<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 16 Test Bias | Principles of Psychological Assessment: With Applied Examples in R</title>
  <meta name="description" content="This is a textbook on Principles of Psychological Assessment. The goal of this book is to provide computational examples to apply principles of psychological assessment into practice." />
  <meta name="generator" content="bookdown 0.41 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 16 Test Bias | Principles of Psychological Assessment: With Applied Examples in R" />
  <meta property="og:type" content="book" />
  <meta property="og:image" content="https://isaactpetersen.github.io/Principles-Psychological-Assessment/Images/cover.png" />
  <meta property="og:description" content="This is a textbook on Principles of Psychological Assessment. The goal of this book is to provide computational examples to apply principles of psychological assessment into practice." />
  <meta name="github-repo" content="isaactpetersen/Principles-Psychological-Assessment" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 16 Test Bias | Principles of Psychological Assessment: With Applied Examples in R" />
  
  <meta name="twitter:description" content="This is a textbook on Principles of Psychological Assessment. The goal of this book is to provide computational examples to apply principles of psychological assessment into practice." />
  <meta name="twitter:image" content="https://isaactpetersen.github.io/Principles-Psychological-Assessment/Images/cover.png" />

<meta name="author" content="Isaac T. Petersen" />


<meta name="date" content="2024-12-22" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="intellectual.html"/>
<link rel="next" href="interview.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/codefolding-lua-1.1/codefolding-lua.css" rel="stylesheet" />
<link href="libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="libs/pagedtable-1.1/js/pagedtable.js"></script>
<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<script async defer src="https://hypothes.is/embed.js"></script>

<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-WZJ45RB1K4"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-WZJ45RB1K4');
</script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>
<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Principles of Psychological Assessment</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Book Cover</a></li>
<li class="chapter" data-level="" data-path="preface.html"><a href="preface.html"><i class="fa fa-check"></i>Preface</a>
<ul>
<li class="chapter" data-level="0.1" data-path="preface.html"><a href="preface.html#openAccess"><i class="fa fa-check"></i><b>0.1</b> Open Access</a></li>
<li class="chapter" data-level="0.2" data-path="preface.html"><a href="preface.html#license"><i class="fa fa-check"></i><b>0.2</b> License</a></li>
<li class="chapter" data-level="0.3" data-path="preface.html"><a href="preface.html#citation"><i class="fa fa-check"></i><b>0.3</b> Citation</a></li>
<li class="chapter" data-level="0.4" data-path="preface.html"><a href="preface.html#accessibility"><i class="fa fa-check"></i><b>0.4</b> Accessibility</a></li>
<li class="chapter" data-level="0.5" data-path="preface.html"><a href="preface.html#contribute"><i class="fa fa-check"></i><b>0.5</b> How to Contribute</a></li>
<li class="chapter" data-level="0.6" data-path="preface.html"><a href="preface.html#acknowledgments"><i class="fa fa-check"></i><b>0.6</b> Acknowledgments</a></li>
</ul></li>
<li class="part"><span><b>I Part I</b></span></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#about"><i class="fa fa-check"></i><b>1.1</b> About This Book</a></li>
<li class="chapter" data-level="1.2" data-path="intro.html"><a href="intro.html#whyR"><i class="fa fa-check"></i><b>1.2</b> Why R?</a></li>
<li class="chapter" data-level="1.3" data-path="intro.html"><a href="intro.html#aboutTheAuthor"><i class="fa fa-check"></i><b>1.3</b> About the Author</a></li>
<li class="chapter" data-level="1.4" data-path="intro.html"><a href="intro.html#whatIsAssessment"><i class="fa fa-check"></i><b>1.4</b> What Is Assessment?</a></li>
<li class="chapter" data-level="1.5" data-path="intro.html"><a href="intro.html#whyCare"><i class="fa fa-check"></i><b>1.5</b> Why Should We Care About Assessment (and Science)?</a>
<ul>
<li class="chapter" data-level="1.5.1" data-path="intro.html"><a href="intro.html#assessmentReplicationCrisis"><i class="fa fa-check"></i><b>1.5.1</b> Assessment and the Replication Crisis in Science</a></li>
<li class="chapter" data-level="1.5.2" data-path="intro.html"><a href="intro.html#scienceVsPseudoscience"><i class="fa fa-check"></i><b>1.5.2</b> Science Versus Pseudoscience in Assessment</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="intro.html"><a href="intro.html#prerequisites"><i class="fa fa-check"></i><b>1.6</b> Prerequisites</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="scoresScales.html"><a href="scoresScales.html"><i class="fa fa-check"></i><b>2</b> Scores and Scales</a>
<ul>
<li class="chapter" data-level="2.1" data-path="scoresScales.html"><a href="scoresScales.html#gettingStarted-scoresScales"><i class="fa fa-check"></i><b>2.1</b> Getting Started</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="scoresScales.html"><a href="scoresScales.html#loadLibraries-scoresScales"><i class="fa fa-check"></i><b>2.1.1</b> Load Libraries</a></li>
<li class="chapter" data-level="2.1.2" data-path="scoresScales.html"><a href="scoresScales.html#prepareData-scoresScales"><i class="fa fa-check"></i><b>2.1.2</b> Prepare Data</a>
<ul>
<li class="chapter" data-level="2.1.2.1" data-path="scoresScales.html"><a href="scoresScales.html#simulateData-scoresScales"><i class="fa fa-check"></i><b>2.1.2.1</b> Simulate Data</a></li>
<li class="chapter" data-level="2.1.2.2" data-path="scoresScales.html"><a href="scoresScales.html#addMissingData-scoresScales"><i class="fa fa-check"></i><b>2.1.2.2</b> Add Missing Data</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="scoresScales.html"><a href="scoresScales.html#dataTypes"><i class="fa fa-check"></i><b>2.2</b> Data Types</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="scoresScales.html"><a href="scoresScales.html#nominalData"><i class="fa fa-check"></i><b>2.2.1</b> Nominal</a></li>
<li class="chapter" data-level="2.2.2" data-path="scoresScales.html"><a href="scoresScales.html#ordinalData"><i class="fa fa-check"></i><b>2.2.2</b> Ordinal</a></li>
<li class="chapter" data-level="2.2.3" data-path="scoresScales.html"><a href="scoresScales.html#intervalData"><i class="fa fa-check"></i><b>2.2.3</b> Interval</a></li>
<li class="chapter" data-level="2.2.4" data-path="scoresScales.html"><a href="scoresScales.html#ratioData"><i class="fa fa-check"></i><b>2.2.4</b> Ratio</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="scoresScales.html"><a href="scoresScales.html#scoreTransformation"><i class="fa fa-check"></i><b>2.3</b> Score Transformation</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="scoresScales.html"><a href="scoresScales.html#rawScores"><i class="fa fa-check"></i><b>2.3.1</b> Raw Scores</a></li>
<li class="chapter" data-level="2.3.2" data-path="scoresScales.html"><a href="scoresScales.html#norm"><i class="fa fa-check"></i><b>2.3.2</b> Norm-Referenced Scores</a>
<ul>
<li class="chapter" data-level="2.3.2.1" data-path="scoresScales.html"><a href="scoresScales.html#percentileRanks"><i class="fa fa-check"></i><b>2.3.2.1</b> Percentile Ranks</a></li>
<li class="chapter" data-level="2.3.2.2" data-path="scoresScales.html"><a href="scoresScales.html#standardizedScores"><i class="fa fa-check"></i><b>2.3.2.2</b> Deviation (Standardized) Scores</a>
<ul>
<li class="chapter" data-level="2.3.2.2.1" data-path="scoresScales.html"><a href="scoresScales.html#zScores"><i class="fa fa-check"></i><b>2.3.2.2.1</b> <em>z</em> scores</a></li>
<li class="chapter" data-level="2.3.2.2.2" data-path="scoresScales.html"><a href="scoresScales.html#tScores"><i class="fa fa-check"></i><b>2.3.2.2.2</b> <em>T</em> scores</a></li>
<li class="chapter" data-level="2.3.2.2.3" data-path="scoresScales.html"><a href="scoresScales.html#standardScores"><i class="fa fa-check"></i><b>2.3.2.2.3</b> Standard scores</a></li>
<li class="chapter" data-level="2.3.2.2.4" data-path="scoresScales.html"><a href="scoresScales.html#scaledScores"><i class="fa fa-check"></i><b>2.3.2.2.4</b> Scaled scores</a></li>
<li class="chapter" data-level="2.3.2.2.5" data-path="scoresScales.html"><a href="scoresScales.html#stanineScores"><i class="fa fa-check"></i><b>2.3.2.2.5</b> Stanine scores</a></li>
</ul></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="scoresScales.html"><a href="scoresScales.html#conclusion-scoresScales"><i class="fa fa-check"></i><b>2.4</b> Conclusion</a></li>
<li class="chapter" data-level="2.5" data-path="scoresScales.html"><a href="scoresScales.html#readings-scoresScales"><i class="fa fa-check"></i><b>2.5</b> Suggested Readings</a></li>
<li class="chapter" data-level="2.6" data-path="scoresScales.html"><a href="scoresScales.html#exercises-scoresScales"><i class="fa fa-check"></i><b>2.6</b> Exercises</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="constructs.html"><a href="constructs.html"><i class="fa fa-check"></i><b>3</b> Constructs</a>
<ul>
<li class="chapter" data-level="3.1" data-path="constructs.html"><a href="constructs.html#constructTypes"><i class="fa fa-check"></i><b>3.1</b> Types of Constructs</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="constructs.html"><a href="constructs.html#reflectiveConstruct"><i class="fa fa-check"></i><b>3.1.1</b> Reflective Construct</a></li>
<li class="chapter" data-level="3.1.2" data-path="constructs.html"><a href="constructs.html#formativeConstruct"><i class="fa fa-check"></i><b>3.1.2</b> Formative Construct</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="constructs.html"><a href="constructs.html#formativeVsReflectiveMeasurementExpectations"><i class="fa fa-check"></i><b>3.2</b> Differences in Measurement Expectations</a></li>
<li class="chapter" data-level="3.3" data-path="constructs.html"><a href="constructs.html#formativeReflectiveIssues"><i class="fa fa-check"></i><b>3.3</b> Practical Issues</a></li>
<li class="chapter" data-level="3.4" data-path="constructs.html"><a href="constructs.html#estimateFormativeReflective"><i class="fa fa-check"></i><b>3.4</b> How To Estimate</a></li>
<li class="chapter" data-level="3.5" data-path="constructs.html"><a href="constructs.html#latentVariableModeling"><i class="fa fa-check"></i><b>3.5</b> Latent Variable Modeling: IRT, SEM, and CFA</a></li>
<li class="chapter" data-level="3.6" data-path="constructs.html"><a href="constructs.html#networkAnalysis"><i class="fa fa-check"></i><b>3.6</b> Alternative Conceptualizations of Phenomena: A Network</a></li>
<li class="chapter" data-level="3.7" data-path="constructs.html"><a href="constructs.html#constructs-conclusion"><i class="fa fa-check"></i><b>3.7</b> Conclusion</a></li>
<li class="chapter" data-level="3.8" data-path="constructs.html"><a href="constructs.html#readings-constructs"><i class="fa fa-check"></i><b>3.8</b> Suggested Readings</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="reliability.html"><a href="reliability.html"><i class="fa fa-check"></i><b>4</b> Reliability</a>
<ul>
<li class="chapter" data-level="4.1" data-path="reliability.html"><a href="reliability.html#ctt"><i class="fa fa-check"></i><b>4.1</b> Classical Test Theory</a>
<ul>
<li class="chapter" data-level="4.1.1" data-path="reliability.html"><a href="reliability.html#cttOverview"><i class="fa fa-check"></i><b>4.1.1</b> Overview</a></li>
<li class="chapter" data-level="4.1.2" data-path="reliability.html"><a href="reliability.html#cttMeasurementModels"><i class="fa fa-check"></i><b>4.1.2</b> Four CTT Measurement Models</a>
<ul>
<li class="chapter" data-level="4.1.2.1" data-path="reliability.html"><a href="reliability.html#cttParallel"><i class="fa fa-check"></i><b>4.1.2.1</b> Parallel</a></li>
<li class="chapter" data-level="4.1.2.2" data-path="reliability.html"><a href="reliability.html#cttTauEquivalent"><i class="fa fa-check"></i><b>4.1.2.2</b> Tau-Equivalent</a></li>
<li class="chapter" data-level="4.1.2.3" data-path="reliability.html"><a href="reliability.html#cttEssentiallyTauEquivalent"><i class="fa fa-check"></i><b>4.1.2.3</b> Essentially Tau-Equivalent</a></li>
<li class="chapter" data-level="4.1.2.4" data-path="reliability.html"><a href="reliability.html#cttCongeneric"><i class="fa fa-check"></i><b>4.1.2.4</b> Congeneric</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="reliability.html"><a href="reliability.html#measurementError"><i class="fa fa-check"></i><b>4.2</b> Measurement Error</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="reliability.html"><a href="reliability.html#systematicError"><i class="fa fa-check"></i><b>4.2.1</b> Systematic (Nonrandom) Error</a></li>
<li class="chapter" data-level="4.2.2" data-path="reliability.html"><a href="reliability.html#randomError"><i class="fa fa-check"></i><b>4.2.2</b> Unsystematic (Random) Error</a></li>
<li class="chapter" data-level="4.2.3" data-path="reliability.html"><a href="reliability.html#withinPersonError"><i class="fa fa-check"></i><b>4.2.3</b> Within-Person Error</a></li>
<li class="chapter" data-level="4.2.4" data-path="reliability.html"><a href="reliability.html#betweenPersonError"><i class="fa fa-check"></i><b>4.2.4</b> Between-Person Error</a></li>
<li class="chapter" data-level="4.2.5" data-path="reliability.html"><a href="reliability.html#measurementErrorTypes"><i class="fa fa-check"></i><b>4.2.5</b> Types of Measurement Error</a>
<ul>
<li class="chapter" data-level="4.2.5.1" data-path="reliability.html"><a href="reliability.html#withinPersonRandomError"><i class="fa fa-check"></i><b>4.2.5.1</b> Within-Person Random Error</a></li>
<li class="chapter" data-level="4.2.5.2" data-path="reliability.html"><a href="reliability.html#withinPersonSystematicError"><i class="fa fa-check"></i><b>4.2.5.2</b> Within-Person Systematic Error</a></li>
<li class="chapter" data-level="4.2.5.3" data-path="reliability.html"><a href="reliability.html#betweenPersonRandomError"><i class="fa fa-check"></i><b>4.2.5.3</b> Between-Person Random Error</a></li>
<li class="chapter" data-level="4.2.5.4" data-path="reliability.html"><a href="reliability.html#betweenPersonSystematicError"><i class="fa fa-check"></i><b>4.2.5.4</b> Between-Person Systematic Error</a></li>
</ul></li>
<li class="chapter" data-level="4.2.6" data-path="reliability.html"><a href="reliability.html#measurementErrorSummary"><i class="fa fa-check"></i><b>4.2.6</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="reliability.html"><a href="reliability.html#overview-reliability"><i class="fa fa-check"></i><b>4.3</b> Overview of Reliability</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="reliability.html"><a href="reliability.html#reliableEnough"><i class="fa fa-check"></i><b>4.3.1</b> How Reliable Is Reliable Enough?</a></li>
<li class="chapter" data-level="4.3.2" data-path="reliability.html"><a href="reliability.html#standardErrorOfMeasurement"><i class="fa fa-check"></i><b>4.3.2</b> Standard Error of Measurement (SEM)</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="reliability.html"><a href="reliability.html#gettingStarted-reliability"><i class="fa fa-check"></i><b>4.4</b> Getting Started</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="reliability.html"><a href="reliability.html#loadLibraries-reliability"><i class="fa fa-check"></i><b>4.4.1</b> Load Libraries</a></li>
<li class="chapter" data-level="4.4.2" data-path="reliability.html"><a href="reliability.html#prepareData-reliability"><i class="fa fa-check"></i><b>4.4.2</b> Prepare Data</a>
<ul>
<li class="chapter" data-level="4.4.2.1" data-path="reliability.html"><a href="reliability.html#simulateData-reliability"><i class="fa fa-check"></i><b>4.4.2.1</b> Simulate Data</a></li>
<li class="chapter" data-level="4.4.2.2" data-path="reliability.html"><a href="reliability.html#addMissingData-reliability"><i class="fa fa-check"></i><b>4.4.2.2</b> Add Missing Data</a></li>
<li class="chapter" data-level="4.4.2.3" data-path="reliability.html"><a href="reliability.html#combineData-reliability"><i class="fa fa-check"></i><b>4.4.2.3</b> Combine Data Into Dataframe</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="reliability.html"><a href="reliability.html#typesOfReliability"><i class="fa fa-check"></i><b>4.5</b> Types of Reliability</a>
<ul>
<li class="chapter" data-level="4.5.1" data-path="reliability.html"><a href="reliability.html#testRetest-reliability"><i class="fa fa-check"></i><b>4.5.1</b> Test–Retest Reliability</a>
<ul>
<li class="chapter" data-level="4.5.1.1" data-path="reliability.html"><a href="reliability.html#stability"><i class="fa fa-check"></i><b>4.5.1.1</b> Coefficient of Stability (and Coefficient of Dependability)</a>
<ul>
<li class="chapter" data-level="4.5.1.1.1" data-path="reliability.html"><a href="reliability.html#correlationConsiderations"><i class="fa fa-check"></i><b>4.5.1.1.1</b> Considerations About the Correlation Coefficient</a></li>
</ul></li>
<li class="chapter" data-level="4.5.1.2" data-path="reliability.html"><a href="reliability.html#agreement"><i class="fa fa-check"></i><b>4.5.1.2</b> Coefficient of Agreement</a></li>
<li class="chapter" data-level="4.5.1.3" data-path="reliability.html"><a href="reliability.html#repeatability"><i class="fa fa-check"></i><b>4.5.1.3</b> Coefficient of Repeatability</a>
<ul>
<li class="chapter" data-level="4.5.1.3.1" data-path="reliability.html"><a href="reliability.html#blandAltmanPlot"><i class="fa fa-check"></i><b>4.5.1.3.1</b> Bland-Altman Plot</a></li>
<li class="chapter" data-level="4.5.1.3.2" data-path="reliability.html"><a href="reliability.html#blandAltmanBias"><i class="fa fa-check"></i><b>4.5.1.3.2</b> Bias</a></li>
<li class="chapter" data-level="4.5.1.3.3" data-path="reliability.html"><a href="reliability.html#loa"><i class="fa fa-check"></i><b>4.5.1.3.3</b> 95% Limits of Agreement</a></li>
<li class="chapter" data-level="4.5.1.3.4" data-path="reliability.html"><a href="reliability.html#repeatabilitySlope"><i class="fa fa-check"></i><b>4.5.1.3.4</b> Regression Slope</a></li>
<li class="chapter" data-level="4.5.1.3.5" data-path="reliability.html"><a href="reliability.html#blandAltmanStats"><i class="fa fa-check"></i><b>4.5.1.3.5</b> Bland-Altman Statistics</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4.5.2" data-path="reliability.html"><a href="reliability.html#interrater-reliability"><i class="fa fa-check"></i><b>4.5.2</b> Inter-Rater Reliability</a>
<ul>
<li class="chapter" data-level="4.5.2.1" data-path="reliability.html"><a href="reliability.html#interraterReliability-continuous"><i class="fa fa-check"></i><b>4.5.2.1</b> Continuous Data</a></li>
<li class="chapter" data-level="4.5.2.2" data-path="reliability.html"><a href="reliability.html#interraterReliability-categorical"><i class="fa fa-check"></i><b>4.5.2.2</b> Categorical Data</a></li>
</ul></li>
<li class="chapter" data-level="4.5.3" data-path="reliability.html"><a href="reliability.html#intrarater-reliability"><i class="fa fa-check"></i><b>4.5.3</b> Intra-Rater Reliability</a></li>
<li class="chapter" data-level="4.5.4" data-path="reliability.html"><a href="reliability.html#parallelForms-reliability"><i class="fa fa-check"></i><b>4.5.4</b> Parallel- (or Alternate-) Forms Reliability</a></li>
<li class="chapter" data-level="4.5.5" data-path="reliability.html"><a href="reliability.html#internalConsistency-reliability"><i class="fa fa-check"></i><b>4.5.5</b> Internal Consistency Reliability</a>
<ul>
<li class="chapter" data-level="4.5.5.1" data-path="reliability.html"><a href="reliability.html#interItemCorrelation-reliability"><i class="fa fa-check"></i><b>4.5.5.1</b> Average Inter-Item Correlation</a></li>
<li class="chapter" data-level="4.5.5.2" data-path="reliability.html"><a href="reliability.html#itemTotalCorrelation-reliability"><i class="fa fa-check"></i><b>4.5.5.2</b> Average Item–Total Correlation</a></li>
<li class="chapter" data-level="4.5.5.3" data-path="reliability.html"><a href="reliability.html#averageVarianceExtracted"><i class="fa fa-check"></i><b>4.5.5.3</b> Average Variance Extracted</a></li>
<li class="chapter" data-level="4.5.5.4" data-path="reliability.html"><a href="reliability.html#splitHalf-reliability"><i class="fa fa-check"></i><b>4.5.5.4</b> Split-Half Reliability</a></li>
<li class="chapter" data-level="4.5.5.5" data-path="reliability.html"><a href="reliability.html#cronbachAlpha"><i class="fa fa-check"></i><b>4.5.5.5</b> Cronbach’s Alpha (Coefficient Alpha)</a></li>
<li class="chapter" data-level="4.5.5.6" data-path="reliability.html"><a href="reliability.html#coefficientOmega"><i class="fa fa-check"></i><b>4.5.5.6</b> Coefficient Omega (<span class="math inline">\(\omega\)</span>)</a>
<ul>
<li class="chapter" data-level="4.5.5.6.1" data-path="reliability.html"><a href="reliability.html#omega-total-omega_t"><i class="fa fa-check"></i><b>4.5.5.6.1</b> Omega Total (<span class="math inline">\(\omega_t\)</span>)</a></li>
<li class="chapter" data-level="4.5.5.6.2" data-path="reliability.html"><a href="reliability.html#omega-hierarchical-omega_h"><i class="fa fa-check"></i><b>4.5.5.6.2</b> Omega Hierarchical (<span class="math inline">\(\omega_h\)</span>)</a></li>
<li class="chapter" data-level="4.5.5.6.3" data-path="reliability.html"><a href="reliability.html#omega-categorical-omega_c"><i class="fa fa-check"></i><b>4.5.5.6.3</b> Omega Categorical (<span class="math inline">\(\omega_C\)</span>)</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4.5.6" data-path="reliability.html"><a href="reliability.html#comparison-reliability"><i class="fa fa-check"></i><b>4.5.6</b> Comparison</a></li>
<li class="chapter" data-level="4.5.7" data-path="reliability.html"><a href="reliability.html#linearComposite-reliability"><i class="fa fa-check"></i><b>4.5.7</b> Reliability of a Linear Composite Score</a></li>
<li class="chapter" data-level="4.5.8" data-path="reliability.html"><a href="reliability.html#differenceScore-reliability"><i class="fa fa-check"></i><b>4.5.8</b> Reliability of a Difference Score</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="reliability.html"><a href="reliability.html#appliedExamples-reliability"><i class="fa fa-check"></i><b>4.6</b> Applied Examples</a></li>
<li class="chapter" data-level="4.7" data-path="reliability.html"><a href="reliability.html#sem-reliability"><i class="fa fa-check"></i><b>4.7</b> Standard Error of Measurement</a></li>
<li class="chapter" data-level="4.8" data-path="reliability.html"><a href="reliability.html#measurementErrorEffects-testRetestReliability"><i class="fa fa-check"></i><b>4.8</b> Influences of Measurement Error on Test–Retest Reliability</a>
<ul>
<li class="chapter" data-level="4.8.1" data-path="reliability.html"><a href="reliability.html#noError"><i class="fa fa-check"></i><b>4.8.1</b> No Error</a>
<ul>
<li class="chapter" data-level="4.8.1.1" data-path="reliability.html"><a href="reliability.html#noError-stability"><i class="fa fa-check"></i><b>4.8.1.1</b> Coefficient of stability</a></li>
<li class="chapter" data-level="4.8.1.2" data-path="reliability.html"><a href="reliability.html#noError-repeatability"><i class="fa fa-check"></i><b>4.8.1.2</b> Coefficient of repeatability</a></li>
<li class="chapter" data-level="4.8.1.3" data-path="reliability.html"><a href="reliability.html#noError-bias"><i class="fa fa-check"></i><b>4.8.1.3</b> Bias</a></li>
</ul></li>
<li class="chapter" data-level="4.8.2" data-path="reliability.html"><a href="reliability.html#withinPersonRandomError-example"><i class="fa fa-check"></i><b>4.8.2</b> Add within-person random error</a>
<ul>
<li class="chapter" data-level="4.8.2.1" data-path="reliability.html"><a href="reliability.html#withinPersonRandomError-stability"><i class="fa fa-check"></i><b>4.8.2.1</b> Coefficient of stability</a></li>
<li class="chapter" data-level="4.8.2.2" data-path="reliability.html"><a href="reliability.html#withinPersonRandomError-repeatability"><i class="fa fa-check"></i><b>4.8.2.2</b> Coefficient of repeatability</a></li>
<li class="chapter" data-level="4.8.2.3" data-path="reliability.html"><a href="reliability.html#withinPersonRandomError-bias"><i class="fa fa-check"></i><b>4.8.2.3</b> Bias</a></li>
<li class="chapter" data-level="4.8.2.4" data-path="reliability.html"><a href="reliability.html#withinPersonRandomError-findings"><i class="fa fa-check"></i><b>4.8.2.4</b> Findings</a></li>
</ul></li>
<li class="chapter" data-level="4.8.3" data-path="reliability.html"><a href="reliability.html#witinPersonSystematicError-example"><i class="fa fa-check"></i><b>4.8.3</b> Add within-person systematic error</a>
<ul>
<li class="chapter" data-level="4.8.3.1" data-path="reliability.html"><a href="reliability.html#withinPersonSystematicError-stability"><i class="fa fa-check"></i><b>4.8.3.1</b> Coefficient of stability</a></li>
<li class="chapter" data-level="4.8.3.2" data-path="reliability.html"><a href="reliability.html#withinPersonSystematicError-repeatability"><i class="fa fa-check"></i><b>4.8.3.2</b> Coefficient of repeatability</a></li>
<li class="chapter" data-level="4.8.3.3" data-path="reliability.html"><a href="reliability.html#withinPersonSystematicError-bias"><i class="fa fa-check"></i><b>4.8.3.3</b> Bias</a></li>
<li class="chapter" data-level="4.8.3.4" data-path="reliability.html"><a href="reliability.html#withinPersonSystematicError-findings"><i class="fa fa-check"></i><b>4.8.3.4</b> Findings</a></li>
</ul></li>
<li class="chapter" data-level="4.8.4" data-path="reliability.html"><a href="reliability.html#betweenPersonRandomError-example"><i class="fa fa-check"></i><b>4.8.4</b> Add between-person random error at T2</a>
<ul>
<li class="chapter" data-level="4.8.4.1" data-path="reliability.html"><a href="reliability.html#betweenPersonRandomError-stability"><i class="fa fa-check"></i><b>4.8.4.1</b> Coefficient of stability</a></li>
<li class="chapter" data-level="4.8.4.2" data-path="reliability.html"><a href="reliability.html#betweenPersonRandomError-repeatability"><i class="fa fa-check"></i><b>4.8.4.2</b> Coefficient of repeatability</a></li>
<li class="chapter" data-level="4.8.4.3" data-path="reliability.html"><a href="reliability.html#betweenPersonRandomError-bias"><i class="fa fa-check"></i><b>4.8.4.3</b> Bias</a></li>
<li class="chapter" data-level="4.8.4.4" data-path="reliability.html"><a href="reliability.html#betweenPersonRandomError-findings"><i class="fa fa-check"></i><b>4.8.4.4</b> Findings</a></li>
</ul></li>
<li class="chapter" data-level="4.8.5" data-path="reliability.html"><a href="reliability.html#betweenPersonSystematicError-example"><i class="fa fa-check"></i><b>4.8.5</b> Add between-person systematic error at T2</a>
<ul>
<li class="chapter" data-level="4.8.5.1" data-path="reliability.html"><a href="reliability.html#betweenPersonSystematicError-stability"><i class="fa fa-check"></i><b>4.8.5.1</b> Coefficient of stability</a></li>
<li class="chapter" data-level="4.8.5.2" data-path="reliability.html"><a href="reliability.html#betweenPersonSystematicError-repeatability"><i class="fa fa-check"></i><b>4.8.5.2</b> Coefficient of repeatability</a></li>
<li class="chapter" data-level="4.8.5.3" data-path="reliability.html"><a href="reliability.html#betweenPersonSystematicError-bias"><i class="fa fa-check"></i><b>4.8.5.3</b> Bias</a></li>
<li class="chapter" data-level="4.8.5.4" data-path="reliability.html"><a href="reliability.html#betweenPersonSystematicError-findings"><i class="fa fa-check"></i><b>4.8.5.4</b> Findings</a></li>
</ul></li>
<li class="chapter" data-level="4.8.6" data-path="reliability.html"><a href="reliability.html#constantVariance-example"><i class="fa fa-check"></i><b>4.8.6</b> Add constant variance of 10 to all participants at all time points</a>
<ul>
<li class="chapter" data-level="4.8.6.1" data-path="reliability.html"><a href="reliability.html#constantVariance-stability"><i class="fa fa-check"></i><b>4.8.6.1</b> Coefficient of stability</a></li>
<li class="chapter" data-level="4.8.6.2" data-path="reliability.html"><a href="reliability.html#constantVariance-repeatability"><i class="fa fa-check"></i><b>4.8.6.2</b> Coefficient of repeatability</a></li>
<li class="chapter" data-level="4.8.6.3" data-path="reliability.html"><a href="reliability.html#constantVariance-bias"><i class="fa fa-check"></i><b>4.8.6.3</b> Bias</a></li>
<li class="chapter" data-level="4.8.6.4" data-path="reliability.html"><a href="reliability.html#constantVariance-findings"><i class="fa fa-check"></i><b>4.8.6.4</b> Findings</a></li>
</ul></li>
<li class="chapter" data-level="4.8.7" data-path="reliability.html"><a href="reliability.html#measurementErrorEffects-testRetestReliability-summary"><i class="fa fa-check"></i><b>4.8.7</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="4.9" data-path="reliability.html"><a href="reliability.html#measurementErrorEffects-associations"><i class="fa fa-check"></i><b>4.9</b> Effect of Measurement Error on Associations</a>
<ul>
<li class="chapter" data-level="4.9.1" data-path="reliability.html"><a href="reliability.html#measurementErrorEffects-attenuationTestRetestReliability"><i class="fa fa-check"></i><b>4.9.1</b> Attenuation of True Test–Retest Reliability Due to Measurement Error</a></li>
<li class="chapter" data-level="4.9.2" data-path="reliability.html"><a href="reliability.html#measurementErrorEffects-disattenuationTestRetestReliability"><i class="fa fa-check"></i><b>4.9.2</b> Disattenuation of Observed Test–Retest Reliability Due to Measurement Error</a></li>
</ul></li>
<li class="chapter" data-level="4.10" data-path="reliability.html"><a href="reliability.html#methodBias"><i class="fa fa-check"></i><b>4.10</b> Method Bias</a>
<ul>
<li class="chapter" data-level="4.10.1" data-path="reliability.html"><a href="reliability.html#methodBias-whyImportant"><i class="fa fa-check"></i><b>4.10.1</b> What Method Biases Are and Why They Are Important</a></li>
<li class="chapter" data-level="4.10.2" data-path="reliability.html"><a href="reliability.html#methodBias-types"><i class="fa fa-check"></i><b>4.10.2</b> Types of Method Biases</a></li>
<li class="chapter" data-level="4.10.3" data-path="reliability.html"><a href="reliability.html#methodBias-remedies"><i class="fa fa-check"></i><b>4.10.3</b> Potential Remedies</a>
<ul>
<li class="chapter" data-level="4.10.3.1" data-path="reliability.html"><a href="reliability.html#randomizedResponseModel"><i class="fa fa-check"></i><b>4.10.3.1</b> Randomized Response Model</a></li>
<li class="chapter" data-level="4.10.3.2" data-path="reliability.html"><a href="reliability.html#mixedModels"><i class="fa fa-check"></i><b>4.10.3.2</b> Mixed Models</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4.11" data-path="reliability.html"><a href="reliability.html#gTheoryReliability"><i class="fa fa-check"></i><b>4.11</b> Generalizability Theory (G-Theory)</a></li>
<li class="chapter" data-level="4.12" data-path="reliability.html"><a href="reliability.html#irt-reliability"><i class="fa fa-check"></i><b>4.12</b> Item Response Theory</a></li>
<li class="chapter" data-level="4.13" data-path="reliability.html"><a href="reliability.html#lowReliabilityProblem"><i class="fa fa-check"></i><b>4.13</b> The Problem of Low Reliability</a></li>
<li class="chapter" data-level="4.14" data-path="reliability.html"><a href="reliability.html#increaseReliability"><i class="fa fa-check"></i><b>4.14</b> Ways to Increase Reliability</a></li>
<li class="chapter" data-level="4.15" data-path="reliability.html"><a href="reliability.html#reliability-conclusion"><i class="fa fa-check"></i><b>4.15</b> Conclusion</a></li>
<li class="chapter" data-level="4.16" data-path="reliability.html"><a href="reliability.html#readings-reliability"><i class="fa fa-check"></i><b>4.16</b> Suggested Readings</a></li>
<li class="chapter" data-level="4.17" data-path="reliability.html"><a href="reliability.html#exercises-reliability"><i class="fa fa-check"></i><b>4.17</b> Exercises</a>
<ul>
<li class="chapter" data-level="4.17.1" data-path="reliability.html"><a href="reliability.html#exercisesQuestions-reliability"><i class="fa fa-check"></i><b>4.17.1</b> Questions</a></li>
<li class="chapter" data-level="4.17.2" data-path="reliability.html"><a href="reliability.html#exercisesAnswers-reliability"><i class="fa fa-check"></i><b>4.17.2</b> Answers</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="validity.html"><a href="validity.html"><i class="fa fa-check"></i><b>5</b> Validity</a>
<ul>
<li class="chapter" data-level="5.1" data-path="validity.html"><a href="validity.html#overview-validity"><i class="fa fa-check"></i><b>5.1</b> Overview of Validity</a></li>
<li class="chapter" data-level="5.2" data-path="validity.html"><a href="validity.html#gettingStarted-validity"><i class="fa fa-check"></i><b>5.2</b> Getting Started</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="validity.html"><a href="validity.html#loadLibraries-validity"><i class="fa fa-check"></i><b>5.2.1</b> Load Libraries</a></li>
<li class="chapter" data-level="5.2.2" data-path="validity.html"><a href="validity.html#prepareData-validity"><i class="fa fa-check"></i><b>5.2.2</b> Prepare Data</a>
<ul>
<li class="chapter" data-level="5.2.2.1" data-path="validity.html"><a href="validity.html#simulateData-validity"><i class="fa fa-check"></i><b>5.2.2.1</b> Simulate Data</a></li>
<li class="chapter" data-level="5.2.2.2" data-path="validity.html"><a href="validity.html#addMissingData-validity"><i class="fa fa-check"></i><b>5.2.2.2</b> Add Missing Data</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="validity.html"><a href="validity.html#ValidityTypes"><i class="fa fa-check"></i><b>5.3</b> Types of Validity</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="validity.html"><a href="validity.html#measurementValidity"><i class="fa fa-check"></i><b>5.3.1</b> Measurement Validity</a>
<ul>
<li class="chapter" data-level="5.3.1.1" data-path="validity.html"><a href="validity.html#faceValidity"><i class="fa fa-check"></i><b>5.3.1.1</b> Face Validity</a></li>
<li class="chapter" data-level="5.3.1.2" data-path="validity.html"><a href="validity.html#contentValidity"><i class="fa fa-check"></i><b>5.3.1.2</b> Content Validity</a></li>
<li class="chapter" data-level="5.3.1.3" data-path="validity.html"><a href="validity.html#criterionValidity"><i class="fa fa-check"></i><b>5.3.1.3</b> Criterion-Related Validity</a>
<ul>
<li class="chapter" data-level="5.3.1.3.1" data-path="validity.html"><a href="validity.html#concurrentValidity"><i class="fa fa-check"></i><b>5.3.1.3.1</b> Concurrent Validity</a></li>
<li class="chapter" data-level="5.3.1.3.2" data-path="validity.html"><a href="validity.html#predictiveValidity"><i class="fa fa-check"></i><b>5.3.1.3.2</b> Predictive Validity</a></li>
<li class="chapter" data-level="5.3.1.3.3" data-path="validity.html"><a href="validity.html#theoryEmpiricism"><i class="fa fa-check"></i><b>5.3.1.3.3</b> Empiricism and Theory</a></li>
</ul></li>
<li class="chapter" data-level="5.3.1.4" data-path="validity.html"><a href="validity.html#constructValidity"><i class="fa fa-check"></i><b>5.3.1.4</b> Construct Validity</a>
<ul>
<li class="chapter" data-level="5.3.1.4.1" data-path="validity.html"><a href="validity.html#nomologicalNetwork"><i class="fa fa-check"></i><b>5.3.1.4.1</b> Nomological Network</a></li>
<li class="chapter" data-level="5.3.1.4.2" data-path="validity.html"><a href="validity.html#MTMM"><i class="fa fa-check"></i><b>5.3.1.4.2</b> Multitrait-Multimethod Matrix (MTMM)</a></li>
</ul></li>
<li class="chapter" data-level="5.3.1.5" data-path="validity.html"><a href="validity.html#incrementalValidity"><i class="fa fa-check"></i><b>5.3.1.5</b> Incremental Validity</a></li>
<li class="chapter" data-level="5.3.1.6" data-path="validity.html"><a href="validity.html#treatmentUtility"><i class="fa fa-check"></i><b>5.3.1.6</b> Treatment Utility of Assessment</a></li>
<li class="chapter" data-level="5.3.1.7" data-path="validity.html"><a href="validity.html#discriminativeValidity"><i class="fa fa-check"></i><b>5.3.1.7</b> Discriminative Validity</a></li>
<li class="chapter" data-level="5.3.1.8" data-path="validity.html"><a href="validity.html#elaborativeValidity"><i class="fa fa-check"></i><b>5.3.1.8</b> Elaborative Validity</a></li>
<li class="chapter" data-level="5.3.1.9" data-path="validity.html"><a href="validity.html#consequentialValidity"><i class="fa fa-check"></i><b>5.3.1.9</b> Consequential Validity</a></li>
<li class="chapter" data-level="5.3.1.10" data-path="validity.html"><a href="validity.html#representationalValidity"><i class="fa fa-check"></i><b>5.3.1.10</b> Representational Validity</a></li>
<li class="chapter" data-level="5.3.1.11" data-path="validity.html"><a href="validity.html#factorialValidity"><i class="fa fa-check"></i><b>5.3.1.11</b> Factorial (Structural) Validity</a></li>
<li class="chapter" data-level="5.3.1.12" data-path="validity.html"><a href="validity.html#ecologicalValidity"><i class="fa fa-check"></i><b>5.3.1.12</b> Ecological Validity</a></li>
<li class="chapter" data-level="5.3.1.13" data-path="validity.html"><a href="validity.html#processFocusedValidity"><i class="fa fa-check"></i><b>5.3.1.13</b> Process-Focused Validity</a></li>
<li class="chapter" data-level="5.3.1.14" data-path="validity.html"><a href="validity.html#diagnosticValidity"><i class="fa fa-check"></i><b>5.3.1.14</b> Diagnostic Validity</a></li>
<li class="chapter" data-level="5.3.1.15" data-path="validity.html"><a href="validity.html#socialValidity"><i class="fa fa-check"></i><b>5.3.1.15</b> Social Validity</a></li>
<li class="chapter" data-level="5.3.1.16" data-path="validity.html"><a href="validity.html#culturalValidity"><i class="fa fa-check"></i><b>5.3.1.16</b> Cultural Validity</a></li>
</ul></li>
<li class="chapter" data-level="5.3.2" data-path="validity.html"><a href="validity.html#researchDesignValidity"><i class="fa fa-check"></i><b>5.3.2</b> Research Design (Experimental) Validity</a>
<ul>
<li class="chapter" data-level="5.3.2.1" data-path="validity.html"><a href="validity.html#internalValidity"><i class="fa fa-check"></i><b>5.3.2.1</b> Internal Validity</a></li>
<li class="chapter" data-level="5.3.2.2" data-path="validity.html"><a href="validity.html#externalValidity"><i class="fa fa-check"></i><b>5.3.2.2</b> External Validity</a>
<ul>
<li class="chapter" data-level="5.3.2.2.1" data-path="validity.html"><a href="validity.html#internalExternalValidityTradeoffs"><i class="fa fa-check"></i><b>5.3.2.2.1</b> Tradeoffs of Internal Validity and External Validity</a></li>
</ul></li>
<li class="chapter" data-level="5.3.2.3" data-path="validity.html"><a href="validity.html#conclusionValidity"><i class="fa fa-check"></i><b>5.3.2.3</b> (Statistical) Conclusion Validity</a></li>
</ul></li>
<li class="chapter" data-level="5.3.3" data-path="validity.html"><a href="validity.html#organizationalFramework-validity"><i class="fa fa-check"></i><b>5.3.3</b> Putting It All Together: An Organizational Framework</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="validity.html"><a href="validity.html#validityAsAProcess"><i class="fa fa-check"></i><b>5.4</b> Validity Is a Process, Not an Outcome</a></li>
<li class="chapter" data-level="5.5" data-path="validity.html"><a href="validity.html#reliabilityVsValidity"><i class="fa fa-check"></i><b>5.5</b> Reliability Versus Validity</a></li>
<li class="chapter" data-level="5.6" data-path="validity.html"><a href="validity.html#effectOfMeasurementErrorOnAssociations"><i class="fa fa-check"></i><b>5.6</b> Effect of Measurement Error on Associations</a>
<ul>
<li class="chapter" data-level="5.6.1" data-path="validity.html"><a href="validity.html#measurementErrorEffects-simulatedData"><i class="fa fa-check"></i><b>5.6.1</b> Test with Simulated Data</a>
<ul>
<li class="chapter" data-level="5.6.1.1" data-path="validity.html"><a href="validity.html#measurementErrorEffects-reliabilityPredictor"><i class="fa fa-check"></i><b>5.6.1.1</b> Reliability of Predictor</a></li>
<li class="chapter" data-level="5.6.1.2" data-path="validity.html"><a href="validity.html#measurementErrorEffects-reliabilityCriterion"><i class="fa fa-check"></i><b>5.6.1.2</b> Reliability of Criterion</a></li>
<li class="chapter" data-level="5.6.1.3" data-path="validity.html"><a href="validity.html#measurementErrorEffects-trueAssociation"><i class="fa fa-check"></i><b>5.6.1.3</b> True Association</a></li>
<li class="chapter" data-level="5.6.1.4" data-path="validity.html"><a href="validity.html#measurementErrorEffects-observedAssociation"><i class="fa fa-check"></i><b>5.6.1.4</b> Observed Association (After Adding Measurement Error)</a></li>
</ul></li>
<li class="chapter" data-level="5.6.2" data-path="validity.html"><a href="validity.html#attenuation"><i class="fa fa-check"></i><b>5.6.2</b> Attenuation of True Correlation Due to Measurement Error</a></li>
<li class="chapter" data-level="5.6.3" data-path="validity.html"><a href="validity.html#disattenuation"><i class="fa fa-check"></i><b>5.6.3</b> Disattenuation of Observed Correlation Due to Measurement Error</a></li>
</ul></li>
<li class="chapter" data-level="5.7" data-path="validity.html"><a href="validity.html#gTheoryValidity"><i class="fa fa-check"></i><b>5.7</b> Generalizability Theory (G-Theory)</a></li>
<li class="chapter" data-level="5.8" data-path="validity.html"><a href="validity.html#ways-to-increase-validity"><i class="fa fa-check"></i><b>5.8</b> Ways to Increase Validity</a></li>
<li class="chapter" data-level="5.9" data-path="validity.html"><a href="validity.html#conclusion-validity"><i class="fa fa-check"></i><b>5.9</b> Conclusion</a></li>
<li class="chapter" data-level="5.10" data-path="validity.html"><a href="validity.html#readings-validity"><i class="fa fa-check"></i><b>5.10</b> Suggested Readings</a></li>
<li class="chapter" data-level="5.11" data-path="validity.html"><a href="validity.html#exercises-validity"><i class="fa fa-check"></i><b>5.11</b> Exercises</a>
<ul>
<li class="chapter" data-level="5.11.1" data-path="validity.html"><a href="validity.html#exercisesQuestions-validity"><i class="fa fa-check"></i><b>5.11.1</b> Questions</a></li>
<li class="chapter" data-level="5.11.2" data-path="validity.html"><a href="validity.html#exercisesAnswers-validity"><i class="fa fa-check"></i><b>5.11.2</b> Answers</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="gTheory.html"><a href="gTheory.html"><i class="fa fa-check"></i><b>6</b> Generalizability Theory</a>
<ul>
<li class="chapter" data-level="6.1" data-path="gTheory.html"><a href="gTheory.html#overview-gTheory"><i class="fa fa-check"></i><b>6.1</b> Overview of Generalizability Theory (G-Theory)</a>
<ul>
<li class="chapter" data-level="6.1.1" data-path="gTheory.html"><a href="gTheory.html#universeGeneralizability"><i class="fa fa-check"></i><b>6.1.1</b> The Universe of Generalizability</a></li>
<li class="chapter" data-level="6.1.2" data-path="gTheory.html"><a href="gTheory.html#universeScore"><i class="fa fa-check"></i><b>6.1.2</b> Universe Score</a></li>
<li class="chapter" data-level="6.1.3" data-path="gTheory.html"><a href="gTheory.html#reliability-gTheory"><i class="fa fa-check"></i><b>6.1.3</b> G-Theory Perspective on Reliability</a></li>
<li class="chapter" data-level="6.1.4" data-path="gTheory.html"><a href="gTheory.html#validity-gTheory"><i class="fa fa-check"></i><b>6.1.4</b> G-Theory Perspective on Validity</a></li>
<li class="chapter" data-level="6.1.5" data-path="gTheory.html"><a href="gTheory.html#generalizabilityStudy"><i class="fa fa-check"></i><b>6.1.5</b> Generalizability Study</a></li>
<li class="chapter" data-level="6.1.6" data-path="gTheory.html"><a href="gTheory.html#decisionStudy"><i class="fa fa-check"></i><b>6.1.6</b> Decision Study</a></li>
<li class="chapter" data-level="6.1.7" data-path="gTheory.html"><a href="gTheory.html#analysis-gTheory"><i class="fa fa-check"></i><b>6.1.7</b> Analysis Approach</a></li>
<li class="chapter" data-level="6.1.8" data-path="gTheory.html"><a href="gTheory.html#challenges-gTheory"><i class="fa fa-check"></i><b>6.1.8</b> Practical Challenges</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="gTheory.html"><a href="gTheory.html#gettingStarted-gTheory"><i class="fa fa-check"></i><b>6.2</b> Getting Started</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="gTheory.html"><a href="gTheory.html#loadLibraries-gTheory"><i class="fa fa-check"></i><b>6.2.1</b> Load Libraries</a></li>
<li class="chapter" data-level="6.2.2" data-path="gTheory.html"><a href="gTheory.html#prepareData-gTheory"><i class="fa fa-check"></i><b>6.2.2</b> Prepare Data</a>
<ul>
<li class="chapter" data-level="6.2.2.1" data-path="gTheory.html"><a href="gTheory.html#generateData-gTheory"><i class="fa fa-check"></i><b>6.2.2.1</b> Generate Data</a></li>
<li class="chapter" data-level="6.2.2.2" data-path="gTheory.html"><a href="gTheory.html#addMissingData-gTheory"><i class="fa fa-check"></i><b>6.2.2.2</b> Add Missing Data</a></li>
<li class="chapter" data-level="6.2.2.3" data-path="gTheory.html"><a href="gTheory.html#combineData-gTheory"><i class="fa fa-check"></i><b>6.2.2.3</b> Combine Data into Dataframe</a></li>
</ul></li>
<li class="chapter" data-level="6.2.3" data-path="gTheory.html"><a href="gTheory.html#universeScore-example"><i class="fa fa-check"></i><b>6.2.3</b> Universe Score for Each Person</a></li>
<li class="chapter" data-level="6.2.4" data-path="gTheory.html"><a href="gTheory.html#gStudy-example"><i class="fa fa-check"></i><b>6.2.4</b> Generalizability (G) Study</a>
<ul>
<li class="chapter" data-level="6.2.4.1" data-path="gTheory.html"><a href="gTheory.html#gTheoryANOVA"><i class="fa fa-check"></i><b>6.2.4.1</b> ANOVA Framework</a></li>
<li class="chapter" data-level="6.2.4.2" data-path="gTheory.html"><a href="gTheory.html#gTheoryMixedModel"><i class="fa fa-check"></i><b>6.2.4.2</b> Mixed Model Framework</a></li>
</ul></li>
<li class="chapter" data-level="6.2.5" data-path="gTheory.html"><a href="gTheory.html#dStudy-example"><i class="fa fa-check"></i><b>6.2.5</b> Decision (D) Study</a></li>
<li class="chapter" data-level="6.2.6" data-path="gTheory.html"><a href="gTheory.html#generalizabilityCoefficient"><i class="fa fa-check"></i><b>6.2.6</b> Generalizability Coefficient</a></li>
<li class="chapter" data-level="6.2.7" data-path="gTheory.html"><a href="gTheory.html#dependabilityCoefficient"><i class="fa fa-check"></i><b>6.2.7</b> Dependability Coefficient</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="gTheory.html"><a href="gTheory.html#conclusion-gTheory"><i class="fa fa-check"></i><b>6.3</b> Conclusion</a></li>
<li class="chapter" data-level="6.4" data-path="gTheory.html"><a href="gTheory.html#readings-gTheory"><i class="fa fa-check"></i><b>6.4</b> Suggested Readings</a></li>
<li class="chapter" data-level="6.5" data-path="gTheory.html"><a href="gTheory.html#exercises-gTheory"><i class="fa fa-check"></i><b>6.5</b> Exercises</a>
<ul>
<li class="chapter" data-level="6.5.1" data-path="gTheory.html"><a href="gTheory.html#exercisesQuestions-gTheory"><i class="fa fa-check"></i><b>6.5.1</b> Questions</a></li>
<li class="chapter" data-level="6.5.2" data-path="gTheory.html"><a href="gTheory.html#exercisesAnswers-gTheory"><i class="fa fa-check"></i><b>6.5.2</b> Answers</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="sem.html"><a href="sem.html"><i class="fa fa-check"></i><b>7</b> Structural Equation Modeling</a>
<ul>
<li class="chapter" data-level="7.1" data-path="sem.html"><a href="sem.html#overview-sem"><i class="fa fa-check"></i><b>7.1</b> Overview of SEM</a></li>
<li class="chapter" data-level="7.2" data-path="sem.html"><a href="sem.html#gettingStarted-sem"><i class="fa fa-check"></i><b>7.2</b> Getting Started</a>
<ul>
<li class="chapter" data-level="7.2.1" data-path="sem.html"><a href="sem.html#loadLibraries-sem"><i class="fa fa-check"></i><b>7.2.1</b> Load Libraries</a></li>
<li class="chapter" data-level="7.2.2" data-path="sem.html"><a href="sem.html#prepareData-sem"><i class="fa fa-check"></i><b>7.2.2</b> Prepare Data</a>
<ul>
<li class="chapter" data-level="7.2.2.1" data-path="sem.html"><a href="sem.html#simulateData-sem"><i class="fa fa-check"></i><b>7.2.2.1</b> Simulate Data</a></li>
<li class="chapter" data-level="7.2.2.2" data-path="sem.html"><a href="sem.html#addMissingData-sem"><i class="fa fa-check"></i><b>7.2.2.2</b> Add Missing Data</a></li>
<li class="chapter" data-level="7.2.2.3" data-path="sem.html"><a href="sem.html#combineData-sem"><i class="fa fa-check"></i><b>7.2.2.3</b> Combine Data into Dataframe</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="sem.html"><a href="sem.html#modelTypes-sem"><i class="fa fa-check"></i><b>7.3</b> Types of Models</a>
<ul>
<li class="chapter" data-level="7.3.1" data-path="sem.html"><a href="sem.html#pathAnalysis-sem"><i class="fa fa-check"></i><b>7.3.1</b> Path Analysis Model</a></li>
<li class="chapter" data-level="7.3.2" data-path="sem.html"><a href="sem.html#semModelComponents"><i class="fa fa-check"></i><b>7.3.2</b> Components of a Structural Equation Model</a>
<ul>
<li class="chapter" data-level="7.3.2.1" data-path="sem.html"><a href="sem.html#measurementModel-sem"><i class="fa fa-check"></i><b>7.3.2.1</b> Measurement Model</a></li>
<li class="chapter" data-level="7.3.2.2" data-path="sem.html"><a href="sem.html#structuralModel-sem"><i class="fa fa-check"></i><b>7.3.2.2</b> Structural Model</a></li>
</ul></li>
<li class="chapter" data-level="7.3.3" data-path="sem.html"><a href="sem.html#cfa-sem"><i class="fa fa-check"></i><b>7.3.3</b> Confirmatory Factor Analysis Model</a></li>
<li class="chapter" data-level="7.3.4" data-path="sem.html"><a href="sem.html#semModel"><i class="fa fa-check"></i><b>7.3.4</b> Structural Equation Model</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="sem.html"><a href="sem.html#formativeReflective-sem"><i class="fa fa-check"></i><b>7.4</b> Estimating Latent Factors</a>
<ul>
<li class="chapter" data-level="7.4.1" data-path="sem.html"><a href="sem.html#modelIdentification-sem"><i class="fa fa-check"></i><b>7.4.1</b> Model Identification</a>
<ul>
<li class="chapter" data-level="7.4.1.1" data-path="sem.html"><a href="sem.html#modelIdentificationTypes-sem"><i class="fa fa-check"></i><b>7.4.1.1</b> Types of Model Identification</a></li>
<li class="chapter" data-level="7.4.1.2" data-path="sem.html"><a href="sem.html#modelIdentificationApproaches-sem"><i class="fa fa-check"></i><b>7.4.1.2</b> Approaches to Model Identification</a>
<ul>
<li class="chapter" data-level="7.4.1.2.1" data-path="sem.html"><a href="sem.html#markerVariable-sem"><i class="fa fa-check"></i><b>7.4.1.2.1</b> Marker Variable Method</a></li>
<li class="chapter" data-level="7.4.1.2.2" data-path="sem.html"><a href="sem.html#effectsCoding-sem"><i class="fa fa-check"></i><b>7.4.1.2.2</b> Effects Coding Method</a></li>
<li class="chapter" data-level="7.4.1.2.3" data-path="sem.html"><a href="sem.html#standardizedLatent-sem"><i class="fa fa-check"></i><b>7.4.1.2.3</b> Standardized Latent Factor Method</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7.4.2" data-path="sem.html"><a href="sem.html#latentFactorTypes-sem"><i class="fa fa-check"></i><b>7.4.2</b> Types of Latent Factors</a>
<ul>
<li class="chapter" data-level="7.4.2.1" data-path="sem.html"><a href="sem.html#reflectiveFactors-sem"><i class="fa fa-check"></i><b>7.4.2.1</b> Reflective Latent Factors</a></li>
<li class="chapter" data-level="7.4.2.2" data-path="sem.html"><a href="sem.html#formativeFactors-sem"><i class="fa fa-check"></i><b>7.4.2.2</b> Formative Latent Factors</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7.5" data-path="sem.html"><a href="sem.html#additionalSEMmodels"><i class="fa fa-check"></i><b>7.5</b> Additional Types of SEM</a>
<ul>
<li class="chapter" data-level="7.5.1" data-path="sem.html"><a href="sem.html#latentClassModels"><i class="fa fa-check"></i><b>7.5.1</b> Latent Class Models</a></li>
<li class="chapter" data-level="7.5.2" data-path="sem.html"><a href="sem.html#MixtureModels"><i class="fa fa-check"></i><b>7.5.2</b> Mixture Models</a></li>
<li class="chapter" data-level="7.5.3" data-path="sem.html"><a href="sem.html#esemModels"><i class="fa fa-check"></i><b>7.5.3</b> Exploratory Structural Equation Models</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="sem.html"><a href="sem.html#dag"><i class="fa fa-check"></i><b>7.6</b> Causal Diagrams: Directed Acyclic Graphs</a></li>
<li class="chapter" data-level="7.7" data-path="sem.html"><a href="sem.html#modelFitIndices-sem"><i class="fa fa-check"></i><b>7.7</b> Model Fit Indices</a></li>
<li class="chapter" data-level="7.8" data-path="sem.html"><a href="sem.html#correlationMatrix-sem"><i class="fa fa-check"></i><b>7.8</b> Correlation Matrix</a></li>
<li class="chapter" data-level="7.9" data-path="sem.html"><a href="sem.html#measurementModelExample-sem"><i class="fa fa-check"></i><b>7.9</b> Measurement Model (of a Given Construct)</a>
<ul>
<li class="chapter" data-level="7.9.1" data-path="sem.html"><a href="sem.html#measurementModelSyntax-sem"><i class="fa fa-check"></i><b>7.9.1</b> Specify the Model</a>
<ul>
<li class="chapter" data-level="7.9.1.1" data-path="sem.html"><a href="sem.html#measurementModelSummary-sem"><i class="fa fa-check"></i><b>7.9.1.1</b> Summary of Model Features</a></li>
<li class="chapter" data-level="7.9.1.2" data-path="sem.html"><a href="sem.html#measurementModelTabular-sem"><i class="fa fa-check"></i><b>7.9.1.2</b> Model Syntax in Table Form:</a></li>
</ul></li>
<li class="chapter" data-level="7.9.2" data-path="sem.html"><a href="sem.html#measurementModelFit-sem"><i class="fa fa-check"></i><b>7.9.2</b> Fit the Model</a></li>
<li class="chapter" data-level="7.9.3" data-path="sem.html"><a href="sem.html#measurementModelOutput-sem"><i class="fa fa-check"></i><b>7.9.3</b> Display Summary Output</a></li>
<li class="chapter" data-level="7.9.4" data-path="sem.html"><a href="sem.html#measurementModelFitCriteria-sem"><i class="fa fa-check"></i><b>7.9.4</b> Estimates of Model Fit</a></li>
<li class="chapter" data-level="7.9.5" data-path="sem.html"><a href="sem.html#measurementModelResiduals-sem"><i class="fa fa-check"></i><b>7.9.5</b> Residuals</a></li>
<li class="chapter" data-level="7.9.6" data-path="sem.html"><a href="sem.html#measurementModelModIndices-sem"><i class="fa fa-check"></i><b>7.9.6</b> Modification Indices</a></li>
<li class="chapter" data-level="7.9.7" data-path="sem.html"><a href="sem.html#measurementModelFactorScores-sem"><i class="fa fa-check"></i><b>7.9.7</b> Factor Scores</a></li>
<li class="chapter" data-level="7.9.8" data-path="sem.html"><a href="sem.html#measurementModelReliability-sem"><i class="fa fa-check"></i><b>7.9.8</b> Internal Consistency Reliability</a></li>
<li class="chapter" data-level="7.9.9" data-path="sem.html"><a href="sem.html#measurementModelPathDiagram-sem"><i class="fa fa-check"></i><b>7.9.9</b> Path Diagram</a></li>
</ul></li>
<li class="chapter" data-level="7.10" data-path="sem.html"><a href="sem.html#cfaExample-sem"><i class="fa fa-check"></i><b>7.10</b> Confirmatory Factor Analysis (CFA)</a>
<ul>
<li class="chapter" data-level="7.10.1" data-path="sem.html"><a href="sem.html#cfaModelSyntax-sem"><i class="fa fa-check"></i><b>7.10.1</b> Specify the Model</a>
<ul>
<li class="chapter" data-level="7.10.1.1" data-path="sem.html"><a href="sem.html#cfaModelTabular-sem"><i class="fa fa-check"></i><b>7.10.1.1</b> Model Syntax in Table Form:</a></li>
</ul></li>
<li class="chapter" data-level="7.10.2" data-path="sem.html"><a href="sem.html#cfaModelFit-sem"><i class="fa fa-check"></i><b>7.10.2</b> Fit the Model</a></li>
<li class="chapter" data-level="7.10.3" data-path="sem.html"><a href="sem.html#cfaModelOutput-sem"><i class="fa fa-check"></i><b>7.10.3</b> Display Summary Output</a></li>
<li class="chapter" data-level="7.10.4" data-path="sem.html"><a href="sem.html#cfaModelFitCriteria-sem"><i class="fa fa-check"></i><b>7.10.4</b> Estimates of Model Fit</a></li>
<li class="chapter" data-level="7.10.5" data-path="sem.html"><a href="sem.html#cfaModelResiduals-sem"><i class="fa fa-check"></i><b>7.10.5</b> Residuals</a></li>
<li class="chapter" data-level="7.10.6" data-path="sem.html"><a href="sem.html#cfaModelModIndices-sem"><i class="fa fa-check"></i><b>7.10.6</b> Modification Indices</a></li>
<li class="chapter" data-level="7.10.7" data-path="sem.html"><a href="sem.html#cfaModelFactorScores-sem"><i class="fa fa-check"></i><b>7.10.7</b> Factor Scores</a></li>
<li class="chapter" data-level="7.10.8" data-path="sem.html"><a href="sem.html#cfaModelReliability-sem"><i class="fa fa-check"></i><b>7.10.8</b> Internal Consistency Reliability</a></li>
<li class="chapter" data-level="7.10.9" data-path="sem.html"><a href="sem.html#cfaModelPathDiagram-sem"><i class="fa fa-check"></i><b>7.10.9</b> Path Diagram</a></li>
<li class="chapter" data-level="7.10.10" data-path="sem.html"><a href="sem.html#cfaModelModification-sem"><i class="fa fa-check"></i><b>7.10.10</b> Modify Model Based on Modification Indices</a>
<ul>
<li class="chapter" data-level="7.10.10.1" data-path="sem.html"><a href="sem.html#cfaModelModifiedSyntax-sem"><i class="fa fa-check"></i><b>7.10.10.1</b> Specify the Model</a></li>
<li class="chapter" data-level="7.10.10.2" data-path="sem.html"><a href="sem.html#cfaModelModifiedFit-sem"><i class="fa fa-check"></i><b>7.10.10.2</b> Fit the Model</a></li>
<li class="chapter" data-level="7.10.10.3" data-path="sem.html"><a href="sem.html#cfaModelModifiedOutput-sem"><i class="fa fa-check"></i><b>7.10.10.3</b> Display Summary Output</a></li>
<li class="chapter" data-level="7.10.10.4" data-path="sem.html"><a href="sem.html#cfaModelModifiedFitCriteria-sem"><i class="fa fa-check"></i><b>7.10.10.4</b> Estimates of Model Fit</a></li>
<li class="chapter" data-level="7.10.10.5" data-path="sem.html"><a href="sem.html#cfaModelModifiedResiduals-sem"><i class="fa fa-check"></i><b>7.10.10.5</b> Residuals</a></li>
<li class="chapter" data-level="7.10.10.6" data-path="sem.html"><a href="sem.html#cfaModelModifiedPathDiagram-sem"><i class="fa fa-check"></i><b>7.10.10.6</b> Path Diagram</a></li>
<li class="chapter" data-level="7.10.10.7" data-path="sem.html"><a href="sem.html#nestedModelComparison-sem"><i class="fa fa-check"></i><b>7.10.10.7</b> Compare Model Fit</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7.11" data-path="sem.html"><a href="sem.html#semModelExample-sem"><i class="fa fa-check"></i><b>7.11</b> Structural Equation Model (SEM)</a>
<ul>
<li class="chapter" data-level="7.11.1" data-path="sem.html"><a href="sem.html#semModelSyntax-sem"><i class="fa fa-check"></i><b>7.11.1</b> Specify the Model</a>
<ul>
<li class="chapter" data-level="7.11.1.1" data-path="sem.html"><a href="sem.html#semModelTabular-sem"><i class="fa fa-check"></i><b>7.11.1.1</b> Model Syntax in Table Form:</a></li>
</ul></li>
<li class="chapter" data-level="7.11.2" data-path="sem.html"><a href="sem.html#semModelFit-sem"><i class="fa fa-check"></i><b>7.11.2</b> Fit the Model</a></li>
<li class="chapter" data-level="7.11.3" data-path="sem.html"><a href="sem.html#semModelOutput-sem"><i class="fa fa-check"></i><b>7.11.3</b> Display Summary Output</a>
<ul>
<li class="chapter" data-level="7.11.3.1" data-path="sem.html"><a href="sem.html#interpreting-lavaan-output"><i class="fa fa-check"></i><b>7.11.3.1</b> Interpreting <code>lavaan</code> Output</a></li>
</ul></li>
<li class="chapter" data-level="7.11.4" data-path="sem.html"><a href="sem.html#semModelFitCriteria-sem"><i class="fa fa-check"></i><b>7.11.4</b> Estimates of Model Fit</a></li>
<li class="chapter" data-level="7.11.5" data-path="sem.html"><a href="sem.html#semModelResiduals-sem"><i class="fa fa-check"></i><b>7.11.5</b> Residuals</a></li>
<li class="chapter" data-level="7.11.6" data-path="sem.html"><a href="sem.html#semModelModIndices-sem"><i class="fa fa-check"></i><b>7.11.6</b> Modification Indices</a></li>
<li class="chapter" data-level="7.11.7" data-path="sem.html"><a href="sem.html#semModelFactorScores-sem"><i class="fa fa-check"></i><b>7.11.7</b> Factor Scores</a></li>
<li class="chapter" data-level="7.11.8" data-path="sem.html"><a href="sem.html#semModelReliability-sem"><i class="fa fa-check"></i><b>7.11.8</b> Internal Consistency Reliability</a></li>
<li class="chapter" data-level="7.11.9" data-path="sem.html"><a href="sem.html#semModelPathDiagram-sem"><i class="fa fa-check"></i><b>7.11.9</b> Path Diagram</a></li>
</ul></li>
<li class="chapter" data-level="7.12" data-path="sem.html"><a href="sem.html#benefits-sem"><i class="fa fa-check"></i><b>7.12</b> Benefits of SEM</a>
<ul>
<li class="chapter" data-level="7.12.1" data-path="sem.html"><a href="sem.html#semMethodBias"><i class="fa fa-check"></i><b>7.12.1</b> Accounting for Method Bias</a></li>
</ul></li>
<li class="chapter" data-level="7.13" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis"><i class="fa fa-check"></i><b>7.13</b> Power Analysis Using Monte Carlo Simulation</a>
<ul>
<li class="chapter" data-level="7.13.1" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-populationModel"><i class="fa fa-check"></i><b>7.13.1</b> Specify Population Model</a>
<ul>
<li class="chapter" data-level="7.13.1.1" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-fixedDefaultValues"><i class="fa fa-check"></i><b>7.13.1.1</b> Show the Model’s Fixed and Default Values</a></li>
<li class="chapter" data-level="7.13.1.2" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-covarianceCorrelationMatrix"><i class="fa fa-check"></i><b>7.13.1.2</b> Model-Implied Covariance and Correlation Matrix</a>
<ul>
<li class="chapter" data-level="7.13.1.2.1" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-covarianceMatrix"><i class="fa fa-check"></i><b>7.13.1.2.1</b> Model-implied covariance matrix:</a></li>
<li class="chapter" data-level="7.13.1.2.2" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-correlationMatrix"><i class="fa fa-check"></i><b>7.13.1.2.2</b> Model-implied correlation matrix</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7.13.2" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-analysisModel"><i class="fa fa-check"></i><b>7.13.2</b> Specify Analysis Model</a></li>
<li class="chapter" data-level="7.13.3" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-dataDistribution"><i class="fa fa-check"></i><b>7.13.3</b> Specify Distribution of Data</a></li>
<li class="chapter" data-level="7.13.4" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-missingness"><i class="fa fa-check"></i><b>7.13.4</b> Specify Extent and Type of Missing Data</a>
<ul>
<li class="chapter" data-level="7.13.4.1" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-missingData"><i class="fa fa-check"></i><b>7.13.4.1</b> Specify Missingness</a></li>
<li class="chapter" data-level="7.13.4.2" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-missingDataPlot"><i class="fa fa-check"></i><b>7.13.4.2</b> Plot of Extent of Missing Data Specified</a></li>
</ul></li>
<li class="chapter" data-level="7.13.5" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-sampleSizeReps"><i class="fa fa-check"></i><b>7.13.5</b> Specify Sample Sizes and Repetitions</a></li>
<li class="chapter" data-level="7.13.6" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-run"><i class="fa fa-check"></i><b>7.13.6</b> Monte Carlo Simulation to Generate Data from the Population Parameter Values</a></li>
<li class="chapter" data-level="7.13.7" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-outputPopulationModel"><i class="fa fa-check"></i><b>7.13.7</b> Summary of Population Model</a></li>
<li class="chapter" data-level="7.13.8" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-outputSimulatedData"><i class="fa fa-check"></i><b>7.13.8</b> Summary of Simulated Data</a></li>
<li class="chapter" data-level="7.13.9" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-parameterSummary"><i class="fa fa-check"></i><b>7.13.9</b> Summary of Parameters</a></li>
<li class="chapter" data-level="7.13.10" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-completionTime"><i class="fa fa-check"></i><b>7.13.10</b> Time to Completion</a></li>
<li class="chapter" data-level="7.13.11" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-fitIndicesCutoffs"><i class="fa fa-check"></i><b>7.13.11</b> Cutoffs of Fit Indices</a>
<ul>
<li class="chapter" data-level="7.13.11.1" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-fitIndicesCutoffsPlot"><i class="fa fa-check"></i><b>7.13.11.1</b> Plot of Cutoffs of Fit Indices</a></li>
<li class="chapter" data-level="7.13.11.2" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-fitIndicesCutoffsSampleSize"><i class="fa fa-check"></i><b>7.13.11.2</b> Cutoffs of Fit Indices at Particular Sample Size</a></li>
</ul></li>
<li class="chapter" data-level="7.13.12" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-power"><i class="fa fa-check"></i><b>7.13.12</b> Statistical Power</a>
<ul>
<li class="chapter" data-level="7.13.12.1" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-powerPlot"><i class="fa fa-check"></i><b>7.13.12.1</b> Plot of Power to Detect Various Parameters as a Function of Sample Size</a></li>
<li class="chapter" data-level="7.13.12.2" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-sampleSizeNeeded"><i class="fa fa-check"></i><b>7.13.12.2</b> Sample Size Needed to Detect a Given Parameter</a></li>
<li class="chapter" data-level="7.13.12.3" data-path="sem.html"><a href="sem.html#monteCarloPowerAnalysis-powerSampleSize"><i class="fa fa-check"></i><b>7.13.12.3</b> Power to Detect Each Parameter at a Given Sample Size</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7.14" data-path="sem.html"><a href="sem.html#generalizability-SEM"><i class="fa fa-check"></i><b>7.14</b> Generalizability Theory</a></li>
<li class="chapter" data-level="7.15" data-path="sem.html"><a href="sem.html#conclusion-sem"><i class="fa fa-check"></i><b>7.15</b> Conclusion</a></li>
<li class="chapter" data-level="7.16" data-path="sem.html"><a href="sem.html#readings-sem"><i class="fa fa-check"></i><b>7.16</b> Suggested Readings</a></li>
<li class="chapter" data-level="7.17" data-path="sem.html"><a href="sem.html#semExercises"><i class="fa fa-check"></i><b>7.17</b> Exercises</a>
<ul>
<li class="chapter" data-level="7.17.1" data-path="sem.html"><a href="sem.html#questions"><i class="fa fa-check"></i><b>7.17.1</b> Questions</a></li>
<li class="chapter" data-level="7.17.2" data-path="sem.html"><a href="sem.html#answers"><i class="fa fa-check"></i><b>7.17.2</b> Answers</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="irt.html"><a href="irt.html"><i class="fa fa-check"></i><b>8</b> Item Response Theory</a>
<ul>
<li class="chapter" data-level="8.1" data-path="irt.html"><a href="irt.html#overview-irt"><i class="fa fa-check"></i><b>8.1</b> Overview of IRT</a>
<ul>
<li class="chapter" data-level="8.1.1" data-path="irt.html"><a href="irt.html#icc"><i class="fa fa-check"></i><b>8.1.1</b> Item Characteristic Curve</a></li>
<li class="chapter" data-level="8.1.2" data-path="irt.html"><a href="irt.html#parameters-irt"><i class="fa fa-check"></i><b>8.1.2</b> Parameters</a>
<ul>
<li class="chapter" data-level="8.1.2.1" data-path="irt.html"><a href="irt.html#itemDifficulty"><i class="fa fa-check"></i><b>8.1.2.1</b> Difficulty (Severity)</a></li>
<li class="chapter" data-level="8.1.2.2" data-path="irt.html"><a href="irt.html#itemDiscrimination"><i class="fa fa-check"></i><b>8.1.2.2</b> Discrimination</a></li>
<li class="chapter" data-level="8.1.2.3" data-path="irt.html"><a href="irt.html#itemGuessing"><i class="fa fa-check"></i><b>8.1.2.3</b> Guessing</a></li>
<li class="chapter" data-level="8.1.2.4" data-path="irt.html"><a href="irt.html#itemCarelessErrors"><i class="fa fa-check"></i><b>8.1.2.4</b> Inattention/Careless Errors</a></li>
</ul></li>
<li class="chapter" data-level="8.1.3" data-path="irt.html"><a href="irt.html#models-irt"><i class="fa fa-check"></i><b>8.1.3</b> Models</a>
<ul>
<li class="chapter" data-level="8.1.3.1" data-path="irt.html"><a href="irt.html#onePL"><i class="fa fa-check"></i><b>8.1.3.1</b> 1-Parameter and Rasch models</a></li>
<li class="chapter" data-level="8.1.3.2" data-path="irt.html"><a href="irt.html#twoPL"><i class="fa fa-check"></i><b>8.1.3.2</b> 2-Parameter</a></li>
<li class="chapter" data-level="8.1.3.3" data-path="irt.html"><a href="irt.html#threePL"><i class="fa fa-check"></i><b>8.1.3.3</b> 3-Parameter</a></li>
<li class="chapter" data-level="8.1.3.4" data-path="irt.html"><a href="irt.html#fourPL"><i class="fa fa-check"></i><b>8.1.3.4</b> 4-Parameter</a></li>
<li class="chapter" data-level="8.1.3.5" data-path="irt.html"><a href="irt.html#grm"><i class="fa fa-check"></i><b>8.1.3.5</b> Graded Response Model</a></li>
</ul></li>
<li class="chapter" data-level="8.1.4" data-path="irt.html"><a href="irt.html#dataTypes-irt"><i class="fa fa-check"></i><b>8.1.4</b> Type of Data</a></li>
<li class="chapter" data-level="8.1.5" data-path="irt.html"><a href="irt.html#sampleSize-irt"><i class="fa fa-check"></i><b>8.1.5</b> Sample Size</a></li>
<li class="chapter" data-level="8.1.6" data-path="irt.html"><a href="irt.html#irtReliability"><i class="fa fa-check"></i><b>8.1.6</b> Reliability (Information)</a></li>
<li class="chapter" data-level="8.1.7" data-path="irt.html"><a href="irt.html#efficientAssessment"><i class="fa fa-check"></i><b>8.1.7</b> Efficient Assessment</a>
<ul>
<li class="chapter" data-level="8.1.7.1" data-path="irt.html"><a href="irt.html#goodMeasure"><i class="fa fa-check"></i><b>8.1.7.1</b> A Good Measure</a></li>
</ul></li>
<li class="chapter" data-level="8.1.8" data-path="irt.html"><a href="irt.html#assumptions-irt"><i class="fa fa-check"></i><b>8.1.8</b> Assumptions of IRT</a>
<ul>
<li class="chapter" data-level="8.1.8.1" data-path="irt.html"><a href="irt.html#monotonicity-irt"><i class="fa fa-check"></i><b>8.1.8.1</b> Monotonicity</a></li>
<li class="chapter" data-level="8.1.8.2" data-path="irt.html"><a href="irt.html#unidimensionality-irt"><i class="fa fa-check"></i><b>8.1.8.2</b> Unidimensionality</a></li>
<li class="chapter" data-level="8.1.8.3" data-path="irt.html"><a href="irt.html#itemInvariance-irt"><i class="fa fa-check"></i><b>8.1.8.3</b> Item Invariance</a></li>
<li class="chapter" data-level="8.1.8.4" data-path="irt.html"><a href="irt.html#localIndependence-irt"><i class="fa fa-check"></i><b>8.1.8.4</b> Local Independence</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8.2" data-path="irt.html"><a href="irt.html#gettingStarted-irt"><i class="fa fa-check"></i><b>8.2</b> Getting Started</a>
<ul>
<li class="chapter" data-level="8.2.1" data-path="irt.html"><a href="irt.html#loadLibraries-irt"><i class="fa fa-check"></i><b>8.2.1</b> Load Libraries</a></li>
<li class="chapter" data-level="8.2.2" data-path="irt.html"><a href="irt.html#loadData-irt"><i class="fa fa-check"></i><b>8.2.2</b> Load Data</a></li>
<li class="chapter" data-level="8.2.3" data-path="irt.html"><a href="irt.html#descriptiveStats-IRT"><i class="fa fa-check"></i><b>8.2.3</b> Descriptive Statistics</a></li>
</ul></li>
<li class="chapter" data-level="8.3" data-path="irt.html"><a href="irt.html#scoringApproaches"><i class="fa fa-check"></i><b>8.3</b> Comparison of Scoring Approaches</a></li>
<li class="chapter" data-level="8.4" data-path="irt.html"><a href="irt.html#irt-onePL"><i class="fa fa-check"></i><b>8.4</b> Rasch Model (1-Parameter Logistic)</a>
<ul>
<li class="chapter" data-level="8.4.1" data-path="irt.html"><a href="irt.html#irt-onePLfit"><i class="fa fa-check"></i><b>8.4.1</b> Fit Model</a></li>
<li class="chapter" data-level="8.4.2" data-path="irt.html"><a href="irt.html#irt-onePLoutput"><i class="fa fa-check"></i><b>8.4.2</b> Model Summary</a></li>
<li class="chapter" data-level="8.4.3" data-path="irt.html"><a href="irt.html#irt-onePLfactorScores"><i class="fa fa-check"></i><b>8.4.3</b> Factor Scores</a></li>
<li class="chapter" data-level="8.4.4" data-path="irt.html"><a href="irt.html#irt-onePLplots"><i class="fa fa-check"></i><b>8.4.4</b> Plots</a>
<ul>
<li class="chapter" data-level="8.4.4.1" data-path="irt.html"><a href="irt.html#irt-onePLtestCurves"><i class="fa fa-check"></i><b>8.4.4.1</b> Test Curves</a>
<ul>
<li class="chapter" data-level="8.4.4.1.1" data-path="irt.html"><a href="irt.html#irt-onePLtcc"><i class="fa fa-check"></i><b>8.4.4.1.1</b> Test Characteristic Curve</a></li>
<li class="chapter" data-level="8.4.4.1.2" data-path="irt.html"><a href="irt.html#irt-onePLtif"><i class="fa fa-check"></i><b>8.4.4.1.2</b> Test Information Curve</a></li>
<li class="chapter" data-level="8.4.4.1.3" data-path="irt.html"><a href="irt.html#irt-onePLreliability"><i class="fa fa-check"></i><b>8.4.4.1.3</b> Test Reliability</a></li>
<li class="chapter" data-level="8.4.4.1.4" data-path="irt.html"><a href="irt.html#irt-onePLsem"><i class="fa fa-check"></i><b>8.4.4.1.4</b> Test Standard Error of Measurement</a></li>
<li class="chapter" data-level="8.4.4.1.5" data-path="irt.html"><a href="irt.html#irt-onePLtifSEM"><i class="fa fa-check"></i><b>8.4.4.1.5</b> Test Information Curve and Test Standard Error of Measurement</a></li>
</ul></li>
<li class="chapter" data-level="8.4.4.2" data-path="irt.html"><a href="irt.html#irt-onePLitemCurves"><i class="fa fa-check"></i><b>8.4.4.2</b> Item Curves</a>
<ul>
<li class="chapter" data-level="8.4.4.2.1" data-path="irt.html"><a href="irt.html#irt-onePLicc"><i class="fa fa-check"></i><b>8.4.4.2.1</b> Item Characteristic Curves</a></li>
<li class="chapter" data-level="8.4.4.2.2" data-path="irt.html"><a href="irt.html#irt-onePLiif"><i class="fa fa-check"></i><b>8.4.4.2.2</b> Item Information Curves</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8.4.5" data-path="irt.html"><a href="irt.html#irt-onePLcfa"><i class="fa fa-check"></i><b>8.4.5</b> CFA</a></li>
<li class="chapter" data-level="8.4.6" data-path="irt.html"><a href="irt.html#irt-onePLmixed"><i class="fa fa-check"></i><b>8.4.6</b> Mixed Model</a></li>
</ul></li>
<li class="chapter" data-level="8.5" data-path="irt.html"><a href="irt.html#irt-twoPL"><i class="fa fa-check"></i><b>8.5</b> Two-Parameter Logistic Model</a>
<ul>
<li class="chapter" data-level="8.5.1" data-path="irt.html"><a href="irt.html#irt-twoPLfit"><i class="fa fa-check"></i><b>8.5.1</b> Fit Model</a></li>
<li class="chapter" data-level="8.5.2" data-path="irt.html"><a href="irt.html#irt-twoPLoutput"><i class="fa fa-check"></i><b>8.5.2</b> Model Summary</a></li>
<li class="chapter" data-level="8.5.3" data-path="irt.html"><a href="irt.html#irt-twoPLfactorScores"><i class="fa fa-check"></i><b>8.5.3</b> Factor Scores</a></li>
<li class="chapter" data-level="8.5.4" data-path="irt.html"><a href="irt.html#irt-twoPLplots"><i class="fa fa-check"></i><b>8.5.4</b> Plots</a>
<ul>
<li class="chapter" data-level="8.5.4.1" data-path="irt.html"><a href="irt.html#irt-twoPLtestCurves"><i class="fa fa-check"></i><b>8.5.4.1</b> Test Curves</a>
<ul>
<li class="chapter" data-level="8.5.4.1.1" data-path="irt.html"><a href="irt.html#irt-twoPLtcc"><i class="fa fa-check"></i><b>8.5.4.1.1</b> Test Characteristic Curve</a></li>
<li class="chapter" data-level="8.5.4.1.2" data-path="irt.html"><a href="irt.html#irt-twoPLtif"><i class="fa fa-check"></i><b>8.5.4.1.2</b> Test Information Curve</a></li>
<li class="chapter" data-level="8.5.4.1.3" data-path="irt.html"><a href="irt.html#irt-twoPLreliability"><i class="fa fa-check"></i><b>8.5.4.1.3</b> Test Reliability</a></li>
<li class="chapter" data-level="8.5.4.1.4" data-path="irt.html"><a href="irt.html#irt-twoPLsem"><i class="fa fa-check"></i><b>8.5.4.1.4</b> Test Standard Error of Measurement</a></li>
<li class="chapter" data-level="8.5.4.1.5" data-path="irt.html"><a href="irt.html#irt-twoPLtifSEM"><i class="fa fa-check"></i><b>8.5.4.1.5</b> Test Information Curve and Standard Errors</a></li>
</ul></li>
<li class="chapter" data-level="8.5.4.2" data-path="irt.html"><a href="irt.html#irt-twoPLitemCurves"><i class="fa fa-check"></i><b>8.5.4.2</b> Item Curves</a>
<ul>
<li class="chapter" data-level="8.5.4.2.1" data-path="irt.html"><a href="irt.html#irt-twoPLicc"><i class="fa fa-check"></i><b>8.5.4.2.1</b> Item Characteristic Curves</a></li>
<li class="chapter" data-level="8.5.4.2.2" data-path="irt.html"><a href="irt.html#irt-twoPLiif"><i class="fa fa-check"></i><b>8.5.4.2.2</b> Item Information Curves</a></li>
</ul></li>
<li class="chapter" data-level="8.5.4.3" data-path="irt.html"><a href="irt.html#irt-twoPLdiscriminationFactorLoading"><i class="fa fa-check"></i><b>8.5.4.3</b> Convert Discrimination To Factor Loading</a></li>
</ul></li>
<li class="chapter" data-level="8.5.5" data-path="irt.html"><a href="irt.html#irt-twoPLcfa"><i class="fa fa-check"></i><b>8.5.5</b> CFA</a></li>
</ul></li>
<li class="chapter" data-level="8.6" data-path="irt.html"><a href="irt.html#irt-twoPLmultidimensional"><i class="fa fa-check"></i><b>8.6</b> Two-Parameter Multidimensional Logistic Model</a>
<ul>
<li class="chapter" data-level="8.6.1" data-path="irt.html"><a href="irt.html#irt-twoPLmultidimensionalFit"><i class="fa fa-check"></i><b>8.6.1</b> Fit Model</a></li>
<li class="chapter" data-level="8.6.2" data-path="irt.html"><a href="irt.html#irt-twoPLmultidimensionalOutput"><i class="fa fa-check"></i><b>8.6.2</b> Model Summary</a></li>
<li class="chapter" data-level="8.6.3" data-path="irt.html"><a href="irt.html#irt-twoPLmultidimensionalFactorScores"><i class="fa fa-check"></i><b>8.6.3</b> Factor Scores</a></li>
<li class="chapter" data-level="8.6.4" data-path="irt.html"><a href="irt.html#irt-twoPLmultidimensionalCompare"><i class="fa fa-check"></i><b>8.6.4</b> Compare model fit</a></li>
<li class="chapter" data-level="8.6.5" data-path="irt.html"><a href="irt.html#irt-twoPLmultidimensionalPlots"><i class="fa fa-check"></i><b>8.6.5</b> Plots</a>
<ul>
<li class="chapter" data-level="8.6.5.1" data-path="irt.html"><a href="irt.html#irt-twoPLmultidimensionalTestCurves"><i class="fa fa-check"></i><b>8.6.5.1</b> Test Curves</a>
<ul>
<li class="chapter" data-level="8.6.5.1.1" data-path="irt.html"><a href="irt.html#irt-twoPLmultidimensionalTCC"><i class="fa fa-check"></i><b>8.6.5.1.1</b> Test Characteristic Curve</a></li>
<li class="chapter" data-level="8.6.5.1.2" data-path="irt.html"><a href="irt.html#irt-twoPLmultidimensionalTIF"><i class="fa fa-check"></i><b>8.6.5.1.2</b> Test Information Curve</a></li>
<li class="chapter" data-level="8.6.5.1.3" data-path="irt.html"><a href="irt.html#irt-twoPLmultidimensionalSEM"><i class="fa fa-check"></i><b>8.6.5.1.3</b> Test Standard Error of Measurement</a></li>
</ul></li>
<li class="chapter" data-level="8.6.5.2" data-path="irt.html"><a href="irt.html#irt-twoPLmultidimensionalItemCurves"><i class="fa fa-check"></i><b>8.6.5.2</b> Item Curves</a>
<ul>
<li class="chapter" data-level="8.6.5.2.1" data-path="irt.html"><a href="irt.html#irt-twoPLmultidimensionalICC"><i class="fa fa-check"></i><b>8.6.5.2.1</b> Item Characteristic Curves</a></li>
<li class="chapter" data-level="8.6.5.2.2" data-path="irt.html"><a href="irt.html#irt-twoPLmultidimensionalIIF"><i class="fa fa-check"></i><b>8.6.5.2.2</b> Item Information Curves</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8.6.6" data-path="irt.html"><a href="irt.html#irt-twoPLmultidimensionalCFA"><i class="fa fa-check"></i><b>8.6.6</b> CFA</a></li>
</ul></li>
<li class="chapter" data-level="8.7" data-path="irt.html"><a href="irt.html#irt-threePL"><i class="fa fa-check"></i><b>8.7</b> Three-Parameter Logistic Model</a>
<ul>
<li class="chapter" data-level="8.7.1" data-path="irt.html"><a href="irt.html#irt-threePLfit"><i class="fa fa-check"></i><b>8.7.1</b> Fit Model</a></li>
<li class="chapter" data-level="8.7.2" data-path="irt.html"><a href="irt.html#irt-threePLoutput"><i class="fa fa-check"></i><b>8.7.2</b> Model Summary</a></li>
<li class="chapter" data-level="8.7.3" data-path="irt.html"><a href="irt.html#irt-threePLfactorScores"><i class="fa fa-check"></i><b>8.7.3</b> Factor Scores</a></li>
<li class="chapter" data-level="8.7.4" data-path="irt.html"><a href="irt.html#irt-threePLplots"><i class="fa fa-check"></i><b>8.7.4</b> Plots</a>
<ul>
<li class="chapter" data-level="8.7.4.1" data-path="irt.html"><a href="irt.html#irt-threePLtestCurves"><i class="fa fa-check"></i><b>8.7.4.1</b> Test Curves</a>
<ul>
<li class="chapter" data-level="8.7.4.1.1" data-path="irt.html"><a href="irt.html#irt-threePLtcc"><i class="fa fa-check"></i><b>8.7.4.1.1</b> Test Characteristic Curve</a></li>
<li class="chapter" data-level="8.7.4.1.2" data-path="irt.html"><a href="irt.html#irt-threePLtif"><i class="fa fa-check"></i><b>8.7.4.1.2</b> Test Information Curve</a></li>
<li class="chapter" data-level="8.7.4.1.3" data-path="irt.html"><a href="irt.html#irt-threePLreliability"><i class="fa fa-check"></i><b>8.7.4.1.3</b> Test Reliability</a></li>
<li class="chapter" data-level="8.7.4.1.4" data-path="irt.html"><a href="irt.html#irt-threePLsem"><i class="fa fa-check"></i><b>8.7.4.1.4</b> Test Standard Error of Measurement</a></li>
<li class="chapter" data-level="8.7.4.1.5" data-path="irt.html"><a href="irt.html#irt-threePLtifSEM"><i class="fa fa-check"></i><b>8.7.4.1.5</b> Test Information Curve and Standard Errors</a></li>
</ul></li>
<li class="chapter" data-level="8.7.4.2" data-path="irt.html"><a href="irt.html#irt-threePLitemCurves"><i class="fa fa-check"></i><b>8.7.4.2</b> Item Curves</a>
<ul>
<li class="chapter" data-level="8.7.4.2.1" data-path="irt.html"><a href="irt.html#irt-threePLicc"><i class="fa fa-check"></i><b>8.7.4.2.1</b> Item Characteristic Curves</a></li>
<li class="chapter" data-level="8.7.4.2.2" data-path="irt.html"><a href="irt.html#irt-threePLiif"><i class="fa fa-check"></i><b>8.7.4.2.2</b> Item Information Curves</a></li>
</ul></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8.8" data-path="irt.html"><a href="irt.html#irt-fourPL"><i class="fa fa-check"></i><b>8.8</b> Four-Parameter Logistic Model</a>
<ul>
<li class="chapter" data-level="8.8.1" data-path="irt.html"><a href="irt.html#irt-fourPLfit"><i class="fa fa-check"></i><b>8.8.1</b> Fit Model</a></li>
<li class="chapter" data-level="8.8.2" data-path="irt.html"><a href="irt.html#irt-fourPLoutput"><i class="fa fa-check"></i><b>8.8.2</b> Model Summary</a></li>
<li class="chapter" data-level="8.8.3" data-path="irt.html"><a href="irt.html#irt-fourPLfactorScores"><i class="fa fa-check"></i><b>8.8.3</b> Factor Scores</a></li>
<li class="chapter" data-level="8.8.4" data-path="irt.html"><a href="irt.html#irt-fourPLplots"><i class="fa fa-check"></i><b>8.8.4</b> Plots</a>
<ul>
<li class="chapter" data-level="8.8.4.1" data-path="irt.html"><a href="irt.html#irt-fourPLtestCurves"><i class="fa fa-check"></i><b>8.8.4.1</b> Test Curves</a>
<ul>
<li class="chapter" data-level="8.8.4.1.1" data-path="irt.html"><a href="irt.html#irt-fourPLtcc"><i class="fa fa-check"></i><b>8.8.4.1.1</b> Test Characteristic Curve</a></li>
<li class="chapter" data-level="8.8.4.1.2" data-path="irt.html"><a href="irt.html#irt-fourPLtif"><i class="fa fa-check"></i><b>8.8.4.1.2</b> Test Information Curve</a></li>
<li class="chapter" data-level="8.8.4.1.3" data-path="irt.html"><a href="irt.html#irt-fourPLreliability"><i class="fa fa-check"></i><b>8.8.4.1.3</b> Test Reliability</a></li>
<li class="chapter" data-level="8.8.4.1.4" data-path="irt.html"><a href="irt.html#irt-fourPLsem"><i class="fa fa-check"></i><b>8.8.4.1.4</b> Test Standard Error of Measurement</a></li>
<li class="chapter" data-level="8.8.4.1.5" data-path="irt.html"><a href="irt.html#irt-fourPLtifSEM"><i class="fa fa-check"></i><b>8.8.4.1.5</b> Test Information Curve and Standard Errors</a></li>
</ul></li>
<li class="chapter" data-level="8.8.4.2" data-path="irt.html"><a href="irt.html#irt-fourPLitemCurves"><i class="fa fa-check"></i><b>8.8.4.2</b> Item Curves</a>
<ul>
<li class="chapter" data-level="8.8.4.2.1" data-path="irt.html"><a href="irt.html#irt-fourPLicc"><i class="fa fa-check"></i><b>8.8.4.2.1</b> Item Characteristic Curves</a></li>
<li class="chapter" data-level="8.8.4.2.2" data-path="irt.html"><a href="irt.html#irt-fourPLiif"><i class="fa fa-check"></i><b>8.8.4.2.2</b> Item Information Curves</a></li>
</ul></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8.9" data-path="irt.html"><a href="irt.html#irt-gradedResponseModel"><i class="fa fa-check"></i><b>8.9</b> Graded Response Model</a>
<ul>
<li class="chapter" data-level="8.9.1" data-path="irt.html"><a href="irt.html#irt-gradedResponseModelFit"><i class="fa fa-check"></i><b>8.9.1</b> Fit Model</a></li>
<li class="chapter" data-level="8.9.2" data-path="irt.html"><a href="irt.html#irt-gradedResponseModelOutput"><i class="fa fa-check"></i><b>8.9.2</b> Model Summary</a></li>
<li class="chapter" data-level="8.9.3" data-path="irt.html"><a href="irt.html#irt-gradedResponseModelFactorScores"><i class="fa fa-check"></i><b>8.9.3</b> Factor Scores</a></li>
<li class="chapter" data-level="8.9.4" data-path="irt.html"><a href="irt.html#irt-gradedResponseModelPlots"><i class="fa fa-check"></i><b>8.9.4</b> Plots</a>
<ul>
<li class="chapter" data-level="8.9.4.1" data-path="irt.html"><a href="irt.html#irt-gradedResponseModelTestCurves"><i class="fa fa-check"></i><b>8.9.4.1</b> Test Curves</a>
<ul>
<li class="chapter" data-level="8.9.4.1.1" data-path="irt.html"><a href="irt.html#irt-gradedResponseModelTCC"><i class="fa fa-check"></i><b>8.9.4.1.1</b> Test Characteristic Curve</a></li>
<li class="chapter" data-level="8.9.4.1.2" data-path="irt.html"><a href="irt.html#irt-gradedResponseModelTIF"><i class="fa fa-check"></i><b>8.9.4.1.2</b> Test Information Curve</a></li>
<li class="chapter" data-level="8.9.4.1.3" data-path="irt.html"><a href="irt.html#irt-gradedResponseModelReliability"><i class="fa fa-check"></i><b>8.9.4.1.3</b> Test Reliability</a></li>
<li class="chapter" data-level="8.9.4.1.4" data-path="irt.html"><a href="irt.html#irt-gradedResponseModelSEM"><i class="fa fa-check"></i><b>8.9.4.1.4</b> Test Standard Error of Measurement</a></li>
<li class="chapter" data-level="8.9.4.1.5" data-path="irt.html"><a href="irt.html#irt-gradedResponseModelTIFsem"><i class="fa fa-check"></i><b>8.9.4.1.5</b> Test Information Curve and Standard Errors</a></li>
</ul></li>
<li class="chapter" data-level="8.9.4.2" data-path="irt.html"><a href="irt.html#irt-gradedResponseModelItemCurves"><i class="fa fa-check"></i><b>8.9.4.2</b> Item Curves</a>
<ul>
<li class="chapter" data-level="8.9.4.2.1" data-path="irt.html"><a href="irt.html#irt-gradedResponseModelICC"><i class="fa fa-check"></i><b>8.9.4.2.1</b> Item Characteristic Curves</a></li>
<li class="chapter" data-level="8.9.4.2.2" data-path="irt.html"><a href="irt.html#irt-gradedResponseModelIIF"><i class="fa fa-check"></i><b>8.9.4.2.2</b> Item Information Curves</a></li>
<li class="chapter" data-level="8.9.4.2.3" data-path="irt.html"><a href="irt.html#irt-gradedResponseModelIRCCC"><i class="fa fa-check"></i><b>8.9.4.2.3</b> Item Response Category Characteristic Curves</a></li>
<li class="chapter" data-level="8.9.4.2.4" data-path="irt.html"><a href="irt.html#irt-gradedResponseModelIOCC"><i class="fa fa-check"></i><b>8.9.4.2.4</b> Item Boundary Characteristic Curves (aka Item Operation Characteristic Curves)</a></li>
</ul></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8.10" data-path="irt.html"><a href="irt.html#conclusion-irt"><i class="fa fa-check"></i><b>8.10</b> Conclusion</a></li>
<li class="chapter" data-level="8.11" data-path="irt.html"><a href="irt.html#readings-irt"><i class="fa fa-check"></i><b>8.11</b> Suggested Readings</a></li>
<li class="chapter" data-level="8.12" data-path="irt.html"><a href="irt.html#exercises-irt"><i class="fa fa-check"></i><b>8.12</b> Exercises</a>
<ul>
<li class="chapter" data-level="8.12.1" data-path="irt.html"><a href="irt.html#exercisesQuestions-irt"><i class="fa fa-check"></i><b>8.12.1</b> Questions</a></li>
<li class="chapter" data-level="8.12.2" data-path="irt.html"><a href="irt.html#exercisesAnswers-irt"><i class="fa fa-check"></i><b>8.12.2</b> Answers</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="prediction.html"><a href="prediction.html"><i class="fa fa-check"></i><b>9</b> Prediction</a>
<ul>
<li class="chapter" data-level="9.1" data-path="prediction.html"><a href="prediction.html#overview-prediction"><i class="fa fa-check"></i><b>9.1</b> Overview of Prediction</a>
<ul>
<li class="chapter" data-level="9.1.1" data-path="prediction.html"><a href="prediction.html#probability"><i class="fa fa-check"></i><b>9.1.1</b> Issues Around Probability</a>
<ul>
<li class="chapter" data-level="9.1.1.1" data-path="prediction.html"><a href="prediction.html#probabilityTypes"><i class="fa fa-check"></i><b>9.1.1.1</b> Types of Probabilities</a>
<ul>
<li class="chapter" data-level="9.1.1.1.1" data-path="prediction.html"><a href="prediction.html#baseRate"><i class="fa fa-check"></i><b>9.1.1.1.1</b> Base Rate (Marginal Probability)</a></li>
<li class="chapter" data-level="9.1.1.1.2" data-path="prediction.html"><a href="prediction.html#jointProbability"><i class="fa fa-check"></i><b>9.1.1.1.2</b> Joint Probability</a></li>
<li class="chapter" data-level="9.1.1.1.3" data-path="prediction.html"><a href="prediction.html#conditionalProbability"><i class="fa fa-check"></i><b>9.1.1.1.3</b> Conditional Probability</a></li>
</ul></li>
<li class="chapter" data-level="9.1.1.2" data-path="prediction.html"><a href="prediction.html#inverseFallacy"><i class="fa fa-check"></i><b>9.1.1.2</b> Confusion of the Inverse</a></li>
<li class="chapter" data-level="9.1.1.3" data-path="prediction.html"><a href="prediction.html#bayesTheorem"><i class="fa fa-check"></i><b>9.1.1.3</b> Bayes’ Theorem</a></li>
</ul></li>
<li class="chapter" data-level="9.1.2" data-path="prediction.html"><a href="prediction.html#predictionAccuracy-overview"><i class="fa fa-check"></i><b>9.1.2</b> Prediction Accuracy</a>
<ul>
<li class="chapter" data-level="9.1.2.1" data-path="prediction.html"><a href="prediction.html#decisionOutcomes-overview"><i class="fa fa-check"></i><b>9.1.2.1</b> Decision Outcomes</a></li>
<li class="chapter" data-level="9.1.2.2" data-path="prediction.html"><a href="prediction.html#percentAccuracy-overview"><i class="fa fa-check"></i><b>9.1.2.2</b> Percent Accuracy</a></li>
<li class="chapter" data-level="9.1.2.3" data-path="prediction.html"><a href="prediction.html#accuracyByChance"><i class="fa fa-check"></i><b>9.1.2.3</b> Percent Accuracy by Chance</a></li>
<li class="chapter" data-level="9.1.2.4" data-path="prediction.html"><a href="prediction.html#predictingFromBaseRate"><i class="fa fa-check"></i><b>9.1.2.4</b> Predicting from the Base Rate</a></li>
<li class="chapter" data-level="9.1.2.5" data-path="prediction.html"><a href="prediction.html#differentErrorsDifferentCosts"><i class="fa fa-check"></i><b>9.1.2.5</b> Different Kinds of Errors Have Different Costs</a></li>
<li class="chapter" data-level="9.1.2.6" data-path="prediction.html"><a href="prediction.html#sensitivitySpecificityPPVnpv"><i class="fa fa-check"></i><b>9.1.2.6</b> Sensitivity, Specificity, PPV, and NPV</a>
<ul>
<li class="chapter" data-level="9.1.2.6.1" data-path="prediction.html"><a href="prediction.html#accuracyCutoff"><i class="fa fa-check"></i><b>9.1.2.6.1</b> Some Accuracy Estimates Depend on the Cutoff</a></li>
</ul></li>
<li class="chapter" data-level="9.1.2.7" data-path="prediction.html"><a href="prediction.html#sdt"><i class="fa fa-check"></i><b>9.1.2.7</b> Signal Detection Theory</a>
<ul>
<li class="chapter" data-level="9.1.2.7.1" data-path="prediction.html"><a href="prediction.html#roc"><i class="fa fa-check"></i><b>9.1.2.7.1</b> Receiver Operating Characteristic (ROC) Curve</a></li>
</ul></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9.2" data-path="prediction.html"><a href="prediction.html#gettingStarted-prediction"><i class="fa fa-check"></i><b>9.2</b> Getting Started</a>
<ul>
<li class="chapter" data-level="9.2.1" data-path="prediction.html"><a href="prediction.html#loadLibraries-prediction"><i class="fa fa-check"></i><b>9.2.1</b> Load Libraries</a></li>
<li class="chapter" data-level="9.2.2" data-path="prediction.html"><a href="prediction.html#prepareData-prediction"><i class="fa fa-check"></i><b>9.2.2</b> Prepare Data</a>
<ul>
<li class="chapter" data-level="9.2.2.1" data-path="prediction.html"><a href="prediction.html#loadData-prediction"><i class="fa fa-check"></i><b>9.2.2.1</b> Load Data</a></li>
<li class="chapter" data-level="9.2.2.2" data-path="prediction.html"><a href="prediction.html#simulateData-prediction"><i class="fa fa-check"></i><b>9.2.2.2</b> Simulate Data</a></li>
<li class="chapter" data-level="9.2.2.3" data-path="prediction.html"><a href="prediction.html#addMissingData-prediction"><i class="fa fa-check"></i><b>9.2.2.3</b> Add Missing Data</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="prediction.html"><a href="prediction.html#rocCurve"><i class="fa fa-check"></i><b>9.3</b> Receiver Operating Characteristic (ROC) Curve</a>
<ul>
<li class="chapter" data-level="9.3.1" data-path="prediction.html"><a href="prediction.html#empiricalROC"><i class="fa fa-check"></i><b>9.3.1</b> Empirical ROC Curve</a></li>
<li class="chapter" data-level="9.3.2" data-path="prediction.html"><a href="prediction.html#smoothROC"><i class="fa fa-check"></i><b>9.3.2</b> Smooth ROC Curve</a></li>
<li class="chapter" data-level="9.3.3" data-path="prediction.html"><a href="prediction.html#youdenJ"><i class="fa fa-check"></i><b>9.3.3</b> Youden’s J Statistic</a></li>
<li class="chapter" data-level="9.3.4" data-path="prediction.html"><a href="prediction.html#topLeftROC"><i class="fa fa-check"></i><b>9.3.4</b> The point closest to the top-left part of the ROC curve with perfect sensitivity and specificity</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="prediction.html"><a href="prediction.html#predictionAccuracy"><i class="fa fa-check"></i><b>9.4</b> Prediction Accuracy Across Cutoffs</a>
<ul>
<li class="chapter" data-level="9.4.1" data-path="prediction.html"><a href="prediction.html#generalPredictionAccuracy"><i class="fa fa-check"></i><b>9.4.1</b> General Prediction Accuracy</a>
<ul>
<li class="chapter" data-level="9.4.1.1" data-path="prediction.html"><a href="prediction.html#scaleDependentAccuracy"><i class="fa fa-check"></i><b>9.4.1.1</b> Scale-Dependent Accuracy Estimates</a>
<ul>
<li class="chapter" data-level="9.4.1.1.1" data-path="prediction.html"><a href="prediction.html#meanError"><i class="fa fa-check"></i><b>9.4.1.1.1</b> Mean Error</a></li>
<li class="chapter" data-level="9.4.1.1.2" data-path="prediction.html"><a href="prediction.html#meanAbsoluteError"><i class="fa fa-check"></i><b>9.4.1.1.2</b> Mean Absolute Error (MAE)</a></li>
<li class="chapter" data-level="9.4.1.1.3" data-path="prediction.html"><a href="prediction.html#meanSquaredError"><i class="fa fa-check"></i><b>9.4.1.1.3</b> Mean Squared Error (MSE)</a></li>
<li class="chapter" data-level="9.4.1.1.4" data-path="prediction.html"><a href="prediction.html#rootMeanSquaredError"><i class="fa fa-check"></i><b>9.4.1.1.4</b> Root Mean Squared Error (RMSE)</a></li>
</ul></li>
<li class="chapter" data-level="9.4.1.2" data-path="prediction.html"><a href="prediction.html#scaleIndependentAccuracy"><i class="fa fa-check"></i><b>9.4.1.2</b> Scale-Independent Accuracy Estimates</a>
<ul>
<li class="chapter" data-level="9.4.1.2.1" data-path="prediction.html"><a href="prediction.html#meanPercentageError"><i class="fa fa-check"></i><b>9.4.1.2.1</b> Mean Percentage Error (MPE)</a></li>
<li class="chapter" data-level="9.4.1.2.2" data-path="prediction.html"><a href="prediction.html#meanAbsolutePercentageError"><i class="fa fa-check"></i><b>9.4.1.2.2</b> Mean Absolute Percentage Error (MAPE)</a></li>
<li class="chapter" data-level="9.4.1.2.3" data-path="prediction.html"><a href="prediction.html#symmetricMeanAbsolutePercentageError"><i class="fa fa-check"></i><b>9.4.1.2.3</b> Symmetric Mean Absolute Percentage Error (sMAPE)</a></li>
<li class="chapter" data-level="9.4.1.2.4" data-path="prediction.html"><a href="prediction.html#meanAbsoluteScaledErrro"><i class="fa fa-check"></i><b>9.4.1.2.4</b> Mean Absolute Scaled Error (MASE)</a></li>
<li class="chapter" data-level="9.4.1.2.5" data-path="prediction.html"><a href="prediction.html#accuracyRatio"><i class="fa fa-check"></i><b>9.4.1.2.5</b> Root Mean Squared Log Error (RMSLE)</a></li>
<li class="chapter" data-level="9.4.1.2.6" data-path="prediction.html"><a href="prediction.html#coefficientOfDetermination"><i class="fa fa-check"></i><b>9.4.1.2.6</b> Coefficient of Determination (<span class="math inline">\(R^2\)</span>)</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9.4.2" data-path="prediction.html"><a href="prediction.html#discrimination"><i class="fa fa-check"></i><b>9.4.2</b> Discrimination</a>
<ul>
<li class="chapter" data-level="9.4.2.1" data-path="prediction.html"><a href="prediction.html#aucROC"><i class="fa fa-check"></i><b>9.4.2.1</b> Area under the ROC curve (AUC)</a></li>
<li class="chapter" data-level="9.4.2.2" data-path="prediction.html"><a href="prediction.html#CPA"><i class="fa fa-check"></i><b>9.4.2.2</b> Coefficient of Predictive Ability (CPA)</a></li>
<li class="chapter" data-level="9.4.2.3" data-path="prediction.html"><a href="prediction.html#spearmanRho"><i class="fa fa-check"></i><b>9.4.2.3</b> Spearman’s Rho (<span class="math inline">\(\rho\)</span>) Rank Correlation</a></li>
<li class="chapter" data-level="9.4.2.4" data-path="prediction.html"><a href="prediction.html#somersD"><i class="fa fa-check"></i><b>9.4.2.4</b> Somers’ <span class="math inline">\(D_{xy}\)</span> Rank Correlation</a></li>
<li class="chapter" data-level="9.4.2.5" data-path="prediction.html"><a href="prediction.html#kendallTau"><i class="fa fa-check"></i><b>9.4.2.5</b> Kendall’s Tau-a (<span class="math inline">\(\tau_A\)</span>) Rank Correlation</a></li>
<li class="chapter" data-level="9.4.2.6" data-path="prediction.html"><a href="prediction.html#concordanceProbability"><i class="fa fa-check"></i><b>9.4.2.6</b> <span class="math inline">\(c\)</span> Index</a></li>
<li class="chapter" data-level="9.4.2.7" data-path="prediction.html"><a href="prediction.html#standardizedRegressionCoefficient"><i class="fa fa-check"></i><b>9.4.2.7</b> Effect Size (<span class="math inline">\(\beta\)</span>) of Regression</a></li>
</ul></li>
<li class="chapter" data-level="9.4.3" data-path="prediction.html"><a href="prediction.html#calibration"><i class="fa fa-check"></i><b>9.4.3</b> Calibration</a>
<ul>
<li class="chapter" data-level="9.4.3.1" data-path="prediction.html"><a href="prediction.html#calibrationPlot"><i class="fa fa-check"></i><b>9.4.3.1</b> Calibration Plot</a></li>
<li class="chapter" data-level="9.4.3.2" data-path="prediction.html"><a href="prediction.html#brierScores"><i class="fa fa-check"></i><b>9.4.3.2</b> Brier Scores</a></li>
<li class="chapter" data-level="9.4.3.3" data-path="prediction.html"><a href="prediction.html#hoslerLemeshow"><i class="fa fa-check"></i><b>9.4.3.3</b> Hosler-Lemeshow Test</a></li>
<li class="chapter" data-level="9.4.3.4" data-path="prediction.html"><a href="prediction.html#spiegelhalterZ"><i class="fa fa-check"></i><b>9.4.3.4</b> Spiegelhalter’s z</a></li>
<li class="chapter" data-level="9.4.3.5" data-path="prediction.html"><a href="prediction.html#calibrationContinuous"><i class="fa fa-check"></i><b>9.4.3.5</b> Calibration for predicting a continuous outcome</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="prediction.html"><a href="prediction.html#cutoffAccuracy"><i class="fa fa-check"></i><b>9.5</b> Prediction Accuracy at a Given Cutoff</a>
<ul>
<li class="chapter" data-level="9.5.1" data-path="prediction.html"><a href="prediction.html#setCutoff"><i class="fa fa-check"></i><b>9.5.1</b> Set a Cutoff</a></li>
<li class="chapter" data-level="9.5.2" data-path="prediction.html"><a href="prediction.html#accuracyAtCutoff"><i class="fa fa-check"></i><b>9.5.2</b> Accuracy at a Given Cutoff</a></li>
<li class="chapter" data-level="9.5.3" data-path="prediction.html"><a href="prediction.html#confusionMatrix"><i class="fa fa-check"></i><b>9.5.3</b> Confusion Matrix aka 2x2 Accuracy Table aka Cross-Tabulation aka Contingency Table</a>
<ul>
<li class="chapter" data-level="9.5.3.1" data-path="prediction.html"><a href="prediction.html#confusionMatrix-number"><i class="fa fa-check"></i><b>9.5.3.1</b> Number</a></li>
<li class="chapter" data-level="9.5.3.2" data-path="prediction.html"><a href="prediction.html#confusionMatrix-numberMargins"><i class="fa fa-check"></i><b>9.5.3.2</b> Number with margins added</a></li>
<li class="chapter" data-level="9.5.3.3" data-path="prediction.html"><a href="prediction.html#confusionMatrix-numberMarginsProportions"><i class="fa fa-check"></i><b>9.5.3.3</b> Proportions</a></li>
<li class="chapter" data-level="9.5.3.4" data-path="prediction.html"><a href="prediction.html#confusionMatrix-numberMarginsProportionsMargins"><i class="fa fa-check"></i><b>9.5.3.4</b> Proportions with margins added</a></li>
</ul></li>
<li class="chapter" data-level="9.5.4" data-path="prediction.html"><a href="prediction.html#truePositive"><i class="fa fa-check"></i><b>9.5.4</b> True Positives (TP)</a></li>
<li class="chapter" data-level="9.5.5" data-path="prediction.html"><a href="prediction.html#trueNegative"><i class="fa fa-check"></i><b>9.5.5</b> True Negatives (TN)</a></li>
<li class="chapter" data-level="9.5.6" data-path="prediction.html"><a href="prediction.html#falsePositive"><i class="fa fa-check"></i><b>9.5.6</b> False Positives (FP)</a></li>
<li class="chapter" data-level="9.5.7" data-path="prediction.html"><a href="prediction.html#falseNegative"><i class="fa fa-check"></i><b>9.5.7</b> False Negatives (FN)</a></li>
<li class="chapter" data-level="9.5.8" data-path="prediction.html"><a href="prediction.html#sampleSize-prediction"><i class="fa fa-check"></i><b>9.5.8</b> Sample Size (<em>N</em>)</a></li>
<li class="chapter" data-level="9.5.9" data-path="prediction.html"><a href="prediction.html#selectionRatio"><i class="fa fa-check"></i><b>9.5.9</b> Selection Ratio (SR)</a></li>
<li class="chapter" data-level="9.5.10" data-path="prediction.html"><a href="prediction.html#pretestProbability"><i class="fa fa-check"></i><b>9.5.10</b> Base Rate (BR)</a></li>
<li class="chapter" data-level="9.5.11" data-path="prediction.html"><a href="prediction.html#pretestOdds"><i class="fa fa-check"></i><b>9.5.11</b> Pretest Odds</a></li>
<li class="chapter" data-level="9.5.12" data-path="prediction.html"><a href="prediction.html#percentAccuracy"><i class="fa fa-check"></i><b>9.5.12</b> Percent Accuracy</a></li>
<li class="chapter" data-level="9.5.13" data-path="prediction.html"><a href="prediction.html#percentAccuracyByChance"><i class="fa fa-check"></i><b>9.5.13</b> Percent Accuracy by Chance</a></li>
<li class="chapter" data-level="9.5.14" data-path="prediction.html"><a href="prediction.html#percentAccuracyPredictingFromBaseRate"><i class="fa fa-check"></i><b>9.5.14</b> Percent Accuracy Predicting from the Base Rate</a></li>
<li class="chapter" data-level="9.5.15" data-path="prediction.html"><a href="prediction.html#relativeImprovementOverChance"><i class="fa fa-check"></i><b>9.5.15</b> Relative Improvement Over Chance (RIOC)</a></li>
<li class="chapter" data-level="9.5.16" data-path="prediction.html"><a href="prediction.html#relativeImprovementOverPredictingFromBaseRate"><i class="fa fa-check"></i><b>9.5.16</b> Relative Improvement Over Predicting from the Base Rate</a></li>
<li class="chapter" data-level="9.5.17" data-path="prediction.html"><a href="prediction.html#sensitivity"><i class="fa fa-check"></i><b>9.5.17</b> Sensitivity (SN)</a></li>
<li class="chapter" data-level="9.5.18" data-path="prediction.html"><a href="prediction.html#specificity"><i class="fa fa-check"></i><b>9.5.18</b> Specificity (SP)</a></li>
<li class="chapter" data-level="9.5.19" data-path="prediction.html"><a href="prediction.html#falseNegativeRate"><i class="fa fa-check"></i><b>9.5.19</b> False Negative Rate (FNR)</a></li>
<li class="chapter" data-level="9.5.20" data-path="prediction.html"><a href="prediction.html#falsePositiveRate"><i class="fa fa-check"></i><b>9.5.20</b> False Positive Rate (FPR)</a></li>
<li class="chapter" data-level="9.5.21" data-path="prediction.html"><a href="prediction.html#ppv"><i class="fa fa-check"></i><b>9.5.21</b> Positive Predictive Value (PPV)</a></li>
<li class="chapter" data-level="9.5.22" data-path="prediction.html"><a href="prediction.html#npv"><i class="fa fa-check"></i><b>9.5.22</b> Negative Predictive Value (NPV)</a></li>
<li class="chapter" data-level="9.5.23" data-path="prediction.html"><a href="prediction.html#falseDiscoveryRate"><i class="fa fa-check"></i><b>9.5.23</b> False Discovery Rate (FDR)</a></li>
<li class="chapter" data-level="9.5.24" data-path="prediction.html"><a href="prediction.html#falseOmissionRate"><i class="fa fa-check"></i><b>9.5.24</b> False Omission Rate (FOR)</a></li>
<li class="chapter" data-level="9.5.25" data-path="prediction.html"><a href="prediction.html#youdenJ-example"><i class="fa fa-check"></i><b>9.5.25</b> Youden’s J Statistic</a></li>
<li class="chapter" data-level="9.5.26" data-path="prediction.html"><a href="prediction.html#balancedAccuracy"><i class="fa fa-check"></i><b>9.5.26</b> Balanced Accuracy</a></li>
<li class="chapter" data-level="9.5.27" data-path="prediction.html"><a href="prediction.html#fScore"><i class="fa fa-check"></i><b>9.5.27</b> F-Score</a></li>
<li class="chapter" data-level="9.5.28" data-path="prediction.html"><a href="prediction.html#matthewsCorrelationCoefficient"><i class="fa fa-check"></i><b>9.5.28</b> Matthews Correlation Coefficient (MCC)</a></li>
<li class="chapter" data-level="9.5.29" data-path="prediction.html"><a href="prediction.html#diagnosticOddsRatio"><i class="fa fa-check"></i><b>9.5.29</b> Diagnostic Odds Ratio</a></li>
<li class="chapter" data-level="9.5.30" data-path="prediction.html"><a href="prediction.html#diagnosticLikelihoodRatio"><i class="fa fa-check"></i><b>9.5.30</b> Diagnostic Likelihood Ratio</a>
<ul>
<li class="chapter" data-level="9.5.30.1" data-path="prediction.html"><a href="prediction.html#positiveLikelihoodRatio"><i class="fa fa-check"></i><b>9.5.30.1</b> Positive Likelihood Ratio (LR+)</a></li>
<li class="chapter" data-level="9.5.30.2" data-path="prediction.html"><a href="prediction.html#negativeLikelihoodRatio"><i class="fa fa-check"></i><b>9.5.30.2</b> Negative Likelihood Ratio (LR−)</a></li>
</ul></li>
<li class="chapter" data-level="9.5.31" data-path="prediction.html"><a href="prediction.html#posttestOdds"><i class="fa fa-check"></i><b>9.5.31</b> Posttest Odds</a></li>
<li class="chapter" data-level="9.5.32" data-path="prediction.html"><a href="prediction.html#posttestProbability"><i class="fa fa-check"></i><b>9.5.32</b> Posttest Probability</a></li>
<li class="chapter" data-level="9.5.33" data-path="prediction.html"><a href="prediction.html#nomogram"><i class="fa fa-check"></i><b>9.5.33</b> Probability Nomogram</a></li>
<li class="chapter" data-level="9.5.34" data-path="prediction.html"><a href="prediction.html#dPrimeSDT"><i class="fa fa-check"></i><b>9.5.34</b> <span class="math inline">\(d&#39;\)</span> Sensitivity from Signal Detection Theory</a></li>
<li class="chapter" data-level="9.5.35" data-path="prediction.html"><a href="prediction.html#aSDT"><i class="fa fa-check"></i><b>9.5.35</b> <span class="math inline">\(A\)</span> (Non-Parametric) Sensitivity from Signal Detection Theory</a></li>
<li class="chapter" data-level="9.5.36" data-path="prediction.html"><a href="prediction.html#betaSDT"><i class="fa fa-check"></i><b>9.5.36</b> <span class="math inline">\(\beta\)</span> Bias from Signal Detection Theory</a></li>
<li class="chapter" data-level="9.5.37" data-path="prediction.html"><a href="prediction.html#cSDT"><i class="fa fa-check"></i><b>9.5.37</b> <span class="math inline">\(c\)</span> Bias from Signal Detection Theory</a></li>
<li class="chapter" data-level="9.5.38" data-path="prediction.html"><a href="prediction.html#bSDT"><i class="fa fa-check"></i><b>9.5.38</b> <span class="math inline">\(b\)</span> (Non-Parametric) Bias from Signal Detection Theory</a></li>
<li class="chapter" data-level="9.5.39" data-path="prediction.html"><a href="prediction.html#miscalibration"><i class="fa fa-check"></i><b>9.5.39</b> Mean Difference between Predicted Versus Observed Values (Miscalibration)</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="prediction.html"><a href="prediction.html#optimalCutoff"><i class="fa fa-check"></i><b>9.6</b> Optimal Cutoff Specification</a>
<ul>
<li class="chapter" data-level="9.6.1" data-path="prediction.html"><a href="prediction.html#decisionTheory"><i class="fa fa-check"></i><b>9.6.1</b> Decision Theory</a>
<ul>
<li class="chapter" data-level="9.6.1.1" data-path="prediction.html"><a href="prediction.html#overallUtilityCutoff"><i class="fa fa-check"></i><b>9.6.1.1</b> Overall utility of a specific cutoff value</a></li>
<li class="chapter" data-level="9.6.1.2" data-path="prediction.html"><a href="prediction.html#utilityRatio"><i class="fa fa-check"></i><b>9.6.1.2</b> Utility ratio</a></li>
</ul></li>
<li class="chapter" data-level="9.6.2" data-path="prediction.html"><a href="prediction.html#informationTheory"><i class="fa fa-check"></i><b>9.6.2</b> Information Theory</a>
<ul>
<li class="chapter" data-level="9.6.2.1" data-path="prediction.html"><a href="prediction.html#informationGain"><i class="fa fa-check"></i><b>9.6.2.1</b> Information Gain</a>
<ul>
<li class="chapter" data-level="9.6.2.1.1" data-path="prediction.html"><a href="prediction.html#IgainTreat2023"><i class="fa fa-check"></i><b>9.6.2.1.1</b> Formula from <span class="citation">Treat &amp; Viken (2023)</span></a></li>
<li class="chapter" data-level="9.6.2.1.2" data-path="prediction.html"><a href="prediction.html#IgainMetz1973"><i class="fa fa-check"></i><b>9.6.2.1.2</b> Alternative formula from <span class="citation">Metz et al. (1973)</span></a></li>
<li class="chapter" data-level="9.6.2.1.3" data-path="prediction.html"><a href="prediction.html#IgainSomoza1989"><i class="fa fa-check"></i><b>9.6.2.1.3</b> Alternative formula from <span class="citation">Somoza et al. (1989)</span></a></li>
<li class="chapter" data-level="9.6.2.1.4" data-path="prediction.html"><a href="prediction.html#IgainExamples"><i class="fa fa-check"></i><b>9.6.2.1.4</b> Examples</a></li>
<li class="chapter" data-level="9.6.2.1.5" data-path="prediction.html"><a href="prediction.html#baseRateEffectsIgain"><i class="fa fa-check"></i><b>9.6.2.1.5</b> Effect of Base Rate</a></li>
</ul></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9.7" data-path="prediction.html"><a href="prediction.html#accuracyAtEveryPossibleCutoff"><i class="fa fa-check"></i><b>9.7</b> Accuracy at Every Possible Cutoff</a>
<ul>
<li class="chapter" data-level="9.7.1" data-path="prediction.html"><a href="prediction.html#specifyUtility"><i class="fa fa-check"></i><b>9.7.1</b> Specify utility of each outcome</a></li>
<li class="chapter" data-level="9.7.2" data-path="prediction.html"><a href="prediction.html#calculateAccuracyAtEveryPossibleCutoff"><i class="fa fa-check"></i><b>9.7.2</b> Calculate Accuracy</a></li>
<li class="chapter" data-level="9.7.3" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics"><i class="fa fa-check"></i><b>9.7.3</b> All Accuracy Statistics</a></li>
<li class="chapter" data-level="9.7.4" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-youdenJ"><i class="fa fa-check"></i><b>9.7.4</b> Youden’s J Statistic</a>
<ul>
<li class="chapter" data-level="9.7.4.1" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-youdenJthreshold"><i class="fa fa-check"></i><b>9.7.4.1</b> Threshold</a></li>
<li class="chapter" data-level="9.7.4.2" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-youdenJaccuracy"><i class="fa fa-check"></i><b>9.7.4.2</b> Accuracy statistics at cutoff of Youden’s J Statistic</a></li>
</ul></li>
<li class="chapter" data-level="9.7.5" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-topLeftROC"><i class="fa fa-check"></i><b>9.7.5</b> Closest to the Top Left of the ROC Curve</a>
<ul>
<li class="chapter" data-level="9.7.5.1" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-topLeftROCthreshold"><i class="fa fa-check"></i><b>9.7.5.1</b> Threshold</a></li>
<li class="chapter" data-level="9.7.5.2" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-topLeftROCaccuracy"><i class="fa fa-check"></i><b>9.7.5.2</b> Accuracy stats at cutoff where the ROC plot is closest to the Top Left</a></li>
</ul></li>
<li class="chapter" data-level="9.7.6" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoff"><i class="fa fa-check"></i><b>9.7.6</b> Cutoff that optimizes each of the following criteria:</a>
<ul>
<li class="chapter" data-level="9.7.6.1" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffPercentAccuracy"><i class="fa fa-check"></i><b>9.7.6.1</b> Percent Accuracy</a></li>
<li class="chapter" data-level="9.7.6.2" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffPercentAccuracyByChance"><i class="fa fa-check"></i><b>9.7.6.2</b> Percent Accuracy by Chance</a></li>
<li class="chapter" data-level="9.7.6.3" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffROIC"><i class="fa fa-check"></i><b>9.7.6.3</b> Relative Improvement Over Chance (ROIC)</a></li>
<li class="chapter" data-level="9.7.6.4" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffRelativeImprovementOverPredictingFromBaseRate"><i class="fa fa-check"></i><b>9.7.6.4</b> Relative Improvement Over Predicting from the Base Rate</a></li>
<li class="chapter" data-level="9.7.6.5" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffSensitivity"><i class="fa fa-check"></i><b>9.7.6.5</b> Sensitivity</a></li>
<li class="chapter" data-level="9.7.6.6" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffSpecificity"><i class="fa fa-check"></i><b>9.7.6.6</b> Specificity</a></li>
<li class="chapter" data-level="9.7.6.7" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffPPV"><i class="fa fa-check"></i><b>9.7.6.7</b> Positive Predictive Value</a></li>
<li class="chapter" data-level="9.7.6.8" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffNPV"><i class="fa fa-check"></i><b>9.7.6.8</b> Negative Predictive Value</a></li>
<li class="chapter" data-level="9.7.6.9" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffYoudenJ"><i class="fa fa-check"></i><b>9.7.6.9</b> Youden’s J Statistic</a></li>
<li class="chapter" data-level="9.7.6.10" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffBalancedAccuracy"><i class="fa fa-check"></i><b>9.7.6.10</b> Balanced Accuracy</a></li>
<li class="chapter" data-level="9.7.6.11" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffF1"><i class="fa fa-check"></i><b>9.7.6.11</b> F1 Score</a></li>
<li class="chapter" data-level="9.7.6.12" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffMCC"><i class="fa fa-check"></i><b>9.7.6.12</b> Matthews Correlation Coefficient</a></li>
<li class="chapter" data-level="9.7.6.13" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffDOR"><i class="fa fa-check"></i><b>9.7.6.13</b> Diagnostic Odds Ratio</a></li>
<li class="chapter" data-level="9.7.6.14" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffPLR"><i class="fa fa-check"></i><b>9.7.6.14</b> Positive Likelihood Ratio</a></li>
<li class="chapter" data-level="9.7.6.15" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffNLR"><i class="fa fa-check"></i><b>9.7.6.15</b> Negative Likelihood Ratio</a></li>
<li class="chapter" data-level="9.7.6.16" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffDprime"><i class="fa fa-check"></i><b>9.7.6.16</b> <span class="math inline">\(d&#39;\)</span> Sensitivity</a></li>
<li class="chapter" data-level="9.7.6.17" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffAsensitivity"><i class="fa fa-check"></i><b>9.7.6.17</b> <span class="math inline">\(A\)</span> (Non-Parametric) Sensitivity</a></li>
<li class="chapter" data-level="9.7.6.18" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffBetaBias"><i class="fa fa-check"></i><b>9.7.6.18</b> <span class="math inline">\(\beta\)</span> Bias</a></li>
<li class="chapter" data-level="9.7.6.19" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffCbias"><i class="fa fa-check"></i><b>9.7.6.19</b> <span class="math inline">\(c\)</span> Bias</a></li>
<li class="chapter" data-level="9.7.6.20" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffBbias"><i class="fa fa-check"></i><b>9.7.6.20</b> <span class="math inline">\(b\)</span> (Non-Parametric) Bias</a></li>
<li class="chapter" data-level="9.7.6.21" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffMiscalibration"><i class="fa fa-check"></i><b>9.7.6.21</b> Mean difference between predicted and observed values (Miscalibration)</a></li>
<li class="chapter" data-level="9.7.6.22" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffOverallUtility"><i class="fa fa-check"></i><b>9.7.6.22</b> Overall Utility</a></li>
<li class="chapter" data-level="9.7.6.23" data-path="prediction.html"><a href="prediction.html#allAccuracyStatistics-cutoffIgain"><i class="fa fa-check"></i><b>9.7.6.23</b> Information Gain</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9.8" data-path="prediction.html"><a href="prediction.html#regression-prediction"><i class="fa fa-check"></i><b>9.8</b> Regression for Prediction of Continuous Outcomes</a></li>
<li class="chapter" data-level="9.9" data-path="prediction.html"><a href="prediction.html#pseudoPrediction"><i class="fa fa-check"></i><b>9.9</b> Pseudo-Prediction</a>
<ul>
<li class="chapter" data-level="9.9.1" data-path="prediction.html"><a href="prediction.html#multiCollinearity"><i class="fa fa-check"></i><b>9.9.1</b> Multicollinearity</a></li>
</ul></li>
<li class="chapter" data-level="9.10" data-path="prediction.html"><a href="prediction.html#waysToImprovePredictionAccuracy"><i class="fa fa-check"></i><b>9.10</b> Ways to Improve Prediction Accuracy</a></li>
<li class="chapter" data-level="9.11" data-path="prediction.html"><a href="prediction.html#conclusion-prediction"><i class="fa fa-check"></i><b>9.11</b> Conclusion</a></li>
<li class="chapter" data-level="9.12" data-path="prediction.html"><a href="prediction.html#readings-prediction"><i class="fa fa-check"></i><b>9.12</b> Suggested Readings</a></li>
<li class="chapter" data-level="9.13" data-path="prediction.html"><a href="prediction.html#exercises-prediction"><i class="fa fa-check"></i><b>9.13</b> Exercises</a>
<ul>
<li class="chapter" data-level="9.13.1" data-path="prediction.html"><a href="prediction.html#exercisesQuestions-prediction"><i class="fa fa-check"></i><b>9.13.1</b> Questions</a></li>
<li class="chapter" data-level="9.13.2" data-path="prediction.html"><a href="prediction.html#exercisesAnswers-prediction"><i class="fa fa-check"></i><b>9.13.2</b> Answers</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="10" data-path="actuarial.html"><a href="actuarial.html"><i class="fa fa-check"></i><b>10</b> Clinical Judgment Versus Algorithmic Prediction</a>
<ul>
<li class="chapter" data-level="10.1" data-path="actuarial.html"><a href="actuarial.html#apporachesToPrediction"><i class="fa fa-check"></i><b>10.1</b> Approaches to Prediction</a>
<ul>
<li class="chapter" data-level="10.1.1" data-path="actuarial.html"><a href="actuarial.html#humanJudgment"><i class="fa fa-check"></i><b>10.1.1</b> Human/Clinical Judgment</a></li>
<li class="chapter" data-level="10.1.2" data-path="actuarial.html"><a href="actuarial.html#actuarialPrediction"><i class="fa fa-check"></i><b>10.1.2</b> Actuarial/Statistical Method</a></li>
<li class="chapter" data-level="10.1.3" data-path="actuarial.html"><a href="actuarial.html#combiningHumanJudgmentActuarialPrediction"><i class="fa fa-check"></i><b>10.1.3</b> Combining Human Judgment and Statistical Algorithms</a></li>
</ul></li>
<li class="chapter" data-level="10.2" data-path="actuarial.html"><a href="actuarial.html#errorsInClinicalJudgment"><i class="fa fa-check"></i><b>10.2</b> Errors in Clinical Judgment</a></li>
<li class="chapter" data-level="10.3" data-path="actuarial.html"><a href="actuarial.html#humansVsComputers"><i class="fa fa-check"></i><b>10.3</b> Humans Versus Computers</a>
<ul>
<li class="chapter" data-level="10.3.1" data-path="actuarial.html"><a href="actuarial.html#advantagesOfComputers"><i class="fa fa-check"></i><b>10.3.1</b> Advantages of Computers</a></li>
<li class="chapter" data-level="10.3.2" data-path="actuarial.html"><a href="actuarial.html#advantagesOfHumans"><i class="fa fa-check"></i><b>10.3.2</b> Advantages of Humans</a></li>
<li class="chapter" data-level="10.3.3" data-path="actuarial.html"><a href="actuarial.html#evidenceOnHumanJudgmentVsActuarialPrediction"><i class="fa fa-check"></i><b>10.3.3</b> Comparison of Evidence</a></li>
</ul></li>
<li class="chapter" data-level="10.4" data-path="actuarial.html"><a href="actuarial.html#modelAccuracy-actuarial"><i class="fa fa-check"></i><b>10.4</b> Accuracy of Different Statistical Models</a></li>
<li class="chapter" data-level="10.5" data-path="actuarial.html"><a href="actuarial.html#gettingStarted-actuarial"><i class="fa fa-check"></i><b>10.5</b> Getting Started</a>
<ul>
<li class="chapter" data-level="10.5.1" data-path="actuarial.html"><a href="actuarial.html#loadLibraries-actuarial"><i class="fa fa-check"></i><b>10.5.1</b> Load Libraries</a></li>
<li class="chapter" data-level="10.5.2" data-path="actuarial.html"><a href="actuarial.html#prepareData-actuarial"><i class="fa fa-check"></i><b>10.5.2</b> Prepare Data</a>
<ul>
<li class="chapter" data-level="10.5.2.1" data-path="actuarial.html"><a href="actuarial.html#loadData-actuarial"><i class="fa fa-check"></i><b>10.5.2.1</b> Load Data</a></li>
<li class="chapter" data-level="10.5.2.2" data-path="actuarial.html"><a href="actuarial.html#simulateData-actuarial"><i class="fa fa-check"></i><b>10.5.2.2</b> Simulate Data</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="10.6" data-path="actuarial.html"><a href="actuarial.html#fittingModels-actuarial"><i class="fa fa-check"></i><b>10.6</b> Fitting the Statistical Models</a>
<ul>
<li class="chapter" data-level="10.6.1" data-path="actuarial.html"><a href="actuarial.html#regressionOnePredictor"><i class="fa fa-check"></i><b>10.6.1</b> Regression with One Predictor</a></li>
<li class="chapter" data-level="10.6.2" data-path="actuarial.html"><a href="actuarial.html#regressionMultiplePredictors"><i class="fa fa-check"></i><b>10.6.2</b> Regression with Multiple Predictors</a></li>
<li class="chapter" data-level="10.6.3" data-path="actuarial.html"><a href="actuarial.html#lasso"><i class="fa fa-check"></i><b>10.6.3</b> Least Absolute Shrinkage and Selection Option (LASSO)</a></li>
<li class="chapter" data-level="10.6.4" data-path="actuarial.html"><a href="actuarial.html#ridgeRegression"><i class="fa fa-check"></i><b>10.6.4</b> Ridge Regression</a></li>
<li class="chapter" data-level="10.6.5" data-path="actuarial.html"><a href="actuarial.html#elasticNet"><i class="fa fa-check"></i><b>10.6.5</b> Elastic Net</a></li>
<li class="chapter" data-level="10.6.6" data-path="actuarial.html"><a href="actuarial.html#randomForest"><i class="fa fa-check"></i><b>10.6.6</b> Random Forest Machine Learning</a></li>
<li class="chapter" data-level="10.6.7" data-path="actuarial.html"><a href="actuarial.html#kfoldCV"><i class="fa fa-check"></i><b>10.6.7</b> <em>k</em>-Fold Cross-Validation</a></li>
<li class="chapter" data-level="10.6.8" data-path="actuarial.html"><a href="actuarial.html#looCV"><i class="fa fa-check"></i><b>10.6.8</b> Leave-One-Out (LOO) Cross-Validation</a></li>
</ul></li>
<li class="chapter" data-level="10.7" data-path="actuarial.html"><a href="actuarial.html#actuarialExamples"><i class="fa fa-check"></i><b>10.7</b> Examples of Using Actuarial Methods</a></li>
<li class="chapter" data-level="10.8" data-path="actuarial.html"><a href="actuarial.html#whyHumanJudgmentIsMoreWidelyUsed"><i class="fa fa-check"></i><b>10.8</b> Why Clinical Judgment is More Widely Used Than Statistical Formulas</a></li>
<li class="chapter" data-level="10.9" data-path="actuarial.html"><a href="actuarial.html#conclusion-actuarial"><i class="fa fa-check"></i><b>10.9</b> Conclusion</a></li>
<li class="chapter" data-level="10.10" data-path="actuarial.html"><a href="actuarial.html#readings-actuarial"><i class="fa fa-check"></i><b>10.10</b> Suggested Readings</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="clinical.html"><a href="clinical.html"><i class="fa fa-check"></i><b>11</b> General Issues in Clinical Assessment</a>
<ul>
<li class="chapter" data-level="11.1" data-path="clinical.html"><a href="clinical.html#historicalPerspectives-clinical"><i class="fa fa-check"></i><b>11.1</b> Historical Perspectives on Clinical Assessment</a></li>
<li class="chapter" data-level="11.2" data-path="clinical.html"><a href="clinical.html#contemporaryTrends-clinical"><i class="fa fa-check"></i><b>11.2</b> Contemporary Trends</a></li>
<li class="chapter" data-level="11.3" data-path="clinical.html"><a href="clinical.html#terminology-clinical"><i class="fa fa-check"></i><b>11.3</b> Terminology</a>
<ul>
<li class="chapter" data-level="11.3.1" data-path="clinical.html"><a href="clinical.html#signsSymptomsSyndromes"><i class="fa fa-check"></i><b>11.3.1</b> The Three S’s: Signs, Symptoms, and Syndromes</a></li>
</ul></li>
<li class="chapter" data-level="11.4" data-path="clinical.html"><a href="clinical.html#pseudoPredictionErrors"><i class="fa fa-check"></i><b>11.4</b> Errors of Pseudo-Prediction</a></li>
<li class="chapter" data-level="11.5" data-path="clinical.html"><a href="clinical.html#conclusion-clinical"><i class="fa fa-check"></i><b>11.5</b> Conclusion</a></li>
<li class="chapter" data-level="11.6" data-path="clinical.html"><a href="clinical.html#readings-clinical"><i class="fa fa-check"></i><b>11.6</b> Suggested Readings</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="evidence-based.html"><a href="evidence-based.html"><i class="fa fa-check"></i><b>12</b> Evidence-Based Assessment</a>
<ul>
<li class="chapter" data-level="12.1" data-path="evidence-based.html"><a href="evidence-based.html#considerations-evidence-based"><i class="fa fa-check"></i><b>12.1</b> Considerations</a></li>
<li class="chapter" data-level="12.2" data-path="evidence-based.html"><a href="evidence-based.html#clinicallyRelevant"><i class="fa fa-check"></i><b>12.2</b> Clinically Relevant</a></li>
<li class="chapter" data-level="12.3" data-path="evidence-based.html"><a href="evidence-based.html#culturallySensitive"><i class="fa fa-check"></i><b>12.3</b> Culturally Sensitive</a></li>
<li class="chapter" data-level="12.4" data-path="evidence-based.html"><a href="evidence-based.html#scientificallySound"><i class="fa fa-check"></i><b>12.4</b> Scientifically Sound</a>
<ul>
<li class="chapter" data-level="12.4.1" data-path="evidence-based.html"><a href="evidence-based.html#excellentTests"><i class="fa fa-check"></i><b>12.4.1</b> Standard for Excellent Tests</a></li>
</ul></li>
<li class="chapter" data-level="12.5" data-path="evidence-based.html"><a href="evidence-based.html#bayesianUpdating"><i class="fa fa-check"></i><b>12.5</b> Bayesian Updating</a></li>
<li class="chapter" data-level="12.6" data-path="evidence-based.html"><a href="evidence-based.html#dimensionalPsychopathology"><i class="fa fa-check"></i><b>12.6</b> Dimensional Approaches to Psychopathology</a>
<ul>
<li class="chapter" data-level="12.6.1" data-path="evidence-based.html"><a href="evidence-based.html#multiStageAssessment"><i class="fa fa-check"></i><b>12.6.1</b> Multi-Stage Approach to Assessment</a></li>
</ul></li>
<li class="chapter" data-level="12.7" data-path="evidence-based.html"><a href="evidence-based.html#reportingGuidelines"><i class="fa fa-check"></i><b>12.7</b> Reporting Guidelines for Publications</a></li>
<li class="chapter" data-level="12.8" data-path="evidence-based.html"><a href="evidence-based.html#manyMeasures"><i class="fa fa-check"></i><b>12.8</b> Many Measures Are Available</a></li>
<li class="chapter" data-level="12.9" data-path="evidence-based.html"><a href="evidence-based.html#conclusion-evidence-based"><i class="fa fa-check"></i><b>12.9</b> Conclusion</a></li>
<li class="chapter" data-level="12.10" data-path="evidence-based.html"><a href="evidence-based.html#readings-evidence-based"><i class="fa fa-check"></i><b>12.10</b> Suggested Readings</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="ethics.html"><a href="ethics.html"><i class="fa fa-check"></i><b>13</b> Ethical Issues in Assessment</a>
<ul>
<li class="chapter" data-level="13.1" data-path="ethics.html"><a href="ethics.html#belmontReport"><i class="fa fa-check"></i><b>13.1</b> Belmont Report</a>
<ul>
<li class="chapter" data-level="13.1.1" data-path="ethics.html"><a href="ethics.html#respectForPersons"><i class="fa fa-check"></i><b>13.1.1</b> Respect for Persons</a></li>
<li class="chapter" data-level="13.1.2" data-path="ethics.html"><a href="ethics.html#beneficence"><i class="fa fa-check"></i><b>13.1.2</b> Beneficence</a></li>
<li class="chapter" data-level="13.1.3" data-path="ethics.html"><a href="ethics.html#justice"><i class="fa fa-check"></i><b>13.1.3</b> Justice</a></li>
</ul></li>
<li class="chapter" data-level="13.2" data-path="ethics.html"><a href="ethics.html#myEthicalAdvice"><i class="fa fa-check"></i><b>13.2</b> My Ethical Advice</a>
<ul>
<li class="chapter" data-level="13.2.1" data-path="ethics.html"><a href="ethics.html#dontDoStupidThings"><i class="fa fa-check"></i><b>13.2.1</b> Don’t Do Stupid Things</a></li>
<li class="chapter" data-level="13.2.2" data-path="ethics.html"><a href="ethics.html#dontDoBadThings"><i class="fa fa-check"></i><b>13.2.2</b> Don’t Do Bad Things</a></li>
<li class="chapter" data-level="13.2.3" data-path="ethics.html"><a href="ethics.html#consult"><i class="fa fa-check"></i><b>13.2.3</b> Consult, Consult, Consult</a></li>
</ul></li>
<li class="chapter" data-level="13.3" data-path="ethics.html"><a href="ethics.html#APAethicsCode"><i class="fa fa-check"></i><b>13.3</b> APA Ethics Code</a>
<ul>
<li class="chapter" data-level="13.3.1" data-path="ethics.html"><a href="ethics.html#competence-APA"><i class="fa fa-check"></i><b>13.3.1</b> Competence</a></li>
<li class="chapter" data-level="13.3.2" data-path="ethics.html"><a href="ethics.html#basesForAssessments-APA"><i class="fa fa-check"></i><b>13.3.2</b> Bases for Assessments</a></li>
<li class="chapter" data-level="13.3.3" data-path="ethics.html"><a href="ethics.html#useOfAssessments-APA"><i class="fa fa-check"></i><b>13.3.3</b> Use of Assessments</a></li>
<li class="chapter" data-level="13.3.4" data-path="ethics.html"><a href="ethics.html#informedConsent-APA"><i class="fa fa-check"></i><b>13.3.4</b> Informed Consent in Assessments</a></li>
<li class="chapter" data-level="13.3.5" data-path="ethics.html"><a href="ethics.html#releaseOfTestData-APA"><i class="fa fa-check"></i><b>13.3.5</b> Release of Test Data</a></li>
<li class="chapter" data-level="13.3.6" data-path="ethics.html"><a href="ethics.html#otherGuidelines-APA"><i class="fa fa-check"></i><b>13.3.6</b> Other Guidelines</a></li>
<li class="chapter" data-level="13.3.7" data-path="ethics.html"><a href="ethics.html#maintainTestSecurity-APA"><i class="fa fa-check"></i><b>13.3.7</b> Maintain Test Security (Integrity)</a></li>
</ul></li>
<li class="chapter" data-level="13.4" data-path="ethics.html"><a href="ethics.html#AERAguidelines"><i class="fa fa-check"></i><b>13.4</b> AERA Guidelines</a></li>
<li class="chapter" data-level="13.5" data-path="ethics.html"><a href="ethics.html#clinicalReportWriting"><i class="fa fa-check"></i><b>13.5</b> Clinical Report Writing</a></li>
<li class="chapter" data-level="13.6" data-path="ethics.html"><a href="ethics.html#openScience"><i class="fa fa-check"></i><b>13.6</b> Open Science</a></li>
<li class="chapter" data-level="13.7" data-path="ethics.html"><a href="ethics.html#conclusion-ethics"><i class="fa fa-check"></i><b>13.7</b> Conclusion</a></li>
<li class="chapter" data-level="13.8" data-path="ethics.html"><a href="ethics.html#readings-ethics"><i class="fa fa-check"></i><b>13.8</b> Suggested Readings</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html"><i class="fa fa-check"></i><b>14</b> Factor Analysis and Principal Component Analysis</a>
<ul>
<li class="chapter" data-level="14.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#factorAnalysisOverview"><i class="fa fa-check"></i><b>14.1</b> Overview of Factor Analysis</a>
<ul>
<li class="chapter" data-level="14.1.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#exampleFactorModelsFromCorrelationMatrices"><i class="fa fa-check"></i><b>14.1.1</b> Example Factor Models from Correlation Matrices</a></li>
<li class="chapter" data-level="14.1.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#indeterminacy"><i class="fa fa-check"></i><b>14.1.2</b> Indeterminacy</a></li>
<li class="chapter" data-level="14.1.3" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#practicalConsiderations-factorAnalysis"><i class="fa fa-check"></i><b>14.1.3</b> Practical Considerations</a></li>
<li class="chapter" data-level="14.1.4" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#factorAnalysisDecisions"><i class="fa fa-check"></i><b>14.1.4</b> Decisions to Make in Factor Analysis</a>
<ul>
<li class="chapter" data-level="14.1.4.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#variablesToInclude-factorAnalysis"><i class="fa fa-check"></i><b>14.1.4.1</b> 1. Variables to Include and their Scaling</a></li>
<li class="chapter" data-level="14.1.4.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#methodOfFactorExtraction"><i class="fa fa-check"></i><b>14.1.4.2</b> 2. Method of Factor Extraction</a>
<ul>
<li class="chapter" data-level="14.1.4.2.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pca"><i class="fa fa-check"></i><b>14.1.4.2.1</b> Principal Component Analysis</a></li>
<li class="chapter" data-level="14.1.4.2.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#factorAnalysis"><i class="fa fa-check"></i><b>14.1.4.2.2</b> Factor Analysis</a></li>
</ul></li>
<li class="chapter" data-level="14.1.4.3" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#efa-cfa"><i class="fa fa-check"></i><b>14.1.4.3</b> 3. EFA or CFA</a>
<ul>
<li class="chapter" data-level="14.1.4.3.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#efa"><i class="fa fa-check"></i><b>14.1.4.3.1</b> Exploratory Factor Analysis (EFA)</a></li>
<li class="chapter" data-level="14.1.4.3.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#cfa"><i class="fa fa-check"></i><b>14.1.4.3.2</b> Confirmatory Factor Analysis (CFA)</a></li>
<li class="chapter" data-level="14.1.4.3.3" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#efa-cfa-esem"><i class="fa fa-check"></i><b>14.1.4.3.3</b> Exploratory Structural Equation Modeling (ESEM)</a></li>
</ul></li>
<li class="chapter" data-level="14.1.4.4" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#factorsToRetain"><i class="fa fa-check"></i><b>14.1.4.4</b> 4. How Many Factors to Retain</a></li>
<li class="chapter" data-level="14.1.4.5" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#factorRotation"><i class="fa fa-check"></i><b>14.1.4.5</b> 5. Factor Rotation</a></li>
<li class="chapter" data-level="14.1.4.6" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#modelSelectionInterpretation"><i class="fa fa-check"></i><b>14.1.4.6</b> 6. Model Selection and Interpretation</a></li>
</ul></li>
<li class="chapter" data-level="14.1.5" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#downfallOfFactorAnalysis"><i class="fa fa-check"></i><b>14.1.5</b> The Downfall of Factor Analysis</a></li>
<li class="chapter" data-level="14.1.6" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#whatToDoWithFactors"><i class="fa fa-check"></i><b>14.1.6</b> What to Do with Factors</a></li>
<li class="chapter" data-level="14.1.7" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#missingDataHandling-factorAnalysis"><i class="fa fa-check"></i><b>14.1.7</b> Missing Data Handling</a></li>
<li class="chapter" data-level="14.1.8" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#troubleshooting-factorAnalysis"><i class="fa fa-check"></i><b>14.1.8</b> Troubleshooting Improper Solutions</a></li>
</ul></li>
<li class="chapter" data-level="14.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#gettingStarted-factorAnalysis"><i class="fa fa-check"></i><b>14.2</b> Getting Started</a>
<ul>
<li class="chapter" data-level="14.2.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#loadLibraries-factorAnalysis"><i class="fa fa-check"></i><b>14.2.1</b> Load Libraries</a></li>
<li class="chapter" data-level="14.2.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#loadData-factorAnalysis"><i class="fa fa-check"></i><b>14.2.2</b> Load Data</a></li>
<li class="chapter" data-level="14.2.3" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#prepareData-factorAnalysis"><i class="fa fa-check"></i><b>14.2.3</b> Prepare Data</a>
<ul>
<li class="chapter" data-level="14.2.3.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#addMissingData-factorAnalysis"><i class="fa fa-check"></i><b>14.2.3.1</b> Add Missing Data</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="14.3" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#descriptiveStatsCorrelations-factorAnalysis"><i class="fa fa-check"></i><b>14.3</b> Descriptive Statistics and Correlations</a>
<ul>
<li class="chapter" data-level="14.3.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#descriptiveStats-factorAnalysis"><i class="fa fa-check"></i><b>14.3.1</b> Descriptive Statistics</a></li>
<li class="chapter" data-level="14.3.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#correlations-factorAnalysis"><i class="fa fa-check"></i><b>14.3.2</b> Correlations</a></li>
</ul></li>
<li class="chapter" data-level="14.4" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#factorAnalysisExamples"><i class="fa fa-check"></i><b>14.4</b> Factor Analysis</a>
<ul>
<li class="chapter" data-level="14.4.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#efaExamples"><i class="fa fa-check"></i><b>14.4.1</b> Exploratory Factor Analysis (EFA)</a>
<ul>
<li class="chapter" data-level="14.4.1.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#determineNumberOfFactors-efaExamples"><i class="fa fa-check"></i><b>14.4.1.1</b> Determine number of factors</a>
<ul>
<li class="chapter" data-level="14.4.1.1.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#screePlot"><i class="fa fa-check"></i><b>14.4.1.1.1</b> Scree Plot</a></li>
<li class="chapter" data-level="14.4.1.1.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#vssPlot"><i class="fa fa-check"></i><b>14.4.1.1.2</b> Very Simple Structure (VSS) Plot</a></li>
</ul></li>
<li class="chapter" data-level="14.4.1.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#runFactorAnalysis-efa"><i class="fa fa-check"></i><b>14.4.1.2</b> Run factor analysis</a>
<ul>
<li class="chapter" data-level="14.4.1.2.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#orthogonal-efa"><i class="fa fa-check"></i><b>14.4.1.2.1</b> Orthogonal (Varimax) rotation</a></li>
<li class="chapter" data-level="14.4.1.2.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#oblique-efa"><i class="fa fa-check"></i><b>14.4.1.2.2</b> Oblique (Oblimin) rotation</a></li>
</ul></li>
<li class="chapter" data-level="14.4.1.3" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#factorLoadings-efa"><i class="fa fa-check"></i><b>14.4.1.3</b> Factor Loadings</a>
<ul>
<li class="chapter" data-level="14.4.1.3.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#orthogonalFactorLoadings-efa"><i class="fa fa-check"></i><b>14.4.1.3.1</b> Orthogonal (Varimax) rotation</a></li>
<li class="chapter" data-level="14.4.1.3.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#obliqueFactorLoadings-efa"><i class="fa fa-check"></i><b>14.4.1.3.2</b> Oblique (Oblimin) rotation</a></li>
</ul></li>
<li class="chapter" data-level="14.4.1.4" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#fitEstimates-efa"><i class="fa fa-check"></i><b>14.4.1.4</b> Estimates of Model Fit</a>
<ul>
<li class="chapter" data-level="14.4.1.4.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#fitEstimatesOrthogonal-efa"><i class="fa fa-check"></i><b>14.4.1.4.1</b> Orthogonal (Varimax) rotation</a></li>
<li class="chapter" data-level="14.4.1.4.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#fitEstimatesOblique-efa"><i class="fa fa-check"></i><b>14.4.1.4.2</b> Oblique (Oblimin) rotation</a></li>
</ul></li>
<li class="chapter" data-level="14.4.1.5" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#factorScores-efa"><i class="fa fa-check"></i><b>14.4.1.5</b> Factor Scores</a>
<ul>
<li class="chapter" data-level="14.4.1.5.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#factorScoresOrthogonal-efa"><i class="fa fa-check"></i><b>14.4.1.5.1</b> Orthogonal (Varimax) rotation</a></li>
<li class="chapter" data-level="14.4.1.5.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#factorScoresOblique-efa"><i class="fa fa-check"></i><b>14.4.1.5.2</b> Oblique (Oblimin) rotation</a></li>
</ul></li>
<li class="chapter" data-level="14.4.1.6" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#factorScoresPlots-efa"><i class="fa fa-check"></i><b>14.4.1.6</b> Plots</a>
<ul>
<li class="chapter" data-level="14.4.1.6.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#factorScoresPlotsOrthogonal-efa"><i class="fa fa-check"></i><b>14.4.1.6.1</b> Orthogonal (Varimax) rotation</a></li>
<li class="chapter" data-level="14.4.1.6.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#factorScoresPlotsOblique-efa"><i class="fa fa-check"></i><b>14.4.1.6.2</b> Oblique (Oblimin) rotation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="14.4.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#cfaExamples"><i class="fa fa-check"></i><b>14.4.2</b> Confirmatory Factor Analysis (CFA)</a>
<ul>
<li class="chapter" data-level="14.4.2.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#syntax-cfa"><i class="fa fa-check"></i><b>14.4.2.1</b> Specify the model</a>
<ul>
<li class="chapter" data-level="14.4.2.1.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#tabular-cfa"><i class="fa fa-check"></i><b>14.4.2.1.1</b> Model syntax in table form:</a></li>
</ul></li>
<li class="chapter" data-level="14.4.2.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#fit-cfa"><i class="fa fa-check"></i><b>14.4.2.2</b> Fit the model</a></li>
<li class="chapter" data-level="14.4.2.3" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#output-cfa"><i class="fa fa-check"></i><b>14.4.2.3</b> Display summary output</a></li>
<li class="chapter" data-level="14.4.2.4" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#fitCriteria-cfa"><i class="fa fa-check"></i><b>14.4.2.4</b> Estimates of model fit</a></li>
<li class="chapter" data-level="14.4.2.5" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#residuals-cfa"><i class="fa fa-check"></i><b>14.4.2.5</b> Residuals</a></li>
<li class="chapter" data-level="14.4.2.6" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#modIndices-cfa"><i class="fa fa-check"></i><b>14.4.2.6</b> Modification indices</a></li>
<li class="chapter" data-level="14.4.2.7" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#factorScores-cfa"><i class="fa fa-check"></i><b>14.4.2.7</b> Factor scores</a>
<ul>
<li class="chapter" data-level="14.4.2.7.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#syntax-cfaEFA"><i class="fa fa-check"></i><b>14.4.2.7.1</b> Compare CFA factor scores to EFA factor scores</a></li>
</ul></li>
<li class="chapter" data-level="14.4.2.8" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#reliability-cfa"><i class="fa fa-check"></i><b>14.4.2.8</b> Internal Consistency Reliability</a></li>
<li class="chapter" data-level="14.4.2.9" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pathDiagram-cfa"><i class="fa fa-check"></i><b>14.4.2.9</b> Path diagram</a></li>
<li class="chapter" data-level="14.4.2.10" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#classExamples-cfa"><i class="fa fa-check"></i><b>14.4.2.10</b> Class Examples</a>
<ul>
<li class="chapter" data-level="14.4.2.10.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#classExamplesModel1-cfa"><i class="fa fa-check"></i><b>14.4.2.10.1</b> Model 1</a></li>
<li class="chapter" data-level="14.4.2.10.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#classExamplesModel2-cfa"><i class="fa fa-check"></i><b>14.4.2.10.2</b> Model 2</a></li>
<li class="chapter" data-level="14.4.2.10.3" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#classExamplesModel3-cfa"><i class="fa fa-check"></i><b>14.4.2.10.3</b> Model 3</a></li>
<li class="chapter" data-level="14.4.2.10.4" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#model4Examples"><i class="fa fa-check"></i><b>14.4.2.10.4</b> Model 4</a></li>
<li class="chapter" data-level="14.4.2.10.5" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#classExamplesModel4-cfa"><i class="fa fa-check"></i><b>14.4.2.10.5</b> Create the covariance matrix</a></li>
</ul></li>
<li class="chapter" data-level="14.4.2.11" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#higherOrderModel"><i class="fa fa-check"></i><b>14.4.2.11</b> Higher-Order Factor Model</a></li>
<li class="chapter" data-level="14.4.2.12" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#bifactorModel"><i class="fa fa-check"></i><b>14.4.2.12</b> Bifactor Model</a>
<ul>
<li class="chapter" data-level="14.4.2.12.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#bifactorModelSyntax"><i class="fa fa-check"></i><b>14.4.2.12.1</b> Specify the model</a></li>
<li class="chapter" data-level="14.4.2.12.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#bifactorModelTabular"><i class="fa fa-check"></i><b>14.4.2.12.2</b> Model syntax in table form:</a></li>
<li class="chapter" data-level="14.4.2.12.3" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#bifactorModelFit"><i class="fa fa-check"></i><b>14.4.2.12.3</b> Fit the model</a></li>
<li class="chapter" data-level="14.4.2.12.4" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#display-summary-output"><i class="fa fa-check"></i><b>14.4.2.12.4</b> Display summary output</a></li>
<li class="chapter" data-level="14.4.2.12.5" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#bifactorModelFitCriteria"><i class="fa fa-check"></i><b>14.4.2.12.5</b> Estimates of model fit</a></li>
<li class="chapter" data-level="14.4.2.12.6" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#bifactorModelResiduals"><i class="fa fa-check"></i><b>14.4.2.12.6</b> Residuals</a></li>
<li class="chapter" data-level="14.4.2.12.7" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#bifactorModelModIndices"><i class="fa fa-check"></i><b>14.4.2.12.7</b> Modification indices</a></li>
<li class="chapter" data-level="14.4.2.12.8" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#bifactorModelReliability"><i class="fa fa-check"></i><b>14.4.2.12.8</b> Internal Consistency Reliability</a></li>
<li class="chapter" data-level="14.4.2.12.9" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#bifactorModelPathDiagram"><i class="fa fa-check"></i><b>14.4.2.12.9</b> Path diagram</a></li>
<li class="chapter" data-level="14.4.2.12.10" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#bifactorModelEquivalent"><i class="fa fa-check"></i><b>14.4.2.12.10</b> Equivalently fitting models</a></li>
</ul></li>
<li class="chapter" data-level="14.4.2.13" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#mtmmCFA"><i class="fa fa-check"></i><b>14.4.2.13</b> Multitrait-Multimethod Model (MTMM)</a>
<ul>
<li class="chapter" data-level="14.4.2.13.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#mtmmSimulate"><i class="fa fa-check"></i><b>14.4.2.13.1</b> Simulate the data</a></li>
<li class="chapter" data-level="14.4.2.13.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#mtmmCFAsyntax"><i class="fa fa-check"></i><b>14.4.2.13.2</b> Specify the model</a></li>
<li class="chapter" data-level="14.4.2.13.3" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#mtmmCFAtabular"><i class="fa fa-check"></i><b>14.4.2.13.3</b> Model syntax in table form:</a></li>
<li class="chapter" data-level="14.4.2.13.4" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#mtmmCFAfit"><i class="fa fa-check"></i><b>14.4.2.13.4</b> Fit the model</a></li>
<li class="chapter" data-level="14.4.2.13.5" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#mtmmCFAoutput"><i class="fa fa-check"></i><b>14.4.2.13.5</b> Display summary output</a></li>
<li class="chapter" data-level="14.4.2.13.6" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#mtmmCFAfitCriteria"><i class="fa fa-check"></i><b>14.4.2.13.6</b> Estimates of model fit</a></li>
<li class="chapter" data-level="14.4.2.13.7" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#mtmmCFAresiduals"><i class="fa fa-check"></i><b>14.4.2.13.7</b> Residuals</a></li>
<li class="chapter" data-level="14.4.2.13.8" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#mtmmCFAmodIndices"><i class="fa fa-check"></i><b>14.4.2.13.8</b> Modification indices</a></li>
<li class="chapter" data-level="14.4.2.13.9" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#mtmmCFAreliability"><i class="fa fa-check"></i><b>14.4.2.13.9</b> Internal Consistency Reliability</a></li>
<li class="chapter" data-level="14.4.2.13.10" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#mtmmCFApathDiagram"><i class="fa fa-check"></i><b>14.4.2.13.10</b> Path diagram</a></li>
<li class="chapter" data-level="14.4.2.13.11" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#mtmmCFAequivalent"><i class="fa fa-check"></i><b>14.4.2.13.11</b> Equivalently fitting models</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="14.4.3" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModel"><i class="fa fa-check"></i><b>14.4.3</b> Exploratory Structural Equation Model (ESEM)</a>
<ul>
<li class="chapter" data-level="14.4.3.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModelExample1"><i class="fa fa-check"></i><b>14.4.3.1</b> Example 1</a>
<ul>
<li class="chapter" data-level="14.4.3.1.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModelSyntax"><i class="fa fa-check"></i><b>14.4.3.1.1</b> Specify the model</a></li>
<li class="chapter" data-level="14.4.3.1.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModelTabular"><i class="fa fa-check"></i><b>14.4.3.1.2</b> Model syntax in table form:</a></li>
<li class="chapter" data-level="14.4.3.1.3" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModelFit"><i class="fa fa-check"></i><b>14.4.3.1.3</b> Fit the model</a></li>
<li class="chapter" data-level="14.4.3.1.4" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemOutput"><i class="fa fa-check"></i><b>14.4.3.1.4</b> Display summary output</a></li>
<li class="chapter" data-level="14.4.3.1.5" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModelFitCriteria"><i class="fa fa-check"></i><b>14.4.3.1.5</b> Estimates of model fit</a></li>
<li class="chapter" data-level="14.4.3.1.6" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModelFitComparison"><i class="fa fa-check"></i><b>14.4.3.1.6</b> Nested model comparison</a></li>
<li class="chapter" data-level="14.4.3.1.7" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModelResiduals"><i class="fa fa-check"></i><b>14.4.3.1.7</b> Residuals</a></li>
<li class="chapter" data-level="14.4.3.1.8" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModelModIndices"><i class="fa fa-check"></i><b>14.4.3.1.8</b> Modification indices</a></li>
<li class="chapter" data-level="14.4.3.1.9" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModelReliability"><i class="fa fa-check"></i><b>14.4.3.1.9</b> Internal Consistency Reliability</a></li>
<li class="chapter" data-level="14.4.3.1.10" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModelPathDiagram"><i class="fa fa-check"></i><b>14.4.3.1.10</b> Path diagram</a></li>
<li class="chapter" data-level="14.4.3.1.11" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModelEquivalent"><i class="fa fa-check"></i><b>14.4.3.1.11</b> Equivalently fitting models</a></li>
</ul></li>
<li class="chapter" data-level="14.4.3.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModelExample2"><i class="fa fa-check"></i><b>14.4.3.2</b> Example 2</a>
<ul>
<li class="chapter" data-level="14.4.3.2.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModelSyntax2"><i class="fa fa-check"></i><b>14.4.3.2.1</b> Specify the model</a></li>
<li class="chapter" data-level="14.4.3.2.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModelTabular2"><i class="fa fa-check"></i><b>14.4.3.2.2</b> Model syntax in table form:</a></li>
<li class="chapter" data-level="14.4.3.2.3" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModelFit2"><i class="fa fa-check"></i><b>14.4.3.2.3</b> Fit the model</a></li>
<li class="chapter" data-level="14.4.3.2.4" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemOutput2"><i class="fa fa-check"></i><b>14.4.3.2.4</b> Display summary output</a></li>
<li class="chapter" data-level="14.4.3.2.5" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModelFitCriteria2"><i class="fa fa-check"></i><b>14.4.3.2.5</b> Estimates of model fit</a></li>
<li class="chapter" data-level="14.4.3.2.6" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModelResiduals2"><i class="fa fa-check"></i><b>14.4.3.2.6</b> Residuals</a></li>
<li class="chapter" data-level="14.4.3.2.7" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModelModIndices2"><i class="fa fa-check"></i><b>14.4.3.2.7</b> Modification indices</a></li>
<li class="chapter" data-level="14.4.3.2.8" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModelReliability2"><i class="fa fa-check"></i><b>14.4.3.2.8</b> Internal Consistency Reliability</a></li>
<li class="chapter" data-level="14.4.3.2.9" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModelPathDiagram2"><i class="fa fa-check"></i><b>14.4.3.2.9</b> Path diagram</a></li>
<li class="chapter" data-level="14.4.3.2.10" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#esemModelEquivalent2"><i class="fa fa-check"></i><b>14.4.3.2.10</b> Equivalently fitting models</a></li>
</ul></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="14.5" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pcaAnalysis"><i class="fa fa-check"></i><b>14.5</b> Principal Component Analysis (PCA)</a>
<ul>
<li class="chapter" data-level="14.5.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pcaAnalysisDetermineNumberOfComponents"><i class="fa fa-check"></i><b>14.5.1</b> Determine number of components</a>
<ul>
<li class="chapter" data-level="14.5.1.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pcaAnalysisScreePlot"><i class="fa fa-check"></i><b>14.5.1.1</b> Scree Plot</a></li>
<li class="chapter" data-level="14.5.1.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pcaAnalysisVSSplot"><i class="fa fa-check"></i><b>14.5.1.2</b> Very Simple Structure (VSS) Plot</a>
<ul>
<li class="chapter" data-level="14.5.1.2.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pcaAnalysisVSSplotOrthogonal"><i class="fa fa-check"></i><b>14.5.1.2.1</b> Orthogonal (Varimax) rotation</a></li>
<li class="chapter" data-level="14.5.1.2.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pcaAnalysisVSSplotOblique"><i class="fa fa-check"></i><b>14.5.1.2.2</b> Oblique (Oblimin) rotation</a></li>
<li class="chapter" data-level="14.5.1.2.3" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pcaAnalysisVSSplotNoRotation"><i class="fa fa-check"></i><b>14.5.1.2.3</b> No rotation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="14.5.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pcaAnalysisFit"><i class="fa fa-check"></i><b>14.5.2</b> Run the Principal Component Analysis</a>
<ul>
<li class="chapter" data-level="14.5.2.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pcaAnalysisFitOrthogonal"><i class="fa fa-check"></i><b>14.5.2.1</b> Orthogonal (Varimax) rotation</a></li>
<li class="chapter" data-level="14.5.2.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pcaAnalysisFitOblique"><i class="fa fa-check"></i><b>14.5.2.2</b> Oblique (Oblimin) rotation</a></li>
</ul></li>
<li class="chapter" data-level="14.5.3" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pcaLoadings"><i class="fa fa-check"></i><b>14.5.3</b> Component Loadings</a>
<ul>
<li class="chapter" data-level="14.5.3.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pcaAnalysisLoadingsOrthogonal"><i class="fa fa-check"></i><b>14.5.3.1</b> Orthogonal (Varimax) rotation</a></li>
<li class="chapter" data-level="14.5.3.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pcaAnalysisLoadingsOblique"><i class="fa fa-check"></i><b>14.5.3.2</b> Oblique (Oblimin) rotation</a></li>
</ul></li>
<li class="chapter" data-level="14.5.4" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pcaAnalysisComponentScores"><i class="fa fa-check"></i><b>14.5.4</b> Component Scores</a>
<ul>
<li class="chapter" data-level="14.5.4.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pcaAnalysisComponentScoresOrthogonal"><i class="fa fa-check"></i><b>14.5.4.1</b> Orthogonal (Varimax) rotation</a></li>
<li class="chapter" data-level="14.5.4.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pcaAnalysisComponentScoresOblique"><i class="fa fa-check"></i><b>14.5.4.2</b> Oblique (Oblimin) rotation</a></li>
</ul></li>
<li class="chapter" data-level="14.5.5" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pcaAnalysisPlots"><i class="fa fa-check"></i><b>14.5.5</b> Plots</a>
<ul>
<li class="chapter" data-level="14.5.5.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pcaBiplot"><i class="fa fa-check"></i><b>14.5.5.1</b> Biplot</a></li>
<li class="chapter" data-level="14.5.5.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pcaAnalysisPlotsOrthogonal"><i class="fa fa-check"></i><b>14.5.5.2</b> Orthogonal (Varimax) rotation</a></li>
<li class="chapter" data-level="14.5.5.3" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#pcaAnalysisPlotsOblique"><i class="fa fa-check"></i><b>14.5.5.3</b> Oblique (Oblimin) rotation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="14.6" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#conclusion-factorAnalysis"><i class="fa fa-check"></i><b>14.6</b> Conclusion</a></li>
<li class="chapter" data-level="14.7" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#readings-factorAnalysis"><i class="fa fa-check"></i><b>14.7</b> Suggested Readings</a></li>
<li class="chapter" data-level="14.8" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#exercises-factorAnalysis"><i class="fa fa-check"></i><b>14.8</b> Exercises</a>
<ul>
<li class="chapter" data-level="14.8.1" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#exercisesQuestions-factorAnalysis"><i class="fa fa-check"></i><b>14.8.1</b> Questions</a></li>
<li class="chapter" data-level="14.8.2" data-path="factor-analysis-PCA.html"><a href="factor-analysis-PCA.html#exercisesAnswers-factorAnalysis"><i class="fa fa-check"></i><b>14.8.2</b> Answers</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>II Part II</b></span></li>
<li class="chapter" data-level="15" data-path="intellectual.html"><a href="intellectual.html"><i class="fa fa-check"></i><b>15</b> Intellectual Assessment</a>
<ul>
<li class="chapter" data-level="15.1" data-path="intellectual.html"><a href="intellectual.html#intelligence"><i class="fa fa-check"></i><b>15.1</b> Defining Intelligence</a></li>
<li class="chapter" data-level="15.2" data-path="intellectual.html"><a href="intellectual.html#historyIntelligence"><i class="fa fa-check"></i><b>15.2</b> History of Intelligence Research</a>
<ul>
<li class="chapter" data-level="15.2.1" data-path="intellectual.html"><a href="intellectual.html#francisGalton"><i class="fa fa-check"></i><b>15.2.1</b> Sir Francis Galton (1822–1911)</a></li>
<li class="chapter" data-level="15.2.2" data-path="intellectual.html"><a href="intellectual.html#jamesCattell"><i class="fa fa-check"></i><b>15.2.2</b> James Cattell (1860–1944)</a></li>
<li class="chapter" data-level="15.2.3" data-path="intellectual.html"><a href="intellectual.html#alfredBinet"><i class="fa fa-check"></i><b>15.2.3</b> Alfred Binet (1857–1911)</a></li>
<li class="chapter" data-level="15.2.4" data-path="intellectual.html"><a href="intellectual.html#lewisTerman"><i class="fa fa-check"></i><b>15.2.4</b> Lewis Terman (1877–1956)</a></li>
<li class="chapter" data-level="15.2.5" data-path="intellectual.html"><a href="intellectual.html#williamStern"><i class="fa fa-check"></i><b>15.2.5</b> William Stern (1871–1938)</a></li>
<li class="chapter" data-level="15.2.6" data-path="intellectual.html"><a href="intellectual.html#robertYerkes"><i class="fa fa-check"></i><b>15.2.6</b> Robert Yerkes (1876–1956)</a></li>
<li class="chapter" data-level="15.2.7" data-path="intellectual.html"><a href="intellectual.html#charlesSpearman"><i class="fa fa-check"></i><b>15.2.7</b> Charles Spearman (1863–1945)</a></li>
<li class="chapter" data-level="15.2.8" data-path="intellectual.html"><a href="intellectual.html#louisLeonThurstone"><i class="fa fa-check"></i><b>15.2.8</b> Louis Leon Thurstone (1887–1955)</a></li>
<li class="chapter" data-level="15.2.9" data-path="intellectual.html"><a href="intellectual.html#fluidCrystallizedIntelligence"><i class="fa fa-check"></i><b>15.2.9</b> Raymond Cattell (1905–1998)</a></li>
<li class="chapter" data-level="15.2.10" data-path="intellectual.html"><a href="intellectual.html#johnHorn"><i class="fa fa-check"></i><b>15.2.10</b> John Horn (1926–2006)</a></li>
<li class="chapter" data-level="15.2.11" data-path="intellectual.html"><a href="intellectual.html#chcModel"><i class="fa fa-check"></i><b>15.2.11</b> John Carroll (1916–2003)</a></li>
<li class="chapter" data-level="15.2.12" data-path="intellectual.html"><a href="intellectual.html#davidWechsler"><i class="fa fa-check"></i><b>15.2.12</b> David Wechsler (1896–1981)</a></li>
</ul></li>
<li class="chapter" data-level="15.3" data-path="intellectual.html"><a href="intellectual.html#alternativeConceptualizationsOfIntelligence"><i class="fa fa-check"></i><b>15.3</b> Alternative Conceptualizations of Intelligence</a></li>
<li class="chapter" data-level="15.4" data-path="intellectual.html"><a href="intellectual.html#purposesOfIntelligenceTests"><i class="fa fa-check"></i><b>15.4</b> Purposes of Intelligence Tests</a></li>
<li class="chapter" data-level="15.5" data-path="intellectual.html"><a href="intellectual.html#achievementAptitude"><i class="fa fa-check"></i><b>15.5</b> Intelligence Versus Achievement Versus Aptitude</a></li>
<li class="chapter" data-level="15.6" data-path="intellectual.html"><a href="intellectual.html#intelligenceTheory"><i class="fa fa-check"></i><b>15.6</b> Theory Influences Intepretation of Scores</a></li>
<li class="chapter" data-level="15.7" data-path="intellectual.html"><a href="intellectual.html#intelligenceTime"><i class="fa fa-check"></i><b>15.7</b> Time-Related Influences</a></li>
<li class="chapter" data-level="15.8" data-path="intellectual.html"><a href="intellectual.html#intelligenceTestsConcerns"><i class="fa fa-check"></i><b>15.8</b> Concerns with Intelligence Tests</a></li>
<li class="chapter" data-level="15.9" data-path="intellectual.html"><a href="intellectual.html#aptitudeTesting"><i class="fa fa-check"></i><b>15.9</b> Aptitude Testing</a></li>
<li class="chapter" data-level="15.10" data-path="intellectual.html"><a href="intellectual.html#iqScales"><i class="fa fa-check"></i><b>15.10</b> Scales</a>
<ul>
<li class="chapter" data-level="15.10.1" data-path="intellectual.html"><a href="intellectual.html#ratioIQ"><i class="fa fa-check"></i><b>15.10.1</b> Ratio IQ</a></li>
<li class="chapter" data-level="15.10.2" data-path="intellectual.html"><a href="intellectual.html#deviationIQ"><i class="fa fa-check"></i><b>15.10.2</b> Deviation IQ</a></li>
<li class="chapter" data-level="15.10.3" data-path="intellectual.html"><a href="intellectual.html#iqDataType"><i class="fa fa-check"></i><b>15.10.3</b> Nominal, Ordinal, Interval, or Ratio?</a></li>
</ul></li>
<li class="chapter" data-level="15.11" data-path="intellectual.html"><a href="intellectual.html#conclusion-intelligence"><i class="fa fa-check"></i><b>15.11</b> Conclusion</a></li>
<li class="chapter" data-level="15.12" data-path="intellectual.html"><a href="intellectual.html#readings-intelligence"><i class="fa fa-check"></i><b>15.12</b> Suggested Readings</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="bias.html"><a href="bias.html"><i class="fa fa-check"></i><b>16</b> Test Bias</a>
<ul>
<li class="chapter" data-level="16.1" data-path="bias.html"><a href="bias.html#overview-bias"><i class="fa fa-check"></i><b>16.1</b> Overview of Bias</a></li>
<li class="chapter" data-level="16.2" data-path="bias.html"><a href="bias.html#detectBias"><i class="fa fa-check"></i><b>16.2</b> Ways to Investigate/Detect Test Bias</a>
<ul>
<li class="chapter" data-level="16.2.1" data-path="bias.html"><a href="bias.html#predictiveBias"><i class="fa fa-check"></i><b>16.2.1</b> Predictive Bias</a>
<ul>
<li class="chapter" data-level="16.2.1.1" data-path="bias.html"><a href="bias.html#typesPredictiveBias"><i class="fa fa-check"></i><b>16.2.1.1</b> Types of Predictive Bias</a>
<ul>
<li class="chapter" data-level="16.2.1.1.1" data-path="bias.html"><a href="bias.html#differentSlopes"><i class="fa fa-check"></i><b>16.2.1.1.1</b> Different Slopes</a></li>
<li class="chapter" data-level="16.2.1.1.2" data-path="bias.html"><a href="bias.html#differentIntercepts"><i class="fa fa-check"></i><b>16.2.1.1.2</b> Different Intercepts</a></li>
<li class="chapter" data-level="16.2.1.1.3" data-path="bias.html"><a href="bias.html#differentInterceptsAndSlopes"><i class="fa fa-check"></i><b>16.2.1.1.3</b> Different Intercepts and Slopes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="16.2.2" data-path="bias.html"><a href="bias.html#testStructureBias"><i class="fa fa-check"></i><b>16.2.2</b> Test Structure Bias</a>
<ul>
<li class="chapter" data-level="16.2.2.1" data-path="bias.html"><a href="bias.html#testStructureBiasEmpirical"><i class="fa fa-check"></i><b>16.2.2.1</b> Empirical Approaches to Identification</a>
<ul>
<li class="chapter" data-level="16.2.2.1.1" data-path="bias.html"><a href="bias.html#itemXgroupTests-bias"><i class="fa fa-check"></i><b>16.2.2.1.1</b> Item <span class="math inline">\(\times\)</span> Group tests (ANOVA)</a></li>
<li class="chapter" data-level="16.2.2.1.2" data-path="bias.html"><a href="bias.html#irt-bias"><i class="fa fa-check"></i><b>16.2.2.1.2</b> Item Response Theory</a></li>
<li class="chapter" data-level="16.2.2.1.3" data-path="bias.html"><a href="bias.html#cfa-bias"><i class="fa fa-check"></i><b>16.2.2.1.3</b> Confirmatory Factor Analysis</a></li>
<li class="chapter" data-level="16.2.2.1.4" data-path="bias.html"><a href="bias.html#sem-bias"><i class="fa fa-check"></i><b>16.2.2.1.4</b> Structural Equation Modeling</a></li>
<li class="chapter" data-level="16.2.2.1.5" data-path="bias.html"><a href="bias.html#sdt-bias"><i class="fa fa-check"></i><b>16.2.2.1.5</b> Signal Detection Theory</a></li>
<li class="chapter" data-level="16.2.2.1.6" data-path="bias.html"><a href="bias.html#evidenceTestStructureBias"><i class="fa fa-check"></i><b>16.2.2.1.6</b> Empirical Evidence of Test Structure Bias</a></li>
</ul></li>
<li class="chapter" data-level="16.2.2.2" data-path="bias.html"><a href="bias.html#testStructureBiasTheoretical"><i class="fa fa-check"></i><b>16.2.2.2</b> Theoretical/Judgmental Approaches to Identification</a>
<ul>
<li class="chapter" data-level="16.2.2.2.1" data-path="bias.html"><a href="bias.html#facialValidityBias"><i class="fa fa-check"></i><b>16.2.2.2.1</b> Facial Validity Bias</a></li>
<li class="chapter" data-level="16.2.2.2.2" data-path="bias.html"><a href="bias.html#contentValidityBias"><i class="fa fa-check"></i><b>16.2.2.2.2</b> Content Validity Bias</a></li>
</ul></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="16.3" data-path="bias.html"><a href="bias.html#biasExamples"><i class="fa fa-check"></i><b>16.3</b> Examples of Bias</a></li>
<li class="chapter" data-level="16.4" data-path="bias.html"><a href="bias.html#fairness"><i class="fa fa-check"></i><b>16.4</b> Test Fairness</a>
<ul>
<li class="chapter" data-level="16.4.1" data-path="bias.html"><a href="bias.html#adverseImpact"><i class="fa fa-check"></i><b>16.4.1</b> Adverse Impact</a></li>
<li class="chapter" data-level="16.4.2" data-path="bias.html"><a href="bias.html#biasVsFairness"><i class="fa fa-check"></i><b>16.4.2</b> Bias Versus Fairness</a></li>
<li class="chapter" data-level="16.4.3" data-path="bias.html"><a href="bias.html#operationalizingFairness"><i class="fa fa-check"></i><b>16.4.3</b> Operationalizing Fairness</a></li>
</ul></li>
<li class="chapter" data-level="16.5" data-path="bias.html"><a href="bias.html#correctForBias"><i class="fa fa-check"></i><b>16.5</b> Correcting For Bias</a>
<ul>
<li class="chapter" data-level="16.5.1" data-path="bias.html"><a href="bias.html#whenDetectingBias"><i class="fa fa-check"></i><b>16.5.1</b> What to Do When Detecting Bias</a></li>
<li class="chapter" data-level="16.5.2" data-path="bias.html"><a href="bias.html#scoreAdjustment"><i class="fa fa-check"></i><b>16.5.2</b> Score Adjustment to Correct for Bias</a>
<ul>
<li class="chapter" data-level="16.5.2.1" data-path="bias.html"><a href="bias.html#whyAdjustScores"><i class="fa fa-check"></i><b>16.5.2.1</b> Why Adjust Scores?</a></li>
<li class="chapter" data-level="16.5.2.2" data-path="bias.html"><a href="bias.html#typesOfScoreAdjustment"><i class="fa fa-check"></i><b>16.5.2.2</b> Types of Score Adjustment</a>
<ul>
<li class="chapter" data-level="16.5.2.2.1" data-path="bias.html"><a href="bias.html#bonusPoints"><i class="fa fa-check"></i><b>16.5.2.2.1</b> Bonus Points</a></li>
<li class="chapter" data-level="16.5.2.2.2" data-path="bias.html"><a href="bias.html#withinGroupNorming"><i class="fa fa-check"></i><b>16.5.2.2.2</b> Within-Group Norming</a></li>
<li class="chapter" data-level="16.5.2.2.3" data-path="bias.html"><a href="bias.html#separateCutoffs"><i class="fa fa-check"></i><b>16.5.2.2.3</b> Separate Cutoffs</a></li>
<li class="chapter" data-level="16.5.2.2.4" data-path="bias.html"><a href="bias.html#topDownSelection"><i class="fa fa-check"></i><b>16.5.2.2.4</b> Top-Down Selection from Different Lists</a></li>
<li class="chapter" data-level="16.5.2.2.5" data-path="bias.html"><a href="bias.html#banding"><i class="fa fa-check"></i><b>16.5.2.2.5</b> Banding</a></li>
<li class="chapter" data-level="16.5.2.2.6" data-path="bias.html"><a href="bias.html#bandingBonusPoints"><i class="fa fa-check"></i><b>16.5.2.2.6</b> Banding with Bonus Points</a></li>
<li class="chapter" data-level="16.5.2.2.7" data-path="bias.html"><a href="bias.html#slidingBand"><i class="fa fa-check"></i><b>16.5.2.2.7</b> Sliding Band</a></li>
<li class="chapter" data-level="16.5.2.2.8" data-path="bias.html"><a href="bias.html#separateTests"><i class="fa fa-check"></i><b>16.5.2.2.8</b> Separate Tests</a></li>
<li class="chapter" data-level="16.5.2.2.9" data-path="bias.html"><a href="bias.html#itemElimination"><i class="fa fa-check"></i><b>16.5.2.2.9</b> Item Elimination based on Group Differences</a></li>
</ul></li>
<li class="chapter" data-level="16.5.2.3" data-path="bias.html"><a href="bias.html#useOfScoreAdjustment"><i class="fa fa-check"></i><b>16.5.2.3</b> Use of Score Adjustment</a></li>
</ul></li>
<li class="chapter" data-level="16.5.3" data-path="bias.html"><a href="bias.html#otherBiasCorrections"><i class="fa fa-check"></i><b>16.5.3</b> Other Ways to Correct for Bias</a>
<ul>
<li class="chapter" data-level="16.5.3.1" data-path="bias.html"><a href="bias.html#useMultiplePredictors"><i class="fa fa-check"></i><b>16.5.3.1</b> Use Multiple Predictors</a></li>
<li class="chapter" data-level="16.5.3.2" data-path="bias.html"><a href="bias.html#changeCriterion"><i class="fa fa-check"></i><b>16.5.3.2</b> Change the Criterion</a></li>
<li class="chapter" data-level="16.5.3.3" data-path="bias.html"><a href="bias.html#removeBiasedItems"><i class="fa fa-check"></i><b>16.5.3.3</b> Remove Biased Items</a></li>
<li class="chapter" data-level="16.5.3.4" data-path="bias.html"><a href="bias.html#resolveBiasedItems"><i class="fa fa-check"></i><b>16.5.3.4</b> Resolve Biased Items</a></li>
<li class="chapter" data-level="16.5.3.5" data-path="bias.html"><a href="bias.html#alternativeTestingModes"><i class="fa fa-check"></i><b>16.5.3.5</b> Use Alternative Modes of Testing</a></li>
<li class="chapter" data-level="16.5.3.6" data-path="bias.html"><a href="bias.html#workRecords"><i class="fa fa-check"></i><b>16.5.3.6</b> Use Work Records</a></li>
<li class="chapter" data-level="16.5.3.7" data-path="bias.html"><a href="bias.html#increaseTimeLimit"><i class="fa fa-check"></i><b>16.5.3.7</b> Increase Time Limit</a></li>
<li class="chapter" data-level="16.5.3.8" data-path="bias.html"><a href="bias.html#motivationSets"><i class="fa fa-check"></i><b>16.5.3.8</b> Use Motivation Sets</a></li>
<li class="chapter" data-level="16.5.3.9" data-path="bias.html"><a href="bias.html#instructionalSets"><i class="fa fa-check"></i><b>16.5.3.9</b> Use Instructional Sets</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="16.6" data-path="bias.html"><a href="bias.html#gettingStarted-bias"><i class="fa fa-check"></i><b>16.6</b> Getting Started</a>
<ul>
<li class="chapter" data-level="16.6.1" data-path="bias.html"><a href="bias.html#loadLibraries-bias"><i class="fa fa-check"></i><b>16.6.1</b> Load Libraries</a></li>
<li class="chapter" data-level="16.6.2" data-path="bias.html"><a href="bias.html#prepareData-bias"><i class="fa fa-check"></i><b>16.6.2</b> Prepare Data</a>
<ul>
<li class="chapter" data-level="16.6.2.1" data-path="bias.html"><a href="bias.html#loadData-bias"><i class="fa fa-check"></i><b>16.6.2.1</b> Load Data</a></li>
<li class="chapter" data-level="16.6.2.2" data-path="bias.html"><a href="bias.html#simulateData-bias"><i class="fa fa-check"></i><b>16.6.2.2</b> Simulate Data</a></li>
<li class="chapter" data-level="16.6.2.3" data-path="bias.html"><a href="bias.html#addMissingData-bias"><i class="fa fa-check"></i><b>16.6.2.3</b> Add Missing Data</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="16.7" data-path="bias.html"><a href="bias.html#examplesUnbiasedTests"><i class="fa fa-check"></i><b>16.7</b> Examples of Unbiased Tests (in Terms of Predictive Bias)</a>
<ul>
<li class="chapter" data-level="16.7.1" data-path="bias.html"><a href="bias.html#unbiasedTestsEqualMeans"><i class="fa fa-check"></i><b>16.7.1</b> Unbiased test where males and females have equal means on predictor and criterion</a></li>
<li class="chapter" data-level="16.7.2" data-path="bias.html"><a href="bias.html#unbiasedTestsFemalesHigherMeans"><i class="fa fa-check"></i><b>16.7.2</b> Unbiased test where females have higher means than males on predictor and criterion</a></li>
<li class="chapter" data-level="16.7.3" data-path="bias.html"><a href="bias.html#unbiasedTestsMalesHigherMeans"><i class="fa fa-check"></i><b>16.7.3</b> Unbiased test where males have higher means than females on predictor and criterion</a></li>
</ul></li>
<li class="chapter" data-level="16.8" data-path="bias.html"><a href="bias.html#predictiveBiasExamples"><i class="fa fa-check"></i><b>16.8</b> Predictive Bias: Different Regression Lines</a>
<ul>
<li class="chapter" data-level="16.8.1" data-path="bias.html"><a href="bias.html#unbiasedPrediction"><i class="fa fa-check"></i><b>16.8.1</b> Example of unbiased prediction (no differences in intercepts or slopes)</a></li>
<li class="chapter" data-level="16.8.2" data-path="bias.html"><a href="bias.html#interceptBias"><i class="fa fa-check"></i><b>16.8.2</b> Example of intercept bias</a></li>
<li class="chapter" data-level="16.8.3" data-path="bias.html"><a href="bias.html#slopeBias"><i class="fa fa-check"></i><b>16.8.3</b> Example of slope bias</a></li>
<li class="chapter" data-level="16.8.4" data-path="bias.html"><a href="bias.html#interceptSlopeBias"><i class="fa fa-check"></i><b>16.8.4</b> Example of intercept and slope bias</a></li>
<li class="chapter" data-level="16.8.5" data-path="bias.html"><a href="bias.html#differentialReliability"><i class="fa fa-check"></i><b>16.8.5</b> Example of different measurement reliability/error across groups</a></li>
</ul></li>
<li class="chapter" data-level="16.9" data-path="bias.html"><a href="bias.html#dif"><i class="fa fa-check"></i><b>16.9</b> Differential Item Functioning (DIF)</a>
<ul>
<li class="chapter" data-level="16.9.1" data-path="bias.html"><a href="bias.html#descriptiveStats-bias"><i class="fa fa-check"></i><b>16.9.1</b> Item Descriptive Statistics</a></li>
<li class="chapter" data-level="16.9.2" data-path="bias.html"><a href="bias.html#difUnconstrained"><i class="fa fa-check"></i><b>16.9.2</b> Unconstrained Model</a>
<ul>
<li class="chapter" data-level="16.9.2.1" data-path="bias.html"><a href="bias.html#difUnconstrainedFit"><i class="fa fa-check"></i><b>16.9.2.1</b> Fit Model</a></li>
<li class="chapter" data-level="16.9.2.2" data-path="bias.html"><a href="bias.html#difUnconstrainedSummary"><i class="fa fa-check"></i><b>16.9.2.2</b> Model Summary</a></li>
</ul></li>
<li class="chapter" data-level="16.9.3" data-path="bias.html"><a href="bias.html#difConstrained"><i class="fa fa-check"></i><b>16.9.3</b> Constrained Model</a>
<ul>
<li class="chapter" data-level="16.9.3.1" data-path="bias.html"><a href="bias.html#difConstrainedFit"><i class="fa fa-check"></i><b>16.9.3.1</b> Fit Model</a></li>
<li class="chapter" data-level="16.9.3.2" data-path="bias.html"><a href="bias.html#difConstrainedSummary"><i class="fa fa-check"></i><b>16.9.3.2</b> Model Summary</a></li>
</ul></li>
<li class="chapter" data-level="16.9.4" data-path="bias.html"><a href="bias.html#difCompareConstrainedUnconstrained"><i class="fa fa-check"></i><b>16.9.4</b> Compare model fit of constrained model to unconstrained model</a></li>
<li class="chapter" data-level="16.9.5" data-path="bias.html"><a href="bias.html#difRemoveConstraints"><i class="fa fa-check"></i><b>16.9.5</b> Identify DIF by iteratively removing constraints from fully constrained model</a>
<ul>
<li class="chapter" data-level="16.9.5.1" data-path="bias.html"><a href="bias.html#difRemoveConstraintsDiscriminationSeverity"><i class="fa fa-check"></i><b>16.9.5.1</b> Items that differ in discrimination and/or severity</a></li>
<li class="chapter" data-level="16.9.5.2" data-path="bias.html"><a href="bias.html#difRemoveConstraintsDiscrimination"><i class="fa fa-check"></i><b>16.9.5.2</b> Items that differ in discrimination</a></li>
<li class="chapter" data-level="16.9.5.3" data-path="bias.html"><a href="bias.html#difRemoveConstraintsSeverity"><i class="fa fa-check"></i><b>16.9.5.3</b> Items that differ in severity</a></li>
</ul></li>
<li class="chapter" data-level="16.9.6" data-path="bias.html"><a href="bias.html#difAddConstraints"><i class="fa fa-check"></i><b>16.9.6</b> Identify DIF by iteratively adding constraints to unconstrained model</a>
<ul>
<li class="chapter" data-level="16.9.6.1" data-path="bias.html"><a href="bias.html#difAddConstraintsDiscriminationSeverity"><i class="fa fa-check"></i><b>16.9.6.1</b> Items that differ in discrimination and/or severity</a></li>
<li class="chapter" data-level="16.9.6.2" data-path="bias.html"><a href="bias.html#difAddConstraintsDiscrimination"><i class="fa fa-check"></i><b>16.9.6.2</b> Items that differ in discrimination</a></li>
<li class="chapter" data-level="16.9.6.3" data-path="bias.html"><a href="bias.html#difAddConstraintsSeverity"><i class="fa fa-check"></i><b>16.9.6.3</b> Items that differ in severity</a></li>
</ul></li>
<li class="chapter" data-level="16.9.7" data-path="bias.html"><a href="bias.html#difEffectSize"><i class="fa fa-check"></i><b>16.9.7</b> Compute effect size of DIF</a>
<ul>
<li class="chapter" data-level="16.9.7.1" data-path="bias.html"><a href="bias.html#difTestLevel"><i class="fa fa-check"></i><b>16.9.7.1</b> Test-level DIF</a></li>
<li class="chapter" data-level="16.9.7.2" data-path="bias.html"><a href="bias.html#difItemLevel"><i class="fa fa-check"></i><b>16.9.7.2</b> Item-level DIF</a></li>
</ul></li>
<li class="chapter" data-level="16.9.8" data-path="bias.html"><a href="bias.html#difItemPlots"><i class="fa fa-check"></i><b>16.9.8</b> Item plots</a></li>
<li class="chapter" data-level="16.9.9" data-path="bias.html"><a href="bias.html#addressingDIF"><i class="fa fa-check"></i><b>16.9.9</b> Addressing DIF</a>
<ul>
<li class="chapter" data-level="16.9.9.1" data-path="bias.html"><a href="bias.html#dropItemForBothGroups"><i class="fa fa-check"></i><b>16.9.9.1</b> Drop item for both groups</a></li>
<li class="chapter" data-level="16.9.9.2" data-path="bias.html"><a href="bias.html#dropItemForOneGroup"><i class="fa fa-check"></i><b>16.9.9.2</b> Drop item for one group but not another group</a></li>
<li class="chapter" data-level="16.9.9.3" data-path="bias.html"><a href="bias.html#resolveItemAcrossGroups"><i class="fa fa-check"></i><b>16.9.9.3</b> Freely estimate item to have different parameters across groups</a></li>
<li class="chapter" data-level="16.9.9.4" data-path="bias.html"><a href="bias.html#difResolveItemsCompareModels"><i class="fa fa-check"></i><b>16.9.9.4</b> Compare model fit</a></li>
<li class="chapter" data-level="16.9.9.5" data-path="bias.html"><a href="bias.html#difNextSteps"><i class="fa fa-check"></i><b>16.9.9.5</b> Next steps</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="16.10" data-path="bias.html"><a href="bias.html#measurementInvariance"><i class="fa fa-check"></i><b>16.10</b> Measurement/Factorial Invariance</a>
<ul>
<li class="chapter" data-level="16.10.1" data-path="bias.html"><a href="bias.html#measurementInvarianceModels"><i class="fa fa-check"></i><b>16.10.1</b> Specify Models</a>
<ul>
<li class="chapter" data-level="16.10.1.1" data-path="bias.html"><a href="bias.html#measurementInvarianceNullModelSyntax"><i class="fa fa-check"></i><b>16.10.1.1</b> Null Model</a></li>
<li class="chapter" data-level="16.10.1.2" data-path="bias.html"><a href="bias.html#measurementInvarianceCFAsyntax"><i class="fa fa-check"></i><b>16.10.1.2</b> CFA Model</a></li>
<li class="chapter" data-level="16.10.1.3" data-path="bias.html"><a href="bias.html#measurementInvarianceConfiguralSyntax"><i class="fa fa-check"></i><b>16.10.1.3</b> Configural Invariance</a></li>
<li class="chapter" data-level="16.10.1.4" data-path="bias.html"><a href="bias.html#measurementInvarianceMetricSyntax"><i class="fa fa-check"></i><b>16.10.1.4</b> Metric (“Weak Factorial”) Invariance</a></li>
<li class="chapter" data-level="16.10.1.5" data-path="bias.html"><a href="bias.html#measurementInvarianceScalarSyntax"><i class="fa fa-check"></i><b>16.10.1.5</b> Scalar (“Strong Factorial”) Invariance</a></li>
<li class="chapter" data-level="16.10.1.6" data-path="bias.html"><a href="bias.html#measurementInvarianceResidualSyntax"><i class="fa fa-check"></i><b>16.10.1.6</b> Residual (“Strict Factorial”) Invariance</a></li>
</ul></li>
<li class="chapter" data-level="16.10.2" data-path="bias.html"><a href="bias.html#measurementInvarianceFitIndices"><i class="fa fa-check"></i><b>16.10.2</b> Specify the Fit Indices of Interest</a></li>
<li class="chapter" data-level="16.10.3" data-path="bias.html"><a href="bias.html#measurementInvarianceNullModel"><i class="fa fa-check"></i><b>16.10.3</b> Null Model</a>
<ul>
<li class="chapter" data-level="16.10.3.1" data-path="bias.html"><a href="bias.html#measurementInvarianceNullModelFit"><i class="fa fa-check"></i><b>16.10.3.1</b> Fit the Model</a></li>
<li class="chapter" data-level="16.10.3.2" data-path="bias.html"><a href="bias.html#measurementInvarianceNullModelSummary"><i class="fa fa-check"></i><b>16.10.3.2</b> Model Summary</a></li>
</ul></li>
<li class="chapter" data-level="16.10.4" data-path="bias.html"><a href="bias.html#measurementInvarianceConfigural"><i class="fa fa-check"></i><b>16.10.4</b> Configural Invariance</a>
<ul>
<li class="chapter" data-level="16.10.4.1" data-path="bias.html"><a href="bias.html#measurementInvarianceConfiguralModelSyntax"><i class="fa fa-check"></i><b>16.10.4.1</b> Model Syntax</a>
<ul>
<li class="chapter" data-level="16.10.4.1.1" data-path="bias.html"><a href="bias.html#measurementInvarianceConfiguralModelFeatures"><i class="fa fa-check"></i><b>16.10.4.1.1</b> Summary of Model Features</a></li>
<li class="chapter" data-level="16.10.4.1.2" data-path="bias.html"><a href="bias.html#measurementInvarianceConfiguralModelTabular"><i class="fa fa-check"></i><b>16.10.4.1.2</b> Model Syntax in Table Form</a></li>
</ul></li>
<li class="chapter" data-level="16.10.4.2" data-path="bias.html"><a href="bias.html#measurementInvarianceConfiguralModelFit"><i class="fa fa-check"></i><b>16.10.4.2</b> Fit the Model</a></li>
<li class="chapter" data-level="16.10.4.3" data-path="bias.html"><a href="bias.html#measurementInvarianceConfiguralModelSummary"><i class="fa fa-check"></i><b>16.10.4.3</b> Model Summary</a></li>
<li class="chapter" data-level="16.10.4.4" data-path="bias.html"><a href="bias.html#measurementInvarianceConfiguralModelFitCriteria"><i class="fa fa-check"></i><b>16.10.4.4</b> Model Fit</a></li>
<li class="chapter" data-level="16.10.4.5" data-path="bias.html"><a href="bias.html#measurementInvarianceEffectSize"><i class="fa fa-check"></i><b>16.10.4.5</b> Effect size of non-invariance</a>
<ul>
<li class="chapter" data-level="16.10.4.5.1" data-path="bias.html"><a href="bias.html#dmacs"><i class="fa fa-check"></i><b>16.10.4.5.1</b> dMACS</a></li>
</ul></li>
<li class="chapter" data-level="16.10.4.6" data-path="bias.html"><a href="bias.html#measurementInvarianceConfiguralModelEquivalenceTest"><i class="fa fa-check"></i><b>16.10.4.6</b> Equivalence Test</a></li>
<li class="chapter" data-level="16.10.4.7" data-path="bias.html"><a href="bias.html#measurementInvarianceConfiguralModelPermutationTest"><i class="fa fa-check"></i><b>16.10.4.7</b> Permutation Test</a></li>
<li class="chapter" data-level="16.10.4.8" data-path="bias.html"><a href="bias.html#measurementInvarianceReliability"><i class="fa fa-check"></i><b>16.10.4.8</b> Internal Consistency Reliability</a></li>
<li class="chapter" data-level="16.10.4.9" data-path="bias.html"><a href="bias.html#measurementInvariancePathDiagram"><i class="fa fa-check"></i><b>16.10.4.9</b> Path Diagram</a></li>
</ul></li>
<li class="chapter" data-level="16.10.5" data-path="bias.html"><a href="bias.html#measurementInvarianceMetricModel"><i class="fa fa-check"></i><b>16.10.5</b> Metric (“Weak Factorial”) Invariance Model</a>
<ul>
<li class="chapter" data-level="16.10.5.1" data-path="bias.html"><a href="bias.html#measurementInvarianceMetricModelSyntax"><i class="fa fa-check"></i><b>16.10.5.1</b> Model Syntax</a>
<ul>
<li class="chapter" data-level="16.10.5.1.1" data-path="bias.html"><a href="bias.html#measurementInvarianceMetricModelFeatures"><i class="fa fa-check"></i><b>16.10.5.1.1</b> Summary of Model Features</a></li>
</ul></li>
<li class="chapter" data-level="16.10.5.2" data-path="bias.html"><a href="bias.html#measurementInvarianceMetricModelTabular"><i class="fa fa-check"></i><b>16.10.5.2</b> Model Syntax in Table Form</a></li>
<li class="chapter" data-level="16.10.5.3" data-path="bias.html"><a href="bias.html#measurementInvarianceMetricModelFit"><i class="fa fa-check"></i><b>16.10.5.3</b> Fit the Model</a></li>
<li class="chapter" data-level="16.10.5.4" data-path="bias.html"><a href="bias.html#measurementInvarianceMetricModelOutput"><i class="fa fa-check"></i><b>16.10.5.4</b> Model Summary</a></li>
<li class="chapter" data-level="16.10.5.5" data-path="bias.html"><a href="bias.html#measurementInvarianceMetricModelFitCriteria"><i class="fa fa-check"></i><b>16.10.5.5</b> Model Fit</a></li>
<li class="chapter" data-level="16.10.5.6" data-path="bias.html"><a href="bias.html#measurementInvarianceMetricModelCompare"><i class="fa fa-check"></i><b>16.10.5.6</b> Compare Model Fit</a>
<ul>
<li class="chapter" data-level="16.10.5.6.1" data-path="bias.html"><a href="bias.html#measurementInvarianceMetricModelNestedModelComparison"><i class="fa fa-check"></i><b>16.10.5.6.1</b> Nested Model (<span class="math inline">\(\chi^2\)</span>) Difference Test</a></li>
<li class="chapter" data-level="16.10.5.6.2" data-path="bias.html"><a href="bias.html#measurementInvarianceMetricModelCompareFitCriteria"><i class="fa fa-check"></i><b>16.10.5.6.2</b> Compare Other Fit Criteria</a></li>
<li class="chapter" data-level="16.10.5.6.3" data-path="bias.html"><a href="bias.html#measurementInvarianceMetricModelScoreBasedTest"><i class="fa fa-check"></i><b>16.10.5.6.3</b> Score-Based Test</a></li>
<li class="chapter" data-level="16.10.5.6.4" data-path="bias.html"><a href="bias.html#measurementInvarianceMetricModelEquivalenceTest"><i class="fa fa-check"></i><b>16.10.5.6.4</b> Equivalence Test</a></li>
<li class="chapter" data-level="16.10.5.6.5" data-path="bias.html"><a href="bias.html#measurementInvarianceMetricModelPermutationTest"><i class="fa fa-check"></i><b>16.10.5.6.5</b> Permutation Test</a></li>
</ul></li>
<li class="chapter" data-level="16.10.5.7" data-path="bias.html"><a href="bias.html#measurementInvarianceMetricModelReliability"><i class="fa fa-check"></i><b>16.10.5.7</b> Internal Consistency Reliability</a></li>
<li class="chapter" data-level="16.10.5.8" data-path="bias.html"><a href="bias.html#measurementInvarianceMetricModelPathDiagram"><i class="fa fa-check"></i><b>16.10.5.8</b> Path Diagram</a></li>
</ul></li>
<li class="chapter" data-level="16.10.6" data-path="bias.html"><a href="bias.html#measurementInvarianceScalarModel"><i class="fa fa-check"></i><b>16.10.6</b> Scalar (“Strong Factorial”) Invariance Model</a>
<ul>
<li class="chapter" data-level="16.10.6.1" data-path="bias.html"><a href="bias.html#measurementInvarianceScalarModelSyntax"><i class="fa fa-check"></i><b>16.10.6.1</b> Model Syntax</a>
<ul>
<li class="chapter" data-level="16.10.6.1.1" data-path="bias.html"><a href="bias.html#measurementInvarianceScalarModelFeatures"><i class="fa fa-check"></i><b>16.10.6.1.1</b> Summary of Model Features</a></li>
</ul></li>
<li class="chapter" data-level="16.10.6.2" data-path="bias.html"><a href="bias.html#measurementInvarianceScalarModelTabular"><i class="fa fa-check"></i><b>16.10.6.2</b> Model Syntax in Table Form</a></li>
<li class="chapter" data-level="16.10.6.3" data-path="bias.html"><a href="bias.html#measurementInvarianceScalarModelFit"><i class="fa fa-check"></i><b>16.10.6.3</b> Fit the Model</a></li>
<li class="chapter" data-level="16.10.6.4" data-path="bias.html"><a href="bias.html#measurementInvarianceScalarModelOutput"><i class="fa fa-check"></i><b>16.10.6.4</b> Model Summary</a></li>
<li class="chapter" data-level="16.10.6.5" data-path="bias.html"><a href="bias.html#measurementInvarianceScalarModelFitCriteria"><i class="fa fa-check"></i><b>16.10.6.5</b> Model Fit</a></li>
<li class="chapter" data-level="16.10.6.6" data-path="bias.html"><a href="bias.html#measurementInvarianceScalarModelCompare"><i class="fa fa-check"></i><b>16.10.6.6</b> Compare Model Fit</a>
<ul>
<li class="chapter" data-level="16.10.6.6.1" data-path="bias.html"><a href="bias.html#measurementInvarianceScalarModelNestedModelComparison"><i class="fa fa-check"></i><b>16.10.6.6.1</b> Nested Model (<span class="math inline">\(\chi^2\)</span>) Difference Test</a></li>
<li class="chapter" data-level="16.10.6.6.2" data-path="bias.html"><a href="bias.html#measurementInvarianceScalarModelCompareFitCriteria"><i class="fa fa-check"></i><b>16.10.6.6.2</b> Compare Other Fit Criteria</a></li>
<li class="chapter" data-level="16.10.6.6.3" data-path="bias.html"><a href="bias.html#measurementInvarianceScalarScoreBasedTest"><i class="fa fa-check"></i><b>16.10.6.6.3</b> Score-Based Test</a></li>
<li class="chapter" data-level="16.10.6.6.4" data-path="bias.html"><a href="bias.html#measurementInvarianceScalarModelEquivalenceTest"><i class="fa fa-check"></i><b>16.10.6.6.4</b> Equivalence Test</a></li>
<li class="chapter" data-level="16.10.6.6.5" data-path="bias.html"><a href="bias.html#measurementInvarianceScalarModelPermutationTest"><i class="fa fa-check"></i><b>16.10.6.6.5</b> Permutation Test</a></li>
</ul></li>
<li class="chapter" data-level="16.10.6.7" data-path="bias.html"><a href="bias.html#measurementInvarianceScalarModelReliability"><i class="fa fa-check"></i><b>16.10.6.7</b> Internal Consistency Reliability</a></li>
<li class="chapter" data-level="16.10.6.8" data-path="bias.html"><a href="bias.html#measurementInvarianceScalarModelPathDiagram"><i class="fa fa-check"></i><b>16.10.6.8</b> Path Diagram</a></li>
</ul></li>
<li class="chapter" data-level="16.10.7" data-path="bias.html"><a href="bias.html#measurementInvarianceResidualModel"><i class="fa fa-check"></i><b>16.10.7</b> Residual (“Strict Factorial”) Invariance Model</a>
<ul>
<li class="chapter" data-level="16.10.7.1" data-path="bias.html"><a href="bias.html#measurementInvarianceResidualModelSyntax"><i class="fa fa-check"></i><b>16.10.7.1</b> Model Syntax</a>
<ul>
<li class="chapter" data-level="16.10.7.1.1" data-path="bias.html"><a href="bias.html#measurementInvarianceResidualModelFeatures"><i class="fa fa-check"></i><b>16.10.7.1.1</b> Summary of Model Features</a></li>
</ul></li>
<li class="chapter" data-level="16.10.7.2" data-path="bias.html"><a href="bias.html#measurementInvarianceResidualModelTabular"><i class="fa fa-check"></i><b>16.10.7.2</b> Model Syntax in Table Form</a></li>
<li class="chapter" data-level="16.10.7.3" data-path="bias.html"><a href="bias.html#measurementInvarianceResidualModelFit"><i class="fa fa-check"></i><b>16.10.7.3</b> Fit the Model</a></li>
<li class="chapter" data-level="16.10.7.4" data-path="bias.html"><a href="bias.html#measurementInvarianceResidualModelOutput"><i class="fa fa-check"></i><b>16.10.7.4</b> Model Summary</a></li>
<li class="chapter" data-level="16.10.7.5" data-path="bias.html"><a href="bias.html#measurementInvarianceResidualModelFitCriteria"><i class="fa fa-check"></i><b>16.10.7.5</b> Model Fit</a></li>
<li class="chapter" data-level="16.10.7.6" data-path="bias.html"><a href="bias.html#measurementInvarianceResidualModelCompare"><i class="fa fa-check"></i><b>16.10.7.6</b> Compare Model Fit</a>
<ul>
<li class="chapter" data-level="16.10.7.6.1" data-path="bias.html"><a href="bias.html#measurementInvarianceResidualModelNestedModelComparison"><i class="fa fa-check"></i><b>16.10.7.6.1</b> Nested Model (<span class="math inline">\(\chi^2\)</span>) Difference Test</a></li>
<li class="chapter" data-level="16.10.7.6.2" data-path="bias.html"><a href="bias.html#measurementInvarianceResidualModelCompareFitCriteria"><i class="fa fa-check"></i><b>16.10.7.6.2</b> Compare Other Fit Criteria</a></li>
<li class="chapter" data-level="16.10.7.6.3" data-path="bias.html"><a href="bias.html#measurementInvarianceResidualModelScoreBasedTest"><i class="fa fa-check"></i><b>16.10.7.6.3</b> Score-Based Test</a></li>
<li class="chapter" data-level="16.10.7.6.4" data-path="bias.html"><a href="bias.html#measurementInvarianceResidualModelEquivalenceTest"><i class="fa fa-check"></i><b>16.10.7.6.4</b> Equivalence Test</a></li>
<li class="chapter" data-level="16.10.7.6.5" data-path="bias.html"><a href="bias.html#measurementInvarianceResidualModelPermutationTest"><i class="fa fa-check"></i><b>16.10.7.6.5</b> Permutation Test</a></li>
</ul></li>
<li class="chapter" data-level="16.10.7.7" data-path="bias.html"><a href="bias.html#measurementInvarianceResidualModelReliability"><i class="fa fa-check"></i><b>16.10.7.7</b> Internal Consistency Reliability</a></li>
<li class="chapter" data-level="16.10.7.8" data-path="bias.html"><a href="bias.html#measurementInvarianceResidualModelPathDiagram"><i class="fa fa-check"></i><b>16.10.7.8</b> Path Diagram</a></li>
</ul></li>
<li class="chapter" data-level="16.10.8" data-path="bias.html"><a href="bias.html#addressingMeasurementNonInvariance"><i class="fa fa-check"></i><b>16.10.8</b> Addressing Measurement Non-Invariance</a>
<ul>
<li class="chapter" data-level="16.10.8.1" data-path="bias.html"><a href="bias.html#measurementInvarianceAddConstraints"><i class="fa fa-check"></i><b>16.10.8.1</b> Iteratively add constraints to identify measurement non-invariance</a>
<ul>
<li class="chapter" data-level="16.10.8.1.1" data-path="bias.html"><a href="bias.html#measurementInvarianceAddConstraintsVariable1"><i class="fa fa-check"></i><b>16.10.8.1.1</b> Variable 1</a></li>
<li class="chapter" data-level="16.10.8.1.2" data-path="bias.html"><a href="bias.html#measurementInvarianceAddConstraintsVariable2"><i class="fa fa-check"></i><b>16.10.8.1.2</b> Variable 2</a></li>
<li class="chapter" data-level="16.10.8.1.3" data-path="bias.html"><a href="bias.html#measurementInvarianceAddConstraintsVariable3"><i class="fa fa-check"></i><b>16.10.8.1.3</b> Variable 3</a></li>
<li class="chapter" data-level="16.10.8.1.4" data-path="bias.html"><a href="bias.html#measurementInvarianceAddConstraintsVariable4"><i class="fa fa-check"></i><b>16.10.8.1.4</b> Variable 4</a></li>
<li class="chapter" data-level="16.10.8.1.5" data-path="bias.html"><a href="bias.html#measurementInvarianceAddConstraintsVariable5"><i class="fa fa-check"></i><b>16.10.8.1.5</b> Variable 5</a></li>
<li class="chapter" data-level="16.10.8.1.6" data-path="bias.html"><a href="bias.html#measurementInvarianceAddConstraintsVariable6"><i class="fa fa-check"></i><b>16.10.8.1.6</b> Variable 6</a></li>
<li class="chapter" data-level="16.10.8.1.7" data-path="bias.html"><a href="bias.html#measurementInvarianceAddConstraintsVariable7"><i class="fa fa-check"></i><b>16.10.8.1.7</b> Variable 7</a></li>
<li class="chapter" data-level="16.10.8.1.8" data-path="bias.html"><a href="bias.html#measurementInvarianceAddConstraintsVariable8"><i class="fa fa-check"></i><b>16.10.8.1.8</b> Variable 8</a></li>
<li class="chapter" data-level="16.10.8.1.9" data-path="bias.html"><a href="bias.html#measurementInvarianceAddConstraintsVariable9"><i class="fa fa-check"></i><b>16.10.8.1.9</b> Variable 9</a></li>
<li class="chapter" data-level="16.10.8.1.10" data-path="bias.html"><a href="bias.html#measurementInvarianceAddConstraintsSummary"><i class="fa fa-check"></i><b>16.10.8.1.10</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="16.10.8.2" data-path="bias.html"><a href="bias.html#measurementInvarianceDropConstraints"><i class="fa fa-check"></i><b>16.10.8.2</b> Iteratively drop constraints to identify measurement non-invariance</a>
<ul>
<li class="chapter" data-level="16.10.8.2.1" data-path="bias.html"><a href="bias.html#measurementInvarianceDropConstraintsVariable1"><i class="fa fa-check"></i><b>16.10.8.2.1</b> Variable 1</a></li>
<li class="chapter" data-level="16.10.8.2.2" data-path="bias.html"><a href="bias.html#measurementInvarianceDropConstraintsVariable2"><i class="fa fa-check"></i><b>16.10.8.2.2</b> Variable 2</a></li>
<li class="chapter" data-level="16.10.8.2.3" data-path="bias.html"><a href="bias.html#measurementInvarianceDropConstraintsVariable3"><i class="fa fa-check"></i><b>16.10.8.2.3</b> Variable 3</a></li>
<li class="chapter" data-level="16.10.8.2.4" data-path="bias.html"><a href="bias.html#measurementInvarianceDropConstraintsVariable4"><i class="fa fa-check"></i><b>16.10.8.2.4</b> Variable 4</a></li>
<li class="chapter" data-level="16.10.8.2.5" data-path="bias.html"><a href="bias.html#measurementInvarianceDropConstraintsVariable5"><i class="fa fa-check"></i><b>16.10.8.2.5</b> Variable 5</a></li>
<li class="chapter" data-level="16.10.8.2.6" data-path="bias.html"><a href="bias.html#measurementInvarianceDropConstraintsVariable6"><i class="fa fa-check"></i><b>16.10.8.2.6</b> Variable 6</a></li>
<li class="chapter" data-level="16.10.8.2.7" data-path="bias.html"><a href="bias.html#measurementInvarianceDropConstraintsVariable7"><i class="fa fa-check"></i><b>16.10.8.2.7</b> Variable 7</a></li>
<li class="chapter" data-level="16.10.8.2.8" data-path="bias.html"><a href="bias.html#measurementInvarianceDropConstraintsVariable8"><i class="fa fa-check"></i><b>16.10.8.2.8</b> Variable 8</a></li>
<li class="chapter" data-level="16.10.8.2.9" data-path="bias.html"><a href="bias.html#measurementInvarianceDropConstraintsVariable9"><i class="fa fa-check"></i><b>16.10.8.2.9</b> Variable 9</a></li>
<li class="chapter" data-level="16.10.8.2.10" data-path="bias.html"><a href="bias.html#measurementInvarianceDropConstraintsSummary"><i class="fa fa-check"></i><b>16.10.8.2.10</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="16.10.8.3" data-path="bias.html"><a href="bias.html#measurementInvarianceNextSteps"><i class="fa fa-check"></i><b>16.10.8.3</b> Next Steps</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="16.11" data-path="bias.html"><a href="bias.html#conclusion-bias"><i class="fa fa-check"></i><b>16.11</b> Conclusion</a></li>
<li class="chapter" data-level="16.12" data-path="bias.html"><a href="bias.html#readings-bias"><i class="fa fa-check"></i><b>16.12</b> Suggested Readings</a></li>
<li class="chapter" data-level="16.13" data-path="bias.html"><a href="bias.html#exercises-bias"><i class="fa fa-check"></i><b>16.13</b> Exercises</a>
<ul>
<li class="chapter" data-level="16.13.1" data-path="bias.html"><a href="bias.html#exercisesQuestions-bias"><i class="fa fa-check"></i><b>16.13.1</b> Questions</a></li>
<li class="chapter" data-level="16.13.2" data-path="bias.html"><a href="bias.html#exercisesAnswers-bias"><i class="fa fa-check"></i><b>16.13.2</b> Answers</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="17" data-path="interview.html"><a href="interview.html"><i class="fa fa-check"></i><b>17</b> The Interview and the DSM</a>
<ul>
<li class="chapter" data-level="17.1" data-path="interview.html"><a href="interview.html#overview-interview"><i class="fa fa-check"></i><b>17.1</b> Overview of Clinical Interviews</a></li>
<li class="chapter" data-level="17.2" data-path="interview.html"><a href="interview.html#interviewTraditions"><i class="fa fa-check"></i><b>17.2</b> Two Traditions: Unstructured and Structured Interviews</a>
<ul>
<li class="chapter" data-level="17.2.1" data-path="interview.html"><a href="interview.html#unstructuredInterview"><i class="fa fa-check"></i><b>17.2.1</b> Unstructured Interview</a></li>
<li class="chapter" data-level="17.2.2" data-path="interview.html"><a href="interview.html#structuredInterview"><i class="fa fa-check"></i><b>17.2.2</b> Structured Interview</a>
<ul>
<li class="chapter" data-level="17.2.2.1" data-path="interview.html"><a href="interview.html#semiStructuredInterview"><i class="fa fa-check"></i><b>17.2.2.1</b> Semi-Structured Interview</a></li>
</ul></li>
<li class="chapter" data-level="17.2.3" data-path="interview.html"><a href="interview.html#structuredVsUnstructuredInterview"><i class="fa fa-check"></i><b>17.2.3</b> Structured Versus Unstructured Interviews</a></li>
</ul></li>
<li class="chapter" data-level="17.3" data-path="interview.html"><a href="interview.html#otherFindingsInterviews"><i class="fa fa-check"></i><b>17.3</b> Other Findings Regarding Interviews</a></li>
<li class="chapter" data-level="17.4" data-path="interview.html"><a href="interview.html#bestPracticeDiagnosis"><i class="fa fa-check"></i><b>17.4</b> Best Practice for Diagnostic Assessment</a></li>
<li class="chapter" data-level="17.5" data-path="interview.html"><a href="interview.html#dsm"><i class="fa fa-check"></i><b>17.5</b> The DSM and ICD</a>
<ul>
<li class="chapter" data-level="17.5.1" data-path="interview.html"><a href="interview.html#dsmStrengths"><i class="fa fa-check"></i><b>17.5.1</b> Strengths</a></li>
<li class="chapter" data-level="17.5.2" data-path="interview.html"><a href="interview.html#dsmConcerns"><i class="fa fa-check"></i><b>17.5.2</b> Concerns</a></li>
<li class="chapter" data-level="17.5.3" data-path="interview.html"><a href="interview.html#alternativeStructuresPsychopathology"><i class="fa fa-check"></i><b>17.5.3</b> Alternative Structures of Psychopathology</a></li>
</ul></li>
<li class="chapter" data-level="17.6" data-path="interview.html"><a href="interview.html#conclusion-interview"><i class="fa fa-check"></i><b>17.6</b> Conclusion</a></li>
<li class="chapter" data-level="17.7" data-path="interview.html"><a href="interview.html#readings-interview"><i class="fa fa-check"></i><b>17.7</b> Suggested Readings</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="objective-personality.html"><a href="objective-personality.html"><i class="fa fa-check"></i><b>18</b> Objective Personality Testing</a>
<ul>
<li class="chapter" data-level="18.1" data-path="objective-personality.html"><a href="objective-personality.html#overview-objective-personality"><i class="fa fa-check"></i><b>18.1</b> Overview of Personality Tests</a>
<ul>
<li class="chapter" data-level="18.1.1" data-path="objective-personality.html"><a href="objective-personality.html#projectivePersonalityTests"><i class="fa fa-check"></i><b>18.1.1</b> Projective Personality Tests</a></li>
<li class="chapter" data-level="18.1.2" data-path="objective-personality.html"><a href="objective-personality.html#objectivePersonalityTests"><i class="fa fa-check"></i><b>18.1.2</b> Objective Personality Tests</a></li>
</ul></li>
<li class="chapter" data-level="18.2" data-path="objective-personality.html"><a href="objective-personality.html#mmpi"><i class="fa fa-check"></i><b>18.2</b> Example of an Objective Personality Test: MMPI</a></li>
<li class="chapter" data-level="18.3" data-path="objective-personality.html"><a href="objective-personality.html#problemsTrueFalseMeasures"><i class="fa fa-check"></i><b>18.3</b> Problems with Objective True/False Measures</a></li>
<li class="chapter" data-level="18.4" data-path="objective-personality.html"><a href="objective-personality.html#developingPersonalityMeasures"><i class="fa fa-check"></i><b>18.4</b> Approaches to Developing Personality Measures</a>
<ul>
<li class="chapter" data-level="18.4.1" data-path="objective-personality.html"><a href="objective-personality.html#externalApproach"><i class="fa fa-check"></i><b>18.4.1</b> External Approach to Scale Contruction</a>
<ul>
<li class="chapter" data-level="18.4.1.1" data-path="objective-personality.html"><a href="objective-personality.html#externalApproachPros"><i class="fa fa-check"></i><b>18.4.1.1</b> Pros</a></li>
<li class="chapter" data-level="18.4.1.2" data-path="objective-personality.html"><a href="objective-personality.html#externalApproachCons"><i class="fa fa-check"></i><b>18.4.1.2</b> Cons</a></li>
<li class="chapter" data-level="18.4.1.3" data-path="objective-personality.html"><a href="objective-personality.html#mmpiExamples"><i class="fa fa-check"></i><b>18.4.1.3</b> MMPI Examples</a></li>
</ul></li>
<li class="chapter" data-level="18.4.2" data-path="objective-personality.html"><a href="objective-personality.html#deductiveApproach"><i class="fa fa-check"></i><b>18.4.2</b> Deductive Approach to Scale Construction</a>
<ul>
<li class="chapter" data-level="18.4.2.1" data-path="objective-personality.html"><a href="objective-personality.html#deductiveApproachPros"><i class="fa fa-check"></i><b>18.4.2.1</b> Pros</a></li>
<li class="chapter" data-level="18.4.2.2" data-path="objective-personality.html"><a href="objective-personality.html#deductiveApproachCons"><i class="fa fa-check"></i><b>18.4.2.2</b> Cons</a></li>
</ul></li>
<li class="chapter" data-level="18.4.3" data-path="objective-personality.html"><a href="objective-personality.html#inductiveApproach"><i class="fa fa-check"></i><b>18.4.3</b> Inductive Approach to Scale Construct</a>
<ul>
<li class="chapter" data-level="18.4.3.1" data-path="objective-personality.html"><a href="objective-personality.html#inductiveApproachPros"><i class="fa fa-check"></i><b>18.4.3.1</b> Pros</a></li>
<li class="chapter" data-level="18.4.3.2" data-path="objective-personality.html"><a href="objective-personality.html#inductiveApproachCons"><i class="fa fa-check"></i><b>18.4.3.2</b> Cons</a></li>
</ul></li>
<li class="chapter" data-level="18.4.4" data-path="objective-personality.html"><a href="objective-personality.html#hybridApproach"><i class="fa fa-check"></i><b>18.4.4</b> Hybrid Approach</a></li>
</ul></li>
<li class="chapter" data-level="18.5" data-path="objective-personality.html"><a href="objective-personality.html#measureDevelopment"><i class="fa fa-check"></i><b>18.5</b> Measure Development and Item Selection</a>
<ul>
<li class="chapter" data-level="18.5.1" data-path="objective-personality.html"><a href="objective-personality.html#responseScale"><i class="fa fa-check"></i><b>18.5.1</b> The Response Scale</a></li>
</ul></li>
<li class="chapter" data-level="18.6" data-path="objective-personality.html"><a href="objective-personality.html#emergingTechniques-personality"><i class="fa fa-check"></i><b>18.6</b> Emerging Techniques</a></li>
<li class="chapter" data-level="18.7" data-path="objective-personality.html"><a href="objective-personality.html#flawedSelfAssessment"><i class="fa fa-check"></i><b>18.7</b> The Flawed Nature of Self-Assessments</a>
<ul>
<li class="chapter" data-level="18.7.1" data-path="objective-personality.html"><a href="objective-personality.html#bias-personality"><i class="fa fa-check"></i><b>18.7.1</b> Response Bias</a></li>
<li class="chapter" data-level="18.7.2" data-path="objective-personality.html"><a href="objective-personality.html#itemAmbiguity"><i class="fa fa-check"></i><b>18.7.2</b> Ambiguity of Items</a></li>
<li class="chapter" data-level="18.7.3" data-path="objective-personality.html"><a href="objective-personality.html#lackOfInsight"><i class="fa fa-check"></i><b>18.7.3</b> Lack of Insight by People to Rate Themselves</a></li>
<li class="chapter" data-level="18.7.4" data-path="objective-personality.html"><a href="objective-personality.html#improveSelfAssessment"><i class="fa fa-check"></i><b>18.7.4</b> Ways to Improve Self-Assessment</a></li>
<li class="chapter" data-level="18.7.5" data-path="objective-personality.html"><a href="objective-personality.html#satisficing"><i class="fa fa-check"></i><b>18.7.5</b> Satisficing (Versus Optimizing)</a>
<ul>
<li class="chapter" data-level="18.7.5.1" data-path="objective-personality.html"><a href="objective-personality.html#optimizing"><i class="fa fa-check"></i><b>18.7.5.1</b> Optimizing</a></li>
<li class="chapter" data-level="18.7.5.2" data-path="objective-personality.html"><a href="objective-personality.html#satisficingVsOptimizing"><i class="fa fa-check"></i><b>18.7.5.2</b> Satisficing</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="18.8" data-path="objective-personality.html"><a href="objective-personality.html#observation-personality"><i class="fa fa-check"></i><b>18.8</b> Observational Assessments</a></li>
<li class="chapter" data-level="18.9" data-path="objective-personality.html"><a href="objective-personality.html#personalityStructure"><i class="fa fa-check"></i><b>18.9</b> Structure of Personality</a></li>
<li class="chapter" data-level="18.10" data-path="objective-personality.html"><a href="objective-personality.html#personalityAcrossTheLifespan"><i class="fa fa-check"></i><b>18.10</b> Personality Across the Lifespan</a></li>
<li class="chapter" data-level="18.11" data-path="objective-personality.html"><a href="objective-personality.html#conclusion-objective-personality"><i class="fa fa-check"></i><b>18.11</b> Conclusion</a></li>
<li class="chapter" data-level="18.12" data-path="objective-personality.html"><a href="objective-personality.html#readings-objective-personality"><i class="fa fa-check"></i><b>18.12</b> Suggested Readings</a></li>
</ul></li>
<li class="chapter" data-level="19" data-path="projective.html"><a href="projective.html"><i class="fa fa-check"></i><b>19</b> Projective Personality Testing</a>
<ul>
<li class="chapter" data-level="19.1" data-path="projective.html"><a href="projective.html#overview-projective-personality"><i class="fa fa-check"></i><b>19.1</b> Overview of Projective Personality Testing</a>
<ul>
<li class="chapter" data-level="19.1.1" data-path="projective.html"><a href="projective.html#projectiveCategorization"><i class="fa fa-check"></i><b>19.1.1</b> Categorization of Responses</a></li>
<li class="chapter" data-level="19.1.2" data-path="projective.html"><a href="projective.html#projectiveHypothesis"><i class="fa fa-check"></i><b>19.1.2</b> Projective Hypothesis</a></li>
<li class="chapter" data-level="19.1.3" data-path="projective.html"><a href="projective.html#projectiveMeasures"><i class="fa fa-check"></i><b>19.1.3</b> Projective Measures</a></li>
<li class="chapter" data-level="19.1.4" data-path="projective.html"><a href="projective.html#whyNotUseProjectiveMeasures"><i class="fa fa-check"></i><b>19.1.4</b> Why Not Use Projective Measures?</a></li>
</ul></li>
<li class="chapter" data-level="19.2" data-path="projective.html"><a href="projective.html#projectiveMeasureExamples"><i class="fa fa-check"></i><b>19.2</b> Examples of Projective Measures</a>
<ul>
<li class="chapter" data-level="19.2.1" data-path="projective.html"><a href="projective.html#rorschach"><i class="fa fa-check"></i><b>19.2.1</b> Rorschach Inkblot Test</a>
<ul>
<li class="chapter" data-level="19.2.1.1" data-path="projective.html"><a href="projective.html#rorschachVariables"><i class="fa fa-check"></i><b>19.2.1.1</b> Variables</a></li>
<li class="chapter" data-level="19.2.1.2" data-path="projective.html"><a href="projective.html#rorschachScoring"><i class="fa fa-check"></i><b>19.2.1.2</b> Scoring</a></li>
</ul></li>
<li class="chapter" data-level="19.2.2" data-path="projective.html"><a href="projective.html#tat"><i class="fa fa-check"></i><b>19.2.2</b> Thematic Apperception Test (TAT)</a></li>
</ul></li>
<li class="chapter" data-level="19.3" data-path="projective.html"><a href="projective.html#mostWidelyUsedAssessmentsChildren"><i class="fa fa-check"></i><b>19.3</b> Most Widely Used Assessments for Children</a></li>
<li class="chapter" data-level="19.4" data-path="projective.html"><a href="projective.html#projectiveScientificStatus"><i class="fa fa-check"></i><b>19.4</b> Evaluating the Scientific Status of Projective Measures</a>
<ul>
<li class="chapter" data-level="19.4.1" data-path="projective.html"><a href="projective.html#projectiveNorms"><i class="fa fa-check"></i><b>19.4.1</b> Utility of Norms</a></li>
<li class="chapter" data-level="19.4.2" data-path="projective.html"><a href="projective.html#projectiveReliability"><i class="fa fa-check"></i><b>19.4.2</b> Reliability of Scores</a></li>
<li class="chapter" data-level="19.4.3" data-path="projective.html"><a href="projective.html#projectiveMeasurementError"><i class="fa fa-check"></i><b>19.4.3</b> Influence of Measurement Error on Scores</a></li>
<li class="chapter" data-level="19.4.4" data-path="projective.html"><a href="projective.html#projectiveValidity"><i class="fa fa-check"></i><b>19.4.4</b> Validity</a>
<ul>
<li class="chapter" data-level="19.4.4.1" data-path="projective.html"><a href="projective.html#criterionValidity-projective"><i class="fa fa-check"></i><b>19.4.4.1</b> Criterion-Related Validity</a></li>
<li class="chapter" data-level="19.4.4.2" data-path="projective.html"><a href="projective.html#convergentValidity-projective"><i class="fa fa-check"></i><b>19.4.4.2</b> Convergent Validity</a></li>
<li class="chapter" data-level="19.4.4.3" data-path="projective.html"><a href="projective.html#incrementalValidity-projective"><i class="fa fa-check"></i><b>19.4.4.3</b> Incremental Validity</a></li>
<li class="chapter" data-level="19.4.4.4" data-path="projective.html"><a href="projective.html#treatmentUtility-projective"><i class="fa fa-check"></i><b>19.4.4.4</b> Treatment Utility</a></li>
</ul></li>
<li class="chapter" data-level="19.4.5" data-path="projective.html"><a href="projective.html#scientificStatusSummary-projective"><i class="fa fa-check"></i><b>19.4.5</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="19.5" data-path="projective.html"><a href="projective.html#conclusion-projective-personality"><i class="fa fa-check"></i><b>19.5</b> Conclusion</a></li>
<li class="chapter" data-level="19.6" data-path="projective.html"><a href="projective.html#readings-projective-personality"><i class="fa fa-check"></i><b>19.6</b> Suggested Readings</a></li>
</ul></li>
<li class="chapter" data-level="20" data-path="psychophysiological.html"><a href="psychophysiological.html"><i class="fa fa-check"></i><b>20</b> Psychophysiological and Ambulatory Assessment</a>
<ul>
<li class="chapter" data-level="20.1" data-path="psychophysiological.html"><a href="psychophysiological.html#rdoc"><i class="fa fa-check"></i><b>20.1</b> NIMH Research Domain Criteria (RDoC)</a>
<ul>
<li class="chapter" data-level="20.1.1" data-path="psychophysiological.html"><a href="psychophysiological.html#rdocMatrix"><i class="fa fa-check"></i><b>20.1.1</b> RDoC Matrix</a></li>
</ul></li>
<li class="chapter" data-level="20.2" data-path="psychophysiological.html"><a href="psychophysiological.html#psychophysiologicalMeasures"><i class="fa fa-check"></i><b>20.2</b> Psychophysiological Measures</a>
<ul>
<li class="chapter" data-level="20.2.1" data-path="psychophysiological.html"><a href="psychophysiological.html#reliability-psychophysiology"><i class="fa fa-check"></i><b>20.2.1</b> Reliability</a></li>
<li class="chapter" data-level="20.2.2" data-path="psychophysiological.html"><a href="psychophysiological.html#psychophysiologicalLotsOfData"><i class="fa fa-check"></i><b>20.2.2</b> Lots of Data</a></li>
<li class="chapter" data-level="20.2.3" data-path="psychophysiological.html"><a href="psychophysiological.html#validity-psychophysiology"><i class="fa fa-check"></i><b>20.2.3</b> Validity</a></li>
<li class="chapter" data-level="20.2.4" data-path="psychophysiological.html"><a href="psychophysiological.html#examplesOfPsychophysiologicalAssessment"><i class="fa fa-check"></i><b>20.2.4</b> Examples</a>
<ul>
<li class="chapter" data-level="20.2.4.1" data-path="psychophysiological.html"><a href="psychophysiological.html#ambulatory"><i class="fa fa-check"></i><b>20.2.4.1</b> Ambulatory Assessment</a>
<ul>
<li class="chapter" data-level="20.2.4.1.1" data-path="psychophysiological.html"><a href="psychophysiological.html#ambulatoryPros"><i class="fa fa-check"></i><b>20.2.4.1.1</b> Pros</a></li>
<li class="chapter" data-level="20.2.4.1.2" data-path="psychophysiological.html"><a href="psychophysiological.html#ambulatoryCons"><i class="fa fa-check"></i><b>20.2.4.1.2</b> Cons</a></li>
</ul></li>
<li class="chapter" data-level="20.2.4.2" data-path="psychophysiological.html"><a href="psychophysiological.html#otherPsyphysiologicalExamples"><i class="fa fa-check"></i><b>20.2.4.2</b> Other Examples</a></li>
</ul></li>
<li class="chapter" data-level="20.2.5" data-path="psychophysiological.html"><a href="psychophysiological.html#virtualReality"><i class="fa fa-check"></i><b>20.2.5</b> Virtual Reality</a></li>
<li class="chapter" data-level="20.2.6" data-path="psychophysiological.html"><a href="psychophysiological.html#usingPsychophysiologicalMeasures"><i class="fa fa-check"></i><b>20.2.6</b> Using Psychophysiological Measures</a></li>
</ul></li>
<li class="chapter" data-level="20.3" data-path="psychophysiological.html"><a href="psychophysiological.html#conclusion-psychophysiological"><i class="fa fa-check"></i><b>20.3</b> Conclusion</a></li>
<li class="chapter" data-level="20.4" data-path="psychophysiological.html"><a href="psychophysiological.html#readings-psychophysiological"><i class="fa fa-check"></i><b>20.4</b> Suggested Readings</a></li>
</ul></li>
<li class="chapter" data-level="21" data-path="cat.html"><a href="cat.html"><i class="fa fa-check"></i><b>21</b> Computers and Adaptive Testing</a>
<ul>
<li class="chapter" data-level="21.1" data-path="cat.html"><a href="cat.html#computerAssessment"><i class="fa fa-check"></i><b>21.1</b> Computer-Administered/Online Assessment</a>
<ul>
<li class="chapter" data-level="21.1.1" data-path="cat.html"><a href="cat.html#computerAdvantages"><i class="fa fa-check"></i><b>21.1.1</b> Advantages</a></li>
<li class="chapter" data-level="21.1.2" data-path="cat.html"><a href="cat.html#computerValidity"><i class="fa fa-check"></i><b>21.1.2</b> Validity Challenges</a></li>
<li class="chapter" data-level="21.1.3" data-path="cat.html"><a href="cat.html#computerEthics"><i class="fa fa-check"></i><b>21.1.3</b> Ethical Challenges</a></li>
<li class="chapter" data-level="21.1.4" data-path="cat.html"><a href="cat.html#computerBestPractices"><i class="fa fa-check"></i><b>21.1.4</b> Best Practices</a></li>
</ul></li>
<li class="chapter" data-level="21.2" data-path="cat.html"><a href="cat.html#adaptiveTesting"><i class="fa fa-check"></i><b>21.2</b> Adaptive Testing</a>
<ul>
<li class="chapter" data-level="21.2.1" data-path="cat.html"><a href="cat.html#adaptiveManual"><i class="fa fa-check"></i><b>21.2.1</b> Manual Administration of Adaptive Testing (Adaptive Testing without IRT)</a>
<ul>
<li class="chapter" data-level="21.2.1.1" data-path="cat.html"><a href="cat.html#skipRules"><i class="fa fa-check"></i><b>21.2.1.1</b> Skip Rules</a></li>
<li class="chapter" data-level="21.2.1.2" data-path="cat.html"><a href="cat.html#basalCeiling"><i class="fa fa-check"></i><b>21.2.1.2</b> Basal and Ceiling Approach</a></li>
<li class="chapter" data-level="21.2.1.3" data-path="cat.html"><a href="cat.html#adaptiveCountdown"><i class="fa fa-check"></i><b>21.2.1.3</b> Countdown Approach</a>
<ul>
<li class="chapter" data-level="21.2.1.3.1" data-path="cat.html"><a href="cat.html#classificationMethod"><i class="fa fa-check"></i><b>21.2.1.3.1</b> Classification Method</a></li>
<li class="chapter" data-level="21.2.1.3.2" data-path="cat.html"><a href="cat.html#fses"><i class="fa fa-check"></i><b>21.2.1.3.2</b> Full Scores on Elevated Scales Method</a></li>
</ul></li>
<li class="chapter" data-level="21.2.1.4" data-path="cat.html"><a href="cat.html#manualAdaptiveTestingSummary"><i class="fa fa-check"></i><b>21.2.1.4</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="21.2.2" data-path="cat.html"><a href="cat.html#catIRT"><i class="fa fa-check"></i><b>21.2.2</b> CAT with IRT</a></li>
</ul></li>
<li class="chapter" data-level="21.3" data-path="cat.html"><a href="cat.html#gettingStarted-cat"><i class="fa fa-check"></i><b>21.3</b> Getting Started</a>
<ul>
<li class="chapter" data-level="21.3.1" data-path="cat.html"><a href="cat.html#loadLibraries-cat"><i class="fa fa-check"></i><b>21.3.1</b> Load Libraries</a></li>
</ul></li>
<li class="chapter" data-level="21.4" data-path="cat.html"><a href="cat.html#exampleUnidimensionalCAT"><i class="fa fa-check"></i><b>21.4</b> Example of Unidimensional CAT</a>
<ul>
<li class="chapter" data-level="21.4.1" data-path="cat.html"><a href="cat.html#definePopulationIRTparameters"><i class="fa fa-check"></i><b>21.4.1</b> Define Population IRT Parameters</a></li>
<li class="chapter" data-level="21.4.2" data-path="cat.html"><a href="cat.html#fitIRTmodel-cat"><i class="fa fa-check"></i><b>21.4.2</b> Fit IRT Model</a></li>
<li class="chapter" data-level="21.4.3" data-path="cat.html"><a href="cat.html#modelOutput-cat"><i class="fa fa-check"></i><b>21.4.3</b> Model Summary</a></li>
<li class="chapter" data-level="21.4.4" data-path="cat.html"><a href="cat.html#modelPlots-cat"><i class="fa fa-check"></i><b>21.4.4</b> Model Plots</a></li>
<li class="chapter" data-level="21.4.5" data-path="cat.html"><a href="cat.html#itemPlots-cat"><i class="fa fa-check"></i><b>21.4.5</b> Item Plots</a></li>
<li class="chapter" data-level="21.4.6" data-path="cat.html"><a href="cat.html#createMathItems-cat"><i class="fa fa-check"></i><b>21.4.6</b> Create Math Items</a></li>
<li class="chapter" data-level="21.4.7" data-path="cat.html"><a href="cat.html#runCAT"><i class="fa fa-check"></i><b>21.4.7</b> Run Computerized Adaptive Test (CAT)</a></li>
<li class="chapter" data-level="21.4.8" data-path="cat.html"><a href="cat.html#catResults"><i class="fa fa-check"></i><b>21.4.8</b> CAT Results</a>
<ul>
<li class="chapter" data-level="21.4.8.1" data-path="cat.html"><a href="cat.html#CATstandardErrors"><i class="fa fa-check"></i><b>21.4.8.1</b> CAT Standard Errors</a></li>
<li class="chapter" data-level="21.4.8.2" data-path="cat.html"><a href="cat.html#cat95CI"><i class="fa fa-check"></i><b>21.4.8.2</b> CAT 95% Confidence Interval</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="21.5" data-path="cat.html"><a href="cat.html#cat-preexisting"><i class="fa fa-check"></i><b>21.5</b> Creating a Computerized Adaptive Test From an Item Response Theory Model</a>
<ul>
<li class="chapter" data-level="21.5.1" data-path="cat.html"><a href="cat.html#cat-createItems"><i class="fa fa-check"></i><b>21.5.1</b> Create Items</a></li>
<li class="chapter" data-level="21.5.2" data-path="cat.html"><a href="cat.html#cat-runModel"><i class="fa fa-check"></i><b>21.5.2</b> Run Computerized Adaptive Test</a></li>
</ul></li>
<li class="chapter" data-level="21.6" data-path="cat.html"><a href="cat.html#conclusion-cat"><i class="fa fa-check"></i><b>21.6</b> Conclusion</a></li>
<li class="chapter" data-level="21.7" data-path="cat.html"><a href="cat.html#readings-cat"><i class="fa fa-check"></i><b>21.7</b> Suggested Readings</a></li>
<li class="chapter" data-level="21.8" data-path="cat.html"><a href="cat.html#exercises-cat"><i class="fa fa-check"></i><b>21.8</b> Exercises</a>
<ul>
<li class="chapter" data-level="21.8.1" data-path="cat.html"><a href="cat.html#exercisesQuestions-cat"><i class="fa fa-check"></i><b>21.8.1</b> Questions</a></li>
<li class="chapter" data-level="21.8.2" data-path="cat.html"><a href="cat.html#exercisesAnswers-cat"><i class="fa fa-check"></i><b>21.8.2</b> Answers</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="22" data-path="behavioral.html"><a href="behavioral.html"><i class="fa fa-check"></i><b>22</b> Behavioral Assessment</a>
<ul>
<li class="chapter" data-level="22.1" data-path="behavioral.html"><a href="behavioral.html#overview-behavioral"><i class="fa fa-check"></i><b>22.1</b> Overview of Behavioral Assessment</a>
<ul>
<li class="chapter" data-level="22.1.1" data-path="behavioral.html"><a href="behavioral.html#observationApproaches"><i class="fa fa-check"></i><b>22.1.1</b> Approaches</a></li>
</ul></li>
<li class="chapter" data-level="22.2" data-path="behavioral.html"><a href="behavioral.html#observationContexts"><i class="fa fa-check"></i><b>22.2</b> Contexts for Observing</a></li>
<li class="chapter" data-level="22.3" data-path="behavioral.html"><a href="behavioral.html#observationCosts"><i class="fa fa-check"></i><b>22.3</b> Costs of Behavioral Observation</a></li>
<li class="chapter" data-level="22.4" data-path="behavioral.html"><a href="behavioral.html#observationDV"><i class="fa fa-check"></i><b>22.4</b> Dependent Variable</a></li>
<li class="chapter" data-level="22.5" data-path="behavioral.html"><a href="behavioral.html#fba"><i class="fa fa-check"></i><b>22.5</b> Functional Behavioral Assessment/Analysis</a></li>
<li class="chapter" data-level="22.6" data-path="behavioral.html"><a href="behavioral.html#mse"><i class="fa fa-check"></i><b>22.6</b> Mental Status Exam</a></li>
<li class="chapter" data-level="22.7" data-path="behavioral.html"><a href="behavioral.html#reliability-observation"><i class="fa fa-check"></i><b>22.7</b> Reliability</a></li>
<li class="chapter" data-level="22.8" data-path="behavioral.html"><a href="behavioral.html#validity-observation"><i class="fa fa-check"></i><b>22.8</b> Validity</a>
<ul>
<li class="chapter" data-level="22.8.1" data-path="behavioral.html"><a href="behavioral.html#reactivity"><i class="fa fa-check"></i><b>22.8.1</b> Reactivity</a></li>
<li class="chapter" data-level="22.8.2" data-path="behavioral.html"><a href="behavioral.html#coding"><i class="fa fa-check"></i><b>22.8.2</b> Coding</a></li>
</ul></li>
<li class="chapter" data-level="22.9" data-path="behavioral.html"><a href="behavioral.html#formsOfMeasurement-observation"><i class="fa fa-check"></i><b>22.9</b> Forms of Measurement</a></li>
<li class="chapter" data-level="22.10" data-path="behavioral.html"><a href="behavioral.html#analogueAssessment"><i class="fa fa-check"></i><b>22.10</b> Analogue (Structured) Observational Assessments</a></li>
<li class="chapter" data-level="22.11" data-path="behavioral.html"><a href="behavioral.html#selfMonitoring"><i class="fa fa-check"></i><b>22.11</b> Self-Monitoring</a></li>
<li class="chapter" data-level="22.12" data-path="behavioral.html"><a href="behavioral.html#behaviorRatingScales"><i class="fa fa-check"></i><b>22.12</b> Behavior Rating Scales</a></li>
<li class="chapter" data-level="22.13" data-path="behavioral.html"><a href="behavioral.html#assessmentTherapeuticProcess"><i class="fa fa-check"></i><b>22.13</b> Assessment of Therapeutic Process</a></li>
<li class="chapter" data-level="22.14" data-path="behavioral.html"><a href="behavioral.html#conclusion-behavioral"><i class="fa fa-check"></i><b>22.14</b> Conclusion</a></li>
<li class="chapter" data-level="22.15" data-path="behavioral.html"><a href="behavioral.html#readings-behavioral"><i class="fa fa-check"></i><b>22.15</b> Suggested Readings</a></li>
</ul></li>
<li class="chapter" data-level="23" data-path="repeated-assessments.html"><a href="repeated-assessments.html"><i class="fa fa-check"></i><b>23</b> Repeated Assessments Across Time</a>
<ul>
<li class="chapter" data-level="23.1" data-path="repeated-assessments.html"><a href="repeated-assessments.html#overview-repeated-assessments"><i class="fa fa-check"></i><b>23.1</b> Overview of Repeated Measurement</a>
<ul>
<li class="chapter" data-level="23.1.1" data-path="repeated-assessments.html"><a href="repeated-assessments.html#approaches-repeated-assessments"><i class="fa fa-check"></i><b>23.1.1</b> Comparison of Approaches</a></li>
</ul></li>
<li class="chapter" data-level="23.2" data-path="repeated-assessments.html"><a href="repeated-assessments.html#examples-repeated-assessments"><i class="fa fa-check"></i><b>23.2</b> Examples of Repeated Measurement</a>
<ul>
<li class="chapter" data-level="23.2.1" data-path="repeated-assessments.html"><a href="repeated-assessments.html#alcoholTimelineFollowback"><i class="fa fa-check"></i><b>23.2.1</b> Alcohol Timeline Followback</a></li>
<li class="chapter" data-level="23.2.2" data-path="repeated-assessments.html"><a href="repeated-assessments.html#ambulatory-repeated-assessments"><i class="fa fa-check"></i><b>23.2.2</b> Ambulatory Assessment</a></li>
</ul></li>
<li class="chapter" data-level="23.3" data-path="repeated-assessments.html"><a href="repeated-assessments.html#testRevision"><i class="fa fa-check"></i><b>23.3</b> Test Revisions</a></li>
<li class="chapter" data-level="23.4" data-path="repeated-assessments.html"><a href="repeated-assessments.html#changeAndStability"><i class="fa fa-check"></i><b>23.4</b> Change and Stability</a></li>
<li class="chapter" data-level="23.5" data-path="repeated-assessments.html"><a href="repeated-assessments.html#assessingChange"><i class="fa fa-check"></i><b>23.5</b> Assessing Change</a>
<ul>
<li class="chapter" data-level="23.5.1" data-path="repeated-assessments.html"><a href="repeated-assessments.html#inferringChange"><i class="fa fa-check"></i><b>23.5.1</b> Inferring Change</a></li>
<li class="chapter" data-level="23.5.2" data-path="repeated-assessments.html"><a href="repeated-assessments.html#potentialConfoundsOfChange"><i class="fa fa-check"></i><b>23.5.2</b> Potential Confounds of Change</a>
<ul>
<li class="chapter" data-level="23.5.2.1" data-path="repeated-assessments.html"><a href="repeated-assessments.html#practiceEffects"><i class="fa fa-check"></i><b>23.5.2.1</b> Practice Effects</a></li>
<li class="chapter" data-level="23.5.2.2" data-path="repeated-assessments.html"><a href="repeated-assessments.html#cohortEffects"><i class="fa fa-check"></i><b>23.5.2.2</b> Cohort Effects</a></li>
<li class="chapter" data-level="23.5.2.3" data-path="repeated-assessments.html"><a href="repeated-assessments.html#timeOfMeasurementEffects"><i class="fa fa-check"></i><b>23.5.2.3</b> Time-of-Measurement Effects</a></li>
</ul></li>
<li class="chapter" data-level="23.5.3" data-path="repeated-assessments.html"><a href="repeated-assessments.html#whenAndHowFrequentlyToAssess"><i class="fa fa-check"></i><b>23.5.3</b> When and How Frequently to Assess</a></li>
<li class="chapter" data-level="23.5.4" data-path="repeated-assessments.html"><a href="repeated-assessments.html#changeScores"><i class="fa fa-check"></i><b>23.5.4</b> Difference Scores</a>
<ul>
<li class="chapter" data-level="23.5.4.1" data-path="repeated-assessments.html"><a href="repeated-assessments.html#superiorApproachesToDifferenceScores"><i class="fa fa-check"></i><b>23.5.4.1</b> Superior Approaches to Difference Scores</a>
<ul>
<li class="chapter" data-level="23.5.4.1.1" data-path="repeated-assessments.html"><a href="repeated-assessments.html#clpm"><i class="fa fa-check"></i><b>23.5.4.1.1</b> Autoregressive Cross-Lagged Models</a></li>
<li class="chapter" data-level="23.5.4.1.2" data-path="repeated-assessments.html"><a href="repeated-assessments.html#lcsm"><i class="fa fa-check"></i><b>23.5.4.1.2</b> Latent Change Score Models</a></li>
<li class="chapter" data-level="23.5.4.1.3" data-path="repeated-assessments.html"><a href="repeated-assessments.html#gcm"><i class="fa fa-check"></i><b>23.5.4.1.3</b> Growth Curve Models</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="23.5.5" data-path="repeated-assessments.html"><a href="repeated-assessments.html#nomotheticIdiographicApproaches"><i class="fa fa-check"></i><b>23.5.5</b> Nomothetic Versus Idiographic</a></li>
<li class="chapter" data-level="23.5.6" data-path="repeated-assessments.html"><a href="repeated-assessments.html#sem-repeated-measures"><i class="fa fa-check"></i><b>23.5.6</b> Structural Equation Modeling</a></li>
<li class="chapter" data-level="23.5.7" data-path="repeated-assessments.html"><a href="repeated-assessments.html#attrition-repeated-measures"><i class="fa fa-check"></i><b>23.5.7</b> Attrition and Systematic Missingness</a></li>
<li class="chapter" data-level="23.5.8" data-path="repeated-assessments.html"><a href="repeated-assessments.html#measurementInvariance-repeated-measures"><i class="fa fa-check"></i><b>23.5.8</b> Measurement Invariance</a></li>
</ul></li>
<li class="chapter" data-level="23.6" data-path="repeated-assessments.html"><a href="repeated-assessments.html#researchDesigns"><i class="fa fa-check"></i><b>23.6</b> Types of Research Designs</a>
<ul>
<li class="chapter" data-level="23.6.1" data-path="repeated-assessments.html"><a href="repeated-assessments.html#crossSectional"><i class="fa fa-check"></i><b>23.6.1</b> Cross-Sectional Design</a></li>
<li class="chapter" data-level="23.6.2" data-path="repeated-assessments.html"><a href="repeated-assessments.html#crossSectionalSequences"><i class="fa fa-check"></i><b>23.6.2</b> Cross-Sectional Sequences Design</a></li>
<li class="chapter" data-level="23.6.3" data-path="repeated-assessments.html"><a href="repeated-assessments.html#timeLag"><i class="fa fa-check"></i><b>23.6.3</b> Time-Lag Design</a></li>
<li class="chapter" data-level="23.6.4" data-path="repeated-assessments.html"><a href="repeated-assessments.html#longitudinal"><i class="fa fa-check"></i><b>23.6.4</b> Longitudinal Design</a></li>
<li class="chapter" data-level="23.6.5" data-path="repeated-assessments.html"><a href="repeated-assessments.html#longitudinalSequences"><i class="fa fa-check"></i><b>23.6.5</b> Longitudinal Sequences Design</a>
<ul>
<li class="chapter" data-level="23.6.5.1" data-path="repeated-assessments.html"><a href="repeated-assessments.html#timeSequential"><i class="fa fa-check"></i><b>23.6.5.1</b> Time-Sequential Design</a></li>
<li class="chapter" data-level="23.6.5.2" data-path="repeated-assessments.html"><a href="repeated-assessments.html#crossSequential"><i class="fa fa-check"></i><b>23.6.5.2</b> Cross-Sequential Design</a></li>
<li class="chapter" data-level="23.6.5.3" data-path="repeated-assessments.html"><a href="repeated-assessments.html#cohortSequential"><i class="fa fa-check"></i><b>23.6.5.3</b> Cohort-Sequential Design</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="23.7" data-path="repeated-assessments.html"><a href="repeated-assessments.html#usingSequentialDesigns"><i class="fa fa-check"></i><b>23.7</b> Using Sequential Designs To Make Developmental Inferences</a></li>
<li class="chapter" data-level="23.8" data-path="repeated-assessments.html"><a href="repeated-assessments.html#stabilityVsContinuity"><i class="fa fa-check"></i><b>23.8</b> Stability Versus Continuity</a></li>
<li class="chapter" data-level="23.9" data-path="repeated-assessments.html"><a href="repeated-assessments.html#heterotypicContinuity"><i class="fa fa-check"></i><b>23.9</b> Heterotypic Continuity</a>
<ul>
<li class="chapter" data-level="23.9.1" data-path="repeated-assessments.html"><a href="repeated-assessments.html#identifyingHeterotypicContinuity"><i class="fa fa-check"></i><b>23.9.1</b> How to Identify Heterotypic Continuity</a></li>
<li class="chapter" data-level="23.9.2" data-path="repeated-assessments.html"><a href="repeated-assessments.html#theChallengeOfHeterotypicContinuity"><i class="fa fa-check"></i><b>23.9.2</b> The Problem</a></li>
<li class="chapter" data-level="23.9.3" data-path="repeated-assessments.html"><a href="repeated-assessments.html#assessingConstructsAcrossDevelopment"><i class="fa fa-check"></i><b>23.9.3</b> Ways to Assess a Construct Across Development</a>
<ul>
<li class="chapter" data-level="23.9.3.1" data-path="repeated-assessments.html"><a href="repeated-assessments.html#allPossibleContent"><i class="fa fa-check"></i><b>23.9.3.1</b> All Possible Content</a></li>
<li class="chapter" data-level="23.9.3.2" data-path="repeated-assessments.html"><a href="repeated-assessments.html#onlyCommonContent"><i class="fa fa-check"></i><b>23.9.3.2</b> Only the Common Content</a></li>
<li class="chapter" data-level="23.9.3.3" data-path="repeated-assessments.html"><a href="repeated-assessments.html#onlyConstructValidContent"><i class="fa fa-check"></i><b>23.9.3.3</b> Only the Construct-Valid Content</a></li>
</ul></li>
<li class="chapter" data-level="23.9.4" data-path="repeated-assessments.html"><a href="repeated-assessments.html#accountingForHeterotypicContinuity"><i class="fa fa-check"></i><b>23.9.4</b> Accounting for Heterotypic Continuity</a>
<ul>
<li class="chapter" data-level="23.9.4.1" data-path="repeated-assessments.html"><a href="repeated-assessments.html#ensuringStatisticalEquivalence"><i class="fa fa-check"></i><b>23.9.4.1</b> Ensuring Statistical Equivalence</a>
<ul>
<li class="chapter" data-level="23.9.4.1.1" data-path="repeated-assessments.html"><a href="repeated-assessments.html#ageNorming"><i class="fa fa-check"></i><b>23.9.4.1.1</b> Age-norming</a></li>
<li class="chapter" data-level="23.9.4.1.2" data-path="repeated-assessments.html"><a href="repeated-assessments.html#averageOrProportionScores"><i class="fa fa-check"></i><b>23.9.4.1.2</b> Average or Proportion Scores</a></li>
<li class="chapter" data-level="23.9.4.1.3" data-path="repeated-assessments.html"><a href="repeated-assessments.html#developmentalScaling"><i class="fa fa-check"></i><b>23.9.4.1.3</b> Developmental Scaling</a></li>
</ul></li>
<li class="chapter" data-level="23.9.4.2" data-path="repeated-assessments.html"><a href="repeated-assessments.html#ensuringTheoreticalEquivalence"><i class="fa fa-check"></i><b>23.9.4.2</b> Ensuring Theoretical Equivalence</a></li>
</ul></li>
<li class="chapter" data-level="23.9.5" data-path="repeated-assessments.html"><a href="repeated-assessments.html#measuresThatSpanLifetime"><i class="fa fa-check"></i><b>23.9.5</b> Are There Measures that Can Span the Lifespan?</a></li>
<li class="chapter" data-level="23.9.6" data-path="repeated-assessments.html"><a href="repeated-assessments.html#piecewiseStudyOfDevelopment"><i class="fa fa-check"></i><b>23.9.6</b> Why not Just Study Development in a Piecewise Way?</a></li>
<li class="chapter" data-level="23.9.7" data-path="repeated-assessments.html"><a href="repeated-assessments.html#studyIndividualBehaviors"><i class="fa fa-check"></i><b>23.9.7</b> Why not Just Study Individual Behaviors?</a></li>
<li class="chapter" data-level="23.9.8" data-path="repeated-assessments.html"><a href="repeated-assessments.html#heterotypicContinuitySummary"><i class="fa fa-check"></i><b>23.9.8</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="23.10" data-path="repeated-assessments.html"><a href="repeated-assessments.html#conclusion-repeated-measures"><i class="fa fa-check"></i><b>23.10</b> Conclusion</a></li>
<li class="chapter" data-level="23.11" data-path="repeated-assessments.html"><a href="repeated-assessments.html#readings-repeated-measures"><i class="fa fa-check"></i><b>23.11</b> Suggested Readings</a></li>
</ul></li>
<li class="chapter" data-level="24" data-path="cognition.html"><a href="cognition.html"><i class="fa fa-check"></i><b>24</b> Assessment of Cognition</a>
<ul>
<li class="chapter" data-level="24.1" data-path="cognition.html"><a href="cognition.html#overview-cognition"><i class="fa fa-check"></i><b>24.1</b> Overview of Cognition</a></li>
<li class="chapter" data-level="24.2" data-path="cognition.html"><a href="cognition.html#cognitionAspects"><i class="fa fa-check"></i><b>24.2</b> Aspects of Cognition Assessed</a></li>
<li class="chapter" data-level="24.3" data-path="cognition.html"><a href="cognition.html#cognitionApproaches"><i class="fa fa-check"></i><b>24.3</b> Approaches to Assessing Cognition</a>
<ul>
<li class="chapter" data-level="24.3.1" data-path="cognition.html"><a href="cognition.html#cognitionSelfReport"><i class="fa fa-check"></i><b>24.3.1</b> Self-Reports of Cognition</a>
<ul>
<li class="chapter" data-level="24.3.1.1" data-path="cognition.html"><a href="cognition.html#cognitionSelfReportPros"><i class="fa fa-check"></i><b>24.3.1.1</b> Pros</a></li>
<li class="chapter" data-level="24.3.1.2" data-path="cognition.html"><a href="cognition.html#cognitionSelfReportCons"><i class="fa fa-check"></i><b>24.3.1.2</b> Cons</a></li>
</ul></li>
<li class="chapter" data-level="24.3.2" data-path="cognition.html"><a href="cognition.html#cognitionThinkAloud"><i class="fa fa-check"></i><b>24.3.2</b> Think-Aloud Approaches (Thought Listing)</a>
<ul>
<li class="chapter" data-level="24.3.2.1" data-path="cognition.html"><a href="cognition.html#cognitionThinkAloudExamples"><i class="fa fa-check"></i><b>24.3.2.1</b> Examples</a>
<ul>
<li class="chapter" data-level="24.3.2.1.1" data-path="cognition.html"><a href="cognition.html#cognitionThinkAloudVideoReplay"><i class="fa fa-check"></i><b>24.3.2.1.1</b> Video Replay</a></li>
<li class="chapter" data-level="24.3.2.1.2" data-path="cognition.html"><a href="cognition.html#cognitionThinkAloudPrivateSpeech"><i class="fa fa-check"></i><b>24.3.2.1.2</b> Private Speech</a></li>
</ul></li>
<li class="chapter" data-level="24.3.2.2" data-path="cognition.html"><a href="cognition.html#cognitionThinkAloudPros"><i class="fa fa-check"></i><b>24.3.2.2</b> Pros</a></li>
<li class="chapter" data-level="24.3.2.3" data-path="cognition.html"><a href="cognition.html#cognitionThinkAloudCons"><i class="fa fa-check"></i><b>24.3.2.3</b> Cons</a></li>
</ul></li>
<li class="chapter" data-level="24.3.3" data-path="cognition.html"><a href="cognition.html#cognitionRandomThoughtSampling"><i class="fa fa-check"></i><b>24.3.3</b> Random Thought Sampling</a>
<ul>
<li class="chapter" data-level="24.3.3.1" data-path="cognition.html"><a href="cognition.html#cognitionRandomThoughtSamplingPros"><i class="fa fa-check"></i><b>24.3.3.1</b> Pros</a></li>
<li class="chapter" data-level="24.3.3.2" data-path="cognition.html"><a href="cognition.html#cognitionRandomThoughtSamplingCons"><i class="fa fa-check"></i><b>24.3.3.2</b> Cons</a></li>
</ul></li>
<li class="chapter" data-level="24.3.4" data-path="cognition.html"><a href="cognition.html#cognitionCognitiveScience"><i class="fa fa-check"></i><b>24.3.4</b> Cognitive Science Approaches</a>
<ul>
<li class="chapter" data-level="24.3.4.1" data-path="cognition.html"><a href="cognition.html#performanceBased"><i class="fa fa-check"></i><b>24.3.4.1</b> Performance-Based Measures</a></li>
<li class="chapter" data-level="24.3.4.2" data-path="cognition.html"><a href="cognition.html#cognitionModeling"><i class="fa fa-check"></i><b>24.3.4.2</b> Cognitive Modeling</a></li>
<li class="chapter" data-level="24.3.4.3" data-path="cognition.html"><a href="cognition.html#challenges-cognition"><i class="fa fa-check"></i><b>24.3.4.3</b> Challenges</a>
<ul>
<li class="chapter" data-level="24.3.4.3.1" data-path="cognition.html"><a href="cognition.html#differenceScores"><i class="fa fa-check"></i><b>24.3.4.3.1</b> Reliability of Difference Scores</a></li>
<li class="chapter" data-level="24.3.4.3.2" data-path="cognition.html"><a href="cognition.html#reliabilityParadox"><i class="fa fa-check"></i><b>24.3.4.3.2</b> Reliability Paradox</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="24.3.5" data-path="cognition.html"><a href="cognition.html#cognitionNeuroscience"><i class="fa fa-check"></i><b>24.3.5</b> Cognitive Neuroscience Approaches</a></li>
</ul></li>
<li class="chapter" data-level="24.4" data-path="cognition.html"><a href="cognition.html#conclusion-cognition"><i class="fa fa-check"></i><b>24.4</b> Conclusion</a></li>
<li class="chapter" data-level="24.5" data-path="cognition.html"><a href="cognition.html#readings-cognition"><i class="fa fa-check"></i><b>24.5</b> Suggested Readings</a></li>
</ul></li>
<li class="chapter" data-level="25" data-path="diversity.html"><a href="diversity.html"><i class="fa fa-check"></i><b>25</b> Cultural and Individual Diversity</a>
<ul>
<li class="chapter" data-level="25.1" data-path="diversity.html"><a href="diversity.html#terminology-diversity"><i class="fa fa-check"></i><b>25.1</b> Terminology</a>
<ul>
<li class="chapter" data-level="25.1.1" data-path="diversity.html"><a href="diversity.html#dei"><i class="fa fa-check"></i><b>25.1.1</b> Diversity, Equity, and Inclusion (DEI)</a>
<ul>
<li class="chapter" data-level="25.1.1.1" data-path="diversity.html"><a href="diversity.html#deiDiversity"><i class="fa fa-check"></i><b>25.1.1.1</b> Diversity</a></li>
<li class="chapter" data-level="25.1.1.2" data-path="diversity.html"><a href="diversity.html#equity"><i class="fa fa-check"></i><b>25.1.1.2</b> Equity</a></li>
<li class="chapter" data-level="25.1.1.3" data-path="diversity.html"><a href="diversity.html#inclusion"><i class="fa fa-check"></i><b>25.1.1.3</b> Inclusion</a></li>
</ul></li>
<li class="chapter" data-level="25.1.2" data-path="diversity.html"><a href="diversity.html#aspectsOfDifference"><i class="fa fa-check"></i><b>25.1.2</b> Aspects of Difference</a></li>
</ul></li>
<li class="chapter" data-level="25.2" data-path="diversity.html"><a href="diversity.html#multiculturalFrameworks"><i class="fa fa-check"></i><b>25.2</b> Assessing Cultural and Individual Diversity: Multicultural Assessment Frameworks</a>
<ul>
<li class="chapter" data-level="25.2.1" data-path="diversity.html"><a href="diversity.html#addressing"><i class="fa fa-check"></i><b>25.2.1</b> The ADDRESSING Model</a></li>
<li class="chapter" data-level="25.2.2" data-path="diversity.html"><a href="diversity.html#dsm5CulturalFormulationInterview"><i class="fa fa-check"></i><b>25.2.2</b> DSM-5 Cultural Formulation Interview</a></li>
<li class="chapter" data-level="25.2.3" data-path="diversity.html"><a href="diversity.html#multiculturalAssessmentProcedure"><i class="fa fa-check"></i><b>25.2.3</b> Multicultural Assessment Procedure</a></li>
<li class="chapter" data-level="25.2.4" data-path="diversity.html"><a href="diversity.html#multiculturalAssessmentInterventionProcess"><i class="fa fa-check"></i><b>25.2.4</b> Multicultural Assessment–Intervention Process</a></li>
</ul></li>
<li class="chapter" data-level="25.3" data-path="diversity.html"><a href="diversity.html#guidelinesForAssessmentWithDiversePopulations"><i class="fa fa-check"></i><b>25.3</b> Assessments with Ethnic, Linguistic, and Culturally Diverse Populations</a>
<ul>
<li class="chapter" data-level="25.3.1" data-path="diversity.html"><a href="diversity.html#researchChallengesDiversePopulations"><i class="fa fa-check"></i><b>25.3.1</b> Methodological and Conceptual Challenges in Research</a>
<ul>
<li class="chapter" data-level="25.3.1.1" data-path="diversity.html"><a href="diversity.html#raceEthnicityCulture"><i class="fa fa-check"></i><b>25.3.1.1</b> Use of Terms: Race, Ethnicity, Culture</a></li>
<li class="chapter" data-level="25.3.1.2" data-path="diversity.html"><a href="diversity.html#individualDifferencesOrGroupCharacteristics"><i class="fa fa-check"></i><b>25.3.1.2</b> Whether to Examine Individual Differences or Group Characteristics</a></li>
<li class="chapter" data-level="25.3.1.3" data-path="diversity.html"><a href="diversity.html#selectingParticipants-diversity"><i class="fa fa-check"></i><b>25.3.1.3</b> Selecting Participants</a></li>
<li class="chapter" data-level="25.3.1.4" data-path="diversity.html"><a href="diversity.html#ensuringFairGroupComparisons"><i class="fa fa-check"></i><b>25.3.1.4</b> Ensuring Fair Group Comparisons</a></li>
<li class="chapter" data-level="25.3.1.5" data-path="diversity.html"><a href="diversity.html#sampling-diversity"><i class="fa fa-check"></i><b>25.3.1.5</b> Sampling</a></li>
<li class="chapter" data-level="25.3.1.6" data-path="diversity.html"><a href="diversity.html#measurementEquivalence"><i class="fa fa-check"></i><b>25.3.1.6</b> Establishing Equivalence of Measures Across Groups</a></li>
<li class="chapter" data-level="25.3.1.7" data-path="diversity.html"><a href="diversity.html#methodsOfAsssessment-diversity"><i class="fa fa-check"></i><b>25.3.1.7</b> Methods of Assessment</a></li>
<li class="chapter" data-level="25.3.1.8" data-path="diversity.html"><a href="diversity.html#interpretationOfData-diversity"><i class="fa fa-check"></i><b>25.3.1.8</b> Interpretation of Data</a></li>
</ul></li>
<li class="chapter" data-level="25.3.2" data-path="diversity.html"><a href="diversity.html#APAguidelines-diversity"><i class="fa fa-check"></i><b>25.3.2</b> APA Guidelines</a></li>
<li class="chapter" data-level="25.3.3" data-path="diversity.html"><a href="diversity.html#trainingAndConsultation"><i class="fa fa-check"></i><b>25.3.3</b> Seek Training and Consultation</a></li>
<li class="chapter" data-level="25.3.4" data-path="diversity.html"><a href="diversity.html#awarenessOfCulturalDifferences"><i class="fa fa-check"></i><b>25.3.4</b> Develop Awareness of Cultural Differences</a></li>
<li class="chapter" data-level="25.3.5" data-path="diversity.html"><a href="diversity.html#culturalHumility"><i class="fa fa-check"></i><b>25.3.5</b> Show Cultural Humility</a></li>
<li class="chapter" data-level="25.3.6" data-path="diversity.html"><a href="diversity.html#avoidAssumptions"><i class="fa fa-check"></i><b>25.3.6</b> Do Not Assume or Stereotype</a></li>
<li class="chapter" data-level="25.3.7" data-path="diversity.html"><a href="diversity.html#adequateReprentations"><i class="fa fa-check"></i><b>25.3.7</b> Ensure Adequate Representation of Under-Represented Groups in Research</a></li>
<li class="chapter" data-level="25.3.8" data-path="diversity.html"><a href="diversity.html#appropriateMeasurement"><i class="fa fa-check"></i><b>25.3.8</b> Use Appropriate Measurement</a>
<ul>
<li class="chapter" data-level="25.3.8.1" data-path="diversity.html"><a href="diversity.html#offensiveContent"><i class="fa fa-check"></i><b>25.3.8.1</b> Do Not Use Offensive Content or Procedures</a></li>
<li class="chapter" data-level="25.3.8.2" data-path="diversity.html"><a href="diversity.html#culturallyValidMeasures"><i class="fa fa-check"></i><b>25.3.8.2</b> Use Culturally Valid Measures</a></li>
<li class="chapter" data-level="25.3.8.3" data-path="diversity.html"><a href="diversity.html#linguisticallyAppropriate"><i class="fa fa-check"></i><b>25.3.8.3</b> Use Linguistically Appropriate Assessment</a></li>
<li class="chapter" data-level="25.3.8.4" data-path="diversity.html"><a href="diversity.html#followUp-diversity"><i class="fa fa-check"></i><b>25.3.8.4</b> Use Follow-Up As Necessary</a></li>
<li class="chapter" data-level="25.3.8.5" data-path="diversity.html"><a href="diversity.html#bias-diversity"><i class="fa fa-check"></i><b>25.3.8.5</b> Examine and Correct for Test Bias</a></li>
</ul></li>
<li class="chapter" data-level="25.3.9" data-path="diversity.html"><a href="diversity.html#pathoplasticity"><i class="fa fa-check"></i><b>25.3.9</b> Consider Pathoplasticity of Constructs</a></li>
<li class="chapter" data-level="25.3.10" data-path="diversity.html"><a href="diversity.html#culturallyInformedCaseConceptualization"><i class="fa fa-check"></i><b>25.3.10</b> Formulate a Culturally Informed Case Conceptualization</a></li>
<li class="chapter" data-level="25.3.11" data-path="diversity.html"><a href="diversity.html#biasClinicalJudgment"><i class="fa fa-check"></i><b>25.3.11</b> Bias in Clinical Judgment</a>
<ul>
<li class="chapter" data-level="25.3.11.1" data-path="diversity.html"><a href="diversity.html#raceBias-clinicalJudgment"><i class="fa fa-check"></i><b>25.3.11.1</b> Race Bias</a></li>
<li class="chapter" data-level="25.3.11.2" data-path="diversity.html"><a href="diversity.html#socialClassBias-clinicalJudgment"><i class="fa fa-check"></i><b>25.3.11.2</b> Social Class Bias</a></li>
<li class="chapter" data-level="25.3.11.3" data-path="diversity.html"><a href="diversity.html#genderBias-clinicalJudgment"><i class="fa fa-check"></i><b>25.3.11.3</b> Gender Bias</a></li>
<li class="chapter" data-level="25.3.11.4" data-path="diversity.html"><a href="diversity.html#reducingBias-clinicalJudgment"><i class="fa fa-check"></i><b>25.3.11.4</b> Recommendations for Reducing Bias in Clinical Judgment</a></li>
</ul></li>
<li class="chapter" data-level="25.3.12" data-path="diversity.html"><a href="diversity.html#appropriateNorms"><i class="fa fa-check"></i><b>25.3.12</b> Use Appropriate Normative Data</a></li>
<li class="chapter" data-level="25.3.13" data-path="diversity.html"><a href="diversity.html#contextualizeGroupDifferences"><i class="fa fa-check"></i><b>25.3.13</b> Contextualize Group-Related Differences</a></li>
<li class="chapter" data-level="25.3.14" data-path="diversity.html"><a href="diversity.html#disseminationImplementation-diversity"><i class="fa fa-check"></i><b>25.3.14</b> Dissemination and Implementation</a></li>
<li class="chapter" data-level="25.3.15" data-path="diversity.html"><a href="diversity.html#inclusiveLanguage"><i class="fa fa-check"></i><b>25.3.15</b> Use Inclusive and Bias-Free Language</a></li>
</ul></li>
<li class="chapter" data-level="25.4" data-path="diversity.html"><a href="diversity.html#conclusion-diversity"><i class="fa fa-check"></i><b>25.4</b> Conclusion</a></li>
<li class="chapter" data-level="25.5" data-path="diversity.html"><a href="diversity.html#readings-diversity"><i class="fa fa-check"></i><b>25.5</b> Suggested Readings</a></li>
</ul></li>
<li class="chapter" data-level="26" data-path="reproducibility.html"><a href="reproducibility.html"><i class="fa fa-check"></i><b>26</b> Reproducibility</a>
<ul>
<li class="chapter" data-level="26.1" data-path="reproducibility.html"><a href="reproducibility.html#session-info"><i class="fa fa-check"></i><b>26.1</b> Session Info</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://psychology.uiowa.edu/people/isaac-petersen" target="blank">Isaac T. Petersen</a></li>
<li><a href="https://github.com/isaactpetersen/Principles-Psychological-Assessment/issues" target="blank">Create a GitHub Issue</a></li>
<li><a href="https://github.com/isaactpetersen/Principles-Psychological-Assessment/pulls" target="blank">Create a GitHub Pull Request</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Principles of Psychological Assessment: With Applied Examples in R</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div class="alert alert-info hints-alert">
  <div class="hints-icon">
    <i class="fa fa-comments-o"></i>
  </div>
  <div class="hints-container">
    <b>I need your help!</b>
    <p>
    I want your feedback to make the book better for you and other readers. If you find typos, errors, or places where the text may be improved, please let me know.
    The best ways to provide feedback are by <a href="https://github.com/isaactpetersen/Principles-Psychological-Assessment">GitHub</a> or <a href="https://hypothes.is">hypothes.is</a> annotations.
    </p>

    <p>
      <i class="fa fa-github"></i>
      <a href="https://github.com/isaactpetersen/Principles-Psychological-Assessment/issues">Opening an issue</a> or <a href="https://github.com/isaactpetersen/Principles-Psychological-Assessment/pulls">submitting a pull request</a> on GitHub: <a href="https://github.com/isaactpetersen/Principles-Psychological-Assessment">https://github.com/isaactpetersen/Principles-Psychological-Assessment</a>
    </p>
    <p>
      <img style="display: inline;" src="https://hypothes.is/organizations/__default__/logo" alt="Hypothesis">
      Adding an annotation using <a href="https://hypothes.is">hypothes.is</a>.
      To add an annotation, <span style="background-color: #3297FD; color: white">select some text</span> and then click the
      <span class="svg-icon--inline"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16" class="annotator-adder-actions__icon">
      <path fill="currentColor" fill-rule="nonzero" d="M15 0c.27 0 .505.099.703.297A.961.961 0 0116 1v15l-4-3H1a.974.974 0 01-.703-.29A.953.953 0 010 12V1C0 .719.096.482.29.29A.966.966 0 011 0h14zM7 3l-.469.063c-.312.041-.656.187-1.031.437-.375.25-.719.646-1.031 1.188C4.156 5.229 4 6 4 7l.002.063.006.062a.896.896 0 01.008.11l-.002.074-.006.066a1.447 1.447 0 00.43 1.188C4.729 8.854 5.082 9 5.5 9c.417 0 .77-.146 1.063-.438C6.854 8.271 7 7.918 7 7.5c0-.417-.146-.77-.438-1.063A1.447 1.447 0 005.5 6c-.073 0-.146.005-.219.016-.073.01-.14.026-.203.046.177-1.03.542-1.632 1.094-1.804L7 4V3zm5 0l-.469.063c-.312.041-.656.187-1.031.437-.375.25-.719.646-1.031 1.188C9.156 5.229 9 6 9 7l.002.063.006.062a.896.896 0 01.008.11l-.002.074-.006.066a1.447 1.447 0 00.43 1.188c.291.291.645.437 1.062.437.417 0 .77-.146 1.063-.438.291-.291.437-.645.437-1.062 0-.417-.146-.77-.438-1.063A1.447 1.447 0 0010.5 6c-.073 0-.146.005-.219.016-.073.01-.14.026-.203.046.177-1.03.542-1.632 1.094-1.804L12 4V3z"></path>
    </svg>
    </span>
      symbol on the pop-up menu.
      To see the annotations of others, click the
      <span class="svg-icon--inline"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16" class=""><g fill-rule="evenodd"><rect fill="none" stroke="none" x="0" y="0" width="16" height="16"></rect><path fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M10 12L6 8l4-4"></path></g></svg>
</span>
      symbol in the upper right-hand corner of the page.
    </p>
  </div>
</div>
<div id="bias" class="section level1 hasAnchor" number="16">
<h1><span class="header-section-number">Chapter 16</span> Test Bias<a href="bias.html#bias" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="overview-bias" class="section level2 hasAnchor" number="16.1">
<h2><span class="header-section-number">16.1</span> Overview of Bias<a href="bias.html#overview-bias" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>There are multiple definitions of the term “bias” depending on the context.
In general, bias is a <a href="reliability.html#systematicError">systematic error</a> <span class="citation">(<a href="#ref-Reynolds2012">Reynolds &amp; Suzuki, 2012</a>)</span>.
<a href="prediction.html#meanError">Mean error</a> is an example of <a href="reliability.html#systematicError">systematic error</a>, and is sometimes called bias.
Cognitive biases are systematic errors in thinking, including confirmation bias and hindsight bias.
<a href="reliability.html#methodBias">Method biases</a> are a form of <a href="reliability.html#systematicError">systematic error</a> that involve the influence of measurement on a person’s score that is not due to the person’s level on the construct.
<a href="reliability.html#methodBias">Method biases</a> include response biases or response styles, including acquiescence and social desirability bias.
Attentional bias refers to the tendency to process some types of stimuli more than others.</p>
<p>Sometimes bias is used to refer in particular to <a href="reliability.html#systematicError">systematic error</a> (in measurement, prediction, etc.) as a function of group membership, where test bias refers to the same score having different meaning for different groups.
Under this meaning, a test is unbiased if a given test score has the same meaning regardless of group membership.
For example, a test is biased if there is differential <a href="validity.html#validity">validity</a> of test scores for groups (e.g., age, education, culture, race, sex).
Test bias would exist, for instance, if a test is a less <a href="validity.html#validity">valid</a> predictor for racial minorities or linguistic minorities.
Test bias would also exist if scores on the Scholastic Aptitude Test (SAT) under-estimate women’s grades in college, for instance.</p>
<p>There are some known instances of test bias, as described in Section <a href="bias.html#biasExamples">16.3</a>.
Research has not produced much empirical evidence of test bias <span class="citation">(<a href="#ref-Brown1999">Brown et al., 1999</a>; <a href="#ref-Hall1999">Hall et al., 1999</a>; <a href="#ref-Jensen1980">Jensen, 1980</a>; <a href="#ref-Kuncel2010a">Kuncel &amp; Hezlett, 2010</a>; <a href="#ref-Reynolds2021">Reynolds et al., 2021</a>; <a href="#ref-Reynolds2012">Reynolds &amp; Suzuki, 2012</a>; <a href="#ref-Sackett2008">Sackett et al., 2008</a>; <a href="#ref-Sackett1994">Sackett &amp; Wilk, 1994</a>)</span>, though some item-level bias is not uncommon.
Moreover, where test bias has been observed, it is often small, unclear, and does not always generalize <span class="citation">(<a href="#ref-Cole1981">N. S. Cole, 1981</a>)</span>.
However, just because there is not much empirical evidence of test bias does not mean that test bias does not exist.
Moreover, just because a test does not show bias does not mean that it should be used.
Furthermore, just because a test does not show bias does not mean that there are not race-, social class-, and gender-related <a href="diversity.html#biasClinicalJudgment">biases in clinical judgment</a> during the assessment process.</p>
<p>It is also worth pointing out that group differences in scores do not necessarily indicate bias.
Group differences in scores could reflect true group differences in the construct.
For instance, women have better verbal abilities, on average, compared to men.
So, if women’s scores on a verbal ability test are higher on average than men’s scores, this would not be sufficient evidence for bias.</p>
<p>There are two broad categories of test bias:</p>
<ol style="list-style-type: decimal">
<li><a href="bias.html#predictiveBias">predictive bias</a></li>
<li><a href="bias.html#testStructureBias">test structure bias</a></li>
</ol>
<p><a href="bias.html#predictiveBias">Predictive bias</a> refers to differences between groups in the relation between the test and criterion.
As with all <a href="validity.html#criterionValidity">criterion-related validity</a> tests, the findings depend on the strength and quality of the criterion.
<a href="bias.html#testStructureBias">Test structure bias</a> refers to differences in the internal test characteristics across groups.</p>
</div>
<div id="detectBias" class="section level2 hasAnchor" number="16.2">
<h2><span class="header-section-number">16.2</span> Ways to Investigate/Detect Test Bias<a href="bias.html#detectBias" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="predictiveBias" class="section level3 hasAnchor" number="16.2.1">
<h3><span class="header-section-number">16.2.1</span> Predictive Bias<a href="bias.html#predictiveBias" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Predictive bias exists when differences emerge between groups in terms of <a href="validity.html#predictiveValidity">predictive validity</a> to a criterion.
It is assessed by using a regression line looking at the association between test score and job performance.
For instance, consider a <a href="prediction.html#confusionMatrix">2x2 confusion matrix</a> used for the standard prediction problem.
A <a href="prediction.html#confusionMatrix">confusion matrix</a> for whom to select for a job is depicted in Figure <a href="bias.html#fig:jobSelection">16.1</a>.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:jobSelection"></span>
<img src="Images/JobSelection.png" alt="2x2 Confusion Matrix for Job Selection. TP = true positive; TN = true negative; FP = false positive; FN = false negative." width="100%" />
<p class="caption">
Figure 16.1: 2x2 Confusion Matrix for Job Selection. TP = true positive; TN = true negative; FP = false positive; FN = false negative.
</p>
</div>
<p>We can also visualize the <a href="prediction.html#confusionMatrix">confusion matrix</a> in terms of a scatterplot of the test scores (i.e., predicted job performance) and the “truth” scores (i.e., actual job performance), as depicted in Figure <a href="bias.html#fig:jobSelection2">16.2</a>.
The predictor (test score) is on the x-axis.
The criterion (job performance) is on the y-axis.
The quadrants reflect the cutoffs (i.e., thresholds) imposed from the 2x2 confusion matrix.
The vertical line reflects the cutoff for selecting someone for a job.
The horizontal line reflects the cutoff for good job performance (i.e., people who should have been selected for the job).</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:jobSelection2"></span>
<img src="Book_files/figure-html/jobSelection2-1.png" alt="2x2 Confusion Matrix for Job Selection in the Form of a Graph With Predicted Performance on the x-Axis and Actual Job Performance on the y-Axis. TP = true positive; TN = true negative; FP = false positive; FN = false negative." width="2100" />
<p class="caption">
Figure 16.2: 2x2 Confusion Matrix for Job Selection in the Form of a Graph With Predicted Performance on the x-Axis and Actual Job Performance on the y-Axis. TP = true positive; TN = true negative; FP = false positive; FN = false negative.
</p>
</div>
<p>The data points in the top right quadrant are <a href="prediction.html#truePositive">true positives</a>: people who the test predicted would do a good job and who did a good job.
The data points in the bottom left quadrant are <a href="prediction.html#trueNegative">true negatives</a>: people who the test predicted would do a poor job and who would have done a poor job.
The data points in the bottom right quadrant are <a href="prediction.html#falsePositive">false positives</a>: people who the test predicted would do a good job and who did a poor job.
The data points in the top left quadrant are <a href="prediction.html#falseNegative">false negatives</a>: people who the test predicted would do a poor job and who would have done a good job.</p>
<p>Figure <a href="bias.html#fig:jobSelection3">16.3</a> depicts a strong predictor.
The best-fit regression line has a steep slope where there are lots of data points that are <a href="prediction.html#truePositive">true positives</a> and <a href="prediction.html#trueNegative">true negatives</a>, with relatively few <a href="prediction.html#falsePositive">false positives</a> and <a href="prediction.html#falseNegative">false negatives</a>.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:jobSelection3"></span>
<img src="Book_files/figure-html/jobSelection3-1.png" alt="Example of a Strong Predictor. TP = true positive; TN = true negative; FP = false positive; FN = false negative." width="2100" />
<p class="caption">
Figure 16.3: Example of a Strong Predictor. TP = true positive; TN = true negative; FP = false positive; FN = false negative.
</p>
</div>
<p>Figure <a href="bias.html#fig:jobSelection4">16.4</a> depicts a poor predictor.
The best-fit regression line has a shallow slope where there are just as many data points that are in the false cells (<a href="prediction.html#falsePositive">false positives</a> and <a href="prediction.html#falseNegative">false negatives</a>) as there are in the true cells (<a href="prediction.html#truePositive">true positives</a> and <a href="prediction.html#trueNegative">true negatives</a>).
In general, the steeper the slope, the better the predictor.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:jobSelection4"></span>
<img src="Book_files/figure-html/jobSelection4-1.png" alt="Example of a Poor Predictor. TP = true positive; TN = true negative; FP = false positive; FN = false negative." width="2100" />
<p class="caption">
Figure 16.4: Example of a Poor Predictor. TP = true positive; TN = true negative; FP = false positive; FN = false negative.
</p>
</div>
<p>We can evaluate predictive bias using a best-fit regression line between the predictor and criterion for each group.</p>
<div id="typesPredictiveBias" class="section level4 hasAnchor" number="16.2.1.1">
<h4><span class="header-section-number">16.2.1.1</span> Types of Predictive Bias<a href="bias.html#typesPredictiveBias" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>There are three types of predictive bias:</p>
<ol style="list-style-type: decimal">
<li><a href="bias.html#differentSlopes">Different slopes</a></li>
<li><a href="bias.html#differentIntercepts">Different intercepts</a></li>
<li><a href="bias.html#differentInterceptsAndSlopes">Different intercepts and slopes</a></li>
</ol>
<p>The slope of the regression line is the steepness of the line.
The intercept of the regression line is the y-value of the point where the line crosses the y-axis (i.e., when <span class="math inline">\(x = 0\)</span>).
If a measure shows predictive test bias, when looking at the regression line for each group, the groups’ regression lines differ in either slopes and/or intercepts.</p>
<div id="differentSlopes" class="section level5 hasAnchor" number="16.2.1.1.1">
<h5><span class="header-section-number">16.2.1.1.1</span> Different Slopes<a href="bias.html#differentSlopes" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Predictive bias in terms of different slopes exists when there are differences in the slope of the regression line between minority and majority groups.
The slope describes the <em>direction</em> and <em>steepness</em> of the regression line.
The slope of a regression line is the amount of change in <span class="math inline">\(y\)</span> for every unit change in <span class="math inline">\(x\)</span> (i.e., rise over run).
Differing slopes indicate differential <a href="validity.html#predictiveValidity">predictive validity</a>, in which the test is a more effective predictor of performance in one group over the other.
Different slopes predictive bias is depicted in Figure <a href="bias.html#fig:testBias1">16.5</a>.
In the figure, the predictor performs well in the majority group.
However, the slope is close to zero in the minority group, indicating that there is no association between the predictor and the criterion for the minority group.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:testBias1"></span>
<img src="Images/testBias-01.png" alt="Test Bias: Different Slopes. TP = true positive; TN = true negative; FP = false positive; FN = false negative." width="100%" />
<p class="caption">
Figure 16.5: Test Bias: Different Slopes. TP = true positive; TN = true negative; FP = false positive; FN = false negative.
</p>
</div>
<p>Different slopes can especially occur if we develop our measure and criterion based on the normative majority group.
Not much evidence has found empirical evidence of different slopes across groups.
However, samples often do not have the power to detect differing slopes <span class="citation">(<a href="#ref-Aguinis2010a">Aguinis et al., 2010</a>)</span>.
Theoretically, to fix biases related to different slopes, you should find another measure that is more predictive for the minority group.
If the predictor is a strong predictor in both groups but shows slight differences in the slope, <a href="bias.html#withinGroupNorming">within-group norming</a> could be used.</p>
</div>
<div id="differentIntercepts" class="section level5 hasAnchor" number="16.2.1.1.2">
<h5><span class="header-section-number">16.2.1.1.2</span> Different Intercepts<a href="bias.html#differentIntercepts" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Predictive bias in terms of different intercepts exists when there are differences in the intercept of the regression line between minority and majority groups.
The <span class="math inline">\(y\)</span>-intercept describes the point on the <span class="math inline">\(y\)</span>-axis that the line intersects with the <span class="math inline">\(y\)</span>-axis (when <span class="math inline">\(x = 0\)</span>).
When the distributions have similar slopes, intercept differences suggest that the measure systematically under- or over-estimates group performance relative to the person’s ability.
The same test score leads to systematically different predictions for the majority and minority groups.
In other words, minority group members get different tests scores than majority group members with the same ability.
Different intercepts predictive bias is depicted in Figure <a href="bias.html#fig:testBias2">16.6</a>.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:testBias2"></span>
<img src="Images/testBias-02.png" alt="Test Bias: Different Intercepts. TP = true positive; TN = true negative; FP = false positive; FN = false negative." width="100%" />
<p class="caption">
Figure 16.6: Test Bias: Different Intercepts. TP = true positive; TN = true negative; FP = false positive; FN = false negative.
</p>
</div>
<p>A higher intercept (relative to zero) indicates that the measure <em>under</em>-estimates a person’s ability (at that test score)—i.e., the person’s job performance is better than what the test score would suggest.
A lower intercept (relative to zero) indicates that the measure <em>over</em>-estimates a person’s ability (at that test score)—i.e., the person’s job performance is worse than what the test score would suggest.
Figure <a href="bias.html#fig:testBias2">16.6</a> indicates that the measure systematically under-estimates the job performance of the minority group.</p>
<p>Performance among members of a minority group could be under- or over-estimated.
For example, historically, women’s grades in math and engineering classes tended to be under-estimated by the Scholastic Aptitude Test [SAT; <span class="citation">M. J. Clark &amp; Grandy (<a href="#ref-Clark1984">1984</a>)</span>].
However, where intercept differences have been observed, measures often show small <em>over</em>-estimation of school and job performance among minority groups <span class="citation">(<a href="#ref-Reynolds2012">Reynolds &amp; Suzuki, 2012</a>)</span>.
For example, women’s physical strength and endurance is over-estimated based on physical ability tests <span class="citation">(<a href="#ref-Sackett1994">Sackett &amp; Wilk, 1994</a>)</span>.
In addition, over-estimation of African Americans’ and Hispanics’ school and job performance has been observed based on cognitive ability tests <span class="citation">(<a href="#ref-Cole1981">N. S. Cole, 1981</a>; <a href="#ref-Reynolds2012">Reynolds &amp; Suzuki, 2012</a>; <a href="#ref-Sackett2008">Sackett et al., 2008</a>; <a href="#ref-Sackett1994">Sackett &amp; Wilk, 1994</a>)</span>.
At the same time, the Black–White difference in job performance is less than the Black–White difference in test performance.</p>
<p>The over-prediction of lower-scoring groups is likely mostly an artifact of <a href="reliability.html#measurementError">measurement error</a> <span class="citation">(<a href="#ref-Gottfredson1994">L. S. Gottfredson, 1994</a>)</span>.
The over-estimation of African Americans’ and Hispanics’ school and job performance may be due to <a href="reliability.html#measurementError">measurement error</a> in the tests.
Moreover, test scores explain only a portion of the variation in job performance.
Black people are far less disadvantaged on the noncognitive determinants of job performance than on the cognitive ones.
Nevertheless, the over-estimation that has been often observed is <em>on average</em>—the performance is not over-estimated for all individuals of the groups even if there is an average over-estimation effect.
In addition, simulation findings indicate that lower intercepts (i.e., over-estimation) among minority groups compared to majority groups could be observed if there are different slopes but not different intercepts in the population, because different slopes are likely to go undetected due to low power <span class="citation">(<a href="#ref-Aguinis2010a">Aguinis et al., 2010</a>)</span>.
That is, if a test shows weaker <a href="validity.html#validity">validity</a> for a minority group than the majority group, it could appear as different intercepts that favor the minority group when, in fact, it reflects shallower slopes of the minority group that go undetected.</p>
<p>Predictive biases in intercepts could especially occur if we develop tests that are based on the majority group, and the items assess constructs other than the construct of interest which are systematically biased in favor of the majority group or against the minority group.
Arguments about reduced power to detect differences are less relevant for intercepts and means than for slopes.</p>
<p>To correct for a bias in intercepts, we could add <a href="bias.html#bonusPoints">bonus points</a> to the scores for the minority group to correct for the amount of the <a href="reliability.html#systematicError">systematic error</a>, and to result in the same regression line.
But if the minority group is over-predicted (as has often been the case where intercept differences have been observed), we would not want to use <a href="bias.html#scoreAdjustment">score adjustment</a> to lower the minority group’s scores.</p>
</div>
<div id="differentInterceptsAndSlopes" class="section level5 hasAnchor" number="16.2.1.1.3">
<h5><span class="header-section-number">16.2.1.1.3</span> Different Intercepts and Slopes<a href="bias.html#differentInterceptsAndSlopes" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Predictive bias in terms of different intercepts and slopes exists when there are differences in the intercept and slope of the regression line between minority and majority groups.
In cases of different intercepts and slopes, there is both differential <a href="validity.html#validity">validity</a> (because the regression lines have different slopes), as well as varying under- and over-estimation of groups’ performance at particular scores.
Different intercepts and slopes predictive bias is depicted in Figure <a href="bias.html#fig:testBias3">16.7</a>.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:testBias3"></span>
<img src="Images/testBias-03.png" alt="Test Bias: Different Intercepts and Slopes. TP = true positive; TN = true negative; FP = false positive; FN = false negative." width="100%" />
<p class="caption">
Figure 16.7: Test Bias: Different Intercepts and Slopes. TP = true positive; TN = true negative; FP = false positive; FN = false negative.
</p>
</div>
<p>In instances of different intercepts and slopes predictive bias, a measure can simultaneously over-estimate and under-estimate a person’s ability at different test scores.
For instance, a measure can under-estimate a person’s ability at higher test scores and can over-estimate a person’s ability at lower test scores.</p>
<p>Different intercepts and slopes across groups is possibly more realistic than just different intercepts or just different slopes.
However, different intercepts and slopes predictive bias is more complicated to study, represent, and resolve.
It is difficult to examine because of complexity, and it is not easy to fix.
Currently, we have nothing to address different intercepts and slopes predictive bias.
We would need to use a different measure or measures for each group.</p>
</div>
</div>
</div>
<div id="testStructureBias" class="section level3 hasAnchor" number="16.2.2">
<h3><span class="header-section-number">16.2.2</span> Test Structure Bias<a href="bias.html#testStructureBias" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>In addition to predictive bias, another type of test bias is test structure bias.
Test structure bias involves differences in internal test characteristics across groups.
Examining test structure bias is different from examining the total score, as is used when examining predictive bias.
Test structure bias can be identified empirically or based on theory/judgment.</p>
<p>Empirically, test structure bias can be examined in multiple ways.</p>
<div id="testStructureBiasEmpirical" class="section level4 hasAnchor" number="16.2.2.1">
<h4><span class="header-section-number">16.2.2.1</span> Empirical Approaches to Identification<a href="bias.html#testStructureBiasEmpirical" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<div id="itemXgroupTests-bias" class="section level5 hasAnchor" number="16.2.2.1.1">
<h5><span class="header-section-number">16.2.2.1.1</span> Item <span class="math inline">\(\times\)</span> Group tests (ANOVA)<a href="bias.html#itemXgroupTests-bias" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Item <span class="math inline">\(\times\)</span> Group tests in analysis of variance (ANOVA) examine whether the difference between groups on the overall score match comparisons among smaller items sets between groups.
Item <span class="math inline">\(\times\)</span> Group tests are used to rule out that items are operating in different ways in different groups.
If the items operate in different ways in different groups, they do not have the same meaning across groups.
For example, if we are going to use a measure for multiple groups, we would expect its items to operate similarly across groups.
So, if women show higher scores on a depression measure compared to men, would also expect them to show similar elevations on each item (e.g., sleep loss).</p>
</div>
<div id="irt-bias" class="section level5 hasAnchor" number="16.2.2.1.2">
<h5><span class="header-section-number">16.2.2.1.2</span> Item Response Theory<a href="bias.html#irt-bias" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Using <a href="irt.html#irt">item response theory</a>, we can examine <a href="bias.html#dif">differential item functioning</a> (DIF).
Evidence of <a href="bias.html#dif">DIF</a>, indicates that there are differences between group in terms of <a href="irt.html#itemDiscrimination">discrimination</a> and/or <a href="irt.html#itemDifficulty">difficulty/severity</a> of items.
Differences between groups in terms of the <a href="irt.html#icc">item characteristic curve</a> (which combines the item’s <a href="irt.html#itemDiscrimination">discrimination</a> and <a href="irt.html#itemDifficulty">severity</a>) would be evidence against <a href="validity.html#constructValidity">construct validity</a> invariance between the groups and would provide evidence of bias.
<a href="bias.html#dif">DIF</a> examines stretching and compression of different groups.
As an example, consider the item “bites others” in relation to externalizing problems.
The item would be expected to show a weaker <a href="irt.html#itemDiscrimination">discrimination</a> and higher <a href="irt.html#itemDifficulty">severity</a> in adults compared to children.
<a href="bias.html#dif">DIF</a> is discussed in Section <a href="bias.html#dif">16.9</a>.</p>
</div>
<div id="cfa-bias" class="section level5 hasAnchor" number="16.2.2.1.3">
<h5><span class="header-section-number">16.2.2.1.3</span> Confirmatory Factor Analysis<a href="bias.html#cfa-bias" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p><a href="sem.html#cfa-sem">Confirmatory factor analysis</a> allows tests of <a href="bias.html#measurementInvariance">measurement invariance</a> (also called factorial invariance).
<a href="bias.html#measurementInvariance">Measurement invariance</a> examines whether the factor structure of the underlying latent variables in the test is consistent across groups.
It also examines whether the manifestation of the construct differs between groups.
<a href="bias.html#measurementInvariance">Measurement invariance</a> is discussed in Section <a href="bias.html#measurementInvariance">16.10</a>.</p>
<p>Even if you find the same slope and intercepts across groups in a prediction model, the measure would still be assessing different constructs across groups if the measure has a different factor structure between the groups.
A different factor structure across groups is depicted in Figure <a href="bias.html#fig:testBias4">16.8</a>.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:testBias4"></span>
<img src="Images/testBias-04.png" alt="Different Factor Structure Across Groups." width="100%" />
<p class="caption">
Figure 16.8: Different Factor Structure Across Groups.
</p>
</div>
<p>An example of a different factor structure across groups is the differentiation of executive functions from two factors to three factors (inhibition, working memory, cognitive flexibility) across childhood <span class="citation">(<a href="#ref-Lee2013">Lee et al., 2013</a>)</span>.</p>
<p>There are different degrees of measurement invariance <span class="citation">(for a review, see <a href="#ref-Putnick2016">Putnick &amp; Bornstein, 2016</a>)</span>:</p>
<ul>
<li>Configural invariance: same number of factors in each group, and which indicators load on which factors are the same in each group (i.e., the same pattern of significant loadings in each group).</li>
<li>Metric (“weak factorial”) invariance: items have the same factor loadings (<a href="irt.html#itemDiscrimination">discrimination</a>) in each group.</li>
<li>Scalar (“strong factorial”) invariance: items have the same intercepts (<a href="irt.html#itemDifficulty">difficulty/severity</a>) in each group.</li>
<li>Residual (“strict factorial”) invariance: items have the same residual/unique variances in each group.</li>
</ul>
</div>
<div id="sem-bias" class="section level5 hasAnchor" number="16.2.2.1.4">
<h5><span class="header-section-number">16.2.2.1.4</span> Structural Equation Modeling<a href="bias.html#sem-bias" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p><a href="sem.html#sem">Structural equation modeling</a> is a <a href="factor-analysis-PCA.html#cfa">confirmatory factor analysis</a> (CFA) model that incorporates prediction.
Structural equation modeling allows examining differences in the underlying structure with differences in prediction in the same model.</p>
</div>
<div id="sdt-bias" class="section level5 hasAnchor" number="16.2.2.1.5">
<h5><span class="header-section-number">16.2.2.1.5</span> Signal Detection Theory<a href="bias.html#sdt-bias" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p><a href="prediction.html#sdt">Signal detection theory</a> is a dynamic measure of bias.
It allows examining the overall bias in selection systems, including both accuracy and errors at various cutoffs (<a href="prediction.html#sensitivity">sensitivity</a>, <a href="prediction.html#specificity">specificity</a>, <a href="prediction.html#ppv">positive predictive value</a>, and <a href="prediction.html#npv">negative predictive value</a>), as well as accuracy across all possible cutoffs (the <a href="prediction.html#auc">area under the receiver operating characteristic curve</a>).
While there may be similar <a href="validity.html#predictiveValidity">predictive validity</a> between groups, the type of errors we are making across groups might differ.
It is important to decide which types of error to emphasize depending on the fairness goals and examining <a href="prediction.html#sensitivity">sensitivity</a>/<a href="prediction.html#specificity">specificity</a> to adjust cutoffs.</p>
</div>
<div id="evidenceTestStructureBias" class="section level5 hasAnchor" number="16.2.2.1.6">
<h5><span class="header-section-number">16.2.2.1.6</span> Empirical Evidence of Test Structure Bias<a href="bias.html#evidenceTestStructureBias" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>It is not uncommon to find items that show differences across groups in <a href="irt.html#itemDifficulty">severity</a> (intercepts) and/or <a href="irt.html#itemDiscrimination">discrimination</a> (factor loadings).
However, cross-group differences in item functioning tend to be small and not consistent across studies, suggesting that some of the differences may reflect Type I errors that result from sampling error and multiple testing.
That said, some instances of cross-group differences in item parameters could reflect test structure bias that is real and important to address.</p>
</div>
</div>
<div id="testStructureBiasTheoretical" class="section level4 hasAnchor" number="16.2.2.2">
<h4><span class="header-section-number">16.2.2.2</span> Theoretical/Judgmental Approaches to Identification<a href="bias.html#testStructureBiasTheoretical" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<div id="facialValidityBias" class="section level5 hasAnchor" number="16.2.2.2.1">
<h5><span class="header-section-number">16.2.2.2.1</span> Facial Validity Bias<a href="bias.html#facialValidityBias" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p><a href="validity.html#faceValidity">Facial validity</a> bias considers the extent to which an average person thinks that an item is biased—i.e., the item has differing <a href="validity.html#validity">validity</a> between minority and majority groups.
If so, the item should be reconsidered.
Does an item disfavor certain groups?
Is the language specific to a particular group?
Is it offensive to some people?
This type of judgment moves into the realm of whether or not an item should be used.</p>
</div>
<div id="contentValidityBias" class="section level5 hasAnchor" number="16.2.2.2.2">
<h5><span class="header-section-number">16.2.2.2.2</span> Content Validity Bias<a href="bias.html#contentValidityBias" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p><a href="validity.html#contentValidity">Content validity</a> bias is determined by judgments of construct experts who look for items that do not do an adequate job assessing the construct between groups.
A construct may include some content facets in one group, but may include different content facets in another group, as depicted in Figure <a href="bias.html#fig:testBias5">16.9</a>.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:testBias5"></span>
<img src="Images/testBias-05.png" alt="Different Content Facets in a Given Construct for Two Groups." width="100%" />
<p class="caption">
Figure 16.9: Different Content Facets in a Given Construct for Two Groups.
</p>
</div>
<p>Examples include information questions and vocabulary questions on the Wechsler Adult Intelligence Scale.
If an item is linguistically complicated, grammatically complex or convoluted, or a double negative, it may be less <a href="validity.html#validity">valid</a> or predictive for rural populations and those with less education.</p>
<p>Also, stereotype threat may contribute to <a href="validity.html#contentValidity">content validity</a> bias.
Stereotype threat occurs when people are or feel at risk of conforming themselves to stereotypes about their social group, thus leading them to show poorer performance in ways that are consistent with the stereotype.
Stereotype threat may partially explain why some women may perform more poorly on some math items than some men.</p>
<p>Another example of <a href="validity.html#contentValidity">content validity</a> bias is when the same measure is used to assess a construct across ages even though the construct shows heterotypic continuity.
Heterotypic continuity occurs when a construct changes in its behavioral manifestation with development <span class="citation">(<a href="#ref-Petersen2020">Petersen et al., 2020</a>)</span>.
That is, the same construct may look different at different points in development.
An example of a construct that shows heterotypic continuity is externalizing problems.
In early childhood, externalizing problems often manifest in overt forms, including physical aggression (e.g., biting) and temper tantrums.
By contrast, in adolescence and adulthood, externalizing problems more often manifest in covert ways, including relational aggression and substance use.
<a href="validity.html#contentValidity">Content validity</a> and <a href="validity.html#faceValidity">facial validity</a> bias judgments are often related, but not always.</p>
</div>
</div>
</div>
</div>
<div id="biasExamples" class="section level2 hasAnchor" number="16.3">
<h2><span class="header-section-number">16.3</span> Examples of Bias<a href="bias.html#biasExamples" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>As described in the overview in Section <a href="bias.html#overview-bias">16.1</a>, there is not much empirical evidence of test bias <span class="citation">(<a href="#ref-Brown1999">Brown et al., 1999</a>; <a href="#ref-Hall1999">Hall et al., 1999</a>; <a href="#ref-Jensen1980">Jensen, 1980</a>; <a href="#ref-Kuncel2010a">Kuncel &amp; Hezlett, 2010</a>; <a href="#ref-Reynolds2021">Reynolds et al., 2021</a>; <a href="#ref-Reynolds2012">Reynolds &amp; Suzuki, 2012</a>; <a href="#ref-Sackett2008">Sackett et al., 2008</a>; <a href="#ref-Sackett1994">Sackett &amp; Wilk, 1994</a>)</span>.
That said, some item-level bias is not uncommon.
One instance of test bias is that, historically, women’s grades in math and engineering classes tended to be under-estimated by the Scholastic Aptitude Test [SAT; <span class="citation">M. J. Clark &amp; Grandy (<a href="#ref-Clark1984">1984</a>)</span>].
<span class="citation">Fernández &amp; Abe (<a href="#ref-Fernandez2018">2018</a>)</span> review the evidence on other instances of test and item bias.
For instance, test bias can occur if a subgroup is less familiar with the language, the stimulus material, or the response procedures, or if they have different <a href="reliability.html#methodBias-types">response styles</a>.
In addition to test bias, there are known patterns of <a href="diversity.html#biasClinicalJudgment">bias in clinical judgment</a>, as described in Section <a href="diversity.html#biasClinicalJudgment">25.3.11</a>.</p>
</div>
<div id="fairness" class="section level2 hasAnchor" number="16.4">
<h2><span class="header-section-number">16.4</span> Test Fairness<a href="bias.html#fairness" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>There is interest in examining more than just the accuracy of measures.
It is also important to examine the <em>errors</em> being made and differentiate the weight or value of different kinds of errors (and correct decisions).
Consider an example of an unbiased test, as depicted in Figure <a href="bias.html#fig:testBias6">16.10</a>, adapted from <span class="citation">L. S. Gottfredson (<a href="#ref-Gottfredson1994">1994</a>)</span>.
Although the example is of a White group and a Black group, we could substitute any two groups into the example (e.g., males versus females).</p>

<div class="figure" style="text-align: center"><span style="display:block;" id="fig:testBias6"></span>
<img src="Images/testBias-06.png" alt="Potential Unfairness in Testing. The ovals represent the distributions of individuals’ performance both on a test and a job performance criterion. TP = true positive; TN = true negative; FP = false positive; FN = false negative. (Adapted from L. S. Gottfredson (1994), Figure 1, p. 958. Gottfredson, L. S. (1994). The science and politics of race-norming. American Psychologist, 49(11), 955–963. https://doi.org/10.1037/0003-066X.49.11.955)" width="100%" />
<p class="caption">
Figure 16.10: Potential Unfairness in Testing. The ovals represent the distributions of individuals’ performance both on a test and a job performance criterion. TP = true positive; TN = true negative; FP = false positive; FN = false negative. (Adapted from <span class="citation">L. S. Gottfredson (<a href="#ref-Gottfredson1994">1994</a>)</span>, Figure 1, p. 958. Gottfredson, L. S. (1994). The science and politics of race-norming. <em>American Psychologist</em>, <em>49</em>(11), 955–963. <a href="https://doi.org/10.1037/0003-066X.49.11.955" class="uri">https://doi.org/10.1037/0003-066X.49.11.955</a>)
</p>
</div>
<p>The example is of an unbiased test between White and Black job applicants.
There are no differences between the two groups in terms of slope.
If we drew a regression line, the line would go through the centroid of both ovals.
Thus, the measure is equally predictive in both groups even though that the Black group failed the test at a higher rate than the White group.
Moreover, there is no difference between the groups in terms of intercept.
Thus, the performance of one group is not over-estimated relative to the performance of the other group.
To demonstrate what a different intercept would look like, Group X shows a different intercept.
In sum, there is no <a href="validity.html#predictiveValidity">predictive validity</a> bias between the two groups.
But just because the test predicts just as well in both groups does not mean that the selection procedures are <em>fair</em>.</p>
<p>Although the test is unbiased, there are differences in the <em>quality</em> of prediction: there are more <a href="prediction.html#falseNegative">false negatives</a> in the Black group compared to the White group.
This gives the White group an advantage and the Black group additional disadvantages.
If the measure showed the same quality of prediction, we would say the test is fair.
The point of the example is that just because a test is unbiased does not mean that the test is fair.</p>
<p>There are two kinds of errors: <a href="prediction.html#falseNegative">false negatives</a> and <a href="prediction.html#falsePositive">false positives</a>.
Each error type has very different implications.
<a href="prediction.html#falseNegative">False negatives</a> would be when the test predicts that an applicant would perform poorly and we do not give them the job even though they would have performed well.
<a href="prediction.html#falseNegative">False negatives</a> have a negative effect on the applicant.
And, in this example, there are more <a href="prediction.html#falseNegative">false negatives</a> in the Black group.
By contrast, <a href="prediction.html#falsePositive">false positives</a> would be when we predict that an applicant would do well, and we give them the job but they perform poorly.
<a href="prediction.html#falsePositive">False positives</a> are a benefit to the applicant but have a negative effect on the employer.
In this example, there are more <a href="prediction.html#falsePositive">false positives</a> in the White group, which is an undeserved benefit based on the <a href="prediction.html#selectionRatio">selection ratio</a>; therefore, the White group benefits.</p>
<p>In sum, equal accuracy of prediction (i.e., equal total number of errors) does not necessarily mean the test is fair; we must examine the types of errors.
Merely ensuring accuracy does not ensure fairness!</p>
<div id="adverseImpact" class="section level3 hasAnchor" number="16.4.1">
<h3><span class="header-section-number">16.4.1</span> Adverse Impact<a href="bias.html#adverseImpact" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Adverse impact is defined as rejecting members of one group at a higher rate than another group.
Adverse impact is different from test <a href="validity.html#validity">validity</a>.
According to federal guidelines, adverse impact is present if the <a href="prediction.html#selectionRatio">selection rate</a> of one group is less than four-fifths (80%) the <a href="prediction.html#selectionRatio">selection rate</a> of the group with the highest <a href="prediction.html#selectionRatio">selection rate</a>.</p>
<p>There is much more evidence of adverse impact than test bias.
Indeed, disparate impact of tests on personnel selection across groups is the norm rather than the exception, even when using <a href="validity.html#validity">valid</a> tests that are unbiased, which in part reflect group-related differences in job-related skills <span class="citation">(<a href="#ref-Gottfredson1994">L. S. Gottfredson, 1994</a>)</span>.
Examples of adverse impact include:</p>
<ul>
<li>physical ability tests, which produce substantial adverse impact against women (despite over-estimation of women’s performance),</li>
<li>cognitive ability tests, which produce substantial impact against some ethnic minority groups, especially Black and Hispanic people (despite over-estimation of Black and Hispanic people’s performance), even though cognitive ability tests tend to be among the strongest predictors of job performance <span class="citation">(<a href="#ref-Sackett2008">Sackett et al., 2008</a>; <a href="#ref-Schmidt1981">Schmidt &amp; Hunter, 1981</a>)</span>, and</li>
<li>personality tests, which produce higher estimates of dominance among men than women; it is unclear whether this has <a href="bias.html#predictiveBias">predictive bias</a>.</li>
</ul>
</div>
<div id="biasVsFairness" class="section level3 hasAnchor" number="16.4.2">
<h3><span class="header-section-number">16.4.2</span> Bias Versus Fairness<a href="bias.html#biasVsFairness" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Whether a measure is accurate or shows test bias is a scientific question.
By contrast, whether a test is fair and thus should be used for a given purpose is not just a scientific question; it is also an <a href="ethics.html#ethics">ethical</a> question.
It involves the consideration of the potential consequences of testing in terms of social values and <a href="validity.html#consequentialValidity">consequential validity</a>.</p>
</div>
<div id="operationalizingFairness" class="section level3 hasAnchor" number="16.4.3">
<h3><span class="header-section-number">16.4.3</span> Operationalizing Fairness<a href="bias.html#operationalizingFairness" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>There are many perspectives to what should be considered when evaluating test fairness <span class="citation">(<a href="#ref-AERA2014">American Educational Research Association et al., 2014</a>; <a href="#ref-Camilli2013">Camilli, 2013</a>; <a href="#ref-GeneralAptitudeTestBattery1989">Committee on the General Aptitude Test Battery et al., 1989</a>; <a href="#ref-Dorans2017">Dorans, 2017</a>; <a href="#ref-Fletcher2021">Fletcher et al., 2021</a>; <a href="#ref-Gipps2009">Gipps &amp; Stobart, 2009</a>; <a href="#ref-Helms2006">Helms, 2006</a>; <a href="#ref-Jonson2022">Jonson &amp; Geisinger, 2022</a>; <a href="#ref-Melikyan2019">Melikyan et al., 2019</a>; <a href="#ref-Sackett2008">Sackett et al., 2008</a>; <a href="#ref-Thorndike1971">Thorndike, 1971</a>; <a href="#ref-Zieky2006">Zieky, 2006</a>, <a href="#ref-Zieky2013">2013</a>)</span>.
As described in <span class="citation">Fletcher et al. (<a href="#ref-Fletcher2021">2021</a>)</span>, there are three primary ways of operationalizing fairness:</p>
<ol style="list-style-type: decimal">
<li>Equal outcomes: the <a href="prediction.html#selectionRatio">selection rate</a> is the same across groups.</li>
<li>Equal opportunity: the <a href="prediction.html#sensitivity">sensitivity</a> (<a href="prediction.html#sensitivity">true positive rate</a>; 1 <span class="math inline">\(-\)</span> <a href="prediction.html#falseNegativeRate">false negative rate</a>) is the same across groups.</li>
<li>Equal odds: the <a href="prediction.html#sensitivity">sensitivity</a> is the same across groups and the <a href="prediction.html#specificity">specificity</a> (<a href="prediction.html#specificity">true negative rate</a>; 1 <span class="math inline">\(-\)</span> <a href="prediction.html#falsePositiveRate">false positive rate</a>) is the same across groups.</li>
</ol>
<p>For example, the job selection procedure shows <em>equal outcomes</em> if the proportion of men selected is equal to the proportion of women selected.
The job selection procedure shows <em>equal opportunity</em> if, among those who show strong job performance, the proportion of classification errors (<a href="prediction.html#falseNegative">false negatives</a>) is the same for men and women.
<a href="prediction.html#roc">Receiver operating characteristic (ROC) curves</a> are depicted for two groups in Figure <a href="bias.html#fig:testBias-ROC">16.11</a>.
A cutoff that represents equal opportunity is depicted with a horizontal line (i.e., the same <a href="prediction.html#sensitivity">sensitivity</a>) in Figure <a href="bias.html#fig:testBias-ROC">16.11</a>.
The job selection procedure shows <em>equal odds</em> if (a), among those who show strong job performance, the proportion of classification errors (<a href="prediction.html#falseNegative">false negatives</a>) is the same for men and women, and (b), among those who show poor job performance, the proportion of classification errors (<a href="prediction.html#falsePositive">false positives</a>) is the same for men and women.
A cutoff that represents equal odds is depicted where the <a href="prediction.html#roc">ROC curve</a> for Group A intersects with the <a href="prediction.html#roc">ROC curve</a> from Group B in Figure <a href="bias.html#fig:testBias-ROC">16.11</a>.
The equal odds approach to fairness is consistent with a National Academy of Sciences committee on fairness <span class="citation">(<a href="#ref-GeneralAptitudeTestBattery1989">Committee on the General Aptitude Test Battery et al., 1989</a>; <a href="#ref-Gottfredson1994">L. S. Gottfredson, 1994</a>)</span>.
Approaches to operationalizing fairness in the context of prediction models are described by <span class="citation">Paulus &amp; Kent (<a href="#ref-Paulus2020">2020</a>)</span>.</p>

<div class="figure" style="text-align: center"><span style="display:block;" id="fig:testBias-ROC"></span>
<img src="Images/testBias-ROC.png" alt="Receiver Operating Characteristic (ROC) Curves for Two Groups. (Figure reprinted from Fletcher et al. (2021), Figure 2, p. 3. Fletcher, R. R., Nakeshimana, A., &amp; Olubeko, O. (2021). Addressing fairness, bias, and appropriate use of artificial intelligence and machine learning in global health. Frontiers in Artificial Intelligence, 3(116). https://doi.org/10.3389/frai.2020.561802)" width="100%" />
<p class="caption">
Figure 16.11: Receiver Operating Characteristic (ROC) Curves for Two Groups. (Figure reprinted from <span class="citation">Fletcher et al. (<a href="#ref-Fletcher2021">2021</a>)</span>, Figure 2, p. 3. Fletcher, R. R., Nakeshimana, A., &amp; Olubeko, O. (2021). Addressing fairness, bias, and appropriate use of artificial intelligence and machine learning in global health. <em>Frontiers in Artificial Intelligence</em>, <em>3</em>(116). <a href="https://doi.org/10.3389/frai.2020.561802">https://doi.org/10.3389/frai.2020.561802</a>)
</p>
</div>
<p>It is not possible to meet all three types of fairness simultaneously (i.e., equal <a href="prediction.html#selectionRatio">selection rates</a>, <a href="prediction.html#sensitivity">sensitivity</a>, and <a href="prediction.html#specificity">specificity</a> across groups) unless the <a href="prediction.html#baseRate">base rates</a> are the same across groups or the selection is perfectly accurate <span class="citation">(<a href="#ref-Fletcher2021">Fletcher et al., 2021</a>)</span>.
In the medical context, equal odds is the most common approach to fairness.
However, using the cutoff associated with equal odds typically reduces overall classification accuracy.
And, changing the cutoff for specific groups can lead to negative consequences.
In the case that equal odds results in a classification accuracy that is too low, it may be worth considering using separate assessment procedures/tests for each group.
In general, it is best to follow one of these approaches to fairness.
It is difficult to get right, so try to minimize negative impact.
Many fairness supporters argue for simpler rules.
In the 1991 Civil Rights Act, <a href="bias.html#scoreAdjustment">score adjustments</a> based on race, gender, and ethnicity (e.g., within-race <a href="scoresScales.html#norm">norming</a> or race-conscious score adjustments) were made illegal in personnel selection <span class="citation">(<a href="#ref-Gottfredson1994">L. S. Gottfredson, 1994</a>)</span>.</p>
<p>Another perspective to fairness is that selection procedures should predict job performance and if they are correlated with any group membership (e.g., race, socioeconomic status, or gender), the test should not be used <span class="citation">(<a href="#ref-Helms2006">Helms, 2006</a>)</span>.
That is, according to Helms, we should not use any test that assesses anything other than the construct of interest (job performance).
Unfortunately, however, no measures like this exist.
Every measure assesses multiple things, and factors such as poverty can have long-lasting impacts across many domains.</p>
<p>Another perspective to fairness is to make the selection procedures equal the number of successes within each group <span class="citation">(<a href="#ref-Thorndike1971">Thorndike, 1971</a>)</span>.
According to this perspective, if you want to do selection, you should hire all people, then look at job performance.
If among successful employees, 60% are White and 40% are Black, then set this selection rate for each group (i.e., hiring 80% White individuals and 20% Black individuals is not okay).
According to this perspective, a selection system is only fair if the majority–minority differences on the selection device used are equal in magnitude to majority–minority differences in job performance.
Selection criteria should be made based on prior distributions of success rates.
However, you likely will not ever really know the true <a href="prediction.html#baseRate">base rate</a> in these situations.
No one uses this approach because you would have a period where you have to accept everyone to find the percent that works.
Also, this would only work in a narrow window of time because the selection pool changes over time.</p>
<p>There are lots of groups and subgroups.
Ensuring fairness is very complex, and there is no way to accomplish the goal of being equally fair to all people.
Therefore, do the best you can and try to minimize negative impact.</p>
</div>
</div>
<div id="correctForBias" class="section level2 hasAnchor" number="16.5">
<h2><span class="header-section-number">16.5</span> Correcting For Bias<a href="bias.html#correctForBias" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="whenDetectingBias" class="section level3 hasAnchor" number="16.5.1">
<h3><span class="header-section-number">16.5.1</span> What to Do When Detecting Bias<a href="bias.html#whenDetectingBias" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>When examining item bias (using <a href="bias.html#dif">differential item functioning</a>/DIF or <a href="bias.html#measurementInvariance">measurement non-invariance</a>) with many items (or measures) across many groups, there can be many tests, which will make it likely that <a href="bias.html#dif">DIF</a>/<a href="bias.html#measurementInvariance">non-invariance</a> will be detected, especially with a large sample.
Some detected <a href="bias.html#dif">DIF</a> may be artificial or trivial, but other <a href="bias.html#dif">DIF</a> may be real and important to address.
It is important to consider how you will proceed when detecting <a href="bias.html#dif">DIF</a>/<a href="bias.html#measurementInvariance">non-invariance</a>.
Considerations of effect size and theory can be important for evaluating the <a href="bias.html#dif">DIF</a>/<a href="bias.html#measurementInvariance">non-invariance</a> and whether it is negligible or important to address.</p>
<p>When detecting bias, there are several steps to take.
First, consider what the bias indicates.
Does the bias present <a href="bias.html#adverseImpact">adverse impact</a> for a minority group?
For what reasons might the bias exist?
Second, examine the effect size of the bias.
If the effects are small, if the bias does not present <a href="bias.html#adverseImpact">adverse impact</a> for a minority group, and if there is no compelling theoretical reason for the bias, the bias might not be sufficient to scrap the instrument for the population.
Some detected bias may be artificial, but other bias may be real.
Gender and cultural differences have shown a number of statistically significant effects for a number of different assessment purposes, but many of the observed effects are quite small and likely trivial, and they do not present compelling reasons to change the assessment <span class="citation">(<a href="#ref-Youngstrom2016">Youngstrom &amp; Van Meter, 2016</a>)</span>.</p>
<p>However, if you find bias, correct for it!
There are a number of <a href="bias.html#scoreAdjustment">score adjustment</a> and <a href="bias.html#otherBiasCorrections">non-score adjustment</a> approaches to correct for bias, as described in Sections <a href="bias.html#scoreAdjustment">16.5.2</a> and <a href="bias.html#otherBiasCorrections">16.5.3</a>.
If the bias occurs at the item level (e.g., <a href="bias.html#testStructureBias">test structure bias</a>), it is generally recommended to <a href="bias.html#removeBiasedItems">remove</a> or <a href="bias.html#resolveBiasedItems">resolve</a> items that show non-negligible bias.
There are three primary options: (1) drop the item for both groups, (2) drop the item for one group but keep it for the other group, or (3) freely estimate the parameters for the item across groups.
Addressing items that show larger bias can also reduce artificial bias in other items <span class="citation">(<a href="#ref-Hagquist2017">Hagquist &amp; Andrich, 2017</a>)</span>.
Thus, researchers are encouraged to handle item bias sequentially from high to low in magnitude.
If the bias occurs at the test score level (e.g., <a href="bias.html#predictiveBias">predictive bias</a>), <a href="bias.html#scoreAdjustment">score adjustments</a> may be considered.</p>
<p>If you do not correct for bias, consider the impact of the test, procedure, and selection procedure when interpreting scores.
Interpret scores with caution and provide necessary caveats in resulting papers or reports regarding the interpretations in question.
In sum, it is important to examine the possibility of bias—it is important to consider how much “erroneous junk” you are introducing into your research.</p>
</div>
<div id="scoreAdjustment" class="section level3 hasAnchor" number="16.5.2">
<h3><span class="header-section-number">16.5.2</span> Score Adjustment to Correct for Bias<a href="bias.html#scoreAdjustment" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Score adjustment involves adjusting scores for a particular group or groups.</p>
<div id="whyAdjustScores" class="section level4 hasAnchor" number="16.5.2.1">
<h4><span class="header-section-number">16.5.2.1</span> Why Adjust Scores?<a href="bias.html#whyAdjustScores" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>There may be several reasons to adjust scores for various groups in a given situation.
First, there may be social goals to adjust scores.
For example, we may want our selection device to yield personnel that better represent the nation or region, including diversity of genders, races, majors, social classes, etc.
Score adjustments are typically discussed with respect to racial minority differences due to historical and systemic inequities.
Our society aims to provide equal opportunity, including the opportunity to gain a fair share (i.e., proportional representation) of jobs.
A diversity of perspectives in a job is a strength; a diversity of perspectives can lead to greater creativity and improved problem-solving.
A second potential reason that we may want to apply score adjustment is to correct for bias.
A third potential reason that we may want to apply score adjustment is to improve the <a href="bias.html#fairness">fairness</a> of a test.</p>
</div>
<div id="typesOfScoreAdjustment" class="section level4 hasAnchor" number="16.5.2.2">
<h4><span class="header-section-number">16.5.2.2</span> Types of Score Adjustment<a href="bias.html#typesOfScoreAdjustment" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>There are a number of potential techniques that have been used in attempts to correct for bias, i.e., to reduce negative impact of the test on an under-represented group.
What is considered an under-represented group may depend on the context.
For instance, men are under-represented compared to women as nurses, preschool teachers, and college students.
However, men may not face the same systemic challenges compared to women, so even though men may show under-representation in some domains, it is arguable whether scores should be adjusted to increase their representation.
Techniques for score adjustment include:</p>
<ul>
<li><a href="bias.html#bonusPoints">Bonus points</a></li>
<li><a href="bias.html#withinGroupNorming">Within-group norming</a></li>
<li><a href="bias.html#separateCutoffs">Separate cutoffs</a></li>
<li><a href="bias.html#topDownSelection">Top-down selection from different lists</a></li>
<li><a href="bias.html#banding">Banding</a></li>
<li><a href="bias.html#bandingBonusPoints">Banding with bonus points</a></li>
<li><a href="bias.html#slidingBand">Sliding band</a></li>
<li><a href="bias.html#separateTests">Separate tests</a></li>
<li><a href="bias.html#itemElimination">Item elimination based on group differences</a></li>
</ul>
<div id="bonusPoints" class="section level5 hasAnchor" number="16.5.2.2.1">
<h5><span class="header-section-number">16.5.2.2.1</span> Bonus Points<a href="bias.html#bonusPoints" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Providing bonus points involves adding a constant number of points to the scores of all individuals who are members of a particular group with the goal of eliminating or reducing group differences.
Bonus points is used to correct for <a href="bias.html#predictiveBias">predictive bias</a> <a href="bias.html#differentIntercepts">differences in intercepts</a> between groups.
An example of bonus points is military veterans in placement for civil service jobs—points are added to the initial score for all veterans (e.g., add 5 points to test scores of all veterans).
An example of using bonus points as a score adjustment is depicted in Figure <a href="bias.html#fig:bonusPoints">16.12</a>.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:bonusPoints"></span>
<img src="Images/adjustment_bonusPoints.png" alt="Using Bonus Points as a Scoring Adjustment." width="100%" />
<p class="caption">
Figure 16.12: Using Bonus Points as a Scoring Adjustment.
</p>
</div>
<p>There are several pros of bonus points.
If the distribution of each group is the same, this will effectively reduce group differences.
Moreover, it is a simple way of impacting test selection and procedure without changing the test, which is therefore a great advantage.
There are several cons of bonus points.
If there are differences in group standard deviations, adding bonus points may not actually correct for bias.
The use of bonus points also obscures what is actually being done to scores, so other methods like using <a href="bias.html#separateCutoffs">separate cutoffs</a> may be more explicit.
In addition, the simplicity of bonus points is also a great disadvantage because it is easily understood and often not viewed as “fair” because some people are getting extra points that others do not.</p>
</div>
<div id="withinGroupNorming" class="section level5 hasAnchor" number="16.5.2.2.2">
<h5><span class="header-section-number">16.5.2.2.2</span> Within-Group Norming<a href="bias.html#withinGroupNorming" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>A <a href="scoresScales.html#norm">norm</a> is the standard of performance that a person’s performance can be compared to.
Within-group norming treats the person’s group in the sample as the <a href="scoresScales.html#norm">norm</a>.
Within-group norming converts an individual’s score to <a href="scoresScales.html#standardizedScores">standardized scores</a> (e.g., T scores) or percentiles within one’s own group.
Then, the people are selected based on the highest standard scores across groups.
Withing-group norming is used to correct for <a href="bias.html#predictiveBias">predictive bias</a> <a href="bias.html#differentSlopes">differences in slopes</a> between groups.
An example of using within-group norming as a score adjustment is depicted in Figure <a href="bias.html#fig:withinGroupNorming">16.13</a>.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:withinGroupNorming"></span>
<img src="Images/adjustment_withinGroupNorming.png" alt="Using Within-Group Norming as a Scoring Adjustment." width="100%" />
<p class="caption">
Figure 16.13: Using Within-Group Norming as a Scoring Adjustment.
</p>
</div>
<p>There are several pros of within-group norming.
First, it accounts for differences in group standard deviations and means, so it does not have the same problem as <a href="bias.html#bonusPoints">bonus points</a> and is generally more effective at eliminating <a href="bias.html#adverseImpact">adverse impact</a> compared to <a href="bias.html#bonusPoints">bonus points</a>.
Second, some general (non-group-specific) <a href="scoresScales.html#norm">norms</a> are clearly irrelevant for characterizing a person’s functioning.
Group-specific <a href="scoresScales.html#norm">norms</a> aim to describe a person’s performance relative to people with a similar background, thus potentially reducing cultural <a href="bias.html#bias">bias</a>.
Third, group-specific <a href="scoresScales.html#norm">norms</a> may better reflect cultural, educational, socioeconomic, and other factors that may influence a person’s score <span class="citation">(<a href="#ref-Burlew2019">Burlew et al., 2019</a>)</span>.
Fourth, group-specific <a href="scoresScales.html#norm">norms</a> may increase <a href="prediction.html#specificity">specificity</a>, and reduce over-pathologizing by preventing giving a diagnosis to people who might not show a condition <span class="citation">(<a href="#ref-Manly2007">Manly &amp; Echemendia, 2007</a>)</span>.</p>
<p>There are several cons of within-group norming.
First, group differences could be maintained if one decides to norm based on a reference sample or, when scores are skewed, a local sample, especially when using standardized scores.
However, percentile scores will consistently eliminate <a href="bias.html#adverseImpact">adverse impact</a>.
Second, using group-specific <a href="scoresScales.html#norm">norms</a> may obscure background variables that explain underlying reasons for group-related differences in test performance <span class="citation">(<a href="#ref-Manly2005">Manly, 2005</a>; <a href="#ref-Manly2007">Manly &amp; Echemendia, 2007</a>)</span>.
Third, group-specific <a href="scoresScales.html#norm">norms</a> do not address the problem if the measure shows test <a href="bias.html#bias">bias</a> <span class="citation">(<a href="#ref-Burlew2019">Burlew et al., 2019</a>)</span>.
Fourth, group-specific <a href="scoresScales.html#norm">norms</a> may reduce <a href="prediction.html#sensitivity">sensitivity</a> to detect conditions <span class="citation">(<a href="#ref-Manly2007">Manly &amp; Echemendia, 2007</a>)</span>.
For instance, they may prevent people from getting treatment who would benefit.
It is worth noting that within-group norming on the basis of sex, gender, and ethnicity is illegal for the basis of personnel selection according to the 1991 Civil Rights Act.</p>
<p>As an example of within-group norming, the National Football League used to use race-norming for identification of concussions.
The effect of race-norming, however, was that it lowered Black players’ concussion risk scores, which prevented many Black players from being identified as having sustained a concussion and from receiving needed treatment.
Race-norming compared the Black football players cognitive test scores to group-specific <a href="scoresScales.html#norm">norms</a>: the cognitive test scores of Black people in the general population (not to common <a href="scoresScales.html#norm">norms</a>).
Using Black-specific <a href="scoresScales.html#norm">norms</a> assumed that Black football players showed lower cognitive ability than other groups, so a low cognitive ability score for a Black player was less likely to be flagged as concerning.
Thus, the race-specific <a href="scoresScales.html#norm">norms</a> led to lower identified rates of concussions among Black football players compared to White football players.
Due to the <a href="bias.html#adverseImpact">adverse impact</a>, Black players sued the National Football League, and the league stopped the controversial practice of race-norming for identification of concussion (<a href="https://www.washingtonpost.com/sports/2021/06/03/nfl-concussion-settlement-race-norming/" class="uri">https://www.washingtonpost.com/sports/2021/06/03/nfl-concussion-settlement-race-norming/</a>; archived at <a href="https://perma.cc/KN3L-5Z7R" class="uri">https://perma.cc/KN3L-5Z7R</a>).</p>
<p>A common question is whether to use group-specific <a href="scoresScales.html#norm">norms</a> or common <a href="scoresScales.html#norm">norms</a>.
Group-specific <a href="scoresScales.html#norm">norms</a> are a controversial practice, and the answer depends.
If you are interested in a person’s absolute functioning (e.g., for determining whether someone is concussed or whether they are suitable to drive), recommendations are to use common <a href="scoresScales.html#norm">norms</a>, not group-specific <a href="scoresScales.html#norm">norms</a> <span class="citation">(<a href="#ref-Barrash2010">Barrash et al., 2010</a>; <a href="#ref-Silverberg2009">Silverberg &amp; Millis, 2009</a>)</span>.
If, by contrast, you are interested in a person’s <em>relative</em> functioning compared to a specific group, within-group norming could make sense if there is an appropriate reference group.
The question about which <a href="scoresScales.html#norm">norms</a> to use are complex, and psychologists should evaluate the cost and benefit of each <a href="scoresScales.html#norm">norm</a>, and use the <a href="scoresScales.html#norm">norm</a> with the greatest benefit and the least cost for the client <span class="citation">(<a href="#ref-Manly2007">Manly &amp; Echemendia, 2007</a>)</span>.</p>
</div>
<div id="separateCutoffs" class="section level5 hasAnchor" number="16.5.2.2.3">
<h5><span class="header-section-number">16.5.2.2.3</span> Separate Cutoffs<a href="bias.html#separateCutoffs" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Using separate cutoffs involves using a separate cutoff score per group and selecting the top number from each group.
That is, using separate cutoffs involves using different criteria for each group.
Using separate cutoffs functions the same as adding <a href="bias.html#bonusPoints">bonus points</a>, but it has greater transparency—i.e., you are lowering the standard for one group compared to another group.
An example of using separate cutoffs as a score adjustment is depicted in Figure <a href="bias.html#fig:separateCutoffs">16.14</a>.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:separateCutoffs"></span>
<img src="Images/adjustment_separateCutoffs.png" alt="Using Separate Cutoffs as a Scoring Adjustment. In this example, the cutoff for the majority group is 128; the cutoff for the minority group is 123." width="100%" />
<p class="caption">
Figure 16.14: Using Separate Cutoffs as a Scoring Adjustment. In this example, the cutoff for the majority group is 128; the cutoff for the minority group is 123.
</p>
</div>
</div>
<div id="topDownSelection" class="section level5 hasAnchor" number="16.5.2.2.4">
<h5><span class="header-section-number">16.5.2.2.4</span> Top-Down Selection from Different Lists<a href="bias.html#topDownSelection" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Top-down selection from different lists involves taking the best from two different lists according to a preset rule as to how many to select from each group.
Top-down selection from different lists functions the same as <a href="bias.html#withinGroupNorming">within-group norming</a>.
An example of using top-down selection from different lists as a score adjustment is depicted in Figure <a href="bias.html#fig:topDownSelectionDifferentLists">16.15</a>.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:topDownSelectionDifferentLists"></span>
<img src="Images/adjustment_separateLists.png" alt="Using Top-Down Selection From Different Lists as a Scoring Adjustment. In this example, the top three candidates are selected from each group." width="100%" />
<p class="caption">
Figure 16.15: Using Top-Down Selection From Different Lists as a Scoring Adjustment. In this example, the top three candidates are selected from each group.
</p>
</div>
</div>
<div id="banding" class="section level5 hasAnchor" number="16.5.2.2.5">
<h5><span class="header-section-number">16.5.2.2.5</span> Banding<a href="bias.html#banding" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Banding uses a tier system that is based on the assumption that individuals within a specific score range are regarded as having equivalent scores.
So that we do not over-estimate small score differences, scores within the same band are seen as equivalent—and the order of selection within the band can be modified depending on selection goals.
The <a href="reliability.html#standardErrorOfMeasurement">standard error of measurement (SEM)</a> is used to estimate the precision (<a href="reliability.html#reliability">reliability</a>) of the test scores, and it is used as the width of the band.</p>
<p>Consider an example: if a person received a score with confidence interval of 18–22, then scores between 18 to 22 are not necessarily different due to random fluctuation (<a href="reliability.html#measurementError">measurement error</a>).
Therefore, scores in that range are considered the same, and we take a band of scores.
However, banding by itself may not result in increased selection of lower scoring groups.
The band provides a subsample of applicants so that we can use other criteria (other than the test) to select a candidate.
Giving “minority preference” involves selecting members of minority group in a given band before selecting members of the majority group.
An example of using banding as a score adjustment is depicted in Figure <a href="bias.html#fig:banding">16.16</a>.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:banding"></span>
<img src="Images/adjustment_banding.png" alt="Using Banding as a Scoring Adjustment." width="100%" />
<p class="caption">
Figure 16.16: Using Banding as a Scoring Adjustment.
</p>
</div>
<p>The problem with banding is that bands are set by the <a href="reliability.html#standardErrorOfMeasurement">standard error of measurement</a>: you can select the first group from the first band, but then whom do you select after the first band?
There is no rationale where to “stop” the band because there are indistinguishable scores on the edges of each band to the next band.
That is, 17 is indistinguishable from 18 (in terms of its confidence interval), 16 is indistinguishable from 17, and so on.
Therefore, banding works okay for the top scores, but if you are going to hire a lot of candidates, it is a problem.
A solution to this problem with banding is to use a <a href="bias.html#slidingBand">sliding band</a>, as described later.</p>
</div>
<div id="bandingBonusPoints" class="section level5 hasAnchor" number="16.5.2.2.6">
<h5><span class="header-section-number">16.5.2.2.6</span> Banding with Bonus Points<a href="bias.html#bandingBonusPoints" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p><a href="bias.html#banding">Banding</a> is often used with <a href="bias.html#bonusPoints">bonus points</a> to reduce the negative impact for minority groups.
An example of using <a href="bias.html#banding">banding</a> with <a href="bias.html#bonusPoints">bonus points</a> as a score adjustment is depicted in Figure <a href="bias.html#fig:bandingBonusPoints">16.17</a>.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:bandingBonusPoints"></span>
<img src="Images/adjustment_bandingBonusPoints.png" alt="Using Banding With Bonus Points as a Scoring Adjustment." width="100%" />
<p class="caption">
Figure 16.17: Using Banding With Bonus Points as a Scoring Adjustment.
</p>
</div>
</div>
<div id="slidingBand" class="section level5 hasAnchor" number="16.5.2.2.7">
<h5><span class="header-section-number">16.5.2.2.7</span> Sliding Band<a href="bias.html#slidingBand" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Using a sliding band is a solution to the problem of which bands to use when using <a href="bias.html#banding">banding</a>.
Using a sliding band can help increase the number of minorities selected.
Using the top band, you select all members of a minority group in the top band, then select members of the majority group with the top score of the band, then slide the band down (based on <a href="reliability.html#standardErrorOfMeasurement">SEM</a>), and repeat.
You work your way down with bands though groups that are indistinguishable based on <a href="reliability.html#standardErrorOfMeasurement">SEM</a>, until getting a cell needed to select a relevant candidate.</p>
<p>For instance, if the top score is 22 and the <a href="reliability.html#standardErrorOfMeasurement">SEM</a> is 4 points, the first band would be: [18, 22].
Here is how you would proceed:</p>
<ol style="list-style-type: decimal">
<li>Select the minority group members who have a score between 18 to 22.</li>
<li>Select the majority group members who have a score of 22.</li>
<li>Slide the band down based on the SEM to the next highest score: [17, 21].</li>
<li>Select the minority group members who have a score between 17 to 21.</li>
<li>Select the majority group members who have a score of 21.</li>
<li>Slide the band down based on the SEM to the next highest score: [16, 20].</li>
<li>…</li>
<li>And so on</li>
</ol>
<p>An example of using a sliding band as a score adjustment is depicted in Figure <a href="bias.html#fig:slidingBand">16.18</a>.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:slidingBand"></span>
<img src="Images/adjustment_slidingBand.png" alt="Using a Sliding Band as a Scoring Adjustment." width="100%" />
<p class="caption">
Figure 16.18: Using a Sliding Band as a Scoring Adjustment.
</p>
</div>
<p>In sum, using a sliding band, scores that are not significantly lower than the highest remaining score should not be treated as different.
Using a sliding band has the same effects on decisions as <a href="bias.html#bonusPoints">bonus points</a> that are the width of the band.
For example, if the <a href="reliability.html#standardErrorOfMeasurement">SEM</a> is 3, it has the same decisions as <a href="bias.html#bonusPoints">bonus points</a> of 3; therefore, any scores within 3 of the highest score are now considered equal.</p>
<p>A sliding band is popular because of its scientific and statistical rationale.
Also, it is more confusing and, therefore, preferred by some because it may be less likely to be sued.
However, a sliding band may not always eliminate <a href="bias.html#adverseImpact">adverse impact</a>.
A sliding band has never been overturned in court (or at least, not yet).</p>
</div>
<div id="separateTests" class="section level5 hasAnchor" number="16.5.2.2.8">
<h5><span class="header-section-number">16.5.2.2.8</span> Separate Tests<a href="bias.html#separateTests" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Using separate tests for each group is another option to reduce bias.
For instance, you might use one test for the majority group and a different test for the minority group, making sure that each test is <a href="validity.html#validity">valid</a> for the relevant group.
Using separate tests is an extreme version of <a href="bias.html#topDownSelection">top-down selection</a> and <a href="bias.html#withinGroupNorming">within-group norming</a>.
Using separate tests would be an option if a measure shows <a href="bias.html#differentSlopes">different slopes</a> <a href="bias.html#predictiveBias">predictive bias</a>.</p>
<p>One way of developing separate tests is to use empirical keying by group: different items for each group are selected based on each item’s association with the criterion in each group.
Empirical keying is an example of <a href="validity.html#theoryEmpiricism">dustbowl empiricism</a> (i.e., relying on empiricism rather than theory).
However, theory can also inform the item selection.</p>
</div>
<div id="itemElimination" class="section level5 hasAnchor" number="16.5.2.2.9">
<h5><span class="header-section-number">16.5.2.2.9</span> Item Elimination based on Group Differences<a href="bias.html#itemElimination" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Items that show large group differences in scores can be eliminated from the test.
If you remove enough items showing differences between groups, you can get similar scores between groups and can get equal group selection.
A problem of item elimination based on group differences is that if you get rid of predictive items, then two goals, equal selection and predictive power, are not met.
If you use this method, you often have to be willing for the measure to show decreases in predictive power.</p>
</div>
</div>
<div id="useOfScoreAdjustment" class="section level4 hasAnchor" number="16.5.2.3">
<h4><span class="header-section-number">16.5.2.3</span> Use of Score Adjustment<a href="bias.html#useOfScoreAdjustment" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Score adjustment can be used in a number of different domains, including tests of aptitude and intelligence.
Score adjustment also comes up in other areas.
For example, the number of drinks it takes to be considered binge drinking differs between men (five) and women (four).
Although the list of score adjustment options is long, they all really reduce to two ways:</p>
<ol style="list-style-type: decimal">
<li><a href="bias.html#bonusPoints">Bonus points</a></li>
<li><a href="bias.html#withinGroupNorming">Within-group norming</a></li>
</ol>
<p><a href="bias.html#bonusPoints">Bonus points</a> and <a href="bias.html#withinGroupNorming">within-group norming</a> are the techniques that are most often used in the real world.
These techniques differ in their degree of obscurity—i.e., confusion that is caused not for scientific reasons, but for social, political, and dissemination and implementation reasons.
Often procedures that are hard to understand are preferred because it is hard to argue against, critique, or game the system.
Basically, you have two options for score adjustment.
One option is to adjust scores by raising scores in one group or lowering the criterion in one group.
The second primary option is to renorm or change the scores.
In sum, you can change the scores, or you can change the decisions you make based on the scores.</p>
</div>
</div>
<div id="otherBiasCorrections" class="section level3 hasAnchor" number="16.5.3">
<h3><span class="header-section-number">16.5.3</span> Other Ways to Correct for Bias<a href="bias.html#otherBiasCorrections" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Because score adjustment is controversial, it is also important to consider other potential ways to correct for bias that do not involve score adjustment.
Strategies other than score adjustment to correct for bias are described by <span class="citation">Sackett et al. (<a href="#ref-Sackett2001">2001</a>)</span>.</p>
<div id="useMultiplePredictors" class="section level4 hasAnchor" number="16.5.3.1">
<h4><span class="header-section-number">16.5.3.1</span> Use Multiple Predictors<a href="bias.html#useMultiplePredictors" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>In general, high-stakes decisions should not be made based on the results from one test.
So, for instance, do not make hiring decisions based just on aptitude assessments.
For example, college admissions decisions are not made just based on SAT scores, but also one’s grades, personal statement, extracurricular activities, letters of recommendation, etc.
Using multiple predictors works best when the predictors are not correlated with the assessment that has <a href="bias.html#adverseImpact">adverse impact</a>, which is difficult to achieve.</p>
<p>There are larger majority–minority subgroup differences in verbal and cognitive ability tests than in noncognitive skills (e.g., motivation, personality, and interpersonal skills).
So, it is important to include assessment of relevant noncognitive skills.
Include as many relevant aspects of the construct as possible for <a href="validity.html#contentValidity">content validity</a>.
For a job, consider as many factors as possible that are relevant for success, e.g., cognitive and noncognitive abilities.</p>
</div>
<div id="changeCriterion" class="section level4 hasAnchor" number="16.5.3.2">
<h4><span class="header-section-number">16.5.3.2</span> Change the Criterion<a href="bias.html#changeCriterion" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Another option is to change the criterion so that the <a href="validity.html#predictiveValidity">predictive validity</a> of tests is less skewed.
It may be that the selection instrument is not biased but the way in which we are thinking about selection procedures is biased.
For example, for judging the quality of universities, there are many different criteria we could use.
It could be valuable to examine the various criteria, and you might find what is driving adverse effects.</p>
</div>
<div id="removeBiasedItems" class="section level4 hasAnchor" number="16.5.3.3">
<h4><span class="header-section-number">16.5.3.3</span> Remove Biased Items<a href="bias.html#removeBiasedItems" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Using <a href="irt.html#irt">item response theory</a> or <a href="factor-analysis-PCA.html#cfa">confirmatory factor analysis</a>, you can identify items that function differently across groups (i.e., <a href="bias.html#dif">differential item functioning</a>/<a href="bias.html#dif">DIF</a> or <a href="bias.html#measurementInvariance">measurement non-invariance</a>).
For instance, you can identify items that show different <a href="irt.html#itemDiscrimination">discrimination</a>/factor loadings or <a href="irt.html#itemDifficulty">difficulty</a>/intercepts by group.
You do not just want to remove items that show mean-level differences in scores (or different rates of endorsement) for one group than another, because there may be true group differences in their level on particular items.
If an item is clearly <a href="validity.html#validity">invalid</a> in one group but <a href="validity.html#validity">valid</a> in another group, another option is to keep the item in one group, and to remove it in another group.</p>
<p>Be careful when removing items because removing items can lead to poorer <a href="validity.html#contentValidity">content validity</a>—i.e., items may no longer be a representative set of the content of the construct.
Removing items also reduces a measure’s <a href="reliability.html#reliability">reliability</a> and ability to detect individual differences <span class="citation">(<a href="#ref-Hagquist2019">Hagquist, 2019</a>; <a href="#ref-Hagquist2017">Hagquist &amp; Andrich, 2017</a>)</span>.
<a href="bias.html#dif">DIF</a> effects tend to be small and inconsistent; removing items showing <a href="bias.html#dif">DIF</a> may not have a big impact.</p>
</div>
<div id="resolveBiasedItems" class="section level4 hasAnchor" number="16.5.3.4">
<h4><span class="header-section-number">16.5.3.4</span> Resolve Biased Items<a href="bias.html#resolveBiasedItems" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Another option, for items identified that show <a href="bias.html#dif">differential item functioning</a> using <a href="irt.html#irt">IRT</a> or <a href="bias.html#measurementInvariance">measurement non-invariance</a> using <a href="factor-analysis-PCA.html#cfa">CFA</a>, is to resolve instead of <a href="bias.html#removeBiasedItems">remove</a> items.
Resolving items involves allowing an item to have a different <a href="irt.html#itemDiscrimination">discrimination</a>/factor loading and/or <a href="irt.html#itemDifficulty">difficulty</a>/intercept parameter for each group.
Allowing item parameters to differ across groups has a very small effect on <a href="reliability.html#reliability">reliability</a> and person separation, so it can be preferable to <a href="bias.html#removeBiasedItems">removing items</a> <span class="citation">(<a href="#ref-Hagquist2019">Hagquist, 2019</a>; <a href="#ref-Hagquist2017">Hagquist &amp; Andrich, 2017</a>)</span>.</p>
</div>
<div id="alternativeTestingModes" class="section level4 hasAnchor" number="16.5.3.5">
<h4><span class="header-section-number">16.5.3.5</span> Use Alternative Modes of Testing<a href="bias.html#alternativeTestingModes" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Another option is to use alternative modes of testing.
For example, you could use audio or video to present test items, rather than requiring a person to read the items, or write answers.
Typical testing and computerized exams are oriented toward the upper-middle class, which is therefore a procedure problem!
McClelland’s <span class="citation">(<a href="#ref-McClelland1973">1973</a>)</span> argument is that we need more real-life testing.
Real-life testing could help address stereotype threats and the effects of learning disabilities.
However, testing in different modalities could change the construct(s) being assessed.</p>
</div>
<div id="workRecords" class="section level4 hasAnchor" number="16.5.3.6">
<h4><span class="header-section-number">16.5.3.6</span> Use Work Records<a href="bias.html#workRecords" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Using work records is based on McClelland’s <span class="citation">(<a href="#ref-McClelland1973">1973</a>)</span> argument to use more realistic and authentic assessments of job-relevant abilities.
Evidence on the value of work records for personnel selection is mixed.
In some cases, use of work records can actually increase <a href="bias.html#adverseImpact">adverse impact</a> on under-represented groups because the primary group typically already has an idea of how to get into the relevant job or is already in the relevant job; therefore, they have a leg up.
It would be acceptable to use work records if you trained people first and then tested, but no one spends the time to do this.</p>
</div>
<div id="increaseTimeLimit" class="section level4 hasAnchor" number="16.5.3.7">
<h4><span class="header-section-number">16.5.3.7</span> Increase Time Limit<a href="bias.html#increaseTimeLimit" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Another option is to allot people more testing time, as long as doing so does not change the construct.
Time limits often lead to greater <a href="reliability.html#measurementError">measurement error</a> because scores conflate pace and quality of work.
Increasing time limits requires convincing stakeholders that job performance is typically not “how fast you do things” but “how well you do them”—i.e., that time does not correlate with outcome of interest.
The utility of increasing time limits depends on the domain.
In some domains, efficiency is crucial (e.g., medicine, pilot).
Increasing time limits is not that effective in reducing group differences, and it may actually increase group differences.</p>
</div>
<div id="motivationSets" class="section level4 hasAnchor" number="16.5.3.8">
<h4><span class="header-section-number">16.5.3.8</span> Use Motivation Sets<a href="bias.html#motivationSets" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Using motivation sets involves finding ways to increase testing motivation for minority groups.
It is probably an error to think that a test assesses just aptitude; therefore, we should also consider an individual’s motivation to test.
Thus, part of the score has to do with ability and some of the score has to do with motivation.
You should try to maximize each examinee’s motivation, so that the person’s score on the measure better captures their true ability score.
Motivation sets could include, for example, using more realistic test stimuli that are clearly applicable to the school or job requirements (i.e., that have <a href="validity.html#faceValidity">face validity</a>) to motivate all test takers.</p>
</div>
<div id="instructionalSets" class="section level4 hasAnchor" number="16.5.3.9">
<h4><span class="header-section-number">16.5.3.9</span> Use Instructional Sets<a href="bias.html#instructionalSets" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Using instructional sets involves coaching and training.
For instance, you could inform examinees about the test content, provide study materials, and recommend test-taking strategies.
This could narrow the gap between groups because there is an implicit assumption that the primary group already has “light” training.
Using instructional sets aims to reduce error variance due to test anxiety, unfamiliar test format, and poor test-taking skills.</p>
<p>Giving minority groups better access to test preparation is based on the assumption that group differences emerge because of different access to test preparation materials.
This could theoretically help to systematically reduce test score differences across groups.
Standardized tests like the SAT/GRE/LSAT/GMAT/MCAT, etc. embrace coaching/training.
For instance, the organization ETS gives training materials for free.
After training, scores on standardized tests show some but minimal improvement.
In general, training yields some improvement on quantitative subscales but minimal change on verbal subscales.
However, the improvements tend to apply across groups, and they do not seem to lessen group differences in scores.</p>
</div>
</div>
</div>
<div id="gettingStarted-bias" class="section level2 hasAnchor" number="16.6">
<h2><span class="header-section-number">16.6</span> Getting Started<a href="bias.html#gettingStarted-bias" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="loadLibraries-bias" class="section level3 hasAnchor" number="16.6.1">
<h3><span class="header-section-number">16.6.1</span> Load Libraries<a href="bias.html#loadLibraries-bias" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1483"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1483-1"><a href="bias.html#cb1483-1" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;petersenlab&quot;</span>) <span class="co">#to install: install.packages(&quot;remotes&quot;); remotes::install_github(&quot;DevPsyLab/petersenlab&quot;)</span></span>
<span id="cb1483-2"><a href="bias.html#cb1483-2" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;lavaan&quot;</span>)</span>
<span id="cb1483-3"><a href="bias.html#cb1483-3" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;semTools&quot;</span>)</span>
<span id="cb1483-4"><a href="bias.html#cb1483-4" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;semPlot&quot;</span>)</span>
<span id="cb1483-5"><a href="bias.html#cb1483-5" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;mirt&quot;</span>)</span>
<span id="cb1483-6"><a href="bias.html#cb1483-6" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;dmacs&quot;</span>) <span class="co">#to install: install.packages(&quot;remotes&quot;); remotes::install_github(&quot;ddueber/dmacs&quot;)</span></span>
<span id="cb1483-7"><a href="bias.html#cb1483-7" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;strucchange&quot;</span>)</span>
<span id="cb1483-8"><a href="bias.html#cb1483-8" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;MOTE&quot;</span>)</span>
<span id="cb1483-9"><a href="bias.html#cb1483-9" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;tidyverse&quot;</span>)</span>
<span id="cb1483-10"><a href="bias.html#cb1483-10" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;here&quot;</span>)</span>
<span id="cb1483-11"><a href="bias.html#cb1483-11" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;tinytex&quot;</span>)</span></code></pre></div>
</details>
</div>
<div id="prepareData-bias" class="section level3 hasAnchor" number="16.6.2">
<h3><span class="header-section-number">16.6.2</span> Prepare Data<a href="bias.html#prepareData-bias" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="loadData-bias" class="section level4 hasAnchor" number="16.6.2.1">
<h4><span class="header-section-number">16.6.2.1</span> Load Data<a href="bias.html#loadData-bias" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><code>cnlsy</code> is a subset of a data set from the Children of the National Longitudinal Survey of Youth Survey (CNLSY).
The CNLSY is a publicly available longitudinal data set provided by the Bureau of Labor Statistics (<a href="https://perma.cc/EH38-HDRN" class="uri">https://perma.cc/EH38-HDRN</a>).
The CNLSY data file for these examples is located on the book’s page of the Open Science Framework (<a href="https://osf.io/3pwza" class="uri">https://osf.io/3pwza</a>).</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1484"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1484-1"><a href="bias.html#cb1484-1" tabindex="-1"></a>cnlsy <span class="ot">&lt;-</span> <span class="fu">read_csv</span>(<span class="fu">here</span>(<span class="st">&quot;Data&quot;</span>, <span class="st">&quot;cnlsy.csv&quot;</span>))</span></code></pre></div>
</details>
</div>
<div id="simulateData-bias" class="section level4 hasAnchor" number="16.6.2.2">
<h4><span class="header-section-number">16.6.2.2</span> Simulate Data<a href="bias.html#simulateData-bias" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>For reproducibility, I set the seed below.
Using the same seed will yield the same answer every time.
There is nothing special about this particular seed.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1485"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1485-1"><a href="bias.html#cb1485-1" tabindex="-1"></a>sampleSize <span class="ot">&lt;-</span> <span class="dv">4000</span></span>
<span id="cb1485-2"><a href="bias.html#cb1485-2" tabindex="-1"></a></span>
<span id="cb1485-3"><a href="bias.html#cb1485-3" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">52242</span>)</span>
<span id="cb1485-4"><a href="bias.html#cb1485-4" tabindex="-1"></a></span>
<span id="cb1485-5"><a href="bias.html#cb1485-5" tabindex="-1"></a>mydataBias <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb1485-6"><a href="bias.html#cb1485-6" tabindex="-1"></a>  <span class="at">ID =</span> <span class="dv">1</span><span class="sc">:</span>sampleSize,</span>
<span id="cb1485-7"><a href="bias.html#cb1485-7" tabindex="-1"></a>  <span class="at">group =</span> <span class="fu">factor</span>(<span class="fu">c</span>(<span class="st">&quot;male&quot;</span>,<span class="st">&quot;female&quot;</span>),</span>
<span id="cb1485-8"><a href="bias.html#cb1485-8" tabindex="-1"></a>                 <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">&quot;male&quot;</span>,<span class="st">&quot;female&quot;</span>)),</span>
<span id="cb1485-9"><a href="bias.html#cb1485-9" tabindex="-1"></a>  <span class="at">unbiasedPredictor1 =</span> <span class="cn">NA</span>,</span>
<span id="cb1485-10"><a href="bias.html#cb1485-10" tabindex="-1"></a>  <span class="at">unbiasedPredictor2 =</span> <span class="cn">NA</span>,</span>
<span id="cb1485-11"><a href="bias.html#cb1485-11" tabindex="-1"></a>  <span class="at">unbiasedPredictor3 =</span> <span class="cn">NA</span>,</span>
<span id="cb1485-12"><a href="bias.html#cb1485-12" tabindex="-1"></a>  <span class="at">unbiasedCriterion1 =</span> <span class="cn">NA</span>,</span>
<span id="cb1485-13"><a href="bias.html#cb1485-13" tabindex="-1"></a>  <span class="at">unbiasedCriterion2 =</span> <span class="cn">NA</span>,</span>
<span id="cb1485-14"><a href="bias.html#cb1485-14" tabindex="-1"></a>  <span class="at">unbiasedCriterion3 =</span> <span class="cn">NA</span>,</span>
<span id="cb1485-15"><a href="bias.html#cb1485-15" tabindex="-1"></a>  <span class="at">predictor =</span> <span class="fu">rnorm</span>(sampleSize, <span class="at">mean =</span> <span class="dv">100</span>, <span class="at">sd =</span> <span class="dv">15</span>),</span>
<span id="cb1485-16"><a href="bias.html#cb1485-16" tabindex="-1"></a>  <span class="at">criterion1 =</span> <span class="cn">NA</span>,</span>
<span id="cb1485-17"><a href="bias.html#cb1485-17" tabindex="-1"></a>  <span class="at">criterion2 =</span> <span class="cn">NA</span>,</span>
<span id="cb1485-18"><a href="bias.html#cb1485-18" tabindex="-1"></a>  <span class="at">criterion3 =</span> <span class="cn">NA</span>,</span>
<span id="cb1485-19"><a href="bias.html#cb1485-19" tabindex="-1"></a>  <span class="at">criterion4 =</span> <span class="cn">NA</span>,</span>
<span id="cb1485-20"><a href="bias.html#cb1485-20" tabindex="-1"></a>  <span class="at">criterion5 =</span> <span class="cn">NA</span>)</span>
<span id="cb1485-21"><a href="bias.html#cb1485-21" tabindex="-1"></a></span>
<span id="cb1485-22"><a href="bias.html#cb1485-22" tabindex="-1"></a>mydataBias<span class="sc">$</span>unbiasedPredictor1 <span class="ot">&lt;-</span> <span class="fu">rnorm</span>(sampleSize, <span class="at">mean =</span> <span class="dv">100</span>, <span class="at">sd =</span> <span class="dv">15</span>)</span>
<span id="cb1485-23"><a href="bias.html#cb1485-23" tabindex="-1"></a>mydataBias<span class="sc">$</span>unbiasedPredictor2[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)] <span class="ot">&lt;-</span> </span>
<span id="cb1485-24"><a href="bias.html#cb1485-24" tabindex="-1"></a>  <span class="fu">rnorm</span>(<span class="fu">length</span>(<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)), <span class="at">mean =</span> <span class="dv">70</span>, <span class="at">sd =</span> <span class="dv">15</span>)</span>
<span id="cb1485-25"><a href="bias.html#cb1485-25" tabindex="-1"></a>mydataBias<span class="sc">$</span>unbiasedPredictor2[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)] <span class="ot">&lt;-</span> </span>
<span id="cb1485-26"><a href="bias.html#cb1485-26" tabindex="-1"></a>  <span class="fu">rnorm</span>(<span class="fu">length</span>(<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)), <span class="at">mean =</span> <span class="dv">130</span>, <span class="at">sd =</span> <span class="dv">15</span>)</span>
<span id="cb1485-27"><a href="bias.html#cb1485-27" tabindex="-1"></a>mydataBias<span class="sc">$</span>unbiasedPredictor3[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)] <span class="ot">&lt;-</span> </span>
<span id="cb1485-28"><a href="bias.html#cb1485-28" tabindex="-1"></a>  <span class="fu">rnorm</span>(<span class="fu">length</span>(<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)), <span class="at">mean =</span> <span class="dv">130</span>, <span class="at">sd =</span> <span class="dv">15</span>)</span>
<span id="cb1485-29"><a href="bias.html#cb1485-29" tabindex="-1"></a>mydataBias<span class="sc">$</span>unbiasedPredictor3[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)] <span class="ot">&lt;-</span> </span>
<span id="cb1485-30"><a href="bias.html#cb1485-30" tabindex="-1"></a>  <span class="fu">rnorm</span>(<span class="fu">length</span>(<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)), <span class="at">mean =</span> <span class="dv">70</span>, <span class="at">sd =</span> <span class="dv">15</span>)</span>
<span id="cb1485-31"><a href="bias.html#cb1485-31" tabindex="-1"></a></span>
<span id="cb1485-32"><a href="bias.html#cb1485-32" tabindex="-1"></a>mydataBias<span class="sc">$</span>unbiasedCriterion1 <span class="ot">&lt;-</span> <span class="dv">1</span> <span class="sc">*</span> mydataBias<span class="sc">$</span>unbiasedPredictor1 <span class="sc">+</span> </span>
<span id="cb1485-33"><a href="bias.html#cb1485-33" tabindex="-1"></a>  <span class="fu">rnorm</span>(sampleSize, <span class="at">mean =</span> <span class="dv">0</span>, <span class="at">sd =</span> <span class="dv">15</span>)</span>
<span id="cb1485-34"><a href="bias.html#cb1485-34" tabindex="-1"></a>mydataBias<span class="sc">$</span>unbiasedCriterion2 <span class="ot">&lt;-</span> <span class="dv">1</span> <span class="sc">*</span> mydataBias<span class="sc">$</span>unbiasedPredictor2 <span class="sc">+</span> </span>
<span id="cb1485-35"><a href="bias.html#cb1485-35" tabindex="-1"></a>  <span class="fu">rnorm</span>(sampleSize, <span class="at">mean =</span> <span class="dv">0</span>, <span class="at">sd =</span> <span class="dv">15</span>)</span>
<span id="cb1485-36"><a href="bias.html#cb1485-36" tabindex="-1"></a>mydataBias<span class="sc">$</span>unbiasedCriterion3 <span class="ot">&lt;-</span> <span class="dv">1</span> <span class="sc">*</span> mydataBias<span class="sc">$</span>unbiasedPredictor3 <span class="sc">+</span> </span>
<span id="cb1485-37"><a href="bias.html#cb1485-37" tabindex="-1"></a>  <span class="fu">rnorm</span>(sampleSize, <span class="at">mean =</span> <span class="dv">0</span>, <span class="at">sd =</span> <span class="dv">15</span>)</span>
<span id="cb1485-38"><a href="bias.html#cb1485-38" tabindex="-1"></a></span>
<span id="cb1485-39"><a href="bias.html#cb1485-39" tabindex="-1"></a>mydataBias<span class="sc">$</span>criterion1[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)] <span class="ot">&lt;-</span> </span>
<span id="cb1485-40"><a href="bias.html#cb1485-40" tabindex="-1"></a>  .<span class="dv">7</span> <span class="sc">*</span> mydataBias<span class="sc">$</span>predictor[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)] <span class="sc">+</span> </span>
<span id="cb1485-41"><a href="bias.html#cb1485-41" tabindex="-1"></a>  <span class="fu">rnorm</span>(<span class="fu">length</span>(<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)), <span class="at">mean =</span> <span class="dv">0</span>, <span class="at">sd =</span> <span class="dv">5</span>)</span>
<span id="cb1485-42"><a href="bias.html#cb1485-42" tabindex="-1"></a>mydataBias<span class="sc">$</span>criterion1[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)] <span class="ot">&lt;-</span> </span>
<span id="cb1485-43"><a href="bias.html#cb1485-43" tabindex="-1"></a>  .<span class="dv">7</span> <span class="sc">*</span> mydataBias<span class="sc">$</span>predictor[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)] <span class="sc">+</span> </span>
<span id="cb1485-44"><a href="bias.html#cb1485-44" tabindex="-1"></a>  <span class="fu">rnorm</span>(<span class="fu">length</span>(<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)), <span class="at">mean =</span> <span class="dv">0</span>, <span class="at">sd =</span> <span class="dv">5</span>)</span>
<span id="cb1485-45"><a href="bias.html#cb1485-45" tabindex="-1"></a>mydataBias<span class="sc">$</span>criterion2[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)] <span class="ot">&lt;-</span> </span>
<span id="cb1485-46"><a href="bias.html#cb1485-46" tabindex="-1"></a>  .<span class="dv">7</span> <span class="sc">*</span> mydataBias<span class="sc">$</span>predictor[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)] <span class="sc">+</span> </span>
<span id="cb1485-47"><a href="bias.html#cb1485-47" tabindex="-1"></a>  <span class="fu">rnorm</span>(<span class="fu">length</span>(<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)), <span class="at">mean =</span> <span class="dv">10</span>, <span class="at">sd =</span> <span class="dv">5</span>)</span>
<span id="cb1485-48"><a href="bias.html#cb1485-48" tabindex="-1"></a>mydataBias<span class="sc">$</span>criterion2[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)] <span class="ot">&lt;-</span> </span>
<span id="cb1485-49"><a href="bias.html#cb1485-49" tabindex="-1"></a>  .<span class="dv">7</span> <span class="sc">*</span> mydataBias<span class="sc">$</span>predictor[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)] <span class="sc">+</span> </span>
<span id="cb1485-50"><a href="bias.html#cb1485-50" tabindex="-1"></a>  <span class="fu">rnorm</span>(<span class="fu">length</span>(<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)), <span class="at">mean =</span> <span class="dv">0</span>, <span class="at">sd =</span> <span class="dv">5</span>)</span>
<span id="cb1485-51"><a href="bias.html#cb1485-51" tabindex="-1"></a>mydataBias<span class="sc">$</span>criterion3[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)] <span class="ot">&lt;-</span> </span>
<span id="cb1485-52"><a href="bias.html#cb1485-52" tabindex="-1"></a>  .<span class="dv">7</span> <span class="sc">*</span> mydataBias<span class="sc">$</span>predictor[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)] <span class="sc">+</span> </span>
<span id="cb1485-53"><a href="bias.html#cb1485-53" tabindex="-1"></a>  <span class="fu">rnorm</span>(<span class="fu">length</span>(<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)), <span class="at">mean =</span> <span class="dv">0</span>, <span class="at">sd =</span> <span class="dv">5</span>)</span>
<span id="cb1485-54"><a href="bias.html#cb1485-54" tabindex="-1"></a>mydataBias<span class="sc">$</span>criterion3[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)] <span class="ot">&lt;-</span> </span>
<span id="cb1485-55"><a href="bias.html#cb1485-55" tabindex="-1"></a>  .<span class="dv">3</span> <span class="sc">*</span> mydataBias<span class="sc">$</span>predictor[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)] <span class="sc">+</span> </span>
<span id="cb1485-56"><a href="bias.html#cb1485-56" tabindex="-1"></a>  <span class="fu">rnorm</span>(<span class="fu">length</span>(<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)), <span class="at">mean =</span> <span class="dv">0</span>, <span class="at">sd =</span> <span class="dv">5</span>)</span>
<span id="cb1485-57"><a href="bias.html#cb1485-57" tabindex="-1"></a>mydataBias<span class="sc">$</span>criterion4[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)] <span class="ot">&lt;-</span> </span>
<span id="cb1485-58"><a href="bias.html#cb1485-58" tabindex="-1"></a>  .<span class="dv">7</span> <span class="sc">*</span> mydataBias<span class="sc">$</span>predictor[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)] <span class="sc">+</span> </span>
<span id="cb1485-59"><a href="bias.html#cb1485-59" tabindex="-1"></a>  <span class="fu">rnorm</span>(<span class="fu">length</span>(<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)), <span class="at">mean =</span> <span class="dv">0</span>, <span class="at">sd =</span> <span class="dv">5</span>)</span>
<span id="cb1485-60"><a href="bias.html#cb1485-60" tabindex="-1"></a>mydataBias<span class="sc">$</span>criterion4[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)] <span class="ot">&lt;-</span> </span>
<span id="cb1485-61"><a href="bias.html#cb1485-61" tabindex="-1"></a>  .<span class="dv">3</span> <span class="sc">*</span> mydataBias<span class="sc">$</span>predictor[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)] <span class="sc">+</span> </span>
<span id="cb1485-62"><a href="bias.html#cb1485-62" tabindex="-1"></a>  <span class="fu">rnorm</span>(<span class="fu">length</span>(<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)), <span class="at">mean =</span> <span class="dv">30</span>, <span class="at">sd =</span> <span class="dv">5</span>)</span>
<span id="cb1485-63"><a href="bias.html#cb1485-63" tabindex="-1"></a>mydataBias<span class="sc">$</span>criterion5[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)] <span class="ot">&lt;-</span> </span>
<span id="cb1485-64"><a href="bias.html#cb1485-64" tabindex="-1"></a>  .<span class="dv">7</span> <span class="sc">*</span> mydataBias<span class="sc">$</span>predictor[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)] <span class="sc">+</span> </span>
<span id="cb1485-65"><a href="bias.html#cb1485-65" tabindex="-1"></a>  <span class="fu">rnorm</span>(<span class="fu">length</span>(<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)), <span class="at">mean =</span> <span class="dv">0</span>, <span class="at">sd =</span> <span class="dv">30</span>)</span>
<span id="cb1485-66"><a href="bias.html#cb1485-66" tabindex="-1"></a>mydataBias<span class="sc">$</span>criterion5[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)] <span class="ot">&lt;-</span> </span>
<span id="cb1485-67"><a href="bias.html#cb1485-67" tabindex="-1"></a>  .<span class="dv">7</span> <span class="sc">*</span> mydataBias<span class="sc">$</span>predictor[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)] <span class="sc">+</span> </span>
<span id="cb1485-68"><a href="bias.html#cb1485-68" tabindex="-1"></a>  <span class="fu">rnorm</span>(<span class="fu">length</span>(<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)), <span class="at">mean =</span> <span class="dv">0</span>, <span class="at">sd =</span> <span class="dv">5</span>)</span></code></pre></div>
</details>
</div>
<div id="addMissingData-bias" class="section level4 hasAnchor" number="16.6.2.3">
<h4><span class="header-section-number">16.6.2.3</span> Add Missing Data<a href="bias.html#addMissingData-bias" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Adding missing data to dataframes helps make examples more realistic to real-life data and helps you get in the habit of programming to account for missing data.</p>
<p><code>HolzingerSwineford1939</code> is a data set from the <code>lavaan</code> package <span class="citation">(<a href="#ref-R-lavaan">Rosseel et al., 2022</a>)</span> that contains mental ability test scores (<code>x1</code>–<code>x9</code>) for seventh- and eighth-grade children.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1486"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1486-1"><a href="bias.html#cb1486-1" tabindex="-1"></a>varNames <span class="ot">&lt;-</span> <span class="fu">names</span>(mydataBias)</span>
<span id="cb1486-2"><a href="bias.html#cb1486-2" tabindex="-1"></a>dimensionsDf <span class="ot">&lt;-</span> <span class="fu">dim</span>(mydataBias[,<span class="sc">-</span><span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">2</span>)])</span>
<span id="cb1486-3"><a href="bias.html#cb1486-3" tabindex="-1"></a>unlistedDf <span class="ot">&lt;-</span> <span class="fu">unlist</span>(mydataBias[,<span class="sc">-</span><span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">2</span>)])</span>
<span id="cb1486-4"><a href="bias.html#cb1486-4" tabindex="-1"></a>unlistedDf[<span class="fu">sample</span>(</span>
<span id="cb1486-5"><a href="bias.html#cb1486-5" tabindex="-1"></a>  <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(unlistedDf),</span>
<span id="cb1486-6"><a href="bias.html#cb1486-6" tabindex="-1"></a>  <span class="at">size =</span> .<span class="dv">01</span> <span class="sc">*</span> <span class="fu">length</span>(unlistedDf))] <span class="ot">&lt;-</span> <span class="cn">NA</span></span>
<span id="cb1486-7"><a href="bias.html#cb1486-7" tabindex="-1"></a>mydataBias <span class="ot">&lt;-</span> <span class="fu">cbind</span>(</span>
<span id="cb1486-8"><a href="bias.html#cb1486-8" tabindex="-1"></a>  mydataBias[,<span class="fu">c</span>(<span class="st">&quot;ID&quot;</span>,<span class="st">&quot;group&quot;</span>)],</span>
<span id="cb1486-9"><a href="bias.html#cb1486-9" tabindex="-1"></a>  <span class="fu">as.data.frame</span>(</span>
<span id="cb1486-10"><a href="bias.html#cb1486-10" tabindex="-1"></a>    <span class="fu">matrix</span>(</span>
<span id="cb1486-11"><a href="bias.html#cb1486-11" tabindex="-1"></a>      unlistedDf,</span>
<span id="cb1486-12"><a href="bias.html#cb1486-12" tabindex="-1"></a>      <span class="at">ncol =</span> dimensionsDf[<span class="dv">2</span>])))</span>
<span id="cb1486-13"><a href="bias.html#cb1486-13" tabindex="-1"></a><span class="fu">names</span>(mydataBias) <span class="ot">&lt;-</span> varNames</span>
<span id="cb1486-14"><a href="bias.html#cb1486-14" tabindex="-1"></a></span>
<span id="cb1486-15"><a href="bias.html#cb1486-15" tabindex="-1"></a><span class="fu">data</span>(<span class="st">&quot;HolzingerSwineford1939&quot;</span>)</span>
<span id="cb1486-16"><a href="bias.html#cb1486-16" tabindex="-1"></a>varNames <span class="ot">&lt;-</span> <span class="fu">names</span>(HolzingerSwineford1939)</span>
<span id="cb1486-17"><a href="bias.html#cb1486-17" tabindex="-1"></a>dimensionsDf <span class="ot">&lt;-</span> <span class="fu">dim</span>(HolzingerSwineford1939[,<span class="fu">paste</span>(<span class="st">&quot;x&quot;</span>, <span class="dv">1</span><span class="sc">:</span><span class="dv">9</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)])</span>
<span id="cb1486-18"><a href="bias.html#cb1486-18" tabindex="-1"></a>unlistedDf <span class="ot">&lt;-</span> <span class="fu">unlist</span>(HolzingerSwineford1939[,<span class="fu">paste</span>(<span class="st">&quot;x&quot;</span>, <span class="dv">1</span><span class="sc">:</span><span class="dv">9</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)])</span>
<span id="cb1486-19"><a href="bias.html#cb1486-19" tabindex="-1"></a>unlistedDf[<span class="fu">sample</span>(</span>
<span id="cb1486-20"><a href="bias.html#cb1486-20" tabindex="-1"></a>  <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(unlistedDf),</span>
<span id="cb1486-21"><a href="bias.html#cb1486-21" tabindex="-1"></a>  <span class="at">size =</span> .<span class="dv">01</span> <span class="sc">*</span> <span class="fu">length</span>(unlistedDf))] <span class="ot">&lt;-</span> <span class="cn">NA</span></span>
<span id="cb1486-22"><a href="bias.html#cb1486-22" tabindex="-1"></a>HolzingerSwineford1939 <span class="ot">&lt;-</span> <span class="fu">cbind</span>(</span>
<span id="cb1486-23"><a href="bias.html#cb1486-23" tabindex="-1"></a>  HolzingerSwineford1939[,<span class="dv">1</span><span class="sc">:</span><span class="dv">6</span>],</span>
<span id="cb1486-24"><a href="bias.html#cb1486-24" tabindex="-1"></a>  <span class="fu">as.data.frame</span>(<span class="fu">matrix</span>(</span>
<span id="cb1486-25"><a href="bias.html#cb1486-25" tabindex="-1"></a>    unlistedDf,</span>
<span id="cb1486-26"><a href="bias.html#cb1486-26" tabindex="-1"></a>    <span class="at">ncol =</span> dimensionsDf[<span class="dv">2</span>])))</span>
<span id="cb1486-27"><a href="bias.html#cb1486-27" tabindex="-1"></a><span class="fu">names</span>(HolzingerSwineford1939) <span class="ot">&lt;-</span> varNames</span></code></pre></div>
</details>
</div>
</div>
</div>
<div id="examplesUnbiasedTests" class="section level2 hasAnchor" number="16.7">
<h2><span class="header-section-number">16.7</span> Examples of Unbiased Tests (in Terms of Predictive Bias)<a href="bias.html#examplesUnbiasedTests" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="unbiasedTestsEqualMeans" class="section level3 hasAnchor" number="16.7.1">
<h3><span class="header-section-number">16.7.1</span> Unbiased test where males and females have equal means on predictor and criterion<a href="bias.html#unbiasedTestsEqualMeans" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Figure <a href="bias.html#fig:unbiasedEqualMeans">16.19</a> depicts an example of an unbiased test where males and females have equal means on the predictor and criterion.
The test is unbiased because there are no significant differences in the regression lines (of <code>predictor</code> predicting <code>criterion</code>) between males and females.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1487"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1487-1"><a href="bias.html#cb1487-1" tabindex="-1"></a><span class="fu">summary</span>(<span class="fu">lm</span>(</span>
<span id="cb1487-2"><a href="bias.html#cb1487-2" tabindex="-1"></a>  unbiasedCriterion1 <span class="sc">~</span> unbiasedPredictor1 <span class="sc">+</span> group <span class="sc">+</span> unbiasedPredictor1<span class="sc">:</span>group,</span>
<span id="cb1487-3"><a href="bias.html#cb1487-3" tabindex="-1"></a>  <span class="at">data =</span> mydataBias))</span></code></pre></div>
</details>
<pre><code>
Call:
lm(formula = unbiasedCriterion1 ~ unbiasedPredictor1 + group + 
    unbiasedPredictor1:group, data = mydataBias)

Residuals:
    Min      1Q  Median      3Q     Max 
-54.623 -10.050  -0.025  10.373  66.811 

Coefficients:
                                Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)                     0.930697   2.295058   0.406    0.685    
unbiasedPredictor1              0.996976   0.022624  44.066   &lt;2e-16 ***
groupfemale                    -1.165200   3.250481  -0.358    0.720    
unbiasedPredictor1:groupfemale -0.004397   0.032115  -0.137    0.891    
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 15.13 on 3913 degrees of freedom
  (83 observations deleted due to missingness)
Multiple R-squared:  0.4963,    Adjusted R-squared:  0.496 
F-statistic:  1285 on 3 and 3913 DF,  p-value: &lt; 2.2e-16</code></pre>
<details class=chunk-details><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1489"><pre class="sourceCode r fold-hide"><code class="sourceCode r"><span id="cb1489-1"><a href="bias.html#cb1489-1" tabindex="-1"></a><span class="fu">plot</span>(</span>
<span id="cb1489-2"><a href="bias.html#cb1489-2" tabindex="-1"></a>  unbiasedCriterion1 <span class="sc">~</span> unbiasedPredictor1,</span>
<span id="cb1489-3"><a href="bias.html#cb1489-3" tabindex="-1"></a>  <span class="at">data =</span> mydataBias,</span>
<span id="cb1489-4"><a href="bias.html#cb1489-4" tabindex="-1"></a>  <span class="at">xlim =</span> <span class="fu">c</span>(</span>
<span id="cb1489-5"><a href="bias.html#cb1489-5" tabindex="-1"></a>    <span class="dv">0</span>, </span>
<span id="cb1489-6"><a href="bias.html#cb1489-6" tabindex="-1"></a>    <span class="fu">max</span>(<span class="fu">c</span>(</span>
<span id="cb1489-7"><a href="bias.html#cb1489-7" tabindex="-1"></a>      mydataBias<span class="sc">$</span>unbiasedCriterion1,</span>
<span id="cb1489-8"><a href="bias.html#cb1489-8" tabindex="-1"></a>      mydataBias<span class="sc">$</span>unbiasedPredictor1),</span>
<span id="cb1489-9"><a href="bias.html#cb1489-9" tabindex="-1"></a>      <span class="at">na.rm =</span> <span class="cn">TRUE</span>)),</span>
<span id="cb1489-10"><a href="bias.html#cb1489-10" tabindex="-1"></a>  <span class="at">ylim =</span> <span class="fu">c</span>(</span>
<span id="cb1489-11"><a href="bias.html#cb1489-11" tabindex="-1"></a>    <span class="dv">0</span>, </span>
<span id="cb1489-12"><a href="bias.html#cb1489-12" tabindex="-1"></a>    <span class="fu">max</span>(<span class="fu">c</span>(</span>
<span id="cb1489-13"><a href="bias.html#cb1489-13" tabindex="-1"></a>      mydataBias<span class="sc">$</span>criterion1,</span>
<span id="cb1489-14"><a href="bias.html#cb1489-14" tabindex="-1"></a>      mydataBias<span class="sc">$</span>predictor),</span>
<span id="cb1489-15"><a href="bias.html#cb1489-15" tabindex="-1"></a>      <span class="at">na.rm =</span> <span class="cn">TRUE</span>)),</span>
<span id="cb1489-16"><a href="bias.html#cb1489-16" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;n&quot;</span>,</span>
<span id="cb1489-17"><a href="bias.html#cb1489-17" tabindex="-1"></a>  <span class="at">xlab =</span> <span class="st">&quot;predictor&quot;</span>,</span>
<span id="cb1489-18"><a href="bias.html#cb1489-18" tabindex="-1"></a>  <span class="at">ylab =</span> <span class="st">&quot;criterion&quot;</span>)</span>
<span id="cb1489-19"><a href="bias.html#cb1489-19" tabindex="-1"></a><span class="fu">points</span>(</span>
<span id="cb1489-20"><a href="bias.html#cb1489-20" tabindex="-1"></a>  mydataBias<span class="sc">$</span>unbiasedPredictor1[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)],</span>
<span id="cb1489-21"><a href="bias.html#cb1489-21" tabindex="-1"></a>  mydataBias<span class="sc">$</span>unbiasedCriterion1[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)],</span>
<span id="cb1489-22"><a href="bias.html#cb1489-22" tabindex="-1"></a>  <span class="at">pch =</span> <span class="dv">20</span>,</span>
<span id="cb1489-23"><a href="bias.html#cb1489-23" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;blue&quot;</span>)</span>
<span id="cb1489-24"><a href="bias.html#cb1489-24" tabindex="-1"></a><span class="fu">points</span>(mydataBias<span class="sc">$</span>unbiasedPredictor1[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)],</span>
<span id="cb1489-25"><a href="bias.html#cb1489-25" tabindex="-1"></a>       mydataBias<span class="sc">$</span>unbiasedCriterion1[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)],</span>
<span id="cb1489-26"><a href="bias.html#cb1489-26" tabindex="-1"></a>       <span class="at">pch =</span> <span class="dv">1</span>,</span>
<span id="cb1489-27"><a href="bias.html#cb1489-27" tabindex="-1"></a>       <span class="at">col =</span> <span class="st">&quot;red&quot;</span>)</span>
<span id="cb1489-28"><a href="bias.html#cb1489-28" tabindex="-1"></a><span class="fu">abline</span>(<span class="fu">lm</span>(</span>
<span id="cb1489-29"><a href="bias.html#cb1489-29" tabindex="-1"></a>  unbiasedCriterion1 <span class="sc">~</span> unbiasedPredictor1,</span>
<span id="cb1489-30"><a href="bias.html#cb1489-30" tabindex="-1"></a>  <span class="at">data =</span> mydataBias[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>),]),</span>
<span id="cb1489-31"><a href="bias.html#cb1489-31" tabindex="-1"></a>  <span class="at">lty =</span> <span class="dv">1</span>,</span>
<span id="cb1489-32"><a href="bias.html#cb1489-32" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;blue&quot;</span>)</span>
<span id="cb1489-33"><a href="bias.html#cb1489-33" tabindex="-1"></a><span class="fu">abline</span>(<span class="fu">lm</span>(</span>
<span id="cb1489-34"><a href="bias.html#cb1489-34" tabindex="-1"></a>  unbiasedCriterion1 <span class="sc">~</span> unbiasedPredictor1,</span>
<span id="cb1489-35"><a href="bias.html#cb1489-35" tabindex="-1"></a>  <span class="at">data =</span> mydataBias[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>),]),</span>
<span id="cb1489-36"><a href="bias.html#cb1489-36" tabindex="-1"></a>  <span class="at">lty =</span> <span class="dv">2</span>,</span>
<span id="cb1489-37"><a href="bias.html#cb1489-37" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;red&quot;</span>)</span>
<span id="cb1489-38"><a href="bias.html#cb1489-38" tabindex="-1"></a><span class="fu">legend</span>(</span>
<span id="cb1489-39"><a href="bias.html#cb1489-39" tabindex="-1"></a>  <span class="st">&quot;bottomright&quot;</span>,</span>
<span id="cb1489-40"><a href="bias.html#cb1489-40" tabindex="-1"></a>  <span class="fu">c</span>(<span class="st">&quot;Male&quot;</span>,<span class="st">&quot;Female&quot;</span>),</span>
<span id="cb1489-41"><a href="bias.html#cb1489-41" tabindex="-1"></a>  <span class="at">lty =</span> <span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">2</span>),</span>
<span id="cb1489-42"><a href="bias.html#cb1489-42" tabindex="-1"></a>  <span class="at">pch =</span> <span class="fu">c</span>(<span class="dv">20</span>,<span class="dv">1</span>),</span>
<span id="cb1489-43"><a href="bias.html#cb1489-43" tabindex="-1"></a>  <span class="at">col =</span> <span class="fu">c</span>(<span class="st">&quot;blue&quot;</span>,<span class="st">&quot;red&quot;</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:unbiasedEqualMeans"></span>
<img src="Book_files/figure-html/unbiasedEqualMeans-1.png" alt="Unbiased Test Where Males and Females Have Equal Means on Predictor and Criterion." width="100%" />
<p class="caption">
Figure 16.19: Unbiased Test Where Males and Females Have Equal Means on Predictor and Criterion.
</p>
</div>
</div>
<div id="unbiasedTestsFemalesHigherMeans" class="section level3 hasAnchor" number="16.7.2">
<h3><span class="header-section-number">16.7.2</span> Unbiased test where females have higher means than males on predictor and criterion<a href="bias.html#unbiasedTestsFemalesHigherMeans" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Figure <a href="bias.html#fig:unbiasedFemaleHigher">16.20</a> depicts an example of an unbiased test where females have higher means than males on the predictor and criterion.
The test is unbiased because there are no differences in the regression lines (of <code>predictor</code> predicting <code>criterion</code>) between males and females.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1490"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1490-1"><a href="bias.html#cb1490-1" tabindex="-1"></a><span class="fu">summary</span>(<span class="fu">lm</span>(</span>
<span id="cb1490-2"><a href="bias.html#cb1490-2" tabindex="-1"></a>  unbiasedCriterion2 <span class="sc">~</span> unbiasedPredictor2 <span class="sc">+</span> group <span class="sc">+</span> unbiasedPredictor2<span class="sc">:</span>group,</span>
<span id="cb1490-3"><a href="bias.html#cb1490-3" tabindex="-1"></a>  <span class="at">data =</span> mydataBias))</span></code></pre></div>
</details>
<pre><code>
Call:
lm(formula = unbiasedCriterion2 ~ unbiasedPredictor2 + group + 
    unbiasedPredictor2:group, data = mydataBias)

Residuals:
    Min      1Q  Median      3Q     Max 
-47.302  -9.989   0.010   9.860  53.791 

Coefficients:
                               Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)                    -1.29332    1.57773  -0.820    0.412    
unbiasedPredictor2              1.02006    0.02218  46.000   &lt;2e-16 ***
groupfemale                     1.21294    3.28319   0.369    0.712    
unbiasedPredictor2:groupfemale -0.01501    0.03125  -0.480    0.631    
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 14.81 on 3919 degrees of freedom
  (77 observations deleted due to missingness)
Multiple R-squared:  0.8412,    Adjusted R-squared:  0.8411 
F-statistic:  6921 on 3 and 3919 DF,  p-value: &lt; 2.2e-16</code></pre>
<details class=chunk-details><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1492"><pre class="sourceCode r fold-hide"><code class="sourceCode r"><span id="cb1492-1"><a href="bias.html#cb1492-1" tabindex="-1"></a><span class="fu">plot</span>(</span>
<span id="cb1492-2"><a href="bias.html#cb1492-2" tabindex="-1"></a>  unbiasedCriterion2 <span class="sc">~</span> unbiasedPredictor2,</span>
<span id="cb1492-3"><a href="bias.html#cb1492-3" tabindex="-1"></a>  <span class="at">data =</span> mydataBias,</span>
<span id="cb1492-4"><a href="bias.html#cb1492-4" tabindex="-1"></a>  <span class="at">xlim =</span> <span class="fu">c</span>(</span>
<span id="cb1492-5"><a href="bias.html#cb1492-5" tabindex="-1"></a>    <span class="dv">0</span>, </span>
<span id="cb1492-6"><a href="bias.html#cb1492-6" tabindex="-1"></a>    <span class="fu">max</span>(<span class="fu">c</span>(</span>
<span id="cb1492-7"><a href="bias.html#cb1492-7" tabindex="-1"></a>      mydataBias<span class="sc">$</span>unbiasedCriterion2,</span>
<span id="cb1492-8"><a href="bias.html#cb1492-8" tabindex="-1"></a>      mydataBias<span class="sc">$</span>unbiasedPredictor2),</span>
<span id="cb1492-9"><a href="bias.html#cb1492-9" tabindex="-1"></a>      <span class="at">na.rm =</span> <span class="cn">TRUE</span>)),</span>
<span id="cb1492-10"><a href="bias.html#cb1492-10" tabindex="-1"></a>  <span class="at">ylim =</span> <span class="fu">c</span>(</span>
<span id="cb1492-11"><a href="bias.html#cb1492-11" tabindex="-1"></a>    <span class="dv">0</span>, </span>
<span id="cb1492-12"><a href="bias.html#cb1492-12" tabindex="-1"></a>    <span class="fu">max</span>(<span class="fu">c</span>(</span>
<span id="cb1492-13"><a href="bias.html#cb1492-13" tabindex="-1"></a>      mydataBias<span class="sc">$</span>criterion1,</span>
<span id="cb1492-14"><a href="bias.html#cb1492-14" tabindex="-1"></a>      mydataBias<span class="sc">$</span>predictor),</span>
<span id="cb1492-15"><a href="bias.html#cb1492-15" tabindex="-1"></a>      <span class="at">na.rm =</span> <span class="cn">TRUE</span>)),</span>
<span id="cb1492-16"><a href="bias.html#cb1492-16" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;n&quot;</span>,</span>
<span id="cb1492-17"><a href="bias.html#cb1492-17" tabindex="-1"></a>  <span class="at">xlab =</span> <span class="st">&quot;predictor&quot;</span>,</span>
<span id="cb1492-18"><a href="bias.html#cb1492-18" tabindex="-1"></a>  <span class="at">ylab =</span> <span class="st">&quot;criterion&quot;</span>)</span>
<span id="cb1492-19"><a href="bias.html#cb1492-19" tabindex="-1"></a><span class="fu">points</span>(</span>
<span id="cb1492-20"><a href="bias.html#cb1492-20" tabindex="-1"></a>  mydataBias<span class="sc">$</span>unbiasedPredictor2[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)],</span>
<span id="cb1492-21"><a href="bias.html#cb1492-21" tabindex="-1"></a>  mydataBias<span class="sc">$</span>unbiasedCriterion2[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)],</span>
<span id="cb1492-22"><a href="bias.html#cb1492-22" tabindex="-1"></a>  <span class="at">pch =</span> <span class="dv">20</span>,</span>
<span id="cb1492-23"><a href="bias.html#cb1492-23" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;blue&quot;</span>)</span>
<span id="cb1492-24"><a href="bias.html#cb1492-24" tabindex="-1"></a><span class="fu">points</span>(</span>
<span id="cb1492-25"><a href="bias.html#cb1492-25" tabindex="-1"></a>  mydataBias<span class="sc">$</span>unbiasedPredictor2[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)],</span>
<span id="cb1492-26"><a href="bias.html#cb1492-26" tabindex="-1"></a>  mydataBias<span class="sc">$</span>unbiasedCriterion2[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)],</span>
<span id="cb1492-27"><a href="bias.html#cb1492-27" tabindex="-1"></a>  <span class="at">pch =</span> <span class="dv">1</span>,</span>
<span id="cb1492-28"><a href="bias.html#cb1492-28" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;red&quot;</span>)</span>
<span id="cb1492-29"><a href="bias.html#cb1492-29" tabindex="-1"></a><span class="fu">abline</span>(<span class="fu">lm</span>(</span>
<span id="cb1492-30"><a href="bias.html#cb1492-30" tabindex="-1"></a>  unbiasedCriterion2 <span class="sc">~</span> unbiasedPredictor2,</span>
<span id="cb1492-31"><a href="bias.html#cb1492-31" tabindex="-1"></a>  <span class="at">data =</span> mydataBias[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>),]),</span>
<span id="cb1492-32"><a href="bias.html#cb1492-32" tabindex="-1"></a>  <span class="at">lty =</span> <span class="dv">1</span>,</span>
<span id="cb1492-33"><a href="bias.html#cb1492-33" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;blue&quot;</span>)</span>
<span id="cb1492-34"><a href="bias.html#cb1492-34" tabindex="-1"></a><span class="fu">abline</span>(<span class="fu">lm</span>(</span>
<span id="cb1492-35"><a href="bias.html#cb1492-35" tabindex="-1"></a>  unbiasedCriterion2 <span class="sc">~</span> unbiasedPredictor2,</span>
<span id="cb1492-36"><a href="bias.html#cb1492-36" tabindex="-1"></a>  <span class="at">data =</span> mydataBias[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>),]),</span>
<span id="cb1492-37"><a href="bias.html#cb1492-37" tabindex="-1"></a>  <span class="at">lty =</span> <span class="dv">2</span>,</span>
<span id="cb1492-38"><a href="bias.html#cb1492-38" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;red&quot;</span>)</span>
<span id="cb1492-39"><a href="bias.html#cb1492-39" tabindex="-1"></a><span class="fu">legend</span>(</span>
<span id="cb1492-40"><a href="bias.html#cb1492-40" tabindex="-1"></a>  <span class="st">&quot;bottomright&quot;</span>,</span>
<span id="cb1492-41"><a href="bias.html#cb1492-41" tabindex="-1"></a>  <span class="fu">c</span>(<span class="st">&quot;Male&quot;</span>,<span class="st">&quot;Female&quot;</span>),</span>
<span id="cb1492-42"><a href="bias.html#cb1492-42" tabindex="-1"></a>  <span class="at">lty =</span> <span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">2</span>),</span>
<span id="cb1492-43"><a href="bias.html#cb1492-43" tabindex="-1"></a>  <span class="at">pch =</span> <span class="fu">c</span>(<span class="dv">20</span>,<span class="dv">1</span>),</span>
<span id="cb1492-44"><a href="bias.html#cb1492-44" tabindex="-1"></a>  <span class="at">col =</span> <span class="fu">c</span>(<span class="st">&quot;blue&quot;</span>,<span class="st">&quot;red&quot;</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:unbiasedFemaleHigher"></span>
<img src="Book_files/figure-html/unbiasedFemaleHigher-1.png" alt="Unbiased Test Where Females Have Higher Means Than Males on Predictor and Criterion." width="100%" />
<p class="caption">
Figure 16.20: Unbiased Test Where Females Have Higher Means Than Males on Predictor and Criterion.
</p>
</div>
</div>
<div id="unbiasedTestsMalesHigherMeans" class="section level3 hasAnchor" number="16.7.3">
<h3><span class="header-section-number">16.7.3</span> Unbiased test where males have higher means than females on predictor and criterion<a href="bias.html#unbiasedTestsMalesHigherMeans" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Figure <a href="bias.html#fig:unbiasedMaleHigher">16.21</a> depicts an example of an unbiased test where males have higher means than females on the predictor and criterion.
The test is unbiased because there are no differences in the regression lines (of <code>predictor</code> predicting <code>criterion</code>) between males and females.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1493"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1493-1"><a href="bias.html#cb1493-1" tabindex="-1"></a><span class="fu">summary</span>(<span class="fu">lm</span>(</span>
<span id="cb1493-2"><a href="bias.html#cb1493-2" tabindex="-1"></a>  unbiasedCriterion3 <span class="sc">~</span> unbiasedPredictor3 <span class="sc">+</span> group <span class="sc">+</span> unbiasedPredictor3<span class="sc">:</span>group,</span>
<span id="cb1493-3"><a href="bias.html#cb1493-3" tabindex="-1"></a>  <span class="at">data =</span> mydataBias))</span></code></pre></div>
</details>
<pre><code>
Call:
lm(formula = unbiasedCriterion3 ~ unbiasedPredictor3 + group + 
    unbiasedPredictor3:group, data = mydataBias)

Residuals:
    Min      1Q  Median      3Q     Max 
-48.613 -10.115  -0.068   9.598  57.126 

Coefficients:
                               Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)                    -1.68352    2.84985  -0.591    0.555    
unbiasedPredictor3              1.01072    0.02179  46.375   &lt;2e-16 ***
groupfemale                     1.42842    3.26227   0.438    0.662    
unbiasedPredictor3:groupfemale -0.01187    0.03109  -0.382    0.703    
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 14.82 on 3916 degrees of freedom
  (80 observations deleted due to missingness)
Multiple R-squared:  0.8376,    Adjusted R-squared:  0.8375 
F-statistic:  6732 on 3 and 3916 DF,  p-value: &lt; 2.2e-16</code></pre>
<details class=chunk-details><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1495"><pre class="sourceCode r fold-hide"><code class="sourceCode r"><span id="cb1495-1"><a href="bias.html#cb1495-1" tabindex="-1"></a><span class="fu">plot</span>(</span>
<span id="cb1495-2"><a href="bias.html#cb1495-2" tabindex="-1"></a>  unbiasedCriterion3 <span class="sc">~</span> unbiasedPredictor3,</span>
<span id="cb1495-3"><a href="bias.html#cb1495-3" tabindex="-1"></a>  <span class="at">data =</span> mydataBias,</span>
<span id="cb1495-4"><a href="bias.html#cb1495-4" tabindex="-1"></a>  <span class="at">xlim =</span> <span class="fu">c</span>(</span>
<span id="cb1495-5"><a href="bias.html#cb1495-5" tabindex="-1"></a>    <span class="dv">0</span>,</span>
<span id="cb1495-6"><a href="bias.html#cb1495-6" tabindex="-1"></a>    <span class="fu">max</span>(<span class="fu">c</span>(</span>
<span id="cb1495-7"><a href="bias.html#cb1495-7" tabindex="-1"></a>      mydataBias<span class="sc">$</span>unbiasedCriterion3,</span>
<span id="cb1495-8"><a href="bias.html#cb1495-8" tabindex="-1"></a>      mydataBias<span class="sc">$</span>unbiasedPredictor3),</span>
<span id="cb1495-9"><a href="bias.html#cb1495-9" tabindex="-1"></a>      <span class="at">na.rm =</span> <span class="cn">TRUE</span>)),</span>
<span id="cb1495-10"><a href="bias.html#cb1495-10" tabindex="-1"></a>  <span class="at">ylim =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="fu">max</span>(<span class="fu">c</span>(</span>
<span id="cb1495-11"><a href="bias.html#cb1495-11" tabindex="-1"></a>    mydataBias<span class="sc">$</span>criterion1,</span>
<span id="cb1495-12"><a href="bias.html#cb1495-12" tabindex="-1"></a>    mydataBias<span class="sc">$</span>predictor),</span>
<span id="cb1495-13"><a href="bias.html#cb1495-13" tabindex="-1"></a>    <span class="at">na.rm =</span> <span class="cn">TRUE</span>)),</span>
<span id="cb1495-14"><a href="bias.html#cb1495-14" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;n&quot;</span>,</span>
<span id="cb1495-15"><a href="bias.html#cb1495-15" tabindex="-1"></a>  <span class="at">xlab =</span> <span class="st">&quot;predictor&quot;</span>,</span>
<span id="cb1495-16"><a href="bias.html#cb1495-16" tabindex="-1"></a>  <span class="at">ylab =</span> <span class="st">&quot;criterion&quot;</span>)</span>
<span id="cb1495-17"><a href="bias.html#cb1495-17" tabindex="-1"></a><span class="fu">points</span>(</span>
<span id="cb1495-18"><a href="bias.html#cb1495-18" tabindex="-1"></a>  mydataBias<span class="sc">$</span>unbiasedPredictor3[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)],</span>
<span id="cb1495-19"><a href="bias.html#cb1495-19" tabindex="-1"></a>  mydataBias<span class="sc">$</span>unbiasedCriterion3[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)],</span>
<span id="cb1495-20"><a href="bias.html#cb1495-20" tabindex="-1"></a>  <span class="at">pch =</span> <span class="dv">20</span>,</span>
<span id="cb1495-21"><a href="bias.html#cb1495-21" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;blue&quot;</span>)</span>
<span id="cb1495-22"><a href="bias.html#cb1495-22" tabindex="-1"></a><span class="fu">points</span>(</span>
<span id="cb1495-23"><a href="bias.html#cb1495-23" tabindex="-1"></a>  mydataBias<span class="sc">$</span>unbiasedPredictor3[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)],</span>
<span id="cb1495-24"><a href="bias.html#cb1495-24" tabindex="-1"></a>  mydataBias<span class="sc">$</span>unbiasedCriterion3[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)],</span>
<span id="cb1495-25"><a href="bias.html#cb1495-25" tabindex="-1"></a>  <span class="at">pch =</span> <span class="dv">1</span>,</span>
<span id="cb1495-26"><a href="bias.html#cb1495-26" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;red&quot;</span>)</span>
<span id="cb1495-27"><a href="bias.html#cb1495-27" tabindex="-1"></a><span class="fu">abline</span>(<span class="fu">lm</span>(</span>
<span id="cb1495-28"><a href="bias.html#cb1495-28" tabindex="-1"></a>  unbiasedCriterion3 <span class="sc">~</span> unbiasedPredictor3,</span>
<span id="cb1495-29"><a href="bias.html#cb1495-29" tabindex="-1"></a>  <span class="at">data =</span> mydataBias[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>),]),</span>
<span id="cb1495-30"><a href="bias.html#cb1495-30" tabindex="-1"></a>  <span class="at">lty =</span> <span class="dv">1</span>,</span>
<span id="cb1495-31"><a href="bias.html#cb1495-31" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;blue&quot;</span>)</span>
<span id="cb1495-32"><a href="bias.html#cb1495-32" tabindex="-1"></a><span class="fu">abline</span>(<span class="fu">lm</span>(</span>
<span id="cb1495-33"><a href="bias.html#cb1495-33" tabindex="-1"></a>  unbiasedCriterion3 <span class="sc">~</span> unbiasedPredictor3,</span>
<span id="cb1495-34"><a href="bias.html#cb1495-34" tabindex="-1"></a>  <span class="at">data =</span> mydataBias[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>),]),</span>
<span id="cb1495-35"><a href="bias.html#cb1495-35" tabindex="-1"></a>  <span class="at">lty =</span> <span class="dv">2</span>,</span>
<span id="cb1495-36"><a href="bias.html#cb1495-36" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;red&quot;</span>)</span>
<span id="cb1495-37"><a href="bias.html#cb1495-37" tabindex="-1"></a><span class="fu">legend</span>(</span>
<span id="cb1495-38"><a href="bias.html#cb1495-38" tabindex="-1"></a>  <span class="st">&quot;bottomright&quot;</span>,</span>
<span id="cb1495-39"><a href="bias.html#cb1495-39" tabindex="-1"></a>  <span class="fu">c</span>(<span class="st">&quot;Male&quot;</span>,<span class="st">&quot;Female&quot;</span>),</span>
<span id="cb1495-40"><a href="bias.html#cb1495-40" tabindex="-1"></a>  <span class="at">lty =</span> <span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">2</span>),</span>
<span id="cb1495-41"><a href="bias.html#cb1495-41" tabindex="-1"></a>  <span class="at">pch =</span> <span class="fu">c</span>(<span class="dv">20</span>,<span class="dv">1</span>),</span>
<span id="cb1495-42"><a href="bias.html#cb1495-42" tabindex="-1"></a>  <span class="at">col =</span> <span class="fu">c</span>(<span class="st">&quot;blue&quot;</span>,<span class="st">&quot;red&quot;</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:unbiasedMaleHigher"></span>
<img src="Book_files/figure-html/unbiasedMaleHigher-1.png" alt="Unbiased Test Where Males Have Higher Means Than Females on Predictor and Criterion." width="100%" />
<p class="caption">
Figure 16.21: Unbiased Test Where Males Have Higher Means Than Females on Predictor and Criterion.
</p>
</div>
</div>
</div>
<div id="predictiveBiasExamples" class="section level2 hasAnchor" number="16.8">
<h2><span class="header-section-number">16.8</span> Predictive Bias: Different Regression Lines<a href="bias.html#predictiveBiasExamples" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="unbiasedPrediction" class="section level3 hasAnchor" number="16.8.1">
<h3><span class="header-section-number">16.8.1</span> Example of unbiased prediction (no differences in intercepts or slopes)<a href="bias.html#unbiasedPrediction" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Figure <a href="bias.html#fig:unbiased2">16.22</a> depicts an example of an unbiased test where males and females have equal means on the predictor and criterion.
The test is unbiased because there are no differences in the regression lines (of <code>predictor</code> predicting <code>criterion1</code>) between males and females.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1496"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1496-1"><a href="bias.html#cb1496-1" tabindex="-1"></a><span class="fu">summary</span>(<span class="fu">lm</span>(</span>
<span id="cb1496-2"><a href="bias.html#cb1496-2" tabindex="-1"></a>  criterion1 <span class="sc">~</span> predictor <span class="sc">+</span> group <span class="sc">+</span> predictor<span class="sc">:</span>group,</span>
<span id="cb1496-3"><a href="bias.html#cb1496-3" tabindex="-1"></a>  <span class="at">data =</span> mydataBias))</span></code></pre></div>
</details>
<pre><code>
Call:
lm(formula = criterion1 ~ predictor + group + predictor:group, 
    data = mydataBias)

Residuals:
    Min      1Q  Median      3Q     Max 
-18.831  -3.338   0.004   3.330  19.335 

Coefficients:
                       Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)           -0.264357   0.747269  -0.354    0.724    
predictor              0.701726   0.007370  95.207   &lt;2e-16 ***
groupfemale           -0.515860   1.059267  -0.487    0.626    
predictor:groupfemale  0.006002   0.010476   0.573    0.567    
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 4.952 on 3908 degrees of freedom
  (88 observations deleted due to missingness)
Multiple R-squared:  0.8225,    Adjusted R-squared:  0.8223 
F-statistic:  6035 on 3 and 3908 DF,  p-value: &lt; 2.2e-16</code></pre>
<details class=chunk-details><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1498"><pre class="sourceCode r fold-hide"><code class="sourceCode r"><span id="cb1498-1"><a href="bias.html#cb1498-1" tabindex="-1"></a><span class="fu">plot</span>(</span>
<span id="cb1498-2"><a href="bias.html#cb1498-2" tabindex="-1"></a>  criterion1 <span class="sc">~</span> predictor,</span>
<span id="cb1498-3"><a href="bias.html#cb1498-3" tabindex="-1"></a>  <span class="at">data =</span> mydataBias,</span>
<span id="cb1498-4"><a href="bias.html#cb1498-4" tabindex="-1"></a>  <span class="at">xlim =</span> <span class="fu">c</span>(</span>
<span id="cb1498-5"><a href="bias.html#cb1498-5" tabindex="-1"></a>    <span class="dv">0</span>,</span>
<span id="cb1498-6"><a href="bias.html#cb1498-6" tabindex="-1"></a>    <span class="fu">max</span>(<span class="fu">c</span>(</span>
<span id="cb1498-7"><a href="bias.html#cb1498-7" tabindex="-1"></a>      mydataBias<span class="sc">$</span>criterion1,</span>
<span id="cb1498-8"><a href="bias.html#cb1498-8" tabindex="-1"></a>      mydataBias<span class="sc">$</span>predictor),</span>
<span id="cb1498-9"><a href="bias.html#cb1498-9" tabindex="-1"></a>      <span class="at">na.rm =</span> <span class="cn">TRUE</span>)),</span>
<span id="cb1498-10"><a href="bias.html#cb1498-10" tabindex="-1"></a>  <span class="at">ylim =</span> <span class="fu">c</span>(</span>
<span id="cb1498-11"><a href="bias.html#cb1498-11" tabindex="-1"></a>    <span class="dv">0</span>, </span>
<span id="cb1498-12"><a href="bias.html#cb1498-12" tabindex="-1"></a>    <span class="fu">max</span>(<span class="fu">c</span>(</span>
<span id="cb1498-13"><a href="bias.html#cb1498-13" tabindex="-1"></a>      mydataBias<span class="sc">$</span>criterion1,</span>
<span id="cb1498-14"><a href="bias.html#cb1498-14" tabindex="-1"></a>      mydataBias<span class="sc">$</span>predictor),</span>
<span id="cb1498-15"><a href="bias.html#cb1498-15" tabindex="-1"></a>      <span class="at">na.rm =</span> <span class="cn">TRUE</span>)),</span>
<span id="cb1498-16"><a href="bias.html#cb1498-16" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;n&quot;</span>,</span>
<span id="cb1498-17"><a href="bias.html#cb1498-17" tabindex="-1"></a>  <span class="at">xlab =</span> <span class="st">&quot;predictor&quot;</span>,</span>
<span id="cb1498-18"><a href="bias.html#cb1498-18" tabindex="-1"></a>  <span class="at">ylab =</span> <span class="st">&quot;criterion&quot;</span>)</span>
<span id="cb1498-19"><a href="bias.html#cb1498-19" tabindex="-1"></a><span class="fu">points</span>(</span>
<span id="cb1498-20"><a href="bias.html#cb1498-20" tabindex="-1"></a>  mydataBias<span class="sc">$</span>predictor[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)],</span>
<span id="cb1498-21"><a href="bias.html#cb1498-21" tabindex="-1"></a>  mydataBias<span class="sc">$</span>criterion1[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)],</span>
<span id="cb1498-22"><a href="bias.html#cb1498-22" tabindex="-1"></a>  <span class="at">pch =</span> <span class="dv">20</span>,</span>
<span id="cb1498-23"><a href="bias.html#cb1498-23" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;blue&quot;</span>)</span>
<span id="cb1498-24"><a href="bias.html#cb1498-24" tabindex="-1"></a><span class="fu">points</span>(</span>
<span id="cb1498-25"><a href="bias.html#cb1498-25" tabindex="-1"></a>  mydataBias<span class="sc">$</span>predictor[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)],</span>
<span id="cb1498-26"><a href="bias.html#cb1498-26" tabindex="-1"></a>  mydataBias<span class="sc">$</span>criterion1[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)],</span>
<span id="cb1498-27"><a href="bias.html#cb1498-27" tabindex="-1"></a>  <span class="at">pch =</span> <span class="dv">1</span>,</span>
<span id="cb1498-28"><a href="bias.html#cb1498-28" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;red&quot;</span>)</span>
<span id="cb1498-29"><a href="bias.html#cb1498-29" tabindex="-1"></a><span class="fu">abline</span>(<span class="fu">lm</span>(</span>
<span id="cb1498-30"><a href="bias.html#cb1498-30" tabindex="-1"></a>  criterion1 <span class="sc">~</span> predictor,</span>
<span id="cb1498-31"><a href="bias.html#cb1498-31" tabindex="-1"></a>  <span class="at">data =</span> mydataBias[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>),]),</span>
<span id="cb1498-32"><a href="bias.html#cb1498-32" tabindex="-1"></a>  <span class="at">lty =</span> <span class="dv">1</span>,</span>
<span id="cb1498-33"><a href="bias.html#cb1498-33" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;blue&quot;</span>)</span>
<span id="cb1498-34"><a href="bias.html#cb1498-34" tabindex="-1"></a><span class="fu">abline</span>(<span class="fu">lm</span>(</span>
<span id="cb1498-35"><a href="bias.html#cb1498-35" tabindex="-1"></a>  criterion1 <span class="sc">~</span> predictor,</span>
<span id="cb1498-36"><a href="bias.html#cb1498-36" tabindex="-1"></a>  <span class="at">data =</span> mydataBias[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>),]),</span>
<span id="cb1498-37"><a href="bias.html#cb1498-37" tabindex="-1"></a>  <span class="at">lty =</span> <span class="dv">2</span>,</span>
<span id="cb1498-38"><a href="bias.html#cb1498-38" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;red&quot;</span>)</span>
<span id="cb1498-39"><a href="bias.html#cb1498-39" tabindex="-1"></a><span class="fu">legend</span>(</span>
<span id="cb1498-40"><a href="bias.html#cb1498-40" tabindex="-1"></a>  <span class="st">&quot;bottomright&quot;</span>,</span>
<span id="cb1498-41"><a href="bias.html#cb1498-41" tabindex="-1"></a>  <span class="fu">c</span>(<span class="st">&quot;Male&quot;</span>,<span class="st">&quot;Female&quot;</span>),</span>
<span id="cb1498-42"><a href="bias.html#cb1498-42" tabindex="-1"></a>  <span class="at">lty =</span> <span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">2</span>),</span>
<span id="cb1498-43"><a href="bias.html#cb1498-43" tabindex="-1"></a>  <span class="at">pch =</span> <span class="fu">c</span>(<span class="dv">20</span>,<span class="dv">1</span>),</span>
<span id="cb1498-44"><a href="bias.html#cb1498-44" tabindex="-1"></a>  <span class="at">col =</span> <span class="fu">c</span>(<span class="st">&quot;blue&quot;</span>,<span class="st">&quot;red&quot;</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:unbiased2"></span>
<img src="Book_files/figure-html/unbiased2-1.png" alt="Example of Unbiased Prediction (No Differences in Intercepts or Slopes Between Males and Females)." width="100%" />
<p class="caption">
Figure 16.22: Example of Unbiased Prediction (No Differences in Intercepts or Slopes Between Males and Females).
</p>
</div>
</div>
<div id="interceptBias" class="section level3 hasAnchor" number="16.8.2">
<h3><span class="header-section-number">16.8.2</span> Example of intercept bias<a href="bias.html#interceptBias" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Figure <a href="bias.html#fig:interceptBias">16.23</a> depicts an example of a biased test due to intercept bias.
There are differences in the intercepts of the regression lines (of <code>predictor</code> predicting <code>criterion2</code>) between males and females: males have a higher intercept than females.
That is, the same score on the predictor results in higher predictions for males than females.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1499"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1499-1"><a href="bias.html#cb1499-1" tabindex="-1"></a><span class="fu">summary</span>(<span class="fu">lm</span>(</span>
<span id="cb1499-2"><a href="bias.html#cb1499-2" tabindex="-1"></a>  criterion2 <span class="sc">~</span> predictor <span class="sc">+</span> group <span class="sc">+</span> predictor<span class="sc">:</span>group,</span>
<span id="cb1499-3"><a href="bias.html#cb1499-3" tabindex="-1"></a>  <span class="at">data =</span> mydataBias))</span></code></pre></div>
</details>
<pre><code>
Call:
lm(formula = criterion2 ~ predictor + group + predictor:group, 
    data = mydataBias)

Residuals:
     Min       1Q   Median       3Q      Max 
-18.6853  -3.4153  -0.0385   3.3979  18.0864 

Coefficients:
                        Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)            10.054048   0.760084  13.228   &lt;2e-16 ***
predictor               0.697987   0.007499  93.075   &lt;2e-16 ***
groupfemale           -10.288893   1.074168  -9.578   &lt;2e-16 ***
predictor:groupfemale   0.004748   0.010625   0.447    0.655    
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 5.037 on 3913 degrees of freedom
  (83 observations deleted due to missingness)
Multiple R-squared:  0.8452,    Adjusted R-squared:  0.8451 
F-statistic:  7124 on 3 and 3913 DF,  p-value: &lt; 2.2e-16</code></pre>
<details class=chunk-details><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1501"><pre class="sourceCode r fold-hide"><code class="sourceCode r"><span id="cb1501-1"><a href="bias.html#cb1501-1" tabindex="-1"></a><span class="fu">plot</span>(</span>
<span id="cb1501-2"><a href="bias.html#cb1501-2" tabindex="-1"></a>  criterion2 <span class="sc">~</span> predictor,</span>
<span id="cb1501-3"><a href="bias.html#cb1501-3" tabindex="-1"></a>  <span class="at">data =</span> mydataBias,</span>
<span id="cb1501-4"><a href="bias.html#cb1501-4" tabindex="-1"></a>  <span class="at">xlim =</span> <span class="fu">c</span>(</span>
<span id="cb1501-5"><a href="bias.html#cb1501-5" tabindex="-1"></a>    <span class="dv">0</span>,</span>
<span id="cb1501-6"><a href="bias.html#cb1501-6" tabindex="-1"></a>    <span class="fu">max</span>(<span class="fu">c</span>(</span>
<span id="cb1501-7"><a href="bias.html#cb1501-7" tabindex="-1"></a>      mydataBias<span class="sc">$</span>criterion2,</span>
<span id="cb1501-8"><a href="bias.html#cb1501-8" tabindex="-1"></a>      mydataBias<span class="sc">$</span>predictor), <span class="at">na.rm =</span> <span class="cn">TRUE</span>)),</span>
<span id="cb1501-9"><a href="bias.html#cb1501-9" tabindex="-1"></a>  <span class="at">ylim =</span> <span class="fu">c</span>(</span>
<span id="cb1501-10"><a href="bias.html#cb1501-10" tabindex="-1"></a>    <span class="dv">0</span>,</span>
<span id="cb1501-11"><a href="bias.html#cb1501-11" tabindex="-1"></a>    <span class="fu">max</span>(<span class="fu">c</span>(</span>
<span id="cb1501-12"><a href="bias.html#cb1501-12" tabindex="-1"></a>      mydataBias<span class="sc">$</span>criterion2,</span>
<span id="cb1501-13"><a href="bias.html#cb1501-13" tabindex="-1"></a>      mydataBias<span class="sc">$</span>predictor),</span>
<span id="cb1501-14"><a href="bias.html#cb1501-14" tabindex="-1"></a>      <span class="at">na.rm =</span> <span class="cn">TRUE</span>)),</span>
<span id="cb1501-15"><a href="bias.html#cb1501-15" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;n&quot;</span>,</span>
<span id="cb1501-16"><a href="bias.html#cb1501-16" tabindex="-1"></a>  <span class="at">xlab =</span> <span class="st">&quot;predictor&quot;</span>,</span>
<span id="cb1501-17"><a href="bias.html#cb1501-17" tabindex="-1"></a>  <span class="at">ylab =</span> <span class="st">&quot;criterion&quot;</span>)</span>
<span id="cb1501-18"><a href="bias.html#cb1501-18" tabindex="-1"></a><span class="fu">points</span>(</span>
<span id="cb1501-19"><a href="bias.html#cb1501-19" tabindex="-1"></a>  mydataBias<span class="sc">$</span>predictor[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)],</span>
<span id="cb1501-20"><a href="bias.html#cb1501-20" tabindex="-1"></a>  mydataBias<span class="sc">$</span>criterion2[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)],</span>
<span id="cb1501-21"><a href="bias.html#cb1501-21" tabindex="-1"></a>  <span class="at">pch =</span> <span class="dv">20</span>,</span>
<span id="cb1501-22"><a href="bias.html#cb1501-22" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;blue&quot;</span>)</span>
<span id="cb1501-23"><a href="bias.html#cb1501-23" tabindex="-1"></a><span class="fu">points</span>(</span>
<span id="cb1501-24"><a href="bias.html#cb1501-24" tabindex="-1"></a>  mydataBias<span class="sc">$</span>predictor[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)],</span>
<span id="cb1501-25"><a href="bias.html#cb1501-25" tabindex="-1"></a>  mydataBias<span class="sc">$</span>criterion2[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)],</span>
<span id="cb1501-26"><a href="bias.html#cb1501-26" tabindex="-1"></a>  <span class="at">pch =</span> <span class="dv">1</span>,</span>
<span id="cb1501-27"><a href="bias.html#cb1501-27" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;red&quot;</span>)</span>
<span id="cb1501-28"><a href="bias.html#cb1501-28" tabindex="-1"></a><span class="fu">abline</span>(<span class="fu">lm</span>(</span>
<span id="cb1501-29"><a href="bias.html#cb1501-29" tabindex="-1"></a>  criterion2 <span class="sc">~</span> predictor,</span>
<span id="cb1501-30"><a href="bias.html#cb1501-30" tabindex="-1"></a>  <span class="at">data =</span> mydataBias[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>),]),</span>
<span id="cb1501-31"><a href="bias.html#cb1501-31" tabindex="-1"></a>  <span class="at">lty =</span> <span class="dv">1</span>,</span>
<span id="cb1501-32"><a href="bias.html#cb1501-32" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;blue&quot;</span>)</span>
<span id="cb1501-33"><a href="bias.html#cb1501-33" tabindex="-1"></a><span class="fu">abline</span>(<span class="fu">lm</span>(</span>
<span id="cb1501-34"><a href="bias.html#cb1501-34" tabindex="-1"></a>  criterion2 <span class="sc">~</span> predictor,</span>
<span id="cb1501-35"><a href="bias.html#cb1501-35" tabindex="-1"></a>  <span class="at">data =</span> mydataBias[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>),]),</span>
<span id="cb1501-36"><a href="bias.html#cb1501-36" tabindex="-1"></a>  <span class="at">lty =</span> <span class="dv">2</span>,</span>
<span id="cb1501-37"><a href="bias.html#cb1501-37" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;red&quot;</span>)</span>
<span id="cb1501-38"><a href="bias.html#cb1501-38" tabindex="-1"></a><span class="fu">legend</span>(</span>
<span id="cb1501-39"><a href="bias.html#cb1501-39" tabindex="-1"></a>  <span class="st">&quot;bottomright&quot;</span>,</span>
<span id="cb1501-40"><a href="bias.html#cb1501-40" tabindex="-1"></a>  <span class="fu">c</span>(<span class="st">&quot;Male&quot;</span>,<span class="st">&quot;Female&quot;</span>),</span>
<span id="cb1501-41"><a href="bias.html#cb1501-41" tabindex="-1"></a>  <span class="at">lty =</span> <span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">2</span>),</span>
<span id="cb1501-42"><a href="bias.html#cb1501-42" tabindex="-1"></a>  <span class="at">pch =</span> <span class="fu">c</span>(<span class="dv">20</span>,<span class="dv">1</span>),</span>
<span id="cb1501-43"><a href="bias.html#cb1501-43" tabindex="-1"></a>  <span class="at">col =</span> <span class="fu">c</span>(<span class="st">&quot;blue&quot;</span>,<span class="st">&quot;red&quot;</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:interceptBias"></span>
<img src="Book_files/figure-html/interceptBias-1.png" alt="Example of Intercept Bias in Prediction (Different Intercepts Between Males and Females)." width="100%" />
<p class="caption">
Figure 16.23: Example of Intercept Bias in Prediction (Different Intercepts Between Males and Females).
</p>
</div>
</div>
<div id="slopeBias" class="section level3 hasAnchor" number="16.8.3">
<h3><span class="header-section-number">16.8.3</span> Example of slope bias<a href="bias.html#slopeBias" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Figure <a href="bias.html#fig:slopeBias">16.24</a> depicts an example of a biased test due to slope bias.
There are differences in the slopes of the regression lines (of <code>predictor</code> predicting <code>criterion3</code>) between males and females: males have a higher slope than females.
That is, scores have stronger <a href="validity.html#predictiveValidity">predictive validity</a> for males than females.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1502"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1502-1"><a href="bias.html#cb1502-1" tabindex="-1"></a><span class="fu">summary</span>(<span class="fu">lm</span>(</span>
<span id="cb1502-2"><a href="bias.html#cb1502-2" tabindex="-1"></a>  criterion3 <span class="sc">~</span> predictor <span class="sc">+</span> group <span class="sc">+</span> predictor<span class="sc">:</span>group,</span>
<span id="cb1502-3"><a href="bias.html#cb1502-3" tabindex="-1"></a>  <span class="at">data =</span> mydataBias))</span></code></pre></div>
</details>
<pre><code>
Call:
lm(formula = criterion3 ~ predictor + group + predictor:group, 
    data = mydataBias)

Residuals:
     Min       1Q   Median       3Q      Max 
-18.3183  -3.3715   0.0256   3.3844  19.3484 

Coefficients:
                       Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)           -1.041940   0.757914  -1.375    0.169    
predictor              0.708670   0.007475  94.804   &lt;2e-16 ***
groupfemale            1.478534   1.074064   1.377    0.169    
predictor:groupfemale -0.413233   0.010621 -38.907   &lt;2e-16 ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 5.031 on 3927 degrees of freedom
  (69 observations deleted due to missingness)
Multiple R-squared:  0.9489,    Adjusted R-squared:  0.9489 
F-statistic: 2.432e+04 on 3 and 3927 DF,  p-value: &lt; 2.2e-16</code></pre>
<details class=chunk-details><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1504"><pre class="sourceCode r fold-hide"><code class="sourceCode r"><span id="cb1504-1"><a href="bias.html#cb1504-1" tabindex="-1"></a><span class="fu">plot</span>(</span>
<span id="cb1504-2"><a href="bias.html#cb1504-2" tabindex="-1"></a>  criterion3 <span class="sc">~</span> predictor,</span>
<span id="cb1504-3"><a href="bias.html#cb1504-3" tabindex="-1"></a>  <span class="at">data =</span> mydataBias,</span>
<span id="cb1504-4"><a href="bias.html#cb1504-4" tabindex="-1"></a>  <span class="at">xlim =</span> <span class="fu">c</span>(</span>
<span id="cb1504-5"><a href="bias.html#cb1504-5" tabindex="-1"></a>    <span class="dv">0</span>,</span>
<span id="cb1504-6"><a href="bias.html#cb1504-6" tabindex="-1"></a>    <span class="fu">max</span>(<span class="fu">c</span>(</span>
<span id="cb1504-7"><a href="bias.html#cb1504-7" tabindex="-1"></a>      mydataBias<span class="sc">$</span>criterion3,</span>
<span id="cb1504-8"><a href="bias.html#cb1504-8" tabindex="-1"></a>      mydataBias<span class="sc">$</span>predictor),</span>
<span id="cb1504-9"><a href="bias.html#cb1504-9" tabindex="-1"></a>      <span class="at">na.rm =</span> <span class="cn">TRUE</span>)),</span>
<span id="cb1504-10"><a href="bias.html#cb1504-10" tabindex="-1"></a>  <span class="at">ylim =</span> <span class="fu">c</span>(</span>
<span id="cb1504-11"><a href="bias.html#cb1504-11" tabindex="-1"></a>    <span class="dv">0</span>,</span>
<span id="cb1504-12"><a href="bias.html#cb1504-12" tabindex="-1"></a>    <span class="fu">max</span>(<span class="fu">c</span>(</span>
<span id="cb1504-13"><a href="bias.html#cb1504-13" tabindex="-1"></a>      mydataBias<span class="sc">$</span>criterion3,</span>
<span id="cb1504-14"><a href="bias.html#cb1504-14" tabindex="-1"></a>      mydataBias<span class="sc">$</span>predictor),</span>
<span id="cb1504-15"><a href="bias.html#cb1504-15" tabindex="-1"></a>      <span class="at">na.rm =</span> <span class="cn">TRUE</span>)),</span>
<span id="cb1504-16"><a href="bias.html#cb1504-16" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;n&quot;</span>,</span>
<span id="cb1504-17"><a href="bias.html#cb1504-17" tabindex="-1"></a>  <span class="at">xlab =</span> <span class="st">&quot;predictor&quot;</span>,</span>
<span id="cb1504-18"><a href="bias.html#cb1504-18" tabindex="-1"></a>  <span class="at">ylab =</span> <span class="st">&quot;criterion&quot;</span>)</span>
<span id="cb1504-19"><a href="bias.html#cb1504-19" tabindex="-1"></a><span class="fu">points</span>(</span>
<span id="cb1504-20"><a href="bias.html#cb1504-20" tabindex="-1"></a>  mydataBias<span class="sc">$</span>predictor[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)],</span>
<span id="cb1504-21"><a href="bias.html#cb1504-21" tabindex="-1"></a>  mydataBias<span class="sc">$</span>criterion3[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)],</span>
<span id="cb1504-22"><a href="bias.html#cb1504-22" tabindex="-1"></a>  <span class="at">pch =</span> <span class="dv">20</span>,</span>
<span id="cb1504-23"><a href="bias.html#cb1504-23" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;blue&quot;</span>)</span>
<span id="cb1504-24"><a href="bias.html#cb1504-24" tabindex="-1"></a><span class="fu">points</span>(</span>
<span id="cb1504-25"><a href="bias.html#cb1504-25" tabindex="-1"></a>  mydataBias<span class="sc">$</span>predictor[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)],</span>
<span id="cb1504-26"><a href="bias.html#cb1504-26" tabindex="-1"></a>  mydataBias<span class="sc">$</span>criterion3[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)],</span>
<span id="cb1504-27"><a href="bias.html#cb1504-27" tabindex="-1"></a>  <span class="at">pch =</span> <span class="dv">1</span>,</span>
<span id="cb1504-28"><a href="bias.html#cb1504-28" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;red&quot;</span>)</span>
<span id="cb1504-29"><a href="bias.html#cb1504-29" tabindex="-1"></a><span class="fu">abline</span>(<span class="fu">lm</span>(</span>
<span id="cb1504-30"><a href="bias.html#cb1504-30" tabindex="-1"></a>  criterion3 <span class="sc">~</span> predictor,</span>
<span id="cb1504-31"><a href="bias.html#cb1504-31" tabindex="-1"></a>  <span class="at">data =</span> mydataBias[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>),]),</span>
<span id="cb1504-32"><a href="bias.html#cb1504-32" tabindex="-1"></a>  <span class="at">lty =</span> <span class="dv">1</span>,</span>
<span id="cb1504-33"><a href="bias.html#cb1504-33" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;blue&quot;</span>)</span>
<span id="cb1504-34"><a href="bias.html#cb1504-34" tabindex="-1"></a><span class="fu">abline</span>(<span class="fu">lm</span>(</span>
<span id="cb1504-35"><a href="bias.html#cb1504-35" tabindex="-1"></a>  criterion3 <span class="sc">~</span> predictor,</span>
<span id="cb1504-36"><a href="bias.html#cb1504-36" tabindex="-1"></a>  <span class="at">data =</span> mydataBias[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>),]),</span>
<span id="cb1504-37"><a href="bias.html#cb1504-37" tabindex="-1"></a>  <span class="at">lty =</span> <span class="dv">2</span>,</span>
<span id="cb1504-38"><a href="bias.html#cb1504-38" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;red&quot;</span>)</span>
<span id="cb1504-39"><a href="bias.html#cb1504-39" tabindex="-1"></a><span class="fu">legend</span>(</span>
<span id="cb1504-40"><a href="bias.html#cb1504-40" tabindex="-1"></a>  <span class="st">&quot;bottomright&quot;</span>,</span>
<span id="cb1504-41"><a href="bias.html#cb1504-41" tabindex="-1"></a>  <span class="fu">c</span>(<span class="st">&quot;Male&quot;</span>,<span class="st">&quot;Female&quot;</span>),</span>
<span id="cb1504-42"><a href="bias.html#cb1504-42" tabindex="-1"></a>  <span class="at">lty =</span> <span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">2</span>),</span>
<span id="cb1504-43"><a href="bias.html#cb1504-43" tabindex="-1"></a>  <span class="at">pch =</span> <span class="fu">c</span>(<span class="dv">20</span>,<span class="dv">1</span>),</span>
<span id="cb1504-44"><a href="bias.html#cb1504-44" tabindex="-1"></a>  <span class="at">col =</span> <span class="fu">c</span>(<span class="st">&quot;blue&quot;</span>,<span class="st">&quot;red&quot;</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:slopeBias"></span>
<img src="Book_files/figure-html/slopeBias-1.png" alt="Example of Slope Bias in Prediction (Different Slopes Between Males And Females)." width="100%" />
<p class="caption">
Figure 16.24: Example of Slope Bias in Prediction (Different Slopes Between Males And Females).
</p>
</div>
</div>
<div id="interceptSlopeBias" class="section level3 hasAnchor" number="16.8.4">
<h3><span class="header-section-number">16.8.4</span> Example of intercept and slope bias<a href="bias.html#interceptSlopeBias" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Figure <a href="bias.html#fig:interceptSlopeBias">16.25</a> depicts an example of a biased test due to intercept and slope bias.
There are differences in the intercepts and slopes of the regression lines (of <code>predictor</code> predicting <code>criterion4</code>) between males and females: males have a higher slope than females.
That is, scores have stronger <a href="validity.html#predictiveValidity">predictive validity</a> for males than females.
Females have a higher intercept than males.
That is, at lower scores, the same score on the predictor results in higher predictions for females than males; at higher scores on the predictor, the same score results in higher predictions for males than females.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1505"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1505-1"><a href="bias.html#cb1505-1" tabindex="-1"></a><span class="fu">summary</span>(<span class="fu">lm</span>(</span>
<span id="cb1505-2"><a href="bias.html#cb1505-2" tabindex="-1"></a>  criterion4 <span class="sc">~</span> predictor <span class="sc">+</span> group <span class="sc">+</span> predictor<span class="sc">:</span>group,</span>
<span id="cb1505-3"><a href="bias.html#cb1505-3" tabindex="-1"></a>  <span class="at">data =</span> mydataBias))</span></code></pre></div>
</details>
<pre><code>
Call:
lm(formula = criterion4 ~ predictor + group + predictor:group, 
    data = mydataBias)

Residuals:
     Min       1Q   Median       3Q      Max 
-18.2805  -3.4400   0.0269   3.3199  17.8824 

Coefficients:
                       Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)           -0.276228   0.767080   -0.36    0.719    
predictor              0.702178   0.007565   92.81   &lt;2e-16 ***
groupfemale           29.156167   1.086254   26.84   &lt;2e-16 ***
predictor:groupfemale -0.392016   0.010743  -36.49   &lt;2e-16 ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 5.099 on 3930 degrees of freedom
  (66 observations deleted due to missingness)
Multiple R-squared:  0.7843,    Adjusted R-squared:  0.7842 
F-statistic:  4764 on 3 and 3930 DF,  p-value: &lt; 2.2e-16</code></pre>
<details class=chunk-details><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1507"><pre class="sourceCode r fold-hide"><code class="sourceCode r"><span id="cb1507-1"><a href="bias.html#cb1507-1" tabindex="-1"></a><span class="fu">plot</span>(</span>
<span id="cb1507-2"><a href="bias.html#cb1507-2" tabindex="-1"></a>  criterion4 <span class="sc">~</span> predictor,</span>
<span id="cb1507-3"><a href="bias.html#cb1507-3" tabindex="-1"></a>  <span class="at">data =</span> mydataBias,</span>
<span id="cb1507-4"><a href="bias.html#cb1507-4" tabindex="-1"></a>  <span class="at">xlim =</span> <span class="fu">c</span>(</span>
<span id="cb1507-5"><a href="bias.html#cb1507-5" tabindex="-1"></a>    <span class="dv">0</span>,</span>
<span id="cb1507-6"><a href="bias.html#cb1507-6" tabindex="-1"></a>    <span class="fu">max</span>(<span class="fu">c</span>(</span>
<span id="cb1507-7"><a href="bias.html#cb1507-7" tabindex="-1"></a>      mydataBias<span class="sc">$</span>criterion4,</span>
<span id="cb1507-8"><a href="bias.html#cb1507-8" tabindex="-1"></a>      mydataBias<span class="sc">$</span>predictor),</span>
<span id="cb1507-9"><a href="bias.html#cb1507-9" tabindex="-1"></a>      <span class="at">na.rm =</span> <span class="cn">TRUE</span>)),</span>
<span id="cb1507-10"><a href="bias.html#cb1507-10" tabindex="-1"></a>  <span class="at">ylim =</span> <span class="fu">c</span>(</span>
<span id="cb1507-11"><a href="bias.html#cb1507-11" tabindex="-1"></a>    <span class="dv">0</span>,</span>
<span id="cb1507-12"><a href="bias.html#cb1507-12" tabindex="-1"></a>    <span class="fu">max</span>(<span class="fu">c</span>(</span>
<span id="cb1507-13"><a href="bias.html#cb1507-13" tabindex="-1"></a>      mydataBias<span class="sc">$</span>criterion4,</span>
<span id="cb1507-14"><a href="bias.html#cb1507-14" tabindex="-1"></a>      mydataBias<span class="sc">$</span>predictor),</span>
<span id="cb1507-15"><a href="bias.html#cb1507-15" tabindex="-1"></a>      <span class="at">na.rm =</span> <span class="cn">TRUE</span>)),</span>
<span id="cb1507-16"><a href="bias.html#cb1507-16" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;n&quot;</span>,</span>
<span id="cb1507-17"><a href="bias.html#cb1507-17" tabindex="-1"></a>  <span class="at">xlab =</span> <span class="st">&quot;predictor&quot;</span>,</span>
<span id="cb1507-18"><a href="bias.html#cb1507-18" tabindex="-1"></a>  <span class="at">ylab =</span> <span class="st">&quot;criterion&quot;</span>)</span>
<span id="cb1507-19"><a href="bias.html#cb1507-19" tabindex="-1"></a><span class="fu">points</span>(</span>
<span id="cb1507-20"><a href="bias.html#cb1507-20" tabindex="-1"></a>  mydataBias<span class="sc">$</span>predictor[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)],</span>
<span id="cb1507-21"><a href="bias.html#cb1507-21" tabindex="-1"></a>  mydataBias<span class="sc">$</span>criterion4[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)],</span>
<span id="cb1507-22"><a href="bias.html#cb1507-22" tabindex="-1"></a>  <span class="at">pch =</span> <span class="dv">20</span>,</span>
<span id="cb1507-23"><a href="bias.html#cb1507-23" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;blue&quot;</span>)</span>
<span id="cb1507-24"><a href="bias.html#cb1507-24" tabindex="-1"></a><span class="fu">points</span>(</span>
<span id="cb1507-25"><a href="bias.html#cb1507-25" tabindex="-1"></a>  mydataBias<span class="sc">$</span>predictor[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)],</span>
<span id="cb1507-26"><a href="bias.html#cb1507-26" tabindex="-1"></a>  mydataBias<span class="sc">$</span>criterion4[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)],</span>
<span id="cb1507-27"><a href="bias.html#cb1507-27" tabindex="-1"></a>  <span class="at">pch =</span> <span class="dv">1</span>,</span>
<span id="cb1507-28"><a href="bias.html#cb1507-28" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;red&quot;</span>)</span>
<span id="cb1507-29"><a href="bias.html#cb1507-29" tabindex="-1"></a><span class="fu">abline</span>(<span class="fu">lm</span>(</span>
<span id="cb1507-30"><a href="bias.html#cb1507-30" tabindex="-1"></a>  criterion4 <span class="sc">~</span> predictor,</span>
<span id="cb1507-31"><a href="bias.html#cb1507-31" tabindex="-1"></a>  <span class="at">data =</span> mydataBias[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>),]),</span>
<span id="cb1507-32"><a href="bias.html#cb1507-32" tabindex="-1"></a>  <span class="at">lty =</span> <span class="dv">1</span>,</span>
<span id="cb1507-33"><a href="bias.html#cb1507-33" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;blue&quot;</span>)</span>
<span id="cb1507-34"><a href="bias.html#cb1507-34" tabindex="-1"></a><span class="fu">abline</span>(<span class="fu">lm</span>(</span>
<span id="cb1507-35"><a href="bias.html#cb1507-35" tabindex="-1"></a>  criterion4 <span class="sc">~</span> predictor,</span>
<span id="cb1507-36"><a href="bias.html#cb1507-36" tabindex="-1"></a>  <span class="at">data =</span> mydataBias[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>),]),</span>
<span id="cb1507-37"><a href="bias.html#cb1507-37" tabindex="-1"></a>  <span class="at">lty =</span> <span class="dv">2</span>,</span>
<span id="cb1507-38"><a href="bias.html#cb1507-38" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;red&quot;</span>)</span>
<span id="cb1507-39"><a href="bias.html#cb1507-39" tabindex="-1"></a><span class="fu">legend</span>(</span>
<span id="cb1507-40"><a href="bias.html#cb1507-40" tabindex="-1"></a>  <span class="st">&quot;bottomright&quot;</span>,</span>
<span id="cb1507-41"><a href="bias.html#cb1507-41" tabindex="-1"></a>  <span class="fu">c</span>(<span class="st">&quot;Male&quot;</span>,<span class="st">&quot;Female&quot;</span>),</span>
<span id="cb1507-42"><a href="bias.html#cb1507-42" tabindex="-1"></a>  <span class="at">lty =</span> <span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">2</span>),</span>
<span id="cb1507-43"><a href="bias.html#cb1507-43" tabindex="-1"></a>  <span class="at">pch =</span> <span class="fu">c</span>(<span class="dv">20</span>,<span class="dv">1</span>),</span>
<span id="cb1507-44"><a href="bias.html#cb1507-44" tabindex="-1"></a>  <span class="at">col =</span> <span class="fu">c</span>(<span class="st">&quot;blue&quot;</span>,<span class="st">&quot;red&quot;</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:interceptSlopeBias"></span>
<img src="Book_files/figure-html/interceptSlopeBias-1.png" alt="Example of Intercept And Slope Bias in Prediction (Different Intercepts and Slopes Between Males and Females)." width="100%" />
<p class="caption">
Figure 16.25: Example of Intercept And Slope Bias in Prediction (Different Intercepts and Slopes Between Males and Females).
</p>
</div>
</div>
<div id="differentialReliability" class="section level3 hasAnchor" number="16.8.5">
<h3><span class="header-section-number">16.8.5</span> Example of different measurement reliability/error across groups<a href="bias.html#differentialReliability" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>In the example depicted in Figure <a href="bias.html#fig:differentMeasurementError">16.26</a>, there are differences in the measurement <a href="reliability.html#reliability">reliability</a>/<a href="reliability.html#measurementError">error</a> on the criterion between males and females: males’ scores have a lower <a href="reliability.html#reliability">reliability</a> (higher <a href="reliability.html#measurementError">measurement error</a>) on the criterion than females’ scores.
That is, we are more confident about a female’s level on the criterion given a particular score on the predictor than we are about a male’s level on the criterion given a particular score on the predictor.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1508"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1508-1"><a href="bias.html#cb1508-1" tabindex="-1"></a><span class="fu">summary</span>(<span class="fu">lm</span>(</span>
<span id="cb1508-2"><a href="bias.html#cb1508-2" tabindex="-1"></a>  criterion5 <span class="sc">~</span> predictor <span class="sc">+</span> group <span class="sc">+</span> predictor<span class="sc">:</span>group,</span>
<span id="cb1508-3"><a href="bias.html#cb1508-3" tabindex="-1"></a>  <span class="at">data =</span> mydataBias))</span></code></pre></div>
</details>
<pre><code>
Call:
lm(formula = criterion5 ~ predictor + group + predictor:group, 
    data = mydataBias)

Residuals:
     Min       1Q   Median       3Q      Max 
-109.295   -7.065    0.077    6.836  108.428 

Coefficients:
                      Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)           -9.28386    3.32617  -2.791  0.00528 ** 
predictor              0.79172    0.03280  24.136  &lt; 2e-16 ***
groupfemale            9.12452    4.70470   1.939  0.05252 .  
predictor:groupfemale -0.09083    0.04654  -1.952  0.05102 .  
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 22.03 on 3920 degrees of freedom
  (76 observations deleted due to missingness)
Multiple R-squared:  0.2087,    Adjusted R-squared:  0.2081 
F-statistic: 344.6 on 3 and 3920 DF,  p-value: &lt; 2.2e-16</code></pre>
<details class=chunk-details><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1510"><pre class="sourceCode r fold-hide"><code class="sourceCode r"><span id="cb1510-1"><a href="bias.html#cb1510-1" tabindex="-1"></a><span class="fu">plot</span>(</span>
<span id="cb1510-2"><a href="bias.html#cb1510-2" tabindex="-1"></a>  criterion5 <span class="sc">~</span> predictor,</span>
<span id="cb1510-3"><a href="bias.html#cb1510-3" tabindex="-1"></a>  <span class="at">data =</span> mydataBias,</span>
<span id="cb1510-4"><a href="bias.html#cb1510-4" tabindex="-1"></a>  <span class="at">xlim =</span> <span class="fu">c</span>(</span>
<span id="cb1510-5"><a href="bias.html#cb1510-5" tabindex="-1"></a>    <span class="dv">0</span>,</span>
<span id="cb1510-6"><a href="bias.html#cb1510-6" tabindex="-1"></a>    <span class="fu">max</span>(<span class="fu">c</span>(mydataBias<span class="sc">$</span>criterion5,</span>
<span id="cb1510-7"><a href="bias.html#cb1510-7" tabindex="-1"></a>          mydataBias<span class="sc">$</span>predictor),</span>
<span id="cb1510-8"><a href="bias.html#cb1510-8" tabindex="-1"></a>        <span class="at">na.rm =</span> <span class="cn">TRUE</span>)),</span>
<span id="cb1510-9"><a href="bias.html#cb1510-9" tabindex="-1"></a>  <span class="at">ylim =</span> <span class="fu">c</span>(</span>
<span id="cb1510-10"><a href="bias.html#cb1510-10" tabindex="-1"></a>    <span class="dv">0</span>, </span>
<span id="cb1510-11"><a href="bias.html#cb1510-11" tabindex="-1"></a>    <span class="fu">max</span>(<span class="fu">c</span>(</span>
<span id="cb1510-12"><a href="bias.html#cb1510-12" tabindex="-1"></a>      mydataBias<span class="sc">$</span>criterion5,</span>
<span id="cb1510-13"><a href="bias.html#cb1510-13" tabindex="-1"></a>      mydataBias<span class="sc">$</span>predictor),</span>
<span id="cb1510-14"><a href="bias.html#cb1510-14" tabindex="-1"></a>      <span class="at">na.rm =</span> <span class="cn">TRUE</span>)),</span>
<span id="cb1510-15"><a href="bias.html#cb1510-15" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;n&quot;</span>,</span>
<span id="cb1510-16"><a href="bias.html#cb1510-16" tabindex="-1"></a>  <span class="at">xlab =</span> <span class="st">&quot;predictor&quot;</span>,</span>
<span id="cb1510-17"><a href="bias.html#cb1510-17" tabindex="-1"></a>  <span class="at">ylab =</span> <span class="st">&quot;criterion&quot;</span>)</span>
<span id="cb1510-18"><a href="bias.html#cb1510-18" tabindex="-1"></a><span class="fu">points</span>(</span>
<span id="cb1510-19"><a href="bias.html#cb1510-19" tabindex="-1"></a>  mydataBias<span class="sc">$</span>predictor[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)],</span>
<span id="cb1510-20"><a href="bias.html#cb1510-20" tabindex="-1"></a>  mydataBias<span class="sc">$</span>criterion5[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>)],</span>
<span id="cb1510-21"><a href="bias.html#cb1510-21" tabindex="-1"></a>  <span class="at">pch =</span> <span class="dv">20</span>,</span>
<span id="cb1510-22"><a href="bias.html#cb1510-22" tabindex="-1"></a>  <span class="at">col =</span> <span class="st">&quot;blue&quot;</span>)</span>
<span id="cb1510-23"><a href="bias.html#cb1510-23" tabindex="-1"></a><span class="fu">points</span>(mydataBias<span class="sc">$</span>predictor[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)],</span>
<span id="cb1510-24"><a href="bias.html#cb1510-24" tabindex="-1"></a>       mydataBias<span class="sc">$</span>criterion5[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>)],</span>
<span id="cb1510-25"><a href="bias.html#cb1510-25" tabindex="-1"></a>       <span class="at">pch =</span> <span class="dv">1</span>,</span>
<span id="cb1510-26"><a href="bias.html#cb1510-26" tabindex="-1"></a>       <span class="at">col =</span> <span class="st">&quot;red&quot;</span>)</span>
<span id="cb1510-27"><a href="bias.html#cb1510-27" tabindex="-1"></a><span class="fu">abline</span>(<span class="fu">lm</span>(criterion5 <span class="sc">~</span> predictor,</span>
<span id="cb1510-28"><a href="bias.html#cb1510-28" tabindex="-1"></a>          <span class="at">data =</span> mydataBias[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;male&quot;</span>),]),</span>
<span id="cb1510-29"><a href="bias.html#cb1510-29" tabindex="-1"></a>       <span class="at">lty =</span> <span class="dv">1</span>,</span>
<span id="cb1510-30"><a href="bias.html#cb1510-30" tabindex="-1"></a>       <span class="at">col =</span> <span class="st">&quot;blue&quot;</span>)</span>
<span id="cb1510-31"><a href="bias.html#cb1510-31" tabindex="-1"></a><span class="fu">abline</span>(<span class="fu">lm</span>(criterion5 <span class="sc">~</span> predictor,</span>
<span id="cb1510-32"><a href="bias.html#cb1510-32" tabindex="-1"></a>          <span class="at">data =</span> mydataBias[<span class="fu">which</span>(mydataBias<span class="sc">$</span>group <span class="sc">==</span> <span class="st">&quot;female&quot;</span>),]),</span>
<span id="cb1510-33"><a href="bias.html#cb1510-33" tabindex="-1"></a>       <span class="at">lty =</span> <span class="dv">2</span>,</span>
<span id="cb1510-34"><a href="bias.html#cb1510-34" tabindex="-1"></a>       <span class="at">col =</span> <span class="st">&quot;red&quot;</span>)</span>
<span id="cb1510-35"><a href="bias.html#cb1510-35" tabindex="-1"></a><span class="fu">legend</span>(</span>
<span id="cb1510-36"><a href="bias.html#cb1510-36" tabindex="-1"></a>  <span class="st">&quot;bottomright&quot;</span>,</span>
<span id="cb1510-37"><a href="bias.html#cb1510-37" tabindex="-1"></a>  <span class="fu">c</span>(<span class="st">&quot;Male&quot;</span>,<span class="st">&quot;Female&quot;</span>),</span>
<span id="cb1510-38"><a href="bias.html#cb1510-38" tabindex="-1"></a>  <span class="at">lty =</span> <span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">2</span>),</span>
<span id="cb1510-39"><a href="bias.html#cb1510-39" tabindex="-1"></a>  <span class="at">pch =</span> <span class="fu">c</span>(<span class="dv">20</span>,<span class="dv">1</span>),</span>
<span id="cb1510-40"><a href="bias.html#cb1510-40" tabindex="-1"></a>  <span class="at">col =</span> <span class="fu">c</span>(<span class="st">&quot;blue&quot;</span>,<span class="st">&quot;red&quot;</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:differentMeasurementError"></span>
<img src="Book_files/figure-html/differentMeasurementError-1.png" alt="Example of Different Measurement Reliability/Error Across Groups." width="100%" />
<p class="caption">
Figure 16.26: Example of Different Measurement Reliability/Error Across Groups.
</p>
</div>
</div>
</div>
<div id="dif" class="section level2 hasAnchor" number="16.9">
<h2><span class="header-section-number">16.9</span> Differential Item Functioning (DIF)<a href="bias.html#dif" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Differential item functioning (DIF) indicates that one or more items functions differently across groups.
That is, one or more of the item parameters for an item differs across groups.
For instance, the <a href="irt.html#itemDifficulty">severity</a> or <a href="irt.html#itemDiscrimination">discrimination</a> of an item could be higher in one group compared to another group.
If an item functions differently across groups, it can lead to biased scores for particular groups.
Thus, when observing group differences in level on the measure or group differences in the measure’s association with other measures, it is unclear whether the observed group differences reflect true group differences or differences in the functioning of the measure across groups.</p>
<p>For instance, consider an item such as “disobedience to authority”, which is thought to reflect externalizing behavior in childhood.
However, in adulthood, disobedience to authority could reflect prosocial functions, such as protesting against societally unjust actions, and may show weaker <a href="validity.html#constructValidity">construct validity</a> with respect to externalizing problems, as operationalized by a weaker <a href="irt.html#itemDiscrimination">discrimination</a> coefficient in adulthood than in childhood.
Tests of differential item functioning are equivalent to tests of <a href="bias.html#measurementInvariance">measurement invariance</a>.</p>
<p>Approaches to addressing DIF are described in Section <a href="bias.html#whenDetectingBias">16.5.1</a>.
If we identify that an item shows non-negligible DIF, we have three primary options (described above): (1) drop the item for both groups, (2) drop the item for one group but keep it for the other group, or (3) freely estimate the parameters (<a href="irt.html#itemDiscrimination">discrimination</a> and <a href="irt.html#itemDifficulty">difficulty</a>) for the item across groups.</p>
<p>Tests of DIF were conducted using the <code>mirt</code> package <span class="citation">(<a href="#ref-R-mirt">Chalmers, 2020</a>)</span>.</p>
<div id="descriptiveStats-bias" class="section level3 hasAnchor" number="16.9.1">
<h3><span class="header-section-number">16.9.1</span> Item Descriptive Statistics<a href="bias.html#descriptiveStats-bias" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1511"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1511-1"><a href="bias.html#cb1511-1" tabindex="-1"></a><span class="fu">itemstats</span>(</span>
<span id="cb1511-2"><a href="bias.html#cb1511-2" tabindex="-1"></a>  <span class="at">data =</span> cnlsy[,<span class="fu">c</span>(</span>
<span id="cb1511-3"><a href="bias.html#cb1511-3" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_1&quot;</span>,<span class="st">&quot;bpi_antisocialT1_2&quot;</span>,<span class="st">&quot;bpi_antisocialT1_3&quot;</span>,</span>
<span id="cb1511-4"><a href="bias.html#cb1511-4" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_4&quot;</span>,<span class="st">&quot;bpi_antisocialT1_5&quot;</span>,<span class="st">&quot;bpi_antisocialT1_6&quot;</span>,</span>
<span id="cb1511-5"><a href="bias.html#cb1511-5" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_7&quot;</span>)],</span>
<span id="cb1511-6"><a href="bias.html#cb1511-6" tabindex="-1"></a>  <span class="at">group =</span> cnlsy<span class="sc">$</span>sex,</span>
<span id="cb1511-7"><a href="bias.html#cb1511-7" tabindex="-1"></a>  <span class="at">ts.tables =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
<pre><code>$female
$female$overall
 N.complete    N mean_total.score sd_total.score ave.r  sd.r alpha SEM.alpha
       1620 5641            2.554          2.132 0.255 0.083 0.681     1.203

$female$itemstats
                      N  mean    sd total.r total.r_if_rm alpha_if_rm
bpi_antisocialT1_1 1928 0.538 0.587   0.662         0.456       0.626
bpi_antisocialT1_2 1925 0.278 0.510   0.648         0.471       0.624
bpi_antisocialT1_3 1925 0.456 0.654   0.625         0.376       0.657
bpi_antisocialT1_4 1933 0.113 0.355   0.523         0.384       0.654
bpi_antisocialT1_5 1649 0.186 0.431   0.594         0.438       0.637
bpi_antisocialT1_6 1658 0.103 0.345   0.572         0.446       0.643
bpi_antisocialT1_7 1944 0.834 0.639   0.553         0.292       0.683

$female$proportions
                       0     1     2    NA
bpi_antisocialT1_1 0.174 0.151 0.016 0.658
bpi_antisocialT1_2 0.257 0.075 0.010 0.659
bpi_antisocialT1_3 0.216 0.094 0.031 0.659
bpi_antisocialT1_4 0.308 0.030 0.004 0.657
bpi_antisocialT1_5 0.243 0.044 0.005 0.708
bpi_antisocialT1_6 0.268 0.023 0.004 0.706
bpi_antisocialT1_7 0.104 0.194 0.046 0.655

$female$total.score_frequency
       0   1   2   3   4  5  6  7  8  9 10 11 12 13
Freq 217 364 349 283 166 91 67 38 12 10 11  4  4  4

$female$total.score_means
                          0        1        2
bpi_antisocialT1_1 1.354919 3.339237 7.216867
bpi_antisocialT1_2 1.846343 4.173789 8.192308
bpi_antisocialT1_3 1.593417 3.846682 5.406667
bpi_antisocialT1_4 2.204952 5.090278 9.045455
bpi_antisocialT1_5 2.010386 4.946721 7.892857
bpi_antisocialT1_6 2.189959 5.701613 9.227273
bpi_antisocialT1_7 1.004357 2.778970 4.746725

$female$total.score_sds
                          0        1        2
bpi_antisocialT1_1 1.262270 1.668474 2.763209
bpi_antisocialT1_2 1.480206 1.809814 2.664498
bpi_antisocialT1_3 1.329902 1.837884 2.783211
bpi_antisocialT1_4 1.743521 2.280698 2.802673
bpi_antisocialT1_5 1.566126 2.196431 3.258485
bpi_antisocialT1_6 1.684322 2.215906 2.844072
bpi_antisocialT1_7 1.232733 1.798814 2.475591


$male
$male$overall
 N.complete    N mean_total.score sd_total.score ave.r  sd.r alpha SEM.alpha
       1645 5891             3.16          2.385 0.266 0.081 0.703       1.3

$male$itemstats
                      N  mean    sd total.r total.r_if_rm alpha_if_rm
bpi_antisocialT1_1 1948 0.615 0.585   0.634         0.449       0.660
bpi_antisocialT1_2 1947 0.372 0.559   0.655         0.485       0.650
bpi_antisocialT1_3 1949 0.520 0.668   0.582         0.346       0.692
bpi_antisocialT1_4 1948 0.233 0.486   0.596         0.441       0.666
bpi_antisocialT1_5 1676 0.367 0.547   0.627         0.454       0.662
bpi_antisocialT1_6 1688 0.177 0.446   0.556         0.406       0.675
bpi_antisocialT1_7 1962 0.834 0.646   0.582         0.357       0.687

$male$proportions
                       0     1     2    NA
bpi_antisocialT1_1 0.145 0.168 0.017 0.669
bpi_antisocialT1_2 0.220 0.097 0.013 0.669
bpi_antisocialT1_3 0.191 0.107 0.033 0.669
bpi_antisocialT1_4 0.263 0.058 0.010 0.669
bpi_antisocialT1_5 0.190 0.085 0.010 0.715
bpi_antisocialT1_6 0.243 0.035 0.008 0.713
bpi_antisocialT1_7 0.102 0.185 0.046 0.667

$male$total.score_frequency
       0   1   2   3   4   5   6  7  8  9 10 11 12 13 14
Freq 163 294 310 259 208 144 104 74 40 25  9  7  5  1  2

$male$total.score_means
                          0        1        2
bpi_antisocialT1_1 1.622478 3.944056 7.397849
bpi_antisocialT1_2 2.134191 4.767821 8.106061
bpi_antisocialT1_3 2.016754 4.400763 5.819277
bpi_antisocialT1_4 2.484091 5.539286 8.177778
bpi_antisocialT1_5 2.174111 4.817073 7.910714
bpi_antisocialT1_6 2.616762 5.810680 8.093023
bpi_antisocialT1_7 1.412500 3.375679 5.782787

$male$total.score_sds
                          0        1        2
bpi_antisocialT1_1 1.515237 1.982218 2.454542
bpi_antisocialT1_2 1.678482 1.955386 2.437709
bpi_antisocialT1_3 1.654589 2.069463 2.774973
bpi_antisocialT1_4 1.845772 2.119687 2.534210
bpi_antisocialT1_5 1.709715 2.047072 2.725481
bpi_antisocialT1_6 1.911288 2.181537 2.982600
bpi_antisocialT1_7 1.522500 1.906515 2.652344</code></pre>
</div>
<div id="difUnconstrained" class="section level3 hasAnchor" number="16.9.2">
<h3><span class="header-section-number">16.9.2</span> Unconstrained Model<a href="bias.html#difUnconstrained" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="difUnconstrainedFit" class="section level4 hasAnchor" number="16.9.2.1">
<h4><span class="header-section-number">16.9.2.1</span> Fit Model<a href="bias.html#difUnconstrainedFit" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1513"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1513-1"><a href="bias.html#cb1513-1" tabindex="-1"></a>unconstrainedModel <span class="ot">&lt;-</span> <span class="fu">multipleGroup</span>(</span>
<span id="cb1513-2"><a href="bias.html#cb1513-2" tabindex="-1"></a>  <span class="at">data =</span> cnlsy[,<span class="fu">c</span>(</span>
<span id="cb1513-3"><a href="bias.html#cb1513-3" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_1&quot;</span>,<span class="st">&quot;bpi_antisocialT1_2&quot;</span>,<span class="st">&quot;bpi_antisocialT1_3&quot;</span>,</span>
<span id="cb1513-4"><a href="bias.html#cb1513-4" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_4&quot;</span>,<span class="st">&quot;bpi_antisocialT1_5&quot;</span>,<span class="st">&quot;bpi_antisocialT1_6&quot;</span>,</span>
<span id="cb1513-5"><a href="bias.html#cb1513-5" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_7&quot;</span>)],</span>
<span id="cb1513-6"><a href="bias.html#cb1513-6" tabindex="-1"></a>  <span class="at">model =</span> <span class="dv">1</span>,</span>
<span id="cb1513-7"><a href="bias.html#cb1513-7" tabindex="-1"></a>  <span class="at">group =</span> cnlsy<span class="sc">$</span>sex,</span>
<span id="cb1513-8"><a href="bias.html#cb1513-8" tabindex="-1"></a>  <span class="at">SE =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
</div>
<div id="difUnconstrainedSummary" class="section level4 hasAnchor" number="16.9.2.2">
<h4><span class="header-section-number">16.9.2.2</span> Model Summary<a href="bias.html#difUnconstrainedSummary" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Items that appear to differ:</p>
<ul>
<li>items 5 and 6 are more discriminating (<span class="math inline">\(a\)</span> parameter) among females than males</li>
<li>item 4 has higher <a href="irt.html#itemDifficulty">difficulty/severity</a> (<span class="math inline">\(b_1\)</span> and <span class="math inline">\(b_2\)</span> parameters) among males than females</li>
</ul>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1514"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1514-1"><a href="bias.html#cb1514-1" tabindex="-1"></a><span class="fu">summary</span>(unconstrainedModel)</span></code></pre></div>
</details>
<pre><code>
----------
GROUP: female 
                      F1    h2
bpi_antisocialT1_1 0.698 0.487
bpi_antisocialT1_2 0.706 0.499
bpi_antisocialT1_3 0.532 0.283
bpi_antisocialT1_4 0.683 0.466
bpi_antisocialT1_5 0.763 0.582
bpi_antisocialT1_6 0.838 0.702
bpi_antisocialT1_7 0.429 0.184

SS loadings:  3.202 
Proportion Var:  0.457 

Factor correlations: 

   F1
F1  1

----------
GROUP: male 
                      F1    h2
bpi_antisocialT1_1 0.663 0.440
bpi_antisocialT1_2 0.736 0.542
bpi_antisocialT1_3 0.531 0.282
bpi_antisocialT1_4 0.713 0.508
bpi_antisocialT1_5 0.691 0.477
bpi_antisocialT1_6 0.702 0.492
bpi_antisocialT1_7 0.507 0.257

SS loadings:  2.999 
Proportion Var:  0.428 

Factor correlations: 

   F1
F1  1</code></pre>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1516"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1516-1"><a href="bias.html#cb1516-1" tabindex="-1"></a><span class="fu">coef</span>(unconstrainedModel,</span>
<span id="cb1516-2"><a href="bias.html#cb1516-2" tabindex="-1"></a>     <span class="at">simplify =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1516-3"><a href="bias.html#cb1516-3" tabindex="-1"></a>     <span class="at">IRTpars =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
<pre><code>$female
$items
                       a     b1    b2
bpi_antisocialT1_1 1.657  0.029 2.451
bpi_antisocialT1_2 1.697  0.953 2.763
bpi_antisocialT1_3 1.069  0.650 2.585
bpi_antisocialT1_4 1.590  1.895 3.465
bpi_antisocialT1_5 2.008  1.281 2.936
bpi_antisocialT1_6 2.614  1.625 2.770
bpi_antisocialT1_7 0.807 -1.184 2.583

$means
F1 
 0 

$cov
   F1
F1  1


$male
$items
                       a     b1    b2
bpi_antisocialT1_1 1.507 -0.238 2.505
bpi_antisocialT1_2 1.853  0.582 2.455
bpi_antisocialT1_3 1.067  0.384 2.480
bpi_antisocialT1_4 1.729  1.165 2.765
bpi_antisocialT1_5 1.625  0.635 2.757
bpi_antisocialT1_6 1.677  1.494 2.873
bpi_antisocialT1_7 1.002 -0.986 2.143

$means
F1 
 0 

$cov
   F1
F1  1</code></pre>
</div>
</div>
<div id="difConstrained" class="section level3 hasAnchor" number="16.9.3">
<h3><span class="header-section-number">16.9.3</span> Constrained Model<a href="bias.html#difConstrained" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Constrain item parameters to be equal across groups (to use as baseline model for identifying <a href="bias.html#dif">DIF</a>).</p>
<div id="difConstrainedFit" class="section level4 hasAnchor" number="16.9.3.1">
<h4><span class="header-section-number">16.9.3.1</span> Fit Model<a href="bias.html#difConstrainedFit" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1518"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1518-1"><a href="bias.html#cb1518-1" tabindex="-1"></a>constrainedModel <span class="ot">&lt;-</span> <span class="fu">multipleGroup</span>(</span>
<span id="cb1518-2"><a href="bias.html#cb1518-2" tabindex="-1"></a>  <span class="at">data =</span> cnlsy[,<span class="fu">c</span>(</span>
<span id="cb1518-3"><a href="bias.html#cb1518-3" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_1&quot;</span>,<span class="st">&quot;bpi_antisocialT1_2&quot;</span>,<span class="st">&quot;bpi_antisocialT1_3&quot;</span>,</span>
<span id="cb1518-4"><a href="bias.html#cb1518-4" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_4&quot;</span>,<span class="st">&quot;bpi_antisocialT1_5&quot;</span>,<span class="st">&quot;bpi_antisocialT1_6&quot;</span>,</span>
<span id="cb1518-5"><a href="bias.html#cb1518-5" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_7&quot;</span>)],</span>
<span id="cb1518-6"><a href="bias.html#cb1518-6" tabindex="-1"></a>  <span class="at">model =</span> <span class="dv">1</span>,</span>
<span id="cb1518-7"><a href="bias.html#cb1518-7" tabindex="-1"></a>  <span class="at">group =</span> cnlsy<span class="sc">$</span>sex,</span>
<span id="cb1518-8"><a href="bias.html#cb1518-8" tabindex="-1"></a>  <span class="at">invariance =</span> <span class="fu">c</span>(<span class="fu">c</span>(</span>
<span id="cb1518-9"><a href="bias.html#cb1518-9" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_1&quot;</span>,<span class="st">&quot;bpi_antisocialT1_2&quot;</span>,<span class="st">&quot;bpi_antisocialT1_3&quot;</span>,</span>
<span id="cb1518-10"><a href="bias.html#cb1518-10" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_4&quot;</span>,<span class="st">&quot;bpi_antisocialT1_5&quot;</span>,<span class="st">&quot;bpi_antisocialT1_6&quot;</span>,</span>
<span id="cb1518-11"><a href="bias.html#cb1518-11" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_7&quot;</span>),</span>
<span id="cb1518-12"><a href="bias.html#cb1518-12" tabindex="-1"></a>    <span class="st">&quot;free_means&quot;</span>, <span class="st">&quot;free_var&quot;</span>),</span>
<span id="cb1518-13"><a href="bias.html#cb1518-13" tabindex="-1"></a>  <span class="at">SE =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
</div>
<div id="difConstrainedSummary" class="section level4 hasAnchor" number="16.9.3.2">
<h4><span class="header-section-number">16.9.3.2</span> Model Summary<a href="bias.html#difConstrainedSummary" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1519"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1519-1"><a href="bias.html#cb1519-1" tabindex="-1"></a><span class="fu">summary</span>(constrainedModel)</span></code></pre></div>
</details>
<pre><code>
----------
GROUP: female 
                      F1    h2
bpi_antisocialT1_1 0.657 0.431
bpi_antisocialT1_2 0.707 0.500
bpi_antisocialT1_3 0.512 0.262
bpi_antisocialT1_4 0.701 0.491
bpi_antisocialT1_5 0.733 0.538
bpi_antisocialT1_6 0.765 0.585
bpi_antisocialT1_7 0.440 0.193

SS loadings:  3 
Proportion Var:  0.429 

Factor correlations: 

   F1
F1  1

----------
GROUP: male 
                      F1    h2
bpi_antisocialT1_1 0.672 0.452
bpi_antisocialT1_2 0.722 0.521
bpi_antisocialT1_3 0.527 0.278
bpi_antisocialT1_4 0.716 0.512
bpi_antisocialT1_5 0.747 0.559
bpi_antisocialT1_6 0.778 0.605
bpi_antisocialT1_7 0.455 0.207

SS loadings:  3.134 
Proportion Var:  0.448 

Factor correlations: 

   F1
F1  1</code></pre>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1521"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1521-1"><a href="bias.html#cb1521-1" tabindex="-1"></a><span class="fu">coef</span>(</span>
<span id="cb1521-2"><a href="bias.html#cb1521-2" tabindex="-1"></a>  constrainedModel,</span>
<span id="cb1521-3"><a href="bias.html#cb1521-3" tabindex="-1"></a>  <span class="at">simplify =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1521-4"><a href="bias.html#cb1521-4" tabindex="-1"></a>  <span class="at">IRTpars =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
<pre><code>$female
$items
                       a     b1    b2
bpi_antisocialT1_1 1.482  0.078 2.798
bpi_antisocialT1_2 1.701  0.973 2.892
bpi_antisocialT1_3 1.013  0.726 2.844
bpi_antisocialT1_4 1.673  1.688 3.278
bpi_antisocialT1_5 1.836  1.135 3.028
bpi_antisocialT1_6 2.020  1.782 3.048
bpi_antisocialT1_7 0.834 -0.967 2.697

$means
F1 
 0 

$cov
   F1
F1  1


$male
$items
                       a     b1    b2
bpi_antisocialT1_1 1.482  0.078 2.798
bpi_antisocialT1_2 1.701  0.973 2.892
bpi_antisocialT1_3 1.013  0.726 2.844
bpi_antisocialT1_4 1.673  1.688 3.278
bpi_antisocialT1_5 1.836  1.135 3.028
bpi_antisocialT1_6 2.020  1.782 3.048
bpi_antisocialT1_7 0.834 -0.967 2.697

$means
  F1 
0.37 

$cov
      F1
F1 1.088</code></pre>
</div>
</div>
<div id="difCompareConstrainedUnconstrained" class="section level3 hasAnchor" number="16.9.4">
<h3><span class="header-section-number">16.9.4</span> Compare model fit of constrained model to unconstrained model<a href="bias.html#difCompareConstrainedUnconstrained" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The constrained model and the unconstrained model are considered “nested” models.
The constrained model is nested within the unconstrained model because the unconstrained model includes all of the terms of the constrained model along with additional terms.
Model fit of nested models can be compared with a chi-square difference test.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1523"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1523-1"><a href="bias.html#cb1523-1" tabindex="-1"></a><span class="fu">anova</span>(constrainedModel, unconstrainedModel)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["AIC"],"name":[1],"type":["dbl"],"align":["right"]},{"label":["SABIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["HQ"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["logLik"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["X2"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["df"],"name":[7],"type":["dbl"],"align":["right"]},{"label":["p"],"name":[8],"type":["dbl"],"align":["right"]}],"data":[{"1":"34331.52","2":"34402.71","3":"34382.72","4":"34475.80","5":"-17142.76","6":"NA","7":"NA","8":"NA","_rn_":"constrainedModel"},{"1":"34247.21","2":"34377.22","3":"34340.70","4":"34510.68","5":"-17081.61","6":"122.3052","7":"19","8":"0","_rn_":"unconstrainedModel"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>The constrained model fits significantly worse than the unconstrained model, which suggests that item parameters differ between males and females.</p>
</div>
<div id="difRemoveConstraints" class="section level3 hasAnchor" number="16.9.5">
<h3><span class="header-section-number">16.9.5</span> Identify DIF by iteratively removing constraints from fully constrained model<a href="bias.html#difRemoveConstraints" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>One way to identify DIF is to iteratively remove constraints from a model in which all parameters are constrained to be the same across groups.
Removing constraints allows individual items to have different item parameters across groups, to identify which items yield a significant improvement in model fit when allowing their <a href="irt.html#itemDiscrimination">discrimination</a> and/or <a href="irt.html#itemDifficulty">severity</a> to differ across groups.
Items 1, 3, 4, 5, 6, and 7 showed <a href="bias.html#dif">DIF</a> in <a href="irt.html#itemDiscrimination">discrimination</a> and/or <a href="irt.html#itemDifficulty">severity</a>, based on a significant chi-square difference test.</p>
<div id="difRemoveConstraintsDiscriminationSeverity" class="section level4 hasAnchor" number="16.9.5.1">
<h4><span class="header-section-number">16.9.5.1</span> Items that differ in discrimination and/or severity<a href="bias.html#difRemoveConstraintsDiscriminationSeverity" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p></p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1524"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1524-1"><a href="bias.html#cb1524-1" tabindex="-1"></a>difDropItems <span class="ot">&lt;-</span> <span class="fu">DIF</span>(</span>
<span id="cb1524-2"><a href="bias.html#cb1524-2" tabindex="-1"></a>  constrainedModel,</span>
<span id="cb1524-3"><a href="bias.html#cb1524-3" tabindex="-1"></a>  <span class="fu">c</span>(<span class="st">&quot;a1&quot;</span>,<span class="st">&quot;d1&quot;</span>,<span class="st">&quot;d2&quot;</span>),</span>
<span id="cb1524-4"><a href="bias.html#cb1524-4" tabindex="-1"></a>  <span class="at">scheme =</span> <span class="st">&quot;drop&quot;</span>,</span>
<span id="cb1524-5"><a href="bias.html#cb1524-5" tabindex="-1"></a>  <span class="at">simplify =</span> <span class="cn">TRUE</span>)</span>
<span id="cb1524-6"><a href="bias.html#cb1524-6" tabindex="-1"></a></span>
<span id="cb1524-7"><a href="bias.html#cb1524-7" tabindex="-1"></a>difDropItems</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["groups"],"name":[1],"type":["chr"],"align":["left"]},{"label":["converged"],"name":[2],"type":["lgl"],"align":["right"]},{"label":["AIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["SABIC"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["HQ"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["X2"],"name":[7],"type":["dbl"],"align":["right"]},{"label":["df"],"name":[8],"type":["dbl"],"align":["right"]},{"label":["p"],"name":[9],"type":["dbl"],"align":["right"]}],"data":[{"1":"female,male","2":"TRUE","3":"-12.009453","4":"-2.722839","5":"-5.331411","6":"6.809791","7":"18.009453","8":"3","9":"4.378795e-04","_rn_":"bpi_antisocialT1_1"},{"1":"female,male","2":"TRUE","3":"1.837879","4":"11.124493","5":"8.515922","6":"20.657123","7":"4.162121","8":"3","9":"2.444818e-01","_rn_":"bpi_antisocialT1_2"},{"1":"female,male","2":"TRUE","3":"-1.820107","4":"7.466507","5":"4.857935","6":"16.999137","7":"7.820107","8":"3","9":"4.987960e-02","_rn_":"bpi_antisocialT1_3"},{"1":"female,male","2":"TRUE","3":"-14.245749","4":"-4.959135","5":"-7.567707","6":"4.573495","7":"20.245749","8":"3","9":"1.509551e-04","_rn_":"bpi_antisocialT1_4"},{"1":"female,male","2":"TRUE","3":"-40.238487","4":"-30.951873","5":"-33.560445","6":"-21.419243","7":"46.238487","8":"3","9":"5.046464e-10","_rn_":"bpi_antisocialT1_5"},{"1":"female,male","2":"TRUE","3":"-4.364028","4":"4.922586","5":"2.314014","6":"14.455216","7":"10.364028","8":"3","9":"1.571222e-02","_rn_":"bpi_antisocialT1_6"},{"1":"female,male","2":"TRUE","3":"-24.727410","4":"-15.440796","5":"-18.049367","6":"-5.908166","7":"30.727410","8":"3","9":"9.701335e-07","_rn_":"bpi_antisocialT1_7"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="difRemoveConstraintsDiscrimination" class="section level4 hasAnchor" number="16.9.5.2">
<h4><span class="header-section-number">16.9.5.2</span> Items that differ in discrimination<a href="bias.html#difRemoveConstraintsDiscrimination" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Items 1, 3, 4, and 5 showed <a href="bias.html#dif">DIF</a> in <a href="irt.html#itemDiscrimination">discrimination</a>, based on a significant chi-square difference test.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1525"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1525-1"><a href="bias.html#cb1525-1" tabindex="-1"></a>difDropItemsDiscrimination <span class="ot">&lt;-</span> <span class="fu">DIF</span>(</span>
<span id="cb1525-2"><a href="bias.html#cb1525-2" tabindex="-1"></a>  constrainedModel,</span>
<span id="cb1525-3"><a href="bias.html#cb1525-3" tabindex="-1"></a>  <span class="fu">c</span>(<span class="st">&quot;a1&quot;</span>),</span>
<span id="cb1525-4"><a href="bias.html#cb1525-4" tabindex="-1"></a>  <span class="at">scheme =</span> <span class="st">&quot;drop&quot;</span>,</span>
<span id="cb1525-5"><a href="bias.html#cb1525-5" tabindex="-1"></a>  <span class="at">simplify =</span> <span class="cn">TRUE</span>)</span>
<span id="cb1525-6"><a href="bias.html#cb1525-6" tabindex="-1"></a></span>
<span id="cb1525-7"><a href="bias.html#cb1525-7" tabindex="-1"></a>difDropItemsDiscrimination</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["groups"],"name":[1],"type":["chr"],"align":["left"]},{"label":["converged"],"name":[2],"type":["lgl"],"align":["right"]},{"label":["AIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["SABIC"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["HQ"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["X2"],"name":[7],"type":["dbl"],"align":["right"]},{"label":["df"],"name":[8],"type":["dbl"],"align":["right"]},{"label":["p"],"name":[9],"type":["dbl"],"align":["right"]}],"data":[{"1":"female,male","2":"TRUE","3":"-13.1370197","4":"-10.0414818","5":"-10.9110057","6":"-6.863938","7":"15.13701974","8":"1","9":"9.998334e-05","_rn_":"bpi_antisocialT1_1"},{"1":"female,male","2":"TRUE","3":"-1.4195676","4":"1.6759704","5":"0.8064464","6":"4.853514","7":"3.41956761","8":"1","9":"6.442789e-02","_rn_":"bpi_antisocialT1_2"},{"1":"female,male","2":"TRUE","3":"-3.5683333","4":"-0.4727953","5":"-1.3423193","6":"2.704748","7":"5.56833331","8":"1","9":"1.828816e-02","_rn_":"bpi_antisocialT1_3"},{"1":"female,male","2":"TRUE","3":"-12.0421738","4":"-8.9466359","5":"-9.8161598","6":"-5.769092","7":"14.04217383","8":"1","9":"1.787562e-04","_rn_":"bpi_antisocialT1_4"},{"1":"female,male","2":"TRUE","3":"-14.2566881","4":"-11.1611501","5":"-12.0306740","6":"-7.983607","7":"16.25668806","8":"1","9":"5.531399e-05","_rn_":"bpi_antisocialT1_5"},{"1":"female,male","2":"TRUE","3":"0.8132294","4":"3.9087674","5":"3.0392435","6":"7.086311","7":"1.18677061","8":"1","9":"2.759819e-01","_rn_":"bpi_antisocialT1_6"},{"1":"female,male","2":"TRUE","3":"1.9620909","4":"5.0576288","5":"4.1881049","6":"8.235172","7":"0.03790914","8":"1","9":"8.456257e-01","_rn_":"bpi_antisocialT1_7"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="difRemoveConstraintsSeverity" class="section level4 hasAnchor" number="16.9.5.3">
<h4><span class="header-section-number">16.9.5.3</span> Items that differ in severity<a href="bias.html#difRemoveConstraintsSeverity" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Items 1, 3, 4, 5, and 7 showed <a href="bias.html#dif">DIF</a> in <a href="irt.html#itemDifficulty">difficulty/severity</a>, based on a significant chi-square difference test.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1526"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1526-1"><a href="bias.html#cb1526-1" tabindex="-1"></a>difDropItemsSeverity <span class="ot">&lt;-</span> <span class="fu">DIF</span>(</span>
<span id="cb1526-2"><a href="bias.html#cb1526-2" tabindex="-1"></a>  constrainedModel,</span>
<span id="cb1526-3"><a href="bias.html#cb1526-3" tabindex="-1"></a>  <span class="fu">c</span>(<span class="st">&quot;d1&quot;</span>,<span class="st">&quot;d2&quot;</span>),</span>
<span id="cb1526-4"><a href="bias.html#cb1526-4" tabindex="-1"></a>  <span class="at">scheme =</span> <span class="st">&quot;drop&quot;</span>,</span>
<span id="cb1526-5"><a href="bias.html#cb1526-5" tabindex="-1"></a>  <span class="at">simplify =</span> <span class="cn">TRUE</span>)</span>
<span id="cb1526-6"><a href="bias.html#cb1526-6" tabindex="-1"></a></span>
<span id="cb1526-7"><a href="bias.html#cb1526-7" tabindex="-1"></a>difDropItemsSeverity</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["groups"],"name":[1],"type":["chr"],"align":["left"]},{"label":["converged"],"name":[2],"type":["lgl"],"align":["right"]},{"label":["AIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["SABIC"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["HQ"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["X2"],"name":[7],"type":["dbl"],"align":["right"]},{"label":["df"],"name":[8],"type":["dbl"],"align":["right"]},{"label":["p"],"name":[9],"type":["dbl"],"align":["right"]}],"data":[{"1":"female,male","2":"TRUE","3":"-10.10127703","4":"-3.910201","5":"-5.649249","6":"2.444886","7":"14.1012770","8":"2","9":"8.668553e-04","_rn_":"bpi_antisocialT1_1"},{"1":"female,male","2":"TRUE","3":"0.08143904","4":"6.272515","5":"4.533467","6":"12.627602","7":"3.9185610","8":"2","9":"1.409598e-01","_rn_":"bpi_antisocialT1_2"},{"1":"female,male","2":"TRUE","3":"-3.05560405","4":"3.135472","5":"1.396424","6":"9.490559","7":"7.0556041","8":"2","9":"2.936940e-02","_rn_":"bpi_antisocialT1_3"},{"1":"female,male","2":"TRUE","3":"-16.11820767","4":"-9.927132","5":"-11.666180","6":"-3.572045","7":"20.1182077","8":"2","9":"4.279438e-05","_rn_":"bpi_antisocialT1_4"},{"1":"female,male","2":"TRUE","3":"-40.86248189","4":"-34.671406","5":"-36.410454","6":"-28.316319","7":"44.8624819","8":"2","9":"1.812324e-10","_rn_":"bpi_antisocialT1_5"},{"1":"female,male","2":"TRUE","3":"3.62091906","4":"9.811995","5":"8.072947","6":"16.167082","7":"0.3790809","8":"2","9":"8.273392e-01","_rn_":"bpi_antisocialT1_6"},{"1":"female,male","2":"TRUE","3":"-23.57170973","4":"-17.380634","5":"-19.119682","6":"-11.025547","7":"27.5717097","8":"2","9":"1.030100e-06","_rn_":"bpi_antisocialT1_7"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
<div id="difAddConstraints" class="section level3 hasAnchor" number="16.9.6">
<h3><span class="header-section-number">16.9.6</span> Identify DIF by iteratively adding constraints to unconstrained model<a href="bias.html#difAddConstraints" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="difAddConstraintsDiscriminationSeverity" class="section level4 hasAnchor" number="16.9.6.1">
<h4><span class="header-section-number">16.9.6.1</span> Items that differ in discrimination and/or severity<a href="bias.html#difAddConstraintsDiscriminationSeverity" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Another way to identify DIF is to iteratively add constraints to a model in which all parameters are allowed to differ across groups.
Adding constraints forces individual items to have the same item parameters across groups, to identify which items yield a significant worsening in model fit when constraining their <a href="irt.html#itemDiscrimination">discrimination</a> and/or <a href="irt.html#itemDifficulty">severity</a> to be the same across groups.
Items 1, 2, 3, 4, 5, and 6 showed <a href="bias.html#dif">DIF</a> in <a href="irt.html#itemDiscrimination">discrimination</a> and/or <a href="irt.html#itemDifficulty">severity</a>, based on a significant chi-square difference test.
The <a href="bias.html#dif">DIF</a> in <a href="irt.html#itemDiscrimination">discrimination</a> and/or <a href="irt.html#itemDifficulty">severity</a> is depicted in Figure <a href="bias.html#fig:difAddDiscriminationSeverity">16.27</a>.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1527"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1527-1"><a href="bias.html#cb1527-1" tabindex="-1"></a>difAddItems <span class="ot">&lt;-</span> <span class="fu">DIF</span>(</span>
<span id="cb1527-2"><a href="bias.html#cb1527-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1527-3"><a href="bias.html#cb1527-3" tabindex="-1"></a>  <span class="fu">c</span>(<span class="st">&quot;a1&quot;</span>,<span class="st">&quot;d1&quot;</span>,<span class="st">&quot;d2&quot;</span>),</span>
<span id="cb1527-4"><a href="bias.html#cb1527-4" tabindex="-1"></a>  <span class="at">scheme =</span> <span class="st">&quot;add&quot;</span>,</span>
<span id="cb1527-5"><a href="bias.html#cb1527-5" tabindex="-1"></a>  <span class="at">simplify =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1527-6"><a href="bias.html#cb1527-6" tabindex="-1"></a>  <span class="at">plotdif =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:difAddDiscriminationSeverity"></span>
<img src="Book_files/figure-html/difAddDiscriminationSeverity-1.png" alt="Item Probability Functions for Examining Differential Item Functioning in Terms of Discrimination and/or Severity." width="100%" />
<p class="caption">
Figure 16.27: Item Probability Functions for Examining Differential Item Functioning in Terms of Discrimination and/or Severity.
</p>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1528"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1528-1"><a href="bias.html#cb1528-1" tabindex="-1"></a>difAddItems</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["groups"],"name":[1],"type":["chr"],"align":["left"]},{"label":["converged"],"name":[2],"type":["lgl"],"align":["right"]},{"label":["AIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["SABIC"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["HQ"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["X2"],"name":[7],"type":["dbl"],"align":["right"]},{"label":["df"],"name":[8],"type":["dbl"],"align":["right"]},{"label":["p"],"name":[9],"type":["dbl"],"align":["right"]}],"data":[{"1":"female,male","2":"TRUE","3":"-14.559887","4":"-5.273273","5":"-7.8818446","6":"4.259357","7":"20.559887","8":"3","9":"1.299240e-04","_rn_":"bpi_antisocialT1_1"},{"1":"female,male","2":"TRUE","3":"-27.694420","4":"-18.407806","5":"-21.0163775","6":"-8.875176","7":"33.694420","8":"3","9":"2.298411e-07","_rn_":"bpi_antisocialT1_2"},{"1":"female,male","2":"TRUE","3":"-6.872892","4":"2.413722","5":"-0.1948493","6":"11.946352","7":"12.872892","8":"3","9":"4.919705e-03","_rn_":"bpi_antisocialT1_3"},{"1":"female,male","2":"TRUE","3":"-75.769683","4":"-66.483069","5":"-69.0916411","6":"-56.950439","7":"81.769683","8":"3","9":"0.000000e+00","_rn_":"bpi_antisocialT1_4"},{"1":"female,male","2":"TRUE","3":"-124.659951","4":"-115.373337","5":"-117.9819087","6":"-105.840707","7":"130.659951","8":"3","9":"0.000000e+00","_rn_":"bpi_antisocialT1_5"},{"1":"female,male","2":"TRUE","3":"-38.409279","4":"-29.122665","5":"-31.7312370","6":"-19.590035","7":"44.409279","8":"3","9":"1.235359e-09","_rn_":"bpi_antisocialT1_6"},{"1":"female,male","2":"TRUE","3":"1.962134","4":"11.248748","5":"8.6401765","6":"20.781378","7":"4.037866","8":"3","9":"2.574042e-01","_rn_":"bpi_antisocialT1_7"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="difAddConstraintsDiscrimination" class="section level4 hasAnchor" number="16.9.6.2">
<h4><span class="header-section-number">16.9.6.2</span> Items that differ in discrimination<a href="bias.html#difAddConstraintsDiscrimination" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Items 6 and 7 showed <a href="bias.html#dif">DIF</a> in <a href="irt.html#itemDiscrimination">discrimination</a>, based on a significant chi-square difference test.
The <a href="bias.html#dif">DIF</a> in <a href="irt.html#itemDiscrimination">discrimination</a> is depicted in Figure <a href="bias.html#fig:difAddDiscrimination">16.28</a>.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1529"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1529-1"><a href="bias.html#cb1529-1" tabindex="-1"></a>difAddItemsDiscrimination <span class="ot">&lt;-</span> <span class="fu">DIF</span>(</span>
<span id="cb1529-2"><a href="bias.html#cb1529-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1529-3"><a href="bias.html#cb1529-3" tabindex="-1"></a>  <span class="fu">c</span>(<span class="st">&quot;a1&quot;</span>),</span>
<span id="cb1529-4"><a href="bias.html#cb1529-4" tabindex="-1"></a>  <span class="at">scheme =</span> <span class="st">&quot;add&quot;</span>,</span>
<span id="cb1529-5"><a href="bias.html#cb1529-5" tabindex="-1"></a>  <span class="at">simplify =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1529-6"><a href="bias.html#cb1529-6" tabindex="-1"></a>  <span class="at">plotdif =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:difAddDiscrimination"></span>
<img src="Book_files/figure-html/difAddDiscrimination-1.png" alt="Item Probability Functions for Examining Differential Item Functioning in Terms of Discrimination." width="100%" />
<p class="caption">
Figure 16.28: Item Probability Functions for Examining Differential Item Functioning in Terms of Discrimination.
</p>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1530"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1530-1"><a href="bias.html#cb1530-1" tabindex="-1"></a>difAddItemsDiscrimination</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["groups"],"name":[1],"type":["chr"],"align":["left"]},{"label":["converged"],"name":[2],"type":["lgl"],"align":["right"]},{"label":["AIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["SABIC"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["HQ"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["X2"],"name":[7],"type":["dbl"],"align":["right"]},{"label":["df"],"name":[8],"type":["dbl"],"align":["right"]},{"label":["p"],"name":[9],"type":["dbl"],"align":["right"]}],"data":[{"1":"female,male","2":"TRUE","3":"1.041531","4":"4.137069","5":"3.2675450","6":"7.314612","7":"0.958469037","8":"1","9":"0.327572904","_rn_":"bpi_antisocialT1_1"},{"1":"female,male","2":"TRUE","3":"1.283788","4":"4.379326","5":"3.5098020","6":"7.556869","7":"0.716212030","8":"1","9":"0.397389250","_rn_":"bpi_antisocialT1_2"},{"1":"female,male","2":"TRUE","3":"1.999766","4":"5.095304","5":"4.2257802","6":"8.272848","7":"0.000233829","8":"1","9":"0.987799649","_rn_":"bpi_antisocialT1_3"},{"1":"female,male","2":"TRUE","3":"1.493041","4":"4.588579","5":"3.7190555","6":"7.766123","7":"0.506958611","8":"1","9":"0.476458422","_rn_":"bpi_antisocialT1_4"},{"1":"female,male","2":"TRUE","3":"-1.231954","4":"1.863584","5":"0.9940598","6":"5.041127","7":"3.231954273","8":"1","9":"0.072214464","_rn_":"bpi_antisocialT1_5"},{"1":"female,male","2":"TRUE","3":"-8.463940","4":"-5.368402","5":"-6.2379255","6":"-2.190858","7":"10.463939561","8":"1","9":"0.001217274","_rn_":"bpi_antisocialT1_6"},{"1":"female,male","2":"TRUE","3":"-1.878169","4":"1.217369","5":"0.3478452","6":"4.394913","7":"3.878168817","8":"1","9":"0.048917884","_rn_":"bpi_antisocialT1_7"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="difAddConstraintsSeverity" class="section level4 hasAnchor" number="16.9.6.3">
<h4><span class="header-section-number">16.9.6.3</span> Items that differ in severity<a href="bias.html#difAddConstraintsSeverity" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Items 1, 2, 3, 4, 5, and 6 showed <a href="bias.html#dif">DIF</a> in <a href="irt.html#itemDifficulty">difficulty/severity</a>, based on a significant chi-square difference test.
The <a href="bias.html#dif">DIF</a> in <a href="irt.html#itemDifficulty">difficulty/severity</a> is depicted in Figure <a href="bias.html#fig:difAddSeverity">16.29</a></p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1531"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1531-1"><a href="bias.html#cb1531-1" tabindex="-1"></a>difAddItemsSeverity <span class="ot">&lt;-</span> <span class="fu">DIF</span>(</span>
<span id="cb1531-2"><a href="bias.html#cb1531-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1531-3"><a href="bias.html#cb1531-3" tabindex="-1"></a>  <span class="fu">c</span>(<span class="st">&quot;d1&quot;</span>,<span class="st">&quot;d2&quot;</span>),</span>
<span id="cb1531-4"><a href="bias.html#cb1531-4" tabindex="-1"></a>  <span class="at">scheme =</span> <span class="st">&quot;add&quot;</span>,</span>
<span id="cb1531-5"><a href="bias.html#cb1531-5" tabindex="-1"></a>  <span class="at">simplify =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1531-6"><a href="bias.html#cb1531-6" tabindex="-1"></a>  <span class="at">plotdif =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:difAddSeverity"></span>
<img src="Book_files/figure-html/difAddSeverity-1.png" alt="Item Probability Functions for Examining Differential Item Functioning in Terms of Severity." width="100%" />
<p class="caption">
Figure 16.29: Item Probability Functions for Examining Differential Item Functioning in Terms of Severity.
</p>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1532"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1532-1"><a href="bias.html#cb1532-1" tabindex="-1"></a>difAddItemsSeverity</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["groups"],"name":[1],"type":["chr"],"align":["left"]},{"label":["converged"],"name":[2],"type":["lgl"],"align":["right"]},{"label":["AIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["SABIC"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["HQ"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["X2"],"name":[7],"type":["dbl"],"align":["right"]},{"label":["df"],"name":[8],"type":["dbl"],"align":["right"]},{"label":["p"],"name":[9],"type":["dbl"],"align":["right"]}],"data":[{"1":"female,male","2":"TRUE","3":"-15.561637","4":"-9.370562","5":"-11.109609","6":"-3.015475","7":"19.5616375","8":"2","9":"5.652550e-05","_rn_":"bpi_antisocialT1_1"},{"1":"female,male","2":"TRUE","3":"-19.151480","4":"-12.960404","5":"-14.699452","6":"-6.605317","7":"23.1514799","8":"2","9":"9.391177e-06","_rn_":"bpi_antisocialT1_2"},{"1":"female,male","2":"TRUE","3":"-8.534472","4":"-2.343396","5":"-4.082444","6":"4.011691","7":"12.5344718","8":"2","9":"1.897466e-03","_rn_":"bpi_antisocialT1_3"},{"1":"female,male","2":"TRUE","3":"-28.940130","4":"-22.749054","5":"-24.488102","6":"-16.393968","7":"32.9401304","8":"2","9":"7.033015e-08","_rn_":"bpi_antisocialT1_4"},{"1":"female,male","2":"TRUE","3":"-92.484733","4":"-86.293657","5":"-88.032705","6":"-79.938570","7":"96.4847331","8":"2","9":"0.000000e+00","_rn_":"bpi_antisocialT1_5"},{"1":"female,male","2":"TRUE","3":"-28.929213","4":"-22.738137","5":"-24.477185","6":"-16.383050","7":"32.9292131","8":"2","9":"7.071511e-08","_rn_":"bpi_antisocialT1_6"},{"1":"female,male","2":"TRUE","3":"3.488790","4":"9.679866","5":"7.940819","6":"16.034953","7":"0.5112096","8":"2","9":"7.744480e-01","_rn_":"bpi_antisocialT1_7"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
<div id="difEffectSize" class="section level3 hasAnchor" number="16.9.7">
<h3><span class="header-section-number">16.9.7</span> Compute effect size of DIF<a href="bias.html#difEffectSize" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Effect size measures of <a href="bias.html#dif">DIF</a> were computed based on expected scores <span class="citation">(<a href="#ref-Meade2010">Meade, 2010</a>)</span> using the <code>mirt</code> package <span class="citation">(<a href="#ref-R-mirt">Chalmers, 2020</a>)</span>.
Some researchers recommend using a simulation-based procedure to examine the impact of <a href="bias.html#dif">DIF</a> on screening accuracy <span class="citation">(<a href="#ref-Gonzalez2021">Gonzalez &amp; Pelham, 2021</a>)</span>.</p>
<div id="difTestLevel" class="section level4 hasAnchor" number="16.9.7.1">
<h4><span class="header-section-number">16.9.7.1</span> Test-level DIF<a href="bias.html#difTestLevel" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>In addition to consideration of <a href="bias.html#dif">differential test functioning</a>, we can also consider whether the test as a whole (i.e., the collection of items) differs in its functioning by group—called differential test functioning.
Differential test functioning is depicted in Figure <a href="bias.html#fig:dtf">16.30</a>.
Estimates of differential test functioning are in Table <a href="#tab:testLevelDIF"><strong>??</strong></a>.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1533"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1533-1"><a href="bias.html#cb1533-1" tabindex="-1"></a><span class="fu">empirical_ES</span>(</span>
<span id="cb1533-2"><a href="bias.html#cb1533-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1533-3"><a href="bias.html#cb1533-3" tabindex="-1"></a>  <span class="at">DIF =</span> <span class="cn">FALSE</span>)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["Effect Size"],"name":[1],"type":["chr"],"align":["left"]},{"label":["Value"],"name":[2],"type":["dbl"],"align":["right"]}],"data":[{"1":"STDS","2":"0.6052089"},{"1":"UTDS","2":"0.6413932"},{"1":"UETSDS","2":"0.6052089"},{"1":"ETSSD","2":"0.3652185"},{"1":"Starks.DTFR","2":"0.6085074"},{"1":"UDTFR","2":"0.6487743"},{"1":"UETSDN","2":"0.6102540"},{"1":"theta.of.max.test.D","2":"1.2157431"},{"1":"Test.Dmax","2":"1.1263941"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>In general, the measure showed greater <a href="irt.html#itemDifficulty">difficulty</a> for females than for males.
That is, at a given construct level, males were more likely than females to endorse the items.
Otherwise said, it takes a higher construct level for females to obtain the same score on the measure as males.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1534"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1534-1"><a href="bias.html#cb1534-1" tabindex="-1"></a><span class="fu">empirical_ES</span>(</span>
<span id="cb1534-2"><a href="bias.html#cb1534-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1534-3"><a href="bias.html#cb1534-3" tabindex="-1"></a>  <span class="at">DIF =</span> <span class="cn">FALSE</span>,</span>
<span id="cb1534-4"><a href="bias.html#cb1534-4" tabindex="-1"></a>  <span class="at">plot =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:dtf"></span>
<img src="Book_files/figure-html/dtf-1.png" alt="Differential Test Functioning by Sex." width="100%" />
<p class="caption">
Figure 16.30: Differential Test Functioning by Sex.
</p>
</div>
</div>
<div id="difItemLevel" class="section level4 hasAnchor" number="16.9.7.2">
<h4><span class="header-section-number">16.9.7.2</span> Item-level DIF<a href="bias.html#difItemLevel" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><a href="bias.html#dif">Differential item functioning</a> is depicted in Figure <a href="bias.html#fig:dif">16.31</a>.
Estimates of <a href="bias.html#dif">differential item functioning</a> are in Table <a href="#tab:itemLevelDIF"><strong>??</strong></a>.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1535"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1535-1"><a href="bias.html#cb1535-1" tabindex="-1"></a><span class="fu">empirical_ES</span>(</span>
<span id="cb1535-2"><a href="bias.html#cb1535-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1535-3"><a href="bias.html#cb1535-3" tabindex="-1"></a>  <span class="at">DIF =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb1535-4"><a href="bias.html#cb1535-4" tabindex="-1"></a>  <span class="fu">round</span>(., <span class="dv">2</span>)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["SIDS"],"name":[1],"type":["dbl"],"align":["right"]},{"label":["UIDS"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["SIDN"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["UIDN"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["ESSD"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["theta.of.max.D"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["max.D"],"name":[7],"type":["dbl"],"align":["right"]},{"label":["mean.ES.foc"],"name":[8],"type":["dbl"],"align":["right"]},{"label":["mean.ES.ref"],"name":[9],"type":["dbl"],"align":["right"]}],"data":[{"1":"0.08","2":"0.08","3":"0.08","4":"0.08","5":"0.27","6":"-0.34","7":"0.11","8":"0.61","9":"0.52","_rn_":"bpi_antisocialT1_1"},{"1":"0.09","2":"0.09","3":"0.09","4":"0.09","5":"0.32","6":"1.05","7":"0.18","8":"0.35","9":"0.26","_rn_":"bpi_antisocialT1_2"},{"1":"0.07","2":"0.07","3":"0.06","4":"0.06","5":"0.25","6":"0.76","7":"0.08","8":"0.50","9":"0.43","_rn_":"bpi_antisocialT1_3"},{"1":"0.11","2":"0.11","3":"0.12","4":"0.12","5":"0.57","6":"2.55","7":"0.40","8":"0.21","9":"0.10","_rn_":"bpi_antisocialT1_4"},{"1":"0.18","2":"0.18","3":"0.18","4":"0.18","5":"0.73","6":"0.88","7":"0.32","8":"0.34","9":"0.16","_rn_":"bpi_antisocialT1_5"},{"1":"0.07","2":"0.08","3":"0.07","4":"0.08","5":"0.40","6":"1.10","7":"0.17","8":"0.15","9":"0.08","_rn_":"bpi_antisocialT1_6"},{"1":"0.00","2":"0.03","3":"0.00","4":"0.04","5":"0.00","6":"3.22","7":"0.13","8":"0.83","9":"0.83","_rn_":"bpi_antisocialT1_7"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>The measure-level differences in functioning appear to be largely driven by greater <a href="irt.html#itemDifficulty">difficulty</a> of items 4 and 5 for females than males.
That is, at a given construct level, males were more likely than females to endorse items 4 and 5, in particular.
Otherwise said, it takes a higher construct level for females than males to endorse items 4 and 5.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1536"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1536-1"><a href="bias.html#cb1536-1" tabindex="-1"></a><span class="fu">empirical_ES</span>(</span>
<span id="cb1536-2"><a href="bias.html#cb1536-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1536-3"><a href="bias.html#cb1536-3" tabindex="-1"></a>  <span class="at">DIF =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1536-4"><a href="bias.html#cb1536-4" tabindex="-1"></a>  <span class="at">plot =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:dif"></span>
<img src="Book_files/figure-html/dif-1.png" alt="Differential Item Functioning by Sex." width="100%" />
<p class="caption">
Figure 16.31: Differential Item Functioning by Sex.
</p>
</div>
</div>
</div>
<div id="difItemPlots" class="section level3 hasAnchor" number="16.9.8">
<h3><span class="header-section-number">16.9.8</span> Item plots<a href="bias.html#difItemPlots" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Plots of item response category characteristic curves by sex are in Figures <a href="bias.html#fig:ircSex1">16.32</a>–<a href="bias.html#fig:ircSex7">16.38</a> below.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1537"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1537-1"><a href="bias.html#cb1537-1" tabindex="-1"></a><span class="fu">itemplot</span>(</span>
<span id="cb1537-2"><a href="bias.html#cb1537-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1537-3"><a href="bias.html#cb1537-3" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_1&quot;</span>,</span>
<span id="cb1537-4"><a href="bias.html#cb1537-4" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;trace&quot;</span>,</span>
<span id="cb1537-5"><a href="bias.html#cb1537-5" tabindex="-1"></a>  <span class="at">par.settings =</span> <span class="fu">standard.theme</span>(<span class="st">&quot;pdf&quot;</span>, <span class="at">color =</span> <span class="cn">FALSE</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:ircSex1"></span>
<img src="Book_files/figure-html/ircSex1-1.png" alt="Item Response Category Characteristic Curves by Sex: Item 1." width="100%" />
<p class="caption">
Figure 16.32: Item Response Category Characteristic Curves by Sex: Item 1.
</p>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1538"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1538-1"><a href="bias.html#cb1538-1" tabindex="-1"></a><span class="fu">itemplot</span>(</span>
<span id="cb1538-2"><a href="bias.html#cb1538-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1538-3"><a href="bias.html#cb1538-3" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_2&quot;</span>,</span>
<span id="cb1538-4"><a href="bias.html#cb1538-4" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;trace&quot;</span>,</span>
<span id="cb1538-5"><a href="bias.html#cb1538-5" tabindex="-1"></a>  <span class="at">par.settings =</span> <span class="fu">standard.theme</span>(<span class="st">&quot;pdf&quot;</span>, <span class="at">color =</span> <span class="cn">FALSE</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:ircSex2"></span>
<img src="Book_files/figure-html/ircSex2-1.png" alt="Item Response Category Characteristic Curves by Sex: Item 2." width="100%" />
<p class="caption">
Figure 16.33: Item Response Category Characteristic Curves by Sex: Item 2.
</p>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1539"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1539-1"><a href="bias.html#cb1539-1" tabindex="-1"></a><span class="fu">itemplot</span>(</span>
<span id="cb1539-2"><a href="bias.html#cb1539-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1539-3"><a href="bias.html#cb1539-3" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_3&quot;</span>,</span>
<span id="cb1539-4"><a href="bias.html#cb1539-4" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;trace&quot;</span>,</span>
<span id="cb1539-5"><a href="bias.html#cb1539-5" tabindex="-1"></a>  <span class="at">par.settings =</span> <span class="fu">standard.theme</span>(<span class="st">&quot;pdf&quot;</span>, <span class="at">color =</span> <span class="cn">FALSE</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:ircSex3"></span>
<img src="Book_files/figure-html/ircSex3-1.png" alt="Item Response Category Characteristic Curves by Sex: Item 3." width="100%" />
<p class="caption">
Figure 16.34: Item Response Category Characteristic Curves by Sex: Item 3.
</p>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1540"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1540-1"><a href="bias.html#cb1540-1" tabindex="-1"></a><span class="fu">itemplot</span>(</span>
<span id="cb1540-2"><a href="bias.html#cb1540-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1540-3"><a href="bias.html#cb1540-3" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_4&quot;</span>,</span>
<span id="cb1540-4"><a href="bias.html#cb1540-4" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;trace&quot;</span>,</span>
<span id="cb1540-5"><a href="bias.html#cb1540-5" tabindex="-1"></a>  <span class="at">par.settings =</span> <span class="fu">standard.theme</span>(<span class="st">&quot;pdf&quot;</span>, <span class="at">color =</span> <span class="cn">FALSE</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:ircSex4"></span>
<img src="Book_files/figure-html/ircSex4-1.png" alt="Item Response Category Characteristic Curves by Sex: Item 4." width="100%" />
<p class="caption">
Figure 16.35: Item Response Category Characteristic Curves by Sex: Item 4.
</p>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1541"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1541-1"><a href="bias.html#cb1541-1" tabindex="-1"></a><span class="fu">itemplot</span>(</span>
<span id="cb1541-2"><a href="bias.html#cb1541-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1541-3"><a href="bias.html#cb1541-3" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_5&quot;</span>,</span>
<span id="cb1541-4"><a href="bias.html#cb1541-4" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;trace&quot;</span>,</span>
<span id="cb1541-5"><a href="bias.html#cb1541-5" tabindex="-1"></a>  <span class="at">par.settings =</span> <span class="fu">standard.theme</span>(<span class="st">&quot;pdf&quot;</span>, <span class="at">color =</span> <span class="cn">FALSE</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:ircSex5"></span>
<img src="Book_files/figure-html/ircSex5-1.png" alt="Item Response Category Characteristic Curves by Sex: Item 5." width="100%" />
<p class="caption">
Figure 16.36: Item Response Category Characteristic Curves by Sex: Item 5.
</p>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1542"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1542-1"><a href="bias.html#cb1542-1" tabindex="-1"></a><span class="fu">itemplot</span>(</span>
<span id="cb1542-2"><a href="bias.html#cb1542-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1542-3"><a href="bias.html#cb1542-3" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_6&quot;</span>,</span>
<span id="cb1542-4"><a href="bias.html#cb1542-4" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;trace&quot;</span>,</span>
<span id="cb1542-5"><a href="bias.html#cb1542-5" tabindex="-1"></a>  <span class="at">par.settings =</span> <span class="fu">standard.theme</span>(<span class="st">&quot;pdf&quot;</span>, <span class="at">color =</span> <span class="cn">FALSE</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:ircSex6"></span>
<img src="Book_files/figure-html/ircSex6-1.png" alt="Item Response Category Characteristic Curves by Sex: Item 6." width="100%" />
<p class="caption">
Figure 16.37: Item Response Category Characteristic Curves by Sex: Item 6.
</p>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1543"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1543-1"><a href="bias.html#cb1543-1" tabindex="-1"></a><span class="fu">itemplot</span>(</span>
<span id="cb1543-2"><a href="bias.html#cb1543-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1543-3"><a href="bias.html#cb1543-3" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_7&quot;</span>,</span>
<span id="cb1543-4"><a href="bias.html#cb1543-4" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;trace&quot;</span>,</span>
<span id="cb1543-5"><a href="bias.html#cb1543-5" tabindex="-1"></a>  <span class="at">par.settings =</span> <span class="fu">standard.theme</span>(<span class="st">&quot;pdf&quot;</span>, <span class="at">color =</span> <span class="cn">FALSE</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:ircSex7"></span>
<img src="Book_files/figure-html/ircSex7-1.png" alt="Item Response Category Characteristic Curves by Sex: Item 7." width="100%" />
<p class="caption">
Figure 16.38: Item Response Category Characteristic Curves by Sex: Item 7.
</p>
</div>
<p>Plots of item <a href="irt.html#irtReliability">information</a> functions by sex are in Figures <a href="bias.html#fig:itemInfo1">16.39</a>–<a href="bias.html#fig:itemInfo7">16.45</a> below.
Items 1, 5, and 6 showed greater <a href="irt.html#irtReliability">information</a> for females than for males.
Items 2, 4, and 7 showed greater <a href="irt.html#irtReliability">information</a> for females than for males.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1544"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1544-1"><a href="bias.html#cb1544-1" tabindex="-1"></a><span class="fu">itemplot</span>(</span>
<span id="cb1544-2"><a href="bias.html#cb1544-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1544-3"><a href="bias.html#cb1544-3" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_1&quot;</span>,</span>
<span id="cb1544-4"><a href="bias.html#cb1544-4" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;info&quot;</span>,</span>
<span id="cb1544-5"><a href="bias.html#cb1544-5" tabindex="-1"></a>  <span class="at">par.settings =</span> <span class="fu">standard.theme</span>(<span class="st">&quot;pdf&quot;</span>, <span class="at">color =</span> <span class="cn">FALSE</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:itemInfo1"></span>
<img src="Book_files/figure-html/itemInfo1-1.png" alt="Item Information Curves by Sex: Item 1." width="100%" />
<p class="caption">
Figure 16.39: Item Information Curves by Sex: Item 1.
</p>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1545"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1545-1"><a href="bias.html#cb1545-1" tabindex="-1"></a><span class="fu">itemplot</span>(</span>
<span id="cb1545-2"><a href="bias.html#cb1545-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1545-3"><a href="bias.html#cb1545-3" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_2&quot;</span>,</span>
<span id="cb1545-4"><a href="bias.html#cb1545-4" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;info&quot;</span>,</span>
<span id="cb1545-5"><a href="bias.html#cb1545-5" tabindex="-1"></a>  <span class="at">par.settings =</span> <span class="fu">standard.theme</span>(<span class="st">&quot;pdf&quot;</span>, <span class="at">color =</span> <span class="cn">FALSE</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:itemInfo2"></span>
<img src="Book_files/figure-html/itemInfo2-1.png" alt="Item Information Curves by Sex: Item 2." width="100%" />
<p class="caption">
Figure 16.40: Item Information Curves by Sex: Item 2.
</p>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1546"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1546-1"><a href="bias.html#cb1546-1" tabindex="-1"></a><span class="fu">itemplot</span>(</span>
<span id="cb1546-2"><a href="bias.html#cb1546-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1546-3"><a href="bias.html#cb1546-3" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_3&quot;</span>,</span>
<span id="cb1546-4"><a href="bias.html#cb1546-4" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;info&quot;</span>,</span>
<span id="cb1546-5"><a href="bias.html#cb1546-5" tabindex="-1"></a>  <span class="at">par.settings =</span> <span class="fu">standard.theme</span>(<span class="st">&quot;pdf&quot;</span>, <span class="at">color =</span> <span class="cn">FALSE</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:itemInfo3"></span>
<img src="Book_files/figure-html/itemInfo3-1.png" alt="Item Information Curves by Sex: Item 3." width="100%" />
<p class="caption">
Figure 16.41: Item Information Curves by Sex: Item 3.
</p>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1547"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1547-1"><a href="bias.html#cb1547-1" tabindex="-1"></a><span class="fu">itemplot</span>(</span>
<span id="cb1547-2"><a href="bias.html#cb1547-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1547-3"><a href="bias.html#cb1547-3" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_4&quot;</span>,</span>
<span id="cb1547-4"><a href="bias.html#cb1547-4" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;info&quot;</span>,</span>
<span id="cb1547-5"><a href="bias.html#cb1547-5" tabindex="-1"></a>  <span class="at">par.settings =</span> <span class="fu">standard.theme</span>(<span class="st">&quot;pdf&quot;</span>, <span class="at">color =</span> <span class="cn">FALSE</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:itemInfo4"></span>
<img src="Book_files/figure-html/itemInfo4-1.png" alt="Item Information Curves by Sex: Item 4." width="100%" />
<p class="caption">
Figure 16.42: Item Information Curves by Sex: Item 4.
</p>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1548"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1548-1"><a href="bias.html#cb1548-1" tabindex="-1"></a><span class="fu">itemplot</span>(</span>
<span id="cb1548-2"><a href="bias.html#cb1548-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1548-3"><a href="bias.html#cb1548-3" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_5&quot;</span>,</span>
<span id="cb1548-4"><a href="bias.html#cb1548-4" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;info&quot;</span>,</span>
<span id="cb1548-5"><a href="bias.html#cb1548-5" tabindex="-1"></a>  <span class="at">par.settings =</span> <span class="fu">standard.theme</span>(<span class="st">&quot;pdf&quot;</span>, <span class="at">color =</span> <span class="cn">FALSE</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:itemInfo5"></span>
<img src="Book_files/figure-html/itemInfo5-1.png" alt="Item Information Curves by Sex: Item 5." width="100%" />
<p class="caption">
Figure 16.43: Item Information Curves by Sex: Item 5.
</p>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1549"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1549-1"><a href="bias.html#cb1549-1" tabindex="-1"></a><span class="fu">itemplot</span>(</span>
<span id="cb1549-2"><a href="bias.html#cb1549-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1549-3"><a href="bias.html#cb1549-3" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_6&quot;</span>,</span>
<span id="cb1549-4"><a href="bias.html#cb1549-4" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;info&quot;</span>,</span>
<span id="cb1549-5"><a href="bias.html#cb1549-5" tabindex="-1"></a>  <span class="at">par.settings =</span> <span class="fu">standard.theme</span>(<span class="st">&quot;pdf&quot;</span>, <span class="at">color =</span> <span class="cn">FALSE</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:itemInfo6"></span>
<img src="Book_files/figure-html/itemInfo6-1.png" alt="Item Information Curves by Sex: Item 6." width="100%" />
<p class="caption">
Figure 16.44: Item Information Curves by Sex: Item 6.
</p>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1550"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1550-1"><a href="bias.html#cb1550-1" tabindex="-1"></a><span class="fu">itemplot</span>(</span>
<span id="cb1550-2"><a href="bias.html#cb1550-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1550-3"><a href="bias.html#cb1550-3" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_7&quot;</span>,</span>
<span id="cb1550-4"><a href="bias.html#cb1550-4" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;info&quot;</span>,</span>
<span id="cb1550-5"><a href="bias.html#cb1550-5" tabindex="-1"></a>  <span class="at">par.settings =</span> <span class="fu">standard.theme</span>(<span class="st">&quot;pdf&quot;</span>, <span class="at">color =</span> <span class="cn">FALSE</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:itemInfo7"></span>
<img src="Book_files/figure-html/itemInfo7-1.png" alt="Item Information Curves by Sex: Item 7." width="100%" />
<p class="caption">
Figure 16.45: Item Information Curves by Sex: Item 7.
</p>
</div>
<p>Plots of expected item scores by sex are are in Figures <a href="bias.html#fig:itemScore1">16.46</a>–<a href="bias.html#fig:itemScore7">16.52</a> below.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1551"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1551-1"><a href="bias.html#cb1551-1" tabindex="-1"></a><span class="fu">itemplot</span>(</span>
<span id="cb1551-2"><a href="bias.html#cb1551-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1551-3"><a href="bias.html#cb1551-3" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_1&quot;</span>,</span>
<span id="cb1551-4"><a href="bias.html#cb1551-4" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;score&quot;</span>,</span>
<span id="cb1551-5"><a href="bias.html#cb1551-5" tabindex="-1"></a>  <span class="at">par.settings =</span> <span class="fu">standard.theme</span>(<span class="st">&quot;pdf&quot;</span>, <span class="at">color =</span> <span class="cn">FALSE</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:itemScore1"></span>
<img src="Book_files/figure-html/itemScore1-1.png" alt="Expected Item Score by Sex: Item 1." width="100%" />
<p class="caption">
Figure 16.46: Expected Item Score by Sex: Item 1.
</p>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1552"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1552-1"><a href="bias.html#cb1552-1" tabindex="-1"></a><span class="fu">itemplot</span>(</span>
<span id="cb1552-2"><a href="bias.html#cb1552-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1552-3"><a href="bias.html#cb1552-3" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_2&quot;</span>,</span>
<span id="cb1552-4"><a href="bias.html#cb1552-4" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;score&quot;</span>,</span>
<span id="cb1552-5"><a href="bias.html#cb1552-5" tabindex="-1"></a>  <span class="at">par.settings =</span> <span class="fu">standard.theme</span>(<span class="st">&quot;pdf&quot;</span>, <span class="at">color =</span> <span class="cn">FALSE</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:itemScore2"></span>
<img src="Book_files/figure-html/itemScore2-1.png" alt="Expected Item Score by Sex: Item 2." width="100%" />
<p class="caption">
Figure 16.47: Expected Item Score by Sex: Item 2.
</p>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1553"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1553-1"><a href="bias.html#cb1553-1" tabindex="-1"></a><span class="fu">itemplot</span>(</span>
<span id="cb1553-2"><a href="bias.html#cb1553-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1553-3"><a href="bias.html#cb1553-3" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_3&quot;</span>,</span>
<span id="cb1553-4"><a href="bias.html#cb1553-4" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;score&quot;</span>,</span>
<span id="cb1553-5"><a href="bias.html#cb1553-5" tabindex="-1"></a>  <span class="at">par.settings =</span> <span class="fu">standard.theme</span>(<span class="st">&quot;pdf&quot;</span>, <span class="at">color =</span> <span class="cn">FALSE</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:itemScore3"></span>
<img src="Book_files/figure-html/itemScore3-1.png" alt="Expected Item Score by Sex: Item 3." width="100%" />
<p class="caption">
Figure 16.48: Expected Item Score by Sex: Item 3.
</p>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1554"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1554-1"><a href="bias.html#cb1554-1" tabindex="-1"></a><span class="fu">itemplot</span>(</span>
<span id="cb1554-2"><a href="bias.html#cb1554-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1554-3"><a href="bias.html#cb1554-3" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_4&quot;</span>,</span>
<span id="cb1554-4"><a href="bias.html#cb1554-4" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;score&quot;</span>,</span>
<span id="cb1554-5"><a href="bias.html#cb1554-5" tabindex="-1"></a>  <span class="at">par.settings =</span> <span class="fu">standard.theme</span>(<span class="st">&quot;pdf&quot;</span>, <span class="at">color =</span> <span class="cn">FALSE</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:itemScore4"></span>
<img src="Book_files/figure-html/itemScore4-1.png" alt="Expected Item Score by Sex: Item 4." width="100%" />
<p class="caption">
Figure 16.49: Expected Item Score by Sex: Item 4.
</p>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1555"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1555-1"><a href="bias.html#cb1555-1" tabindex="-1"></a><span class="fu">itemplot</span>(</span>
<span id="cb1555-2"><a href="bias.html#cb1555-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1555-3"><a href="bias.html#cb1555-3" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_5&quot;</span>,</span>
<span id="cb1555-4"><a href="bias.html#cb1555-4" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;score&quot;</span>,</span>
<span id="cb1555-5"><a href="bias.html#cb1555-5" tabindex="-1"></a>  <span class="at">par.settings =</span> <span class="fu">standard.theme</span>(<span class="st">&quot;pdf&quot;</span>, <span class="at">color =</span> <span class="cn">FALSE</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:itemScore5"></span>
<img src="Book_files/figure-html/itemScore5-1.png" alt="Expected Item Score by Sex: Item 5." width="100%" />
<p class="caption">
Figure 16.50: Expected Item Score by Sex: Item 5.
</p>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1556"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1556-1"><a href="bias.html#cb1556-1" tabindex="-1"></a><span class="fu">itemplot</span>(</span>
<span id="cb1556-2"><a href="bias.html#cb1556-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1556-3"><a href="bias.html#cb1556-3" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_6&quot;</span>,</span>
<span id="cb1556-4"><a href="bias.html#cb1556-4" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;score&quot;</span>,</span>
<span id="cb1556-5"><a href="bias.html#cb1556-5" tabindex="-1"></a>  <span class="at">par.settings =</span> <span class="fu">standard.theme</span>(<span class="st">&quot;pdf&quot;</span>, <span class="at">color =</span> <span class="cn">FALSE</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:itemScore6"></span>
<img src="Book_files/figure-html/itemScore6-1.png" alt="Expected Item Score by Sex: Item 6." width="100%" />
<p class="caption">
Figure 16.51: Expected Item Score by Sex: Item 6.
</p>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1557"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1557-1"><a href="bias.html#cb1557-1" tabindex="-1"></a><span class="fu">itemplot</span>(</span>
<span id="cb1557-2"><a href="bias.html#cb1557-2" tabindex="-1"></a>  unconstrainedModel,</span>
<span id="cb1557-3"><a href="bias.html#cb1557-3" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_7&quot;</span>,</span>
<span id="cb1557-4"><a href="bias.html#cb1557-4" tabindex="-1"></a>  <span class="at">type =</span> <span class="st">&quot;score&quot;</span>,</span>
<span id="cb1557-5"><a href="bias.html#cb1557-5" tabindex="-1"></a>  <span class="at">par.settings =</span> <span class="fu">standard.theme</span>(<span class="st">&quot;pdf&quot;</span>, <span class="at">color =</span> <span class="cn">FALSE</span>))</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:itemScore7"></span>
<img src="Book_files/figure-html/itemScore7-1.png" alt="Expected Item Score by Sex: Item 7." width="100%" />
<p class="caption">
Figure 16.52: Expected Item Score by Sex: Item 7.
</p>
</div>
</div>
<div id="addressingDIF" class="section level3 hasAnchor" number="16.9.9">
<h3><span class="header-section-number">16.9.9</span> Addressing DIF<a href="bias.html#addressingDIF" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Based on the analyses above, item 5 shows the largest magnitude of <a href="bias.html#dif">DIF</a>.
Specifically, item 5 has a stronger <a href="irt.html#itemDiscrimination">discrimination</a> parameter for women than men.
So, we should handle item 5 first.
If we deem the <a href="bias.html#dif">DIF</a> for item 5 to be non-negligible, we have three primary options: (1) drop item 5 for both men and women, (2) drop item 5 for men but keep it for women, or (3) freely estimate the parameters (<a href="irt.html#itemDiscrimination">discrimination</a> and <a href="irt.html#itemDifficulty">difficulty</a>) for item 5 across groups.</p>
<div id="dropItemForBothGroups" class="section level4 hasAnchor" number="16.9.9.1">
<h4><span class="header-section-number">16.9.9.1</span> Drop item for both groups<a href="bias.html#dropItemForBothGroups" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>The first option is to drop item 5 for both men and women groups.
You might do this if you want to use a measure that has only those items that function equivalently across groups.
However, this can lead to lower <a href="reliability.html#reliability">reliability</a> and weaker ability to detect individual differences.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1558"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1558-1"><a href="bias.html#cb1558-1" tabindex="-1"></a>constrainedModelDropItem5 <span class="ot">&lt;-</span> <span class="fu">multipleGroup</span>(</span>
<span id="cb1558-2"><a href="bias.html#cb1558-2" tabindex="-1"></a>  <span class="at">data =</span> cnlsy[,<span class="fu">c</span>(</span>
<span id="cb1558-3"><a href="bias.html#cb1558-3" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_1&quot;</span>,<span class="st">&quot;bpi_antisocialT1_2&quot;</span>,<span class="st">&quot;bpi_antisocialT1_3&quot;</span>,</span>
<span id="cb1558-4"><a href="bias.html#cb1558-4" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_4&quot;</span>,<span class="st">&quot;bpi_antisocialT1_6&quot;</span>,<span class="st">&quot;bpi_antisocialT1_7&quot;</span>)],</span>
<span id="cb1558-5"><a href="bias.html#cb1558-5" tabindex="-1"></a>  <span class="at">model =</span> <span class="dv">1</span>,</span>
<span id="cb1558-6"><a href="bias.html#cb1558-6" tabindex="-1"></a>  <span class="at">group =</span> cnlsy<span class="sc">$</span>sex,</span>
<span id="cb1558-7"><a href="bias.html#cb1558-7" tabindex="-1"></a>  <span class="at">invariance =</span> <span class="fu">c</span>(<span class="fu">c</span>(</span>
<span id="cb1558-8"><a href="bias.html#cb1558-8" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_1&quot;</span>,<span class="st">&quot;bpi_antisocialT1_2&quot;</span>,<span class="st">&quot;bpi_antisocialT1_3&quot;</span>,</span>
<span id="cb1558-9"><a href="bias.html#cb1558-9" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_4&quot;</span>,<span class="st">&quot;bpi_antisocialT1_6&quot;</span>,<span class="st">&quot;bpi_antisocialT1_7&quot;</span>),</span>
<span id="cb1558-10"><a href="bias.html#cb1558-10" tabindex="-1"></a>    <span class="st">&quot;free_means&quot;</span>, <span class="st">&quot;free_var&quot;</span>),</span>
<span id="cb1558-11"><a href="bias.html#cb1558-11" tabindex="-1"></a>  <span class="at">SE =</span> <span class="cn">TRUE</span>)</span>
<span id="cb1558-12"><a href="bias.html#cb1558-12" tabindex="-1"></a></span>
<span id="cb1558-13"><a href="bias.html#cb1558-13" tabindex="-1"></a><span class="fu">coef</span>(constrainedModelDropItem5, <span class="at">simplify =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
</div>
<div id="dropItemForOneGroup" class="section level4 hasAnchor" number="16.9.9.2">
<h4><span class="header-section-number">16.9.9.2</span> Drop item for one group but not another group<a href="bias.html#dropItemForOneGroup" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>A second option is to drop item 5 for men but to keep it for women.
You might do this if the item is <a href="validity.html#validity">invalid</a> for men, but still <a href="validity.html#validity">valid</a> for women.
The coefficients show an item parameter for item 5 for men, but it was constrained to be equal to the parameter for women, and data were removed from the estimation of item 5 for men.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1559"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1559-1"><a href="bias.html#cb1559-1" tabindex="-1"></a>dropItem5ForMen <span class="ot">&lt;-</span> cnlsy</span>
<span id="cb1559-2"><a href="bias.html#cb1559-2" tabindex="-1"></a>dropItem5ForMen[<span class="fu">which</span>(</span>
<span id="cb1559-3"><a href="bias.html#cb1559-3" tabindex="-1"></a>  dropItem5ForMen<span class="sc">$</span>sex <span class="sc">==</span> <span class="st">&quot;male&quot;</span>),</span>
<span id="cb1559-4"><a href="bias.html#cb1559-4" tabindex="-1"></a>  <span class="st">&quot;bpi_antisocialT1_5&quot;</span>] <span class="ot">&lt;-</span> <span class="cn">NA</span></span>
<span id="cb1559-5"><a href="bias.html#cb1559-5" tabindex="-1"></a></span>
<span id="cb1559-6"><a href="bias.html#cb1559-6" tabindex="-1"></a>constrainedModelDropItem5ForMen <span class="ot">&lt;-</span> <span class="fu">multipleGroup</span>(</span>
<span id="cb1559-7"><a href="bias.html#cb1559-7" tabindex="-1"></a>  <span class="at">data =</span> dropItem5ForMen[,<span class="fu">c</span>(</span>
<span id="cb1559-8"><a href="bias.html#cb1559-8" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_1&quot;</span>,<span class="st">&quot;bpi_antisocialT1_2&quot;</span>,<span class="st">&quot;bpi_antisocialT1_3&quot;</span>,</span>
<span id="cb1559-9"><a href="bias.html#cb1559-9" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_4&quot;</span>,<span class="st">&quot;bpi_antisocialT1_5&quot;</span>,<span class="st">&quot;bpi_antisocialT1_6&quot;</span>,</span>
<span id="cb1559-10"><a href="bias.html#cb1559-10" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_7&quot;</span>)],</span>
<span id="cb1559-11"><a href="bias.html#cb1559-11" tabindex="-1"></a>  <span class="at">model =</span> <span class="dv">1</span>,</span>
<span id="cb1559-12"><a href="bias.html#cb1559-12" tabindex="-1"></a>  <span class="at">group =</span> dropItem5ForMen<span class="sc">$</span>sex,</span>
<span id="cb1559-13"><a href="bias.html#cb1559-13" tabindex="-1"></a>  <span class="at">invariance =</span> <span class="fu">c</span>(<span class="fu">c</span>(</span>
<span id="cb1559-14"><a href="bias.html#cb1559-14" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_1&quot;</span>,<span class="st">&quot;bpi_antisocialT1_2&quot;</span>,<span class="st">&quot;bpi_antisocialT1_3&quot;</span>,</span>
<span id="cb1559-15"><a href="bias.html#cb1559-15" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_4&quot;</span>,<span class="st">&quot;bpi_antisocialT1_5&quot;</span>,<span class="st">&quot;bpi_antisocialT1_6&quot;</span>,</span>
<span id="cb1559-16"><a href="bias.html#cb1559-16" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_7&quot;</span>),</span>
<span id="cb1559-17"><a href="bias.html#cb1559-17" tabindex="-1"></a>    <span class="st">&quot;free_means&quot;</span>, <span class="st">&quot;free_var&quot;</span>),</span>
<span id="cb1559-18"><a href="bias.html#cb1559-18" tabindex="-1"></a>  <span class="at">SE =</span> <span class="cn">TRUE</span>)</span>
<span id="cb1559-19"><a href="bias.html#cb1559-19" tabindex="-1"></a></span>
<span id="cb1559-20"><a href="bias.html#cb1559-20" tabindex="-1"></a><span class="fu">coef</span>(constrainedModelDropItem5ForMen, <span class="at">simplify =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
</div>
<div id="resolveItemAcrossGroups" class="section level4 hasAnchor" number="16.9.9.3">
<h4><span class="header-section-number">16.9.9.3</span> Freely estimate item to have different parameters across groups<a href="bias.html#resolveItemAcrossGroups" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Alternatively, we can resolve <a href="bias.html#dif">DIF</a> by allowing the item to have different parameters (<a href="irt.html#itemDiscrimination">discrimination</a> and <a href="irt.html#itemDifficulty">difficulty</a>) across both groups.
You might do this if the item is <a href="validity.html#validity">valid</a> for both men and women, and it has importantly different item parameters nonetheless.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1560"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1560-1"><a href="bias.html#cb1560-1" tabindex="-1"></a>constrainedModelResolveItem5 <span class="ot">&lt;-</span> <span class="fu">multipleGroup</span>(</span>
<span id="cb1560-2"><a href="bias.html#cb1560-2" tabindex="-1"></a>  <span class="at">data =</span> cnlsy[,<span class="fu">c</span>(</span>
<span id="cb1560-3"><a href="bias.html#cb1560-3" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_1&quot;</span>,<span class="st">&quot;bpi_antisocialT1_2&quot;</span>,<span class="st">&quot;bpi_antisocialT1_3&quot;</span>,</span>
<span id="cb1560-4"><a href="bias.html#cb1560-4" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_4&quot;</span>,<span class="st">&quot;bpi_antisocialT1_5&quot;</span>,<span class="st">&quot;bpi_antisocialT1_6&quot;</span>,</span>
<span id="cb1560-5"><a href="bias.html#cb1560-5" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_7&quot;</span>)],</span>
<span id="cb1560-6"><a href="bias.html#cb1560-6" tabindex="-1"></a>  <span class="at">model =</span> <span class="dv">1</span>,</span>
<span id="cb1560-7"><a href="bias.html#cb1560-7" tabindex="-1"></a>  <span class="at">group =</span> cnlsy<span class="sc">$</span>sex,</span>
<span id="cb1560-8"><a href="bias.html#cb1560-8" tabindex="-1"></a>  <span class="at">invariance =</span> <span class="fu">c</span>(<span class="fu">c</span>(</span>
<span id="cb1560-9"><a href="bias.html#cb1560-9" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_1&quot;</span>,<span class="st">&quot;bpi_antisocialT1_2&quot;</span>,<span class="st">&quot;bpi_antisocialT1_3&quot;</span>,</span>
<span id="cb1560-10"><a href="bias.html#cb1560-10" tabindex="-1"></a>    <span class="st">&quot;bpi_antisocialT1_4&quot;</span>,<span class="st">&quot;bpi_antisocialT1_6&quot;</span>,<span class="st">&quot;bpi_antisocialT1_7&quot;</span>),</span>
<span id="cb1560-11"><a href="bias.html#cb1560-11" tabindex="-1"></a>    <span class="st">&quot;free_means&quot;</span>, <span class="st">&quot;free_var&quot;</span>),</span>
<span id="cb1560-12"><a href="bias.html#cb1560-12" tabindex="-1"></a>  <span class="at">SE =</span> <span class="cn">TRUE</span>)</span>
<span id="cb1560-13"><a href="bias.html#cb1560-13" tabindex="-1"></a></span>
<span id="cb1560-14"><a href="bias.html#cb1560-14" tabindex="-1"></a><span class="fu">coef</span>(constrainedModelResolveItem5, <span class="at">simplify =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
</div>
<div id="difResolveItemsCompareModels" class="section level4 hasAnchor" number="16.9.9.4">
<h4><span class="header-section-number">16.9.9.4</span> Compare model fit<a href="bias.html#difResolveItemsCompareModels" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>We can compare the model fit against the fully constrained model.
In this case, all three of these approaches for handling <a href="bias.html#dif">DIF</a> resulted in improvement in model fit based on AIC and, for the two nested models, the chi-square difference test.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1561"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1561-1"><a href="bias.html#cb1561-1" tabindex="-1"></a><span class="fu">anova</span>(constrainedModel, constrainedModelDropItem5)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["AIC"],"name":[1],"type":["dbl"],"align":["right"]},{"label":["SABIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["HQ"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["logLik"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["X2"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["df"],"name":[7],"type":["dbl"],"align":["right"]},{"label":["p"],"name":[8],"type":["dbl"],"align":["right"]}],"data":[{"1":"34331.52","2":"34402.71","3":"34382.72","4":"34475.80","5":"-17142.76","6":"NA","7":"NA","8":"NA","_rn_":"constrainedModel"},{"1":"30777.88","2":"30839.79","3":"30822.40","4":"30903.34","5":"-15368.94","6":"3547.64","7":"2913","8":"3.441691e-15","_rn_":"constrainedModelDropItem5"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1562"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1562-1"><a href="bias.html#cb1562-1" tabindex="-1"></a><span class="fu">anova</span>(constrainedModel, constrainedModelDropItem5ForMen)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["AIC"],"name":[1],"type":["dbl"],"align":["right"]},{"label":["SABIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["HQ"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["logLik"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["X2"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["df"],"name":[7],"type":["dbl"],"align":["right"]},{"label":["p"],"name":[8],"type":["dbl"],"align":["right"]}],"data":[{"1":"34331.52","2":"34402.71","3":"34382.72","4":"34475.80","5":"-17142.76","6":"NA","7":"NA","8":"NA","_rn_":"constrainedModel"},{"1":"32179.07","2":"32250.27","3":"32230.27","4":"32323.36","5":"-16066.54","6":"2152.443","7":"0","8":"NaN","_rn_":"constrainedModelDropItem5ForMen"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1563"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1563-1"><a href="bias.html#cb1563-1" tabindex="-1"></a><span class="fu">anova</span>(constrainedModel, constrainedModelResolveItem5)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["AIC"],"name":[1],"type":["dbl"],"align":["right"]},{"label":["SABIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["HQ"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["logLik"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["X2"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["df"],"name":[7],"type":["dbl"],"align":["right"]},{"label":["p"],"name":[8],"type":["dbl"],"align":["right"]}],"data":[{"1":"34331.52","2":"34402.71","3":"34382.72","4":"34475.80","5":"-17142.76","6":"NA","7":"NA","8":"NA","_rn_":"constrainedModel"},{"1":"34291.28","2":"34371.76","3":"34349.16","4":"34454.38","5":"-17119.64","6":"46.23856","7":"3","8":"5.046279e-10","_rn_":"constrainedModelResolveItem5"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="difNextSteps" class="section level4 hasAnchor" number="16.9.9.5">
<h4><span class="header-section-number">16.9.9.5</span> Next steps<a href="bias.html#difNextSteps" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Whichever of these approaches we select, we would then identify and handle the remaining non-negligent <a href="bias.html#dif">DIF</a> sequentially by magnitude.
For instance, if we chose to resolve <a href="bias.html#dif">DIF</a> in item 5 by allowing the item to have different parameters across both groups, we would then iteratively drop constraints to see which items continue to show non-negligible <a href="bias.html#dif">DIF</a>.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1564"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1564-1"><a href="bias.html#cb1564-1" tabindex="-1"></a>difDropItemsNew <span class="ot">&lt;-</span> <span class="fu">DIF</span>(</span>
<span id="cb1564-2"><a href="bias.html#cb1564-2" tabindex="-1"></a>  constrainedModelResolveItem5,</span>
<span id="cb1564-3"><a href="bias.html#cb1564-3" tabindex="-1"></a>  <span class="fu">c</span>(<span class="st">&quot;a1&quot;</span>,<span class="st">&quot;d1&quot;</span>,<span class="st">&quot;d2&quot;</span>),</span>
<span id="cb1564-4"><a href="bias.html#cb1564-4" tabindex="-1"></a>  <span class="at">scheme =</span> <span class="st">&quot;drop&quot;</span>,</span>
<span id="cb1564-5"><a href="bias.html#cb1564-5" tabindex="-1"></a>  <span class="at">simplify =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
<table>
<caption><span id="tab:itemLevelDIFAfterResolving">Table 16.1: </span>Differential Item Functioning After Resolving DIF in Item 5.</caption>
<colgroup>
<col width="16%" />
<col width="10%" />
<col width="8%" />
<col width="10%" />
<col width="10%" />
<col width="10%" />
<col width="9%" />
<col width="9%" />
<col width="2%" />
<col width="8%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"></th>
<th align="left">groups</th>
<th align="left">converged</th>
<th align="right">AIC</th>
<th align="right">SABIC</th>
<th align="right">HQ</th>
<th align="right">BIC</th>
<th align="right">X2</th>
<th align="right">df</th>
<th align="right">p</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">bpi_antisocialT1_1</td>
<td align="left">female,male</td>
<td align="left">TRUE</td>
<td align="right">-5.5042001</td>
<td align="right">3.7824138</td>
<td align="right">1.1738421</td>
<td align="right">13.3150439</td>
<td align="right">11.5042001</td>
<td align="right">3</td>
<td align="right">0.0092897</td>
</tr>
<tr class="even">
<td align="left">bpi_antisocialT1_2</td>
<td align="left">female,male</td>
<td align="left">TRUE</td>
<td align="right">3.8539227</td>
<td align="right">13.1405366</td>
<td align="right">10.5319649</td>
<td align="right">22.6731667</td>
<td align="right">2.1460773</td>
<td align="right">3</td>
<td align="right">0.5426474</td>
</tr>
<tr class="odd">
<td align="left">bpi_antisocialT1_3</td>
<td align="left">female,male</td>
<td align="left">TRUE</td>
<td align="right">1.4534865</td>
<td align="right">10.7401004</td>
<td align="right">8.1315287</td>
<td align="right">20.2727305</td>
<td align="right">4.5465135</td>
<td align="right">3</td>
<td align="right">0.2081787</td>
</tr>
<tr class="even">
<td align="left">bpi_antisocialT1_4</td>
<td align="left">female,male</td>
<td align="left">TRUE</td>
<td align="right">-26.4683563</td>
<td align="right">-17.1817424</td>
<td align="right">-19.7903141</td>
<td align="right">-7.6491123</td>
<td align="right">32.4683563</td>
<td align="right">3</td>
<td align="right">0.0000004</td>
</tr>
<tr class="odd">
<td align="left">bpi_antisocialT1_5</td>
<td align="left">female,male</td>
<td align="left">TRUE</td>
<td align="right">0.0000966</td>
<td align="right">0.0000966</td>
<td align="right">0.0000966</td>
<td align="right">0.0000966</td>
<td align="right">-0.0000966</td>
<td align="right">0</td>
<td align="right">NaN</td>
</tr>
<tr class="even">
<td align="left">bpi_antisocialT1_6</td>
<td align="left">female,male</td>
<td align="left">TRUE</td>
<td align="right">-9.9043878</td>
<td align="right">-0.6177739</td>
<td align="right">-3.2263456</td>
<td align="right">8.9148562</td>
<td align="right">15.9043878</td>
<td align="right">3</td>
<td align="right">0.0011863</td>
</tr>
<tr class="odd">
<td align="left">bpi_antisocialT1_7</td>
<td align="left">female,male</td>
<td align="left">TRUE</td>
<td align="right">-16.5941405</td>
<td align="right">-7.3075266</td>
<td align="right">-9.9160983</td>
<td align="right">2.2251035</td>
<td align="right">22.5941405</td>
<td align="right">3</td>
<td align="right">0.0000491</td>
</tr>
</tbody>
</table>
<p>Estimates of <a href="bias.html#dif">DIF</a> after resolving item 5 are in Table <a href="bias.html#tab:itemLevelDIFAfterResolving">16.1</a>.
Of the remaining <a href="bias.html#dif">DIF</a>, item 4 appears to have the largest <a href="bias.html#dif">DIF</a>.
If we deem item 4 to show non-negligible <a href="bias.html#dif">DIF</a>, we would address it using one of the three approaches above and then see which items continue to show the largest <a href="bias.html#dif">DIF</a>, address it if necessary, then identify the remaining <a href="bias.html#dif">DIF</a>, etc.</p>
</div>
</div>
</div>
<div id="measurementInvariance" class="section level2 hasAnchor" number="16.10">
<h2><span class="header-section-number">16.10</span> Measurement/Factorial Invariance<a href="bias.html#measurementInvariance" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Before making comparisons across groups in terms of associations between constructs or their level on the construct, it is important to establish <em>measurement invariance</em> (also called <em>factorial invariance</em>) across the groups <span class="citation">(<a href="#ref-Millsap2011">Millsap, 2011</a>)</span>.
Tests of measurement invariance are equivalent to tests of <a href="bias.html#dif">differential item functioning</a>.
Measurement invariance can help provide greater confidence that the measure functions equivalently across groups, that is, the items have the same strength of association with the latent factor, and the latent factor is on the same metric (i.e., the items have the same level when accounting for the latent factor).
For instance, If you observe differences between groups in level of depression without establishing measurement invariance across the groups, you do not know whether the differences observed reflect true group-related differences in depression or differences in the functioning of the measure across the groups.</p>
<p>Measurement invariance can be tested using many methods, including <a href="factor-analysis-PCA.html#cfa">confirmatory factor analysis</a> and <a href="irt.html#irt">IRT</a>.
<a href="factor-analysis-PCA.html#cfa">CFA</a> approaches to testing measurement invariance include multi-group <a href="factor-analysis-PCA.html#cfa">CFA</a> (MGCFA), multiple-indicator, multiple-causes (MIMIC) models, moderated nonlinear factor analysis (MNLFA), score-based tests <span class="citation">(<a href="#ref-Wang2014a">T. Wang et al., 2014</a>)</span>, and the alignment method <span class="citation">(<a href="#ref-Lai2021">Lai, 2021</a>)</span>.
The <a href="irt.html#irt">IRT</a> approach to testing measurement (non-)invariance is to examine whether items show <a href="bias.html#dif">differential item functioning</a>.</p>
<p>The tests of measurement invariance in <a href="factor-analysis-PCA.html#cfa">confirmatory factor analysis</a> (CFA) models were fit in the <code>lavaan</code> package <span class="citation">(<a href="#ref-R-lavaan">Rosseel et al., 2022</a>)</span>.
The examples were adapted from <code>lavaan</code> documentation: <a href="https://lavaan.ugent.be/tutorial/groups.html" class="uri">https://lavaan.ugent.be/tutorial/groups.html</a> (archived at <a href="https://perma.cc/2FBK-RSAH" class="uri">https://perma.cc/2FBK-RSAH</a>).
Procedures for testing measurement invariance are outlined in <span class="citation">Putnick &amp; Bornstein (<a href="#ref-Putnick2016">2016</a>)</span>.
In addition to nested chi-square difference tests (<span class="math inline">\(\chi^2_{\text{diff}}\)</span>), I also demonstrate permutation procedures for testing measurement invariance, as described by <span class="citation">Jorgensen et al. (<a href="#ref-Jorgensen2018">2018</a>)</span>.
Also, you are encouraged to read about MIMIC models <span class="citation">(<a href="#ref-Cheng2016">Cheng et al., 2016</a>; <a href="#ref-Wang2009b">W.-C. Wang et al., 2009</a>)</span>, score-based tests <span class="citation">(<a href="#ref-Wang2014a">T. Wang et al., 2014</a>)</span>, and MNLFA that allows for testing measurement invariance across continuous moderators <span class="citation">(<a href="#ref-Bauer2020">Bauer et al., 2020</a>; <a href="#ref-Curran2014">Curran et al., 2014</a>; <a href="#ref-Gottfredson2019">N. C. Gottfredson et al., 2019</a>)</span>.
MNLFA is implemented in the <code>mnlfa</code> package <span class="citation">(<a href="#ref-R-mnlfa">Robitzsch, 2019</a>)</span>.
You can also generate syntax for conducting MNLFA in Mplus software <span class="citation">(<a href="#ref-Muthen2019">Muthén &amp; Muthén, 2019</a>)</span> using the <code>aMNLFA</code> package <span class="citation">(<a href="#ref-R-aMNLFA">V. Cole et al., 2018</a>)</span>.
The alignment method allows many groups to be compared <span class="citation">(<a href="#ref-Han2019">Han et al., 2019</a>)</span>.</p>
<p>Model fit of nested models can be compared with a chi-square difference test, as a way of testing measurement invariance.
In this approach to testing measurement invariance, first model fit is evaluated in a configural invariance model, in which the same number of factors is specified in each group, and which indicators load on which factors are the same in each group.
Then, successive constraints are made across groups, including constraining factor loadings, intercepts, and residuals across groups.
The metric (“weak factorial”) invariance model is similar to the configural invariance model, but it constrains the factor loadings to be the same across groups.
The scalar (“strong factorial”) invariance model keeps the constraints of the metric invariance model, but it also constrains the intercepts to be the same across groups.
The residual (“strict factorial”) invariance model keeps the constraints of the scalar invariance model, but it also constrains the residuals to be the same across groups.
The fit of each constrained model is compared to the previous model without such constraints.
That is, the fit of the metric invariance model is compared to the fit of the configural invariance model, the fit of the scalar invariance model is compared to the fit of the metric invariance model, and the fit of the residual invariance model is compared to the fit of the scalar invariance model.</p>
<p>The chi-square difference test is sensitive to sample size, and trivial differences in fit can be detected with large samples <span class="citation">(<a href="#ref-Cheung2002">Cheung &amp; Rensvold, 2002</a>)</span>.
As a result, researchers also recommend examining change in additional criteria, including CFI, RMSEA, and SRMR <span class="citation">(<a href="#ref-Chen2007a">F. F. Chen, 2007</a>)</span>.
<span class="citation">Cheung &amp; Rensvold (<a href="#ref-Cheung2002">2002</a>)</span> recommend a cutoff of <span class="math inline">\(\Delta \text{CFI} \geq -.01\)</span> for identifying measurement non-invariance.
<span class="citation">F. F. Chen (<a href="#ref-Chen2007a">2007</a>)</span> recommends the following cutoffs for identifying measurement non-invariance:</p>
<ul>
<li>with a small sample size (total <span class="math inline">\(N \leq 300\)</span>):
<ul>
<li>testing invariance of factor loadings:
<ul>
<li><span class="math inline">\(\Delta \text{CFI} \geq -.005\)</span></li>
<li>supplemented by <span class="math inline">\(\Delta \text{RMSEA} \geq .010\)</span> or <span class="math inline">\(\Delta \text{SRMR} \geq .025\)</span></li>
</ul></li>
<li>testing invariance of intercepts or residuals:
<ul>
<li><span class="math inline">\(\Delta \text{CFI} \geq -.005\)</span></li>
<li>supplemented by <span class="math inline">\(\Delta \text{RMSEA} \geq .010\)</span> or <span class="math inline">\(\Delta \text{SRMR} \geq .005\)</span></li>
</ul></li>
</ul></li>
<li>with an adequate sample size (total <span class="math inline">\(N &gt; 300\)</span>):
<ul>
<li>testing invariance of factor loadings:
<ul>
<li><span class="math inline">\(\Delta \text{CFI} \geq -.010\)</span></li>
<li>supplemented by <span class="math inline">\(\Delta \text{RMSEA} \geq .015\)</span> or <span class="math inline">\(\Delta \text{SRMR} \geq .030\)</span></li>
</ul></li>
<li>testing invariance of intercepts or residuals:
<ul>
<li><span class="math inline">\(\Delta \text{CFI} \geq -.005\)</span></li>
<li>supplemented by <span class="math inline">\(\Delta \text{RMSEA} \geq .015\)</span> or <span class="math inline">\(\Delta \text{SRMR} \geq .010\)</span></li>
</ul></li>
</ul></li>
</ul>
<p><span class="citation">Little et al. (<a href="#ref-Little2007">2007</a>)</span> suggested that researchers establish at least partial invariance of factor loadings (metric invariance) to compare covariances across groups, and at least partial invariance of intercepts (scalar invariance) to compare mean levels across groups.
Partial invariance refers to invariance with some but not all indicators.
So, to examine associations with other variables, invariance of at least some factor loadings would be preferable.
To examine differences in level or growth, invariance of at least some factor loadings and intercepts would be preferable.
Residual invariance is considered overly restrictive, and it is not generally expected that one establish residual invariance <span class="citation">(<a href="#ref-Little2013">Little, 2013</a>)</span>.</p>
<p>Although measurement invariance is important to test, it is also worth noting that tests of measurement invariance and <a href="bias.html#dif">DIF</a> rest on various fundamentally untestable assumptions related to scale setting <span class="citation">(<a href="#ref-Raykovinpress">Raykov et al., 2020</a>)</span>.
For instance, to give the latent factor units, oftentimes researchers set one item’s factor loading to be equal across time (i.e., the marker variable).
Results of factorial invariance tests can depend highly on which item is used as the anchor item for setting the scale of the latent factor <span class="citation">(<a href="#ref-Belzak2020">Belzak &amp; Bauer, 2020</a>)</span>.
And subsequent tests of measurement invariance then rest on the fundamentally untestable assumption that changes in the level of the latent factor precipitates the same amount of change in the item across groups.
If this scaling is not proper, however, it could lead to improper conclusions.
Researchers are, therefore, not conclusively able to establish measurement invariance.</p>
<p>Several possible approaches may help address this.
First, it can be helpful to place greater focus on <em>degree</em> of measurement invariance and confidence (rather than presence versus absence of measurement invariance).
To the extent that non-invariance is trivial in effect size, it provides the researcher with greater confidence that they can use the measure to assess a construct in a comparable way over time.
A number of studies describe how to test the effect size of measurement invariance <span class="citation">(<a href="#ref-Gunn2020">Gunn et al., 2020</a>; e.g., <a href="#ref-Liu2017a">Liu et al., 2017</a>)</span> or differential item functioning <span class="citation">(<a href="#ref-Gonzalez2021">Gonzalez &amp; Pelham, 2021</a>; e.g., <a href="#ref-Meade2010">Meade, 2010</a>)</span>.
Second, there may be important robustness checks.
I describe each in greater detail below.
One important sensitivity analysis could be to see if measurement invariance holds when using different marker variables.
This would provide greater evidence that their apparent measurement invariance was not specific to one marker variable (i.e., one particular set of assumptions).
A second robustness check would be to use effects coding, in which the average of items’ factor loadings is equal to 1, so the metric of the latent variable is on the metric of all of the items rather than just one item <span class="citation">(<a href="#ref-Little2006a">Little et al., 2006</a>)</span>.
A third robustness check would be to use regularization, which is an alternative method to select the anchor items and to identify differential item functioning based on a machine learning technique that applies penalization to remove parameters that have little impact on model fit <span class="citation">(<a href="#ref-Bauer2020">Bauer et al., 2020</a>; <a href="#ref-Belzak2020">Belzak &amp; Bauer, 2020</a>)</span>.</p>
<p>Another issue with measurement invariance is that the null hypothesis that adding constraints across groups will not worsen fit is likely always false, because there are often at least slight differences across groups in factor loadings, intercepts, or residuals that reflect sampling variability.
However, we are not interested in trivial differences across groups that reflect sampling variability.
Instead, we are interested in the extent to which there are substantive, meaningful differences across the groups of large enough magnitude to be practically meaningful.
Thus, it can also be helpful to consider whether the measures show <em>approximate measurement invariance</em> <span class="citation">(<a href="#ref-VanDeSchoot2015">Van De Schoot et al., 2015</a>)</span>.
For details on how to test approximate measurement invariance, see <span class="citation">Van De Schoot et al. (<a href="#ref-VanDeSchoot2013">2013</a>)</span>.</p>
<p>When detecting measurement non-invariance in a given parameter (e.g., factor loadings, intercepts, or residuals), one can identify the specific items that show measurement non-invariance in one of two primary ways: (1) starting with a model that allows the given parameter to differ across groups, a researcher can iteratively add constraints to identify the item(s) for which measurement invariance fails, or (2) starting with a model that constrains the given parameter to be the same across groups, a researcher can iteratively remove constraints to identify the item(s) for which measurement invariance becomes established (and by process of elimination, the items for which measurement invariance does not become established).</p>
<p>Approaches to addressing measurement non-invariance are described in Section <a href="bias.html#whenDetectingBias">16.5.1</a>.
If we identify that an item shows non-negligible non-invariance, we have three primary options (described above): (1) drop the item for both groups, (2) drop the item for one group but keep it for the other group, or (3) freely estimate the parameters (factor loadings and/or intercepts) for the item across groups.</p>
<p>Using the traditional chi-square difference test, tests of measurement invariance compare the model fit to a model with perfect fit.
As the sample size grows larger, smaller differences in model fit will be detected as significant, thus rejecting measurement invariance even if the model fits well.
So, it can also be useful to compare the model fit to a null hypothesis that the fit is poor, instead of the null hypothesis that the model is a perfect fit.
<em>Chi-square equivalence tests</em> use the poor model fit as the null hypothesis.
Thus, a significant chi-square equivalence test suggests that the equality constraints are plausible, providing support for the alternative hypothesis that the model fit is acceptable <span class="citation">(<a href="#ref-Counsell2020">Counsell et al., 2020</a>)</span>.</p>
<div id="measurementInvarianceModels" class="section level3 hasAnchor" number="16.10.1">
<h3><span class="header-section-number">16.10.1</span> Specify Models<a href="bias.html#measurementInvarianceModels" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="measurementInvarianceNullModelSyntax" class="section level4 hasAnchor" number="16.10.1.1">
<h4><span class="header-section-number">16.10.1.1</span> Null Model<a href="bias.html#measurementInvarianceNullModelSyntax" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Fix residual variances and intercepts of manifest variables to be equal across groups.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1565"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1565-1"><a href="bias.html#cb1565-1" tabindex="-1"></a>nullModel <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1565-2"><a href="bias.html#cb1565-2" tabindex="-1"></a><span class="st">#Fix residual variances of manifest variables to be equal across groups</span></span>
<span id="cb1565-3"><a href="bias.html#cb1565-3" tabindex="-1"></a><span class="st">x1 ~~ c(psi1, psi1)*x1</span></span>
<span id="cb1565-4"><a href="bias.html#cb1565-4" tabindex="-1"></a><span class="st">x2 ~~ c(psi2, psi2)*x2</span></span>
<span id="cb1565-5"><a href="bias.html#cb1565-5" tabindex="-1"></a><span class="st">x3 ~~ c(psi3, psi3)*x3</span></span>
<span id="cb1565-6"><a href="bias.html#cb1565-6" tabindex="-1"></a><span class="st">x4 ~~ c(psi4, psi4)*x4</span></span>
<span id="cb1565-7"><a href="bias.html#cb1565-7" tabindex="-1"></a><span class="st">x5 ~~ c(psi5, psi5)*x5</span></span>
<span id="cb1565-8"><a href="bias.html#cb1565-8" tabindex="-1"></a><span class="st">x6 ~~ c(psi6, psi6)*x6</span></span>
<span id="cb1565-9"><a href="bias.html#cb1565-9" tabindex="-1"></a><span class="st">x7 ~~ c(psi7, psi7)*x7</span></span>
<span id="cb1565-10"><a href="bias.html#cb1565-10" tabindex="-1"></a><span class="st">x8 ~~ c(psi8, psi8)*x8</span></span>
<span id="cb1565-11"><a href="bias.html#cb1565-11" tabindex="-1"></a><span class="st">x9 ~~ c(psi9, psi9)*x9</span></span>
<span id="cb1565-12"><a href="bias.html#cb1565-12" tabindex="-1"></a></span>
<span id="cb1565-13"><a href="bias.html#cb1565-13" tabindex="-1"></a><span class="st">#Fix intercepts of manifest variables to be equal across groups</span></span>
<span id="cb1565-14"><a href="bias.html#cb1565-14" tabindex="-1"></a><span class="st">x1 ~ c(tau1, tau1)*1</span></span>
<span id="cb1565-15"><a href="bias.html#cb1565-15" tabindex="-1"></a><span class="st">x2 ~ c(tau2, tau2)*1</span></span>
<span id="cb1565-16"><a href="bias.html#cb1565-16" tabindex="-1"></a><span class="st">x3 ~ c(tau3, tau3)*1</span></span>
<span id="cb1565-17"><a href="bias.html#cb1565-17" tabindex="-1"></a><span class="st">x4 ~ c(tau4, tau4)*1</span></span>
<span id="cb1565-18"><a href="bias.html#cb1565-18" tabindex="-1"></a><span class="st">x5 ~ c(tau5, tau5)*1</span></span>
<span id="cb1565-19"><a href="bias.html#cb1565-19" tabindex="-1"></a><span class="st">x6 ~ c(tau6, tau6)*1</span></span>
<span id="cb1565-20"><a href="bias.html#cb1565-20" tabindex="-1"></a><span class="st">x7 ~ c(tau7, tau7)*1</span></span>
<span id="cb1565-21"><a href="bias.html#cb1565-21" tabindex="-1"></a><span class="st">x8 ~ c(tau8, tau8)*1</span></span>
<span id="cb1565-22"><a href="bias.html#cb1565-22" tabindex="-1"></a><span class="st">x9 ~ c(tau9, tau9)*1</span></span>
<span id="cb1565-23"><a href="bias.html#cb1565-23" tabindex="-1"></a><span class="st">&#39;</span></span></code></pre></div>
</details>
</div>
<div id="measurementInvarianceCFAsyntax" class="section level4 hasAnchor" number="16.10.1.2">
<h4><span class="header-section-number">16.10.1.2</span> CFA Model<a href="bias.html#measurementInvarianceCFAsyntax" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1566"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1566-1"><a href="bias.html#cb1566-1" tabindex="-1"></a>cfaModel <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1566-2"><a href="bias.html#cb1566-2" tabindex="-1"></a><span class="st"> #Factor loadings</span></span>
<span id="cb1566-3"><a href="bias.html#cb1566-3" tabindex="-1"></a><span class="st"> visual  =~ x1 + x2 + x3</span></span>
<span id="cb1566-4"><a href="bias.html#cb1566-4" tabindex="-1"></a><span class="st"> textual =~ x4 + x5 + x6</span></span>
<span id="cb1566-5"><a href="bias.html#cb1566-5" tabindex="-1"></a><span class="st"> speed   =~ x7 + x8 + x9</span></span>
<span id="cb1566-6"><a href="bias.html#cb1566-6" tabindex="-1"></a><span class="st">&#39;</span></span></code></pre></div>
</details>
</div>
<div id="measurementInvarianceConfiguralSyntax" class="section level4 hasAnchor" number="16.10.1.3">
<h4><span class="header-section-number">16.10.1.3</span> Configural Invariance<a href="bias.html#measurementInvarianceConfiguralSyntax" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Specify the same number of factors in each group, and which indicators load on which factors are the same in each group.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1567"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1567-1"><a href="bias.html#cb1567-1" tabindex="-1"></a>cfaModel_configuralInvariance <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1567-2"><a href="bias.html#cb1567-2" tabindex="-1"></a><span class="st"> #Factor loadings (free the factor loading of the first indicator)</span></span>
<span id="cb1567-3"><a href="bias.html#cb1567-3" tabindex="-1"></a><span class="st"> visual  =~ NA*x1 + x2 + x3</span></span>
<span id="cb1567-4"><a href="bias.html#cb1567-4" tabindex="-1"></a><span class="st"> textual =~ NA*x4 + x5 + x6</span></span>
<span id="cb1567-5"><a href="bias.html#cb1567-5" tabindex="-1"></a><span class="st"> speed   =~ NA*x7 + x8 + x9</span></span>
<span id="cb1567-6"><a href="bias.html#cb1567-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1567-7"><a href="bias.html#cb1567-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero</span></span>
<span id="cb1567-8"><a href="bias.html#cb1567-8" tabindex="-1"></a><span class="st"> visual ~ 0</span></span>
<span id="cb1567-9"><a href="bias.html#cb1567-9" tabindex="-1"></a><span class="st"> textual ~ 0</span></span>
<span id="cb1567-10"><a href="bias.html#cb1567-10" tabindex="-1"></a><span class="st"> speed ~ 0</span></span>
<span id="cb1567-11"><a href="bias.html#cb1567-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1567-12"><a href="bias.html#cb1567-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one</span></span>
<span id="cb1567-13"><a href="bias.html#cb1567-13" tabindex="-1"></a><span class="st"> visual ~~ 1*visual</span></span>
<span id="cb1567-14"><a href="bias.html#cb1567-14" tabindex="-1"></a><span class="st"> textual ~~ 1*textual</span></span>
<span id="cb1567-15"><a href="bias.html#cb1567-15" tabindex="-1"></a><span class="st"> speed ~~ 1*speed</span></span>
<span id="cb1567-16"><a href="bias.html#cb1567-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1567-17"><a href="bias.html#cb1567-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1567-18"><a href="bias.html#cb1567-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1567-19"><a href="bias.html#cb1567-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1567-20"><a href="bias.html#cb1567-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1567-21"><a href="bias.html#cb1567-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1567-22"><a href="bias.html#cb1567-22" tabindex="-1"></a><span class="st"> #Estimate residual variances of manifest variables</span></span>
<span id="cb1567-23"><a href="bias.html#cb1567-23" tabindex="-1"></a><span class="st"> x1 ~~ x1</span></span>
<span id="cb1567-24"><a href="bias.html#cb1567-24" tabindex="-1"></a><span class="st"> x2 ~~ x2</span></span>
<span id="cb1567-25"><a href="bias.html#cb1567-25" tabindex="-1"></a><span class="st"> x3 ~~ x3</span></span>
<span id="cb1567-26"><a href="bias.html#cb1567-26" tabindex="-1"></a><span class="st"> x4 ~~ x4</span></span>
<span id="cb1567-27"><a href="bias.html#cb1567-27" tabindex="-1"></a><span class="st"> x5 ~~ x5</span></span>
<span id="cb1567-28"><a href="bias.html#cb1567-28" tabindex="-1"></a><span class="st"> x6 ~~ x6</span></span>
<span id="cb1567-29"><a href="bias.html#cb1567-29" tabindex="-1"></a><span class="st"> x7 ~~ x7</span></span>
<span id="cb1567-30"><a href="bias.html#cb1567-30" tabindex="-1"></a><span class="st"> x8 ~~ x8</span></span>
<span id="cb1567-31"><a href="bias.html#cb1567-31" tabindex="-1"></a><span class="st"> x9 ~~ x9</span></span>
<span id="cb1567-32"><a href="bias.html#cb1567-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1567-33"><a href="bias.html#cb1567-33" tabindex="-1"></a><span class="st"> #Free intercepts of manifest variables</span></span>
<span id="cb1567-34"><a href="bias.html#cb1567-34" tabindex="-1"></a><span class="st"> x1 ~ NA*1</span></span>
<span id="cb1567-35"><a href="bias.html#cb1567-35" tabindex="-1"></a><span class="st"> x2 ~ NA*1</span></span>
<span id="cb1567-36"><a href="bias.html#cb1567-36" tabindex="-1"></a><span class="st"> x3 ~ NA*1</span></span>
<span id="cb1567-37"><a href="bias.html#cb1567-37" tabindex="-1"></a><span class="st"> x4 ~ NA*1</span></span>
<span id="cb1567-38"><a href="bias.html#cb1567-38" tabindex="-1"></a><span class="st"> x5 ~ NA*1</span></span>
<span id="cb1567-39"><a href="bias.html#cb1567-39" tabindex="-1"></a><span class="st"> x6 ~ NA*1</span></span>
<span id="cb1567-40"><a href="bias.html#cb1567-40" tabindex="-1"></a><span class="st"> x7 ~ NA*1</span></span>
<span id="cb1567-41"><a href="bias.html#cb1567-41" tabindex="-1"></a><span class="st"> x8 ~ NA*1</span></span>
<span id="cb1567-42"><a href="bias.html#cb1567-42" tabindex="-1"></a><span class="st"> x9 ~ NA*1</span></span>
<span id="cb1567-43"><a href="bias.html#cb1567-43" tabindex="-1"></a><span class="st">&#39;</span></span></code></pre></div>
</details>
</div>
<div id="measurementInvarianceMetricSyntax" class="section level4 hasAnchor" number="16.10.1.4">
<h4><span class="header-section-number">16.10.1.4</span> Metric (“Weak Factorial”) Invariance<a href="bias.html#measurementInvarianceMetricSyntax" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Specify invariance of factor loadings across groups.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1568"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1568-1"><a href="bias.html#cb1568-1" tabindex="-1"></a>cfaModel_metricInvariance <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1568-2"><a href="bias.html#cb1568-2" tabindex="-1"></a><span class="st"> #Fix factor loadings to be the same across groups</span></span>
<span id="cb1568-3"><a href="bias.html#cb1568-3" tabindex="-1"></a><span class="st"> visual  =~ c(lambdax1,lambdax1)*x1 + c(lambdax2,lambdax2)*x2 + c(lambdax3,lambdax3)*x3</span></span>
<span id="cb1568-4"><a href="bias.html#cb1568-4" tabindex="-1"></a><span class="st"> textual =~ c(lambdax4,lambdax4)*x4 + c(lambdax5,lambdax5)*x5 + c(lambdax6,lambdax6)*x6</span></span>
<span id="cb1568-5"><a href="bias.html#cb1568-5" tabindex="-1"></a><span class="st"> speed   =~ c(lambdax7,lambdax7)*x7 + c(lambdax8,lambdax8)*x8 + c(lambdax9,lambdax9)*x9</span></span>
<span id="cb1568-6"><a href="bias.html#cb1568-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1568-7"><a href="bias.html#cb1568-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero</span></span>
<span id="cb1568-8"><a href="bias.html#cb1568-8" tabindex="-1"></a><span class="st"> visual ~ 0</span></span>
<span id="cb1568-9"><a href="bias.html#cb1568-9" tabindex="-1"></a><span class="st"> textual ~ 0</span></span>
<span id="cb1568-10"><a href="bias.html#cb1568-10" tabindex="-1"></a><span class="st"> speed ~ 0</span></span>
<span id="cb1568-11"><a href="bias.html#cb1568-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1568-12"><a href="bias.html#cb1568-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one in group 1; free latent variances in group 2</span></span>
<span id="cb1568-13"><a href="bias.html#cb1568-13" tabindex="-1"></a><span class="st"> visual ~~ c(1, NA)*visual</span></span>
<span id="cb1568-14"><a href="bias.html#cb1568-14" tabindex="-1"></a><span class="st"> textual ~~ c(1, NA)*textual</span></span>
<span id="cb1568-15"><a href="bias.html#cb1568-15" tabindex="-1"></a><span class="st"> speed ~~ c(1, NA)*speed</span></span>
<span id="cb1568-16"><a href="bias.html#cb1568-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1568-17"><a href="bias.html#cb1568-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1568-18"><a href="bias.html#cb1568-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1568-19"><a href="bias.html#cb1568-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1568-20"><a href="bias.html#cb1568-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1568-21"><a href="bias.html#cb1568-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1568-22"><a href="bias.html#cb1568-22" tabindex="-1"></a><span class="st"> #Estimate residual variances of manifest variables</span></span>
<span id="cb1568-23"><a href="bias.html#cb1568-23" tabindex="-1"></a><span class="st"> x1 ~~ x1</span></span>
<span id="cb1568-24"><a href="bias.html#cb1568-24" tabindex="-1"></a><span class="st"> x2 ~~ x2</span></span>
<span id="cb1568-25"><a href="bias.html#cb1568-25" tabindex="-1"></a><span class="st"> x3 ~~ x3</span></span>
<span id="cb1568-26"><a href="bias.html#cb1568-26" tabindex="-1"></a><span class="st"> x4 ~~ x4</span></span>
<span id="cb1568-27"><a href="bias.html#cb1568-27" tabindex="-1"></a><span class="st"> x5 ~~ x5</span></span>
<span id="cb1568-28"><a href="bias.html#cb1568-28" tabindex="-1"></a><span class="st"> x6 ~~ x6</span></span>
<span id="cb1568-29"><a href="bias.html#cb1568-29" tabindex="-1"></a><span class="st"> x7 ~~ x7</span></span>
<span id="cb1568-30"><a href="bias.html#cb1568-30" tabindex="-1"></a><span class="st"> x8 ~~ x8</span></span>
<span id="cb1568-31"><a href="bias.html#cb1568-31" tabindex="-1"></a><span class="st"> x9 ~~ x9</span></span>
<span id="cb1568-32"><a href="bias.html#cb1568-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1568-33"><a href="bias.html#cb1568-33" tabindex="-1"></a><span class="st"> #Free intercepts of manifest variables</span></span>
<span id="cb1568-34"><a href="bias.html#cb1568-34" tabindex="-1"></a><span class="st"> x1 ~ NA*1</span></span>
<span id="cb1568-35"><a href="bias.html#cb1568-35" tabindex="-1"></a><span class="st"> x2 ~ NA*1</span></span>
<span id="cb1568-36"><a href="bias.html#cb1568-36" tabindex="-1"></a><span class="st"> x3 ~ NA*1</span></span>
<span id="cb1568-37"><a href="bias.html#cb1568-37" tabindex="-1"></a><span class="st"> x4 ~ NA*1</span></span>
<span id="cb1568-38"><a href="bias.html#cb1568-38" tabindex="-1"></a><span class="st"> x5 ~ NA*1</span></span>
<span id="cb1568-39"><a href="bias.html#cb1568-39" tabindex="-1"></a><span class="st"> x6 ~ NA*1</span></span>
<span id="cb1568-40"><a href="bias.html#cb1568-40" tabindex="-1"></a><span class="st"> x7 ~ NA*1</span></span>
<span id="cb1568-41"><a href="bias.html#cb1568-41" tabindex="-1"></a><span class="st"> x8 ~ NA*1</span></span>
<span id="cb1568-42"><a href="bias.html#cb1568-42" tabindex="-1"></a><span class="st"> x9 ~ NA*1</span></span>
<span id="cb1568-43"><a href="bias.html#cb1568-43" tabindex="-1"></a><span class="st">&#39;</span></span></code></pre></div>
</details>
</div>
<div id="measurementInvarianceScalarSyntax" class="section level4 hasAnchor" number="16.10.1.5">
<h4><span class="header-section-number">16.10.1.5</span> Scalar (“Strong Factorial”) Invariance<a href="bias.html#measurementInvarianceScalarSyntax" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Specify invariance of factor loadings and intercepts across groups.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1569"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1569-1"><a href="bias.html#cb1569-1" tabindex="-1"></a>cfaModel_scalarInvariance <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1569-2"><a href="bias.html#cb1569-2" tabindex="-1"></a><span class="st"> #Fix factor loadings to be the same across groups</span></span>
<span id="cb1569-3"><a href="bias.html#cb1569-3" tabindex="-1"></a><span class="st"> visual  =~ c(lambdax1,lambdax1)*x1 + c(lambdax2,lambdax2)*x2 + c(lambdax3,lambdax3)*x3</span></span>
<span id="cb1569-4"><a href="bias.html#cb1569-4" tabindex="-1"></a><span class="st"> textual =~ c(lambdax4,lambdax4)*x4 + c(lambdax5,lambdax5)*x5 + c(lambdax6,lambdax6)*x6</span></span>
<span id="cb1569-5"><a href="bias.html#cb1569-5" tabindex="-1"></a><span class="st"> speed   =~ c(lambdax7,lambdax7)*x7 + c(lambdax8,lambdax8)*x8 + c(lambdax9,lambdax9)*x9</span></span>
<span id="cb1569-6"><a href="bias.html#cb1569-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1569-7"><a href="bias.html#cb1569-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero in group 1; free latent means in group 2</span></span>
<span id="cb1569-8"><a href="bias.html#cb1569-8" tabindex="-1"></a><span class="st"> visual ~ c(0, NA)*1</span></span>
<span id="cb1569-9"><a href="bias.html#cb1569-9" tabindex="-1"></a><span class="st"> textual ~ c(0, NA)*1</span></span>
<span id="cb1569-10"><a href="bias.html#cb1569-10" tabindex="-1"></a><span class="st"> speed ~ c(0, NA)*1</span></span>
<span id="cb1569-11"><a href="bias.html#cb1569-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1569-12"><a href="bias.html#cb1569-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one in group 1; free latent variances in group 2</span></span>
<span id="cb1569-13"><a href="bias.html#cb1569-13" tabindex="-1"></a><span class="st"> visual ~~ c(1, NA)*visual</span></span>
<span id="cb1569-14"><a href="bias.html#cb1569-14" tabindex="-1"></a><span class="st"> textual ~~ c(1, NA)*textual</span></span>
<span id="cb1569-15"><a href="bias.html#cb1569-15" tabindex="-1"></a><span class="st"> speed ~~ c(1, NA)*speed</span></span>
<span id="cb1569-16"><a href="bias.html#cb1569-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1569-17"><a href="bias.html#cb1569-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1569-18"><a href="bias.html#cb1569-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1569-19"><a href="bias.html#cb1569-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1569-20"><a href="bias.html#cb1569-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1569-21"><a href="bias.html#cb1569-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1569-22"><a href="bias.html#cb1569-22" tabindex="-1"></a><span class="st"> #Estimate residual variances of manifest variables</span></span>
<span id="cb1569-23"><a href="bias.html#cb1569-23" tabindex="-1"></a><span class="st"> x1 ~~ x1</span></span>
<span id="cb1569-24"><a href="bias.html#cb1569-24" tabindex="-1"></a><span class="st"> x2 ~~ x2</span></span>
<span id="cb1569-25"><a href="bias.html#cb1569-25" tabindex="-1"></a><span class="st"> x3 ~~ x3</span></span>
<span id="cb1569-26"><a href="bias.html#cb1569-26" tabindex="-1"></a><span class="st"> x4 ~~ x4</span></span>
<span id="cb1569-27"><a href="bias.html#cb1569-27" tabindex="-1"></a><span class="st"> x5 ~~ x5</span></span>
<span id="cb1569-28"><a href="bias.html#cb1569-28" tabindex="-1"></a><span class="st"> x6 ~~ x6</span></span>
<span id="cb1569-29"><a href="bias.html#cb1569-29" tabindex="-1"></a><span class="st"> x7 ~~ x7</span></span>
<span id="cb1569-30"><a href="bias.html#cb1569-30" tabindex="-1"></a><span class="st"> x8 ~~ x8</span></span>
<span id="cb1569-31"><a href="bias.html#cb1569-31" tabindex="-1"></a><span class="st"> x9 ~~ x9</span></span>
<span id="cb1569-32"><a href="bias.html#cb1569-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1569-33"><a href="bias.html#cb1569-33" tabindex="-1"></a><span class="st"> #Fix intercepts of manifest variables across groups</span></span>
<span id="cb1569-34"><a href="bias.html#cb1569-34" tabindex="-1"></a><span class="st"> x1 ~ c(intx1, intx1)*1</span></span>
<span id="cb1569-35"><a href="bias.html#cb1569-35" tabindex="-1"></a><span class="st"> x2 ~ c(intx2, intx2)*1</span></span>
<span id="cb1569-36"><a href="bias.html#cb1569-36" tabindex="-1"></a><span class="st"> x3 ~ c(intx3, intx3)*1</span></span>
<span id="cb1569-37"><a href="bias.html#cb1569-37" tabindex="-1"></a><span class="st"> x4 ~ c(intx4, intx4)*1</span></span>
<span id="cb1569-38"><a href="bias.html#cb1569-38" tabindex="-1"></a><span class="st"> x5 ~ c(intx5, intx5)*1</span></span>
<span id="cb1569-39"><a href="bias.html#cb1569-39" tabindex="-1"></a><span class="st"> x6 ~ c(intx6, intx6)*1</span></span>
<span id="cb1569-40"><a href="bias.html#cb1569-40" tabindex="-1"></a><span class="st"> x7 ~ c(intx7, intx7)*1</span></span>
<span id="cb1569-41"><a href="bias.html#cb1569-41" tabindex="-1"></a><span class="st"> x8 ~ c(intx8, intx8)*1</span></span>
<span id="cb1569-42"><a href="bias.html#cb1569-42" tabindex="-1"></a><span class="st"> x9 ~ c(intx9, intx9)*1</span></span>
<span id="cb1569-43"><a href="bias.html#cb1569-43" tabindex="-1"></a><span class="st">&#39;</span></span></code></pre></div>
</details>
</div>
<div id="measurementInvarianceResidualSyntax" class="section level4 hasAnchor" number="16.10.1.6">
<h4><span class="header-section-number">16.10.1.6</span> Residual (“Strict Factorial”) Invariance<a href="bias.html#measurementInvarianceResidualSyntax" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Specify invariance of factor loadings, intercepts, and residuals across groups.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1570"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1570-1"><a href="bias.html#cb1570-1" tabindex="-1"></a>cfaModel_residualInvariance <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1570-2"><a href="bias.html#cb1570-2" tabindex="-1"></a><span class="st"> #Fix factor loadings to be the same across groups</span></span>
<span id="cb1570-3"><a href="bias.html#cb1570-3" tabindex="-1"></a><span class="st"> visual  =~ c(lambdax1,lambdax1)*x1 + c(lambdax2,lambdax2)*x2 + c(lambdax3,lambdax3)*x3</span></span>
<span id="cb1570-4"><a href="bias.html#cb1570-4" tabindex="-1"></a><span class="st"> textual =~ c(lambdax4,lambdax4)*x4 + c(lambdax5,lambdax5)*x5 + c(lambdax6,lambdax6)*x6</span></span>
<span id="cb1570-5"><a href="bias.html#cb1570-5" tabindex="-1"></a><span class="st"> speed   =~ c(lambdax7,lambdax7)*x7 + c(lambdax8,lambdax8)*x8 + c(lambdax9,lambdax9)*x9</span></span>
<span id="cb1570-6"><a href="bias.html#cb1570-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1570-7"><a href="bias.html#cb1570-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero</span></span>
<span id="cb1570-8"><a href="bias.html#cb1570-8" tabindex="-1"></a><span class="st"> visual ~ c(0, NA)*1</span></span>
<span id="cb1570-9"><a href="bias.html#cb1570-9" tabindex="-1"></a><span class="st"> textual ~ c(0, NA)*1</span></span>
<span id="cb1570-10"><a href="bias.html#cb1570-10" tabindex="-1"></a><span class="st"> speed ~ c(0, NA)*1</span></span>
<span id="cb1570-11"><a href="bias.html#cb1570-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1570-12"><a href="bias.html#cb1570-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one in group 1; free latent variances in group 2</span></span>
<span id="cb1570-13"><a href="bias.html#cb1570-13" tabindex="-1"></a><span class="st"> visual ~~ c(1, NA)*visual</span></span>
<span id="cb1570-14"><a href="bias.html#cb1570-14" tabindex="-1"></a><span class="st"> textual ~~ c(1, NA)*textual</span></span>
<span id="cb1570-15"><a href="bias.html#cb1570-15" tabindex="-1"></a><span class="st"> speed ~~ c(1, NA)*speed</span></span>
<span id="cb1570-16"><a href="bias.html#cb1570-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1570-17"><a href="bias.html#cb1570-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1570-18"><a href="bias.html#cb1570-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1570-19"><a href="bias.html#cb1570-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1570-20"><a href="bias.html#cb1570-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1570-21"><a href="bias.html#cb1570-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1570-22"><a href="bias.html#cb1570-22" tabindex="-1"></a><span class="st"> #Fix residual variances of manifest variables across groups</span></span>
<span id="cb1570-23"><a href="bias.html#cb1570-23" tabindex="-1"></a><span class="st"> x1 ~~ c(residx1, residx1)*x1</span></span>
<span id="cb1570-24"><a href="bias.html#cb1570-24" tabindex="-1"></a><span class="st"> x2 ~~ c(residx2, residx2)*x2</span></span>
<span id="cb1570-25"><a href="bias.html#cb1570-25" tabindex="-1"></a><span class="st"> x3 ~~ c(residx3, residx3)*x3</span></span>
<span id="cb1570-26"><a href="bias.html#cb1570-26" tabindex="-1"></a><span class="st"> x4 ~~ c(residx4, residx4)*x4</span></span>
<span id="cb1570-27"><a href="bias.html#cb1570-27" tabindex="-1"></a><span class="st"> x5 ~~ c(residx5, residx5)*x5</span></span>
<span id="cb1570-28"><a href="bias.html#cb1570-28" tabindex="-1"></a><span class="st"> x6 ~~ c(residx6, residx6)*x6</span></span>
<span id="cb1570-29"><a href="bias.html#cb1570-29" tabindex="-1"></a><span class="st"> x7 ~~ c(residx7, residx7)*x7</span></span>
<span id="cb1570-30"><a href="bias.html#cb1570-30" tabindex="-1"></a><span class="st"> x8 ~~ c(residx8, residx8)*x8</span></span>
<span id="cb1570-31"><a href="bias.html#cb1570-31" tabindex="-1"></a><span class="st"> x9 ~~ c(residx9, residx9)*x9</span></span>
<span id="cb1570-32"><a href="bias.html#cb1570-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1570-33"><a href="bias.html#cb1570-33" tabindex="-1"></a><span class="st"> #Fix intercepts of manifest variables across groups</span></span>
<span id="cb1570-34"><a href="bias.html#cb1570-34" tabindex="-1"></a><span class="st"> x1 ~ c(intx1, intx1)*1</span></span>
<span id="cb1570-35"><a href="bias.html#cb1570-35" tabindex="-1"></a><span class="st"> x2 ~ c(intx2, intx2)*1</span></span>
<span id="cb1570-36"><a href="bias.html#cb1570-36" tabindex="-1"></a><span class="st"> x3 ~ c(intx3, intx3)*1</span></span>
<span id="cb1570-37"><a href="bias.html#cb1570-37" tabindex="-1"></a><span class="st"> x4 ~ c(intx4, intx4)*1</span></span>
<span id="cb1570-38"><a href="bias.html#cb1570-38" tabindex="-1"></a><span class="st"> x5 ~ c(intx5, intx5)*1</span></span>
<span id="cb1570-39"><a href="bias.html#cb1570-39" tabindex="-1"></a><span class="st"> x6 ~ c(intx6, intx6)*1</span></span>
<span id="cb1570-40"><a href="bias.html#cb1570-40" tabindex="-1"></a><span class="st"> x7 ~ c(intx7, intx7)*1</span></span>
<span id="cb1570-41"><a href="bias.html#cb1570-41" tabindex="-1"></a><span class="st"> x8 ~ c(intx8, intx8)*1</span></span>
<span id="cb1570-42"><a href="bias.html#cb1570-42" tabindex="-1"></a><span class="st"> x9 ~ c(intx9, intx9)*1</span></span>
<span id="cb1570-43"><a href="bias.html#cb1570-43" tabindex="-1"></a><span class="st">&#39;</span></span></code></pre></div>
</details>
</div>
</div>
<div id="measurementInvarianceFitIndices" class="section level3 hasAnchor" number="16.10.2">
<h3><span class="header-section-number">16.10.2</span> Specify the Fit Indices of Interest<a href="bias.html#measurementInvarianceFitIndices" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1571"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1571-1"><a href="bias.html#cb1571-1" tabindex="-1"></a>myAFIs <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">&quot;chisq&quot;</span>, <span class="st">&quot;chisq.scaled&quot;</span>,</span>
<span id="cb1571-2"><a href="bias.html#cb1571-2" tabindex="-1"></a>            <span class="st">&quot;rmsea&quot;</span>, <span class="st">&quot;cfi&quot;</span>, <span class="st">&quot;tli&quot;</span>, <span class="st">&quot;srmr&quot;</span>,</span>
<span id="cb1571-3"><a href="bias.html#cb1571-3" tabindex="-1"></a>            <span class="st">&quot;rmsea.robust&quot;</span>, <span class="st">&quot;cfi.robust&quot;</span>, <span class="st">&quot;tli.robust&quot;</span>)</span>
<span id="cb1571-4"><a href="bias.html#cb1571-4" tabindex="-1"></a>moreAFIs <span class="ot">&lt;-</span> <span class="cn">NULL</span> <span class="co"># c(&quot;gammaHat&quot;,&quot;gammaHat.scaled&quot;)</span></span></code></pre></div>
</details>
</div>
<div id="measurementInvarianceNullModel" class="section level3 hasAnchor" number="16.10.3">
<h3><span class="header-section-number">16.10.3</span> Null Model<a href="bias.html#measurementInvarianceNullModel" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="measurementInvarianceNullModelFit" class="section level4 hasAnchor" number="16.10.3.1">
<h4><span class="header-section-number">16.10.3.1</span> Fit the Model<a href="bias.html#measurementInvarianceNullModelFit" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1572"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1572-1"><a href="bias.html#cb1572-1" tabindex="-1"></a>nullModelFit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1572-2"><a href="bias.html#cb1572-2" tabindex="-1"></a>  nullModel,</span>
<span id="cb1572-3"><a href="bias.html#cb1572-3" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1572-4"><a href="bias.html#cb1572-4" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1572-5"><a href="bias.html#cb1572-5" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1572-6"><a href="bias.html#cb1572-6" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span></code></pre></div>
</details>
</div>
<div id="measurementInvarianceNullModelSummary" class="section level4 hasAnchor" number="16.10.3.2">
<h4><span class="header-section-number">16.10.3.2</span> Model Summary<a href="bias.html#measurementInvarianceNullModelSummary" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1573"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1573-1"><a href="bias.html#cb1573-1" tabindex="-1"></a><span class="fu">summary</span>(</span>
<span id="cb1573-2"><a href="bias.html#cb1573-2" tabindex="-1"></a>  nullModelFit,</span>
<span id="cb1573-3"><a href="bias.html#cb1573-3" tabindex="-1"></a>  <span class="at">fit.measures =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1573-4"><a href="bias.html#cb1573-4" tabindex="-1"></a>  <span class="at">standardized =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1573-5"><a href="bias.html#cb1573-5" tabindex="-1"></a>  <span class="at">rsquare =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
<pre><code>lavaan 0.6-19 ended normally after 39 iterations

  Estimator                                         ML
  Optimization method                           NLMINB
  Number of model parameters                        36
  Number of equality constraints                    18

  Number of observations per group:                   
    Pasteur                                        156
    Grant-White                                    145
  Number of missing patterns per group:               
    Pasteur                                          9
    Grant-White                                      9

Model Test User Model:
                                              Standard      Scaled
  Test Statistic                              1034.341    1001.376
  Degrees of freedom                                90          90
  P-value (Chi-square)                           0.000       0.000
  Scaling correction factor                                  1.033
    Yuan-Bentler correction (Mplus variant)                       
  Test statistic for each group:
    Pasteur                                    477.005     477.005
    Grant-White                                524.371     524.371

Model Test Baseline Model:

  Test statistic                               932.665     909.019
  Degrees of freedom                                72          72
  P-value                                        0.000       0.000
  Scaling correction factor                                  1.026

User Model versus Baseline Model:

  Comparative Fit Index (CFI)                    0.000       0.000
  Tucker-Lewis Index (TLI)                       0.122       0.129
                                                                  
  Robust Comparative Fit Index (CFI)                         0.000
  Robust Tucker-Lewis Index (TLI)                            0.125

Loglikelihood and Information Criteria:

  Loglikelihood user model (H0)              -4111.291   -4111.291
  Scaling correction factor                                  0.520
      for the MLR correction                                      
  Loglikelihood unrestricted model (H1)      -3594.120   -3594.120
  Scaling correction factor                                  1.034
      for the MLR correction                                      
                                                                  
  Akaike (AIC)                                8258.582    8258.582
  Bayesian (BIC)                              8325.310    8325.310
  Sample-size adjusted Bayesian (SABIC)       8268.224    8268.224

Root Mean Square Error of Approximation:

  RMSEA                                          0.264       0.259
  90 Percent confidence interval - lower         0.250       0.245
  90 Percent confidence interval - upper         0.279       0.274
  P-value H_0: RMSEA &lt;= 0.050                    0.000       0.000
  P-value H_0: RMSEA &gt;= 0.080                    1.000       1.000
                                                                  
  Robust RMSEA                                               0.267
  90 Percent confidence interval - lower                     0.252
  90 Percent confidence interval - upper                     0.282
  P-value H_0: Robust RMSEA &lt;= 0.050                         0.000
  P-value H_0: Robust RMSEA &gt;= 0.080                         1.000

Standardized Root Mean Square Residual:

  SRMR                                           0.284       0.284

Parameter Estimates:

  Standard errors                             Sandwich
  Information bread                           Observed
  Observed information based on                Hessian


Group 1 [Pasteur]:

Intercepts:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
    x1      (tau1)    4.935    0.067   73.255    0.000    4.935    4.236
    x2      (tau2)    6.085    0.068   89.601    0.000    6.085    5.173
    x3      (tau3)    2.261    0.066   34.411    0.000    2.261    2.000
    x4      (tau4)    3.050    0.067   45.313    0.000    3.050    2.634
    x5      (tau5)    4.332    0.074   58.509    0.000    4.332    3.389
    x6      (tau6)    2.195    0.063   34.800    0.000    2.195    2.013
    x7      (tau7)    4.196    0.063   66.358    0.000    4.196    3.850
    x8      (tau8)    5.524    0.059   94.261    0.000    5.524    5.451
    x9      (tau9)    5.378    0.059   91.869    0.000    5.378    5.322

Variances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
    x1      (psi1)    1.357    0.120   11.283    0.000    1.357    1.000
    x2      (psi2)    1.384    0.123   11.273    0.000    1.384    1.000
    x3      (psi3)    1.278    0.077   16.500    0.000    1.278    1.000
    x4      (psi4)    1.341    0.114   11.780    0.000    1.341    1.000
    x5      (psi5)    1.634    0.116   14.144    0.000    1.634    1.000
    x6      (psi6)    1.189    0.116   10.243    0.000    1.189    1.000
    x7      (psi7)    1.188    0.090   13.206    0.000    1.188    1.000
    x8      (psi8)    1.027    0.106    9.674    0.000    1.027    1.000
    x9      (psi9)    1.021    0.090   11.389    0.000    1.021    1.000


Group 2 [Grant-White]:

Intercepts:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
    x1      (tau1)    4.935    0.067   73.255    0.000    4.935    4.236
    x2      (tau2)    6.085    0.068   89.601    0.000    6.085    5.173
    x3      (tau3)    2.261    0.066   34.411    0.000    2.261    2.000
    x4      (tau4)    3.050    0.067   45.313    0.000    3.050    2.634
    x5      (tau5)    4.332    0.074   58.509    0.000    4.332    3.389
    x6      (tau6)    2.195    0.063   34.800    0.000    2.195    2.013
    x7      (tau7)    4.196    0.063   66.358    0.000    4.196    3.850
    x8      (tau8)    5.524    0.059   94.261    0.000    5.524    5.451
    x9      (tau9)    5.378    0.059   91.869    0.000    5.378    5.322

Variances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
    x1      (psi1)    1.357    0.120   11.283    0.000    1.357    1.000
    x2      (psi2)    1.384    0.123   11.273    0.000    1.384    1.000
    x3      (psi3)    1.278    0.077   16.500    0.000    1.278    1.000
    x4      (psi4)    1.341    0.114   11.780    0.000    1.341    1.000
    x5      (psi5)    1.634    0.116   14.144    0.000    1.634    1.000
    x6      (psi6)    1.189    0.116   10.243    0.000    1.189    1.000
    x7      (psi7)    1.188    0.090   13.206    0.000    1.188    1.000
    x8      (psi8)    1.027    0.106    9.674    0.000    1.027    1.000
    x9      (psi9)    1.021    0.090   11.389    0.000    1.021    1.000</code></pre>
</div>
</div>
<div id="measurementInvarianceConfigural" class="section level3 hasAnchor" number="16.10.4">
<h3><span class="header-section-number">16.10.4</span> Configural Invariance<a href="bias.html#measurementInvarianceConfigural" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Specify the same number of factors in each group, and which indicators load on which factors are the same in each group.</p>
<div id="measurementInvarianceConfiguralModelSyntax" class="section level4 hasAnchor" number="16.10.4.1">
<h4><span class="header-section-number">16.10.4.1</span> Model Syntax<a href="bias.html#measurementInvarianceConfiguralModelSyntax" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1575"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1575-1"><a href="bias.html#cb1575-1" tabindex="-1"></a>configuralInvarianceModel <span class="ot">&lt;-</span> <span class="fu">measEq.syntax</span>(</span>
<span id="cb1575-2"><a href="bias.html#cb1575-2" tabindex="-1"></a>  <span class="at">configural.model =</span> cfaModel,</span>
<span id="cb1575-3"><a href="bias.html#cb1575-3" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1575-4"><a href="bias.html#cb1575-4" tabindex="-1"></a>  <span class="at">ID.fac =</span> <span class="st">&quot;std.lv&quot;</span>,</span>
<span id="cb1575-5"><a href="bias.html#cb1575-5" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>)</span></code></pre></div>
</details>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1576"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1576-1"><a href="bias.html#cb1576-1" tabindex="-1"></a><span class="fu">cat</span>(<span class="fu">as.character</span>(configuralInvarianceModel))</span></code></pre></div>
</details>
<pre><code>## LOADINGS:

visual =~ c(NA, NA)*x1 + c(lambda.1_1.g1, lambda.1_1.g2)*x1
visual =~ c(NA, NA)*x2 + c(lambda.2_1.g1, lambda.2_1.g2)*x2
visual =~ c(NA, NA)*x3 + c(lambda.3_1.g1, lambda.3_1.g2)*x3
textual =~ c(NA, NA)*x4 + c(lambda.4_2.g1, lambda.4_2.g2)*x4
textual =~ c(NA, NA)*x5 + c(lambda.5_2.g1, lambda.5_2.g2)*x5
textual =~ c(NA, NA)*x6 + c(lambda.6_2.g1, lambda.6_2.g2)*x6
speed =~ c(NA, NA)*x7 + c(lambda.7_3.g1, lambda.7_3.g2)*x7
speed =~ c(NA, NA)*x8 + c(lambda.8_3.g1, lambda.8_3.g2)*x8
speed =~ c(NA, NA)*x9 + c(lambda.9_3.g1, lambda.9_3.g2)*x9

## INTERCEPTS:

x1 ~ c(NA, NA)*1 + c(nu.1.g1, nu.1.g2)*1
x2 ~ c(NA, NA)*1 + c(nu.2.g1, nu.2.g2)*1
x3 ~ c(NA, NA)*1 + c(nu.3.g1, nu.3.g2)*1
x4 ~ c(NA, NA)*1 + c(nu.4.g1, nu.4.g2)*1
x5 ~ c(NA, NA)*1 + c(nu.5.g1, nu.5.g2)*1
x6 ~ c(NA, NA)*1 + c(nu.6.g1, nu.6.g2)*1
x7 ~ c(NA, NA)*1 + c(nu.7.g1, nu.7.g2)*1
x8 ~ c(NA, NA)*1 + c(nu.8.g1, nu.8.g2)*1
x9 ~ c(NA, NA)*1 + c(nu.9.g1, nu.9.g2)*1

## UNIQUE-FACTOR VARIANCES:

x1 ~~ c(NA, NA)*x1 + c(theta.1_1.g1, theta.1_1.g2)*x1
x2 ~~ c(NA, NA)*x2 + c(theta.2_2.g1, theta.2_2.g2)*x2
x3 ~~ c(NA, NA)*x3 + c(theta.3_3.g1, theta.3_3.g2)*x3
x4 ~~ c(NA, NA)*x4 + c(theta.4_4.g1, theta.4_4.g2)*x4
x5 ~~ c(NA, NA)*x5 + c(theta.5_5.g1, theta.5_5.g2)*x5
x6 ~~ c(NA, NA)*x6 + c(theta.6_6.g1, theta.6_6.g2)*x6
x7 ~~ c(NA, NA)*x7 + c(theta.7_7.g1, theta.7_7.g2)*x7
x8 ~~ c(NA, NA)*x8 + c(theta.8_8.g1, theta.8_8.g2)*x8
x9 ~~ c(NA, NA)*x9 + c(theta.9_9.g1, theta.9_9.g2)*x9

## LATENT MEANS/INTERCEPTS:

visual ~ c(0, 0)*1 + c(alpha.1.g1, alpha.1.g2)*1
textual ~ c(0, 0)*1 + c(alpha.2.g1, alpha.2.g2)*1
speed ~ c(0, 0)*1 + c(alpha.3.g1, alpha.3.g2)*1

## COMMON-FACTOR VARIANCES:

visual ~~ c(1, 1)*visual + c(psi.1_1.g1, psi.1_1.g2)*visual
textual ~~ c(1, 1)*textual + c(psi.2_2.g1, psi.2_2.g2)*textual
speed ~~ c(1, 1)*speed + c(psi.3_3.g1, psi.3_3.g2)*speed

## COMMON-FACTOR COVARIANCES:

visual ~~ c(NA, NA)*textual + c(psi.2_1.g1, psi.2_1.g2)*textual
visual ~~ c(NA, NA)*speed + c(psi.3_1.g1, psi.3_1.g2)*speed
textual ~~ c(NA, NA)*speed + c(psi.3_2.g1, psi.3_2.g2)*speed</code></pre>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1578"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1578-1"><a href="bias.html#cb1578-1" tabindex="-1"></a>configuralInvarianceSyntax <span class="ot">&lt;-</span> <span class="fu">as.character</span>(configuralInvarianceModel)</span></code></pre></div>
</details>
<div id="measurementInvarianceConfiguralModelFeatures" class="section level5 hasAnchor" number="16.10.4.1.1">
<h5><span class="header-section-number">16.10.4.1.1</span> Summary of Model Features<a href="bias.html#measurementInvarianceConfiguralModelFeatures" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1579"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1579-1"><a href="bias.html#cb1579-1" tabindex="-1"></a><span class="fu">summary</span>(configuralInvarianceModel)</span></code></pre></div>
</details>
<pre><code>This lavaan model syntax specifies a CFA with 9 manifest indicators of 3 common factor(s).

To identify the location and scale of each common factor, the factor means and variances were fixed to 0 and 1, respectively, unless equality constraints on measurement parameters allow them to be freed.

Pattern matrix indicating num(eric), ord(ered), and lat(ent) indicators per factor:

   visual textual speed
x1 num                 
x2 num                 
x3 num                 
x4        num          
x5        num          
x6        num          
x7                num  
x8                num  
x9                num  


This model hypothesizes only configural invariance.</code></pre>
</div>
<div id="measurementInvarianceConfiguralModelTabular" class="section level5 hasAnchor" number="16.10.4.1.2">
<h5><span class="header-section-number">16.10.4.1.2</span> Model Syntax in Table Form<a href="bias.html#measurementInvarianceConfiguralModelTabular" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1581"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1581-1"><a href="bias.html#cb1581-1" tabindex="-1"></a><span class="fu">lavaanify</span>(cfaModel, <span class="at">ngroups =</span> <span class="dv">2</span>)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["id"],"name":[1],"type":["int"],"align":["right"]},{"label":["lhs"],"name":[2],"type":["chr"],"align":["left"]},{"label":["op"],"name":[3],"type":["chr"],"align":["left"]},{"label":["rhs"],"name":[4],"type":["chr"],"align":["left"]},{"label":["user"],"name":[5],"type":["int"],"align":["right"]},{"label":["block"],"name":[6],"type":["int"],"align":["right"]},{"label":["group"],"name":[7],"type":["int"],"align":["right"]},{"label":["free"],"name":[8],"type":["int"],"align":["right"]},{"label":["ustart"],"name":[9],"type":["dbl"],"align":["right"]},{"label":["exo"],"name":[10],"type":["int"],"align":["right"]},{"label":["label"],"name":[11],"type":["chr"],"align":["left"]},{"label":["plabel"],"name":[12],"type":["chr"],"align":["left"]}],"data":[{"1":"1","2":"visual","3":"=~","4":"x1","5":"1","6":"1","7":"1","8":"1","9":"NA","10":"0","11":"","12":".p1."},{"1":"2","2":"visual","3":"=~","4":"x2","5":"1","6":"1","7":"1","8":"2","9":"NA","10":"0","11":"","12":".p2."},{"1":"3","2":"visual","3":"=~","4":"x3","5":"1","6":"1","7":"1","8":"3","9":"NA","10":"0","11":"","12":".p3."},{"1":"4","2":"textual","3":"=~","4":"x4","5":"1","6":"1","7":"1","8":"4","9":"NA","10":"0","11":"","12":".p4."},{"1":"5","2":"textual","3":"=~","4":"x5","5":"1","6":"1","7":"1","8":"5","9":"NA","10":"0","11":"","12":".p5."},{"1":"6","2":"textual","3":"=~","4":"x6","5":"1","6":"1","7":"1","8":"6","9":"NA","10":"0","11":"","12":".p6."},{"1":"7","2":"speed","3":"=~","4":"x7","5":"1","6":"1","7":"1","8":"7","9":"NA","10":"0","11":"","12":".p7."},{"1":"8","2":"speed","3":"=~","4":"x8","5":"1","6":"1","7":"1","8":"8","9":"NA","10":"0","11":"","12":".p8."},{"1":"9","2":"speed","3":"=~","4":"x9","5":"1","6":"1","7":"1","8":"9","9":"NA","10":"0","11":"","12":".p9."},{"1":"10","2":"x1","3":"~~","4":"x1","5":"0","6":"1","7":"1","8":"0","9":"0","10":"0","11":"","12":".p10."},{"1":"11","2":"x2","3":"~~","4":"x2","5":"0","6":"1","7":"1","8":"0","9":"0","10":"0","11":"","12":".p11."},{"1":"12","2":"x3","3":"~~","4":"x3","5":"0","6":"1","7":"1","8":"0","9":"0","10":"0","11":"","12":".p12."},{"1":"13","2":"x4","3":"~~","4":"x4","5":"0","6":"1","7":"1","8":"0","9":"0","10":"0","11":"","12":".p13."},{"1":"14","2":"x5","3":"~~","4":"x5","5":"0","6":"1","7":"1","8":"0","9":"0","10":"0","11":"","12":".p14."},{"1":"15","2":"x6","3":"~~","4":"x6","5":"0","6":"1","7":"1","8":"0","9":"0","10":"0","11":"","12":".p15."},{"1":"16","2":"x7","3":"~~","4":"x7","5":"0","6":"1","7":"1","8":"0","9":"0","10":"0","11":"","12":".p16."},{"1":"17","2":"x8","3":"~~","4":"x8","5":"0","6":"1","7":"1","8":"0","9":"0","10":"0","11":"","12":".p17."},{"1":"18","2":"x9","3":"~~","4":"x9","5":"0","6":"1","7":"1","8":"0","9":"0","10":"0","11":"","12":".p18."},{"1":"19","2":"visual","3":"~~","4":"visual","5":"0","6":"1","7":"1","8":"0","9":"0","10":"0","11":"","12":".p19."},{"1":"20","2":"textual","3":"~~","4":"textual","5":"0","6":"1","7":"1","8":"0","9":"0","10":"0","11":"","12":".p20."},{"1":"21","2":"speed","3":"~~","4":"speed","5":"0","6":"1","7":"1","8":"0","9":"0","10":"0","11":"","12":".p21."},{"1":"22","2":"visual","3":"=~","4":"x1","5":"1","6":"2","7":"2","8":"10","9":"NA","10":"0","11":"","12":".p22."},{"1":"23","2":"visual","3":"=~","4":"x2","5":"1","6":"2","7":"2","8":"11","9":"NA","10":"0","11":"","12":".p23."},{"1":"24","2":"visual","3":"=~","4":"x3","5":"1","6":"2","7":"2","8":"12","9":"NA","10":"0","11":"","12":".p24."},{"1":"25","2":"textual","3":"=~","4":"x4","5":"1","6":"2","7":"2","8":"13","9":"NA","10":"0","11":"","12":".p25."},{"1":"26","2":"textual","3":"=~","4":"x5","5":"1","6":"2","7":"2","8":"14","9":"NA","10":"0","11":"","12":".p26."},{"1":"27","2":"textual","3":"=~","4":"x6","5":"1","6":"2","7":"2","8":"15","9":"NA","10":"0","11":"","12":".p27."},{"1":"28","2":"speed","3":"=~","4":"x7","5":"1","6":"2","7":"2","8":"16","9":"NA","10":"0","11":"","12":".p28."},{"1":"29","2":"speed","3":"=~","4":"x8","5":"1","6":"2","7":"2","8":"17","9":"NA","10":"0","11":"","12":".p29."},{"1":"30","2":"speed","3":"=~","4":"x9","5":"1","6":"2","7":"2","8":"18","9":"NA","10":"0","11":"","12":".p30."},{"1":"31","2":"x1","3":"~~","4":"x1","5":"0","6":"2","7":"2","8":"0","9":"0","10":"0","11":"","12":".p31."},{"1":"32","2":"x2","3":"~~","4":"x2","5":"0","6":"2","7":"2","8":"0","9":"0","10":"0","11":"","12":".p32."},{"1":"33","2":"x3","3":"~~","4":"x3","5":"0","6":"2","7":"2","8":"0","9":"0","10":"0","11":"","12":".p33."},{"1":"34","2":"x4","3":"~~","4":"x4","5":"0","6":"2","7":"2","8":"0","9":"0","10":"0","11":"","12":".p34."},{"1":"35","2":"x5","3":"~~","4":"x5","5":"0","6":"2","7":"2","8":"0","9":"0","10":"0","11":"","12":".p35."},{"1":"36","2":"x6","3":"~~","4":"x6","5":"0","6":"2","7":"2","8":"0","9":"0","10":"0","11":"","12":".p36."},{"1":"37","2":"x7","3":"~~","4":"x7","5":"0","6":"2","7":"2","8":"0","9":"0","10":"0","11":"","12":".p37."},{"1":"38","2":"x8","3":"~~","4":"x8","5":"0","6":"2","7":"2","8":"0","9":"0","10":"0","11":"","12":".p38."},{"1":"39","2":"x9","3":"~~","4":"x9","5":"0","6":"2","7":"2","8":"0","9":"0","10":"0","11":"","12":".p39."},{"1":"40","2":"visual","3":"~~","4":"visual","5":"0","6":"2","7":"2","8":"0","9":"0","10":"0","11":"","12":".p40."},{"1":"41","2":"textual","3":"~~","4":"textual","5":"0","6":"2","7":"2","8":"0","9":"0","10":"0","11":"","12":".p41."},{"1":"42","2":"speed","3":"~~","4":"speed","5":"0","6":"2","7":"2","8":"0","9":"0","10":"0","11":"","12":".p42."}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
<div id="measurementInvarianceConfiguralModelFit" class="section level4 hasAnchor" number="16.10.4.2">
<h4><span class="header-section-number">16.10.4.2</span> Fit the Model<a href="bias.html#measurementInvarianceConfiguralModelFit" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1582"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1582-1"><a href="bias.html#cb1582-1" tabindex="-1"></a>configuralInvarianceModel_fit <span class="ot">&lt;-</span> <span class="fu">cfa</span>(</span>
<span id="cb1582-2"><a href="bias.html#cb1582-2" tabindex="-1"></a>  configuralInvarianceSyntax,</span>
<span id="cb1582-3"><a href="bias.html#cb1582-3" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1582-4"><a href="bias.html#cb1582-4" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1582-5"><a href="bias.html#cb1582-5" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1582-6"><a href="bias.html#cb1582-6" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span></code></pre></div>
</details>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1583"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1583-1"><a href="bias.html#cb1583-1" tabindex="-1"></a>configuralInvarianceModelFullSyntax_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1583-2"><a href="bias.html#cb1583-2" tabindex="-1"></a>  cfaModel_configuralInvariance,</span>
<span id="cb1583-3"><a href="bias.html#cb1583-3" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1583-4"><a href="bias.html#cb1583-4" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1583-5"><a href="bias.html#cb1583-5" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1583-6"><a href="bias.html#cb1583-6" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span></code></pre></div>
</details>
</div>
<div id="measurementInvarianceConfiguralModelSummary" class="section level4 hasAnchor" number="16.10.4.3">
<h4><span class="header-section-number">16.10.4.3</span> Model Summary<a href="bias.html#measurementInvarianceConfiguralModelSummary" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1584"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1584-1"><a href="bias.html#cb1584-1" tabindex="-1"></a><span class="fu">summary</span>(</span>
<span id="cb1584-2"><a href="bias.html#cb1584-2" tabindex="-1"></a>  configuralInvarianceModel_fit,</span>
<span id="cb1584-3"><a href="bias.html#cb1584-3" tabindex="-1"></a>  <span class="at">fit.measures =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1584-4"><a href="bias.html#cb1584-4" tabindex="-1"></a>  <span class="at">standardized =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1584-5"><a href="bias.html#cb1584-5" tabindex="-1"></a>  <span class="at">rsquare =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
<pre><code>lavaan 0.6-19 ended normally after 64 iterations

  Estimator                                         ML
  Optimization method                           NLMINB
  Number of model parameters                        60

  Number of observations per group:                   
    Pasteur                                        156
    Grant-White                                    145
  Number of missing patterns per group:               
    Pasteur                                          9
    Grant-White                                      9

Model Test User Model:
                                              Standard      Scaled
  Test Statistic                               109.425     114.970
  Degrees of freedom                                48          48
  P-value (Chi-square)                           0.000       0.000
  Scaling correction factor                                  0.952
    Yuan-Bentler correction (Mplus variant)                       
  Test statistic for each group:
    Pasteur                                     63.000      63.000
    Grant-White                                 51.970      51.970

Model Test Baseline Model:

  Test statistic                               932.665     909.019
  Degrees of freedom                                72          72
  P-value                                        0.000       0.000
  Scaling correction factor                                  1.026

User Model versus Baseline Model:

  Comparative Fit Index (CFI)                    0.929       0.920
  Tucker-Lewis Index (TLI)                       0.893       0.880
                                                                  
  Robust Comparative Fit Index (CFI)                         0.929
  Robust Tucker-Lewis Index (TLI)                            0.893

Loglikelihood and Information Criteria:

  Loglikelihood user model (H0)              -3648.833   -3648.833
  Scaling correction factor                                  1.100
      for the MLR correction                                      
  Loglikelihood unrestricted model (H1)      -3594.120   -3594.120
  Scaling correction factor                                  1.034
      for the MLR correction                                      
                                                                  
  Akaike (AIC)                                7417.666    7417.666
  Bayesian (BIC)                              7640.092    7640.092
  Sample-size adjusted Bayesian (SABIC)       7449.806    7449.806

Root Mean Square Error of Approximation:

  RMSEA                                          0.092       0.096
  90 Percent confidence interval - lower         0.069       0.073
  90 Percent confidence interval - upper         0.115       0.120
  P-value H_0: RMSEA &lt;= 0.050                    0.002       0.001
  P-value H_0: RMSEA &gt;= 0.080                    0.821       0.883
                                                                  
  Robust RMSEA                                               0.093
  90 Percent confidence interval - lower                     0.070
  90 Percent confidence interval - upper                     0.117
  P-value H_0: Robust RMSEA &lt;= 0.050                         0.002
  P-value H_0: Robust RMSEA &gt;= 0.080                         0.833

Standardized Root Mean Square Residual:

  SRMR                                           0.066       0.066

Parameter Estimates:

  Standard errors                             Sandwich
  Information bread                           Observed
  Observed information based on                Hessian


Group 1 [Pasteur]:

Latent Variables:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual =~                                                             
    x1      (l.1_)    0.999    0.172    5.810    0.000    0.999    0.846
    x2      (l.2_)    0.443    0.147    3.022    0.003    0.443    0.361
    x3      (l.3_)    0.624    0.134    4.662    0.000    0.624    0.541
  textual =~                                                            
    x4      (l.4_)    0.922    0.085   10.830    0.000    0.922    0.811
    x5      (l.5_)    1.126    0.072   15.723    0.000    1.126    0.864
    x6      (l.6_)    0.833    0.081   10.242    0.000    0.833    0.843
  speed =~                                                              
    x7      (l.7_)    0.570    0.119    4.790    0.000    0.570    0.527
    x8      (l.8_)    0.676    0.110    6.133    0.000    0.676    0.688
    x9      (l.9_)    0.587    0.118    4.998    0.000    0.587    0.589

Covariances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual ~~                                                             
    textul  (p.2_)    0.473    0.095    4.951    0.000    0.473    0.473
    speed  (p.3_1)    0.319    0.148    2.150    0.032    0.319    0.319
  textual ~~                                                            
    speed  (p.3_2)    0.311    0.100    3.099    0.002    0.311    0.311

Intercepts:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
   .x1      (n.1.)    4.947    0.095   52.269    0.000    4.947    4.191
   .x2      (n.2.)    5.984    0.098   60.949    0.000    5.984    4.880
   .x3      (n.3.)    2.510    0.093   26.912    0.000    2.510    2.178
   .x4      (n.4.)    2.814    0.091   30.873    0.000    2.814    2.476
   .x5      (n.5.)    4.002    0.105   38.256    0.000    4.002    3.070
   .x6      (n.6.)    1.929    0.079   24.376    0.000    1.929    1.953
   .x7      (n.7.)    4.439    0.087   51.126    0.000    4.439    4.104
   .x8      (n.8.)    5.563    0.079   70.133    0.000    5.563    5.663
   .x9      (n.9.)    5.426    0.080   67.537    0.000    5.426    5.443
    visual  (a.1.)    0.000                               0.000    0.000
    textual (a.2.)    0.000                               0.000    0.000
    speed   (a.3.)    0.000                               0.000    0.000

Variances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
   .x1      (t.1_)    0.396    0.298    1.330    0.183    0.396    0.284
   .x2      (t.2_)    1.308    0.175    7.482    0.000    1.308    0.870
   .x3      (t.3_)    0.939    0.158    5.959    0.000    0.939    0.707
   .x4      (t.4_)    0.441    0.072    6.091    0.000    0.441    0.342
   .x5      (t.5_)    0.431    0.083    5.169    0.000    0.431    0.254
   .x6      (t.6_)    0.282    0.056    5.063    0.000    0.282    0.289
   .x7      (t.7_)    0.845    0.132    6.394    0.000    0.845    0.723
   .x8      (t.8_)    0.509    0.109    4.673    0.000    0.509    0.527
   .x9      (t.9_)    0.649    0.134    4.824    0.000    0.649    0.653
    visual  (p.1_)    1.000                               1.000    1.000
    textual (p.2_)    1.000                               1.000    1.000
    speed   (p.3_)    1.000                               1.000    1.000

R-Square:
                   Estimate
    x1                0.716
    x2                0.130
    x3                0.293
    x4                0.658
    x5                0.746
    x6                0.711
    x7                0.277
    x8                0.473
    x9                0.347


Group 2 [Grant-White]:

Latent Variables:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual =~                                                             
    x1      (l.1_)    0.781    0.128    6.099    0.000    0.781    0.679
    x2      (l.2_)    0.566    0.103    5.489    0.000    0.566    0.510
    x3      (l.3_)    0.717    0.100    7.165    0.000    0.717    0.690
  textual =~                                                            
    x4      (l.4_)    0.958    0.083   11.489    0.000    0.958    0.856
    x5      (l.5_)    0.973    0.090   10.798    0.000    0.973    0.835
    x6      (l.6_)    0.947    0.088   10.753    0.000    0.947    0.830
  speed =~                                                              
    x7      (l.7_)    0.672    0.090    7.446    0.000    0.672    0.649
    x8      (l.8_)    0.833    0.112    7.447    0.000    0.833    0.796
    x9      (l.9_)    0.732    0.124    5.924    0.000    0.732    0.714

Covariances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual ~~                                                             
    textul  (p.2_)    0.545    0.096    5.648    0.000    0.545    0.545
    speed  (p.3_1)    0.531    0.148    3.593    0.000    0.531    0.531
  textual ~~                                                            
    speed  (p.3_2)    0.339    0.142    2.392    0.017    0.339    0.339

Intercepts:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
   .x1      (n.1.)    4.929    0.096   51.433    0.000    4.929    4.286
   .x2      (n.2.)    6.198    0.092   67.052    0.000    6.198    5.587
   .x3      (n.3.)    1.992    0.086   23.048    0.000    1.992    1.916
   .x4      (n.4.)    3.317    0.093   35.551    0.000    3.317    2.966
   .x5      (n.5.)    4.712    0.096   49.039    0.000    4.712    4.045
   .x6      (n.6.)    2.466    0.094   26.150    0.000    2.466    2.163
   .x7      (n.7.)    3.923    0.087   45.300    0.000    3.923    3.793
   .x8      (n.8.)    5.488    0.087   63.174    0.000    5.488    5.246
   .x9      (n.9.)    5.338    0.085   62.640    0.000    5.338    5.210
    visual  (a.1.)    0.000                               0.000    0.000
    textual (a.2.)    0.000                               0.000    0.000
    speed   (a.3.)    0.000                               0.000    0.000

Variances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
   .x1      (t.1_)    0.712    0.183    3.884    0.000    0.712    0.538
   .x2      (t.2_)    0.911    0.143    6.364    0.000    0.911    0.740
   .x3      (t.3_)    0.566    0.123    4.601    0.000    0.566    0.524
   .x4      (t.4_)    0.333    0.066    5.025    0.000    0.333    0.267
   .x5      (t.5_)    0.411    0.074    5.570    0.000    0.411    0.303
   .x6      (t.6_)    0.404    0.078    5.182    0.000    0.404    0.311
   .x7      (t.7_)    0.619    0.101    6.105    0.000    0.619    0.578
   .x8      (t.8_)    0.401    0.160    2.512    0.012    0.401    0.366
   .x9      (t.9_)    0.514    0.140    3.684    0.000    0.514    0.490
    visual  (p.1_)    1.000                               1.000    1.000
    textual (p.2_)    1.000                               1.000    1.000
    speed   (p.3_)    1.000                               1.000    1.000

R-Square:
                   Estimate
    x1                0.462
    x2                0.260
    x3                0.476
    x4                0.733
    x5                0.697
    x6                0.689
    x7                0.422
    x8                0.634
    x9                0.510</code></pre>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1586"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1586-1"><a href="bias.html#cb1586-1" tabindex="-1"></a><span class="fu">summary</span>(</span>
<span id="cb1586-2"><a href="bias.html#cb1586-2" tabindex="-1"></a>  configuralInvarianceModelFullSyntax_fit,</span>
<span id="cb1586-3"><a href="bias.html#cb1586-3" tabindex="-1"></a>  <span class="at">fit.measures =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1586-4"><a href="bias.html#cb1586-4" tabindex="-1"></a>  <span class="at">standardized =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1586-5"><a href="bias.html#cb1586-5" tabindex="-1"></a>  <span class="at">rsquare =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
<pre><code>lavaan 0.6-19 ended normally after 64 iterations

  Estimator                                         ML
  Optimization method                           NLMINB
  Number of model parameters                        60

  Number of observations per group:                   
    Pasteur                                        156
    Grant-White                                    145
  Number of missing patterns per group:               
    Pasteur                                          9
    Grant-White                                      9

Model Test User Model:
                                              Standard      Scaled
  Test Statistic                               109.425     114.970
  Degrees of freedom                                48          48
  P-value (Chi-square)                           0.000       0.000
  Scaling correction factor                                  0.952
    Yuan-Bentler correction (Mplus variant)                       
  Test statistic for each group:
    Pasteur                                     63.000      63.000
    Grant-White                                 51.970      51.970

Model Test Baseline Model:

  Test statistic                               932.665     909.019
  Degrees of freedom                                72          72
  P-value                                        0.000       0.000
  Scaling correction factor                                  1.026

User Model versus Baseline Model:

  Comparative Fit Index (CFI)                    0.929       0.920
  Tucker-Lewis Index (TLI)                       0.893       0.880
                                                                  
  Robust Comparative Fit Index (CFI)                         0.929
  Robust Tucker-Lewis Index (TLI)                            0.893

Loglikelihood and Information Criteria:

  Loglikelihood user model (H0)              -3648.833   -3648.833
  Scaling correction factor                                  1.100
      for the MLR correction                                      
  Loglikelihood unrestricted model (H1)      -3594.120   -3594.120
  Scaling correction factor                                  1.034
      for the MLR correction                                      
                                                                  
  Akaike (AIC)                                7417.666    7417.666
  Bayesian (BIC)                              7640.092    7640.092
  Sample-size adjusted Bayesian (SABIC)       7449.806    7449.806

Root Mean Square Error of Approximation:

  RMSEA                                          0.092       0.096
  90 Percent confidence interval - lower         0.069       0.073
  90 Percent confidence interval - upper         0.115       0.120
  P-value H_0: RMSEA &lt;= 0.050                    0.002       0.001
  P-value H_0: RMSEA &gt;= 0.080                    0.821       0.883
                                                                  
  Robust RMSEA                                               0.093
  90 Percent confidence interval - lower                     0.070
  90 Percent confidence interval - upper                     0.117
  P-value H_0: Robust RMSEA &lt;= 0.050                         0.002
  P-value H_0: Robust RMSEA &gt;= 0.080                         0.833

Standardized Root Mean Square Residual:

  SRMR                                           0.066       0.066

Parameter Estimates:

  Standard errors                             Sandwich
  Information bread                           Observed
  Observed information based on                Hessian


Group 1 [Pasteur]:

Latent Variables:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual =~                                                             
    x1                0.999    0.172    5.810    0.000    0.999    0.846
    x2                0.443    0.147    3.022    0.003    0.443    0.361
    x3                0.624    0.134    4.662    0.000    0.624    0.541
  textual =~                                                            
    x4                0.922    0.085   10.830    0.000    0.922    0.811
    x5                1.126    0.072   15.723    0.000    1.126    0.864
    x6                0.833    0.081   10.242    0.000    0.833    0.843
  speed =~                                                              
    x7                0.570    0.119    4.790    0.000    0.570    0.527
    x8                0.676    0.110    6.133    0.000    0.676    0.688
    x9                0.587    0.118    4.998    0.000    0.587    0.589

Covariances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual ~~                                                             
    textual           0.473    0.095    4.951    0.000    0.473    0.473
    speed             0.319    0.148    2.150    0.032    0.319    0.319
  textual ~~                                                            
    speed             0.311    0.100    3.099    0.002    0.311    0.311

Intercepts:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
    visual            0.000                               0.000    0.000
    textual           0.000                               0.000    0.000
    speed             0.000                               0.000    0.000
   .x1                4.947    0.095   52.269    0.000    4.947    4.191
   .x2                5.984    0.098   60.949    0.000    5.984    4.880
   .x3                2.510    0.093   26.912    0.000    2.510    2.178
   .x4                2.814    0.091   30.873    0.000    2.814    2.476
   .x5                4.002    0.105   38.256    0.000    4.002    3.070
   .x6                1.929    0.079   24.376    0.000    1.929    1.953
   .x7                4.439    0.087   51.126    0.000    4.439    4.104
   .x8                5.563    0.079   70.133    0.000    5.563    5.663
   .x9                5.426    0.080   67.537    0.000    5.426    5.443

Variances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
    visual            1.000                               1.000    1.000
    textual           1.000                               1.000    1.000
    speed             1.000                               1.000    1.000
   .x1                0.396    0.298    1.330    0.183    0.396    0.284
   .x2                1.308    0.175    7.482    0.000    1.308    0.870
   .x3                0.939    0.158    5.959    0.000    0.939    0.707
   .x4                0.441    0.072    6.091    0.000    0.441    0.342
   .x5                0.431    0.083    5.169    0.000    0.431    0.254
   .x6                0.282    0.056    5.063    0.000    0.282    0.289
   .x7                0.845    0.132    6.394    0.000    0.845    0.723
   .x8                0.509    0.109    4.673    0.000    0.509    0.527
   .x9                0.649    0.134    4.824    0.000    0.649    0.653

R-Square:
                   Estimate
    x1                0.716
    x2                0.130
    x3                0.293
    x4                0.658
    x5                0.746
    x6                0.711
    x7                0.277
    x8                0.473
    x9                0.347


Group 2 [Grant-White]:

Latent Variables:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual =~                                                             
    x1                0.781    0.128    6.099    0.000    0.781    0.679
    x2                0.566    0.103    5.489    0.000    0.566    0.510
    x3                0.717    0.100    7.165    0.000    0.717    0.690
  textual =~                                                            
    x4                0.958    0.083   11.489    0.000    0.958    0.856
    x5                0.973    0.090   10.798    0.000    0.973    0.835
    x6                0.947    0.088   10.753    0.000    0.947    0.830
  speed =~                                                              
    x7                0.672    0.090    7.446    0.000    0.672    0.649
    x8                0.833    0.112    7.447    0.000    0.833    0.796
    x9                0.732    0.124    5.924    0.000    0.732    0.714

Covariances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual ~~                                                             
    textual           0.545    0.096    5.648    0.000    0.545    0.545
    speed             0.531    0.148    3.593    0.000    0.531    0.531
  textual ~~                                                            
    speed             0.339    0.142    2.392    0.017    0.339    0.339

Intercepts:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
    visual            0.000                               0.000    0.000
    textual           0.000                               0.000    0.000
    speed             0.000                               0.000    0.000
   .x1                4.929    0.096   51.433    0.000    4.929    4.286
   .x2                6.198    0.092   67.052    0.000    6.198    5.587
   .x3                1.992    0.086   23.048    0.000    1.992    1.916
   .x4                3.317    0.093   35.551    0.000    3.317    2.966
   .x5                4.712    0.096   49.039    0.000    4.712    4.045
   .x6                2.466    0.094   26.150    0.000    2.466    2.163
   .x7                3.923    0.087   45.300    0.000    3.923    3.793
   .x8                5.488    0.087   63.174    0.000    5.488    5.246
   .x9                5.338    0.085   62.640    0.000    5.338    5.210

Variances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
    visual            1.000                               1.000    1.000
    textual           1.000                               1.000    1.000
    speed             1.000                               1.000    1.000
   .x1                0.712    0.183    3.884    0.000    0.712    0.538
   .x2                0.911    0.143    6.364    0.000    0.911    0.740
   .x3                0.566    0.123    4.601    0.000    0.566    0.524
   .x4                0.333    0.066    5.025    0.000    0.333    0.267
   .x5                0.411    0.074    5.570    0.000    0.411    0.303
   .x6                0.404    0.078    5.182    0.000    0.404    0.311
   .x7                0.619    0.101    6.105    0.000    0.619    0.578
   .x8                0.401    0.160    2.512    0.012    0.401    0.366
   .x9                0.514    0.140    3.684    0.000    0.514    0.490

R-Square:
                   Estimate
    x1                0.462
    x2                0.260
    x3                0.476
    x4                0.733
    x5                0.697
    x6                0.689
    x7                0.422
    x8                0.634
    x9                0.510</code></pre>
</div>
<div id="measurementInvarianceConfiguralModelFitCriteria" class="section level4 hasAnchor" number="16.10.4.4">
<h4><span class="header-section-number">16.10.4.4</span> Model Fit<a href="bias.html#measurementInvarianceConfiguralModelFitCriteria" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>You can specify the null model as the baseline model using: <code>baseline.model = nullModelFit</code></p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1588"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1588-1"><a href="bias.html#cb1588-1" tabindex="-1"></a><span class="fu">fitMeasures</span>(configuralInvarianceModel_fit)</span></code></pre></div>
</details>
<pre><code>                         npar                          fmin 
                       60.000                         0.182 
                        chisq                            df 
                      109.425                        48.000 
                       pvalue                  chisq.scaled 
                        0.000                       114.970 
                    df.scaled                 pvalue.scaled 
                       48.000                         0.000 
         chisq.scaling.factor                baseline.chisq 
                        0.952                       932.665 
                  baseline.df               baseline.pvalue 
                       72.000                         0.000 
        baseline.chisq.scaled            baseline.df.scaled 
                      909.019                        72.000 
       baseline.pvalue.scaled baseline.chisq.scaling.factor 
                        0.000                         1.026 
                          cfi                           tli 
                        0.929                         0.893 
                   cfi.scaled                    tli.scaled 
                        0.920                         0.880 
                   cfi.robust                    tli.robust 
                        0.929                         0.893 
                         nnfi                           rfi 
                        0.893                         0.824 
                          nfi                          pnfi 
                        0.883                         0.588 
                          ifi                           rni 
                        0.931                         0.929 
                  nnfi.scaled                    rfi.scaled 
                        0.880                         0.810 
                   nfi.scaled                   pnfi.scaled 
                        0.874                         0.582 
                   ifi.scaled                    rni.scaled 
                        0.922                         0.920 
                  nnfi.robust                    rni.robust 
                        0.893                         0.929 
                         logl             unrestricted.logl 
                    -3648.833                     -3594.120 
                          aic                           bic 
                     7417.666                      7640.092 
                       ntotal                          bic2 
                      301.000                      7449.806 
            scaling.factor.h1             scaling.factor.h0 
                        1.034                         1.100 
                        rmsea                rmsea.ci.lower 
                        0.092                         0.069 
               rmsea.ci.upper                rmsea.ci.level 
                        0.115                         0.900 
                 rmsea.pvalue                rmsea.close.h0 
                        0.002                         0.050 
        rmsea.notclose.pvalue             rmsea.notclose.h0 
                        0.821                         0.080 
                 rmsea.scaled         rmsea.ci.lower.scaled 
                        0.096                         0.073 
        rmsea.ci.upper.scaled           rmsea.pvalue.scaled 
                        0.120                         0.001 
 rmsea.notclose.pvalue.scaled                  rmsea.robust 
                        0.883                         0.093 
        rmsea.ci.lower.robust         rmsea.ci.upper.robust 
                        0.070                         0.117 
          rmsea.pvalue.robust  rmsea.notclose.pvalue.robust 
                        0.002                         0.833 
                          rmr                    rmr_nomean 
                        0.081                         0.089 
                         srmr                  srmr_bentler 
                        0.066                         0.066 
          srmr_bentler_nomean                          crmr 
                        0.072                         0.072 
                  crmr_nomean                    srmr_mplus 
                        0.080                         0.066 
            srmr_mplus_nomean                         cn_05 
                        0.072                       180.268 
                        cn_01                           gfi 
                      203.682                         0.995 
                         agfi                          pgfi 
                        0.989                         0.442 
                          mfi                          ecvi 
                        0.903                         0.762 </code></pre>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1590"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1590-1"><a href="bias.html#cb1590-1" tabindex="-1"></a>configuralInvarianceModelFitIndices <span class="ot">&lt;-</span> <span class="fu">fitMeasures</span>(</span>
<span id="cb1590-2"><a href="bias.html#cb1590-2" tabindex="-1"></a>  configuralInvarianceModel_fit)[<span class="fu">c</span>(</span>
<span id="cb1590-3"><a href="bias.html#cb1590-3" tabindex="-1"></a>    <span class="st">&quot;cfi.robust&quot;</span>, <span class="st">&quot;rmsea.robust&quot;</span>, <span class="st">&quot;srmr&quot;</span>)]</span>
<span id="cb1590-4"><a href="bias.html#cb1590-4" tabindex="-1"></a></span>
<span id="cb1590-5"><a href="bias.html#cb1590-5" tabindex="-1"></a>configuralInvarianceModel_chisquare <span class="ot">&lt;-</span> <span class="fu">fitMeasures</span>(</span>
<span id="cb1590-6"><a href="bias.html#cb1590-6" tabindex="-1"></a>  configuralInvarianceModel_fit)[<span class="fu">c</span>(<span class="st">&quot;chisq.scaled&quot;</span>)]</span>
<span id="cb1590-7"><a href="bias.html#cb1590-7" tabindex="-1"></a></span>
<span id="cb1590-8"><a href="bias.html#cb1590-8" tabindex="-1"></a>configuralInvarianceModel_chisquareScaling <span class="ot">&lt;-</span> <span class="fu">fitMeasures</span>(</span>
<span id="cb1590-9"><a href="bias.html#cb1590-9" tabindex="-1"></a>  configuralInvarianceModel_fit)[<span class="fu">c</span>(<span class="st">&quot;chisq.scaling.factor&quot;</span>)]</span>
<span id="cb1590-10"><a href="bias.html#cb1590-10" tabindex="-1"></a></span>
<span id="cb1590-11"><a href="bias.html#cb1590-11" tabindex="-1"></a>configuralInvarianceModel_df <span class="ot">&lt;-</span> <span class="fu">fitMeasures</span>(</span>
<span id="cb1590-12"><a href="bias.html#cb1590-12" tabindex="-1"></a>  configuralInvarianceModel_fit)[<span class="fu">c</span>(<span class="st">&quot;df.scaled&quot;</span>)]</span>
<span id="cb1590-13"><a href="bias.html#cb1590-13" tabindex="-1"></a></span>
<span id="cb1590-14"><a href="bias.html#cb1590-14" tabindex="-1"></a>configuralInvarianceModel_N <span class="ot">&lt;-</span> <span class="fu">lavInspect</span>(</span>
<span id="cb1590-15"><a href="bias.html#cb1590-15" tabindex="-1"></a>  configuralInvarianceModel_fit,</span>
<span id="cb1590-16"><a href="bias.html#cb1590-16" tabindex="-1"></a>  <span class="at">what =</span> <span class="st">&quot;ntotal&quot;</span>)</span></code></pre></div>
</details>
</div>
<div id="measurementInvarianceEffectSize" class="section level4 hasAnchor" number="16.10.4.5">
<h4><span class="header-section-number">16.10.4.5</span> Effect size of non-invariance<a href="bias.html#measurementInvarianceEffectSize" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<div id="dmacs" class="section level5 hasAnchor" number="16.10.4.5.1">
<h5><span class="header-section-number">16.10.4.5.1</span> dMACS<a href="bias.html#dmacs" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>The effect size of measurement non-invariance, as described by <span class="citation">Nye et al. (<a href="#ref-Nye2019">2019</a>)</span>, was calculated using the <code>dmacs</code> package <span class="citation">(<a href="#ref-R-dmacs">Dueber, 2019</a>)</span>.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1591"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1591-1"><a href="bias.html#cb1591-1" tabindex="-1"></a><span class="fu">lavaan_dmacs</span>(configuralInvarianceModel_fit)</span></code></pre></div>
</details>
<pre><code>$DMACS
      visual   textual     speed
x1 0.1865718        NA        NA
x2 0.2102487        NA        NA
x3 0.4774264        NA        NA
x4        NA 0.4450412        NA
x5        NA 0.5889006        NA
x6        NA 0.5196869        NA
x7        NA        NA 0.4941585
x8        NA        NA 0.1714779
x9        NA        NA 0.1669673

$ItemDeltaMean
        visual   textual       speed
x1 -0.01829609        NA          NA
x2  0.21407860        NA          NA
x3 -0.51808612        NA          NA
x4          NA 0.5031129          NA
x5          NA 0.7100353          NA
x6          NA 0.5368256          NA
x7          NA        NA -0.51547978
x8          NA        NA -0.07512450
x9          NA        NA -0.08770776

$MeanDiff
    visual    textual      speed 
-0.3223036  1.7499738 -0.6783120 </code></pre>
</div>
</div>
<div id="measurementInvarianceConfiguralModelEquivalenceTest" class="section level4 hasAnchor" number="16.10.4.6">
<h4><span class="header-section-number">16.10.4.6</span> Equivalence Test<a href="bias.html#measurementInvarianceConfiguralModelEquivalenceTest" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>The <a href="https://github.com/DevPsyLab/petersenlab"><code>petersenlab</code></a> package <span class="citation">(<a href="#ref-R-petersenlab">Petersen, 2024b</a>)</span> contains the <code>equiv_chi()</code> function from <span class="citation">Counsell et al. (<a href="#ref-Counsell2020">2020</a>)</span> that performs an equivalence test: <a href="https://osf.io/cqu8v" class="uri">https://osf.io/cqu8v</a>.
An equivalence test evaluates the null hypothesis that a model is equivalent to another model.
In this case, the equivalence test evaluates the null hypothesis that a model is equivalent to another model in terms of (poor) fit.
Here, we operationalize a mediocre-fitting model as a model whose RMSEA is .08 or greater.
So, the equivalence test evaluates whether our invariance model fits significantly better than a mediocre-fitting model.
Thus, a statistically significant <em>p</em>-value indicates that our invariance model fits significantly better than the mediocre-fitting model (i.e., that invariance is established), whereas a non-significant <em>p</em>-value indicates that our invariance model does not fit significantly better than the mediocre-fitting model (i.e., that invariance is failed).</p>
<p>The chi-square equivalence test is non-significant, suggesting that the model fit of the configural invariance model is not acceptable (i.e., it is not better than the mediocre-fitting model).
In other words, configural invariance failed.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1593"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1593-1"><a href="bias.html#cb1593-1" tabindex="-1"></a><span class="fu">equiv_chi</span>(</span>
<span id="cb1593-2"><a href="bias.html#cb1593-2" tabindex="-1"></a>  <span class="at">alpha =</span> .<span class="dv">05</span>,</span>
<span id="cb1593-3"><a href="bias.html#cb1593-3" tabindex="-1"></a>  <span class="at">chi =</span> configuralInvarianceModel_chisquare,</span>
<span id="cb1593-4"><a href="bias.html#cb1593-4" tabindex="-1"></a>  <span class="at">df =</span> configuralInvarianceModel_df,</span>
<span id="cb1593-5"><a href="bias.html#cb1593-5" tabindex="-1"></a>  <span class="at">m =</span> <span class="dv">2</span>,</span>
<span id="cb1593-6"><a href="bias.html#cb1593-6" tabindex="-1"></a>  <span class="at">N_sample =</span> configuralInvarianceModel_N,</span>
<span id="cb1593-7"><a href="bias.html#cb1593-7" tabindex="-1"></a>  <span class="at">popRMSEA =</span> .<span class="dv">08</span>)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["chi"],"name":[1],"type":["dbl"],"align":["right"]},{"label":["Fml"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["popep"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["popdelt"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["pval"],"name":[5],"type":["dbl"],"align":["right"]}],"data":[{"1":"114.9699","2":"0.3845147","3":"0.1536","4":"45.9264","5":"0.8914912","_rn_":"chisq.scaled"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceConfiguralModelPermutationTest" class="section level4 hasAnchor" number="16.10.4.7">
<h4><span class="header-section-number">16.10.4.7</span> Permutation Test<a href="bias.html#measurementInvarianceConfiguralModelPermutationTest" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Permutation procedures for testing measurement invariance are described in <span class="citation">Jorgensen et al. (<a href="#ref-Jorgensen2018">2018</a>)</span>.
The permutation test evaluates the null hypothesis that the model fit is not worse than the best-possible fitting model.
A significant <em>p</em>-value indicates that the model fits significanlty worse than the best-possible fitting model.
A non-significant <em>p</em>-value indicates that the model does not fit significantly worse than the best-possible fitting model.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1594"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1594-1"><a href="bias.html#cb1594-1" tabindex="-1"></a>numPermutations <span class="ot">&lt;-</span> <span class="dv">100</span></span></code></pre></div>
</details>
<p>For reproducibility, I set the seed below.
Using the same seed will yield the same answer every time.
There is nothing special about this particular seed.
You can specify the null model as the baseline model using: <code>baseline.model = nullModelFit</code>.
<strong>Warning</strong>: this code takes a while to run based on <span class="math inline">\(100\)</span> iterations.
You can reduce the number of iterations to be faster.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1595"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1595-1"><a href="bias.html#cb1595-1" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">52242</span>)</span>
<span id="cb1595-2"><a href="bias.html#cb1595-2" tabindex="-1"></a>configuralInvarianceTest <span class="ot">&lt;-</span> <span class="fu">permuteMeasEq</span>(</span>
<span id="cb1595-3"><a href="bias.html#cb1595-3" tabindex="-1"></a>  <span class="at">nPermute =</span> numPermutations,</span>
<span id="cb1595-4"><a href="bias.html#cb1595-4" tabindex="-1"></a>  <span class="at">modelType =</span> <span class="st">&quot;mgcfa&quot;</span>,</span>
<span id="cb1595-5"><a href="bias.html#cb1595-5" tabindex="-1"></a>  <span class="at">con =</span> configuralInvarianceModel_fit,</span>
<span id="cb1595-6"><a href="bias.html#cb1595-6" tabindex="-1"></a>  <span class="at">uncon =</span> <span class="cn">NULL</span>,</span>
<span id="cb1595-7"><a href="bias.html#cb1595-7" tabindex="-1"></a>  <span class="at">AFIs =</span> myAFIs,</span>
<span id="cb1595-8"><a href="bias.html#cb1595-8" tabindex="-1"></a>  <span class="at">moreAFIs =</span> moreAFIs,</span>
<span id="cb1595-9"><a href="bias.html#cb1595-9" tabindex="-1"></a>  <span class="at">parallelType =</span> <span class="st">&quot;multicore&quot;</span>, <span class="co">#only &#39;snow&#39; works on Windows, but right now, it is throwing an error</span></span>
<span id="cb1595-10"><a href="bias.html#cb1595-10" tabindex="-1"></a>  <span class="at">iseed =</span> <span class="dv">52242</span>)</span></code></pre></div>
</details>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1596"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1596-1"><a href="bias.html#cb1596-1" tabindex="-1"></a><span class="fu">RNGkind</span>(<span class="st">&quot;default&quot;</span>, <span class="st">&quot;default&quot;</span>, <span class="st">&quot;default&quot;</span>)</span>
<span id="cb1596-2"><a href="bias.html#cb1596-2" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">52242</span>)</span>
<span id="cb1596-3"><a href="bias.html#cb1596-3" tabindex="-1"></a></span>
<span id="cb1596-4"><a href="bias.html#cb1596-4" tabindex="-1"></a>configuralInvarianceTest</span></code></pre></div>
</details>
<pre><code>Omnibus p value based on parametric chi-squared difference test:

Chisq diff    Df diff Pr(&gt;Chisq) 
    114.97      48.00       0.00 


Omnibus p values based on nonparametric permutation method: 

             AFI.Difference p.value
chisq               109.425    0.17
chisq.scaled        114.970    0.15
rmsea                 0.092    0.17
cfi                   0.929    0.15
tli                   0.893    0.15
srmr                  0.066    0.15
rmsea.robust          0.093    0.15
cfi.robust            0.929    0.14
tli.robust            0.893    0.14</code></pre>
<p>The <em>p</em>-values are non-significant, indicating that the model does not fit significantly worse than the best-possible fitting model.
In other words, configural invariance held.</p>
</div>
<div id="measurementInvarianceReliability" class="section level4 hasAnchor" number="16.10.4.8">
<h4><span class="header-section-number">16.10.4.8</span> Internal Consistency Reliability<a href="bias.html#measurementInvarianceReliability" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><a href="reliability.html#internalConsistency-reliability">Internal consistency reliability</a> of items composing the latent factors, as quantified by <a href="reliability.html#coefficientOmega">omega (<span class="math inline">\(\omega\)</span>)</a> and <a href="reliability.html#averageVarianceExtracted">average variance extracted</a> (AVE), was estimated using the <code>semTools</code> package <span class="citation">(<a href="#ref-R-semTools">Jorgensen et al., 2021</a>)</span>.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1598"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1598-1"><a href="bias.html#cb1598-1" tabindex="-1"></a><span class="fu">compRelSEM</span>(configuralInvarianceModel_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["group"],"name":[1],"type":["chr"],"align":["left"]},{"label":["visual"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["textual"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["speed"],"name":[4],"type":["dbl"],"align":["right"]}],"data":[{"1":"Pasteur","2":"0.5919833","3":"0.8744231","4":"0.6272631"},{"1":"Grant-White","2":"0.6594568","3":"0.8770047","4":"0.7697249"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1599"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1599-1"><a href="bias.html#cb1599-1" tabindex="-1"></a><span class="fu">AVE</span>(configuralInvarianceModel_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["group"],"name":[1],"type":["chr"],"align":["left"]},{"label":["visual"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["textual"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["speed"],"name":[4],"type":["dbl"],"align":["right"]}],"data":[{"1":"Pasteur","2":"0.3740814","3":"0.7069305","4":"0.3596373"},{"1":"Grant-White","2":"0.3983114","3":"0.7054973","4":"0.5231958"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvariancePathDiagram" class="section level4 hasAnchor" number="16.10.4.9">
<h4><span class="header-section-number">16.10.4.9</span> Path Diagram<a href="bias.html#measurementInvariancePathDiagram" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>A path diagram of the model generated using the <code>semPlot</code> package <span class="citation">(<a href="#ref-R-semPlot">Epskamp, 2022</a>)</span> is in the figures below (“1” = group 1; “2” = group 2).</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1600"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1600-1"><a href="bias.html#cb1600-1" tabindex="-1"></a><span class="fu">semPaths</span>(</span>
<span id="cb1600-2"><a href="bias.html#cb1600-2" tabindex="-1"></a>  configuralInvarianceModel_fit,</span>
<span id="cb1600-3"><a href="bias.html#cb1600-3" tabindex="-1"></a>  <span class="at">what =</span> <span class="st">&quot;est&quot;</span>,</span>
<span id="cb1600-4"><a href="bias.html#cb1600-4" tabindex="-1"></a>  <span class="at">layout =</span> <span class="st">&quot;tree2&quot;</span>,</span>
<span id="cb1600-5"><a href="bias.html#cb1600-5" tabindex="-1"></a>  <span class="at">ask =</span> <span class="cn">FALSE</span>,</span>
<span id="cb1600-6"><a href="bias.html#cb1600-6" tabindex="-1"></a>  <span class="at">edge.label.cex =</span> <span class="fl">1.2</span>)</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:measurementInvariancePathDiagram-1"></span>
<img src="Book_files/figure-html/measurementInvariancePathDiagram-1.png" alt="Configural Invariance Model in Confirmatory Factor Analysis (1 = Group 1; 2 = Group 2)." width="100%" />
<p class="caption">
Figure 16.53: Configural Invariance Model in Confirmatory Factor Analysis (1 = Group 1; 2 = Group 2).
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:measurementInvariancePathDiagram-2"></span>
<img src="Book_files/figure-html/measurementInvariancePathDiagram-2.png" alt="Configural Invariance Model in Confirmatory Factor Analysis (1 = Group 1; 2 = Group 2)." width="100%" />
<p class="caption">
Figure 16.54: Configural Invariance Model in Confirmatory Factor Analysis (1 = Group 1; 2 = Group 2).
</p>
</div>
</div>
</div>
<div id="measurementInvarianceMetricModel" class="section level3 hasAnchor" number="16.10.5">
<h3><span class="header-section-number">16.10.5</span> Metric (“Weak Factorial”) Invariance Model<a href="bias.html#measurementInvarianceMetricModel" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Specify invariance of factor loadings across groups.</p>
<div id="measurementInvarianceMetricModelSyntax" class="section level4 hasAnchor" number="16.10.5.1">
<h4><span class="header-section-number">16.10.5.1</span> Model Syntax<a href="bias.html#measurementInvarianceMetricModelSyntax" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1601"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1601-1"><a href="bias.html#cb1601-1" tabindex="-1"></a>metricInvarianceModel <span class="ot">&lt;-</span> <span class="fu">measEq.syntax</span>(</span>
<span id="cb1601-2"><a href="bias.html#cb1601-2" tabindex="-1"></a>  <span class="at">configural.model =</span> cfaModel,</span>
<span id="cb1601-3"><a href="bias.html#cb1601-3" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1601-4"><a href="bias.html#cb1601-4" tabindex="-1"></a>  <span class="at">ID.fac =</span> <span class="st">&quot;std.lv&quot;</span>,</span>
<span id="cb1601-5"><a href="bias.html#cb1601-5" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1601-6"><a href="bias.html#cb1601-6" tabindex="-1"></a>  <span class="at">group.equal =</span> <span class="st">&quot;loadings&quot;</span>)</span></code></pre></div>
</details>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1602"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1602-1"><a href="bias.html#cb1602-1" tabindex="-1"></a><span class="fu">cat</span>(<span class="fu">as.character</span>(metricInvarianceModel))</span></code></pre></div>
</details>
<pre><code>## LOADINGS:

visual =~ c(NA, NA)*x1 + c(lambda.1_1, lambda.1_1)*x1
visual =~ c(NA, NA)*x2 + c(lambda.2_1, lambda.2_1)*x2
visual =~ c(NA, NA)*x3 + c(lambda.3_1, lambda.3_1)*x3
textual =~ c(NA, NA)*x4 + c(lambda.4_2, lambda.4_2)*x4
textual =~ c(NA, NA)*x5 + c(lambda.5_2, lambda.5_2)*x5
textual =~ c(NA, NA)*x6 + c(lambda.6_2, lambda.6_2)*x6
speed =~ c(NA, NA)*x7 + c(lambda.7_3, lambda.7_3)*x7
speed =~ c(NA, NA)*x8 + c(lambda.8_3, lambda.8_3)*x8
speed =~ c(NA, NA)*x9 + c(lambda.9_3, lambda.9_3)*x9

## INTERCEPTS:

x1 ~ c(NA, NA)*1 + c(nu.1.g1, nu.1.g2)*1
x2 ~ c(NA, NA)*1 + c(nu.2.g1, nu.2.g2)*1
x3 ~ c(NA, NA)*1 + c(nu.3.g1, nu.3.g2)*1
x4 ~ c(NA, NA)*1 + c(nu.4.g1, nu.4.g2)*1
x5 ~ c(NA, NA)*1 + c(nu.5.g1, nu.5.g2)*1
x6 ~ c(NA, NA)*1 + c(nu.6.g1, nu.6.g2)*1
x7 ~ c(NA, NA)*1 + c(nu.7.g1, nu.7.g2)*1
x8 ~ c(NA, NA)*1 + c(nu.8.g1, nu.8.g2)*1
x9 ~ c(NA, NA)*1 + c(nu.9.g1, nu.9.g2)*1

## UNIQUE-FACTOR VARIANCES:

x1 ~~ c(NA, NA)*x1 + c(theta.1_1.g1, theta.1_1.g2)*x1
x2 ~~ c(NA, NA)*x2 + c(theta.2_2.g1, theta.2_2.g2)*x2
x3 ~~ c(NA, NA)*x3 + c(theta.3_3.g1, theta.3_3.g2)*x3
x4 ~~ c(NA, NA)*x4 + c(theta.4_4.g1, theta.4_4.g2)*x4
x5 ~~ c(NA, NA)*x5 + c(theta.5_5.g1, theta.5_5.g2)*x5
x6 ~~ c(NA, NA)*x6 + c(theta.6_6.g1, theta.6_6.g2)*x6
x7 ~~ c(NA, NA)*x7 + c(theta.7_7.g1, theta.7_7.g2)*x7
x8 ~~ c(NA, NA)*x8 + c(theta.8_8.g1, theta.8_8.g2)*x8
x9 ~~ c(NA, NA)*x9 + c(theta.9_9.g1, theta.9_9.g2)*x9

## LATENT MEANS/INTERCEPTS:

visual ~ c(0, 0)*1 + c(alpha.1.g1, alpha.1.g2)*1
textual ~ c(0, 0)*1 + c(alpha.2.g1, alpha.2.g2)*1
speed ~ c(0, 0)*1 + c(alpha.3.g1, alpha.3.g2)*1

## COMMON-FACTOR VARIANCES:

visual ~~ c(1, NA)*visual + c(psi.1_1.g1, psi.1_1.g2)*visual
textual ~~ c(1, NA)*textual + c(psi.2_2.g1, psi.2_2.g2)*textual
speed ~~ c(1, NA)*speed + c(psi.3_3.g1, psi.3_3.g2)*speed

## COMMON-FACTOR COVARIANCES:

visual ~~ c(NA, NA)*textual + c(psi.2_1.g1, psi.2_1.g2)*textual
visual ~~ c(NA, NA)*speed + c(psi.3_1.g1, psi.3_1.g2)*speed
textual ~~ c(NA, NA)*speed + c(psi.3_2.g1, psi.3_2.g2)*speed</code></pre>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1604"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1604-1"><a href="bias.html#cb1604-1" tabindex="-1"></a>metricInvarianceSyntax <span class="ot">&lt;-</span> <span class="fu">as.character</span>(metricInvarianceModel)</span></code></pre></div>
</details>
<div id="measurementInvarianceMetricModelFeatures" class="section level5 hasAnchor" number="16.10.5.1.1">
<h5><span class="header-section-number">16.10.5.1.1</span> Summary of Model Features<a href="bias.html#measurementInvarianceMetricModelFeatures" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1605"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1605-1"><a href="bias.html#cb1605-1" tabindex="-1"></a><span class="fu">summary</span>(metricInvarianceModel)</span></code></pre></div>
</details>
<pre><code>This lavaan model syntax specifies a CFA with 9 manifest indicators of 3 common factor(s).

To identify the location and scale of each common factor, the factor means and variances were fixed to 0 and 1, respectively, unless equality constraints on measurement parameters allow them to be freed.

Pattern matrix indicating num(eric), ord(ered), and lat(ent) indicators per factor:

   visual textual speed
x1 num                 
x2 num                 
x3 num                 
x4        num          
x5        num          
x6        num          
x7                num  
x8                num  
x9                num  

The following types of parameter were constrained to equality across groups:

    loadings</code></pre>
</div>
</div>
<div id="measurementInvarianceMetricModelTabular" class="section level4 hasAnchor" number="16.10.5.2">
<h4><span class="header-section-number">16.10.5.2</span> Model Syntax in Table Form<a href="bias.html#measurementInvarianceMetricModelTabular" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1607"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1607-1"><a href="bias.html#cb1607-1" tabindex="-1"></a><span class="fu">lavaanify</span>(metricInvarianceModel, <span class="at">ngroups =</span> <span class="dv">2</span>)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["id"],"name":[1],"type":["int"],"align":["right"]},{"label":["lhs"],"name":[2],"type":["chr"],"align":["left"]},{"label":["op"],"name":[3],"type":["chr"],"align":["left"]},{"label":["rhs"],"name":[4],"type":["chr"],"align":["left"]},{"label":["user"],"name":[5],"type":["int"],"align":["right"]},{"label":["block"],"name":[6],"type":["int"],"align":["right"]},{"label":["group"],"name":[7],"type":["int"],"align":["right"]},{"label":["free"],"name":[8],"type":["int"],"align":["right"]},{"label":["ustart"],"name":[9],"type":["dbl"],"align":["right"]},{"label":["exo"],"name":[10],"type":["int"],"align":["right"]},{"label":["label"],"name":[11],"type":["chr"],"align":["left"]},{"label":["plabel"],"name":[12],"type":["chr"],"align":["left"]}],"data":[{"1":"1","2":"visual","3":"=~","4":"x1","5":"1","6":"1","7":"1","8":"1","9":"NA","10":"0","11":"lambda.1_1","12":".p1."},{"1":"2","2":"visual","3":"=~","4":"x2","5":"1","6":"1","7":"1","8":"2","9":"NA","10":"0","11":"lambda.2_1","12":".p2."},{"1":"3","2":"visual","3":"=~","4":"x3","5":"1","6":"1","7":"1","8":"3","9":"NA","10":"0","11":"lambda.3_1","12":".p3."},{"1":"4","2":"textual","3":"=~","4":"x4","5":"1","6":"1","7":"1","8":"4","9":"NA","10":"0","11":"lambda.4_2","12":".p4."},{"1":"5","2":"textual","3":"=~","4":"x5","5":"1","6":"1","7":"1","8":"5","9":"NA","10":"0","11":"lambda.5_2","12":".p5."},{"1":"6","2":"textual","3":"=~","4":"x6","5":"1","6":"1","7":"1","8":"6","9":"NA","10":"0","11":"lambda.6_2","12":".p6."},{"1":"7","2":"speed","3":"=~","4":"x7","5":"1","6":"1","7":"1","8":"7","9":"NA","10":"0","11":"lambda.7_3","12":".p7."},{"1":"8","2":"speed","3":"=~","4":"x8","5":"1","6":"1","7":"1","8":"8","9":"NA","10":"0","11":"lambda.8_3","12":".p8."},{"1":"9","2":"speed","3":"=~","4":"x9","5":"1","6":"1","7":"1","8":"9","9":"NA","10":"0","11":"lambda.9_3","12":".p9."},{"1":"10","2":"x1","3":"~1","4":"","5":"1","6":"1","7":"1","8":"10","9":"NA","10":"0","11":"nu.1.g1","12":".p10."},{"1":"11","2":"x2","3":"~1","4":"","5":"1","6":"1","7":"1","8":"11","9":"NA","10":"0","11":"nu.2.g1","12":".p11."},{"1":"12","2":"x3","3":"~1","4":"","5":"1","6":"1","7":"1","8":"12","9":"NA","10":"0","11":"nu.3.g1","12":".p12."},{"1":"13","2":"x4","3":"~1","4":"","5":"1","6":"1","7":"1","8":"13","9":"NA","10":"0","11":"nu.4.g1","12":".p13."},{"1":"14","2":"x5","3":"~1","4":"","5":"1","6":"1","7":"1","8":"14","9":"NA","10":"0","11":"nu.5.g1","12":".p14."},{"1":"15","2":"x6","3":"~1","4":"","5":"1","6":"1","7":"1","8":"15","9":"NA","10":"0","11":"nu.6.g1","12":".p15."},{"1":"16","2":"x7","3":"~1","4":"","5":"1","6":"1","7":"1","8":"16","9":"NA","10":"0","11":"nu.7.g1","12":".p16."},{"1":"17","2":"x8","3":"~1","4":"","5":"1","6":"1","7":"1","8":"17","9":"NA","10":"0","11":"nu.8.g1","12":".p17."},{"1":"18","2":"x9","3":"~1","4":"","5":"1","6":"1","7":"1","8":"18","9":"NA","10":"0","11":"nu.9.g1","12":".p18."},{"1":"19","2":"x1","3":"~~","4":"x1","5":"1","6":"1","7":"1","8":"19","9":"NA","10":"0","11":"theta.1_1.g1","12":".p19."},{"1":"20","2":"x2","3":"~~","4":"x2","5":"1","6":"1","7":"1","8":"20","9":"NA","10":"0","11":"theta.2_2.g1","12":".p20."},{"1":"21","2":"x3","3":"~~","4":"x3","5":"1","6":"1","7":"1","8":"21","9":"NA","10":"0","11":"theta.3_3.g1","12":".p21."},{"1":"22","2":"x4","3":"~~","4":"x4","5":"1","6":"1","7":"1","8":"22","9":"NA","10":"0","11":"theta.4_4.g1","12":".p22."},{"1":"23","2":"x5","3":"~~","4":"x5","5":"1","6":"1","7":"1","8":"23","9":"NA","10":"0","11":"theta.5_5.g1","12":".p23."},{"1":"24","2":"x6","3":"~~","4":"x6","5":"1","6":"1","7":"1","8":"24","9":"NA","10":"0","11":"theta.6_6.g1","12":".p24."},{"1":"25","2":"x7","3":"~~","4":"x7","5":"1","6":"1","7":"1","8":"25","9":"NA","10":"0","11":"theta.7_7.g1","12":".p25."},{"1":"26","2":"x8","3":"~~","4":"x8","5":"1","6":"1","7":"1","8":"26","9":"NA","10":"0","11":"theta.8_8.g1","12":".p26."},{"1":"27","2":"x9","3":"~~","4":"x9","5":"1","6":"1","7":"1","8":"27","9":"NA","10":"0","11":"theta.9_9.g1","12":".p27."},{"1":"28","2":"visual","3":"~1","4":"","5":"1","6":"1","7":"1","8":"0","9":"0","10":"0","11":"alpha.1.g1","12":".p28."},{"1":"29","2":"textual","3":"~1","4":"","5":"1","6":"1","7":"1","8":"0","9":"0","10":"0","11":"alpha.2.g1","12":".p29."},{"1":"30","2":"speed","3":"~1","4":"","5":"1","6":"1","7":"1","8":"0","9":"0","10":"0","11":"alpha.3.g1","12":".p30."},{"1":"31","2":"visual","3":"~~","4":"visual","5":"1","6":"1","7":"1","8":"0","9":"1","10":"0","11":"psi.1_1.g1","12":".p31."},{"1":"32","2":"textual","3":"~~","4":"textual","5":"1","6":"1","7":"1","8":"0","9":"1","10":"0","11":"psi.2_2.g1","12":".p32."},{"1":"33","2":"speed","3":"~~","4":"speed","5":"1","6":"1","7":"1","8":"0","9":"1","10":"0","11":"psi.3_3.g1","12":".p33."},{"1":"34","2":"visual","3":"~~","4":"textual","5":"1","6":"1","7":"1","8":"28","9":"NA","10":"0","11":"psi.2_1.g1","12":".p34."},{"1":"35","2":"visual","3":"~~","4":"speed","5":"1","6":"1","7":"1","8":"29","9":"NA","10":"0","11":"psi.3_1.g1","12":".p35."},{"1":"36","2":"textual","3":"~~","4":"speed","5":"1","6":"1","7":"1","8":"30","9":"NA","10":"0","11":"psi.3_2.g1","12":".p36."},{"1":"37","2":"visual","3":"=~","4":"x1","5":"1","6":"2","7":"2","8":"31","9":"NA","10":"0","11":"lambda.1_1","12":".p37."},{"1":"38","2":"visual","3":"=~","4":"x2","5":"1","6":"2","7":"2","8":"32","9":"NA","10":"0","11":"lambda.2_1","12":".p38."},{"1":"39","2":"visual","3":"=~","4":"x3","5":"1","6":"2","7":"2","8":"33","9":"NA","10":"0","11":"lambda.3_1","12":".p39."},{"1":"40","2":"textual","3":"=~","4":"x4","5":"1","6":"2","7":"2","8":"34","9":"NA","10":"0","11":"lambda.4_2","12":".p40."},{"1":"41","2":"textual","3":"=~","4":"x5","5":"1","6":"2","7":"2","8":"35","9":"NA","10":"0","11":"lambda.5_2","12":".p41."},{"1":"42","2":"textual","3":"=~","4":"x6","5":"1","6":"2","7":"2","8":"36","9":"NA","10":"0","11":"lambda.6_2","12":".p42."},{"1":"43","2":"speed","3":"=~","4":"x7","5":"1","6":"2","7":"2","8":"37","9":"NA","10":"0","11":"lambda.7_3","12":".p43."},{"1":"44","2":"speed","3":"=~","4":"x8","5":"1","6":"2","7":"2","8":"38","9":"NA","10":"0","11":"lambda.8_3","12":".p44."},{"1":"45","2":"speed","3":"=~","4":"x9","5":"1","6":"2","7":"2","8":"39","9":"NA","10":"0","11":"lambda.9_3","12":".p45."},{"1":"46","2":"x1","3":"~1","4":"","5":"1","6":"2","7":"2","8":"40","9":"NA","10":"0","11":"nu.1.g2","12":".p46."},{"1":"47","2":"x2","3":"~1","4":"","5":"1","6":"2","7":"2","8":"41","9":"NA","10":"0","11":"nu.2.g2","12":".p47."},{"1":"48","2":"x3","3":"~1","4":"","5":"1","6":"2","7":"2","8":"42","9":"NA","10":"0","11":"nu.3.g2","12":".p48."},{"1":"49","2":"x4","3":"~1","4":"","5":"1","6":"2","7":"2","8":"43","9":"NA","10":"0","11":"nu.4.g2","12":".p49."},{"1":"50","2":"x5","3":"~1","4":"","5":"1","6":"2","7":"2","8":"44","9":"NA","10":"0","11":"nu.5.g2","12":".p50."},{"1":"51","2":"x6","3":"~1","4":"","5":"1","6":"2","7":"2","8":"45","9":"NA","10":"0","11":"nu.6.g2","12":".p51."},{"1":"52","2":"x7","3":"~1","4":"","5":"1","6":"2","7":"2","8":"46","9":"NA","10":"0","11":"nu.7.g2","12":".p52."},{"1":"53","2":"x8","3":"~1","4":"","5":"1","6":"2","7":"2","8":"47","9":"NA","10":"0","11":"nu.8.g2","12":".p53."},{"1":"54","2":"x9","3":"~1","4":"","5":"1","6":"2","7":"2","8":"48","9":"NA","10":"0","11":"nu.9.g2","12":".p54."},{"1":"55","2":"x1","3":"~~","4":"x1","5":"1","6":"2","7":"2","8":"49","9":"NA","10":"0","11":"theta.1_1.g2","12":".p55."},{"1":"56","2":"x2","3":"~~","4":"x2","5":"1","6":"2","7":"2","8":"50","9":"NA","10":"0","11":"theta.2_2.g2","12":".p56."},{"1":"57","2":"x3","3":"~~","4":"x3","5":"1","6":"2","7":"2","8":"51","9":"NA","10":"0","11":"theta.3_3.g2","12":".p57."},{"1":"58","2":"x4","3":"~~","4":"x4","5":"1","6":"2","7":"2","8":"52","9":"NA","10":"0","11":"theta.4_4.g2","12":".p58."},{"1":"59","2":"x5","3":"~~","4":"x5","5":"1","6":"2","7":"2","8":"53","9":"NA","10":"0","11":"theta.5_5.g2","12":".p59."},{"1":"60","2":"x6","3":"~~","4":"x6","5":"1","6":"2","7":"2","8":"54","9":"NA","10":"0","11":"theta.6_6.g2","12":".p60."},{"1":"61","2":"x7","3":"~~","4":"x7","5":"1","6":"2","7":"2","8":"55","9":"NA","10":"0","11":"theta.7_7.g2","12":".p61."},{"1":"62","2":"x8","3":"~~","4":"x8","5":"1","6":"2","7":"2","8":"56","9":"NA","10":"0","11":"theta.8_8.g2","12":".p62."},{"1":"63","2":"x9","3":"~~","4":"x9","5":"1","6":"2","7":"2","8":"57","9":"NA","10":"0","11":"theta.9_9.g2","12":".p63."},{"1":"64","2":"visual","3":"~1","4":"","5":"1","6":"2","7":"2","8":"0","9":"0","10":"0","11":"alpha.1.g2","12":".p64."},{"1":"65","2":"textual","3":"~1","4":"","5":"1","6":"2","7":"2","8":"0","9":"0","10":"0","11":"alpha.2.g2","12":".p65."},{"1":"66","2":"speed","3":"~1","4":"","5":"1","6":"2","7":"2","8":"0","9":"0","10":"0","11":"alpha.3.g2","12":".p66."},{"1":"67","2":"visual","3":"~~","4":"visual","5":"1","6":"2","7":"2","8":"58","9":"NA","10":"0","11":"psi.1_1.g2","12":".p67."},{"1":"68","2":"textual","3":"~~","4":"textual","5":"1","6":"2","7":"2","8":"59","9":"NA","10":"0","11":"psi.2_2.g2","12":".p68."},{"1":"69","2":"speed","3":"~~","4":"speed","5":"1","6":"2","7":"2","8":"60","9":"NA","10":"0","11":"psi.3_3.g2","12":".p69."},{"1":"70","2":"visual","3":"~~","4":"textual","5":"1","6":"2","7":"2","8":"61","9":"NA","10":"0","11":"psi.2_1.g2","12":".p70."},{"1":"71","2":"visual","3":"~~","4":"speed","5":"1","6":"2","7":"2","8":"62","9":"NA","10":"0","11":"psi.3_1.g2","12":".p71."},{"1":"72","2":"textual","3":"~~","4":"speed","5":"1","6":"2","7":"2","8":"63","9":"NA","10":"0","11":"psi.3_2.g2","12":".p72."},{"1":"73","2":".p1.","3":"==","4":".p37.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"74","2":".p2.","3":"==","4":".p38.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"75","2":".p3.","3":"==","4":".p39.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"76","2":".p4.","3":"==","4":".p40.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"77","2":".p5.","3":"==","4":".p41.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"78","2":".p6.","3":"==","4":".p42.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"79","2":".p7.","3":"==","4":".p43.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"80","2":".p8.","3":"==","4":".p44.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"81","2":".p9.","3":"==","4":".p45.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1608"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1608-1"><a href="bias.html#cb1608-1" tabindex="-1"></a><span class="fu">lavaanify</span>(cfaModel_metricInvariance, <span class="at">ngroups =</span> <span class="dv">2</span>)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["id"],"name":[1],"type":["int"],"align":["right"]},{"label":["lhs"],"name":[2],"type":["chr"],"align":["left"]},{"label":["op"],"name":[3],"type":["chr"],"align":["left"]},{"label":["rhs"],"name":[4],"type":["chr"],"align":["left"]},{"label":["user"],"name":[5],"type":["int"],"align":["right"]},{"label":["block"],"name":[6],"type":["int"],"align":["right"]},{"label":["group"],"name":[7],"type":["int"],"align":["right"]},{"label":["free"],"name":[8],"type":["int"],"align":["right"]},{"label":["ustart"],"name":[9],"type":["dbl"],"align":["right"]},{"label":["exo"],"name":[10],"type":["int"],"align":["right"]},{"label":["label"],"name":[11],"type":["chr"],"align":["left"]},{"label":["plabel"],"name":[12],"type":["chr"],"align":["left"]}],"data":[{"1":"1","2":"visual","3":"=~","4":"x1","5":"1","6":"1","7":"1","8":"1","9":"NA","10":"0","11":"lambdax1","12":".p1."},{"1":"2","2":"visual","3":"=~","4":"x2","5":"1","6":"1","7":"1","8":"2","9":"NA","10":"0","11":"lambdax2","12":".p2."},{"1":"3","2":"visual","3":"=~","4":"x3","5":"1","6":"1","7":"1","8":"3","9":"NA","10":"0","11":"lambdax3","12":".p3."},{"1":"4","2":"textual","3":"=~","4":"x4","5":"1","6":"1","7":"1","8":"4","9":"NA","10":"0","11":"lambdax4","12":".p4."},{"1":"5","2":"textual","3":"=~","4":"x5","5":"1","6":"1","7":"1","8":"5","9":"NA","10":"0","11":"lambdax5","12":".p5."},{"1":"6","2":"textual","3":"=~","4":"x6","5":"1","6":"1","7":"1","8":"6","9":"NA","10":"0","11":"lambdax6","12":".p6."},{"1":"7","2":"speed","3":"=~","4":"x7","5":"1","6":"1","7":"1","8":"7","9":"NA","10":"0","11":"lambdax7","12":".p7."},{"1":"8","2":"speed","3":"=~","4":"x8","5":"1","6":"1","7":"1","8":"8","9":"NA","10":"0","11":"lambdax8","12":".p8."},{"1":"9","2":"speed","3":"=~","4":"x9","5":"1","6":"1","7":"1","8":"9","9":"NA","10":"0","11":"lambdax9","12":".p9."},{"1":"10","2":"visual","3":"~1","4":"","5":"1","6":"1","7":"1","8":"0","9":"0","10":"0","11":"","12":".p10."},{"1":"11","2":"textual","3":"~1","4":"","5":"1","6":"1","7":"1","8":"0","9":"0","10":"0","11":"","12":".p11."},{"1":"12","2":"speed","3":"~1","4":"","5":"1","6":"1","7":"1","8":"0","9":"0","10":"0","11":"","12":".p12."},{"1":"13","2":"visual","3":"~~","4":"visual","5":"1","6":"1","7":"1","8":"0","9":"1","10":"0","11":"","12":".p13."},{"1":"14","2":"textual","3":"~~","4":"textual","5":"1","6":"1","7":"1","8":"0","9":"1","10":"0","11":"","12":".p14."},{"1":"15","2":"speed","3":"~~","4":"speed","5":"1","6":"1","7":"1","8":"0","9":"1","10":"0","11":"","12":".p15."},{"1":"16","2":"visual","3":"~~","4":"textual","5":"1","6":"1","7":"1","8":"10","9":"NA","10":"0","11":"","12":".p16."},{"1":"17","2":"visual","3":"~~","4":"speed","5":"1","6":"1","7":"1","8":"11","9":"NA","10":"0","11":"","12":".p17."},{"1":"18","2":"textual","3":"~~","4":"speed","5":"1","6":"1","7":"1","8":"12","9":"NA","10":"0","11":"","12":".p18."},{"1":"19","2":"x1","3":"~~","4":"x1","5":"1","6":"1","7":"1","8":"13","9":"NA","10":"0","11":"","12":".p19."},{"1":"20","2":"x2","3":"~~","4":"x2","5":"1","6":"1","7":"1","8":"14","9":"NA","10":"0","11":"","12":".p20."},{"1":"21","2":"x3","3":"~~","4":"x3","5":"1","6":"1","7":"1","8":"15","9":"NA","10":"0","11":"","12":".p21."},{"1":"22","2":"x4","3":"~~","4":"x4","5":"1","6":"1","7":"1","8":"16","9":"NA","10":"0","11":"","12":".p22."},{"1":"23","2":"x5","3":"~~","4":"x5","5":"1","6":"1","7":"1","8":"17","9":"NA","10":"0","11":"","12":".p23."},{"1":"24","2":"x6","3":"~~","4":"x6","5":"1","6":"1","7":"1","8":"18","9":"NA","10":"0","11":"","12":".p24."},{"1":"25","2":"x7","3":"~~","4":"x7","5":"1","6":"1","7":"1","8":"19","9":"NA","10":"0","11":"","12":".p25."},{"1":"26","2":"x8","3":"~~","4":"x8","5":"1","6":"1","7":"1","8":"20","9":"NA","10":"0","11":"","12":".p26."},{"1":"27","2":"x9","3":"~~","4":"x9","5":"1","6":"1","7":"1","8":"21","9":"NA","10":"0","11":"","12":".p27."},{"1":"28","2":"x1","3":"~1","4":"","5":"1","6":"1","7":"1","8":"22","9":"NA","10":"0","11":"","12":".p28."},{"1":"29","2":"x2","3":"~1","4":"","5":"1","6":"1","7":"1","8":"23","9":"NA","10":"0","11":"","12":".p29."},{"1":"30","2":"x3","3":"~1","4":"","5":"1","6":"1","7":"1","8":"24","9":"NA","10":"0","11":"","12":".p30."},{"1":"31","2":"x4","3":"~1","4":"","5":"1","6":"1","7":"1","8":"25","9":"NA","10":"0","11":"","12":".p31."},{"1":"32","2":"x5","3":"~1","4":"","5":"1","6":"1","7":"1","8":"26","9":"NA","10":"0","11":"","12":".p32."},{"1":"33","2":"x6","3":"~1","4":"","5":"1","6":"1","7":"1","8":"27","9":"NA","10":"0","11":"","12":".p33."},{"1":"34","2":"x7","3":"~1","4":"","5":"1","6":"1","7":"1","8":"28","9":"NA","10":"0","11":"","12":".p34."},{"1":"35","2":"x8","3":"~1","4":"","5":"1","6":"1","7":"1","8":"29","9":"NA","10":"0","11":"","12":".p35."},{"1":"36","2":"x9","3":"~1","4":"","5":"1","6":"1","7":"1","8":"30","9":"NA","10":"0","11":"","12":".p36."},{"1":"37","2":"visual","3":"=~","4":"x1","5":"1","6":"2","7":"2","8":"31","9":"NA","10":"0","11":"lambdax1","12":".p37."},{"1":"38","2":"visual","3":"=~","4":"x2","5":"1","6":"2","7":"2","8":"32","9":"NA","10":"0","11":"lambdax2","12":".p38."},{"1":"39","2":"visual","3":"=~","4":"x3","5":"1","6":"2","7":"2","8":"33","9":"NA","10":"0","11":"lambdax3","12":".p39."},{"1":"40","2":"textual","3":"=~","4":"x4","5":"1","6":"2","7":"2","8":"34","9":"NA","10":"0","11":"lambdax4","12":".p40."},{"1":"41","2":"textual","3":"=~","4":"x5","5":"1","6":"2","7":"2","8":"35","9":"NA","10":"0","11":"lambdax5","12":".p41."},{"1":"42","2":"textual","3":"=~","4":"x6","5":"1","6":"2","7":"2","8":"36","9":"NA","10":"0","11":"lambdax6","12":".p42."},{"1":"43","2":"speed","3":"=~","4":"x7","5":"1","6":"2","7":"2","8":"37","9":"NA","10":"0","11":"lambdax7","12":".p43."},{"1":"44","2":"speed","3":"=~","4":"x8","5":"1","6":"2","7":"2","8":"38","9":"NA","10":"0","11":"lambdax8","12":".p44."},{"1":"45","2":"speed","3":"=~","4":"x9","5":"1","6":"2","7":"2","8":"39","9":"NA","10":"0","11":"lambdax9","12":".p45."},{"1":"46","2":"visual","3":"~1","4":"","5":"1","6":"2","7":"2","8":"0","9":"0","10":"0","11":"","12":".p46."},{"1":"47","2":"textual","3":"~1","4":"","5":"1","6":"2","7":"2","8":"0","9":"0","10":"0","11":"","12":".p47."},{"1":"48","2":"speed","3":"~1","4":"","5":"1","6":"2","7":"2","8":"0","9":"0","10":"0","11":"","12":".p48."},{"1":"49","2":"visual","3":"~~","4":"visual","5":"1","6":"2","7":"2","8":"40","9":"NA","10":"0","11":"","12":".p49."},{"1":"50","2":"textual","3":"~~","4":"textual","5":"1","6":"2","7":"2","8":"41","9":"NA","10":"0","11":"","12":".p50."},{"1":"51","2":"speed","3":"~~","4":"speed","5":"1","6":"2","7":"2","8":"42","9":"NA","10":"0","11":"","12":".p51."},{"1":"52","2":"visual","3":"~~","4":"textual","5":"1","6":"2","7":"2","8":"43","9":"NA","10":"0","11":"","12":".p52."},{"1":"53","2":"visual","3":"~~","4":"speed","5":"1","6":"2","7":"2","8":"44","9":"NA","10":"0","11":"","12":".p53."},{"1":"54","2":"textual","3":"~~","4":"speed","5":"1","6":"2","7":"2","8":"45","9":"NA","10":"0","11":"","12":".p54."},{"1":"55","2":"x1","3":"~~","4":"x1","5":"1","6":"2","7":"2","8":"46","9":"NA","10":"0","11":"","12":".p55."},{"1":"56","2":"x2","3":"~~","4":"x2","5":"1","6":"2","7":"2","8":"47","9":"NA","10":"0","11":"","12":".p56."},{"1":"57","2":"x3","3":"~~","4":"x3","5":"1","6":"2","7":"2","8":"48","9":"NA","10":"0","11":"","12":".p57."},{"1":"58","2":"x4","3":"~~","4":"x4","5":"1","6":"2","7":"2","8":"49","9":"NA","10":"0","11":"","12":".p58."},{"1":"59","2":"x5","3":"~~","4":"x5","5":"1","6":"2","7":"2","8":"50","9":"NA","10":"0","11":"","12":".p59."},{"1":"60","2":"x6","3":"~~","4":"x6","5":"1","6":"2","7":"2","8":"51","9":"NA","10":"0","11":"","12":".p60."},{"1":"61","2":"x7","3":"~~","4":"x7","5":"1","6":"2","7":"2","8":"52","9":"NA","10":"0","11":"","12":".p61."},{"1":"62","2":"x8","3":"~~","4":"x8","5":"1","6":"2","7":"2","8":"53","9":"NA","10":"0","11":"","12":".p62."},{"1":"63","2":"x9","3":"~~","4":"x9","5":"1","6":"2","7":"2","8":"54","9":"NA","10":"0","11":"","12":".p63."},{"1":"64","2":"x1","3":"~1","4":"","5":"1","6":"2","7":"2","8":"55","9":"NA","10":"0","11":"","12":".p64."},{"1":"65","2":"x2","3":"~1","4":"","5":"1","6":"2","7":"2","8":"56","9":"NA","10":"0","11":"","12":".p65."},{"1":"66","2":"x3","3":"~1","4":"","5":"1","6":"2","7":"2","8":"57","9":"NA","10":"0","11":"","12":".p66."},{"1":"67","2":"x4","3":"~1","4":"","5":"1","6":"2","7":"2","8":"58","9":"NA","10":"0","11":"","12":".p67."},{"1":"68","2":"x5","3":"~1","4":"","5":"1","6":"2","7":"2","8":"59","9":"NA","10":"0","11":"","12":".p68."},{"1":"69","2":"x6","3":"~1","4":"","5":"1","6":"2","7":"2","8":"60","9":"NA","10":"0","11":"","12":".p69."},{"1":"70","2":"x7","3":"~1","4":"","5":"1","6":"2","7":"2","8":"61","9":"NA","10":"0","11":"","12":".p70."},{"1":"71","2":"x8","3":"~1","4":"","5":"1","6":"2","7":"2","8":"62","9":"NA","10":"0","11":"","12":".p71."},{"1":"72","2":"x9","3":"~1","4":"","5":"1","6":"2","7":"2","8":"63","9":"NA","10":"0","11":"","12":".p72."},{"1":"73","2":".p1.","3":"==","4":".p37.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"74","2":".p2.","3":"==","4":".p38.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"75","2":".p3.","3":"==","4":".p39.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"76","2":".p4.","3":"==","4":".p40.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"77","2":".p5.","3":"==","4":".p41.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"78","2":".p6.","3":"==","4":".p42.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"79","2":".p7.","3":"==","4":".p43.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"80","2":".p8.","3":"==","4":".p44.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"81","2":".p9.","3":"==","4":".p45.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceMetricModelFit" class="section level4 hasAnchor" number="16.10.5.3">
<h4><span class="header-section-number">16.10.5.3</span> Fit the Model<a href="bias.html#measurementInvarianceMetricModelFit" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1609"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1609-1"><a href="bias.html#cb1609-1" tabindex="-1"></a>metricInvarianceModel_fit <span class="ot">&lt;-</span> <span class="fu">cfa</span>(</span>
<span id="cb1609-2"><a href="bias.html#cb1609-2" tabindex="-1"></a>  metricInvarianceSyntax,</span>
<span id="cb1609-3"><a href="bias.html#cb1609-3" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1609-4"><a href="bias.html#cb1609-4" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1609-5"><a href="bias.html#cb1609-5" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1609-6"><a href="bias.html#cb1609-6" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span></code></pre></div>
</details>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1610"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1610-1"><a href="bias.html#cb1610-1" tabindex="-1"></a>metricInvarianceModelFullSyntax_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1610-2"><a href="bias.html#cb1610-2" tabindex="-1"></a>  cfaModel_metricInvariance,</span>
<span id="cb1610-3"><a href="bias.html#cb1610-3" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1610-4"><a href="bias.html#cb1610-4" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1610-5"><a href="bias.html#cb1610-5" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1610-6"><a href="bias.html#cb1610-6" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span></code></pre></div>
</details>
</div>
<div id="measurementInvarianceMetricModelOutput" class="section level4 hasAnchor" number="16.10.5.4">
<h4><span class="header-section-number">16.10.5.4</span> Model Summary<a href="bias.html#measurementInvarianceMetricModelOutput" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1611"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1611-1"><a href="bias.html#cb1611-1" tabindex="-1"></a><span class="fu">summary</span>(</span>
<span id="cb1611-2"><a href="bias.html#cb1611-2" tabindex="-1"></a>  metricInvarianceModel_fit,</span>
<span id="cb1611-3"><a href="bias.html#cb1611-3" tabindex="-1"></a>  <span class="at">fit.measures =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1611-4"><a href="bias.html#cb1611-4" tabindex="-1"></a>  <span class="at">standardized =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1611-5"><a href="bias.html#cb1611-5" tabindex="-1"></a>  <span class="at">rsquare =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
<pre><code>lavaan 0.6-19 ended normally after 73 iterations

  Estimator                                         ML
  Optimization method                           NLMINB
  Number of model parameters                        63
  Number of equality constraints                     9

  Number of observations per group:                   
    Pasteur                                        156
    Grant-White                                    145
  Number of missing patterns per group:               
    Pasteur                                          9
    Grant-White                                      9

Model Test User Model:
                                              Standard      Scaled
  Test Statistic                               116.313     117.955
  Degrees of freedom                                54          54
  P-value (Chi-square)                           0.000       0.000
  Scaling correction factor                                  0.986
    Yuan-Bentler correction (Mplus variant)                       
  Test statistic for each group:
    Pasteur                                     64.366      64.366
    Grant-White                                 53.590      53.590

Model Test Baseline Model:

  Test statistic                               932.665     909.019
  Degrees of freedom                                72          72
  P-value                                        0.000       0.000
  Scaling correction factor                                  1.026

User Model versus Baseline Model:

  Comparative Fit Index (CFI)                    0.928       0.924
  Tucker-Lewis Index (TLI)                       0.903       0.898
                                                                  
  Robust Comparative Fit Index (CFI)                         0.928
  Robust Tucker-Lewis Index (TLI)                            0.905

Loglikelihood and Information Criteria:

  Loglikelihood user model (H0)              -3652.277   -3652.277
  Scaling correction factor                                  0.928
      for the MLR correction                                      
  Loglikelihood unrestricted model (H1)      -3594.120   -3594.120
  Scaling correction factor                                  1.034
      for the MLR correction                                      
                                                                  
  Akaike (AIC)                                7412.554    7412.554
  Bayesian (BIC)                              7612.738    7612.738
  Sample-size adjusted Bayesian (SABIC)       7441.480    7441.480

Root Mean Square Error of Approximation:

  RMSEA                                          0.088       0.089
  90 Percent confidence interval - lower         0.066       0.067
  90 Percent confidence interval - upper         0.109       0.111
  P-value H_0: RMSEA &lt;= 0.050                    0.004       0.003
  P-value H_0: RMSEA &gt;= 0.080                    0.730       0.757
                                                                  
  Robust RMSEA                                               0.088
  90 Percent confidence interval - lower                     0.066
  90 Percent confidence interval - upper                     0.111
  P-value H_0: Robust RMSEA &lt;= 0.050                         0.004
  P-value H_0: Robust RMSEA &gt;= 0.080                         0.740

Standardized Root Mean Square Residual:

  SRMR                                           0.069       0.069

Parameter Estimates:

  Standard errors                             Sandwich
  Information bread                           Observed
  Observed information based on                Hessian


Group 1 [Pasteur]:

Latent Variables:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual =~                                                             
    x1      (l.1_)    0.885    0.122    7.258    0.000    0.885    0.759
    x2      (l.2_)    0.536    0.088    6.106    0.000    0.536    0.432
    x3      (l.3_)    0.707    0.087    8.148    0.000    0.707    0.608
  textual =~                                                            
    x4      (l.4_)    0.935    0.072   12.929    0.000    0.935    0.813
    x5      (l.5_)    1.045    0.076   13.740    0.000    1.045    0.832
    x6      (l.6_)    0.874    0.070   12.496    0.000    0.874    0.865
  speed =~                                                              
    x7      (l.7_)    0.550    0.075    7.346    0.000    0.550    0.510
    x8      (l.8_)    0.675    0.087    7.784    0.000    0.675    0.686
    x9      (l.9_)    0.602    0.088    6.867    0.000    0.602    0.603

Covariances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual ~~                                                             
    textul  (p.2_)    0.473    0.105    4.484    0.000    0.473    0.473
    speed  (p.3_1)    0.347    0.125    2.778    0.005    0.347    0.347
  textual ~~                                                            
    speed  (p.3_2)    0.320    0.096    3.342    0.001    0.320    0.320

Intercepts:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
   .x1      (n.1.)    4.948    0.095   52.314    0.000    4.948    4.243
   .x2      (n.2.)    5.984    0.098   60.949    0.000    5.984    4.823
   .x3      (n.3.)    2.510    0.093   26.891    0.000    2.510    2.157
   .x4      (n.4.)    2.814    0.091   30.886    0.000    2.814    2.448
   .x5      (n.5.)    4.002    0.105   38.265    0.000    4.002    3.186
   .x6      (n.6.)    1.929    0.079   24.377    0.000    1.929    1.907
   .x7      (n.7.)    4.439    0.087   51.128    0.000    4.439    4.117
   .x8      (n.8.)    5.564    0.079   70.170    0.000    5.564    5.655
   .x9      (n.9.)    5.426    0.080   67.539    0.000    5.426    5.435
    visual  (a.1.)    0.000                               0.000    0.000
    textual (a.2.)    0.000                               0.000    0.000
    speed   (a.3.)    0.000                               0.000    0.000

Variances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
   .x1      (t.1_)    0.577    0.166    3.482    0.000    0.577    0.424
   .x2      (t.2_)    1.252    0.162    7.718    0.000    1.252    0.813
   .x3      (t.3_)    0.853    0.127    6.737    0.000    0.853    0.630
   .x4      (t.4_)    0.447    0.072    6.189    0.000    0.447    0.339
   .x5      (t.5_)    0.486    0.075    6.445    0.000    0.486    0.308
   .x6      (t.6_)    0.258    0.057    4.499    0.000    0.258    0.252
   .x7      (t.7_)    0.860    0.114    7.549    0.000    0.860    0.739
   .x8      (t.8_)    0.512    0.097    5.296    0.000    0.512    0.529
   .x9      (t.9_)    0.635    0.118    5.376    0.000    0.635    0.637
    visual  (p.1_)    1.000                               1.000    1.000
    textual (p.2_)    1.000                               1.000    1.000
    speed   (p.3_)    1.000                               1.000    1.000

R-Square:
                   Estimate
    x1                0.576
    x2                0.187
    x3                0.370
    x4                0.661
    x5                0.692
    x6                0.748
    x7                0.261
    x8                0.471
    x9                0.363


Group 2 [Grant-White]:

Latent Variables:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual =~                                                             
    x1      (l.1_)    0.885    0.122    7.258    0.000    0.844    0.723
    x2      (l.2_)    0.536    0.088    6.106    0.000    0.512    0.467
    x3      (l.3_)    0.707    0.087    8.148    0.000    0.675    0.655
  textual =~                                                            
    x4      (l.4_)    0.935    0.072   12.929    0.000    0.937    0.846
    x5      (l.5_)    1.045    0.076   13.740    0.000    1.047    0.864
    x6      (l.6_)    0.874    0.070   12.496    0.000    0.876    0.798
  speed =~                                                              
    x7      (l.7_)    0.550    0.075    7.346    0.000    0.674    0.649
    x8      (l.8_)    0.675    0.087    7.784    0.000    0.826    0.790
    x9      (l.9_)    0.602    0.088    6.867    0.000    0.736    0.719

Covariances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual ~~                                                             
    textul  (p.2_)    0.520    0.134    3.867    0.000    0.544    0.544
    speed  (p.3_1)    0.638    0.211    3.023    0.002    0.546    0.546
  textual ~~                                                            
    speed  (p.3_2)    0.429    0.193    2.219    0.026    0.350    0.350

Intercepts:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
   .x1      (n.1.)    4.930    0.096   51.453    0.000    4.930    4.220
   .x2      (n.2.)    6.198    0.092   67.063    0.000    6.198    5.654
   .x3      (n.3.)    1.992    0.086   23.042    0.000    1.992    1.934
   .x4      (n.4.)    3.318    0.093   35.501    0.000    3.318    2.997
   .x5      (n.5.)    4.714    0.097   48.693    0.000    4.714    3.891
   .x6      (n.6.)    2.467    0.094   26.165    0.000    2.467    2.246
   .x7      (n.7.)    3.923    0.087   45.299    0.000    3.923    3.782
   .x8      (n.8.)    5.488    0.087   63.174    0.000    5.488    5.250
   .x9      (n.9.)    5.338    0.085   62.628    0.000    5.338    5.216
    visual  (a.1.)    0.000                               0.000    0.000
    textual (a.2.)    0.000                               0.000    0.000
    speed   (a.3.)    0.000                               0.000    0.000

Variances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
   .x1      (t.1_)    0.652    0.168    3.877    0.000    0.652    0.478
   .x2      (t.2_)    0.940    0.145    6.473    0.000    0.940    0.782
   .x3      (t.3_)    0.606    0.113    5.376    0.000    0.606    0.571
   .x4      (t.4_)    0.348    0.066    5.257    0.000    0.348    0.284
   .x5      (t.5_)    0.371    0.074    5.021    0.000    0.371    0.253
   .x6      (t.6_)    0.439    0.076    5.738    0.000    0.439    0.364
   .x7      (t.7_)    0.622    0.098    6.338    0.000    0.622    0.578
   .x8      (t.8_)    0.410    0.148    2.765    0.006    0.410    0.375
   .x9      (t.9_)    0.505    0.128    3.957    0.000    0.505    0.482
    visual  (p.1_)    0.910    0.234    3.895    0.000    1.000    1.000
    textual (p.2_)    1.004    0.191    5.257    0.000    1.000    1.000
    speed   (p.3_)    1.498    0.357    4.201    0.000    1.000    1.000

R-Square:
                   Estimate
    x1                0.522
    x2                0.218
    x3                0.429
    x4                0.716
    x5                0.747
    x6                0.636
    x7                0.422
    x8                0.625
    x9                0.518</code></pre>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1613"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1613-1"><a href="bias.html#cb1613-1" tabindex="-1"></a><span class="fu">summary</span>(</span>
<span id="cb1613-2"><a href="bias.html#cb1613-2" tabindex="-1"></a>  metricInvarianceModelFullSyntax_fit,</span>
<span id="cb1613-3"><a href="bias.html#cb1613-3" tabindex="-1"></a>  <span class="at">fit.measures =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1613-4"><a href="bias.html#cb1613-4" tabindex="-1"></a>  <span class="at">standardized =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1613-5"><a href="bias.html#cb1613-5" tabindex="-1"></a>  <span class="at">rsquare =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
<pre><code>lavaan 0.6-19 ended normally after 73 iterations

  Estimator                                         ML
  Optimization method                           NLMINB
  Number of model parameters                        63
  Number of equality constraints                     9

  Number of observations per group:                   
    Pasteur                                        156
    Grant-White                                    145
  Number of missing patterns per group:               
    Pasteur                                          9
    Grant-White                                      9

Model Test User Model:
                                              Standard      Scaled
  Test Statistic                               116.313     117.955
  Degrees of freedom                                54          54
  P-value (Chi-square)                           0.000       0.000
  Scaling correction factor                                  0.986
    Yuan-Bentler correction (Mplus variant)                       
  Test statistic for each group:
    Pasteur                                     64.366      64.366
    Grant-White                                 53.590      53.590

Model Test Baseline Model:

  Test statistic                               932.665     909.019
  Degrees of freedom                                72          72
  P-value                                        0.000       0.000
  Scaling correction factor                                  1.026

User Model versus Baseline Model:

  Comparative Fit Index (CFI)                    0.928       0.924
  Tucker-Lewis Index (TLI)                       0.903       0.898
                                                                  
  Robust Comparative Fit Index (CFI)                         0.928
  Robust Tucker-Lewis Index (TLI)                            0.905

Loglikelihood and Information Criteria:

  Loglikelihood user model (H0)              -3652.277   -3652.277
  Scaling correction factor                                  0.928
      for the MLR correction                                      
  Loglikelihood unrestricted model (H1)      -3594.120   -3594.120
  Scaling correction factor                                  1.034
      for the MLR correction                                      
                                                                  
  Akaike (AIC)                                7412.554    7412.554
  Bayesian (BIC)                              7612.738    7612.738
  Sample-size adjusted Bayesian (SABIC)       7441.480    7441.480

Root Mean Square Error of Approximation:

  RMSEA                                          0.088       0.089
  90 Percent confidence interval - lower         0.066       0.067
  90 Percent confidence interval - upper         0.109       0.111
  P-value H_0: RMSEA &lt;= 0.050                    0.004       0.003
  P-value H_0: RMSEA &gt;= 0.080                    0.730       0.757
                                                                  
  Robust RMSEA                                               0.088
  90 Percent confidence interval - lower                     0.066
  90 Percent confidence interval - upper                     0.111
  P-value H_0: Robust RMSEA &lt;= 0.050                         0.004
  P-value H_0: Robust RMSEA &gt;= 0.080                         0.740

Standardized Root Mean Square Residual:

  SRMR                                           0.069       0.069

Parameter Estimates:

  Standard errors                             Sandwich
  Information bread                           Observed
  Observed information based on                Hessian


Group 1 [Pasteur]:

Latent Variables:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual =~                                                             
    x1      (lmb1)    0.885    0.122    7.258    0.000    0.885    0.759
    x2      (lmb2)    0.536    0.088    6.106    0.000    0.536    0.432
    x3      (lmb3)    0.707    0.087    8.148    0.000    0.707    0.608
  textual =~                                                            
    x4      (lmb4)    0.935    0.072   12.929    0.000    0.935    0.813
    x5      (lmb5)    1.045    0.076   13.740    0.000    1.045    0.832
    x6      (lmb6)    0.874    0.070   12.496    0.000    0.874    0.865
  speed =~                                                              
    x7      (lmb7)    0.550    0.075    7.346    0.000    0.550    0.510
    x8      (lmb8)    0.675    0.087    7.784    0.000    0.675    0.686
    x9      (lmb9)    0.602    0.088    6.867    0.000    0.602    0.603

Covariances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual ~~                                                             
    textual           0.473    0.105    4.484    0.000    0.473    0.473
    speed             0.347    0.125    2.778    0.005    0.347    0.347
  textual ~~                                                            
    speed             0.320    0.096    3.342    0.001    0.320    0.320

Intercepts:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
    visual            0.000                               0.000    0.000
    textual           0.000                               0.000    0.000
    speed             0.000                               0.000    0.000
   .x1                4.948    0.095   52.314    0.000    4.948    4.243
   .x2                5.984    0.098   60.949    0.000    5.984    4.823
   .x3                2.510    0.093   26.891    0.000    2.510    2.157
   .x4                2.814    0.091   30.886    0.000    2.814    2.448
   .x5                4.002    0.105   38.265    0.000    4.002    3.186
   .x6                1.929    0.079   24.377    0.000    1.929    1.907
   .x7                4.439    0.087   51.128    0.000    4.439    4.117
   .x8                5.564    0.079   70.170    0.000    5.564    5.655
   .x9                5.426    0.080   67.539    0.000    5.426    5.435

Variances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
    visual            1.000                               1.000    1.000
    textual           1.000                               1.000    1.000
    speed             1.000                               1.000    1.000
   .x1                0.577    0.166    3.482    0.000    0.577    0.424
   .x2                1.252    0.162    7.718    0.000    1.252    0.813
   .x3                0.853    0.127    6.737    0.000    0.853    0.630
   .x4                0.447    0.072    6.189    0.000    0.447    0.339
   .x5                0.486    0.075    6.445    0.000    0.486    0.308
   .x6                0.258    0.057    4.499    0.000    0.258    0.252
   .x7                0.860    0.114    7.549    0.000    0.860    0.739
   .x8                0.512    0.097    5.296    0.000    0.512    0.529
   .x9                0.635    0.118    5.376    0.000    0.635    0.637

R-Square:
                   Estimate
    x1                0.576
    x2                0.187
    x3                0.370
    x4                0.661
    x5                0.692
    x6                0.748
    x7                0.261
    x8                0.471
    x9                0.363


Group 2 [Grant-White]:

Latent Variables:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual =~                                                             
    x1      (lmb1)    0.885    0.122    7.258    0.000    0.844    0.723
    x2      (lmb2)    0.536    0.088    6.106    0.000    0.512    0.467
    x3      (lmb3)    0.707    0.087    8.148    0.000    0.675    0.655
  textual =~                                                            
    x4      (lmb4)    0.935    0.072   12.929    0.000    0.937    0.846
    x5      (lmb5)    1.045    0.076   13.740    0.000    1.047    0.864
    x6      (lmb6)    0.874    0.070   12.496    0.000    0.876    0.798
  speed =~                                                              
    x7      (lmb7)    0.550    0.075    7.346    0.000    0.674    0.649
    x8      (lmb8)    0.675    0.087    7.784    0.000    0.826    0.790
    x9      (lmb9)    0.602    0.088    6.867    0.000    0.736    0.719

Covariances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual ~~                                                             
    textual           0.520    0.134    3.867    0.000    0.544    0.544
    speed             0.638    0.211    3.023    0.002    0.546    0.546
  textual ~~                                                            
    speed             0.429    0.193    2.219    0.026    0.350    0.350

Intercepts:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
    visual            0.000                               0.000    0.000
    textual           0.000                               0.000    0.000
    speed             0.000                               0.000    0.000
   .x1                4.930    0.096   51.453    0.000    4.930    4.220
   .x2                6.198    0.092   67.063    0.000    6.198    5.654
   .x3                1.992    0.086   23.042    0.000    1.992    1.934
   .x4                3.318    0.093   35.501    0.000    3.318    2.997
   .x5                4.714    0.097   48.693    0.000    4.714    3.891
   .x6                2.467    0.094   26.165    0.000    2.467    2.246
   .x7                3.923    0.087   45.299    0.000    3.923    3.782
   .x8                5.488    0.087   63.174    0.000    5.488    5.250
   .x9                5.338    0.085   62.628    0.000    5.338    5.216

Variances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
    visual            0.910    0.234    3.895    0.000    1.000    1.000
    textual           1.004    0.191    5.257    0.000    1.000    1.000
    speed             1.498    0.357    4.201    0.000    1.000    1.000
   .x1                0.652    0.168    3.877    0.000    0.652    0.478
   .x2                0.940    0.145    6.473    0.000    0.940    0.782
   .x3                0.606    0.113    5.376    0.000    0.606    0.571
   .x4                0.348    0.066    5.257    0.000    0.348    0.284
   .x5                0.371    0.074    5.021    0.000    0.371    0.253
   .x6                0.439    0.076    5.738    0.000    0.439    0.364
   .x7                0.622    0.098    6.338    0.000    0.622    0.578
   .x8                0.410    0.148    2.765    0.006    0.410    0.375
   .x9                0.505    0.128    3.957    0.000    0.505    0.482

R-Square:
                   Estimate
    x1                0.522
    x2                0.218
    x3                0.429
    x4                0.716
    x5                0.747
    x6                0.636
    x7                0.422
    x8                0.625
    x9                0.518</code></pre>
</div>
<div id="measurementInvarianceMetricModelFitCriteria" class="section level4 hasAnchor" number="16.10.5.5">
<h4><span class="header-section-number">16.10.5.5</span> Model Fit<a href="bias.html#measurementInvarianceMetricModelFitCriteria" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>You can specify the null model as the baseline model using: <code>baseline.model = nullModelFit</code></p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1615"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1615-1"><a href="bias.html#cb1615-1" tabindex="-1"></a><span class="fu">fitMeasures</span>(metricInvarianceModel_fit)</span></code></pre></div>
</details>
<pre><code>                         npar                          fmin 
                       54.000                         0.193 
                        chisq                            df 
                      116.313                        54.000 
                       pvalue                  chisq.scaled 
                        0.000                       117.955 
                    df.scaled                 pvalue.scaled 
                       54.000                         0.000 
         chisq.scaling.factor                baseline.chisq 
                        0.986                       932.665 
                  baseline.df               baseline.pvalue 
                       72.000                         0.000 
        baseline.chisq.scaled            baseline.df.scaled 
                      909.019                        72.000 
       baseline.pvalue.scaled baseline.chisq.scaling.factor 
                        0.000                         1.026 
                          cfi                           tli 
                        0.928                         0.903 
                   cfi.scaled                    tli.scaled 
                        0.924                         0.898 
                   cfi.robust                    tli.robust 
                        0.928                         0.905 
                         nnfi                           rfi 
                        0.903                         0.834 
                          nfi                          pnfi 
                        0.875                         0.656 
                          ifi                           rni 
                        0.929                         0.928 
                  nnfi.scaled                    rfi.scaled 
                        0.898                         0.827 
                   nfi.scaled                   pnfi.scaled 
                        0.870                         0.653 
                   ifi.scaled                    rni.scaled 
                        0.925                         0.924 
                  nnfi.robust                    rni.robust 
                        0.905                         0.928 
                         logl             unrestricted.logl 
                    -3652.277                     -3594.120 
                          aic                           bic 
                     7412.554                      7612.738 
                       ntotal                          bic2 
                      301.000                      7441.480 
            scaling.factor.h1             scaling.factor.h0 
                        1.034                         0.928 
                        rmsea                rmsea.ci.lower 
                        0.088                         0.066 
               rmsea.ci.upper                rmsea.ci.level 
                        0.109                         0.900 
                 rmsea.pvalue                rmsea.close.h0 
                        0.004                         0.050 
        rmsea.notclose.pvalue             rmsea.notclose.h0 
                        0.730                         0.080 
                 rmsea.scaled         rmsea.ci.lower.scaled 
                        0.089                         0.067 
        rmsea.ci.upper.scaled           rmsea.pvalue.scaled 
                        0.111                         0.003 
 rmsea.notclose.pvalue.scaled                  rmsea.robust 
                        0.757                         0.088 
        rmsea.ci.lower.robust         rmsea.ci.upper.robust 
                        0.066                         0.111 
          rmsea.pvalue.robust  rmsea.notclose.pvalue.robust 
                        0.004                         0.740 
                          rmr                    rmr_nomean 
                        0.085                         0.093 
                         srmr                  srmr_bentler 
                        0.069                         0.069 
          srmr_bentler_nomean                          crmr 
                        0.075                         0.073 
                  crmr_nomean                    srmr_mplus 
                        0.081                         0.072 
            srmr_mplus_nomean                         cn_05 
                        0.074                       187.721 
                        cn_01                           gfi 
                      210.793                         0.995 
                         agfi                          pgfi 
                        0.989                         0.497 
                          mfi                          ecvi 
                        0.902                         0.745 </code></pre>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1617"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1617-1"><a href="bias.html#cb1617-1" tabindex="-1"></a>metricInvarianceModelFitIndices <span class="ot">&lt;-</span> <span class="fu">fitMeasures</span>(</span>
<span id="cb1617-2"><a href="bias.html#cb1617-2" tabindex="-1"></a>  metricInvarianceModel_fit)[<span class="fu">c</span>(</span>
<span id="cb1617-3"><a href="bias.html#cb1617-3" tabindex="-1"></a>    <span class="st">&quot;cfi.robust&quot;</span>, <span class="st">&quot;rmsea.robust&quot;</span>, <span class="st">&quot;srmr&quot;</span>)]</span>
<span id="cb1617-4"><a href="bias.html#cb1617-4" tabindex="-1"></a></span>
<span id="cb1617-5"><a href="bias.html#cb1617-5" tabindex="-1"></a>metricInvarianceModel_chisquare <span class="ot">&lt;-</span> <span class="fu">fitMeasures</span>(</span>
<span id="cb1617-6"><a href="bias.html#cb1617-6" tabindex="-1"></a>  metricInvarianceModel_fit)[<span class="fu">c</span>(<span class="st">&quot;chisq.scaled&quot;</span>)]</span>
<span id="cb1617-7"><a href="bias.html#cb1617-7" tabindex="-1"></a></span>
<span id="cb1617-8"><a href="bias.html#cb1617-8" tabindex="-1"></a>metricInvarianceModel_chisquareScaling <span class="ot">&lt;-</span> <span class="fu">fitMeasures</span>(</span>
<span id="cb1617-9"><a href="bias.html#cb1617-9" tabindex="-1"></a>  metricInvarianceModel_fit)[<span class="fu">c</span>(<span class="st">&quot;chisq.scaling.factor&quot;</span>)]</span>
<span id="cb1617-10"><a href="bias.html#cb1617-10" tabindex="-1"></a></span>
<span id="cb1617-11"><a href="bias.html#cb1617-11" tabindex="-1"></a>metricInvarianceModel_df <span class="ot">&lt;-</span> <span class="fu">fitMeasures</span>(</span>
<span id="cb1617-12"><a href="bias.html#cb1617-12" tabindex="-1"></a>  metricInvarianceModel_fit)[<span class="fu">c</span>(<span class="st">&quot;df.scaled&quot;</span>)]</span>
<span id="cb1617-13"><a href="bias.html#cb1617-13" tabindex="-1"></a></span>
<span id="cb1617-14"><a href="bias.html#cb1617-14" tabindex="-1"></a>metricInvarianceModel_N <span class="ot">&lt;-</span> <span class="fu">lavInspect</span>(</span>
<span id="cb1617-15"><a href="bias.html#cb1617-15" tabindex="-1"></a>  metricInvarianceModel_fit, <span class="at">what =</span> <span class="st">&quot;ntotal&quot;</span>)</span></code></pre></div>
</details>
</div>
<div id="measurementInvarianceMetricModelCompare" class="section level4 hasAnchor" number="16.10.5.6">
<h4><span class="header-section-number">16.10.5.6</span> Compare Model Fit<a href="bias.html#measurementInvarianceMetricModelCompare" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<div id="measurementInvarianceMetricModelNestedModelComparison" class="section level5 hasAnchor" number="16.10.5.6.1">
<h5><span class="header-section-number">16.10.5.6.1</span> Nested Model (<span class="math inline">\(\chi^2\)</span>) Difference Test<a href="bias.html#measurementInvarianceMetricModelNestedModelComparison" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>The configural invariance model and the metric (“weak factorial”) invariance model are considered “nested” models.
The metric invariance model is nested within the configural invariance model because the configural invariance model includes all of the terms of the metric invariance model along with additional terms.
Model fit of nested models can be compared with a chi-square difference test, also known as a likelihood ratio test or deviance test.
A significant chi-square difference test would indicate that the simplified model with additional constraints (the metric invariance model) is significantly worse fitting than the more complex model that has fewer constraints (the configural invariance model).</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1618"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1618-1"><a href="bias.html#cb1618-1" tabindex="-1"></a><span class="fu">anova</span>(configuralInvarianceModel_fit, metricInvarianceModel_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Df"],"name":[1],"type":["int"],"align":["right"]},{"label":["AIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Chisq"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Chisq diff"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Df diff"],"name":[6],"type":["int"],"align":["right"]},{"label":["Pr(>Chisq)"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"48","2":"7417.666","3":"7640.092","4":"109.4252","5":"NA","6":"NA","7":"NA","_rn_":"configuralInvarianceModel_fit"},{"1":"54","2":"7412.554","3":"7612.738","4":"116.3133","5":"5.464443","6":"6","7":"0.4857647","_rn_":"metricInvarianceModel_fit"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>The metric invariance model did not fit significantly worse than the configural invariance model, so metric invariance held.
This provides evidence of measurement invariance of factor loadings across groups.
Measurement invariance of factor loadings across groups provides support for examining whether the groups show different associations of the factor with other constructs <span class="citation">(<a href="#ref-Little2007">Little et al., 2007</a>)</span>.</p>
<p>The <a href="https://github.com/DevPsyLab/petersenlab"><code>petersenlab</code></a> package <span class="citation">(<a href="#ref-R-petersenlab">Petersen, 2024b</a>)</span> contains the <code>satorraBentlerScaledChiSquareDifferenceTestStatistic()</code> function that performs a Satorra-Bentler scaled chi-square difference test:
Below is a Satorra-Bentler scaled chi-square difference test, where <span class="math inline">\(c0\)</span> and <span class="math inline">\(c1\)</span> are the scaling correction factor for the nested model and comparison model, respectively; <span class="math inline">\(d0\)</span> and <span class="math inline">\(d1\)</span> are the degrees of freedom of the nested model and comparison model, respectively; and <span class="math inline">\(T0\)</span> and <span class="math inline">\(T1\)</span> are the chi-square values of the nested model and comparison model, respectively.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1619"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1619-1"><a href="bias.html#cb1619-1" tabindex="-1"></a>metricInvarianceModel_chisquareDiff <span class="ot">&lt;-</span> </span>
<span id="cb1619-2"><a href="bias.html#cb1619-2" tabindex="-1"></a>  <span class="fu">satorraBentlerScaledChiSquareDifferenceTestStatistic</span>(</span>
<span id="cb1619-3"><a href="bias.html#cb1619-3" tabindex="-1"></a>    <span class="at">T0 =</span> metricInvarianceModel_chisquare,</span>
<span id="cb1619-4"><a href="bias.html#cb1619-4" tabindex="-1"></a>    <span class="at">c0 =</span> metricInvarianceModel_chisquareScaling,</span>
<span id="cb1619-5"><a href="bias.html#cb1619-5" tabindex="-1"></a>    <span class="at">d0 =</span> metricInvarianceModel_df,</span>
<span id="cb1619-6"><a href="bias.html#cb1619-6" tabindex="-1"></a>    <span class="at">T1 =</span> configuralInvarianceModel_chisquare,</span>
<span id="cb1619-7"><a href="bias.html#cb1619-7" tabindex="-1"></a>    <span class="at">c1 =</span> configuralInvarianceModel_chisquareScaling,</span>
<span id="cb1619-8"><a href="bias.html#cb1619-8" tabindex="-1"></a>    <span class="at">d1 =</span> configuralInvarianceModel_df)</span>
<span id="cb1619-9"><a href="bias.html#cb1619-9" tabindex="-1"></a></span>
<span id="cb1619-10"><a href="bias.html#cb1619-10" tabindex="-1"></a>metricInvarianceModel_chisquareDiff</span></code></pre></div>
</details>
<pre><code>chisq.scaled 
    5.464443 </code></pre>
</div>
<div id="measurementInvarianceMetricModelCompareFitCriteria" class="section level5 hasAnchor" number="16.10.5.6.2">
<h5><span class="header-section-number">16.10.5.6.2</span> Compare Other Fit Criteria<a href="bias.html#measurementInvarianceMetricModelCompareFitCriteria" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1621"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1621-1"><a href="bias.html#cb1621-1" tabindex="-1"></a><span class="fu">round</span>(metricInvarianceModelFitIndices <span class="sc">-</span> configuralInvarianceModelFitIndices, <span class="at">digits =</span> <span class="dv">3</span>)</span></code></pre></div>
</details>
<pre><code>  cfi.robust rmsea.robust         srmr 
      -0.001       -0.005        0.003 </code></pre>
</div>
<div id="measurementInvarianceMetricModelScoreBasedTest" class="section level5 hasAnchor" number="16.10.5.6.3">
<h5><span class="header-section-number">16.10.5.6.3</span> Score-Based Test<a href="bias.html#measurementInvarianceMetricModelScoreBasedTest" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Score-based tests of measurement invariance are implemented using the <code>strucchange</code> package and are described by <span class="citation">T. Wang et al. (<a href="#ref-Wang2014a">2014</a>)</span>.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1623"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1623-1"><a href="bias.html#cb1623-1" tabindex="-1"></a><span class="fu">coef</span>(metricInvarianceModel_fit)</span></code></pre></div>
</details>
<pre><code>  lambda.1_1   lambda.2_1   lambda.3_1   lambda.4_2   lambda.5_2   lambda.6_2 
       0.885        0.536        0.707        0.935        1.045        0.874 
  lambda.7_3   lambda.8_3   lambda.9_3      nu.1.g1      nu.2.g1      nu.3.g1 
       0.550        0.675        0.602        4.948        5.984        2.510 
     nu.4.g1      nu.5.g1      nu.6.g1      nu.7.g1      nu.8.g1      nu.9.g1 
       2.814        4.002        1.929        4.439        5.564        5.426 
theta.1_1.g1 theta.2_2.g1 theta.3_3.g1 theta.4_4.g1 theta.5_5.g1 theta.6_6.g1 
       0.577        1.252        0.853        0.447        0.486        0.258 
theta.7_7.g1 theta.8_8.g1 theta.9_9.g1   psi.2_1.g1   psi.3_1.g1   psi.3_2.g1 
       0.860        0.512        0.635        0.473        0.347        0.320 
  lambda.1_1   lambda.2_1   lambda.3_1   lambda.4_2   lambda.5_2   lambda.6_2 
       0.885        0.536        0.707        0.935        1.045        0.874 
  lambda.7_3   lambda.8_3   lambda.9_3      nu.1.g2      nu.2.g2      nu.3.g2 
       0.550        0.675        0.602        4.930        6.198        1.992 
     nu.4.g2      nu.5.g2      nu.6.g2      nu.7.g2      nu.8.g2      nu.9.g2 
       3.318        4.714        2.467        3.923        5.488        5.338 
theta.1_1.g2 theta.2_2.g2 theta.3_3.g2 theta.4_4.g2 theta.5_5.g2 theta.6_6.g2 
       0.652        0.940        0.606        0.348        0.371        0.439 
theta.7_7.g2 theta.8_8.g2 theta.9_9.g2   psi.1_1.g2   psi.2_2.g2   psi.3_3.g2 
       0.622        0.410        0.505        0.910        1.004        1.498 
  psi.2_1.g2   psi.3_1.g2   psi.3_2.g2 
       0.520        0.638        0.429 </code></pre>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1625"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1625-1"><a href="bias.html#cb1625-1" tabindex="-1"></a><span class="fu">coef</span>(metricInvarianceModel_fit)[<span class="dv">1</span><span class="sc">:</span><span class="dv">9</span>]</span></code></pre></div>
</details>
<pre><code>lambda.1_1 lambda.2_1 lambda.3_1 lambda.4_2 lambda.5_2 lambda.6_2 lambda.7_3 
 0.8850052  0.5363099  0.7072357  0.9347277  1.0449934  0.8743059  0.5504042 
lambda.8_3 lambda.9_3 
 0.6751277  0.6015120 </code></pre>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1627"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1627-1"><a href="bias.html#cb1627-1" tabindex="-1"></a>strucchange<span class="sc">::</span><span class="fu">sctest</span>(</span>
<span id="cb1627-2"><a href="bias.html#cb1627-2" tabindex="-1"></a>  metricInvarianceModel_fit,</span>
<span id="cb1627-3"><a href="bias.html#cb1627-3" tabindex="-1"></a>  <span class="at">order.by =</span> HolzingerSwineford1939<span class="sc">$</span>school,</span>
<span id="cb1627-4"><a href="bias.html#cb1627-4" tabindex="-1"></a>  <span class="at">parm =</span> <span class="dv">1</span><span class="sc">:</span><span class="dv">9</span>,</span>
<span id="cb1627-5"><a href="bias.html#cb1627-5" tabindex="-1"></a>  <span class="at">functional =</span> <span class="st">&quot;LMuo&quot;</span>)</span></code></pre></div>
</details>
<pre><code>
    M-fluctuation test

data:  metricInvarianceModel_fit
f(efp) = 5.0601, p-value = 0.829</code></pre>
<p>A score-based test and expected parameter change (EPC) estimates <span class="citation">(<a href="#ref-Oberski2014">Oberski, 2014</a>; <a href="#ref-Oberski2015">Oberski et al., 2015</a>)</span> are provided by the <code>lavaan</code> package <span class="citation">(<a href="#ref-R-lavaan">Rosseel et al., 2022</a>)</span>.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1629"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1629-1"><a href="bias.html#cb1629-1" tabindex="-1"></a><span class="fu">lavTestScore</span>(</span>
<span id="cb1629-2"><a href="bias.html#cb1629-2" tabindex="-1"></a>  metricInvarianceModel_fit,</span>
<span id="cb1629-3"><a href="bias.html#cb1629-3" tabindex="-1"></a>  <span class="at">epc =</span> <span class="cn">TRUE</span></span>
<span id="cb1629-4"><a href="bias.html#cb1629-4" tabindex="-1"></a>)</span></code></pre></div>
</details>
<pre><code>$test

total score test:

   test    X2 df p.value
1 score 6.199  9    0.72

$uni

univariate score tests:

   lhs op   rhs    X2 df p.value
1 .p1. == .p37. 1.303  1   0.254
2 .p2. == .p38. 0.468  1   0.494
3 .p3. == .p39. 0.334  1   0.563
4 .p4. == .p40. 0.314  1   0.575
5 .p5. == .p41. 4.447  1   0.035
6 .p6. == .p42. 2.539  1   0.111
7 .p7. == .p43. 0.027  1   0.869
8 .p8. == .p44. 0.005  1   0.942
9 .p9. == .p45. 0.007  1   0.932

$epc

expected parameter changes (epc) and expected parameter values (epv):

       lhs op     rhs block group free        label plabel   est    epc   epv
1   visual =~      x1     1     1    1   lambda.1_1   .p1. 0.885  0.072 0.957
2   visual =~      x2     1     1    2   lambda.2_1   .p2. 0.536 -0.060 0.476
3   visual =~      x3     1     1    3   lambda.3_1   .p3. 0.707 -0.052 0.655
4  textual =~      x4     1     1    4   lambda.4_2   .p4. 0.935 -0.018 0.916
5  textual =~      x5     1     1    5   lambda.5_2   .p5. 1.045  0.081 1.126
6  textual =~      x6     1     1    6   lambda.6_2   .p6. 0.874 -0.042 0.832
7    speed =~      x7     1     1    7   lambda.7_3   .p7. 0.550  0.013 0.563
8    speed =~      x8     1     1    8   lambda.8_3   .p8. 0.675 -0.004 0.671
9    speed =~      x9     1     1    9   lambda.9_3   .p9. 0.602 -0.004 0.597
10      x1 ~1             1     1   10      nu.1.g1  .p10. 4.948  0.000 4.948
11      x2 ~1             1     1   11      nu.2.g1  .p11. 5.984  0.000 5.984
12      x3 ~1             1     1   12      nu.3.g1  .p12. 2.510  0.000 2.510
13      x4 ~1             1     1   13      nu.4.g1  .p13. 2.814  0.000 2.814
14      x5 ~1             1     1   14      nu.5.g1  .p14. 4.002  0.000 4.002
15      x6 ~1             1     1   15      nu.6.g1  .p15. 1.929  0.000 1.929
16      x7 ~1             1     1   16      nu.7.g1  .p16. 4.439  0.000 4.439
17      x8 ~1             1     1   17      nu.8.g1  .p17. 5.564  0.000 5.564
18      x9 ~1             1     1   18      nu.9.g1  .p18. 5.426  0.000 5.426
19      x1 ~~      x1     1     1   19 theta.1_1.g1  .p19. 0.577 -0.094 0.483
20      x2 ~~      x2     1     1   20 theta.2_2.g1  .p20. 1.252  0.029 1.280
21      x3 ~~      x3     1     1   21 theta.3_3.g1  .p21. 0.853  0.048 0.901
22      x4 ~~      x4     1     1   22 theta.4_4.g1  .p22. 0.447  0.005 0.453
23      x5 ~~      x5     1     1   23 theta.5_5.g1  .p23. 0.486 -0.048 0.439
24      x6 ~~      x6     1     1   24 theta.6_6.g1  .p24. 0.258  0.026 0.284
25      x7 ~~      x7     1     1   25 theta.7_7.g1  .p25. 0.860 -0.007 0.853
26      x8 ~~      x8     1     1   26 theta.8_8.g1  .p26. 0.512  0.004 0.516
27      x9 ~~      x9     1     1   27 theta.9_9.g1  .p27. 0.635  0.002 0.637
28  visual ~1             1     1    0   alpha.1.g1  .p28. 0.000     NA    NA
29 textual ~1             1     1    0   alpha.2.g1  .p29. 0.000     NA    NA
30   speed ~1             1     1    0   alpha.3.g1  .p30. 0.000     NA    NA
31  visual ~~  visual     1     1    0   psi.1_1.g1  .p31. 1.000     NA    NA
32 textual ~~ textual     1     1    0   psi.2_2.g1  .p32. 1.000     NA    NA
33   speed ~~   speed     1     1    0   psi.3_3.g1  .p33. 1.000     NA    NA
34  visual ~~ textual     1     1   28   psi.2_1.g1  .p34. 0.473 -0.009 0.464
35  visual ~~   speed     1     1   29   psi.3_1.g1  .p35. 0.347 -0.007 0.341
36 textual ~~   speed     1     1   30   psi.3_2.g1  .p36. 0.320  0.001 0.320
37  visual =~      x1     2     2   31   lambda.1_1  .p37. 0.885 -0.056 0.829
38  visual =~      x2     2     2   32   lambda.2_1  .p38. 0.536  0.046 0.583
39  visual =~      x3     2     2   33   lambda.3_1  .p39. 0.707  0.033 0.740
40 textual =~      x4     2     2   34   lambda.4_2  .p40. 0.935  0.023 0.958
41 textual =~      x5     2     2   35   lambda.5_2  .p41. 1.045 -0.072 0.973
42 textual =~      x6     2     2   36   lambda.6_2  .p42. 0.874  0.065 0.939
43   speed =~      x7     2     2   37   lambda.7_3  .p43. 0.550 -0.005 0.545
44   speed =~      x8     2     2   38   lambda.8_3  .p44. 0.675  0.002 0.677
45   speed =~      x9     2     2   39   lambda.9_3  .p45. 0.602  0.002 0.604
46      x1 ~1             2     2   40      nu.1.g2  .p46. 4.930  0.000 4.930
47      x2 ~1             2     2   41      nu.2.g2  .p47. 6.198  0.000 6.198
48      x3 ~1             2     2   42      nu.3.g2  .p48. 1.992  0.000 1.992
49      x4 ~1             2     2   43      nu.4.g2  .p49. 3.318  0.000 3.318
50      x5 ~1             2     2   44      nu.5.g2  .p50. 4.714  0.000 4.714
51      x6 ~1             2     2   45      nu.6.g2  .p51. 2.467  0.000 2.467
52      x7 ~1             2     2   46      nu.7.g2  .p52. 3.923  0.000 3.923
53      x8 ~1             2     2   47      nu.8.g2  .p53. 5.488  0.000 5.488
54      x9 ~1             2     2   48      nu.9.g2  .p54. 5.338  0.000 5.338
55      x1 ~~      x1     2     2   49 theta.1_1.g2  .p55. 0.652  0.051 0.702
56      x2 ~~      x2     2     2   50 theta.2_2.g2  .p56. 0.940 -0.015 0.924
57      x3 ~~      x3     2     2   51 theta.3_3.g2  .p57. 0.606 -0.022 0.584
58      x4 ~~      x4     2     2   52 theta.4_4.g2  .p58. 0.348 -0.018 0.330
59      x5 ~~      x5     2     2   53 theta.5_5.g2  .p59. 0.371  0.050 0.421
60      x6 ~~      x6     2     2   54 theta.6_6.g2  .p60. 0.439 -0.027 0.412
61      x7 ~~      x7     2     2   55 theta.7_7.g2  .p61. 0.622  0.003 0.625
62      x8 ~~      x8     2     2   56 theta.8_8.g2  .p62. 0.410 -0.002 0.408
63      x9 ~~      x9     2     2   57 theta.9_9.g2  .p63. 0.505 -0.001 0.504
64  visual ~1             2     2    0   alpha.1.g2  .p64. 0.000     NA    NA
65 textual ~1             2     2    0   alpha.2.g2  .p65. 0.000     NA    NA
66   speed ~1             2     2    0   alpha.3.g2  .p66. 0.000     NA    NA
67  visual ~~  visual     2     2   58   psi.1_1.g2  .p67. 0.910 -0.002 0.907
68 textual ~~ textual     2     2   59   psi.2_2.g2  .p68. 1.004  0.000 1.005
69   speed ~~   speed     2     2   60   psi.3_3.g2  .p69. 1.498 -0.001 1.497
70  visual ~~ textual     2     2   61   psi.2_1.g2  .p70. 0.520  0.003 0.523
71  visual ~~   speed     2     2   62   psi.3_1.g2  .p71. 0.638  0.003 0.641
72 textual ~~   speed     2     2   63   psi.3_2.g2  .p72. 0.429  0.001 0.430
   sepc.lv sepc.all sepc.nox
1    0.072    0.062    0.062
2   -0.060   -0.048   -0.048
3   -0.052   -0.045   -0.045
4   -0.018   -0.016   -0.016
5    0.081    0.065    0.065
6   -0.042   -0.042   -0.042
7    0.013    0.012    0.012
8   -0.004   -0.004   -0.004
9   -0.004   -0.004   -0.004
10   0.000    0.000    0.000
11   0.000    0.000    0.000
12   0.000    0.000    0.000
13   0.000    0.000    0.000
14   0.000    0.000    0.000
15   0.000    0.000    0.000
16   0.000    0.000    0.000
17   0.000    0.000    0.000
18   0.000    0.000    0.000
19  -0.577   -0.424   -0.424
20   1.252    0.813    0.813
21   0.853    0.630    0.630
22   0.447    0.339    0.339
23  -0.486   -0.308   -0.308
24   0.258    0.252    0.252
25  -0.860   -0.739   -0.739
26   0.512    0.529    0.529
27   0.635    0.637    0.637
28      NA       NA       NA
29      NA       NA       NA
30      NA       NA       NA
31      NA       NA       NA
32      NA       NA       NA
33      NA       NA       NA
34  -0.009   -0.009   -0.009
35  -0.007   -0.007   -0.007
36   0.001    0.001    0.001
37  -0.053   -0.045   -0.045
38   0.044    0.040    0.040
39   0.031    0.030    0.030
40   0.023    0.021    0.021
41  -0.072   -0.060   -0.060
42   0.065    0.059    0.059
43  -0.006   -0.006   -0.006
44   0.002    0.002    0.002
45   0.003    0.003    0.003
46   0.000    0.000    0.000
47   0.000    0.000    0.000
48   0.000    0.000    0.000
49   0.000    0.000    0.000
50   0.000    0.000    0.000
51   0.000    0.000    0.000
52   0.000    0.000    0.000
53   0.000    0.000    0.000
54   0.000    0.000    0.000
55   0.652    0.478    0.478
56  -0.940   -0.782   -0.782
57  -0.606   -0.571   -0.571
58  -0.348   -0.284   -0.284
59   0.371    0.253    0.253
60  -0.439   -0.364   -0.364
61   0.622    0.578    0.578
62  -0.410   -0.375   -0.375
63  -0.505   -0.482   -0.482
64      NA       NA       NA
65      NA       NA       NA
66      NA       NA       NA
67  -1.000   -1.000   -1.000
68   1.000    1.000    1.000
69  -1.000   -1.000   -1.000
70   0.003    0.003    0.003
71   0.002    0.002    0.002
72   0.000    0.000    0.000</code></pre>
</div>
<div id="measurementInvarianceMetricModelEquivalenceTest" class="section level5 hasAnchor" number="16.10.5.6.4">
<h5><span class="header-section-number">16.10.5.6.4</span> Equivalence Test<a href="bias.html#measurementInvarianceMetricModelEquivalenceTest" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>The <a href="https://github.com/DevPsyLab/petersenlab"><code>petersenlab</code></a> package <span class="citation">(<a href="#ref-R-petersenlab">Petersen, 2024b</a>)</span> contains the <code>equiv_chi()</code> function from <span class="citation">Counsell et al. (<a href="#ref-Counsell2020">2020</a>)</span> that performs an equivalence test: <a href="https://osf.io/cqu8v" class="uri">https://osf.io/cqu8v</a>.
The chi-square equivalence test is non-significant, suggesting that the model fit is not acceptable.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1631"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1631-1"><a href="bias.html#cb1631-1" tabindex="-1"></a><span class="fu">equiv_chi</span>(</span>
<span id="cb1631-2"><a href="bias.html#cb1631-2" tabindex="-1"></a>  <span class="at">alpha =</span> .<span class="dv">05</span>,</span>
<span id="cb1631-3"><a href="bias.html#cb1631-3" tabindex="-1"></a>  <span class="at">chi =</span> metricInvarianceModel_chisquare,</span>
<span id="cb1631-4"><a href="bias.html#cb1631-4" tabindex="-1"></a>  <span class="at">df =</span> metricInvarianceModel_df,</span>
<span id="cb1631-5"><a href="bias.html#cb1631-5" tabindex="-1"></a>  <span class="at">m =</span> <span class="dv">2</span>,</span>
<span id="cb1631-6"><a href="bias.html#cb1631-6" tabindex="-1"></a>  <span class="at">N_sample =</span> metricInvarianceModel_N,</span>
<span id="cb1631-7"><a href="bias.html#cb1631-7" tabindex="-1"></a>  <span class="at">popRMSEA =</span> .<span class="dv">08</span>)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["chi"],"name":[1],"type":["dbl"],"align":["right"]},{"label":["Fml"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["popep"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["popdelt"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["pval"],"name":[5],"type":["dbl"],"align":["right"]}],"data":[{"1":"117.9554","2":"0.3944997","3":"0.1728","4":"51.6672","5":"0.7639705","_rn_":"chisq.scaled"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>Moreover, the equivalence test of the chi-square difference test is non-significant, suggesting that the degree of worsening of model fit is not acceptable.
In other words, metric invariance failed.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1632"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1632-1"><a href="bias.html#cb1632-1" tabindex="-1"></a>metricInvarianceModel_dfDiff <span class="ot">&lt;-</span> </span>
<span id="cb1632-2"><a href="bias.html#cb1632-2" tabindex="-1"></a>  metricInvarianceModel_df <span class="sc">-</span> configuralInvarianceModel_df</span>
<span id="cb1632-3"><a href="bias.html#cb1632-3" tabindex="-1"></a></span>
<span id="cb1632-4"><a href="bias.html#cb1632-4" tabindex="-1"></a><span class="fu">equiv_chi</span>(</span>
<span id="cb1632-5"><a href="bias.html#cb1632-5" tabindex="-1"></a>  <span class="at">alpha =</span> .<span class="dv">05</span>,</span>
<span id="cb1632-6"><a href="bias.html#cb1632-6" tabindex="-1"></a>  <span class="at">chi =</span> metricInvarianceModel_chisquareDiff,</span>
<span id="cb1632-7"><a href="bias.html#cb1632-7" tabindex="-1"></a>  <span class="at">df =</span> metricInvarianceModel_dfDiff,</span>
<span id="cb1632-8"><a href="bias.html#cb1632-8" tabindex="-1"></a>  <span class="at">m =</span> <span class="dv">2</span>,</span>
<span id="cb1632-9"><a href="bias.html#cb1632-9" tabindex="-1"></a>  <span class="at">N_sample =</span> metricInvarianceModel_N,</span>
<span id="cb1632-10"><a href="bias.html#cb1632-10" tabindex="-1"></a>  <span class="at">popRMSEA =</span> .<span class="dv">08</span>)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["chi"],"name":[1],"type":["dbl"],"align":["right"]},{"label":["Fml"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["popep"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["popdelt"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["pval"],"name":[5],"type":["dbl"],"align":["right"]}],"data":[{"1":"5.464443","2":"0.01827573","3":"0.0192","4":"5.7408","5":"0.1274538","_rn_":"chisq.scaled"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceMetricModelPermutationTest" class="section level5 hasAnchor" number="16.10.5.6.5">
<h5><span class="header-section-number">16.10.5.6.5</span> Permutation Test<a href="bias.html#measurementInvarianceMetricModelPermutationTest" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Permutation procedures for testing measurement invariance are described in <span class="citation">(<a href="#ref-Jorgensen2018">Jorgensen et al., 2018</a>)</span>.</p>
<p>For reproducibility, I set the seed below.
Using the same seed will yield the same answer every time.
There is nothing special about this particular seed.
You can specify the null model as the baseline model using: <code>baseline.model = nullModelFit</code>.
<strong>Warning</strong>: this code takes a while to run based on <span class="math inline">\(100\)</span> iterations.
You can reduce the number of iterations to be faster.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1633"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1633-1"><a href="bias.html#cb1633-1" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">52242</span>)</span>
<span id="cb1633-2"><a href="bias.html#cb1633-2" tabindex="-1"></a>metricInvarianceTest <span class="ot">&lt;-</span> <span class="fu">permuteMeasEq</span>(</span>
<span id="cb1633-3"><a href="bias.html#cb1633-3" tabindex="-1"></a>  <span class="at">nPermute =</span> numPermutations,</span>
<span id="cb1633-4"><a href="bias.html#cb1633-4" tabindex="-1"></a>  <span class="at">modelType =</span> <span class="st">&quot;mgcfa&quot;</span>,</span>
<span id="cb1633-5"><a href="bias.html#cb1633-5" tabindex="-1"></a>  <span class="at">con =</span> metricInvarianceModel_fit,</span>
<span id="cb1633-6"><a href="bias.html#cb1633-6" tabindex="-1"></a>  <span class="at">uncon =</span> configuralInvarianceModel_fit,</span>
<span id="cb1633-7"><a href="bias.html#cb1633-7" tabindex="-1"></a>  <span class="at">AFIs =</span> myAFIs,</span>
<span id="cb1633-8"><a href="bias.html#cb1633-8" tabindex="-1"></a>  <span class="at">moreAFIs =</span> moreAFIs,</span>
<span id="cb1633-9"><a href="bias.html#cb1633-9" tabindex="-1"></a>  <span class="at">parallelType =</span> <span class="st">&quot;multicore&quot;</span>, <span class="co">#only &#39;snow&#39; works on Windows, but right now, it is throwing an error</span></span>
<span id="cb1633-10"><a href="bias.html#cb1633-10" tabindex="-1"></a>  <span class="at">iseed =</span> <span class="dv">52242</span>)</span></code></pre></div>
</details>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1634"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1634-1"><a href="bias.html#cb1634-1" tabindex="-1"></a><span class="fu">RNGkind</span>(<span class="st">&quot;default&quot;</span>, <span class="st">&quot;default&quot;</span>, <span class="st">&quot;default&quot;</span>)</span>
<span id="cb1634-2"><a href="bias.html#cb1634-2" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">52242</span>)</span>
<span id="cb1634-3"><a href="bias.html#cb1634-3" tabindex="-1"></a></span>
<span id="cb1634-4"><a href="bias.html#cb1634-4" tabindex="-1"></a>metricInvarianceTest</span></code></pre></div>
</details>
<pre><code>Omnibus p value based on parametric chi-squared difference test:

Chisq diff    Df diff Pr(&gt;Chisq) 
     5.464      6.000      0.486 


Omnibus p values based on nonparametric permutation method: 

             AFI.Difference p.value
chisq                 6.888    0.57
chisq.scaled          2.986    0.59
rmsea                -0.005    0.60
cfi                  -0.001    0.57
tli                   0.011    0.61
srmr                  0.003    0.71
rmsea.robust         -0.005    0.61
cfi.robust           -0.001    0.59
tli.robust            0.011    0.64</code></pre>
<p>The <em>p</em>-values are non-significant, indicating that the model does not fit significantly worse than the configural invariance model.
In other words, metric invariance held.</p>
</div>
</div>
<div id="measurementInvarianceMetricModelReliability" class="section level4 hasAnchor" number="16.10.5.7">
<h4><span class="header-section-number">16.10.5.7</span> Internal Consistency Reliability<a href="bias.html#measurementInvarianceMetricModelReliability" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><a href="reliability.html#internalConsistency-reliability">Internal consistency reliability</a> of items composing the latent factors, as quantified by <a href="reliability.html#coefficientOmega">omega (<span class="math inline">\(\omega\)</span>)</a> and <a href="reliability.html#averageVarianceExtracted">average variance extracted</a> (AVE), was estimated using the <code>semTools</code> package <span class="citation">(<a href="#ref-R-semTools">Jorgensen et al., 2021</a>)</span>.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1636"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1636-1"><a href="bias.html#cb1636-1" tabindex="-1"></a><span class="fu">compRelSEM</span>(metricInvarianceModel_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["group"],"name":[1],"type":["chr"],"align":["left"]},{"label":["visual"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["textual"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["speed"],"name":[4],"type":["dbl"],"align":["right"]}],"data":[{"1":"Pasteur","2":"0.6287810","3":"0.8578309","4":"0.6234457"},{"1":"Grant-White","2":"0.6377762","3":"0.8666432","4":"0.7694884"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1637"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1637-1"><a href="bias.html#cb1637-1" tabindex="-1"></a><span class="fu">AVE</span>(metricInvarianceModel_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["group"],"name":[1],"type":["chr"],"align":["left"]},{"label":["visual"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["textual"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["speed"],"name":[4],"type":["dbl"],"align":["right"]}],"data":[{"1":"Pasteur","2":"0.3713712","3":"0.6861439","4":"0.3579097"},{"1":"Grant-White","2":"0.3939286","3":"0.7009211","4":"0.5225752"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceMetricModelPathDiagram" class="section level4 hasAnchor" number="16.10.5.8">
<h4><span class="header-section-number">16.10.5.8</span> Path Diagram<a href="bias.html#measurementInvarianceMetricModelPathDiagram" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>A path diagram of the model generated using the <code>semPlot</code> package <span class="citation">(<a href="#ref-R-semPlot">Epskamp, 2022</a>)</span> is below.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1638"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1638-1"><a href="bias.html#cb1638-1" tabindex="-1"></a><span class="fu">semPaths</span>(</span>
<span id="cb1638-2"><a href="bias.html#cb1638-2" tabindex="-1"></a>  metricInvarianceModel_fit,</span>
<span id="cb1638-3"><a href="bias.html#cb1638-3" tabindex="-1"></a>  <span class="at">what =</span> <span class="st">&quot;est&quot;</span>,</span>
<span id="cb1638-4"><a href="bias.html#cb1638-4" tabindex="-1"></a>  <span class="at">layout =</span> <span class="st">&quot;tree2&quot;</span>,</span>
<span id="cb1638-5"><a href="bias.html#cb1638-5" tabindex="-1"></a>  <span class="at">ask =</span> <span class="cn">FALSE</span>,</span>
<span id="cb1638-6"><a href="bias.html#cb1638-6" tabindex="-1"></a>  <span class="at">edge.label.cex =</span> <span class="fl">1.2</span>)</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:measurementInvarianceMetricModelPathDiagram-1"></span>
<img src="Book_files/figure-html/measurementInvarianceMetricModelPathDiagram-1.png" alt="Metric Invariance Model in Confirmatory Factor Analysis." width="100%" />
<p class="caption">
Figure 16.55: Metric Invariance Model in Confirmatory Factor Analysis.
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:measurementInvarianceMetricModelPathDiagram-2"></span>
<img src="Book_files/figure-html/measurementInvarianceMetricModelPathDiagram-2.png" alt="Metric Invariance Model in Confirmatory Factor Analysis." width="100%" />
<p class="caption">
Figure 16.56: Metric Invariance Model in Confirmatory Factor Analysis.
</p>
</div>
</div>
</div>
<div id="measurementInvarianceScalarModel" class="section level3 hasAnchor" number="16.10.6">
<h3><span class="header-section-number">16.10.6</span> Scalar (“Strong Factorial”) Invariance Model<a href="bias.html#measurementInvarianceScalarModel" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Specify invariance of factor loadings and intercepts across groups.</p>
<div id="measurementInvarianceScalarModelSyntax" class="section level4 hasAnchor" number="16.10.6.1">
<h4><span class="header-section-number">16.10.6.1</span> Model Syntax<a href="bias.html#measurementInvarianceScalarModelSyntax" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1639"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1639-1"><a href="bias.html#cb1639-1" tabindex="-1"></a>scalarInvarianceModel <span class="ot">&lt;-</span> <span class="fu">measEq.syntax</span>(</span>
<span id="cb1639-2"><a href="bias.html#cb1639-2" tabindex="-1"></a>  <span class="at">configural.model =</span> cfaModel,</span>
<span id="cb1639-3"><a href="bias.html#cb1639-3" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1639-4"><a href="bias.html#cb1639-4" tabindex="-1"></a>  <span class="at">ID.fac =</span> <span class="st">&quot;std.lv&quot;</span>,</span>
<span id="cb1639-5"><a href="bias.html#cb1639-5" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1639-6"><a href="bias.html#cb1639-6" tabindex="-1"></a>  <span class="at">group.equal =</span> <span class="fu">c</span>(<span class="st">&quot;loadings&quot;</span>,<span class="st">&quot;intercepts&quot;</span>))</span></code></pre></div>
</details>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1640"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1640-1"><a href="bias.html#cb1640-1" tabindex="-1"></a><span class="fu">cat</span>(<span class="fu">as.character</span>(scalarInvarianceModel))</span></code></pre></div>
</details>
<pre><code>## LOADINGS:

visual =~ c(NA, NA)*x1 + c(lambda.1_1, lambda.1_1)*x1
visual =~ c(NA, NA)*x2 + c(lambda.2_1, lambda.2_1)*x2
visual =~ c(NA, NA)*x3 + c(lambda.3_1, lambda.3_1)*x3
textual =~ c(NA, NA)*x4 + c(lambda.4_2, lambda.4_2)*x4
textual =~ c(NA, NA)*x5 + c(lambda.5_2, lambda.5_2)*x5
textual =~ c(NA, NA)*x6 + c(lambda.6_2, lambda.6_2)*x6
speed =~ c(NA, NA)*x7 + c(lambda.7_3, lambda.7_3)*x7
speed =~ c(NA, NA)*x8 + c(lambda.8_3, lambda.8_3)*x8
speed =~ c(NA, NA)*x9 + c(lambda.9_3, lambda.9_3)*x9

## INTERCEPTS:

x1 ~ c(NA, NA)*1 + c(nu.1, nu.1)*1
x2 ~ c(NA, NA)*1 + c(nu.2, nu.2)*1
x3 ~ c(NA, NA)*1 + c(nu.3, nu.3)*1
x4 ~ c(NA, NA)*1 + c(nu.4, nu.4)*1
x5 ~ c(NA, NA)*1 + c(nu.5, nu.5)*1
x6 ~ c(NA, NA)*1 + c(nu.6, nu.6)*1
x7 ~ c(NA, NA)*1 + c(nu.7, nu.7)*1
x8 ~ c(NA, NA)*1 + c(nu.8, nu.8)*1
x9 ~ c(NA, NA)*1 + c(nu.9, nu.9)*1

## UNIQUE-FACTOR VARIANCES:

x1 ~~ c(NA, NA)*x1 + c(theta.1_1.g1, theta.1_1.g2)*x1
x2 ~~ c(NA, NA)*x2 + c(theta.2_2.g1, theta.2_2.g2)*x2
x3 ~~ c(NA, NA)*x3 + c(theta.3_3.g1, theta.3_3.g2)*x3
x4 ~~ c(NA, NA)*x4 + c(theta.4_4.g1, theta.4_4.g2)*x4
x5 ~~ c(NA, NA)*x5 + c(theta.5_5.g1, theta.5_5.g2)*x5
x6 ~~ c(NA, NA)*x6 + c(theta.6_6.g1, theta.6_6.g2)*x6
x7 ~~ c(NA, NA)*x7 + c(theta.7_7.g1, theta.7_7.g2)*x7
x8 ~~ c(NA, NA)*x8 + c(theta.8_8.g1, theta.8_8.g2)*x8
x9 ~~ c(NA, NA)*x9 + c(theta.9_9.g1, theta.9_9.g2)*x9

## LATENT MEANS/INTERCEPTS:

visual ~ c(0, NA)*1 + c(alpha.1.g1, alpha.1.g2)*1
textual ~ c(0, NA)*1 + c(alpha.2.g1, alpha.2.g2)*1
speed ~ c(0, NA)*1 + c(alpha.3.g1, alpha.3.g2)*1

## COMMON-FACTOR VARIANCES:

visual ~~ c(1, NA)*visual + c(psi.1_1.g1, psi.1_1.g2)*visual
textual ~~ c(1, NA)*textual + c(psi.2_2.g1, psi.2_2.g2)*textual
speed ~~ c(1, NA)*speed + c(psi.3_3.g1, psi.3_3.g2)*speed

## COMMON-FACTOR COVARIANCES:

visual ~~ c(NA, NA)*textual + c(psi.2_1.g1, psi.2_1.g2)*textual
visual ~~ c(NA, NA)*speed + c(psi.3_1.g1, psi.3_1.g2)*speed
textual ~~ c(NA, NA)*speed + c(psi.3_2.g1, psi.3_2.g2)*speed</code></pre>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1642"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1642-1"><a href="bias.html#cb1642-1" tabindex="-1"></a>scalarInvarianceSyntax <span class="ot">&lt;-</span> <span class="fu">as.character</span>(scalarInvarianceModel)</span></code></pre></div>
</details>
<div id="measurementInvarianceScalarModelFeatures" class="section level5 hasAnchor" number="16.10.6.1.1">
<h5><span class="header-section-number">16.10.6.1.1</span> Summary of Model Features<a href="bias.html#measurementInvarianceScalarModelFeatures" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1643"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1643-1"><a href="bias.html#cb1643-1" tabindex="-1"></a><span class="fu">summary</span>(scalarInvarianceModel)</span></code></pre></div>
</details>
<pre><code>This lavaan model syntax specifies a CFA with 9 manifest indicators of 3 common factor(s).

To identify the location and scale of each common factor, the factor means and variances were fixed to 0 and 1, respectively, unless equality constraints on measurement parameters allow them to be freed.

Pattern matrix indicating num(eric), ord(ered), and lat(ent) indicators per factor:

   visual textual speed
x1 num                 
x2 num                 
x3 num                 
x4        num          
x5        num          
x6        num          
x7                num  
x8                num  
x9                num  

The following types of parameter were constrained to equality across groups:

    loadings
    intercepts</code></pre>
</div>
</div>
<div id="measurementInvarianceScalarModelTabular" class="section level4 hasAnchor" number="16.10.6.2">
<h4><span class="header-section-number">16.10.6.2</span> Model Syntax in Table Form<a href="bias.html#measurementInvarianceScalarModelTabular" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1645"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1645-1"><a href="bias.html#cb1645-1" tabindex="-1"></a><span class="fu">lavaanify</span>(scalarInvarianceModel, <span class="at">ngroups =</span> <span class="dv">2</span>)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["id"],"name":[1],"type":["int"],"align":["right"]},{"label":["lhs"],"name":[2],"type":["chr"],"align":["left"]},{"label":["op"],"name":[3],"type":["chr"],"align":["left"]},{"label":["rhs"],"name":[4],"type":["chr"],"align":["left"]},{"label":["user"],"name":[5],"type":["int"],"align":["right"]},{"label":["block"],"name":[6],"type":["int"],"align":["right"]},{"label":["group"],"name":[7],"type":["int"],"align":["right"]},{"label":["free"],"name":[8],"type":["int"],"align":["right"]},{"label":["ustart"],"name":[9],"type":["dbl"],"align":["right"]},{"label":["exo"],"name":[10],"type":["int"],"align":["right"]},{"label":["label"],"name":[11],"type":["chr"],"align":["left"]},{"label":["plabel"],"name":[12],"type":["chr"],"align":["left"]}],"data":[{"1":"1","2":"visual","3":"=~","4":"x1","5":"1","6":"1","7":"1","8":"1","9":"NA","10":"0","11":"lambda.1_1","12":".p1."},{"1":"2","2":"visual","3":"=~","4":"x2","5":"1","6":"1","7":"1","8":"2","9":"NA","10":"0","11":"lambda.2_1","12":".p2."},{"1":"3","2":"visual","3":"=~","4":"x3","5":"1","6":"1","7":"1","8":"3","9":"NA","10":"0","11":"lambda.3_1","12":".p3."},{"1":"4","2":"textual","3":"=~","4":"x4","5":"1","6":"1","7":"1","8":"4","9":"NA","10":"0","11":"lambda.4_2","12":".p4."},{"1":"5","2":"textual","3":"=~","4":"x5","5":"1","6":"1","7":"1","8":"5","9":"NA","10":"0","11":"lambda.5_2","12":".p5."},{"1":"6","2":"textual","3":"=~","4":"x6","5":"1","6":"1","7":"1","8":"6","9":"NA","10":"0","11":"lambda.6_2","12":".p6."},{"1":"7","2":"speed","3":"=~","4":"x7","5":"1","6":"1","7":"1","8":"7","9":"NA","10":"0","11":"lambda.7_3","12":".p7."},{"1":"8","2":"speed","3":"=~","4":"x8","5":"1","6":"1","7":"1","8":"8","9":"NA","10":"0","11":"lambda.8_3","12":".p8."},{"1":"9","2":"speed","3":"=~","4":"x9","5":"1","6":"1","7":"1","8":"9","9":"NA","10":"0","11":"lambda.9_3","12":".p9."},{"1":"10","2":"x1","3":"~1","4":"","5":"1","6":"1","7":"1","8":"10","9":"NA","10":"0","11":"nu.1","12":".p10."},{"1":"11","2":"x2","3":"~1","4":"","5":"1","6":"1","7":"1","8":"11","9":"NA","10":"0","11":"nu.2","12":".p11."},{"1":"12","2":"x3","3":"~1","4":"","5":"1","6":"1","7":"1","8":"12","9":"NA","10":"0","11":"nu.3","12":".p12."},{"1":"13","2":"x4","3":"~1","4":"","5":"1","6":"1","7":"1","8":"13","9":"NA","10":"0","11":"nu.4","12":".p13."},{"1":"14","2":"x5","3":"~1","4":"","5":"1","6":"1","7":"1","8":"14","9":"NA","10":"0","11":"nu.5","12":".p14."},{"1":"15","2":"x6","3":"~1","4":"","5":"1","6":"1","7":"1","8":"15","9":"NA","10":"0","11":"nu.6","12":".p15."},{"1":"16","2":"x7","3":"~1","4":"","5":"1","6":"1","7":"1","8":"16","9":"NA","10":"0","11":"nu.7","12":".p16."},{"1":"17","2":"x8","3":"~1","4":"","5":"1","6":"1","7":"1","8":"17","9":"NA","10":"0","11":"nu.8","12":".p17."},{"1":"18","2":"x9","3":"~1","4":"","5":"1","6":"1","7":"1","8":"18","9":"NA","10":"0","11":"nu.9","12":".p18."},{"1":"19","2":"x1","3":"~~","4":"x1","5":"1","6":"1","7":"1","8":"19","9":"NA","10":"0","11":"theta.1_1.g1","12":".p19."},{"1":"20","2":"x2","3":"~~","4":"x2","5":"1","6":"1","7":"1","8":"20","9":"NA","10":"0","11":"theta.2_2.g1","12":".p20."},{"1":"21","2":"x3","3":"~~","4":"x3","5":"1","6":"1","7":"1","8":"21","9":"NA","10":"0","11":"theta.3_3.g1","12":".p21."},{"1":"22","2":"x4","3":"~~","4":"x4","5":"1","6":"1","7":"1","8":"22","9":"NA","10":"0","11":"theta.4_4.g1","12":".p22."},{"1":"23","2":"x5","3":"~~","4":"x5","5":"1","6":"1","7":"1","8":"23","9":"NA","10":"0","11":"theta.5_5.g1","12":".p23."},{"1":"24","2":"x6","3":"~~","4":"x6","5":"1","6":"1","7":"1","8":"24","9":"NA","10":"0","11":"theta.6_6.g1","12":".p24."},{"1":"25","2":"x7","3":"~~","4":"x7","5":"1","6":"1","7":"1","8":"25","9":"NA","10":"0","11":"theta.7_7.g1","12":".p25."},{"1":"26","2":"x8","3":"~~","4":"x8","5":"1","6":"1","7":"1","8":"26","9":"NA","10":"0","11":"theta.8_8.g1","12":".p26."},{"1":"27","2":"x9","3":"~~","4":"x9","5":"1","6":"1","7":"1","8":"27","9":"NA","10":"0","11":"theta.9_9.g1","12":".p27."},{"1":"28","2":"visual","3":"~1","4":"","5":"1","6":"1","7":"1","8":"0","9":"0","10":"0","11":"alpha.1.g1","12":".p28."},{"1":"29","2":"textual","3":"~1","4":"","5":"1","6":"1","7":"1","8":"0","9":"0","10":"0","11":"alpha.2.g1","12":".p29."},{"1":"30","2":"speed","3":"~1","4":"","5":"1","6":"1","7":"1","8":"0","9":"0","10":"0","11":"alpha.3.g1","12":".p30."},{"1":"31","2":"visual","3":"~~","4":"visual","5":"1","6":"1","7":"1","8":"0","9":"1","10":"0","11":"psi.1_1.g1","12":".p31."},{"1":"32","2":"textual","3":"~~","4":"textual","5":"1","6":"1","7":"1","8":"0","9":"1","10":"0","11":"psi.2_2.g1","12":".p32."},{"1":"33","2":"speed","3":"~~","4":"speed","5":"1","6":"1","7":"1","8":"0","9":"1","10":"0","11":"psi.3_3.g1","12":".p33."},{"1":"34","2":"visual","3":"~~","4":"textual","5":"1","6":"1","7":"1","8":"28","9":"NA","10":"0","11":"psi.2_1.g1","12":".p34."},{"1":"35","2":"visual","3":"~~","4":"speed","5":"1","6":"1","7":"1","8":"29","9":"NA","10":"0","11":"psi.3_1.g1","12":".p35."},{"1":"36","2":"textual","3":"~~","4":"speed","5":"1","6":"1","7":"1","8":"30","9":"NA","10":"0","11":"psi.3_2.g1","12":".p36."},{"1":"37","2":"visual","3":"=~","4":"x1","5":"1","6":"2","7":"2","8":"31","9":"NA","10":"0","11":"lambda.1_1","12":".p37."},{"1":"38","2":"visual","3":"=~","4":"x2","5":"1","6":"2","7":"2","8":"32","9":"NA","10":"0","11":"lambda.2_1","12":".p38."},{"1":"39","2":"visual","3":"=~","4":"x3","5":"1","6":"2","7":"2","8":"33","9":"NA","10":"0","11":"lambda.3_1","12":".p39."},{"1":"40","2":"textual","3":"=~","4":"x4","5":"1","6":"2","7":"2","8":"34","9":"NA","10":"0","11":"lambda.4_2","12":".p40."},{"1":"41","2":"textual","3":"=~","4":"x5","5":"1","6":"2","7":"2","8":"35","9":"NA","10":"0","11":"lambda.5_2","12":".p41."},{"1":"42","2":"textual","3":"=~","4":"x6","5":"1","6":"2","7":"2","8":"36","9":"NA","10":"0","11":"lambda.6_2","12":".p42."},{"1":"43","2":"speed","3":"=~","4":"x7","5":"1","6":"2","7":"2","8":"37","9":"NA","10":"0","11":"lambda.7_3","12":".p43."},{"1":"44","2":"speed","3":"=~","4":"x8","5":"1","6":"2","7":"2","8":"38","9":"NA","10":"0","11":"lambda.8_3","12":".p44."},{"1":"45","2":"speed","3":"=~","4":"x9","5":"1","6":"2","7":"2","8":"39","9":"NA","10":"0","11":"lambda.9_3","12":".p45."},{"1":"46","2":"x1","3":"~1","4":"","5":"1","6":"2","7":"2","8":"40","9":"NA","10":"0","11":"nu.1","12":".p46."},{"1":"47","2":"x2","3":"~1","4":"","5":"1","6":"2","7":"2","8":"41","9":"NA","10":"0","11":"nu.2","12":".p47."},{"1":"48","2":"x3","3":"~1","4":"","5":"1","6":"2","7":"2","8":"42","9":"NA","10":"0","11":"nu.3","12":".p48."},{"1":"49","2":"x4","3":"~1","4":"","5":"1","6":"2","7":"2","8":"43","9":"NA","10":"0","11":"nu.4","12":".p49."},{"1":"50","2":"x5","3":"~1","4":"","5":"1","6":"2","7":"2","8":"44","9":"NA","10":"0","11":"nu.5","12":".p50."},{"1":"51","2":"x6","3":"~1","4":"","5":"1","6":"2","7":"2","8":"45","9":"NA","10":"0","11":"nu.6","12":".p51."},{"1":"52","2":"x7","3":"~1","4":"","5":"1","6":"2","7":"2","8":"46","9":"NA","10":"0","11":"nu.7","12":".p52."},{"1":"53","2":"x8","3":"~1","4":"","5":"1","6":"2","7":"2","8":"47","9":"NA","10":"0","11":"nu.8","12":".p53."},{"1":"54","2":"x9","3":"~1","4":"","5":"1","6":"2","7":"2","8":"48","9":"NA","10":"0","11":"nu.9","12":".p54."},{"1":"55","2":"x1","3":"~~","4":"x1","5":"1","6":"2","7":"2","8":"49","9":"NA","10":"0","11":"theta.1_1.g2","12":".p55."},{"1":"56","2":"x2","3":"~~","4":"x2","5":"1","6":"2","7":"2","8":"50","9":"NA","10":"0","11":"theta.2_2.g2","12":".p56."},{"1":"57","2":"x3","3":"~~","4":"x3","5":"1","6":"2","7":"2","8":"51","9":"NA","10":"0","11":"theta.3_3.g2","12":".p57."},{"1":"58","2":"x4","3":"~~","4":"x4","5":"1","6":"2","7":"2","8":"52","9":"NA","10":"0","11":"theta.4_4.g2","12":".p58."},{"1":"59","2":"x5","3":"~~","4":"x5","5":"1","6":"2","7":"2","8":"53","9":"NA","10":"0","11":"theta.5_5.g2","12":".p59."},{"1":"60","2":"x6","3":"~~","4":"x6","5":"1","6":"2","7":"2","8":"54","9":"NA","10":"0","11":"theta.6_6.g2","12":".p60."},{"1":"61","2":"x7","3":"~~","4":"x7","5":"1","6":"2","7":"2","8":"55","9":"NA","10":"0","11":"theta.7_7.g2","12":".p61."},{"1":"62","2":"x8","3":"~~","4":"x8","5":"1","6":"2","7":"2","8":"56","9":"NA","10":"0","11":"theta.8_8.g2","12":".p62."},{"1":"63","2":"x9","3":"~~","4":"x9","5":"1","6":"2","7":"2","8":"57","9":"NA","10":"0","11":"theta.9_9.g2","12":".p63."},{"1":"64","2":"visual","3":"~1","4":"","5":"1","6":"2","7":"2","8":"58","9":"NA","10":"0","11":"alpha.1.g2","12":".p64."},{"1":"65","2":"textual","3":"~1","4":"","5":"1","6":"2","7":"2","8":"59","9":"NA","10":"0","11":"alpha.2.g2","12":".p65."},{"1":"66","2":"speed","3":"~1","4":"","5":"1","6":"2","7":"2","8":"60","9":"NA","10":"0","11":"alpha.3.g2","12":".p66."},{"1":"67","2":"visual","3":"~~","4":"visual","5":"1","6":"2","7":"2","8":"61","9":"NA","10":"0","11":"psi.1_1.g2","12":".p67."},{"1":"68","2":"textual","3":"~~","4":"textual","5":"1","6":"2","7":"2","8":"62","9":"NA","10":"0","11":"psi.2_2.g2","12":".p68."},{"1":"69","2":"speed","3":"~~","4":"speed","5":"1","6":"2","7":"2","8":"63","9":"NA","10":"0","11":"psi.3_3.g2","12":".p69."},{"1":"70","2":"visual","3":"~~","4":"textual","5":"1","6":"2","7":"2","8":"64","9":"NA","10":"0","11":"psi.2_1.g2","12":".p70."},{"1":"71","2":"visual","3":"~~","4":"speed","5":"1","6":"2","7":"2","8":"65","9":"NA","10":"0","11":"psi.3_1.g2","12":".p71."},{"1":"72","2":"textual","3":"~~","4":"speed","5":"1","6":"2","7":"2","8":"66","9":"NA","10":"0","11":"psi.3_2.g2","12":".p72."},{"1":"73","2":".p1.","3":"==","4":".p37.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"74","2":".p2.","3":"==","4":".p38.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"75","2":".p3.","3":"==","4":".p39.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"76","2":".p4.","3":"==","4":".p40.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"77","2":".p5.","3":"==","4":".p41.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"78","2":".p6.","3":"==","4":".p42.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"79","2":".p7.","3":"==","4":".p43.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"80","2":".p8.","3":"==","4":".p44.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"81","2":".p9.","3":"==","4":".p45.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"82","2":".p10.","3":"==","4":".p46.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"83","2":".p11.","3":"==","4":".p47.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"84","2":".p12.","3":"==","4":".p48.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"85","2":".p13.","3":"==","4":".p49.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"86","2":".p14.","3":"==","4":".p50.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"87","2":".p15.","3":"==","4":".p51.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"88","2":".p16.","3":"==","4":".p52.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"89","2":".p17.","3":"==","4":".p53.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"90","2":".p18.","3":"==","4":".p54.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1646"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1646-1"><a href="bias.html#cb1646-1" tabindex="-1"></a><span class="fu">lavaanify</span>(cfaModel_scalarInvariance, <span class="at">ngroups =</span> <span class="dv">2</span>)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["id"],"name":[1],"type":["int"],"align":["right"]},{"label":["lhs"],"name":[2],"type":["chr"],"align":["left"]},{"label":["op"],"name":[3],"type":["chr"],"align":["left"]},{"label":["rhs"],"name":[4],"type":["chr"],"align":["left"]},{"label":["user"],"name":[5],"type":["int"],"align":["right"]},{"label":["block"],"name":[6],"type":["int"],"align":["right"]},{"label":["group"],"name":[7],"type":["int"],"align":["right"]},{"label":["free"],"name":[8],"type":["int"],"align":["right"]},{"label":["ustart"],"name":[9],"type":["dbl"],"align":["right"]},{"label":["exo"],"name":[10],"type":["int"],"align":["right"]},{"label":["label"],"name":[11],"type":["chr"],"align":["left"]},{"label":["plabel"],"name":[12],"type":["chr"],"align":["left"]}],"data":[{"1":"1","2":"visual","3":"=~","4":"x1","5":"1","6":"1","7":"1","8":"1","9":"NA","10":"0","11":"lambdax1","12":".p1."},{"1":"2","2":"visual","3":"=~","4":"x2","5":"1","6":"1","7":"1","8":"2","9":"NA","10":"0","11":"lambdax2","12":".p2."},{"1":"3","2":"visual","3":"=~","4":"x3","5":"1","6":"1","7":"1","8":"3","9":"NA","10":"0","11":"lambdax3","12":".p3."},{"1":"4","2":"textual","3":"=~","4":"x4","5":"1","6":"1","7":"1","8":"4","9":"NA","10":"0","11":"lambdax4","12":".p4."},{"1":"5","2":"textual","3":"=~","4":"x5","5":"1","6":"1","7":"1","8":"5","9":"NA","10":"0","11":"lambdax5","12":".p5."},{"1":"6","2":"textual","3":"=~","4":"x6","5":"1","6":"1","7":"1","8":"6","9":"NA","10":"0","11":"lambdax6","12":".p6."},{"1":"7","2":"speed","3":"=~","4":"x7","5":"1","6":"1","7":"1","8":"7","9":"NA","10":"0","11":"lambdax7","12":".p7."},{"1":"8","2":"speed","3":"=~","4":"x8","5":"1","6":"1","7":"1","8":"8","9":"NA","10":"0","11":"lambdax8","12":".p8."},{"1":"9","2":"speed","3":"=~","4":"x9","5":"1","6":"1","7":"1","8":"9","9":"NA","10":"0","11":"lambdax9","12":".p9."},{"1":"10","2":"visual","3":"~1","4":"","5":"1","6":"1","7":"1","8":"0","9":"0","10":"0","11":"","12":".p10."},{"1":"11","2":"textual","3":"~1","4":"","5":"1","6":"1","7":"1","8":"0","9":"0","10":"0","11":"","12":".p11."},{"1":"12","2":"speed","3":"~1","4":"","5":"1","6":"1","7":"1","8":"0","9":"0","10":"0","11":"","12":".p12."},{"1":"13","2":"visual","3":"~~","4":"visual","5":"1","6":"1","7":"1","8":"0","9":"1","10":"0","11":"","12":".p13."},{"1":"14","2":"textual","3":"~~","4":"textual","5":"1","6":"1","7":"1","8":"0","9":"1","10":"0","11":"","12":".p14."},{"1":"15","2":"speed","3":"~~","4":"speed","5":"1","6":"1","7":"1","8":"0","9":"1","10":"0","11":"","12":".p15."},{"1":"16","2":"visual","3":"~~","4":"textual","5":"1","6":"1","7":"1","8":"10","9":"NA","10":"0","11":"","12":".p16."},{"1":"17","2":"visual","3":"~~","4":"speed","5":"1","6":"1","7":"1","8":"11","9":"NA","10":"0","11":"","12":".p17."},{"1":"18","2":"textual","3":"~~","4":"speed","5":"1","6":"1","7":"1","8":"12","9":"NA","10":"0","11":"","12":".p18."},{"1":"19","2":"x1","3":"~~","4":"x1","5":"1","6":"1","7":"1","8":"13","9":"NA","10":"0","11":"","12":".p19."},{"1":"20","2":"x2","3":"~~","4":"x2","5":"1","6":"1","7":"1","8":"14","9":"NA","10":"0","11":"","12":".p20."},{"1":"21","2":"x3","3":"~~","4":"x3","5":"1","6":"1","7":"1","8":"15","9":"NA","10":"0","11":"","12":".p21."},{"1":"22","2":"x4","3":"~~","4":"x4","5":"1","6":"1","7":"1","8":"16","9":"NA","10":"0","11":"","12":".p22."},{"1":"23","2":"x5","3":"~~","4":"x5","5":"1","6":"1","7":"1","8":"17","9":"NA","10":"0","11":"","12":".p23."},{"1":"24","2":"x6","3":"~~","4":"x6","5":"1","6":"1","7":"1","8":"18","9":"NA","10":"0","11":"","12":".p24."},{"1":"25","2":"x7","3":"~~","4":"x7","5":"1","6":"1","7":"1","8":"19","9":"NA","10":"0","11":"","12":".p25."},{"1":"26","2":"x8","3":"~~","4":"x8","5":"1","6":"1","7":"1","8":"20","9":"NA","10":"0","11":"","12":".p26."},{"1":"27","2":"x9","3":"~~","4":"x9","5":"1","6":"1","7":"1","8":"21","9":"NA","10":"0","11":"","12":".p27."},{"1":"28","2":"x1","3":"~1","4":"","5":"1","6":"1","7":"1","8":"22","9":"NA","10":"0","11":"intx1","12":".p28."},{"1":"29","2":"x2","3":"~1","4":"","5":"1","6":"1","7":"1","8":"23","9":"NA","10":"0","11":"intx2","12":".p29."},{"1":"30","2":"x3","3":"~1","4":"","5":"1","6":"1","7":"1","8":"24","9":"NA","10":"0","11":"intx3","12":".p30."},{"1":"31","2":"x4","3":"~1","4":"","5":"1","6":"1","7":"1","8":"25","9":"NA","10":"0","11":"intx4","12":".p31."},{"1":"32","2":"x5","3":"~1","4":"","5":"1","6":"1","7":"1","8":"26","9":"NA","10":"0","11":"intx5","12":".p32."},{"1":"33","2":"x6","3":"~1","4":"","5":"1","6":"1","7":"1","8":"27","9":"NA","10":"0","11":"intx6","12":".p33."},{"1":"34","2":"x7","3":"~1","4":"","5":"1","6":"1","7":"1","8":"28","9":"NA","10":"0","11":"intx7","12":".p34."},{"1":"35","2":"x8","3":"~1","4":"","5":"1","6":"1","7":"1","8":"29","9":"NA","10":"0","11":"intx8","12":".p35."},{"1":"36","2":"x9","3":"~1","4":"","5":"1","6":"1","7":"1","8":"30","9":"NA","10":"0","11":"intx9","12":".p36."},{"1":"37","2":"visual","3":"=~","4":"x1","5":"1","6":"2","7":"2","8":"31","9":"NA","10":"0","11":"lambdax1","12":".p37."},{"1":"38","2":"visual","3":"=~","4":"x2","5":"1","6":"2","7":"2","8":"32","9":"NA","10":"0","11":"lambdax2","12":".p38."},{"1":"39","2":"visual","3":"=~","4":"x3","5":"1","6":"2","7":"2","8":"33","9":"NA","10":"0","11":"lambdax3","12":".p39."},{"1":"40","2":"textual","3":"=~","4":"x4","5":"1","6":"2","7":"2","8":"34","9":"NA","10":"0","11":"lambdax4","12":".p40."},{"1":"41","2":"textual","3":"=~","4":"x5","5":"1","6":"2","7":"2","8":"35","9":"NA","10":"0","11":"lambdax5","12":".p41."},{"1":"42","2":"textual","3":"=~","4":"x6","5":"1","6":"2","7":"2","8":"36","9":"NA","10":"0","11":"lambdax6","12":".p42."},{"1":"43","2":"speed","3":"=~","4":"x7","5":"1","6":"2","7":"2","8":"37","9":"NA","10":"0","11":"lambdax7","12":".p43."},{"1":"44","2":"speed","3":"=~","4":"x8","5":"1","6":"2","7":"2","8":"38","9":"NA","10":"0","11":"lambdax8","12":".p44."},{"1":"45","2":"speed","3":"=~","4":"x9","5":"1","6":"2","7":"2","8":"39","9":"NA","10":"0","11":"lambdax9","12":".p45."},{"1":"46","2":"visual","3":"~1","4":"","5":"1","6":"2","7":"2","8":"40","9":"NA","10":"0","11":"","12":".p46."},{"1":"47","2":"textual","3":"~1","4":"","5":"1","6":"2","7":"2","8":"41","9":"NA","10":"0","11":"","12":".p47."},{"1":"48","2":"speed","3":"~1","4":"","5":"1","6":"2","7":"2","8":"42","9":"NA","10":"0","11":"","12":".p48."},{"1":"49","2":"visual","3":"~~","4":"visual","5":"1","6":"2","7":"2","8":"43","9":"NA","10":"0","11":"","12":".p49."},{"1":"50","2":"textual","3":"~~","4":"textual","5":"1","6":"2","7":"2","8":"44","9":"NA","10":"0","11":"","12":".p50."},{"1":"51","2":"speed","3":"~~","4":"speed","5":"1","6":"2","7":"2","8":"45","9":"NA","10":"0","11":"","12":".p51."},{"1":"52","2":"visual","3":"~~","4":"textual","5":"1","6":"2","7":"2","8":"46","9":"NA","10":"0","11":"","12":".p52."},{"1":"53","2":"visual","3":"~~","4":"speed","5":"1","6":"2","7":"2","8":"47","9":"NA","10":"0","11":"","12":".p53."},{"1":"54","2":"textual","3":"~~","4":"speed","5":"1","6":"2","7":"2","8":"48","9":"NA","10":"0","11":"","12":".p54."},{"1":"55","2":"x1","3":"~~","4":"x1","5":"1","6":"2","7":"2","8":"49","9":"NA","10":"0","11":"","12":".p55."},{"1":"56","2":"x2","3":"~~","4":"x2","5":"1","6":"2","7":"2","8":"50","9":"NA","10":"0","11":"","12":".p56."},{"1":"57","2":"x3","3":"~~","4":"x3","5":"1","6":"2","7":"2","8":"51","9":"NA","10":"0","11":"","12":".p57."},{"1":"58","2":"x4","3":"~~","4":"x4","5":"1","6":"2","7":"2","8":"52","9":"NA","10":"0","11":"","12":".p58."},{"1":"59","2":"x5","3":"~~","4":"x5","5":"1","6":"2","7":"2","8":"53","9":"NA","10":"0","11":"","12":".p59."},{"1":"60","2":"x6","3":"~~","4":"x6","5":"1","6":"2","7":"2","8":"54","9":"NA","10":"0","11":"","12":".p60."},{"1":"61","2":"x7","3":"~~","4":"x7","5":"1","6":"2","7":"2","8":"55","9":"NA","10":"0","11":"","12":".p61."},{"1":"62","2":"x8","3":"~~","4":"x8","5":"1","6":"2","7":"2","8":"56","9":"NA","10":"0","11":"","12":".p62."},{"1":"63","2":"x9","3":"~~","4":"x9","5":"1","6":"2","7":"2","8":"57","9":"NA","10":"0","11":"","12":".p63."},{"1":"64","2":"x1","3":"~1","4":"","5":"1","6":"2","7":"2","8":"58","9":"NA","10":"0","11":"intx1","12":".p64."},{"1":"65","2":"x2","3":"~1","4":"","5":"1","6":"2","7":"2","8":"59","9":"NA","10":"0","11":"intx2","12":".p65."},{"1":"66","2":"x3","3":"~1","4":"","5":"1","6":"2","7":"2","8":"60","9":"NA","10":"0","11":"intx3","12":".p66."},{"1":"67","2":"x4","3":"~1","4":"","5":"1","6":"2","7":"2","8":"61","9":"NA","10":"0","11":"intx4","12":".p67."},{"1":"68","2":"x5","3":"~1","4":"","5":"1","6":"2","7":"2","8":"62","9":"NA","10":"0","11":"intx5","12":".p68."},{"1":"69","2":"x6","3":"~1","4":"","5":"1","6":"2","7":"2","8":"63","9":"NA","10":"0","11":"intx6","12":".p69."},{"1":"70","2":"x7","3":"~1","4":"","5":"1","6":"2","7":"2","8":"64","9":"NA","10":"0","11":"intx7","12":".p70."},{"1":"71","2":"x8","3":"~1","4":"","5":"1","6":"2","7":"2","8":"65","9":"NA","10":"0","11":"intx8","12":".p71."},{"1":"72","2":"x9","3":"~1","4":"","5":"1","6":"2","7":"2","8":"66","9":"NA","10":"0","11":"intx9","12":".p72."},{"1":"73","2":".p1.","3":"==","4":".p37.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"74","2":".p2.","3":"==","4":".p38.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"75","2":".p3.","3":"==","4":".p39.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"76","2":".p4.","3":"==","4":".p40.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"77","2":".p5.","3":"==","4":".p41.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"78","2":".p6.","3":"==","4":".p42.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"79","2":".p7.","3":"==","4":".p43.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"80","2":".p8.","3":"==","4":".p44.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"81","2":".p9.","3":"==","4":".p45.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"82","2":".p28.","3":"==","4":".p64.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"83","2":".p29.","3":"==","4":".p65.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"84","2":".p30.","3":"==","4":".p66.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"85","2":".p31.","3":"==","4":".p67.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"86","2":".p32.","3":"==","4":".p68.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"87","2":".p33.","3":"==","4":".p69.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"88","2":".p34.","3":"==","4":".p70.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"89","2":".p35.","3":"==","4":".p71.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"90","2":".p36.","3":"==","4":".p72.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceScalarModelFit" class="section level4 hasAnchor" number="16.10.6.3">
<h4><span class="header-section-number">16.10.6.3</span> Fit the Model<a href="bias.html#measurementInvarianceScalarModelFit" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1647"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1647-1"><a href="bias.html#cb1647-1" tabindex="-1"></a>scalarInvarianceModel_fit <span class="ot">&lt;-</span> <span class="fu">cfa</span>(</span>
<span id="cb1647-2"><a href="bias.html#cb1647-2" tabindex="-1"></a>  scalarInvarianceSyntax,</span>
<span id="cb1647-3"><a href="bias.html#cb1647-3" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1647-4"><a href="bias.html#cb1647-4" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1647-5"><a href="bias.html#cb1647-5" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1647-6"><a href="bias.html#cb1647-6" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span></code></pre></div>
</details>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1648"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1648-1"><a href="bias.html#cb1648-1" tabindex="-1"></a>scalarInvarianceModelFullSyntax_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1648-2"><a href="bias.html#cb1648-2" tabindex="-1"></a>  cfaModel_scalarInvariance,</span>
<span id="cb1648-3"><a href="bias.html#cb1648-3" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1648-4"><a href="bias.html#cb1648-4" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1648-5"><a href="bias.html#cb1648-5" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1648-6"><a href="bias.html#cb1648-6" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span></code></pre></div>
</details>
</div>
<div id="measurementInvarianceScalarModelOutput" class="section level4 hasAnchor" number="16.10.6.4">
<h4><span class="header-section-number">16.10.6.4</span> Model Summary<a href="bias.html#measurementInvarianceScalarModelOutput" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1649"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1649-1"><a href="bias.html#cb1649-1" tabindex="-1"></a><span class="fu">summary</span>(</span>
<span id="cb1649-2"><a href="bias.html#cb1649-2" tabindex="-1"></a>  scalarInvarianceModel_fit,</span>
<span id="cb1649-3"><a href="bias.html#cb1649-3" tabindex="-1"></a>  <span class="at">fit.measures =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1649-4"><a href="bias.html#cb1649-4" tabindex="-1"></a>  <span class="at">standardized =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1649-5"><a href="bias.html#cb1649-5" tabindex="-1"></a>  <span class="at">rsquare =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
<pre><code>lavaan 0.6-19 ended normally after 66 iterations

  Estimator                                         ML
  Optimization method                           NLMINB
  Number of model parameters                        66
  Number of equality constraints                    18

  Number of observations per group:                   
    Pasteur                                        156
    Grant-White                                    145
  Number of missing patterns per group:               
    Pasteur                                          9
    Grant-White                                      9

Model Test User Model:
                                              Standard      Scaled
  Test Statistic                               156.955     159.442
  Degrees of freedom                                60          60
  P-value (Chi-square)                           0.000       0.000
  Scaling correction factor                                  0.984
    Yuan-Bentler correction (Mplus variant)                       
  Test statistic for each group:
    Pasteur                                     86.521      86.521
    Grant-White                                 72.921      72.921

Model Test Baseline Model:

  Test statistic                               932.665     909.019
  Degrees of freedom                                72          72
  P-value                                        0.000       0.000
  Scaling correction factor                                  1.026

User Model versus Baseline Model:

  Comparative Fit Index (CFI)                    0.887       0.881
  Tucker-Lewis Index (TLI)                       0.865       0.857
                                                                  
  Robust Comparative Fit Index (CFI)                         0.890
  Robust Tucker-Lewis Index (TLI)                            0.868

Loglikelihood and Information Criteria:

  Loglikelihood user model (H0)              -3672.597   -3672.597
  Scaling correction factor                                  0.797
      for the MLR correction                                      
  Loglikelihood unrestricted model (H1)      -3594.120   -3594.120
  Scaling correction factor                                  1.034
      for the MLR correction                                      
                                                                  
  Akaike (AIC)                                7441.195    7441.195
  Bayesian (BIC)                              7619.136    7619.136
  Sample-size adjusted Bayesian (SABIC)       7466.908    7466.908

Root Mean Square Error of Approximation:

  RMSEA                                          0.104       0.105
  90 Percent confidence interval - lower         0.084       0.085
  90 Percent confidence interval - upper         0.124       0.125
  P-value H_0: RMSEA &lt;= 0.050                    0.000       0.000
  P-value H_0: RMSEA &gt;= 0.080                    0.975       0.980
                                                                  
  Robust RMSEA                                               0.104
  90 Percent confidence interval - lower                     0.083
  90 Percent confidence interval - upper                     0.124
  P-value H_0: Robust RMSEA &lt;= 0.050                         0.000
  P-value H_0: Robust RMSEA &gt;= 0.080                         0.972

Standardized Root Mean Square Residual:

  SRMR                                           0.080       0.080

Parameter Estimates:

  Standard errors                             Sandwich
  Information bread                           Observed
  Observed information based on                Hessian


Group 1 [Pasteur]:

Latent Variables:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual =~                                                             
    x1      (l.1_)    0.874    0.132    6.630    0.000    0.874    0.751
    x2      (l.2_)    0.512    0.086    5.984    0.000    0.512    0.411
    x3      (l.3_)    0.722    0.095    7.610    0.000    0.722    0.604
  textual =~                                                            
    x4      (l.4_)    0.921    0.071   12.920    0.000    0.921    0.807
    x5      (l.5_)    1.059    0.077   13.763    0.000    1.059    0.837
    x6      (l.6_)    0.875    0.066   13.259    0.000    0.875    0.865
  speed =~                                                              
    x7      (l.7_)    0.566    0.081    6.946    0.000    0.566    0.512
    x8      (l.8_)    0.657    0.082    8.028    0.000    0.657    0.669
    x9      (l.9_)    0.602    0.085    7.049    0.000    0.602    0.604

Covariances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual ~~                                                             
    textul  (p.2_)    0.473    0.108    4.397    0.000    0.473    0.473
    speed  (p.3_1)    0.355    0.125    2.842    0.004    0.355    0.355
  textual ~~                                                            
    speed  (p.3_2)    0.322    0.095    3.377    0.001    0.322    0.322

Intercepts:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
   .x1      (nu.1)    5.016    0.094   53.146    0.000    5.016    4.309
   .x2      (nu.2)    6.156    0.088   69.594    0.000    6.156    4.944
   .x3      (nu.3)    2.292    0.098   23.329    0.000    2.292    1.916
   .x4      (nu.4)    2.780    0.086   32.469    0.000    2.780    2.435
   .x5      (nu.5)    4.035    0.104   38.981    0.000    4.035    3.189
   .x6      (nu.6)    1.928    0.074   25.882    0.000    1.928    1.907
   .x7      (nu.7)    4.244    0.078   54.164    0.000    4.244    3.840
   .x8      (nu.8)    5.627    0.076   73.915    0.000    5.627    5.726
   .x9      (nu.9)    5.474    0.073   74.934    0.000    5.474    5.492
    visual  (a.1.)    0.000                               0.000    0.000
    textual (a.2.)    0.000                               0.000    0.000
    speed   (a.3.)    0.000                               0.000    0.000

Variances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
   .x1      (t.1_)    0.591    0.179    3.295    0.001    0.591    0.436
   .x2      (t.2_)    1.288    0.160    8.075    0.000    1.288    0.831
   .x3      (t.3_)    0.910    0.151    6.046    0.000    0.910    0.636
   .x4      (t.4_)    0.454    0.073    6.245    0.000    0.454    0.349
   .x5      (t.5_)    0.480    0.076    6.295    0.000    0.480    0.300
   .x6      (t.6_)    0.257    0.058    4.401    0.000    0.257    0.252
   .x7      (t.7_)    0.901    0.129    7.000    0.000    0.901    0.738
   .x8      (t.8_)    0.534    0.091    5.838    0.000    0.534    0.553
   .x9      (t.9_)    0.631    0.115    5.469    0.000    0.631    0.635
    visual  (p.1_)    1.000                               1.000    1.000
    textual (p.2_)    1.000                               1.000    1.000
    speed   (p.3_)    1.000                               1.000    1.000

R-Square:
                   Estimate
    x1                0.564
    x2                0.169
    x3                0.364
    x4                0.651
    x5                0.700
    x6                0.748
    x7                0.262
    x8                0.447
    x9                0.365


Group 2 [Grant-White]:

Latent Variables:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual =~                                                             
    x1      (l.1_)    0.874    0.132    6.630    0.000    0.827    0.711
    x2      (l.2_)    0.512    0.086    5.984    0.000    0.485    0.441
    x3      (l.3_)    0.722    0.095    7.610    0.000    0.684    0.649
  textual =~                                                            
    x4      (l.4_)    0.921    0.071   12.920    0.000    0.921    0.838
    x5      (l.5_)    1.059    0.077   13.763    0.000    1.059    0.869
    x6      (l.6_)    0.875    0.066   13.259    0.000    0.875    0.797
  speed =~                                                              
    x7      (l.7_)    0.566    0.081    6.946    0.000    0.692    0.651
    x8      (l.8_)    0.657    0.082    8.028    0.000    0.803    0.773
    x9      (l.9_)    0.602    0.085    7.049    0.000    0.736    0.722

Covariances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual ~~                                                             
    textul  (p.2_)    0.519    0.136    3.829    0.000    0.549    0.549
    speed  (p.3_1)    0.649    0.215    3.014    0.003    0.561    0.561
  textual ~~                                                            
    speed  (p.3_2)    0.442    0.189    2.335    0.020    0.361    0.361

Intercepts:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
   .x1      (nu.1)    5.016    0.094   53.146    0.000    5.016    4.312
   .x2      (nu.2)    6.156    0.088   69.594    0.000    6.156    5.602
   .x3      (nu.3)    2.292    0.098   23.329    0.000    2.292    2.177
   .x4      (nu.4)    2.780    0.086   32.469    0.000    2.780    2.529
   .x5      (nu.5)    4.035    0.104   38.981    0.000    4.035    3.312
   .x6      (nu.6)    1.928    0.074   25.882    0.000    1.928    1.756
   .x7      (nu.7)    4.244    0.078   54.164    0.000    4.244    3.990
   .x8      (nu.8)    5.627    0.076   73.915    0.000    5.627    5.412
   .x9      (nu.9)    5.474    0.073   74.934    0.000    5.474    5.370
    visual  (a.1.)   -0.193    0.175   -1.101    0.271   -0.204   -0.204
    textual (a.2.)    0.617    0.135    4.587    0.000    0.617    0.617
    speed   (a.3.)   -0.294    0.161   -1.828    0.068   -0.240   -0.240

Variances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
   .x1      (t.1_)    0.668    0.176    3.793    0.000    0.668    0.494
   .x2      (t.2_)    0.973    0.152    6.384    0.000    0.973    0.805
   .x3      (t.3_)    0.641    0.133    4.833    0.000    0.641    0.578
   .x4      (t.4_)    0.359    0.066    5.464    0.000    0.359    0.297
   .x5      (t.5_)    0.363    0.074    4.883    0.000    0.363    0.245
   .x6      (t.6_)    0.441    0.075    5.851    0.000    0.441    0.366
   .x7      (t.7_)    0.652    0.108    6.016    0.000    0.652    0.577
   .x8      (t.8_)    0.436    0.152    2.872    0.004    0.436    0.403
   .x9      (t.9_)    0.497    0.127    3.918    0.000    0.497    0.478
    visual  (p.1_)    0.896    0.235    3.816    0.000    1.000    1.000
    textual (p.2_)    1.000    0.194    5.157    0.000    1.000    1.000
    speed   (p.3_)    1.495    0.360    4.156    0.000    1.000    1.000

R-Square:
                   Estimate
    x1                0.506
    x2                0.195
    x3                0.422
    x4                0.703
    x5                0.755
    x6                0.634
    x7                0.423
    x8                0.597
    x9                0.522</code></pre>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1651"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1651-1"><a href="bias.html#cb1651-1" tabindex="-1"></a><span class="fu">summary</span>(</span>
<span id="cb1651-2"><a href="bias.html#cb1651-2" tabindex="-1"></a>  scalarInvarianceModelFullSyntax_fit,</span>
<span id="cb1651-3"><a href="bias.html#cb1651-3" tabindex="-1"></a>  <span class="at">fit.measures =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1651-4"><a href="bias.html#cb1651-4" tabindex="-1"></a>  <span class="at">standardized =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1651-5"><a href="bias.html#cb1651-5" tabindex="-1"></a>  <span class="at">rsquare =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
<pre><code>lavaan 0.6-19 ended normally after 66 iterations

  Estimator                                         ML
  Optimization method                           NLMINB
  Number of model parameters                        66
  Number of equality constraints                    18

  Number of observations per group:                   
    Pasteur                                        156
    Grant-White                                    145
  Number of missing patterns per group:               
    Pasteur                                          9
    Grant-White                                      9

Model Test User Model:
                                              Standard      Scaled
  Test Statistic                               156.955     159.442
  Degrees of freedom                                60          60
  P-value (Chi-square)                           0.000       0.000
  Scaling correction factor                                  0.984
    Yuan-Bentler correction (Mplus variant)                       
  Test statistic for each group:
    Pasteur                                     86.520      86.520
    Grant-White                                 72.921      72.921

Model Test Baseline Model:

  Test statistic                               932.665     909.019
  Degrees of freedom                                72          72
  P-value                                        0.000       0.000
  Scaling correction factor                                  1.026

User Model versus Baseline Model:

  Comparative Fit Index (CFI)                    0.887       0.881
  Tucker-Lewis Index (TLI)                       0.865       0.857
                                                                  
  Robust Comparative Fit Index (CFI)                         0.890
  Robust Tucker-Lewis Index (TLI)                            0.868

Loglikelihood and Information Criteria:

  Loglikelihood user model (H0)              -3672.597   -3672.597
  Scaling correction factor                                  0.797
      for the MLR correction                                      
  Loglikelihood unrestricted model (H1)      -3594.120   -3594.120
  Scaling correction factor                                  1.034
      for the MLR correction                                      
                                                                  
  Akaike (AIC)                                7441.195    7441.195
  Bayesian (BIC)                              7619.136    7619.136
  Sample-size adjusted Bayesian (SABIC)       7466.908    7466.908

Root Mean Square Error of Approximation:

  RMSEA                                          0.104       0.105
  90 Percent confidence interval - lower         0.084       0.085
  90 Percent confidence interval - upper         0.124       0.125
  P-value H_0: RMSEA &lt;= 0.050                    0.000       0.000
  P-value H_0: RMSEA &gt;= 0.080                    0.975       0.980
                                                                  
  Robust RMSEA                                               0.104
  90 Percent confidence interval - lower                     0.083
  90 Percent confidence interval - upper                     0.124
  P-value H_0: Robust RMSEA &lt;= 0.050                         0.000
  P-value H_0: Robust RMSEA &gt;= 0.080                         0.972

Standardized Root Mean Square Residual:

  SRMR                                           0.080       0.080

Parameter Estimates:

  Standard errors                             Sandwich
  Information bread                           Observed
  Observed information based on                Hessian


Group 1 [Pasteur]:

Latent Variables:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual =~                                                             
    x1      (lmb1)    0.874    0.132    6.630    0.000    0.874    0.751
    x2      (lmb2)    0.512    0.086    5.984    0.000    0.512    0.411
    x3      (lmb3)    0.722    0.095    7.610    0.000    0.722    0.604
  textual =~                                                            
    x4      (lmb4)    0.921    0.071   12.920    0.000    0.921    0.807
    x5      (lmb5)    1.059    0.077   13.763    0.000    1.059    0.837
    x6      (lmb6)    0.875    0.066   13.259    0.000    0.875    0.865
  speed =~                                                              
    x7      (lmb7)    0.566    0.081    6.946    0.000    0.566    0.512
    x8      (lmb8)    0.657    0.082    8.028    0.000    0.657    0.669
    x9      (lmb9)    0.602    0.085    7.049    0.000    0.602    0.604

Covariances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual ~~                                                             
    textual           0.473    0.108    4.397    0.000    0.473    0.473
    speed             0.355    0.125    2.842    0.004    0.355    0.355
  textual ~~                                                            
    speed             0.322    0.095    3.377    0.001    0.322    0.322

Intercepts:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
    visual            0.000                               0.000    0.000
    textual           0.000                               0.000    0.000
    speed             0.000                               0.000    0.000
   .x1      (int1)    5.016    0.094   53.146    0.000    5.016    4.309
   .x2      (int2)    6.156    0.088   69.594    0.000    6.156    4.944
   .x3      (int3)    2.292    0.098   23.329    0.000    2.292    1.916
   .x4      (int4)    2.780    0.086   32.469    0.000    2.780    2.435
   .x5      (int5)    4.035    0.104   38.981    0.000    4.035    3.189
   .x6      (int6)    1.928    0.074   25.882    0.000    1.928    1.907
   .x7      (int7)    4.244    0.078   54.164    0.000    4.244    3.840
   .x8      (int8)    5.627    0.076   73.915    0.000    5.627    5.726
   .x9      (int9)    5.474    0.073   74.934    0.000    5.474    5.492

Variances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
    visual            1.000                               1.000    1.000
    textual           1.000                               1.000    1.000
    speed             1.000                               1.000    1.000
   .x1                0.591    0.179    3.295    0.001    0.591    0.436
   .x2                1.288    0.160    8.075    0.000    1.288    0.831
   .x3                0.910    0.151    6.046    0.000    0.910    0.636
   .x4                0.454    0.073    6.245    0.000    0.454    0.349
   .x5                0.480    0.076    6.295    0.000    0.480    0.300
   .x6                0.257    0.058    4.401    0.000    0.257    0.252
   .x7                0.901    0.129    7.000    0.000    0.901    0.738
   .x8                0.534    0.091    5.838    0.000    0.534    0.553
   .x9                0.631    0.115    5.469    0.000    0.631    0.635

R-Square:
                   Estimate
    x1                0.564
    x2                0.169
    x3                0.364
    x4                0.651
    x5                0.700
    x6                0.748
    x7                0.262
    x8                0.447
    x9                0.365


Group 2 [Grant-White]:

Latent Variables:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual =~                                                             
    x1      (lmb1)    0.874    0.132    6.630    0.000    0.827    0.711
    x2      (lmb2)    0.512    0.086    5.984    0.000    0.485    0.441
    x3      (lmb3)    0.722    0.095    7.610    0.000    0.684    0.649
  textual =~                                                            
    x4      (lmb4)    0.921    0.071   12.920    0.000    0.921    0.838
    x5      (lmb5)    1.059    0.077   13.763    0.000    1.059    0.869
    x6      (lmb6)    0.875    0.066   13.259    0.000    0.875    0.797
  speed =~                                                              
    x7      (lmb7)    0.566    0.081    6.946    0.000    0.692    0.651
    x8      (lmb8)    0.657    0.082    8.028    0.000    0.803    0.773
    x9      (lmb9)    0.602    0.085    7.049    0.000    0.736    0.722

Covariances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual ~~                                                             
    textual           0.519    0.136    3.829    0.000    0.549    0.549
    speed             0.649    0.215    3.014    0.003    0.561    0.561
  textual ~~                                                            
    speed             0.442    0.189    2.335    0.020    0.361    0.361

Intercepts:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
    visual           -0.193    0.175   -1.101    0.271   -0.204   -0.204
    textual           0.617    0.135    4.587    0.000    0.617    0.617
    speed            -0.294    0.161   -1.828    0.068   -0.240   -0.240
   .x1      (int1)    5.016    0.094   53.146    0.000    5.016    4.312
   .x2      (int2)    6.156    0.088   69.594    0.000    6.156    5.602
   .x3      (int3)    2.292    0.098   23.329    0.000    2.292    2.177
   .x4      (int4)    2.780    0.086   32.469    0.000    2.780    2.529
   .x5      (int5)    4.035    0.104   38.981    0.000    4.035    3.312
   .x6      (int6)    1.928    0.074   25.882    0.000    1.928    1.756
   .x7      (int7)    4.244    0.078   54.164    0.000    4.244    3.990
   .x8      (int8)    5.627    0.076   73.915    0.000    5.627    5.412
   .x9      (int9)    5.474    0.073   74.934    0.000    5.474    5.370

Variances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
    visual            0.896    0.235    3.815    0.000    1.000    1.000
    textual           1.000    0.194    5.157    0.000    1.000    1.000
    speed             1.495    0.360    4.156    0.000    1.000    1.000
   .x1                0.668    0.176    3.793    0.000    0.668    0.494
   .x2                0.973    0.152    6.384    0.000    0.973    0.805
   .x3                0.641    0.133    4.833    0.000    0.641    0.578
   .x4                0.359    0.066    5.464    0.000    0.359    0.297
   .x5                0.363    0.074    4.883    0.000    0.363    0.245
   .x6                0.441    0.075    5.851    0.000    0.441    0.366
   .x7                0.652    0.108    6.016    0.000    0.652    0.577
   .x8                0.436    0.152    2.872    0.004    0.436    0.403
   .x9                0.497    0.127    3.918    0.000    0.497    0.478

R-Square:
                   Estimate
    x1                0.506
    x2                0.195
    x3                0.422
    x4                0.703
    x5                0.755
    x6                0.634
    x7                0.423
    x8                0.597
    x9                0.522</code></pre>
</div>
<div id="measurementInvarianceScalarModelFitCriteria" class="section level4 hasAnchor" number="16.10.6.5">
<h4><span class="header-section-number">16.10.6.5</span> Model Fit<a href="bias.html#measurementInvarianceScalarModelFitCriteria" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>You can specify the null model as the baseline model using: <code>baseline.model = nullModelFit</code></p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1653"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1653-1"><a href="bias.html#cb1653-1" tabindex="-1"></a><span class="fu">fitMeasures</span>(scalarInvarianceModel_fit)</span></code></pre></div>
</details>
<pre><code>                         npar                          fmin 
                       48.000                         0.261 
                        chisq                            df 
                      156.955                        60.000 
                       pvalue                  chisq.scaled 
                        0.000                       159.442 
                    df.scaled                 pvalue.scaled 
                       60.000                         0.000 
         chisq.scaling.factor                baseline.chisq 
                        0.984                       932.665 
                  baseline.df               baseline.pvalue 
                       72.000                         0.000 
        baseline.chisq.scaled            baseline.df.scaled 
                      909.019                        72.000 
       baseline.pvalue.scaled baseline.chisq.scaling.factor 
                        0.000                         1.026 
                          cfi                           tli 
                        0.887                         0.865 
                   cfi.scaled                    tli.scaled 
                        0.881                         0.857 
                   cfi.robust                    tli.robust 
                        0.890                         0.868 
                         nnfi                           rfi 
                        0.865                         0.798 
                          nfi                          pnfi 
                        0.832                         0.693 
                          ifi                           rni 
                        0.889                         0.887 
                  nnfi.scaled                    rfi.scaled 
                        0.857                         0.790 
                   nfi.scaled                   pnfi.scaled 
                        0.825                         0.687 
                   ifi.scaled                    rni.scaled 
                        0.883                         0.881 
                  nnfi.robust                    rni.robust 
                        0.868                         0.890 
                         logl             unrestricted.logl 
                    -3672.597                     -3594.120 
                          aic                           bic 
                     7441.195                      7619.136 
                       ntotal                          bic2 
                      301.000                      7466.908 
            scaling.factor.h1             scaling.factor.h0 
                        1.034                         0.797 
                        rmsea                rmsea.ci.lower 
                        0.104                         0.084 
               rmsea.ci.upper                rmsea.ci.level 
                        0.124                         0.900 
                 rmsea.pvalue                rmsea.close.h0 
                        0.000                         0.050 
        rmsea.notclose.pvalue             rmsea.notclose.h0 
                        0.975                         0.080 
                 rmsea.scaled         rmsea.ci.lower.scaled 
                        0.105                         0.085 
        rmsea.ci.upper.scaled           rmsea.pvalue.scaled 
                        0.125                         0.000 
 rmsea.notclose.pvalue.scaled                  rmsea.robust 
                        0.980                         0.104 
        rmsea.ci.lower.robust         rmsea.ci.upper.robust 
                        0.083                         0.124 
          rmsea.pvalue.robust  rmsea.notclose.pvalue.robust 
                        0.000                         0.972 
                          rmr                    rmr_nomean 
                        0.097                         0.095 
                         srmr                  srmr_bentler 
                        0.080                         0.080 
          srmr_bentler_nomean                          crmr 
                        0.076                         0.084 
                  crmr_nomean                    srmr_mplus 
                        0.081                         0.084 
            srmr_mplus_nomean                         cn_05 
                        0.075                       152.660 
                        cn_01                           gfi 
                      170.490                         0.993 
                         agfi                          pgfi 
                        0.987                         0.552 
                          mfi                          ecvi 
                        0.851                         0.840 </code></pre>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1655"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1655-1"><a href="bias.html#cb1655-1" tabindex="-1"></a>scalarInvarianceModelFitIndices <span class="ot">&lt;-</span> <span class="fu">fitMeasures</span>(</span>
<span id="cb1655-2"><a href="bias.html#cb1655-2" tabindex="-1"></a>  scalarInvarianceModel_fit)[<span class="fu">c</span>(</span>
<span id="cb1655-3"><a href="bias.html#cb1655-3" tabindex="-1"></a>    <span class="st">&quot;cfi.robust&quot;</span>, <span class="st">&quot;rmsea.robust&quot;</span>, <span class="st">&quot;srmr&quot;</span>)]</span>
<span id="cb1655-4"><a href="bias.html#cb1655-4" tabindex="-1"></a></span>
<span id="cb1655-5"><a href="bias.html#cb1655-5" tabindex="-1"></a>scalarInvarianceModel_chisquare <span class="ot">&lt;-</span> <span class="fu">fitMeasures</span>(</span>
<span id="cb1655-6"><a href="bias.html#cb1655-6" tabindex="-1"></a>  scalarInvarianceModel_fit)[<span class="fu">c</span>(<span class="st">&quot;chisq.scaled&quot;</span>)]</span>
<span id="cb1655-7"><a href="bias.html#cb1655-7" tabindex="-1"></a></span>
<span id="cb1655-8"><a href="bias.html#cb1655-8" tabindex="-1"></a>scalarInvarianceModel_chisquareScaling <span class="ot">&lt;-</span> <span class="fu">fitMeasures</span>(</span>
<span id="cb1655-9"><a href="bias.html#cb1655-9" tabindex="-1"></a>  scalarInvarianceModel_fit)[<span class="fu">c</span>(<span class="st">&quot;chisq.scaling.factor&quot;</span>)]</span>
<span id="cb1655-10"><a href="bias.html#cb1655-10" tabindex="-1"></a></span>
<span id="cb1655-11"><a href="bias.html#cb1655-11" tabindex="-1"></a>scalarInvarianceModel_df <span class="ot">&lt;-</span> <span class="fu">fitMeasures</span>(</span>
<span id="cb1655-12"><a href="bias.html#cb1655-12" tabindex="-1"></a>  scalarInvarianceModel_fit)[<span class="fu">c</span>(<span class="st">&quot;df.scaled&quot;</span>)]</span>
<span id="cb1655-13"><a href="bias.html#cb1655-13" tabindex="-1"></a></span>
<span id="cb1655-14"><a href="bias.html#cb1655-14" tabindex="-1"></a>scalarInvarianceModel_N <span class="ot">&lt;-</span> <span class="fu">lavInspect</span>(</span>
<span id="cb1655-15"><a href="bias.html#cb1655-15" tabindex="-1"></a>  scalarInvarianceModel_fit, <span class="at">what =</span> <span class="st">&quot;ntotal&quot;</span>)</span></code></pre></div>
</details>
</div>
<div id="measurementInvarianceScalarModelCompare" class="section level4 hasAnchor" number="16.10.6.6">
<h4><span class="header-section-number">16.10.6.6</span> Compare Model Fit<a href="bias.html#measurementInvarianceScalarModelCompare" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<div id="measurementInvarianceScalarModelNestedModelComparison" class="section level5 hasAnchor" number="16.10.6.6.1">
<h5><span class="header-section-number">16.10.6.6.1</span> Nested Model (<span class="math inline">\(\chi^2\)</span>) Difference Test<a href="bias.html#measurementInvarianceScalarModelNestedModelComparison" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>The metric invariance model and the scalar (“strong factorial”) invariance model are considered “nested” models.
The scalar invariance model is nested within the metric invariance model because the metric invariance model includes all of the terms of the metric invariance model along with additional terms.
Model fit of nested models can be compared with a chi-square difference test, also known as a likelihood ratio test or deviance test.
A significant chi-square difference test would indicate that the simplified model with additional constraints (the scalar invariance model) is significantly worse fitting than the more complex model that has fewer constraints (the metric invariance model).</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1656"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1656-1"><a href="bias.html#cb1656-1" tabindex="-1"></a><span class="fu">anova</span>(metricInvarianceModel_fit, scalarInvarianceModel_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Df"],"name":[1],"type":["int"],"align":["right"]},{"label":["AIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Chisq"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Chisq diff"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Df diff"],"name":[6],"type":["int"],"align":["right"]},{"label":["Pr(>Chisq)"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"54","2":"7412.554","3":"7612.738","4":"116.3133","5":"NA","6":"NA","7":"NA","_rn_":"metricInvarianceModel_fit"},{"1":"60","2":"7441.195","3":"7619.136","4":"156.9546","5":"41.928","6":"6","7":"1.899955e-07","_rn_":"scalarInvarianceModel_fit"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>The scalar invariance model fit significantly worse than the configural invariance model, so scalar invariance did not hold.
This provides evidence of measurement non-invariance of indicator intercepts across groups.
Measurement non-invariance of indicator intercepts poses challenges to being able to meaningfully compare levels on the latent factor across groups <span class="citation">(<a href="#ref-Little2007">Little et al., 2007</a>)</span>.</p>
<p>The <a href="https://github.com/DevPsyLab/petersenlab"><code>petersenlab</code></a> package <span class="citation">(<a href="#ref-R-petersenlab">Petersen, 2024b</a>)</span> contains the <code>satorraBentlerScaledChiSquareDifferenceTestStatistic()</code> function that performs a Satorra-Bentler scaled chi-square difference test:</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1657"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1657-1"><a href="bias.html#cb1657-1" tabindex="-1"></a>scalarInvarianceModel_chisquareDiff <span class="ot">&lt;-</span> </span>
<span id="cb1657-2"><a href="bias.html#cb1657-2" tabindex="-1"></a>  <span class="fu">satorraBentlerScaledChiSquareDifferenceTestStatistic</span>(</span>
<span id="cb1657-3"><a href="bias.html#cb1657-3" tabindex="-1"></a>    <span class="at">T0 =</span> scalarInvarianceModel_chisquare,</span>
<span id="cb1657-4"><a href="bias.html#cb1657-4" tabindex="-1"></a>    <span class="at">c0 =</span> scalarInvarianceModel_chisquareScaling,</span>
<span id="cb1657-5"><a href="bias.html#cb1657-5" tabindex="-1"></a>    <span class="at">d0 =</span> scalarInvarianceModel_df,</span>
<span id="cb1657-6"><a href="bias.html#cb1657-6" tabindex="-1"></a>    <span class="at">T1 =</span> metricInvarianceModel_chisquare,</span>
<span id="cb1657-7"><a href="bias.html#cb1657-7" tabindex="-1"></a>    <span class="at">c1 =</span> metricInvarianceModel_chisquareScaling,</span>
<span id="cb1657-8"><a href="bias.html#cb1657-8" tabindex="-1"></a>    <span class="at">d1 =</span> metricInvarianceModel_df)</span>
<span id="cb1657-9"><a href="bias.html#cb1657-9" tabindex="-1"></a></span>
<span id="cb1657-10"><a href="bias.html#cb1657-10" tabindex="-1"></a>scalarInvarianceModel_chisquareDiff</span></code></pre></div>
</details>
<pre><code>chisq.scaled 
      41.928 </code></pre>
</div>
<div id="measurementInvarianceScalarModelCompareFitCriteria" class="section level5 hasAnchor" number="16.10.6.6.2">
<h5><span class="header-section-number">16.10.6.6.2</span> Compare Other Fit Criteria<a href="bias.html#measurementInvarianceScalarModelCompareFitCriteria" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1659"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1659-1"><a href="bias.html#cb1659-1" tabindex="-1"></a><span class="fu">round</span>(</span>
<span id="cb1659-2"><a href="bias.html#cb1659-2" tabindex="-1"></a>  scalarInvarianceModelFitIndices <span class="sc">-</span> metricInvarianceModelFitIndices,</span>
<span id="cb1659-3"><a href="bias.html#cb1659-3" tabindex="-1"></a>  <span class="at">digits =</span> <span class="dv">3</span>)</span></code></pre></div>
</details>
<pre><code>  cfi.robust rmsea.robust         srmr 
      -0.038        0.015        0.011 </code></pre>
</div>
<div id="measurementInvarianceScalarScoreBasedTest" class="section level5 hasAnchor" number="16.10.6.6.3">
<h5><span class="header-section-number">16.10.6.6.3</span> Score-Based Test<a href="bias.html#measurementInvarianceScalarScoreBasedTest" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Score-based tests of measurement invariance are implemented using the <code>strucchange</code> package and are described by <span class="citation">T. Wang et al. (<a href="#ref-Wang2014a">2014</a>)</span>.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1661"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1661-1"><a href="bias.html#cb1661-1" tabindex="-1"></a><span class="fu">coef</span>(scalarInvarianceModel_fit)</span></code></pre></div>
</details>
<pre><code>  lambda.1_1   lambda.2_1   lambda.3_1   lambda.4_2   lambda.5_2   lambda.6_2 
       0.874        0.512        0.722        0.921        1.059        0.875 
  lambda.7_3   lambda.8_3   lambda.9_3         nu.1         nu.2         nu.3 
       0.566        0.657        0.602        5.016        6.156        2.292 
        nu.4         nu.5         nu.6         nu.7         nu.8         nu.9 
       2.780        4.035        1.928        4.244        5.627        5.474 
theta.1_1.g1 theta.2_2.g1 theta.3_3.g1 theta.4_4.g1 theta.5_5.g1 theta.6_6.g1 
       0.591        1.288        0.910        0.454        0.480        0.257 
theta.7_7.g1 theta.8_8.g1 theta.9_9.g1   psi.2_1.g1   psi.3_1.g1   psi.3_2.g1 
       0.901        0.534        0.631        0.473        0.355        0.322 
  lambda.1_1   lambda.2_1   lambda.3_1   lambda.4_2   lambda.5_2   lambda.6_2 
       0.874        0.512        0.722        0.921        1.059        0.875 
  lambda.7_3   lambda.8_3   lambda.9_3         nu.1         nu.2         nu.3 
       0.566        0.657        0.602        5.016        6.156        2.292 
        nu.4         nu.5         nu.6         nu.7         nu.8         nu.9 
       2.780        4.035        1.928        4.244        5.627        5.474 
theta.1_1.g2 theta.2_2.g2 theta.3_3.g2 theta.4_4.g2 theta.5_5.g2 theta.6_6.g2 
       0.668        0.973        0.641        0.359        0.363        0.441 
theta.7_7.g2 theta.8_8.g2 theta.9_9.g2   alpha.1.g2   alpha.2.g2   alpha.3.g2 
       0.652        0.436        0.497       -0.193        0.617       -0.294 
  psi.1_1.g2   psi.2_2.g2   psi.3_3.g2   psi.2_1.g2   psi.3_1.g2   psi.3_2.g2 
       0.896        1.000        1.495        0.519        0.649        0.442 </code></pre>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1663"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1663-1"><a href="bias.html#cb1663-1" tabindex="-1"></a><span class="fu">coef</span>(scalarInvarianceModel_fit)[<span class="dv">10</span><span class="sc">:</span><span class="dv">18</span>]</span></code></pre></div>
</details>
<pre><code>    nu.1     nu.2     nu.3     nu.4     nu.5     nu.6     nu.7     nu.8 
5.015537 6.155701 2.292365 2.779781 4.034564 1.928163 4.243593 5.626570 
    nu.9 
5.474024 </code></pre>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1665"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1665-1"><a href="bias.html#cb1665-1" tabindex="-1"></a>strucchange<span class="sc">::</span><span class="fu">sctest</span>(</span>
<span id="cb1665-2"><a href="bias.html#cb1665-2" tabindex="-1"></a>  scalarInvarianceModel_fit,</span>
<span id="cb1665-3"><a href="bias.html#cb1665-3" tabindex="-1"></a>  <span class="at">order.by =</span> HolzingerSwineford1939<span class="sc">$</span>school,</span>
<span id="cb1665-4"><a href="bias.html#cb1665-4" tabindex="-1"></a>  <span class="at">parm =</span> <span class="dv">10</span><span class="sc">:</span><span class="dv">18</span>,</span>
<span id="cb1665-5"><a href="bias.html#cb1665-5" tabindex="-1"></a>  <span class="at">functional =</span> <span class="st">&quot;LMuo&quot;</span>)</span></code></pre></div>
</details>
<pre><code>
    M-fluctuation test

data:  scalarInvarianceModel_fit
f(efp) = 44.639, p-value = 1.076e-06</code></pre>
<p>A score-based test and expected parameter change (EPC) estimates <span class="citation">(<a href="#ref-Oberski2014">Oberski, 2014</a>; <a href="#ref-Oberski2015">Oberski et al., 2015</a>)</span> are provided by the <code>lavaan</code> package <span class="citation">(<a href="#ref-R-lavaan">Rosseel et al., 2022</a>)</span>.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1667"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1667-1"><a href="bias.html#cb1667-1" tabindex="-1"></a><span class="fu">lavTestScore</span>(</span>
<span id="cb1667-2"><a href="bias.html#cb1667-2" tabindex="-1"></a>  scalarInvarianceModel_fit,</span>
<span id="cb1667-3"><a href="bias.html#cb1667-3" tabindex="-1"></a>  <span class="at">epc =</span> <span class="cn">TRUE</span></span>
<span id="cb1667-4"><a href="bias.html#cb1667-4" tabindex="-1"></a>)</span></code></pre></div>
</details>
<pre><code>$test

total score test:

   test     X2 df p.value
1 score 46.446 18       0

$uni

univariate score tests:

     lhs op   rhs     X2 df p.value
1   .p1. == .p37.  2.216  1   0.137
2   .p2. == .p38.  0.115  1   0.734
3   .p3. == .p39.  1.483  1   0.223
4   .p4. == .p40.  0.039  1   0.843
5   .p5. == .p41.  2.724  1   0.099
6   .p6. == .p42.  2.205  1   0.138
7   .p7. == .p43.  0.088  1   0.767
8   .p8. == .p44.  0.035  1   0.851
9   .p9. == .p45.  0.010  1   0.922
10 .p10. == .p46.  6.336  1   0.012
11 .p11. == .p47.  7.421  1   0.006
12 .p12. == .p48. 20.450  1   0.000
13 .p13. == .p49.  1.216  1   0.270
14 .p14. == .p50.  1.146  1   0.284
15 .p15. == .p51.  0.001  1   0.976
16 .p16. == .p52. 15.301  1   0.000
17 .p17. == .p53.  4.102  1   0.043
18 .p18. == .p54.  1.602  1   0.206

$epc

expected parameter changes (epc) and expected parameter values (epv):

       lhs op     rhs block group free        label plabel    est    epc    epv
1   visual =~      x1     1     1    1   lambda.1_1   .p1.  0.874  0.100  0.974
2   visual =~      x2     1     1    2   lambda.2_1   .p2.  0.512 -0.038  0.474
3   visual =~      x3     1     1    3   lambda.3_1   .p3.  0.722 -0.098  0.624
4  textual =~      x4     1     1    4   lambda.4_2   .p4.  0.921 -0.006  0.915
5  textual =~      x5     1     1    5   lambda.5_2   .p5.  1.059  0.067  1.126
6  textual =~      x6     1     1    6   lambda.6_2   .p6.  0.875 -0.042  0.833
7    speed =~      x7     1     1    7   lambda.7_3   .p7.  0.566 -0.024  0.542
8    speed =~      x8     1     1    8   lambda.8_3   .p8.  0.657  0.012  0.669
9    speed =~      x9     1     1    9   lambda.9_3   .p9.  0.602  0.004  0.606
10      x1 ~1             1     1   10         nu.1  .p10.  5.016 -0.068  4.948
11      x2 ~1             1     1   11         nu.2  .p11.  6.156 -0.172  5.984
12      x3 ~1             1     1   12         nu.3  .p12.  2.292  0.217  2.510
13      x4 ~1             1     1   13         nu.4  .p13.  2.780  0.034  2.814
14      x5 ~1             1     1   14         nu.5  .p14.  4.035 -0.032  4.002
15      x6 ~1             1     1   15         nu.6  .p15.  1.928  0.001  1.929
16      x7 ~1             1     1   16         nu.7  .p16.  4.244  0.195  4.439
17      x8 ~1             1     1   17         nu.8  .p17.  5.627 -0.063  5.563
18      x9 ~1             1     1   18         nu.9  .p18.  5.474 -0.048  5.426
19      x1 ~~      x1     1     1   19 theta.1_1.g1  .p19.  0.591 -0.132  0.459
20      x2 ~~      x2     1     1   20 theta.2_2.g1  .p20.  1.288  0.022  1.310
21      x3 ~~      x3     1     1   21 theta.3_3.g1  .p21.  0.910  0.086  0.997
22      x4 ~~      x4     1     1   22 theta.4_4.g1  .p22.  0.454  0.000  0.454
23      x5 ~~      x5     1     1   23 theta.5_5.g1  .p23.  0.480 -0.042  0.437
24      x6 ~~      x6     1     1   24 theta.6_6.g1  .p24.  0.257  0.026  0.283
25      x7 ~~      x7     1     1   25 theta.7_7.g1  .p25.  0.901  0.013  0.914
26      x8 ~~      x8     1     1   26 theta.8_8.g1  .p26.  0.534 -0.010  0.524
27      x9 ~~      x9     1     1   27 theta.9_9.g1  .p27.  0.631 -0.002  0.629
28  visual ~1             1     1    0   alpha.1.g1  .p28.  0.000     NA     NA
29 textual ~1             1     1    0   alpha.2.g1  .p29.  0.000     NA     NA
30   speed ~1             1     1    0   alpha.3.g1  .p30.  0.000     NA     NA
31  visual ~~  visual     1     1    0   psi.1_1.g1  .p31.  1.000     NA     NA
32 textual ~~ textual     1     1    0   psi.2_2.g1  .p32.  1.000     NA     NA
33   speed ~~   speed     1     1    0   psi.3_3.g1  .p33.  1.000     NA     NA
34  visual ~~ textual     1     1   28   psi.2_1.g1  .p34.  0.473 -0.013  0.460
35  visual ~~   speed     1     1   29   psi.3_1.g1  .p35.  0.355 -0.010  0.345
36 textual ~~   speed     1     1   30   psi.3_2.g1  .p36.  0.322  0.000  0.322
37  visual =~      x1     2     2   31   lambda.1_1  .p37.  0.874 -0.035  0.839
38  visual =~      x2     2     2   32   lambda.2_1  .p38.  0.512  0.068  0.580
39  visual =~      x3     2     2   33   lambda.3_1  .p39.  0.722 -0.002  0.721
40 textual =~      x4     2     2   34   lambda.4_2  .p40.  0.921  0.038  0.959
41 textual =~      x5     2     2   35   lambda.5_2  .p41.  1.059 -0.086  0.973
42 textual =~      x6     2     2   36   lambda.6_2  .p42.  0.875  0.067  0.942
43   speed =~      x7     2     2   37   lambda.7_3  .p43.  0.566 -0.036  0.530
44   speed =~      x8     2     2   38   lambda.8_3  .p44.  0.657  0.017  0.674
45   speed =~      x9     2     2   39   lambda.9_3  .p45.  0.602  0.009  0.611
46      x1 ~1             2     2   40         nu.1  .p46.  5.016  0.066  5.081
47      x2 ~1             2     2   41         nu.2  .p47.  6.156  0.148  6.304
48      x3 ~1             2     2   42         nu.3  .p48.  2.292 -0.169  2.123
49      x4 ~1             2     2   43         nu.4  .p49.  2.780 -0.053  2.727
50      x5 ~1             2     2   44         nu.5  .p50.  4.035  0.080  4.115
51      x6 ~1             2     2   45         nu.6  .p51.  1.928 -0.042  1.886
52      x7 ~1             2     2   46         nu.7  .p52.  4.244 -0.158  4.086
53      x8 ~1             2     2   47         nu.8  .p53.  5.627  0.068  5.695
54      x9 ~1             2     2   48         nu.9  .p54.  5.474  0.051  5.525
55      x1 ~~      x1     2     2   49 theta.1_1.g2  .p55.  0.668  0.029  0.697
56      x2 ~~      x2     2     2   50 theta.2_2.g2  .p56.  0.973 -0.021  0.952
57      x3 ~~      x3     2     2   51 theta.3_3.g2  .p57.  0.641 -0.002  0.639
58      x4 ~~      x4     2     2   52 theta.4_4.g2  .p58.  0.359 -0.026  0.333
59      x5 ~~      x5     2     2   53 theta.5_5.g2  .p59.  0.363  0.064  0.427
60      x6 ~~      x6     2     2   54 theta.6_6.g2  .p60.  0.441 -0.029  0.411
61      x7 ~~      x7     2     2   55 theta.7_7.g2  .p61.  0.652  0.021  0.674
62      x8 ~~      x8     2     2   56 theta.8_8.g2  .p62.  0.436 -0.015  0.420
63      x9 ~~      x9     2     2   57 theta.9_9.g2  .p63.  0.497 -0.004  0.493
64  visual ~1             2     2   58   alpha.1.g2  .p64. -0.193  0.011 -0.182
65 textual ~1             2     2   59   alpha.2.g2  .p65.  0.617 -0.001  0.617
66   speed ~1             2     2   60   alpha.3.g2  .p66. -0.294 -0.013 -0.307
67  visual ~~  visual     2     2   61   psi.1_1.g2  .p67.  0.896  0.001  0.897
68 textual ~~ textual     2     2   62   psi.2_2.g2  .p68.  1.000  0.001  1.000
69   speed ~~   speed     2     2   63   psi.3_3.g2  .p69.  1.495 -0.003  1.492
70  visual ~~ textual     2     2   64   psi.2_1.g2  .p70.  0.519  0.004  0.524
71  visual ~~   speed     2     2   65   psi.3_1.g2  .p71.  0.649  0.002  0.650
72 textual ~~   speed     2     2   66   psi.3_2.g2  .p72.  0.442  0.000  0.442
   sepc.lv sepc.all sepc.nox
1    0.100    0.086    0.086
2   -0.038   -0.031   -0.031
3   -0.098   -0.082   -0.082
4   -0.006   -0.005   -0.005
5    0.067    0.053    0.053
6   -0.042   -0.042   -0.042
7   -0.024   -0.021   -0.021
8    0.012    0.012    0.012
9    0.004    0.004    0.004
10  -0.068   -0.058   -0.058
11  -0.172   -0.138   -0.138
12   0.217    0.182    0.182
13   0.034    0.030    0.030
14  -0.032   -0.026   -0.026
15   0.001    0.001    0.001
16   0.195    0.177    0.177
17  -0.063   -0.064   -0.064
18  -0.048   -0.048   -0.048
19  -0.591   -0.436   -0.436
20   1.288    0.831    0.831
21   0.910    0.636    0.636
22   0.454    0.349    0.349
23  -0.480   -0.300   -0.300
24   0.257    0.252    0.252
25   0.901    0.738    0.738
26  -0.534   -0.553   -0.553
27  -0.631   -0.635   -0.635
28      NA       NA       NA
29      NA       NA       NA
30      NA       NA       NA
31      NA       NA       NA
32      NA       NA       NA
33      NA       NA       NA
34  -0.013   -0.013   -0.013
35  -0.010   -0.010   -0.010
36   0.000    0.000    0.000
37  -0.033   -0.028   -0.028
38   0.064    0.059    0.059
39  -0.002   -0.001   -0.001
40   0.038    0.034    0.034
41  -0.086   -0.070   -0.070
42   0.067    0.061    0.061
43  -0.044   -0.041   -0.041
44   0.021    0.020    0.020
45   0.011    0.011    0.011
46   0.066    0.057    0.057
47   0.148    0.135    0.135
48  -0.169   -0.161   -0.161
49  -0.053   -0.048   -0.048
50   0.080    0.066    0.066
51  -0.042   -0.038   -0.038
52  -0.158   -0.148   -0.148
53   0.068    0.066    0.066
54   0.051    0.050    0.050
55   0.668    0.494    0.494
56  -0.973   -0.805   -0.805
57  -0.641   -0.578   -0.578
58  -0.359   -0.297   -0.297
59   0.363    0.245    0.245
60  -0.441   -0.366   -0.366
61   0.652    0.577    0.577
62  -0.436   -0.403   -0.403
63  -0.497   -0.478   -0.478
64   0.012    0.012    0.012
65  -0.001   -0.001   -0.001
66  -0.011   -0.011   -0.011
67   1.000    1.000    1.000
68   1.000    1.000    1.000
69  -1.000   -1.000   -1.000
70   0.004    0.004    0.004
71   0.001    0.001    0.001
72   0.000    0.000    0.000</code></pre>
</div>
<div id="measurementInvarianceScalarModelEquivalenceTest" class="section level5 hasAnchor" number="16.10.6.6.4">
<h5><span class="header-section-number">16.10.6.6.4</span> Equivalence Test<a href="bias.html#measurementInvarianceScalarModelEquivalenceTest" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>The <a href="https://github.com/DevPsyLab/petersenlab"><code>petersenlab</code></a> package <span class="citation">(<a href="#ref-R-petersenlab">Petersen, 2024b</a>)</span> contains the <code>equiv_chi()</code> function from <span class="citation">Counsell et al. (<a href="#ref-Counsell2020">2020</a>)</span> that performs an equivalence test: <a href="https://osf.io/cqu8v" class="uri">https://osf.io/cqu8v</a>.
The chi-square equivalence test is non-significant, suggesting that the model fit is not acceptable.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1669"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1669-1"><a href="bias.html#cb1669-1" tabindex="-1"></a><span class="fu">equiv_chi</span>(</span>
<span id="cb1669-2"><a href="bias.html#cb1669-2" tabindex="-1"></a>  <span class="at">alpha =</span> .<span class="dv">05</span>,</span>
<span id="cb1669-3"><a href="bias.html#cb1669-3" tabindex="-1"></a>  <span class="at">chi =</span> scalarInvarianceModel_chisquare,</span>
<span id="cb1669-4"><a href="bias.html#cb1669-4" tabindex="-1"></a>  <span class="at">df =</span> scalarInvarianceModel_df,</span>
<span id="cb1669-5"><a href="bias.html#cb1669-5" tabindex="-1"></a>  <span class="at">m =</span> <span class="dv">2</span>,</span>
<span id="cb1669-6"><a href="bias.html#cb1669-6" tabindex="-1"></a>  <span class="at">N_sample =</span> scalarInvarianceModel_N,</span>
<span id="cb1669-7"><a href="bias.html#cb1669-7" tabindex="-1"></a>  <span class="at">popRMSEA =</span> .<span class="dv">08</span>)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["chi"],"name":[1],"type":["dbl"],"align":["right"]},{"label":["Fml"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["popep"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["popdelt"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["pval"],"name":[5],"type":["dbl"],"align":["right"]}],"data":[{"1":"159.4416","2":"0.5332494","3":"0.192","4":"57.408","5":"0.9817801","_rn_":"chisq.scaled"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>Moreover, the equivalence test of the chi-square difference test is non-significant, suggesting that the degree of worsening of model fit is not acceptable.
In other words, scalar invariance failed.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1670"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1670-1"><a href="bias.html#cb1670-1" tabindex="-1"></a>scalarInvarianceModel_dfDiff <span class="ot">&lt;-</span> </span>
<span id="cb1670-2"><a href="bias.html#cb1670-2" tabindex="-1"></a>  scalarInvarianceModel_df <span class="sc">-</span> metricInvarianceModel_df</span>
<span id="cb1670-3"><a href="bias.html#cb1670-3" tabindex="-1"></a></span>
<span id="cb1670-4"><a href="bias.html#cb1670-4" tabindex="-1"></a><span class="fu">equiv_chi</span>(</span>
<span id="cb1670-5"><a href="bias.html#cb1670-5" tabindex="-1"></a>  <span class="at">alpha =</span> .<span class="dv">05</span>,</span>
<span id="cb1670-6"><a href="bias.html#cb1670-6" tabindex="-1"></a>  <span class="at">chi =</span> scalarInvarianceModel_chisquareDiff,</span>
<span id="cb1670-7"><a href="bias.html#cb1670-7" tabindex="-1"></a>  <span class="at">df =</span> scalarInvarianceModel_dfDiff,</span>
<span id="cb1670-8"><a href="bias.html#cb1670-8" tabindex="-1"></a>  <span class="at">m =</span> <span class="dv">2</span>,</span>
<span id="cb1670-9"><a href="bias.html#cb1670-9" tabindex="-1"></a>  <span class="at">N_sample =</span> scalarInvarianceModel_N,</span>
<span id="cb1670-10"><a href="bias.html#cb1670-10" tabindex="-1"></a>  <span class="at">popRMSEA =</span> .<span class="dv">08</span>)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["chi"],"name":[1],"type":["dbl"],"align":["right"]},{"label":["Fml"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["popep"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["popdelt"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["pval"],"name":[5],"type":["dbl"],"align":["right"]}],"data":[{"1":"41.928","2":"0.1402274","3":"0.0192","4":"5.7408","5":"0.9997378","_rn_":"chisq.scaled"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceScalarModelPermutationTest" class="section level5 hasAnchor" number="16.10.6.6.5">
<h5><span class="header-section-number">16.10.6.6.5</span> Permutation Test<a href="bias.html#measurementInvarianceScalarModelPermutationTest" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Permutation procedures for testing measurement invariance are described in <span class="citation">(<a href="#ref-Jorgensen2018">Jorgensen et al., 2018</a>)</span>.
For reproducibility, I set the seed below.
Using the same seed will yield the same answer every time.
There is nothing special about this particular seed.
You can specify the null model as the baseline model using: <code>baseline.model = nullModelFit</code></p>
<p><strong>Warning</strong>: this code takes a while to run based on 100 iterations.
You can reduce the number of iterations to be faster.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1671"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1671-1"><a href="bias.html#cb1671-1" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">52242</span>)</span>
<span id="cb1671-2"><a href="bias.html#cb1671-2" tabindex="-1"></a>scalarInvarianceTest <span class="ot">&lt;-</span> <span class="fu">permuteMeasEq</span>(</span>
<span id="cb1671-3"><a href="bias.html#cb1671-3" tabindex="-1"></a>  <span class="at">nPermute =</span> numPermutations,</span>
<span id="cb1671-4"><a href="bias.html#cb1671-4" tabindex="-1"></a>  <span class="at">modelType =</span> <span class="st">&quot;mgcfa&quot;</span>,</span>
<span id="cb1671-5"><a href="bias.html#cb1671-5" tabindex="-1"></a>  <span class="at">con =</span> scalarInvarianceModel_fit,</span>
<span id="cb1671-6"><a href="bias.html#cb1671-6" tabindex="-1"></a>  <span class="at">uncon =</span> metricInvarianceModel_fit,</span>
<span id="cb1671-7"><a href="bias.html#cb1671-7" tabindex="-1"></a>  <span class="at">AFIs =</span> myAFIs,</span>
<span id="cb1671-8"><a href="bias.html#cb1671-8" tabindex="-1"></a>  <span class="at">moreAFIs =</span> moreAFIs,</span>
<span id="cb1671-9"><a href="bias.html#cb1671-9" tabindex="-1"></a>  <span class="at">parallelType =</span> <span class="st">&quot;multicore&quot;</span>, <span class="co">#only &#39;snow&#39; works on Windows, but right now, it is throwing an error</span></span>
<span id="cb1671-10"><a href="bias.html#cb1671-10" tabindex="-1"></a>  <span class="at">iseed =</span> <span class="dv">52242</span>)</span></code></pre></div>
</details>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1672"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1672-1"><a href="bias.html#cb1672-1" tabindex="-1"></a><span class="fu">RNGkind</span>(<span class="st">&quot;default&quot;</span>, <span class="st">&quot;default&quot;</span>, <span class="st">&quot;default&quot;</span>)</span>
<span id="cb1672-2"><a href="bias.html#cb1672-2" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">52242</span>)</span>
<span id="cb1672-3"><a href="bias.html#cb1672-3" tabindex="-1"></a></span>
<span id="cb1672-4"><a href="bias.html#cb1672-4" tabindex="-1"></a>scalarInvarianceTest</span></code></pre></div>
</details>
<pre><code>Omnibus p value based on parametric chi-squared difference test:

Chisq diff    Df diff Pr(&gt;Chisq) 
    41.928      6.000      0.000 


Omnibus p values based on nonparametric permutation method: 

             AFI.Difference p.value
chisq                40.641       0
chisq.scaled         41.486       0
rmsea                 0.016       0
cfi                  -0.040       0
tli                  -0.039       0
srmr                  0.011       0
rmsea.robust          0.015       0
cfi.robust           -0.038       0
tli.robust           -0.036       0</code></pre>
<p>The <em>p</em>-values are significant, indicating that the model fit significantly worse than the metric invariance model.
In other words, scalar invariance failed.</p>
</div>
</div>
<div id="measurementInvarianceScalarModelReliability" class="section level4 hasAnchor" number="16.10.6.7">
<h4><span class="header-section-number">16.10.6.7</span> Internal Consistency Reliability<a href="bias.html#measurementInvarianceScalarModelReliability" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><a href="reliability.html#internalConsistency-reliability">Internal consistency reliability</a> of items composing the latent factors, as quantified by <a href="reliability.html#coefficientOmega">omega (<span class="math inline">\(\omega\)</span>)</a> and <a href="reliability.html#averageVarianceExtracted">average variance extracted</a> (AVE), was estimated using the <code>semTools</code> package <span class="citation">(<a href="#ref-R-semTools">Jorgensen et al., 2021</a>)</span>.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1674"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1674-1"><a href="bias.html#cb1674-1" tabindex="-1"></a><span class="fu">compRelSEM</span>(scalarInvarianceModel_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["group"],"name":[1],"type":["chr"],"align":["left"]},{"label":["visual"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["textual"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["speed"],"name":[4],"type":["dbl"],"align":["right"]}],"data":[{"1":"Pasteur","2":"0.6171126","3":"0.8583892","4":"0.6220281"},{"1":"Grant-White","2":"0.6162424","3":"0.8634281","4":"0.7663053"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1675"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1675-1"><a href="bias.html#cb1675-1" tabindex="-1"></a><span class="fu">AVE</span>(scalarInvarianceModel_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["group"],"name":[1],"type":["chr"],"align":["left"]},{"label":["visual"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["textual"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["speed"],"name":[4],"type":["dbl"],"align":["right"]}],"data":[{"1":"Pasteur","2":"0.3659807","3":"0.6874279","4":"0.3559319"},{"1":"Grant-White","2":"0.3821957","3":"0.6991725","4":"0.5187171"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceScalarModelPathDiagram" class="section level4 hasAnchor" number="16.10.6.8">
<h4><span class="header-section-number">16.10.6.8</span> Path Diagram<a href="bias.html#measurementInvarianceScalarModelPathDiagram" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>A path diagram of the model generated using the <code>semPlot</code> package <span class="citation">(<a href="#ref-R-semPlot">Epskamp, 2022</a>)</span> is below.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1676"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1676-1"><a href="bias.html#cb1676-1" tabindex="-1"></a><span class="fu">semPaths</span>(</span>
<span id="cb1676-2"><a href="bias.html#cb1676-2" tabindex="-1"></a>  scalarInvarianceModel_fit,</span>
<span id="cb1676-3"><a href="bias.html#cb1676-3" tabindex="-1"></a>  <span class="at">what =</span> <span class="st">&quot;est&quot;</span>,</span>
<span id="cb1676-4"><a href="bias.html#cb1676-4" tabindex="-1"></a>  <span class="at">layout =</span> <span class="st">&quot;tree2&quot;</span>,</span>
<span id="cb1676-5"><a href="bias.html#cb1676-5" tabindex="-1"></a>  <span class="at">ask =</span> <span class="cn">FALSE</span>,</span>
<span id="cb1676-6"><a href="bias.html#cb1676-6" tabindex="-1"></a>  <span class="at">edge.label.cex =</span> <span class="fl">1.2</span>)</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:measurementInvarianceScalarModelPathDiagram-1"></span>
<img src="Book_files/figure-html/measurementInvarianceScalarModelPathDiagram-1.png" alt="Scalar Invariance Model in Confirmatory Factor Analysis." width="100%" />
<p class="caption">
Figure 16.57: Scalar Invariance Model in Confirmatory Factor Analysis.
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:measurementInvarianceScalarModelPathDiagram-2"></span>
<img src="Book_files/figure-html/measurementInvarianceScalarModelPathDiagram-2.png" alt="Scalar Invariance Model in Confirmatory Factor Analysis." width="100%" />
<p class="caption">
Figure 16.58: Scalar Invariance Model in Confirmatory Factor Analysis.
</p>
</div>
</div>
</div>
<div id="measurementInvarianceResidualModel" class="section level3 hasAnchor" number="16.10.7">
<h3><span class="header-section-number">16.10.7</span> Residual (“Strict Factorial”) Invariance Model<a href="bias.html#measurementInvarianceResidualModel" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Specify invariance of factor loadings, intercepts, and residuals across groups.</p>
<div id="measurementInvarianceResidualModelSyntax" class="section level4 hasAnchor" number="16.10.7.1">
<h4><span class="header-section-number">16.10.7.1</span> Model Syntax<a href="bias.html#measurementInvarianceResidualModelSyntax" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1677"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1677-1"><a href="bias.html#cb1677-1" tabindex="-1"></a>residualInvarianceModel <span class="ot">&lt;-</span> <span class="fu">measEq.syntax</span>(</span>
<span id="cb1677-2"><a href="bias.html#cb1677-2" tabindex="-1"></a>  <span class="at">configural.model =</span> cfaModel,</span>
<span id="cb1677-3"><a href="bias.html#cb1677-3" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1677-4"><a href="bias.html#cb1677-4" tabindex="-1"></a>  <span class="at">ID.fac =</span> <span class="st">&quot;std.lv&quot;</span>,</span>
<span id="cb1677-5"><a href="bias.html#cb1677-5" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1677-6"><a href="bias.html#cb1677-6" tabindex="-1"></a>  <span class="at">group.equal =</span> <span class="fu">c</span>(<span class="st">&quot;loadings&quot;</span>,<span class="st">&quot;intercepts&quot;</span>,<span class="st">&quot;residuals&quot;</span>))</span></code></pre></div>
</details>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1678"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1678-1"><a href="bias.html#cb1678-1" tabindex="-1"></a><span class="fu">cat</span>(<span class="fu">as.character</span>(residualInvarianceModel))</span></code></pre></div>
</details>
<pre><code>## LOADINGS:

visual =~ c(NA, NA)*x1 + c(lambda.1_1, lambda.1_1)*x1
visual =~ c(NA, NA)*x2 + c(lambda.2_1, lambda.2_1)*x2
visual =~ c(NA, NA)*x3 + c(lambda.3_1, lambda.3_1)*x3
textual =~ c(NA, NA)*x4 + c(lambda.4_2, lambda.4_2)*x4
textual =~ c(NA, NA)*x5 + c(lambda.5_2, lambda.5_2)*x5
textual =~ c(NA, NA)*x6 + c(lambda.6_2, lambda.6_2)*x6
speed =~ c(NA, NA)*x7 + c(lambda.7_3, lambda.7_3)*x7
speed =~ c(NA, NA)*x8 + c(lambda.8_3, lambda.8_3)*x8
speed =~ c(NA, NA)*x9 + c(lambda.9_3, lambda.9_3)*x9

## INTERCEPTS:

x1 ~ c(NA, NA)*1 + c(nu.1, nu.1)*1
x2 ~ c(NA, NA)*1 + c(nu.2, nu.2)*1
x3 ~ c(NA, NA)*1 + c(nu.3, nu.3)*1
x4 ~ c(NA, NA)*1 + c(nu.4, nu.4)*1
x5 ~ c(NA, NA)*1 + c(nu.5, nu.5)*1
x6 ~ c(NA, NA)*1 + c(nu.6, nu.6)*1
x7 ~ c(NA, NA)*1 + c(nu.7, nu.7)*1
x8 ~ c(NA, NA)*1 + c(nu.8, nu.8)*1
x9 ~ c(NA, NA)*1 + c(nu.9, nu.9)*1

## UNIQUE-FACTOR VARIANCES:

x1 ~~ c(NA, NA)*x1 + c(theta.1_1, theta.1_1)*x1
x2 ~~ c(NA, NA)*x2 + c(theta.2_2, theta.2_2)*x2
x3 ~~ c(NA, NA)*x3 + c(theta.3_3, theta.3_3)*x3
x4 ~~ c(NA, NA)*x4 + c(theta.4_4, theta.4_4)*x4
x5 ~~ c(NA, NA)*x5 + c(theta.5_5, theta.5_5)*x5
x6 ~~ c(NA, NA)*x6 + c(theta.6_6, theta.6_6)*x6
x7 ~~ c(NA, NA)*x7 + c(theta.7_7, theta.7_7)*x7
x8 ~~ c(NA, NA)*x8 + c(theta.8_8, theta.8_8)*x8
x9 ~~ c(NA, NA)*x9 + c(theta.9_9, theta.9_9)*x9

## LATENT MEANS/INTERCEPTS:

visual ~ c(0, NA)*1 + c(alpha.1.g1, alpha.1.g2)*1
textual ~ c(0, NA)*1 + c(alpha.2.g1, alpha.2.g2)*1
speed ~ c(0, NA)*1 + c(alpha.3.g1, alpha.3.g2)*1

## COMMON-FACTOR VARIANCES:

visual ~~ c(1, NA)*visual + c(psi.1_1.g1, psi.1_1.g2)*visual
textual ~~ c(1, NA)*textual + c(psi.2_2.g1, psi.2_2.g2)*textual
speed ~~ c(1, NA)*speed + c(psi.3_3.g1, psi.3_3.g2)*speed

## COMMON-FACTOR COVARIANCES:

visual ~~ c(NA, NA)*textual + c(psi.2_1.g1, psi.2_1.g2)*textual
visual ~~ c(NA, NA)*speed + c(psi.3_1.g1, psi.3_1.g2)*speed
textual ~~ c(NA, NA)*speed + c(psi.3_2.g1, psi.3_2.g2)*speed</code></pre>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1680"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1680-1"><a href="bias.html#cb1680-1" tabindex="-1"></a>residualInvarianceSyntax <span class="ot">&lt;-</span> <span class="fu">as.character</span>(residualInvarianceModel)</span></code></pre></div>
</details>
<div id="measurementInvarianceResidualModelFeatures" class="section level5 hasAnchor" number="16.10.7.1.1">
<h5><span class="header-section-number">16.10.7.1.1</span> Summary of Model Features<a href="bias.html#measurementInvarianceResidualModelFeatures" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1681"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1681-1"><a href="bias.html#cb1681-1" tabindex="-1"></a><span class="fu">summary</span>(residualInvarianceModel)</span></code></pre></div>
</details>
<pre><code>This lavaan model syntax specifies a CFA with 9 manifest indicators of 3 common factor(s).

To identify the location and scale of each common factor, the factor means and variances were fixed to 0 and 1, respectively, unless equality constraints on measurement parameters allow them to be freed.

Pattern matrix indicating num(eric), ord(ered), and lat(ent) indicators per factor:

   visual textual speed
x1 num                 
x2 num                 
x3 num                 
x4        num          
x5        num          
x6        num          
x7                num  
x8                num  
x9                num  

The following types of parameter were constrained to equality across groups:

    loadings
    intercepts
    residuals</code></pre>
</div>
</div>
<div id="measurementInvarianceResidualModelTabular" class="section level4 hasAnchor" number="16.10.7.2">
<h4><span class="header-section-number">16.10.7.2</span> Model Syntax in Table Form<a href="bias.html#measurementInvarianceResidualModelTabular" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1683"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1683-1"><a href="bias.html#cb1683-1" tabindex="-1"></a><span class="fu">lavaanify</span>(residualInvarianceModel, <span class="at">ngroups =</span> <span class="dv">2</span>)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["id"],"name":[1],"type":["int"],"align":["right"]},{"label":["lhs"],"name":[2],"type":["chr"],"align":["left"]},{"label":["op"],"name":[3],"type":["chr"],"align":["left"]},{"label":["rhs"],"name":[4],"type":["chr"],"align":["left"]},{"label":["user"],"name":[5],"type":["int"],"align":["right"]},{"label":["block"],"name":[6],"type":["int"],"align":["right"]},{"label":["group"],"name":[7],"type":["int"],"align":["right"]},{"label":["free"],"name":[8],"type":["int"],"align":["right"]},{"label":["ustart"],"name":[9],"type":["dbl"],"align":["right"]},{"label":["exo"],"name":[10],"type":["int"],"align":["right"]},{"label":["label"],"name":[11],"type":["chr"],"align":["left"]},{"label":["plabel"],"name":[12],"type":["chr"],"align":["left"]}],"data":[{"1":"1","2":"visual","3":"=~","4":"x1","5":"1","6":"1","7":"1","8":"1","9":"NA","10":"0","11":"lambda.1_1","12":".p1."},{"1":"2","2":"visual","3":"=~","4":"x2","5":"1","6":"1","7":"1","8":"2","9":"NA","10":"0","11":"lambda.2_1","12":".p2."},{"1":"3","2":"visual","3":"=~","4":"x3","5":"1","6":"1","7":"1","8":"3","9":"NA","10":"0","11":"lambda.3_1","12":".p3."},{"1":"4","2":"textual","3":"=~","4":"x4","5":"1","6":"1","7":"1","8":"4","9":"NA","10":"0","11":"lambda.4_2","12":".p4."},{"1":"5","2":"textual","3":"=~","4":"x5","5":"1","6":"1","7":"1","8":"5","9":"NA","10":"0","11":"lambda.5_2","12":".p5."},{"1":"6","2":"textual","3":"=~","4":"x6","5":"1","6":"1","7":"1","8":"6","9":"NA","10":"0","11":"lambda.6_2","12":".p6."},{"1":"7","2":"speed","3":"=~","4":"x7","5":"1","6":"1","7":"1","8":"7","9":"NA","10":"0","11":"lambda.7_3","12":".p7."},{"1":"8","2":"speed","3":"=~","4":"x8","5":"1","6":"1","7":"1","8":"8","9":"NA","10":"0","11":"lambda.8_3","12":".p8."},{"1":"9","2":"speed","3":"=~","4":"x9","5":"1","6":"1","7":"1","8":"9","9":"NA","10":"0","11":"lambda.9_3","12":".p9."},{"1":"10","2":"x1","3":"~1","4":"","5":"1","6":"1","7":"1","8":"10","9":"NA","10":"0","11":"nu.1","12":".p10."},{"1":"11","2":"x2","3":"~1","4":"","5":"1","6":"1","7":"1","8":"11","9":"NA","10":"0","11":"nu.2","12":".p11."},{"1":"12","2":"x3","3":"~1","4":"","5":"1","6":"1","7":"1","8":"12","9":"NA","10":"0","11":"nu.3","12":".p12."},{"1":"13","2":"x4","3":"~1","4":"","5":"1","6":"1","7":"1","8":"13","9":"NA","10":"0","11":"nu.4","12":".p13."},{"1":"14","2":"x5","3":"~1","4":"","5":"1","6":"1","7":"1","8":"14","9":"NA","10":"0","11":"nu.5","12":".p14."},{"1":"15","2":"x6","3":"~1","4":"","5":"1","6":"1","7":"1","8":"15","9":"NA","10":"0","11":"nu.6","12":".p15."},{"1":"16","2":"x7","3":"~1","4":"","5":"1","6":"1","7":"1","8":"16","9":"NA","10":"0","11":"nu.7","12":".p16."},{"1":"17","2":"x8","3":"~1","4":"","5":"1","6":"1","7":"1","8":"17","9":"NA","10":"0","11":"nu.8","12":".p17."},{"1":"18","2":"x9","3":"~1","4":"","5":"1","6":"1","7":"1","8":"18","9":"NA","10":"0","11":"nu.9","12":".p18."},{"1":"19","2":"x1","3":"~~","4":"x1","5":"1","6":"1","7":"1","8":"19","9":"NA","10":"0","11":"theta.1_1","12":".p19."},{"1":"20","2":"x2","3":"~~","4":"x2","5":"1","6":"1","7":"1","8":"20","9":"NA","10":"0","11":"theta.2_2","12":".p20."},{"1":"21","2":"x3","3":"~~","4":"x3","5":"1","6":"1","7":"1","8":"21","9":"NA","10":"0","11":"theta.3_3","12":".p21."},{"1":"22","2":"x4","3":"~~","4":"x4","5":"1","6":"1","7":"1","8":"22","9":"NA","10":"0","11":"theta.4_4","12":".p22."},{"1":"23","2":"x5","3":"~~","4":"x5","5":"1","6":"1","7":"1","8":"23","9":"NA","10":"0","11":"theta.5_5","12":".p23."},{"1":"24","2":"x6","3":"~~","4":"x6","5":"1","6":"1","7":"1","8":"24","9":"NA","10":"0","11":"theta.6_6","12":".p24."},{"1":"25","2":"x7","3":"~~","4":"x7","5":"1","6":"1","7":"1","8":"25","9":"NA","10":"0","11":"theta.7_7","12":".p25."},{"1":"26","2":"x8","3":"~~","4":"x8","5":"1","6":"1","7":"1","8":"26","9":"NA","10":"0","11":"theta.8_8","12":".p26."},{"1":"27","2":"x9","3":"~~","4":"x9","5":"1","6":"1","7":"1","8":"27","9":"NA","10":"0","11":"theta.9_9","12":".p27."},{"1":"28","2":"visual","3":"~1","4":"","5":"1","6":"1","7":"1","8":"0","9":"0","10":"0","11":"alpha.1.g1","12":".p28."},{"1":"29","2":"textual","3":"~1","4":"","5":"1","6":"1","7":"1","8":"0","9":"0","10":"0","11":"alpha.2.g1","12":".p29."},{"1":"30","2":"speed","3":"~1","4":"","5":"1","6":"1","7":"1","8":"0","9":"0","10":"0","11":"alpha.3.g1","12":".p30."},{"1":"31","2":"visual","3":"~~","4":"visual","5":"1","6":"1","7":"1","8":"0","9":"1","10":"0","11":"psi.1_1.g1","12":".p31."},{"1":"32","2":"textual","3":"~~","4":"textual","5":"1","6":"1","7":"1","8":"0","9":"1","10":"0","11":"psi.2_2.g1","12":".p32."},{"1":"33","2":"speed","3":"~~","4":"speed","5":"1","6":"1","7":"1","8":"0","9":"1","10":"0","11":"psi.3_3.g1","12":".p33."},{"1":"34","2":"visual","3":"~~","4":"textual","5":"1","6":"1","7":"1","8":"28","9":"NA","10":"0","11":"psi.2_1.g1","12":".p34."},{"1":"35","2":"visual","3":"~~","4":"speed","5":"1","6":"1","7":"1","8":"29","9":"NA","10":"0","11":"psi.3_1.g1","12":".p35."},{"1":"36","2":"textual","3":"~~","4":"speed","5":"1","6":"1","7":"1","8":"30","9":"NA","10":"0","11":"psi.3_2.g1","12":".p36."},{"1":"37","2":"visual","3":"=~","4":"x1","5":"1","6":"2","7":"2","8":"31","9":"NA","10":"0","11":"lambda.1_1","12":".p37."},{"1":"38","2":"visual","3":"=~","4":"x2","5":"1","6":"2","7":"2","8":"32","9":"NA","10":"0","11":"lambda.2_1","12":".p38."},{"1":"39","2":"visual","3":"=~","4":"x3","5":"1","6":"2","7":"2","8":"33","9":"NA","10":"0","11":"lambda.3_1","12":".p39."},{"1":"40","2":"textual","3":"=~","4":"x4","5":"1","6":"2","7":"2","8":"34","9":"NA","10":"0","11":"lambda.4_2","12":".p40."},{"1":"41","2":"textual","3":"=~","4":"x5","5":"1","6":"2","7":"2","8":"35","9":"NA","10":"0","11":"lambda.5_2","12":".p41."},{"1":"42","2":"textual","3":"=~","4":"x6","5":"1","6":"2","7":"2","8":"36","9":"NA","10":"0","11":"lambda.6_2","12":".p42."},{"1":"43","2":"speed","3":"=~","4":"x7","5":"1","6":"2","7":"2","8":"37","9":"NA","10":"0","11":"lambda.7_3","12":".p43."},{"1":"44","2":"speed","3":"=~","4":"x8","5":"1","6":"2","7":"2","8":"38","9":"NA","10":"0","11":"lambda.8_3","12":".p44."},{"1":"45","2":"speed","3":"=~","4":"x9","5":"1","6":"2","7":"2","8":"39","9":"NA","10":"0","11":"lambda.9_3","12":".p45."},{"1":"46","2":"x1","3":"~1","4":"","5":"1","6":"2","7":"2","8":"40","9":"NA","10":"0","11":"nu.1","12":".p46."},{"1":"47","2":"x2","3":"~1","4":"","5":"1","6":"2","7":"2","8":"41","9":"NA","10":"0","11":"nu.2","12":".p47."},{"1":"48","2":"x3","3":"~1","4":"","5":"1","6":"2","7":"2","8":"42","9":"NA","10":"0","11":"nu.3","12":".p48."},{"1":"49","2":"x4","3":"~1","4":"","5":"1","6":"2","7":"2","8":"43","9":"NA","10":"0","11":"nu.4","12":".p49."},{"1":"50","2":"x5","3":"~1","4":"","5":"1","6":"2","7":"2","8":"44","9":"NA","10":"0","11":"nu.5","12":".p50."},{"1":"51","2":"x6","3":"~1","4":"","5":"1","6":"2","7":"2","8":"45","9":"NA","10":"0","11":"nu.6","12":".p51."},{"1":"52","2":"x7","3":"~1","4":"","5":"1","6":"2","7":"2","8":"46","9":"NA","10":"0","11":"nu.7","12":".p52."},{"1":"53","2":"x8","3":"~1","4":"","5":"1","6":"2","7":"2","8":"47","9":"NA","10":"0","11":"nu.8","12":".p53."},{"1":"54","2":"x9","3":"~1","4":"","5":"1","6":"2","7":"2","8":"48","9":"NA","10":"0","11":"nu.9","12":".p54."},{"1":"55","2":"x1","3":"~~","4":"x1","5":"1","6":"2","7":"2","8":"49","9":"NA","10":"0","11":"theta.1_1","12":".p55."},{"1":"56","2":"x2","3":"~~","4":"x2","5":"1","6":"2","7":"2","8":"50","9":"NA","10":"0","11":"theta.2_2","12":".p56."},{"1":"57","2":"x3","3":"~~","4":"x3","5":"1","6":"2","7":"2","8":"51","9":"NA","10":"0","11":"theta.3_3","12":".p57."},{"1":"58","2":"x4","3":"~~","4":"x4","5":"1","6":"2","7":"2","8":"52","9":"NA","10":"0","11":"theta.4_4","12":".p58."},{"1":"59","2":"x5","3":"~~","4":"x5","5":"1","6":"2","7":"2","8":"53","9":"NA","10":"0","11":"theta.5_5","12":".p59."},{"1":"60","2":"x6","3":"~~","4":"x6","5":"1","6":"2","7":"2","8":"54","9":"NA","10":"0","11":"theta.6_6","12":".p60."},{"1":"61","2":"x7","3":"~~","4":"x7","5":"1","6":"2","7":"2","8":"55","9":"NA","10":"0","11":"theta.7_7","12":".p61."},{"1":"62","2":"x8","3":"~~","4":"x8","5":"1","6":"2","7":"2","8":"56","9":"NA","10":"0","11":"theta.8_8","12":".p62."},{"1":"63","2":"x9","3":"~~","4":"x9","5":"1","6":"2","7":"2","8":"57","9":"NA","10":"0","11":"theta.9_9","12":".p63."},{"1":"64","2":"visual","3":"~1","4":"","5":"1","6":"2","7":"2","8":"58","9":"NA","10":"0","11":"alpha.1.g2","12":".p64."},{"1":"65","2":"textual","3":"~1","4":"","5":"1","6":"2","7":"2","8":"59","9":"NA","10":"0","11":"alpha.2.g2","12":".p65."},{"1":"66","2":"speed","3":"~1","4":"","5":"1","6":"2","7":"2","8":"60","9":"NA","10":"0","11":"alpha.3.g2","12":".p66."},{"1":"67","2":"visual","3":"~~","4":"visual","5":"1","6":"2","7":"2","8":"61","9":"NA","10":"0","11":"psi.1_1.g2","12":".p67."},{"1":"68","2":"textual","3":"~~","4":"textual","5":"1","6":"2","7":"2","8":"62","9":"NA","10":"0","11":"psi.2_2.g2","12":".p68."},{"1":"69","2":"speed","3":"~~","4":"speed","5":"1","6":"2","7":"2","8":"63","9":"NA","10":"0","11":"psi.3_3.g2","12":".p69."},{"1":"70","2":"visual","3":"~~","4":"textual","5":"1","6":"2","7":"2","8":"64","9":"NA","10":"0","11":"psi.2_1.g2","12":".p70."},{"1":"71","2":"visual","3":"~~","4":"speed","5":"1","6":"2","7":"2","8":"65","9":"NA","10":"0","11":"psi.3_1.g2","12":".p71."},{"1":"72","2":"textual","3":"~~","4":"speed","5":"1","6":"2","7":"2","8":"66","9":"NA","10":"0","11":"psi.3_2.g2","12":".p72."},{"1":"73","2":".p1.","3":"==","4":".p37.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"74","2":".p2.","3":"==","4":".p38.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"75","2":".p3.","3":"==","4":".p39.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"76","2":".p4.","3":"==","4":".p40.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"77","2":".p5.","3":"==","4":".p41.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"78","2":".p6.","3":"==","4":".p42.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"79","2":".p7.","3":"==","4":".p43.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"80","2":".p8.","3":"==","4":".p44.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"81","2":".p9.","3":"==","4":".p45.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"82","2":".p10.","3":"==","4":".p46.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"83","2":".p11.","3":"==","4":".p47.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"84","2":".p12.","3":"==","4":".p48.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"85","2":".p13.","3":"==","4":".p49.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"86","2":".p14.","3":"==","4":".p50.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"87","2":".p15.","3":"==","4":".p51.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"88","2":".p16.","3":"==","4":".p52.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"89","2":".p17.","3":"==","4":".p53.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"90","2":".p18.","3":"==","4":".p54.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"91","2":".p19.","3":"==","4":".p55.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"92","2":".p20.","3":"==","4":".p56.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"93","2":".p21.","3":"==","4":".p57.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"94","2":".p22.","3":"==","4":".p58.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"95","2":".p23.","3":"==","4":".p59.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"96","2":".p24.","3":"==","4":".p60.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"97","2":".p25.","3":"==","4":".p61.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"98","2":".p26.","3":"==","4":".p62.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"99","2":".p27.","3":"==","4":".p63.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1684"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1684-1"><a href="bias.html#cb1684-1" tabindex="-1"></a><span class="fu">lavaanify</span>(cfaModel_residualInvariance, <span class="at">ngroups =</span> <span class="dv">2</span>)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["id"],"name":[1],"type":["int"],"align":["right"]},{"label":["lhs"],"name":[2],"type":["chr"],"align":["left"]},{"label":["op"],"name":[3],"type":["chr"],"align":["left"]},{"label":["rhs"],"name":[4],"type":["chr"],"align":["left"]},{"label":["user"],"name":[5],"type":["int"],"align":["right"]},{"label":["block"],"name":[6],"type":["int"],"align":["right"]},{"label":["group"],"name":[7],"type":["int"],"align":["right"]},{"label":["free"],"name":[8],"type":["int"],"align":["right"]},{"label":["ustart"],"name":[9],"type":["dbl"],"align":["right"]},{"label":["exo"],"name":[10],"type":["int"],"align":["right"]},{"label":["label"],"name":[11],"type":["chr"],"align":["left"]},{"label":["plabel"],"name":[12],"type":["chr"],"align":["left"]}],"data":[{"1":"1","2":"visual","3":"=~","4":"x1","5":"1","6":"1","7":"1","8":"1","9":"NA","10":"0","11":"lambdax1","12":".p1."},{"1":"2","2":"visual","3":"=~","4":"x2","5":"1","6":"1","7":"1","8":"2","9":"NA","10":"0","11":"lambdax2","12":".p2."},{"1":"3","2":"visual","3":"=~","4":"x3","5":"1","6":"1","7":"1","8":"3","9":"NA","10":"0","11":"lambdax3","12":".p3."},{"1":"4","2":"textual","3":"=~","4":"x4","5":"1","6":"1","7":"1","8":"4","9":"NA","10":"0","11":"lambdax4","12":".p4."},{"1":"5","2":"textual","3":"=~","4":"x5","5":"1","6":"1","7":"1","8":"5","9":"NA","10":"0","11":"lambdax5","12":".p5."},{"1":"6","2":"textual","3":"=~","4":"x6","5":"1","6":"1","7":"1","8":"6","9":"NA","10":"0","11":"lambdax6","12":".p6."},{"1":"7","2":"speed","3":"=~","4":"x7","5":"1","6":"1","7":"1","8":"7","9":"NA","10":"0","11":"lambdax7","12":".p7."},{"1":"8","2":"speed","3":"=~","4":"x8","5":"1","6":"1","7":"1","8":"8","9":"NA","10":"0","11":"lambdax8","12":".p8."},{"1":"9","2":"speed","3":"=~","4":"x9","5":"1","6":"1","7":"1","8":"9","9":"NA","10":"0","11":"lambdax9","12":".p9."},{"1":"10","2":"visual","3":"~1","4":"","5":"1","6":"1","7":"1","8":"0","9":"0","10":"0","11":"","12":".p10."},{"1":"11","2":"textual","3":"~1","4":"","5":"1","6":"1","7":"1","8":"0","9":"0","10":"0","11":"","12":".p11."},{"1":"12","2":"speed","3":"~1","4":"","5":"1","6":"1","7":"1","8":"0","9":"0","10":"0","11":"","12":".p12."},{"1":"13","2":"visual","3":"~~","4":"visual","5":"1","6":"1","7":"1","8":"0","9":"1","10":"0","11":"","12":".p13."},{"1":"14","2":"textual","3":"~~","4":"textual","5":"1","6":"1","7":"1","8":"0","9":"1","10":"0","11":"","12":".p14."},{"1":"15","2":"speed","3":"~~","4":"speed","5":"1","6":"1","7":"1","8":"0","9":"1","10":"0","11":"","12":".p15."},{"1":"16","2":"visual","3":"~~","4":"textual","5":"1","6":"1","7":"1","8":"10","9":"NA","10":"0","11":"","12":".p16."},{"1":"17","2":"visual","3":"~~","4":"speed","5":"1","6":"1","7":"1","8":"11","9":"NA","10":"0","11":"","12":".p17."},{"1":"18","2":"textual","3":"~~","4":"speed","5":"1","6":"1","7":"1","8":"12","9":"NA","10":"0","11":"","12":".p18."},{"1":"19","2":"x1","3":"~~","4":"x1","5":"1","6":"1","7":"1","8":"13","9":"NA","10":"0","11":"residx1","12":".p19."},{"1":"20","2":"x2","3":"~~","4":"x2","5":"1","6":"1","7":"1","8":"14","9":"NA","10":"0","11":"residx2","12":".p20."},{"1":"21","2":"x3","3":"~~","4":"x3","5":"1","6":"1","7":"1","8":"15","9":"NA","10":"0","11":"residx3","12":".p21."},{"1":"22","2":"x4","3":"~~","4":"x4","5":"1","6":"1","7":"1","8":"16","9":"NA","10":"0","11":"residx4","12":".p22."},{"1":"23","2":"x5","3":"~~","4":"x5","5":"1","6":"1","7":"1","8":"17","9":"NA","10":"0","11":"residx5","12":".p23."},{"1":"24","2":"x6","3":"~~","4":"x6","5":"1","6":"1","7":"1","8":"18","9":"NA","10":"0","11":"residx6","12":".p24."},{"1":"25","2":"x7","3":"~~","4":"x7","5":"1","6":"1","7":"1","8":"19","9":"NA","10":"0","11":"residx7","12":".p25."},{"1":"26","2":"x8","3":"~~","4":"x8","5":"1","6":"1","7":"1","8":"20","9":"NA","10":"0","11":"residx8","12":".p26."},{"1":"27","2":"x9","3":"~~","4":"x9","5":"1","6":"1","7":"1","8":"21","9":"NA","10":"0","11":"residx9","12":".p27."},{"1":"28","2":"x1","3":"~1","4":"","5":"1","6":"1","7":"1","8":"22","9":"NA","10":"0","11":"intx1","12":".p28."},{"1":"29","2":"x2","3":"~1","4":"","5":"1","6":"1","7":"1","8":"23","9":"NA","10":"0","11":"intx2","12":".p29."},{"1":"30","2":"x3","3":"~1","4":"","5":"1","6":"1","7":"1","8":"24","9":"NA","10":"0","11":"intx3","12":".p30."},{"1":"31","2":"x4","3":"~1","4":"","5":"1","6":"1","7":"1","8":"25","9":"NA","10":"0","11":"intx4","12":".p31."},{"1":"32","2":"x5","3":"~1","4":"","5":"1","6":"1","7":"1","8":"26","9":"NA","10":"0","11":"intx5","12":".p32."},{"1":"33","2":"x6","3":"~1","4":"","5":"1","6":"1","7":"1","8":"27","9":"NA","10":"0","11":"intx6","12":".p33."},{"1":"34","2":"x7","3":"~1","4":"","5":"1","6":"1","7":"1","8":"28","9":"NA","10":"0","11":"intx7","12":".p34."},{"1":"35","2":"x8","3":"~1","4":"","5":"1","6":"1","7":"1","8":"29","9":"NA","10":"0","11":"intx8","12":".p35."},{"1":"36","2":"x9","3":"~1","4":"","5":"1","6":"1","7":"1","8":"30","9":"NA","10":"0","11":"intx9","12":".p36."},{"1":"37","2":"visual","3":"=~","4":"x1","5":"1","6":"2","7":"2","8":"31","9":"NA","10":"0","11":"lambdax1","12":".p37."},{"1":"38","2":"visual","3":"=~","4":"x2","5":"1","6":"2","7":"2","8":"32","9":"NA","10":"0","11":"lambdax2","12":".p38."},{"1":"39","2":"visual","3":"=~","4":"x3","5":"1","6":"2","7":"2","8":"33","9":"NA","10":"0","11":"lambdax3","12":".p39."},{"1":"40","2":"textual","3":"=~","4":"x4","5":"1","6":"2","7":"2","8":"34","9":"NA","10":"0","11":"lambdax4","12":".p40."},{"1":"41","2":"textual","3":"=~","4":"x5","5":"1","6":"2","7":"2","8":"35","9":"NA","10":"0","11":"lambdax5","12":".p41."},{"1":"42","2":"textual","3":"=~","4":"x6","5":"1","6":"2","7":"2","8":"36","9":"NA","10":"0","11":"lambdax6","12":".p42."},{"1":"43","2":"speed","3":"=~","4":"x7","5":"1","6":"2","7":"2","8":"37","9":"NA","10":"0","11":"lambdax7","12":".p43."},{"1":"44","2":"speed","3":"=~","4":"x8","5":"1","6":"2","7":"2","8":"38","9":"NA","10":"0","11":"lambdax8","12":".p44."},{"1":"45","2":"speed","3":"=~","4":"x9","5":"1","6":"2","7":"2","8":"39","9":"NA","10":"0","11":"lambdax9","12":".p45."},{"1":"46","2":"visual","3":"~1","4":"","5":"1","6":"2","7":"2","8":"40","9":"NA","10":"0","11":"","12":".p46."},{"1":"47","2":"textual","3":"~1","4":"","5":"1","6":"2","7":"2","8":"41","9":"NA","10":"0","11":"","12":".p47."},{"1":"48","2":"speed","3":"~1","4":"","5":"1","6":"2","7":"2","8":"42","9":"NA","10":"0","11":"","12":".p48."},{"1":"49","2":"visual","3":"~~","4":"visual","5":"1","6":"2","7":"2","8":"43","9":"NA","10":"0","11":"","12":".p49."},{"1":"50","2":"textual","3":"~~","4":"textual","5":"1","6":"2","7":"2","8":"44","9":"NA","10":"0","11":"","12":".p50."},{"1":"51","2":"speed","3":"~~","4":"speed","5":"1","6":"2","7":"2","8":"45","9":"NA","10":"0","11":"","12":".p51."},{"1":"52","2":"visual","3":"~~","4":"textual","5":"1","6":"2","7":"2","8":"46","9":"NA","10":"0","11":"","12":".p52."},{"1":"53","2":"visual","3":"~~","4":"speed","5":"1","6":"2","7":"2","8":"47","9":"NA","10":"0","11":"","12":".p53."},{"1":"54","2":"textual","3":"~~","4":"speed","5":"1","6":"2","7":"2","8":"48","9":"NA","10":"0","11":"","12":".p54."},{"1":"55","2":"x1","3":"~~","4":"x1","5":"1","6":"2","7":"2","8":"49","9":"NA","10":"0","11":"residx1","12":".p55."},{"1":"56","2":"x2","3":"~~","4":"x2","5":"1","6":"2","7":"2","8":"50","9":"NA","10":"0","11":"residx2","12":".p56."},{"1":"57","2":"x3","3":"~~","4":"x3","5":"1","6":"2","7":"2","8":"51","9":"NA","10":"0","11":"residx3","12":".p57."},{"1":"58","2":"x4","3":"~~","4":"x4","5":"1","6":"2","7":"2","8":"52","9":"NA","10":"0","11":"residx4","12":".p58."},{"1":"59","2":"x5","3":"~~","4":"x5","5":"1","6":"2","7":"2","8":"53","9":"NA","10":"0","11":"residx5","12":".p59."},{"1":"60","2":"x6","3":"~~","4":"x6","5":"1","6":"2","7":"2","8":"54","9":"NA","10":"0","11":"residx6","12":".p60."},{"1":"61","2":"x7","3":"~~","4":"x7","5":"1","6":"2","7":"2","8":"55","9":"NA","10":"0","11":"residx7","12":".p61."},{"1":"62","2":"x8","3":"~~","4":"x8","5":"1","6":"2","7":"2","8":"56","9":"NA","10":"0","11":"residx8","12":".p62."},{"1":"63","2":"x9","3":"~~","4":"x9","5":"1","6":"2","7":"2","8":"57","9":"NA","10":"0","11":"residx9","12":".p63."},{"1":"64","2":"x1","3":"~1","4":"","5":"1","6":"2","7":"2","8":"58","9":"NA","10":"0","11":"intx1","12":".p64."},{"1":"65","2":"x2","3":"~1","4":"","5":"1","6":"2","7":"2","8":"59","9":"NA","10":"0","11":"intx2","12":".p65."},{"1":"66","2":"x3","3":"~1","4":"","5":"1","6":"2","7":"2","8":"60","9":"NA","10":"0","11":"intx3","12":".p66."},{"1":"67","2":"x4","3":"~1","4":"","5":"1","6":"2","7":"2","8":"61","9":"NA","10":"0","11":"intx4","12":".p67."},{"1":"68","2":"x5","3":"~1","4":"","5":"1","6":"2","7":"2","8":"62","9":"NA","10":"0","11":"intx5","12":".p68."},{"1":"69","2":"x6","3":"~1","4":"","5":"1","6":"2","7":"2","8":"63","9":"NA","10":"0","11":"intx6","12":".p69."},{"1":"70","2":"x7","3":"~1","4":"","5":"1","6":"2","7":"2","8":"64","9":"NA","10":"0","11":"intx7","12":".p70."},{"1":"71","2":"x8","3":"~1","4":"","5":"1","6":"2","7":"2","8":"65","9":"NA","10":"0","11":"intx8","12":".p71."},{"1":"72","2":"x9","3":"~1","4":"","5":"1","6":"2","7":"2","8":"66","9":"NA","10":"0","11":"intx9","12":".p72."},{"1":"73","2":".p1.","3":"==","4":".p37.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"74","2":".p2.","3":"==","4":".p38.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"75","2":".p3.","3":"==","4":".p39.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"76","2":".p4.","3":"==","4":".p40.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"77","2":".p5.","3":"==","4":".p41.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"78","2":".p6.","3":"==","4":".p42.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"79","2":".p7.","3":"==","4":".p43.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"80","2":".p8.","3":"==","4":".p44.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"81","2":".p9.","3":"==","4":".p45.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"82","2":".p19.","3":"==","4":".p55.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"83","2":".p20.","3":"==","4":".p56.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"84","2":".p21.","3":"==","4":".p57.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"85","2":".p22.","3":"==","4":".p58.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"86","2":".p23.","3":"==","4":".p59.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"87","2":".p24.","3":"==","4":".p60.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"88","2":".p25.","3":"==","4":".p61.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"89","2":".p26.","3":"==","4":".p62.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"90","2":".p27.","3":"==","4":".p63.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"91","2":".p28.","3":"==","4":".p64.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"92","2":".p29.","3":"==","4":".p65.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"93","2":".p30.","3":"==","4":".p66.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"94","2":".p31.","3":"==","4":".p67.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"95","2":".p32.","3":"==","4":".p68.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"96","2":".p33.","3":"==","4":".p69.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"97","2":".p34.","3":"==","4":".p70.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"98","2":".p35.","3":"==","4":".p71.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""},{"1":"99","2":".p36.","3":"==","4":".p72.","5":"2","6":"0","7":"0","8":"0","9":"NA","10":"0","11":"","12":""}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceResidualModelFit" class="section level4 hasAnchor" number="16.10.7.3">
<h4><span class="header-section-number">16.10.7.3</span> Fit the Model<a href="bias.html#measurementInvarianceResidualModelFit" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1685"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1685-1"><a href="bias.html#cb1685-1" tabindex="-1"></a>residualInvarianceModel_fit <span class="ot">&lt;-</span> <span class="fu">cfa</span>(</span>
<span id="cb1685-2"><a href="bias.html#cb1685-2" tabindex="-1"></a>  residualInvarianceSyntax,</span>
<span id="cb1685-3"><a href="bias.html#cb1685-3" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1685-4"><a href="bias.html#cb1685-4" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1685-5"><a href="bias.html#cb1685-5" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1685-6"><a href="bias.html#cb1685-6" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span></code></pre></div>
</details>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1686"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1686-1"><a href="bias.html#cb1686-1" tabindex="-1"></a>residualInvarianceModelFullSyntax_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1686-2"><a href="bias.html#cb1686-2" tabindex="-1"></a>  cfaModel_residualInvariance,</span>
<span id="cb1686-3"><a href="bias.html#cb1686-3" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1686-4"><a href="bias.html#cb1686-4" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1686-5"><a href="bias.html#cb1686-5" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1686-6"><a href="bias.html#cb1686-6" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span></code></pre></div>
</details>
</div>
<div id="measurementInvarianceResidualModelOutput" class="section level4 hasAnchor" number="16.10.7.4">
<h4><span class="header-section-number">16.10.7.4</span> Model Summary<a href="bias.html#measurementInvarianceResidualModelOutput" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1687"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1687-1"><a href="bias.html#cb1687-1" tabindex="-1"></a><span class="fu">summary</span>(</span>
<span id="cb1687-2"><a href="bias.html#cb1687-2" tabindex="-1"></a>  residualInvarianceModel_fit,</span>
<span id="cb1687-3"><a href="bias.html#cb1687-3" tabindex="-1"></a>  <span class="at">fit.measures =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1687-4"><a href="bias.html#cb1687-4" tabindex="-1"></a>  <span class="at">standardized =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1687-5"><a href="bias.html#cb1687-5" tabindex="-1"></a>  <span class="at">rsquare =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
<pre><code>lavaan 0.6-19 ended normally after 62 iterations

  Estimator                                         ML
  Optimization method                           NLMINB
  Number of model parameters                        66
  Number of equality constraints                    27

  Number of observations per group:                   
    Pasteur                                        156
    Grant-White                                    145
  Number of missing patterns per group:               
    Pasteur                                          9
    Grant-White                                      9

Model Test User Model:
                                              Standard      Scaled
  Test Statistic                               172.960     172.714
  Degrees of freedom                                69          69
  P-value (Chi-square)                           0.000       0.000
  Scaling correction factor                                  1.001
    Yuan-Bentler correction (Mplus variant)                       
  Test statistic for each group:
    Pasteur                                     87.707      87.707
    Grant-White                                 85.007      85.007

Model Test Baseline Model:

  Test statistic                               932.665     909.019
  Degrees of freedom                                72          72
  P-value                                        0.000       0.000
  Scaling correction factor                                  1.026

User Model versus Baseline Model:

  Comparative Fit Index (CFI)                    0.879       0.876
  Tucker-Lewis Index (TLI)                       0.874       0.871
                                                                  
  Robust Comparative Fit Index (CFI)                         0.883
  Robust Tucker-Lewis Index (TLI)                            0.878

Loglikelihood and Information Criteria:

  Loglikelihood user model (H0)              -3680.600   -3680.600
  Scaling correction factor                                  0.645
      for the MLR correction                                      
  Loglikelihood unrestricted model (H1)      -3594.120   -3594.120
  Scaling correction factor                                  1.034
      for the MLR correction                                      
                                                                  
  Akaike (AIC)                                7439.201    7439.201
  Bayesian (BIC)                              7583.778    7583.778
  Sample-size adjusted Bayesian (SABIC)       7460.092    7460.092

Root Mean Square Error of Approximation:

  RMSEA                                          0.100       0.100
  90 Percent confidence interval - lower         0.082       0.081
  90 Percent confidence interval - upper         0.119       0.119
  P-value H_0: RMSEA &lt;= 0.050                    0.000       0.000
  P-value H_0: RMSEA &gt;= 0.080                    0.962       0.961
                                                                  
  Robust RMSEA                                               0.100
  90 Percent confidence interval - lower                     0.081
  90 Percent confidence interval - upper                     0.119
  P-value H_0: Robust RMSEA &lt;= 0.050                         0.000
  P-value H_0: Robust RMSEA &gt;= 0.080                         0.955

Standardized Root Mean Square Residual:

  SRMR                                           0.086       0.086

Parameter Estimates:

  Standard errors                             Sandwich
  Information bread                           Observed
  Observed information based on                Hessian


Group 1 [Pasteur]:

Latent Variables:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual =~                                                             
    x1      (l.1_)    0.862    0.126    6.855    0.000    0.862    0.728
    x2      (l.2_)    0.516    0.091    5.678    0.000    0.516    0.436
    x3      (l.3_)    0.745    0.112    6.630    0.000    0.745    0.651
  textual =~                                                            
    x4      (l.4_)    0.928    0.071   13.094    0.000    0.928    0.828
    x5      (l.5_)    1.069    0.076   14.113    0.000    1.069    0.855
    x6      (l.6_)    0.882    0.062   14.125    0.000    0.882    0.830
  speed =~                                                              
    x7      (l.7_)    0.577    0.087    6.658    0.000    0.577    0.544
    x8      (l.8_)    0.669    0.083    8.109    0.000    0.669    0.688
    x9      (l.9_)    0.632    0.085    7.461    0.000    0.632    0.649

Covariances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual ~~                                                             
    textul  (p.2_)    0.430    0.124    3.472    0.001    0.430    0.430
    speed  (p.3_1)    0.351    0.129    2.710    0.007    0.351    0.351
  textual ~~                                                            
    speed  (p.3_2)    0.302    0.093    3.250    0.001    0.302    0.302

Intercepts:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
   .x1      (nu.1)    5.025    0.095   52.938    0.000    5.025    4.243
   .x2      (nu.2)    6.138    0.086   71.711    0.000    6.138    5.188
   .x3      (nu.3)    2.332    0.099   23.546    0.000    2.332    2.039
   .x4      (nu.4)    2.784    0.086   32.453    0.000    2.784    2.483
   .x5      (nu.5)    4.030    0.102   39.369    0.000    4.030    3.224
   .x6      (nu.6)    1.928    0.075   25.828    0.000    1.928    1.815
   .x7      (nu.7)    4.270    0.076   55.911    0.000    4.270    4.027
   .x8      (nu.8)    5.619    0.075   74.597    0.000    5.619    5.775
   .x9      (nu.9)    5.470    0.073   74.756    0.000    5.470    5.616
    visual  (a.1.)    0.000                               0.000    0.000
    textual (a.2.)    0.000                               0.000    0.000
    speed   (a.3.)    0.000                               0.000    0.000

Variances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
   .x1      (t.1_)    0.660    0.165    4.008    0.000    0.660    0.470
   .x2      (t.2_)    1.133    0.110   10.337    0.000    1.133    0.810
   .x3      (t.3_)    0.754    0.131    5.749    0.000    0.754    0.576
   .x4      (t.4_)    0.396    0.050    7.904    0.000    0.396    0.315
   .x5      (t.5_)    0.420    0.057    7.385    0.000    0.420    0.269
   .x6      (t.6_)    0.350    0.048    7.311    0.000    0.350    0.310
   .x7      (t.7_)    0.792    0.094    8.413    0.000    0.792    0.704
   .x8      (t.8_)    0.499    0.095    5.244    0.000    0.499    0.527
   .x9      (t.9_)    0.549    0.100    5.479    0.000    0.549    0.579
    visual  (p.1_)    1.000                               1.000    1.000
    textual (p.2_)    1.000                               1.000    1.000
    speed   (p.3_)    1.000                               1.000    1.000

R-Square:
                   Estimate
    x1                0.530
    x2                0.190
    x3                0.424
    x4                0.685
    x5                0.731
    x6                0.690
    x7                0.296
    x8                0.473
    x9                0.421


Group 2 [Grant-White]:

Latent Variables:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual =~                                                             
    x1      (l.1_)    0.862    0.126    6.855    0.000    0.796    0.700
    x2      (l.2_)    0.516    0.091    5.678    0.000    0.476    0.408
    x3      (l.3_)    0.745    0.112    6.630    0.000    0.687    0.621
  textual =~                                                            
    x4      (l.4_)    0.928    0.071   13.094    0.000    0.926    0.827
    x5      (l.5_)    1.069    0.076   14.113    0.000    1.068    0.855
    x6      (l.6_)    0.882    0.062   14.125    0.000    0.881    0.830
  speed =~                                                              
    x7      (l.7_)    0.577    0.087    6.658    0.000    0.670    0.601
    x8      (l.8_)    0.669    0.083    8.109    0.000    0.777    0.740
    x9      (l.9_)    0.632    0.085    7.461    0.000    0.733    0.703

Covariances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual ~~                                                             
    textul  (p.2_)    0.519    0.135    3.852    0.000    0.563    0.563
    speed  (p.3_1)    0.643    0.202    3.183    0.001    0.600    0.600
  textual ~~                                                            
    speed  (p.3_2)    0.426    0.181    2.351    0.019    0.368    0.368

Intercepts:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
   .x1      (nu.1)    5.025    0.095   52.938    0.000    5.025    4.420
   .x2      (nu.2)    6.138    0.086   71.711    0.000    6.138    5.263
   .x3      (nu.3)    2.332    0.099   23.546    0.000    2.332    2.106
   .x4      (nu.4)    2.784    0.086   32.453    0.000    2.784    2.486
   .x5      (nu.5)    4.030    0.102   39.369    0.000    4.030    3.228
   .x6      (nu.6)    1.928    0.075   25.828    0.000    1.928    1.817
   .x7      (nu.7)    4.270    0.076   55.911    0.000    4.270    3.835
   .x8      (nu.8)    5.619    0.075   74.597    0.000    5.619    5.353
   .x9      (nu.9)    5.470    0.073   74.756    0.000    5.470    5.247
    visual  (a.1.)   -0.208    0.181   -1.149    0.251   -0.225   -0.225
    textual (a.2.)    0.611    0.132    4.646    0.000    0.612    0.612
    speed   (a.3.)   -0.283    0.155   -1.824    0.068   -0.244   -0.244

Variances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
   .x1      (t.1_)    0.660    0.165    4.008    0.000    0.660    0.510
   .x2      (t.2_)    1.133    0.110   10.337    0.000    1.133    0.833
   .x3      (t.3_)    0.754    0.131    5.749    0.000    0.754    0.615
   .x4      (t.4_)    0.396    0.050    7.904    0.000    0.396    0.316
   .x5      (t.5_)    0.420    0.057    7.385    0.000    0.420    0.269
   .x6      (t.6_)    0.350    0.048    7.311    0.000    0.350    0.311
   .x7      (t.7_)    0.792    0.094    8.413    0.000    0.792    0.638
   .x8      (t.8_)    0.499    0.095    5.244    0.000    0.499    0.453
   .x9      (t.9_)    0.549    0.100    5.479    0.000    0.549    0.505
    visual  (p.1_)    0.852    0.217    3.932    0.000    1.000    1.000
    textual (p.2_)    0.997    0.193    5.160    0.000    1.000    1.000
    speed   (p.3_)    1.346    0.328    4.101    0.000    1.000    1.000

R-Square:
                   Estimate
    x1                0.490
    x2                0.167
    x3                0.385
    x4                0.684
    x5                0.731
    x6                0.689
    x7                0.362
    x8                0.547
    x9                0.495</code></pre>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1689"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1689-1"><a href="bias.html#cb1689-1" tabindex="-1"></a><span class="fu">summary</span>(</span>
<span id="cb1689-2"><a href="bias.html#cb1689-2" tabindex="-1"></a>  residualInvarianceModelFullSyntax_fit,</span>
<span id="cb1689-3"><a href="bias.html#cb1689-3" tabindex="-1"></a>  <span class="at">fit.measures =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1689-4"><a href="bias.html#cb1689-4" tabindex="-1"></a>  <span class="at">standardized =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1689-5"><a href="bias.html#cb1689-5" tabindex="-1"></a>  <span class="at">rsquare =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</details>
<pre><code>lavaan 0.6-19 ended normally after 59 iterations

  Estimator                                         ML
  Optimization method                           NLMINB
  Number of model parameters                        66
  Number of equality constraints                    27

  Number of observations per group:                   
    Pasteur                                        156
    Grant-White                                    145
  Number of missing patterns per group:               
    Pasteur                                          9
    Grant-White                                      9

Model Test User Model:
                                              Standard      Scaled
  Test Statistic                               172.960     172.714
  Degrees of freedom                                69          69
  P-value (Chi-square)                           0.000       0.000
  Scaling correction factor                                  1.001
    Yuan-Bentler correction (Mplus variant)                       
  Test statistic for each group:
    Pasteur                                     87.707      87.707
    Grant-White                                 85.007      85.007

Model Test Baseline Model:

  Test statistic                               932.665     909.019
  Degrees of freedom                                72          72
  P-value                                        0.000       0.000
  Scaling correction factor                                  1.026

User Model versus Baseline Model:

  Comparative Fit Index (CFI)                    0.879       0.876
  Tucker-Lewis Index (TLI)                       0.874       0.871
                                                                  
  Robust Comparative Fit Index (CFI)                         0.883
  Robust Tucker-Lewis Index (TLI)                            0.878

Loglikelihood and Information Criteria:

  Loglikelihood user model (H0)              -3680.600   -3680.600
  Scaling correction factor                                  0.645
      for the MLR correction                                      
  Loglikelihood unrestricted model (H1)      -3594.120   -3594.120
  Scaling correction factor                                  1.034
      for the MLR correction                                      
                                                                  
  Akaike (AIC)                                7439.201    7439.201
  Bayesian (BIC)                              7583.778    7583.778
  Sample-size adjusted Bayesian (SABIC)       7460.092    7460.092

Root Mean Square Error of Approximation:

  RMSEA                                          0.100       0.100
  90 Percent confidence interval - lower         0.082       0.081
  90 Percent confidence interval - upper         0.119       0.119
  P-value H_0: RMSEA &lt;= 0.050                    0.000       0.000
  P-value H_0: RMSEA &gt;= 0.080                    0.962       0.961
                                                                  
  Robust RMSEA                                               0.100
  90 Percent confidence interval - lower                     0.081
  90 Percent confidence interval - upper                     0.119
  P-value H_0: Robust RMSEA &lt;= 0.050                         0.000
  P-value H_0: Robust RMSEA &gt;= 0.080                         0.955

Standardized Root Mean Square Residual:

  SRMR                                           0.086       0.086

Parameter Estimates:

  Standard errors                             Sandwich
  Information bread                           Observed
  Observed information based on                Hessian


Group 1 [Pasteur]:

Latent Variables:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual =~                                                             
    x1      (lmb1)    0.862    0.126    6.855    0.000    0.862    0.728
    x2      (lmb2)    0.516    0.091    5.678    0.000    0.516    0.436
    x3      (lmb3)    0.745    0.112    6.630    0.000    0.745    0.651
  textual =~                                                            
    x4      (lmb4)    0.928    0.071   13.094    0.000    0.928    0.828
    x5      (lmb5)    1.069    0.076   14.113    0.000    1.069    0.855
    x6      (lmb6)    0.882    0.062   14.125    0.000    0.882    0.830
  speed =~                                                              
    x7      (lmb7)    0.577    0.087    6.658    0.000    0.577    0.544
    x8      (lmb8)    0.669    0.083    8.110    0.000    0.669    0.688
    x9      (lmb9)    0.632    0.085    7.461    0.000    0.632    0.649

Covariances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual ~~                                                             
    textual           0.430    0.124    3.472    0.001    0.430    0.430
    speed             0.351    0.129    2.710    0.007    0.351    0.351
  textual ~~                                                            
    speed             0.302    0.093    3.250    0.001    0.302    0.302

Intercepts:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
    visual            0.000                               0.000    0.000
    textual           0.000                               0.000    0.000
    speed             0.000                               0.000    0.000
   .x1      (int1)    5.025    0.095   52.938    0.000    5.025    4.243
   .x2      (int2)    6.138    0.086   71.710    0.000    6.138    5.188
   .x3      (int3)    2.332    0.099   23.546    0.000    2.332    2.039
   .x4      (int4)    2.784    0.086   32.453    0.000    2.784    2.483
   .x5      (int5)    4.030    0.102   39.369    0.000    4.030    3.224
   .x6      (int6)    1.928    0.075   25.828    0.000    1.928    1.815
   .x7      (int7)    4.270    0.076   55.911    0.000    4.270    4.027
   .x8      (int8)    5.619    0.075   74.598    0.000    5.619    5.775
   .x9      (int9)    5.470    0.073   74.756    0.000    5.470    5.616

Variances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
    visual            1.000                               1.000    1.000
    textual           1.000                               1.000    1.000
    speed             1.000                               1.000    1.000
   .x1      (rsd1)    0.660    0.165    4.008    0.000    0.660    0.470
   .x2      (rsd2)    1.133    0.110   10.337    0.000    1.133    0.810
   .x3      (rsd3)    0.754    0.131    5.749    0.000    0.754    0.576
   .x4      (rsd4)    0.396    0.050    7.904    0.000    0.396    0.315
   .x5      (rsd5)    0.420    0.057    7.385    0.000    0.420    0.269
   .x6      (rsd6)    0.350    0.048    7.311    0.000    0.350    0.310
   .x7      (rsd7)    0.792    0.094    8.414    0.000    0.792    0.704
   .x8      (rsd8)    0.499    0.095    5.244    0.000    0.499    0.527
   .x9      (rsd9)    0.549    0.100    5.479    0.000    0.549    0.579

R-Square:
                   Estimate
    x1                0.530
    x2                0.190
    x3                0.424
    x4                0.685
    x5                0.731
    x6                0.690
    x7                0.296
    x8                0.473
    x9                0.421


Group 2 [Grant-White]:

Latent Variables:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual =~                                                             
    x1      (lmb1)    0.862    0.126    6.855    0.000    0.796    0.700
    x2      (lmb2)    0.516    0.091    5.678    0.000    0.476    0.408
    x3      (lmb3)    0.745    0.112    6.630    0.000    0.687    0.621
  textual =~                                                            
    x4      (lmb4)    0.928    0.071   13.094    0.000    0.926    0.827
    x5      (lmb5)    1.069    0.076   14.113    0.000    1.068    0.855
    x6      (lmb6)    0.882    0.062   14.125    0.000    0.881    0.830
  speed =~                                                              
    x7      (lmb7)    0.577    0.087    6.658    0.000    0.670    0.601
    x8      (lmb8)    0.669    0.083    8.110    0.000    0.777    0.740
    x9      (lmb9)    0.632    0.085    7.461    0.000    0.733    0.703

Covariances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
  visual ~~                                                             
    textual           0.519    0.135    3.852    0.000    0.563    0.563
    speed             0.643    0.202    3.183    0.001    0.600    0.600
  textual ~~                                                            
    speed             0.426    0.181    2.351    0.019    0.368    0.368

Intercepts:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
    visual           -0.208    0.181   -1.149    0.251   -0.225   -0.225
    textual           0.611    0.132    4.646    0.000    0.612    0.612
    speed            -0.283    0.155   -1.824    0.068   -0.244   -0.244
   .x1      (int1)    5.025    0.095   52.938    0.000    5.025    4.420
   .x2      (int2)    6.138    0.086   71.710    0.000    6.138    5.263
   .x3      (int3)    2.332    0.099   23.546    0.000    2.332    2.106
   .x4      (int4)    2.784    0.086   32.453    0.000    2.784    2.486
   .x5      (int5)    4.030    0.102   39.369    0.000    4.030    3.228
   .x6      (int6)    1.928    0.075   25.828    0.000    1.928    1.817
   .x7      (int7)    4.270    0.076   55.911    0.000    4.270    3.835
   .x8      (int8)    5.619    0.075   74.598    0.000    5.619    5.353
   .x9      (int9)    5.470    0.073   74.756    0.000    5.470    5.247

Variances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)   Std.lv  Std.all
    visual            0.852    0.217    3.932    0.000    1.000    1.000
    textual           0.997    0.193    5.160    0.000    1.000    1.000
    speed             1.346    0.328    4.101    0.000    1.000    1.000
   .x1      (rsd1)    0.660    0.165    4.008    0.000    0.660    0.510
   .x2      (rsd2)    1.133    0.110   10.337    0.000    1.133    0.833
   .x3      (rsd3)    0.754    0.131    5.749    0.000    0.754    0.615
   .x4      (rsd4)    0.396    0.050    7.904    0.000    0.396    0.316
   .x5      (rsd5)    0.420    0.057    7.385    0.000    0.420    0.269
   .x6      (rsd6)    0.350    0.048    7.311    0.000    0.350    0.311
   .x7      (rsd7)    0.792    0.094    8.414    0.000    0.792    0.638
   .x8      (rsd8)    0.499    0.095    5.244    0.000    0.499    0.453
   .x9      (rsd9)    0.549    0.100    5.479    0.000    0.549    0.505

R-Square:
                   Estimate
    x1                0.490
    x2                0.167
    x3                0.385
    x4                0.684
    x5                0.731
    x6                0.689
    x7                0.362
    x8                0.547
    x9                0.495</code></pre>
</div>
<div id="measurementInvarianceResidualModelFitCriteria" class="section level4 hasAnchor" number="16.10.7.5">
<h4><span class="header-section-number">16.10.7.5</span> Model Fit<a href="bias.html#measurementInvarianceResidualModelFitCriteria" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>You can specify the null model as the baseline model using: <code>baseline.model = nullModelFit</code></p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1691"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1691-1"><a href="bias.html#cb1691-1" tabindex="-1"></a><span class="fu">fitMeasures</span>(residualInvarianceModel_fit)</span></code></pre></div>
</details>
<pre><code>                         npar                          fmin 
                       39.000                         0.287 
                        chisq                            df 
                      172.960                        69.000 
                       pvalue                  chisq.scaled 
                        0.000                       172.714 
                    df.scaled                 pvalue.scaled 
                       69.000                         0.000 
         chisq.scaling.factor                baseline.chisq 
                        1.001                       932.665 
                  baseline.df               baseline.pvalue 
                       72.000                         0.000 
        baseline.chisq.scaled            baseline.df.scaled 
                      909.019                        72.000 
       baseline.pvalue.scaled baseline.chisq.scaling.factor 
                        0.000                         1.026 
                          cfi                           tli 
                        0.879                         0.874 
                   cfi.scaled                    tli.scaled 
                        0.876                         0.871 
                   cfi.robust                    tli.robust 
                        0.883                         0.878 
                         nnfi                           rfi 
                        0.874                         0.806 
                          nfi                          pnfi 
                        0.815                         0.781 
                          ifi                           rni 
                        0.880                         0.879 
                  nnfi.scaled                    rfi.scaled 
                        0.871                         0.802 
                   nfi.scaled                   pnfi.scaled 
                        0.810                         0.776 
                   ifi.scaled                    rni.scaled 
                        0.877                         0.876 
                  nnfi.robust                    rni.robust 
                        0.878                         0.883 
                         logl             unrestricted.logl 
                    -3680.600                     -3594.120 
                          aic                           bic 
                     7439.201                      7583.778 
                       ntotal                          bic2 
                      301.000                      7460.092 
            scaling.factor.h1             scaling.factor.h0 
                        1.034                         0.645 
                        rmsea                rmsea.ci.lower 
                        0.100                         0.082 
               rmsea.ci.upper                rmsea.ci.level 
                        0.119                         0.900 
                 rmsea.pvalue                rmsea.close.h0 
                        0.000                         0.050 
        rmsea.notclose.pvalue             rmsea.notclose.h0 
                        0.962                         0.080 
                 rmsea.scaled         rmsea.ci.lower.scaled 
                        0.100                         0.081 
        rmsea.ci.upper.scaled           rmsea.pvalue.scaled 
                        0.119                         0.000 
 rmsea.notclose.pvalue.scaled                  rmsea.robust 
                        0.961                         0.100 
        rmsea.ci.lower.robust         rmsea.ci.upper.robust 
                        0.081                         0.119 
          rmsea.pvalue.robust  rmsea.notclose.pvalue.robust 
                        0.000                         0.955 
                          rmr                    rmr_nomean 
                        0.103                         0.102 
                         srmr                  srmr_bentler 
                        0.086                         0.086 
          srmr_bentler_nomean                          crmr 
                        0.083                         0.086 
                  crmr_nomean                    srmr_mplus 
                        0.083                         0.104 
            srmr_mplus_nomean                         cn_05 
                        0.084                       156.566 
                        cn_01                           gfi 
                      173.684                         0.992 
                         agfi                          pgfi 
                        0.988                         0.634 
                          mfi                          ecvi 
                        0.841                         0.834 </code></pre>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1693"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1693-1"><a href="bias.html#cb1693-1" tabindex="-1"></a>residualInvarianceModelFitIndices <span class="ot">&lt;-</span> <span class="fu">fitMeasures</span>(</span>
<span id="cb1693-2"><a href="bias.html#cb1693-2" tabindex="-1"></a>  residualInvarianceModel_fit)[<span class="fu">c</span>(</span>
<span id="cb1693-3"><a href="bias.html#cb1693-3" tabindex="-1"></a>    <span class="st">&quot;cfi.robust&quot;</span>, <span class="st">&quot;rmsea.robust&quot;</span>, <span class="st">&quot;srmr&quot;</span>)]</span>
<span id="cb1693-4"><a href="bias.html#cb1693-4" tabindex="-1"></a></span>
<span id="cb1693-5"><a href="bias.html#cb1693-5" tabindex="-1"></a>residualInvarianceModel_chisquare <span class="ot">&lt;-</span> <span class="fu">fitMeasures</span>(</span>
<span id="cb1693-6"><a href="bias.html#cb1693-6" tabindex="-1"></a>  residualInvarianceModel_fit)[<span class="fu">c</span>(<span class="st">&quot;chisq.scaled&quot;</span>)]</span>
<span id="cb1693-7"><a href="bias.html#cb1693-7" tabindex="-1"></a></span>
<span id="cb1693-8"><a href="bias.html#cb1693-8" tabindex="-1"></a>residualInvarianceModel_chisquareScaling <span class="ot">&lt;-</span> <span class="fu">fitMeasures</span>(</span>
<span id="cb1693-9"><a href="bias.html#cb1693-9" tabindex="-1"></a>  residualInvarianceModel_fit)[<span class="fu">c</span>(<span class="st">&quot;chisq.scaling.factor&quot;</span>)]</span>
<span id="cb1693-10"><a href="bias.html#cb1693-10" tabindex="-1"></a></span>
<span id="cb1693-11"><a href="bias.html#cb1693-11" tabindex="-1"></a>residualInvarianceModel_df <span class="ot">&lt;-</span> <span class="fu">fitMeasures</span>(</span>
<span id="cb1693-12"><a href="bias.html#cb1693-12" tabindex="-1"></a>  residualInvarianceModel_fit)[<span class="fu">c</span>(<span class="st">&quot;df.scaled&quot;</span>)]</span>
<span id="cb1693-13"><a href="bias.html#cb1693-13" tabindex="-1"></a></span>
<span id="cb1693-14"><a href="bias.html#cb1693-14" tabindex="-1"></a>residualInvarianceModel_N <span class="ot">&lt;-</span> <span class="fu">lavInspect</span>(</span>
<span id="cb1693-15"><a href="bias.html#cb1693-15" tabindex="-1"></a>  residualInvarianceModel_fit,</span>
<span id="cb1693-16"><a href="bias.html#cb1693-16" tabindex="-1"></a>  <span class="at">what =</span> <span class="st">&quot;ntotal&quot;</span>)</span></code></pre></div>
</details>
</div>
<div id="measurementInvarianceResidualModelCompare" class="section level4 hasAnchor" number="16.10.7.6">
<h4><span class="header-section-number">16.10.7.6</span> Compare Model Fit<a href="bias.html#measurementInvarianceResidualModelCompare" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<div id="measurementInvarianceResidualModelNestedModelComparison" class="section level5 hasAnchor" number="16.10.7.6.1">
<h5><span class="header-section-number">16.10.7.6.1</span> Nested Model (<span class="math inline">\(\chi^2\)</span>) Difference Test<a href="bias.html#measurementInvarianceResidualModelNestedModelComparison" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>The scalar invariance model and the residual invariance model are considered “nested” models.
The residual invariance model is nested within the scalar invariance model because the scalar invariance model includes all of the terms of the residual invariance model along with additional terms.
Model fit of nested models can be compared with a chi-square difference test, also known as a likelihood ratio test or deviance test.
A significant chi-square difference test would indicate that the simplified model with additional constraints (the residual invariance model) is significantly worse fitting than the more complex model that has fewer constraints (the scalar invariance model).</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1694"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1694-1"><a href="bias.html#cb1694-1" tabindex="-1"></a><span class="fu">anova</span>(scalarInvarianceModel_fit, residualInvarianceModel_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Df"],"name":[1],"type":["int"],"align":["right"]},{"label":["AIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Chisq"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Chisq diff"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Df diff"],"name":[6],"type":["int"],"align":["right"]},{"label":["Pr(>Chisq)"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"60","2":"7441.195","3":"7619.136","4":"156.9546","5":"NA","6":"NA","7":"NA","_rn_":"scalarInvarianceModel_fit"},{"1":"69","2":"7439.201","3":"7583.778","4":"172.9604","5":"14.35586","6":"9","7":"0.1102176","_rn_":"residualInvarianceModel_fit"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>In this instance, you do not need to examine whether residual invariance held because the preceding level of measurement invariance (scalar invariance) did not hold.</p>
<p>The <a href="https://github.com/DevPsyLab/petersenlab"><code>petersenlab</code></a> package <span class="citation">(<a href="#ref-R-petersenlab">Petersen, 2024b</a>)</span> contains the <code>satorraBentlerScaledChiSquareDifferenceTestStatistic()</code> function that performs a Satorra-Bentler scaled chi-square difference test:</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1695"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1695-1"><a href="bias.html#cb1695-1" tabindex="-1"></a>residualInvarianceModel_chisquareDiff <span class="ot">&lt;-</span> </span>
<span id="cb1695-2"><a href="bias.html#cb1695-2" tabindex="-1"></a>  <span class="fu">satorraBentlerScaledChiSquareDifferenceTestStatistic</span>(</span>
<span id="cb1695-3"><a href="bias.html#cb1695-3" tabindex="-1"></a>    <span class="at">T0 =</span> residualInvarianceModel_chisquare,</span>
<span id="cb1695-4"><a href="bias.html#cb1695-4" tabindex="-1"></a>    <span class="at">c0 =</span> residualInvarianceModel_chisquareScaling,</span>
<span id="cb1695-5"><a href="bias.html#cb1695-5" tabindex="-1"></a>    <span class="at">d0 =</span> residualInvarianceModel_df,</span>
<span id="cb1695-6"><a href="bias.html#cb1695-6" tabindex="-1"></a>    <span class="at">T1 =</span> scalarInvarianceModel_chisquare,</span>
<span id="cb1695-7"><a href="bias.html#cb1695-7" tabindex="-1"></a>    <span class="at">c1 =</span> scalarInvarianceModel_chisquareScaling,</span>
<span id="cb1695-8"><a href="bias.html#cb1695-8" tabindex="-1"></a>    <span class="at">d1 =</span> scalarInvarianceModel_df)</span>
<span id="cb1695-9"><a href="bias.html#cb1695-9" tabindex="-1"></a></span>
<span id="cb1695-10"><a href="bias.html#cb1695-10" tabindex="-1"></a>residualInvarianceModel_chisquareDiff</span></code></pre></div>
</details>
<pre><code>chisq.scaled 
    14.35586 </code></pre>
</div>
<div id="measurementInvarianceResidualModelCompareFitCriteria" class="section level5 hasAnchor" number="16.10.7.6.2">
<h5><span class="header-section-number">16.10.7.6.2</span> Compare Other Fit Criteria<a href="bias.html#measurementInvarianceResidualModelCompareFitCriteria" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1697"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1697-1"><a href="bias.html#cb1697-1" tabindex="-1"></a><span class="fu">round</span>(residualInvarianceModelFitIndices <span class="sc">-</span> scalarInvarianceModelFitIndices,</span>
<span id="cb1697-2"><a href="bias.html#cb1697-2" tabindex="-1"></a>      <span class="at">digits =</span> <span class="dv">3</span>)</span></code></pre></div>
</details>
<pre><code>  cfi.robust rmsea.robust         srmr 
      -0.007       -0.004        0.006 </code></pre>
</div>
<div id="measurementInvarianceResidualModelScoreBasedTest" class="section level5 hasAnchor" number="16.10.7.6.3">
<h5><span class="header-section-number">16.10.7.6.3</span> Score-Based Test<a href="bias.html#measurementInvarianceResidualModelScoreBasedTest" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Score-based tests of measurement invariance are implemented using the <code>strucchange</code> package and are described by <span class="citation">T. Wang et al. (<a href="#ref-Wang2014a">2014</a>)</span>.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1699"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1699-1"><a href="bias.html#cb1699-1" tabindex="-1"></a><span class="fu">coef</span>(residualInvarianceModel_fit)</span></code></pre></div>
</details>
<pre><code>lambda.1_1 lambda.2_1 lambda.3_1 lambda.4_2 lambda.5_2 lambda.6_2 lambda.7_3 
     0.862      0.516      0.745      0.928      1.069      0.882      0.577 
lambda.8_3 lambda.9_3       nu.1       nu.2       nu.3       nu.4       nu.5 
     0.669      0.632      5.025      6.138      2.332      2.784      4.030 
      nu.6       nu.7       nu.8       nu.9  theta.1_1  theta.2_2  theta.3_3 
     1.928      4.270      5.619      5.470      0.660      1.133      0.754 
 theta.4_4  theta.5_5  theta.6_6  theta.7_7  theta.8_8  theta.9_9 psi.2_1.g1 
     0.396      0.420      0.350      0.792      0.499      0.549      0.430 
psi.3_1.g1 psi.3_2.g1 lambda.1_1 lambda.2_1 lambda.3_1 lambda.4_2 lambda.5_2 
     0.351      0.302      0.862      0.516      0.745      0.928      1.069 
lambda.6_2 lambda.7_3 lambda.8_3 lambda.9_3       nu.1       nu.2       nu.3 
     0.882      0.577      0.669      0.632      5.025      6.138      2.332 
      nu.4       nu.5       nu.6       nu.7       nu.8       nu.9  theta.1_1 
     2.784      4.030      1.928      4.270      5.619      5.470      0.660 
 theta.2_2  theta.3_3  theta.4_4  theta.5_5  theta.6_6  theta.7_7  theta.8_8 
     1.133      0.754      0.396      0.420      0.350      0.792      0.499 
 theta.9_9 alpha.1.g2 alpha.2.g2 alpha.3.g2 psi.1_1.g2 psi.2_2.g2 psi.3_3.g2 
     0.549     -0.208      0.611     -0.283      0.852      0.997      1.346 
psi.2_1.g2 psi.3_1.g2 psi.3_2.g2 
     0.519      0.643      0.426 </code></pre>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1701"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1701-1"><a href="bias.html#cb1701-1" tabindex="-1"></a><span class="fu">coef</span>(residualInvarianceModel_fit)[<span class="dv">19</span><span class="sc">:</span><span class="dv">27</span>]</span></code></pre></div>
</details>
<pre><code>theta.1_1 theta.2_2 theta.3_3 theta.4_4 theta.5_5 theta.6_6 theta.7_7 theta.8_8 
0.6598353 1.1333770 0.7536794 0.3957809 0.4196626 0.3504637 0.7916721 0.4987961 
theta.9_9 
0.5490913 </code></pre>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1703"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1703-1"><a href="bias.html#cb1703-1" tabindex="-1"></a>strucchange<span class="sc">::</span><span class="fu">sctest</span>(</span>
<span id="cb1703-2"><a href="bias.html#cb1703-2" tabindex="-1"></a>  residualInvarianceModel_fit,</span>
<span id="cb1703-3"><a href="bias.html#cb1703-3" tabindex="-1"></a>  <span class="at">order.by =</span> HolzingerSwineford1939<span class="sc">$</span>school,</span>
<span id="cb1703-4"><a href="bias.html#cb1703-4" tabindex="-1"></a>  <span class="at">parm =</span> <span class="dv">19</span><span class="sc">:</span><span class="dv">27</span>,</span>
<span id="cb1703-5"><a href="bias.html#cb1703-5" tabindex="-1"></a>  <span class="at">functional =</span> <span class="st">&quot;LMuo&quot;</span>)</span></code></pre></div>
</details>
<pre><code>
    M-fluctuation test

data:  residualInvarianceModel_fit
f(efp) = 10.64, p-value = 0.3012</code></pre>
<p>A score-based test and expected parameter change (EPC) estimates <span class="citation">(<a href="#ref-Oberski2014">Oberski, 2014</a>; <a href="#ref-Oberski2015">Oberski et al., 2015</a>)</span> are provided by the <code>lavaan</code> package <span class="citation">(<a href="#ref-R-lavaan">Rosseel et al., 2022</a>)</span>.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1705"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1705-1"><a href="bias.html#cb1705-1" tabindex="-1"></a><span class="fu">lavTestScore</span>(</span>
<span id="cb1705-2"><a href="bias.html#cb1705-2" tabindex="-1"></a>  residualInvarianceModel_fit,</span>
<span id="cb1705-3"><a href="bias.html#cb1705-3" tabindex="-1"></a>  <span class="at">epc =</span> <span class="cn">TRUE</span></span>
<span id="cb1705-4"><a href="bias.html#cb1705-4" tabindex="-1"></a>)</span></code></pre></div>
</details>
<pre><code>$test

total score test:

   test     X2 df p.value
1 score 60.136 27       0

$uni

univariate score tests:

     lhs op   rhs     X2 df p.value
1   .p1. == .p37.  0.011  1   0.915
2   .p2. == .p38.  0.043  1   0.836
3   .p3. == .p39.  0.062  1   0.803
4   .p4. == .p40.  0.089  1   0.766
5   .p5. == .p41.  4.083  1   0.043
6   .p6. == .p42.  5.815  1   0.016
7   .p7. == .p43.  0.026  1   0.872
8   .p8. == .p44.  0.003  1   0.954
9   .p9. == .p45.  0.006  1   0.939
10 .p10. == .p46.  6.331  1   0.012
11 .p11. == .p47.  7.812  1   0.005
12 .p12. == .p48. 20.213  1   0.000
13 .p13. == .p49.  1.231  1   0.267
14 .p14. == .p50.  1.155  1   0.283
15 .p15. == .p51.  0.001  1   0.982
16 .p16. == .p52. 14.986  1   0.000
17 .p17. == .p53.  3.574  1   0.059
18 .p18. == .p54.  1.735  1   0.188
19 .p19. == .p55.  0.055  1   0.815
20 .p20. == .p56.  2.026  1   0.155
21 .p21. == .p57.  1.867  1   0.172
22 .p22. == .p58.  0.754  1   0.385
23 .p23. == .p59.  0.619  1   0.432
24 .p24. == .p60.  3.852  1   0.050
25 .p25. == .p61.  2.273  1   0.132
26 .p26. == .p62.  0.674  1   0.412
27 .p27. == .p63.  1.954  1   0.162

$epc

expected parameter changes (epc) and expected parameter values (epv):

       lhs op     rhs block group free      label plabel    est    epc    epv
1   visual =~      x1     1     1    1 lambda.1_1   .p1.  0.862  0.083  0.945
2   visual =~      x2     1     1    2 lambda.2_1   .p2.  0.516 -0.017  0.499
3   visual =~      x3     1     1    3 lambda.3_1   .p3.  0.745 -0.095  0.650
4  textual =~      x4     1     1    4 lambda.4_2   .p4.  0.928 -0.006  0.922
5  textual =~      x5     1     1    5 lambda.5_2   .p5.  1.069  0.060  1.130
6  textual =~      x6     1     1    6 lambda.6_2   .p6.  0.882 -0.054  0.828
7    speed =~      x7     1     1    7 lambda.7_3   .p7.  0.577 -0.036  0.541
8    speed =~      x8     1     1    8 lambda.8_3   .p8.  0.669  0.006  0.676
9    speed =~      x9     1     1    9 lambda.9_3   .p9.  0.632 -0.028  0.604
10      x1 ~1             1     1   10       nu.1  .p10.  5.025 -0.076  4.949
11      x2 ~1             1     1   11       nu.2  .p11.  6.138 -0.154  5.984
12      x3 ~1             1     1   12       nu.3  .p12.  2.332  0.177  2.509
13      x4 ~1             1     1   13       nu.4  .p13.  2.784  0.030  2.814
14      x5 ~1             1     1   14       nu.5  .p14.  4.030 -0.029  4.002
15      x6 ~1             1     1   15       nu.6  .p15.  1.928  0.001  1.929
16      x7 ~1             1     1   16       nu.7  .p16.  4.270  0.168  4.439
17      x8 ~1             1     1   17       nu.8  .p17.  5.619 -0.055  5.564
18      x9 ~1             1     1   18       nu.9  .p18.  5.470 -0.043  5.427
19      x1 ~~      x1     1     1   19  theta.1_1  .p19.  0.660 -0.145  0.514
20      x2 ~~      x2     1     1   20  theta.2_2  .p20.  1.133  0.145  1.279
21      x3 ~~      x3     1     1   21  theta.3_3  .p21.  0.754  0.194  0.947
22      x4 ~~      x4     1     1   22  theta.4_4  .p22.  0.396  0.046  0.442
23      x5 ~~      x5     1     1   23  theta.5_5  .p23.  0.420  0.009  0.428
24      x6 ~~      x6     1     1   24  theta.6_6  .p24.  0.350 -0.059  0.291
25      x7 ~~      x7     1     1   25  theta.7_7  .p25.  0.792  0.115  0.906
26      x8 ~~      x8     1     1   26  theta.8_8  .p26.  0.499  0.016  0.515
27      x9 ~~      x9     1     1   27  theta.9_9  .p27.  0.549  0.084  0.633
28  visual ~1             1     1    0 alpha.1.g1  .p28.  0.000     NA     NA
29 textual ~1             1     1    0 alpha.2.g1  .p29.  0.000     NA     NA
30   speed ~1             1     1    0 alpha.3.g1  .p30.  0.000     NA     NA
31  visual ~~  visual     1     1    0 psi.1_1.g1  .p31.  1.000     NA     NA
32 textual ~~ textual     1     1    0 psi.2_2.g1  .p32.  1.000     NA     NA
33   speed ~~   speed     1     1    0 psi.3_3.g1  .p33.  1.000     NA     NA
34  visual ~~ textual     1     1   28 psi.2_1.g1  .p34.  0.430 -0.002  0.428
35  visual ~~   speed     1     1   29 psi.3_1.g1  .p35.  0.351  0.007  0.358
36 textual ~~   speed     1     1   30 psi.3_2.g1  .p36.  0.302  0.007  0.309
37  visual =~      x1     2     2   31 lambda.1_1  .p37.  0.862  0.000  0.862
38  visual =~      x2     2     2   32 lambda.2_1  .p38.  0.516  0.063  0.579
39  visual =~      x3     2     2   33 lambda.3_1  .p39.  0.745 -0.024  0.721
40 textual =~      x4     2     2   34 lambda.4_2  .p40.  0.928  0.036  0.963
41 textual =~      x5     2     2   35 lambda.5_2  .p41.  1.069 -0.091  0.978
42 textual =~      x6     2     2   36 lambda.6_2  .p42.  0.882  0.061  0.944
43   speed =~      x7     2     2   37 lambda.7_3  .p43.  0.577 -0.035  0.542
44   speed =~      x8     2     2   38 lambda.8_3  .p44.  0.669  0.020  0.689
45   speed =~      x9     2     2   39 lambda.9_3  .p45.  0.632  0.033  0.665
46      x1 ~1             2     2   40       nu.1  .p46.  5.025  0.075  5.100
47      x2 ~1             2     2   41       nu.2  .p47.  6.138  0.176  6.314
48      x3 ~1             2     2   42       nu.3  .p48.  2.332 -0.197  2.135
49      x4 ~1             2     2   43       nu.4  .p49.  2.784 -0.057  2.726
50      x5 ~1             2     2   44       nu.5  .p50.  4.030  0.085  4.115
51      x6 ~1             2     2   45       nu.6  .p51.  1.928 -0.040  1.888
52      x7 ~1             2     2   46       nu.7  .p52.  4.270 -0.185  4.086
53      x8 ~1             2     2   47       nu.8  .p53.  5.619  0.074  5.694
54      x9 ~1             2     2   48       nu.9  .p54.  5.470  0.065  5.535
55      x1 ~~      x1     2     2   49  theta.1_1  .p55.  0.660  0.021  0.681
56      x2 ~~      x2     2     2   50  theta.2_2  .p56.  1.133 -0.162  0.971
57      x3 ~~      x3     2     2   51  theta.3_3  .p57.  0.754 -0.091  0.663
58      x4 ~~      x4     2     2   52  theta.4_4  .p58.  0.396 -0.064  0.332
59      x5 ~~      x5     2     2   53  theta.5_5  .p59.  0.420  0.009  0.428
60      x6 ~~      x6     2     2   54  theta.6_6  .p60.  0.350  0.062  0.412
61      x7 ~~      x7     2     2   55  theta.7_7  .p61.  0.792 -0.088  0.704
62      x8 ~~      x8     2     2   56  theta.8_8  .p62.  0.499 -0.044  0.455
63      x9 ~~      x9     2     2   57  theta.9_9  .p63.  0.549 -0.095  0.454
64  visual ~1             2     2   58 alpha.1.g2  .p64. -0.208  0.009 -0.200
65 textual ~1             2     2   59 alpha.2.g2  .p65.  0.611  0.002  0.614
66   speed ~1             2     2   60 alpha.3.g2  .p66. -0.283 -0.016 -0.298
67  visual ~~  visual     2     2   61 psi.1_1.g2  .p67.  0.852  0.016  0.868
68 textual ~~ textual     2     2   62 psi.2_2.g2  .p68.  0.997 -0.004  0.993
69   speed ~~   speed     2     2   63 psi.3_3.g2  .p69.  1.346  0.008  1.354
70  visual ~~ textual     2     2   64 psi.2_1.g2  .p70.  0.519 -0.002  0.517
71  visual ~~   speed     2     2   65 psi.3_1.g2  .p71.  0.643 -0.014  0.629
72 textual ~~   speed     2     2   66 psi.3_2.g2  .p72.  0.426 -0.009  0.417
   sepc.lv sepc.all sepc.nox
1    0.083    0.070    0.070
2   -0.017   -0.014   -0.014
3   -0.095   -0.083   -0.083
4   -0.006   -0.005   -0.005
5    0.060    0.048    0.048
6   -0.054   -0.051   -0.051
7   -0.036   -0.034   -0.034
8    0.006    0.006    0.006
9   -0.028   -0.029   -0.029
10  -0.076   -0.065   -0.065
11  -0.154   -0.130   -0.130
12   0.177    0.155    0.155
13   0.030    0.027    0.027
14  -0.029   -0.023   -0.023
15   0.001    0.001    0.001
16   0.168    0.159    0.159
17  -0.055   -0.057   -0.057
18  -0.043   -0.044   -0.044
19  -0.660   -0.470   -0.470
20   1.133    0.810    0.810
21   0.754    0.576    0.576
22   0.396    0.315    0.315
23   0.420    0.269    0.269
24  -0.350   -0.310   -0.310
25   0.792    0.704    0.704
26   0.499    0.527    0.527
27   0.549    0.579    0.579
28      NA       NA       NA
29      NA       NA       NA
30      NA       NA       NA
31      NA       NA       NA
32      NA       NA       NA
33      NA       NA       NA
34  -0.002   -0.002   -0.002
35   0.007    0.007    0.007
36   0.007    0.007    0.007
37   0.000    0.000    0.000
38   0.058    0.050    0.050
39  -0.022   -0.020   -0.020
40   0.036    0.032    0.032
41  -0.091   -0.073   -0.073
42   0.061    0.058    0.058
43  -0.041   -0.037   -0.037
44   0.023    0.022    0.022
45   0.038    0.037    0.037
46   0.075    0.066    0.066
47   0.176    0.151    0.151
48  -0.197   -0.178   -0.178
49  -0.057   -0.051   -0.051
50   0.085    0.068    0.068
51  -0.040   -0.038   -0.038
52  -0.185   -0.166   -0.166
53   0.074    0.071    0.071
54   0.065    0.063    0.063
55   0.660    0.510    0.510
56  -1.133   -0.833   -0.833
57  -0.754   -0.615   -0.615
58  -0.396   -0.316   -0.316
59   0.420    0.269    0.269
60   0.350    0.311    0.311
61  -0.792   -0.638   -0.638
62  -0.499   -0.453   -0.453
63  -0.549   -0.505   -0.505
64   0.009    0.009    0.009
65   0.002    0.002    0.002
66  -0.013   -0.013   -0.013
67   1.000    1.000    1.000
68  -1.000   -1.000   -1.000
69   1.000    1.000    1.000
70  -0.002   -0.002   -0.002
71  -0.013   -0.013   -0.013
72  -0.007   -0.007   -0.007</code></pre>
</div>
<div id="measurementInvarianceResidualModelEquivalenceTest" class="section level5 hasAnchor" number="16.10.7.6.4">
<h5><span class="header-section-number">16.10.7.6.4</span> Equivalence Test<a href="bias.html#measurementInvarianceResidualModelEquivalenceTest" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>The <a href="https://github.com/DevPsyLab/petersenlab"><code>petersenlab</code></a> package <span class="citation">(<a href="#ref-R-petersenlab">Petersen, 2024b</a>)</span> contains the <code>equiv_chi()</code> function from <span class="citation">Counsell et al. (<a href="#ref-Counsell2020">2020</a>)</span> that performs an equivalence test: <a href="https://osf.io/cqu8v" class="uri">https://osf.io/cqu8v</a>.
The chi-square equivalence test is non-significant, suggesting that the model fit is not acceptable.
The equivalence test of the chi-square difference test is non-significant, suggesting that the degree of worsening of model fit is not acceptable.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1707"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1707-1"><a href="bias.html#cb1707-1" tabindex="-1"></a><span class="fu">equiv_chi</span>(</span>
<span id="cb1707-2"><a href="bias.html#cb1707-2" tabindex="-1"></a>  <span class="at">alpha =</span> .<span class="dv">05</span>,</span>
<span id="cb1707-3"><a href="bias.html#cb1707-3" tabindex="-1"></a>  <span class="at">chi =</span> residualInvarianceModel_chisquare,</span>
<span id="cb1707-4"><a href="bias.html#cb1707-4" tabindex="-1"></a>  <span class="at">df =</span> residualInvarianceModel_df,</span>
<span id="cb1707-5"><a href="bias.html#cb1707-5" tabindex="-1"></a>  <span class="at">m =</span> <span class="dv">2</span>,</span>
<span id="cb1707-6"><a href="bias.html#cb1707-6" tabindex="-1"></a>  <span class="at">N_sample =</span> residualInvarianceModel_N,</span>
<span id="cb1707-7"><a href="bias.html#cb1707-7" tabindex="-1"></a>  <span class="at">popRMSEA =</span> .<span class="dv">08</span>)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["chi"],"name":[1],"type":["dbl"],"align":["right"]},{"label":["Fml"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["popep"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["popdelt"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["pval"],"name":[5],"type":["dbl"],"align":["right"]}],"data":[{"1":"172.7138","2":"0.5776383","3":"0.2208","4":"66.0192","5":"0.9629861","_rn_":"chisq.scaled"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>Moreover, the equivalence test of the chi-square difference test is non-significant, suggesting that the degree of worsening of model fit is not acceptable.
In other words, residual invariance failed.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1708"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1708-1"><a href="bias.html#cb1708-1" tabindex="-1"></a>residualInvarianceModel_dfDiff <span class="ot">&lt;-</span> </span>
<span id="cb1708-2"><a href="bias.html#cb1708-2" tabindex="-1"></a>  residualInvarianceModel_df <span class="sc">-</span> scalarInvarianceModel_df</span>
<span id="cb1708-3"><a href="bias.html#cb1708-3" tabindex="-1"></a></span>
<span id="cb1708-4"><a href="bias.html#cb1708-4" tabindex="-1"></a><span class="fu">equiv_chi</span>(</span>
<span id="cb1708-5"><a href="bias.html#cb1708-5" tabindex="-1"></a>  <span class="at">alpha =</span> .<span class="dv">05</span>,</span>
<span id="cb1708-6"><a href="bias.html#cb1708-6" tabindex="-1"></a>  <span class="at">chi =</span> residualInvarianceModel_chisquareDiff,</span>
<span id="cb1708-7"><a href="bias.html#cb1708-7" tabindex="-1"></a>  <span class="at">df =</span> residualInvarianceModel_dfDiff,</span>
<span id="cb1708-8"><a href="bias.html#cb1708-8" tabindex="-1"></a>  <span class="at">m =</span> <span class="dv">2</span>,</span>
<span id="cb1708-9"><a href="bias.html#cb1708-9" tabindex="-1"></a>  <span class="at">N_sample =</span> residualInvarianceModel_N,</span>
<span id="cb1708-10"><a href="bias.html#cb1708-10" tabindex="-1"></a>  <span class="at">popRMSEA =</span> .<span class="dv">08</span>)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["chi"],"name":[1],"type":["dbl"],"align":["right"]},{"label":["Fml"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["popep"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["popdelt"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["pval"],"name":[5],"type":["dbl"],"align":["right"]}],"data":[{"1":"14.35586","2":"0.04801292","3":"0.0288","4":"8.6112","5":"0.3636516","_rn_":"chisq.scaled"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceResidualModelPermutationTest" class="section level5 hasAnchor" number="16.10.7.6.5">
<h5><span class="header-section-number">16.10.7.6.5</span> Permutation Test<a href="bias.html#measurementInvarianceResidualModelPermutationTest" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Permutation procedures for testing measurement invariance are described in <span class="citation">(<a href="#ref-Jorgensen2018">Jorgensen et al., 2018</a>)</span>.
For reproducibility, I set the seed below.
Using the same seed will yield the same answer every time.
There is nothing special about this particular seed.
You can specify the null model as the baseline model using: <code>baseline.model = nullModelFit</code>.</p>
<p><strong>Warning</strong>: this code takes a while to run based on <span class="math inline">\(100\)</span> iterations.
You can reduce the number of iterations to be faster.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1709"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1709-1"><a href="bias.html#cb1709-1" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">52242</span>)</span>
<span id="cb1709-2"><a href="bias.html#cb1709-2" tabindex="-1"></a>residualInvarianceTest <span class="ot">&lt;-</span> <span class="fu">permuteMeasEq</span>(</span>
<span id="cb1709-3"><a href="bias.html#cb1709-3" tabindex="-1"></a>  <span class="at">nPermute =</span> numPermutations,</span>
<span id="cb1709-4"><a href="bias.html#cb1709-4" tabindex="-1"></a>  <span class="at">modelType =</span> <span class="st">&quot;mgcfa&quot;</span>,</span>
<span id="cb1709-5"><a href="bias.html#cb1709-5" tabindex="-1"></a>  <span class="at">con =</span> residualInvarianceModel_fit,</span>
<span id="cb1709-6"><a href="bias.html#cb1709-6" tabindex="-1"></a>  <span class="at">uncon =</span> scalarInvarianceModel_fit,</span>
<span id="cb1709-7"><a href="bias.html#cb1709-7" tabindex="-1"></a>  <span class="at">AFIs =</span> myAFIs,</span>
<span id="cb1709-8"><a href="bias.html#cb1709-8" tabindex="-1"></a>  <span class="at">moreAFIs =</span> moreAFIs,</span>
<span id="cb1709-9"><a href="bias.html#cb1709-9" tabindex="-1"></a>  <span class="at">parallelType =</span> <span class="st">&quot;multicore&quot;</span>, <span class="co">#only &#39;snow&#39; works on Windows, but right now, it is throwing an error</span></span>
<span id="cb1709-10"><a href="bias.html#cb1709-10" tabindex="-1"></a>  <span class="at">iseed =</span> <span class="dv">52242</span>)</span></code></pre></div>
</details>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1710"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1710-1"><a href="bias.html#cb1710-1" tabindex="-1"></a><span class="fu">RNGkind</span>(<span class="st">&quot;default&quot;</span>, <span class="st">&quot;default&quot;</span>, <span class="st">&quot;default&quot;</span>)</span>
<span id="cb1710-2"><a href="bias.html#cb1710-2" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">52242</span>)</span>
<span id="cb1710-3"><a href="bias.html#cb1710-3" tabindex="-1"></a></span>
<span id="cb1710-4"><a href="bias.html#cb1710-4" tabindex="-1"></a>residualInvarianceTest</span></code></pre></div>
</details>
<pre><code>Omnibus p value based on parametric chi-squared difference test:

Chisq diff    Df diff Pr(&gt;Chisq) 
    14.356      9.000      0.110 


Omnibus p values based on nonparametric permutation method: 

             AFI.Difference p.value
chisq                16.006    0.15
chisq.scaled         13.272    0.24
rmsea                -0.004    0.38
cfi                  -0.008    0.15
tli                   0.009    0.52
srmr                  0.006    0.20
rmsea.robust         -0.004    0.38
cfi.robust           -0.007    0.15
tli.robust            0.010    0.50</code></pre>
<p>The <em>p</em>-values are non-significant, indicating that the model did not fit significantly worse than the scalar invariance model.</p>
</div>
</div>
<div id="measurementInvarianceResidualModelReliability" class="section level4 hasAnchor" number="16.10.7.7">
<h4><span class="header-section-number">16.10.7.7</span> Internal Consistency Reliability<a href="bias.html#measurementInvarianceResidualModelReliability" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><a href="reliability.html#internalConsistency-reliability">Internal consistency reliability</a> of items composing the latent factors, as quantified by <a href="reliability.html#coefficientOmega">omega (<span class="math inline">\(\omega\)</span>)</a> and <a href="reliability.html#averageVarianceExtracted">average variance extracted</a> (AVE), was estimated using the <code>semTools</code> package <span class="citation">(<a href="#ref-R-semTools">Jorgensen et al., 2021</a>)</span>.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1712"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1712-1"><a href="bias.html#cb1712-1" tabindex="-1"></a><span class="fu">compRelSEM</span>(residualInvarianceModel_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["group"],"name":[1],"type":["chr"],"align":["left"]},{"label":["visual"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["textual"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["speed"],"name":[4],"type":["dbl"],"align":["right"]}],"data":[{"1":"Pasteur","2":"0.6253136","3":"0.8730352","4":"0.6590115"},{"1":"Grant-White","2":"0.5938595","3":"0.8754604","4":"0.7310116"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1713"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1713-1"><a href="bias.html#cb1713-1" tabindex="-1"></a><span class="fu">AVE</span>(residualInvarianceModel_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["group"],"name":[1],"type":["chr"],"align":["left"]},{"label":["visual"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["textual"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["speed"],"name":[4],"type":["dbl"],"align":["right"]}],"data":[{"1":"Pasteur","2":"0.3696721","3":"0.6992522","4":"0.3770485"},{"1":"Grant-White","2":"0.3671495","3":"0.7090124","4":"0.4947666"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceResidualModelPathDiagram" class="section level4 hasAnchor" number="16.10.7.8">
<h4><span class="header-section-number">16.10.7.8</span> Path Diagram<a href="bias.html#measurementInvarianceResidualModelPathDiagram" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>A path diagram of the model generated using the <code>semPlot</code> package <span class="citation">(<a href="#ref-R-semPlot">Epskamp, 2022</a>)</span> is below.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1714"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1714-1"><a href="bias.html#cb1714-1" tabindex="-1"></a><span class="fu">semPaths</span>(</span>
<span id="cb1714-2"><a href="bias.html#cb1714-2" tabindex="-1"></a>  residualInvarianceModel_fit,</span>
<span id="cb1714-3"><a href="bias.html#cb1714-3" tabindex="-1"></a>  <span class="at">what =</span> <span class="st">&quot;est&quot;</span>,</span>
<span id="cb1714-4"><a href="bias.html#cb1714-4" tabindex="-1"></a>  <span class="at">layout =</span> <span class="st">&quot;tree2&quot;</span>,</span>
<span id="cb1714-5"><a href="bias.html#cb1714-5" tabindex="-1"></a>  <span class="at">ask =</span> <span class="cn">FALSE</span>,</span>
<span id="cb1714-6"><a href="bias.html#cb1714-6" tabindex="-1"></a>  <span class="at">edge.label.cex =</span> <span class="fl">1.2</span>)</span></code></pre></div>
</details>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:measurementInvarianceResidualModelPathDiagram-1"></span>
<img src="Book_files/figure-html/measurementInvarianceResidualModelPathDiagram-1.png" alt="Residual Invariance Model in Confirmatory Factor Analysis." width="100%" />
<p class="caption">
Figure 16.59: Residual Invariance Model in Confirmatory Factor Analysis.
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:measurementInvarianceResidualModelPathDiagram-2"></span>
<img src="Book_files/figure-html/measurementInvarianceResidualModelPathDiagram-2.png" alt="Residual Invariance Model in Confirmatory Factor Analysis." width="100%" />
<p class="caption">
Figure 16.60: Residual Invariance Model in Confirmatory Factor Analysis.
</p>
</div>
</div>
</div>
<div id="addressingMeasurementNonInvariance" class="section level3 hasAnchor" number="16.10.8">
<h3><span class="header-section-number">16.10.8</span> Addressing Measurement Non-Invariance<a href="bias.html#addressingMeasurementNonInvariance" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>In the example above, we detected measurement non-invariance of intercepts, but not factor loadings.
Thus, we would want to identify which item(s) show non-invariant intercepts across groups.
When detecting measurement non-invariance in a given parameter (e.g., factor loadings, intercepts, or residuals), one can identify the specific items that show measurement non-invariance in one of two primary ways: (1) starting with a model that allows the given parameter to differ across groups, a researcher can iteratively add constraints to identify the item(s) for which measurement invariance fails, or (2) starting with a model that constrains the given parameter to be the same across groups, a researcher can iteratively remove constraints to identify the item(s) for which measurement invariance becomes established (and by process of elimination, the items for which measurement invariance does not become established).</p>
<div id="measurementInvarianceAddConstraints" class="section level4 hasAnchor" number="16.10.8.1">
<h4><span class="header-section-number">16.10.8.1</span> Iteratively add constraints to identify measurement non-invariance<a href="bias.html#measurementInvarianceAddConstraints" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>In the example above, we detected measurement non-invariance of intercepts.
One approach to identify measurement non-invariance is to iteratively add constraints to a model that allows the given parameter (intercepts) to differ across groups.
So, we will use the metric invariance model as the baseline model (in which item intercepts are allowed to differ across groups), and iteratively add constraints to identify which item(s) show non-invariant intercepts across groups.</p>
<div id="measurementInvarianceAddConstraintsVariable1" class="section level5 hasAnchor" number="16.10.8.1.1">
<h5><span class="header-section-number">16.10.8.1.1</span> Variable 1<a href="bias.html#measurementInvarianceAddConstraintsVariable1" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Variable 1 does not have non-invariant intercepts.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1715"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1715-1"><a href="bias.html#cb1715-1" tabindex="-1"></a>cfaModel_metricInvarianceV1 <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1715-2"><a href="bias.html#cb1715-2" tabindex="-1"></a><span class="st"> #Fix factor loadings to be the same across groups</span></span>
<span id="cb1715-3"><a href="bias.html#cb1715-3" tabindex="-1"></a><span class="st"> visual  =~ c(lambdax1,lambdax1)*x1 + c(lambdax2,lambdax2)*x2 + c(lambdax3,lambdax3)*x3</span></span>
<span id="cb1715-4"><a href="bias.html#cb1715-4" tabindex="-1"></a><span class="st"> textual =~ c(lambdax4,lambdax4)*x4 + c(lambdax5,lambdax5)*x5 + c(lambdax6,lambdax6)*x6</span></span>
<span id="cb1715-5"><a href="bias.html#cb1715-5" tabindex="-1"></a><span class="st"> speed   =~ c(lambdax7,lambdax7)*x7 + c(lambdax8,lambdax8)*x8 + c(lambdax9,lambdax9)*x9</span></span>
<span id="cb1715-6"><a href="bias.html#cb1715-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1715-7"><a href="bias.html#cb1715-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero</span></span>
<span id="cb1715-8"><a href="bias.html#cb1715-8" tabindex="-1"></a><span class="st"> visual ~ 0</span></span>
<span id="cb1715-9"><a href="bias.html#cb1715-9" tabindex="-1"></a><span class="st"> textual ~ 0</span></span>
<span id="cb1715-10"><a href="bias.html#cb1715-10" tabindex="-1"></a><span class="st"> speed ~ 0</span></span>
<span id="cb1715-11"><a href="bias.html#cb1715-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1715-12"><a href="bias.html#cb1715-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one in group 1; free latent variances in group 2</span></span>
<span id="cb1715-13"><a href="bias.html#cb1715-13" tabindex="-1"></a><span class="st"> visual ~~ c(1, NA)*visual</span></span>
<span id="cb1715-14"><a href="bias.html#cb1715-14" tabindex="-1"></a><span class="st"> textual ~~ c(1, NA)*textual</span></span>
<span id="cb1715-15"><a href="bias.html#cb1715-15" tabindex="-1"></a><span class="st"> speed ~~ c(1, NA)*speed</span></span>
<span id="cb1715-16"><a href="bias.html#cb1715-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1715-17"><a href="bias.html#cb1715-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1715-18"><a href="bias.html#cb1715-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1715-19"><a href="bias.html#cb1715-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1715-20"><a href="bias.html#cb1715-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1715-21"><a href="bias.html#cb1715-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1715-22"><a href="bias.html#cb1715-22" tabindex="-1"></a><span class="st"> #Estimate residual variances of manifest variables</span></span>
<span id="cb1715-23"><a href="bias.html#cb1715-23" tabindex="-1"></a><span class="st"> x1 ~~ x1</span></span>
<span id="cb1715-24"><a href="bias.html#cb1715-24" tabindex="-1"></a><span class="st"> x2 ~~ x2</span></span>
<span id="cb1715-25"><a href="bias.html#cb1715-25" tabindex="-1"></a><span class="st"> x3 ~~ x3</span></span>
<span id="cb1715-26"><a href="bias.html#cb1715-26" tabindex="-1"></a><span class="st"> x4 ~~ x4</span></span>
<span id="cb1715-27"><a href="bias.html#cb1715-27" tabindex="-1"></a><span class="st"> x5 ~~ x5</span></span>
<span id="cb1715-28"><a href="bias.html#cb1715-28" tabindex="-1"></a><span class="st"> x6 ~~ x6</span></span>
<span id="cb1715-29"><a href="bias.html#cb1715-29" tabindex="-1"></a><span class="st"> x7 ~~ x7</span></span>
<span id="cb1715-30"><a href="bias.html#cb1715-30" tabindex="-1"></a><span class="st"> x8 ~~ x8</span></span>
<span id="cb1715-31"><a href="bias.html#cb1715-31" tabindex="-1"></a><span class="st"> x9 ~~ x9</span></span>
<span id="cb1715-32"><a href="bias.html#cb1715-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1715-33"><a href="bias.html#cb1715-33" tabindex="-1"></a><span class="st"> #Iteratively fix intercepts of manifest variables across groups</span></span>
<span id="cb1715-34"><a href="bias.html#cb1715-34" tabindex="-1"></a><span class="st"> x1 ~ c(intx1, intx1)*1</span></span>
<span id="cb1715-35"><a href="bias.html#cb1715-35" tabindex="-1"></a><span class="st"> x2 ~ NA*1</span></span>
<span id="cb1715-36"><a href="bias.html#cb1715-36" tabindex="-1"></a><span class="st"> x3 ~ NA*1</span></span>
<span id="cb1715-37"><a href="bias.html#cb1715-37" tabindex="-1"></a><span class="st"> x4 ~ NA*1</span></span>
<span id="cb1715-38"><a href="bias.html#cb1715-38" tabindex="-1"></a><span class="st"> x5 ~ NA*1</span></span>
<span id="cb1715-39"><a href="bias.html#cb1715-39" tabindex="-1"></a><span class="st"> x6 ~ NA*1</span></span>
<span id="cb1715-40"><a href="bias.html#cb1715-40" tabindex="-1"></a><span class="st"> x7 ~ NA*1</span></span>
<span id="cb1715-41"><a href="bias.html#cb1715-41" tabindex="-1"></a><span class="st"> x8 ~ NA*1</span></span>
<span id="cb1715-42"><a href="bias.html#cb1715-42" tabindex="-1"></a><span class="st"> x9 ~ NA*1</span></span>
<span id="cb1715-43"><a href="bias.html#cb1715-43" tabindex="-1"></a><span class="st">&#39;</span></span>
<span id="cb1715-44"><a href="bias.html#cb1715-44" tabindex="-1"></a></span>
<span id="cb1715-45"><a href="bias.html#cb1715-45" tabindex="-1"></a>cfaModel_metricInvarianceV1_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1715-46"><a href="bias.html#cb1715-46" tabindex="-1"></a>  cfaModel_metricInvarianceV1,</span>
<span id="cb1715-47"><a href="bias.html#cb1715-47" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1715-48"><a href="bias.html#cb1715-48" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1715-49"><a href="bias.html#cb1715-49" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1715-50"><a href="bias.html#cb1715-50" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span>
<span id="cb1715-51"><a href="bias.html#cb1715-51" tabindex="-1"></a></span>
<span id="cb1715-52"><a href="bias.html#cb1715-52" tabindex="-1"></a><span class="fu">anova</span>(metricInvarianceModel_fit, cfaModel_metricInvarianceV1_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Df"],"name":[1],"type":["int"],"align":["right"]},{"label":["AIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Chisq"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Chisq diff"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Df diff"],"name":[6],"type":["int"],"align":["right"]},{"label":["Pr(>Chisq)"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"54","2":"7412.554","3":"7612.738","4":"116.3133","5":"NA","6":"NA","7":"NA","_rn_":"metricInvarianceModel_fit"},{"1":"55","2":"7410.573","3":"7607.050","4":"116.3323","5":"0.01905591","6":"1","7":"0.8902063","_rn_":"cfaModel_metricInvarianceV1_fit"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceAddConstraintsVariable2" class="section level5 hasAnchor" number="16.10.8.1.2">
<h5><span class="header-section-number">16.10.8.1.2</span> Variable 2<a href="bias.html#measurementInvarianceAddConstraintsVariable2" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Variable 2 has non-invariant intercepts.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1716"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1716-1"><a href="bias.html#cb1716-1" tabindex="-1"></a>cfaModel_metricInvarianceV2 <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1716-2"><a href="bias.html#cb1716-2" tabindex="-1"></a><span class="st"> #Fix factor loadings to be the same across groups</span></span>
<span id="cb1716-3"><a href="bias.html#cb1716-3" tabindex="-1"></a><span class="st"> visual  =~ c(lambdax1,lambdax1)*x1 + c(lambdax2,lambdax2)*x2 + c(lambdax3,lambdax3)*x3</span></span>
<span id="cb1716-4"><a href="bias.html#cb1716-4" tabindex="-1"></a><span class="st"> textual =~ c(lambdax4,lambdax4)*x4 + c(lambdax5,lambdax5)*x5 + c(lambdax6,lambdax6)*x6</span></span>
<span id="cb1716-5"><a href="bias.html#cb1716-5" tabindex="-1"></a><span class="st"> speed   =~ c(lambdax7,lambdax7)*x7 + c(lambdax8,lambdax8)*x8 + c(lambdax9,lambdax9)*x9</span></span>
<span id="cb1716-6"><a href="bias.html#cb1716-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1716-7"><a href="bias.html#cb1716-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero</span></span>
<span id="cb1716-8"><a href="bias.html#cb1716-8" tabindex="-1"></a><span class="st"> visual ~ 0</span></span>
<span id="cb1716-9"><a href="bias.html#cb1716-9" tabindex="-1"></a><span class="st"> textual ~ 0</span></span>
<span id="cb1716-10"><a href="bias.html#cb1716-10" tabindex="-1"></a><span class="st"> speed ~ 0</span></span>
<span id="cb1716-11"><a href="bias.html#cb1716-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1716-12"><a href="bias.html#cb1716-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one in group 1; free latent variances in group 2</span></span>
<span id="cb1716-13"><a href="bias.html#cb1716-13" tabindex="-1"></a><span class="st"> visual ~~ c(1, NA)*visual</span></span>
<span id="cb1716-14"><a href="bias.html#cb1716-14" tabindex="-1"></a><span class="st"> textual ~~ c(1, NA)*textual</span></span>
<span id="cb1716-15"><a href="bias.html#cb1716-15" tabindex="-1"></a><span class="st"> speed ~~ c(1, NA)*speed</span></span>
<span id="cb1716-16"><a href="bias.html#cb1716-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1716-17"><a href="bias.html#cb1716-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1716-18"><a href="bias.html#cb1716-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1716-19"><a href="bias.html#cb1716-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1716-20"><a href="bias.html#cb1716-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1716-21"><a href="bias.html#cb1716-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1716-22"><a href="bias.html#cb1716-22" tabindex="-1"></a><span class="st"> #Estimate residual variances of manifest variables</span></span>
<span id="cb1716-23"><a href="bias.html#cb1716-23" tabindex="-1"></a><span class="st"> x1 ~~ x1</span></span>
<span id="cb1716-24"><a href="bias.html#cb1716-24" tabindex="-1"></a><span class="st"> x2 ~~ x2</span></span>
<span id="cb1716-25"><a href="bias.html#cb1716-25" tabindex="-1"></a><span class="st"> x3 ~~ x3</span></span>
<span id="cb1716-26"><a href="bias.html#cb1716-26" tabindex="-1"></a><span class="st"> x4 ~~ x4</span></span>
<span id="cb1716-27"><a href="bias.html#cb1716-27" tabindex="-1"></a><span class="st"> x5 ~~ x5</span></span>
<span id="cb1716-28"><a href="bias.html#cb1716-28" tabindex="-1"></a><span class="st"> x6 ~~ x6</span></span>
<span id="cb1716-29"><a href="bias.html#cb1716-29" tabindex="-1"></a><span class="st"> x7 ~~ x7</span></span>
<span id="cb1716-30"><a href="bias.html#cb1716-30" tabindex="-1"></a><span class="st"> x8 ~~ x8</span></span>
<span id="cb1716-31"><a href="bias.html#cb1716-31" tabindex="-1"></a><span class="st"> x9 ~~ x9</span></span>
<span id="cb1716-32"><a href="bias.html#cb1716-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1716-33"><a href="bias.html#cb1716-33" tabindex="-1"></a><span class="st"> #Iteratively fix intercepts of manifest variables across groups</span></span>
<span id="cb1716-34"><a href="bias.html#cb1716-34" tabindex="-1"></a><span class="st"> x1 ~ c(intx1, intx1)*1</span></span>
<span id="cb1716-35"><a href="bias.html#cb1716-35" tabindex="-1"></a><span class="st"> x2 ~ c(intx1, intx1)*1</span></span>
<span id="cb1716-36"><a href="bias.html#cb1716-36" tabindex="-1"></a><span class="st"> x3 ~ NA*1</span></span>
<span id="cb1716-37"><a href="bias.html#cb1716-37" tabindex="-1"></a><span class="st"> x4 ~ NA*1</span></span>
<span id="cb1716-38"><a href="bias.html#cb1716-38" tabindex="-1"></a><span class="st"> x5 ~ NA*1</span></span>
<span id="cb1716-39"><a href="bias.html#cb1716-39" tabindex="-1"></a><span class="st"> x6 ~ NA*1</span></span>
<span id="cb1716-40"><a href="bias.html#cb1716-40" tabindex="-1"></a><span class="st"> x7 ~ NA*1</span></span>
<span id="cb1716-41"><a href="bias.html#cb1716-41" tabindex="-1"></a><span class="st"> x8 ~ NA*1</span></span>
<span id="cb1716-42"><a href="bias.html#cb1716-42" tabindex="-1"></a><span class="st"> x9 ~ NA*1</span></span>
<span id="cb1716-43"><a href="bias.html#cb1716-43" tabindex="-1"></a><span class="st">&#39;</span></span>
<span id="cb1716-44"><a href="bias.html#cb1716-44" tabindex="-1"></a></span>
<span id="cb1716-45"><a href="bias.html#cb1716-45" tabindex="-1"></a>cfaModel_metricInvarianceV2_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1716-46"><a href="bias.html#cb1716-46" tabindex="-1"></a>  cfaModel_metricInvarianceV2,</span>
<span id="cb1716-47"><a href="bias.html#cb1716-47" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1716-48"><a href="bias.html#cb1716-48" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1716-49"><a href="bias.html#cb1716-49" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1716-50"><a href="bias.html#cb1716-50" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span>
<span id="cb1716-51"><a href="bias.html#cb1716-51" tabindex="-1"></a></span>
<span id="cb1716-52"><a href="bias.html#cb1716-52" tabindex="-1"></a><span class="fu">anova</span>(metricInvarianceModel_fit, cfaModel_metricInvarianceV2_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Df"],"name":[1],"type":["int"],"align":["right"]},{"label":["AIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Chisq"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Chisq diff"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Df diff"],"name":[6],"type":["int"],"align":["right"]},{"label":["Pr(>Chisq)"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"54","2":"7412.554","3":"7612.738","4":"116.3133","5":"NA","6":"NA","7":"NA","_rn_":"metricInvarianceModel_fit"},{"1":"57","2":"7578.349","3":"7767.412","4":"288.1087","5":"165.0147","6":"3","7":"1.516417e-35","_rn_":"cfaModel_metricInvarianceV2_fit"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>Variable 2 appears to have larger intercepts in group 2 than in group 1:</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1717"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1717-1"><a href="bias.html#cb1717-1" tabindex="-1"></a><span class="fu">summary</span>(metricInvarianceModel_fit)</span></code></pre></div>
</details>
<pre><code>lavaan 0.6-19 ended normally after 73 iterations

  Estimator                                         ML
  Optimization method                           NLMINB
  Number of model parameters                        63
  Number of equality constraints                     9

  Number of observations per group:                   
    Pasteur                                        156
    Grant-White                                    145
  Number of missing patterns per group:               
    Pasteur                                          9
    Grant-White                                      9

Model Test User Model:
                                              Standard      Scaled
  Test Statistic                               116.313     117.955
  Degrees of freedom                                54          54
  P-value (Chi-square)                           0.000       0.000
  Scaling correction factor                                  0.986
    Yuan-Bentler correction (Mplus variant)                       
  Test statistic for each group:
    Pasteur                                     64.366      64.366
    Grant-White                                 53.590      53.590

Parameter Estimates:

  Standard errors                             Sandwich
  Information bread                           Observed
  Observed information based on                Hessian


Group 1 [Pasteur]:

Latent Variables:
                   Estimate  Std.Err  z-value  P(&gt;|z|)
  visual =~                                           
    x1      (l.1_)    0.885    0.122    7.258    0.000
    x2      (l.2_)    0.536    0.088    6.106    0.000
    x3      (l.3_)    0.707    0.087    8.148    0.000
  textual =~                                          
    x4      (l.4_)    0.935    0.072   12.929    0.000
    x5      (l.5_)    1.045    0.076   13.740    0.000
    x6      (l.6_)    0.874    0.070   12.496    0.000
  speed =~                                            
    x7      (l.7_)    0.550    0.075    7.346    0.000
    x8      (l.8_)    0.675    0.087    7.784    0.000
    x9      (l.9_)    0.602    0.088    6.867    0.000

Covariances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)
  visual ~~                                           
    textul  (p.2_)    0.473    0.105    4.484    0.000
    speed  (p.3_1)    0.347    0.125    2.778    0.005
  textual ~~                                          
    speed  (p.3_2)    0.320    0.096    3.342    0.001

Intercepts:
                   Estimate  Std.Err  z-value  P(&gt;|z|)
   .x1      (n.1.)    4.948    0.095   52.314    0.000
   .x2      (n.2.)    5.984    0.098   60.949    0.000
   .x3      (n.3.)    2.510    0.093   26.891    0.000
   .x4      (n.4.)    2.814    0.091   30.886    0.000
   .x5      (n.5.)    4.002    0.105   38.265    0.000
   .x6      (n.6.)    1.929    0.079   24.377    0.000
   .x7      (n.7.)    4.439    0.087   51.128    0.000
   .x8      (n.8.)    5.564    0.079   70.170    0.000
   .x9      (n.9.)    5.426    0.080   67.539    0.000
    visual  (a.1.)    0.000                           
    textual (a.2.)    0.000                           
    speed   (a.3.)    0.000                           

Variances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)
   .x1      (t.1_)    0.577    0.166    3.482    0.000
   .x2      (t.2_)    1.252    0.162    7.718    0.000
   .x3      (t.3_)    0.853    0.127    6.737    0.000
   .x4      (t.4_)    0.447    0.072    6.189    0.000
   .x5      (t.5_)    0.486    0.075    6.445    0.000
   .x6      (t.6_)    0.258    0.057    4.499    0.000
   .x7      (t.7_)    0.860    0.114    7.549    0.000
   .x8      (t.8_)    0.512    0.097    5.296    0.000
   .x9      (t.9_)    0.635    0.118    5.376    0.000
    visual  (p.1_)    1.000                           
    textual (p.2_)    1.000                           
    speed   (p.3_)    1.000                           


Group 2 [Grant-White]:

Latent Variables:
                   Estimate  Std.Err  z-value  P(&gt;|z|)
  visual =~                                           
    x1      (l.1_)    0.885    0.122    7.258    0.000
    x2      (l.2_)    0.536    0.088    6.106    0.000
    x3      (l.3_)    0.707    0.087    8.148    0.000
  textual =~                                          
    x4      (l.4_)    0.935    0.072   12.929    0.000
    x5      (l.5_)    1.045    0.076   13.740    0.000
    x6      (l.6_)    0.874    0.070   12.496    0.000
  speed =~                                            
    x7      (l.7_)    0.550    0.075    7.346    0.000
    x8      (l.8_)    0.675    0.087    7.784    0.000
    x9      (l.9_)    0.602    0.088    6.867    0.000

Covariances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)
  visual ~~                                           
    textul  (p.2_)    0.520    0.134    3.867    0.000
    speed  (p.3_1)    0.638    0.211    3.023    0.002
  textual ~~                                          
    speed  (p.3_2)    0.429    0.193    2.219    0.026

Intercepts:
                   Estimate  Std.Err  z-value  P(&gt;|z|)
   .x1      (n.1.)    4.930    0.096   51.453    0.000
   .x2      (n.2.)    6.198    0.092   67.063    0.000
   .x3      (n.3.)    1.992    0.086   23.042    0.000
   .x4      (n.4.)    3.318    0.093   35.501    0.000
   .x5      (n.5.)    4.714    0.097   48.693    0.000
   .x6      (n.6.)    2.467    0.094   26.165    0.000
   .x7      (n.7.)    3.923    0.087   45.299    0.000
   .x8      (n.8.)    5.488    0.087   63.174    0.000
   .x9      (n.9.)    5.338    0.085   62.628    0.000
    visual  (a.1.)    0.000                           
    textual (a.2.)    0.000                           
    speed   (a.3.)    0.000                           

Variances:
                   Estimate  Std.Err  z-value  P(&gt;|z|)
   .x1      (t.1_)    0.652    0.168    3.877    0.000
   .x2      (t.2_)    0.940    0.145    6.473    0.000
   .x3      (t.3_)    0.606    0.113    5.376    0.000
   .x4      (t.4_)    0.348    0.066    5.257    0.000
   .x5      (t.5_)    0.371    0.074    5.021    0.000
   .x6      (t.6_)    0.439    0.076    5.738    0.000
   .x7      (t.7_)    0.622    0.098    6.338    0.000
   .x8      (t.8_)    0.410    0.148    2.765    0.006
   .x9      (t.9_)    0.505    0.128    3.957    0.000
    visual  (p.1_)    0.910    0.234    3.895    0.000
    textual (p.2_)    1.004    0.191    5.257    0.000
    speed   (p.3_)    1.498    0.357    4.201    0.000</code></pre>
<p>Thus, in subsequent models, we would allow variable 2 to have different intercepts across groups.</p>
</div>
<div id="measurementInvarianceAddConstraintsVariable3" class="section level5 hasAnchor" number="16.10.8.1.3">
<h5><span class="header-section-number">16.10.8.1.3</span> Variable 3<a href="bias.html#measurementInvarianceAddConstraintsVariable3" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Variable 3 has non-invariant intercepts.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1719"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1719-1"><a href="bias.html#cb1719-1" tabindex="-1"></a>cfaModel_metricInvarianceV3 <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1719-2"><a href="bias.html#cb1719-2" tabindex="-1"></a><span class="st"> #Fix factor loadings to be the same across groups</span></span>
<span id="cb1719-3"><a href="bias.html#cb1719-3" tabindex="-1"></a><span class="st"> visual  =~ c(lambdax1,lambdax1)*x1 + c(lambdax2,lambdax2)*x2 + c(lambdax3,lambdax3)*x3</span></span>
<span id="cb1719-4"><a href="bias.html#cb1719-4" tabindex="-1"></a><span class="st"> textual =~ c(lambdax4,lambdax4)*x4 + c(lambdax5,lambdax5)*x5 + c(lambdax6,lambdax6)*x6</span></span>
<span id="cb1719-5"><a href="bias.html#cb1719-5" tabindex="-1"></a><span class="st"> speed   =~ c(lambdax7,lambdax7)*x7 + c(lambdax8,lambdax8)*x8 + c(lambdax9,lambdax9)*x9</span></span>
<span id="cb1719-6"><a href="bias.html#cb1719-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1719-7"><a href="bias.html#cb1719-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero</span></span>
<span id="cb1719-8"><a href="bias.html#cb1719-8" tabindex="-1"></a><span class="st"> visual ~ 0</span></span>
<span id="cb1719-9"><a href="bias.html#cb1719-9" tabindex="-1"></a><span class="st"> textual ~ 0</span></span>
<span id="cb1719-10"><a href="bias.html#cb1719-10" tabindex="-1"></a><span class="st"> speed ~ 0</span></span>
<span id="cb1719-11"><a href="bias.html#cb1719-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1719-12"><a href="bias.html#cb1719-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one in group 1; free latent variances in group 2</span></span>
<span id="cb1719-13"><a href="bias.html#cb1719-13" tabindex="-1"></a><span class="st"> visual ~~ c(1, NA)*visual</span></span>
<span id="cb1719-14"><a href="bias.html#cb1719-14" tabindex="-1"></a><span class="st"> textual ~~ c(1, NA)*textual</span></span>
<span id="cb1719-15"><a href="bias.html#cb1719-15" tabindex="-1"></a><span class="st"> speed ~~ c(1, NA)*speed</span></span>
<span id="cb1719-16"><a href="bias.html#cb1719-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1719-17"><a href="bias.html#cb1719-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1719-18"><a href="bias.html#cb1719-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1719-19"><a href="bias.html#cb1719-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1719-20"><a href="bias.html#cb1719-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1719-21"><a href="bias.html#cb1719-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1719-22"><a href="bias.html#cb1719-22" tabindex="-1"></a><span class="st"> #Estimate residual variances of manifest variables</span></span>
<span id="cb1719-23"><a href="bias.html#cb1719-23" tabindex="-1"></a><span class="st"> x1 ~~ x1</span></span>
<span id="cb1719-24"><a href="bias.html#cb1719-24" tabindex="-1"></a><span class="st"> x2 ~~ x2</span></span>
<span id="cb1719-25"><a href="bias.html#cb1719-25" tabindex="-1"></a><span class="st"> x3 ~~ x3</span></span>
<span id="cb1719-26"><a href="bias.html#cb1719-26" tabindex="-1"></a><span class="st"> x4 ~~ x4</span></span>
<span id="cb1719-27"><a href="bias.html#cb1719-27" tabindex="-1"></a><span class="st"> x5 ~~ x5</span></span>
<span id="cb1719-28"><a href="bias.html#cb1719-28" tabindex="-1"></a><span class="st"> x6 ~~ x6</span></span>
<span id="cb1719-29"><a href="bias.html#cb1719-29" tabindex="-1"></a><span class="st"> x7 ~~ x7</span></span>
<span id="cb1719-30"><a href="bias.html#cb1719-30" tabindex="-1"></a><span class="st"> x8 ~~ x8</span></span>
<span id="cb1719-31"><a href="bias.html#cb1719-31" tabindex="-1"></a><span class="st"> x9 ~~ x9</span></span>
<span id="cb1719-32"><a href="bias.html#cb1719-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1719-33"><a href="bias.html#cb1719-33" tabindex="-1"></a><span class="st"> #Iteratively fix intercepts of manifest variables across groups</span></span>
<span id="cb1719-34"><a href="bias.html#cb1719-34" tabindex="-1"></a><span class="st"> x1 ~ c(intx1, intx1)*1</span></span>
<span id="cb1719-35"><a href="bias.html#cb1719-35" tabindex="-1"></a><span class="st"> x2 ~ NA*1</span></span>
<span id="cb1719-36"><a href="bias.html#cb1719-36" tabindex="-1"></a><span class="st"> x3 ~ c(intx1, intx1)*1</span></span>
<span id="cb1719-37"><a href="bias.html#cb1719-37" tabindex="-1"></a><span class="st"> x4 ~ NA*1</span></span>
<span id="cb1719-38"><a href="bias.html#cb1719-38" tabindex="-1"></a><span class="st"> x5 ~ NA*1</span></span>
<span id="cb1719-39"><a href="bias.html#cb1719-39" tabindex="-1"></a><span class="st"> x6 ~ NA*1</span></span>
<span id="cb1719-40"><a href="bias.html#cb1719-40" tabindex="-1"></a><span class="st"> x7 ~ NA*1</span></span>
<span id="cb1719-41"><a href="bias.html#cb1719-41" tabindex="-1"></a><span class="st"> x8 ~ NA*1</span></span>
<span id="cb1719-42"><a href="bias.html#cb1719-42" tabindex="-1"></a><span class="st"> x9 ~ NA*1</span></span>
<span id="cb1719-43"><a href="bias.html#cb1719-43" tabindex="-1"></a><span class="st">&#39;</span></span>
<span id="cb1719-44"><a href="bias.html#cb1719-44" tabindex="-1"></a></span>
<span id="cb1719-45"><a href="bias.html#cb1719-45" tabindex="-1"></a>cfaModel_metricInvarianceV3_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1719-46"><a href="bias.html#cb1719-46" tabindex="-1"></a>  cfaModel_metricInvarianceV3,</span>
<span id="cb1719-47"><a href="bias.html#cb1719-47" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1719-48"><a href="bias.html#cb1719-48" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1719-49"><a href="bias.html#cb1719-49" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1719-50"><a href="bias.html#cb1719-50" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span>
<span id="cb1719-51"><a href="bias.html#cb1719-51" tabindex="-1"></a></span>
<span id="cb1719-52"><a href="bias.html#cb1719-52" tabindex="-1"></a><span class="fu">anova</span>(metricInvarianceModel_fit, cfaModel_metricInvarianceV3_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Df"],"name":[1],"type":["int"],"align":["right"]},{"label":["AIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Chisq"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Chisq diff"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Df diff"],"name":[6],"type":["int"],"align":["right"]},{"label":["Pr(>Chisq)"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"54","2":"7412.554","3":"7612.738","4":"116.3133","5":"NA","6":"NA","7":"NA","_rn_":"metricInvarianceModel_fit"},{"1":"57","2":"7962.938","3":"8152.000","4":"672.6972","5":"363.5409","6":"3","7":"1.743816e-78","_rn_":"cfaModel_metricInvarianceV3_fit"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceAddConstraintsVariable4" class="section level5 hasAnchor" number="16.10.8.1.4">
<h5><span class="header-section-number">16.10.8.1.4</span> Variable 4<a href="bias.html#measurementInvarianceAddConstraintsVariable4" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Variable 4 has non-invariant intercepts.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1720"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1720-1"><a href="bias.html#cb1720-1" tabindex="-1"></a>cfaModel_metricInvarianceV4 <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1720-2"><a href="bias.html#cb1720-2" tabindex="-1"></a><span class="st"> #Fix factor loadings to be the same across groups</span></span>
<span id="cb1720-3"><a href="bias.html#cb1720-3" tabindex="-1"></a><span class="st"> visual  =~ c(lambdax1,lambdax1)*x1 + c(lambdax2,lambdax2)*x2 + c(lambdax3,lambdax3)*x3</span></span>
<span id="cb1720-4"><a href="bias.html#cb1720-4" tabindex="-1"></a><span class="st"> textual =~ c(lambdax4,lambdax4)*x4 + c(lambdax5,lambdax5)*x5 + c(lambdax6,lambdax6)*x6</span></span>
<span id="cb1720-5"><a href="bias.html#cb1720-5" tabindex="-1"></a><span class="st"> speed   =~ c(lambdax7,lambdax7)*x7 + c(lambdax8,lambdax8)*x8 + c(lambdax9,lambdax9)*x9</span></span>
<span id="cb1720-6"><a href="bias.html#cb1720-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1720-7"><a href="bias.html#cb1720-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero</span></span>
<span id="cb1720-8"><a href="bias.html#cb1720-8" tabindex="-1"></a><span class="st"> visual ~ 0</span></span>
<span id="cb1720-9"><a href="bias.html#cb1720-9" tabindex="-1"></a><span class="st"> textual ~ 0</span></span>
<span id="cb1720-10"><a href="bias.html#cb1720-10" tabindex="-1"></a><span class="st"> speed ~ 0</span></span>
<span id="cb1720-11"><a href="bias.html#cb1720-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1720-12"><a href="bias.html#cb1720-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one in group 1; free latent variances in group 2</span></span>
<span id="cb1720-13"><a href="bias.html#cb1720-13" tabindex="-1"></a><span class="st"> visual ~~ c(1, NA)*visual</span></span>
<span id="cb1720-14"><a href="bias.html#cb1720-14" tabindex="-1"></a><span class="st"> textual ~~ c(1, NA)*textual</span></span>
<span id="cb1720-15"><a href="bias.html#cb1720-15" tabindex="-1"></a><span class="st"> speed ~~ c(1, NA)*speed</span></span>
<span id="cb1720-16"><a href="bias.html#cb1720-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1720-17"><a href="bias.html#cb1720-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1720-18"><a href="bias.html#cb1720-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1720-19"><a href="bias.html#cb1720-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1720-20"><a href="bias.html#cb1720-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1720-21"><a href="bias.html#cb1720-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1720-22"><a href="bias.html#cb1720-22" tabindex="-1"></a><span class="st"> #Estimate residual variances of manifest variables</span></span>
<span id="cb1720-23"><a href="bias.html#cb1720-23" tabindex="-1"></a><span class="st"> x1 ~~ x1</span></span>
<span id="cb1720-24"><a href="bias.html#cb1720-24" tabindex="-1"></a><span class="st"> x2 ~~ x2</span></span>
<span id="cb1720-25"><a href="bias.html#cb1720-25" tabindex="-1"></a><span class="st"> x3 ~~ x3</span></span>
<span id="cb1720-26"><a href="bias.html#cb1720-26" tabindex="-1"></a><span class="st"> x4 ~~ x4</span></span>
<span id="cb1720-27"><a href="bias.html#cb1720-27" tabindex="-1"></a><span class="st"> x5 ~~ x5</span></span>
<span id="cb1720-28"><a href="bias.html#cb1720-28" tabindex="-1"></a><span class="st"> x6 ~~ x6</span></span>
<span id="cb1720-29"><a href="bias.html#cb1720-29" tabindex="-1"></a><span class="st"> x7 ~~ x7</span></span>
<span id="cb1720-30"><a href="bias.html#cb1720-30" tabindex="-1"></a><span class="st"> x8 ~~ x8</span></span>
<span id="cb1720-31"><a href="bias.html#cb1720-31" tabindex="-1"></a><span class="st"> x9 ~~ x9</span></span>
<span id="cb1720-32"><a href="bias.html#cb1720-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1720-33"><a href="bias.html#cb1720-33" tabindex="-1"></a><span class="st"> #Iteratively fix intercepts of manifest variables across groups</span></span>
<span id="cb1720-34"><a href="bias.html#cb1720-34" tabindex="-1"></a><span class="st"> x1 ~ c(intx1, intx1)*1</span></span>
<span id="cb1720-35"><a href="bias.html#cb1720-35" tabindex="-1"></a><span class="st"> x2 ~ NA*1</span></span>
<span id="cb1720-36"><a href="bias.html#cb1720-36" tabindex="-1"></a><span class="st"> x3 ~ NA*1</span></span>
<span id="cb1720-37"><a href="bias.html#cb1720-37" tabindex="-1"></a><span class="st"> x4 ~ c(intx1, intx1)*1</span></span>
<span id="cb1720-38"><a href="bias.html#cb1720-38" tabindex="-1"></a><span class="st"> x5 ~ NA*1</span></span>
<span id="cb1720-39"><a href="bias.html#cb1720-39" tabindex="-1"></a><span class="st"> x6 ~ NA*1</span></span>
<span id="cb1720-40"><a href="bias.html#cb1720-40" tabindex="-1"></a><span class="st"> x7 ~ NA*1</span></span>
<span id="cb1720-41"><a href="bias.html#cb1720-41" tabindex="-1"></a><span class="st"> x8 ~ NA*1</span></span>
<span id="cb1720-42"><a href="bias.html#cb1720-42" tabindex="-1"></a><span class="st"> x9 ~ NA*1</span></span>
<span id="cb1720-43"><a href="bias.html#cb1720-43" tabindex="-1"></a><span class="st">&#39;</span></span>
<span id="cb1720-44"><a href="bias.html#cb1720-44" tabindex="-1"></a></span>
<span id="cb1720-45"><a href="bias.html#cb1720-45" tabindex="-1"></a>cfaModel_metricInvarianceV4_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1720-46"><a href="bias.html#cb1720-46" tabindex="-1"></a>  cfaModel_metricInvarianceV4,</span>
<span id="cb1720-47"><a href="bias.html#cb1720-47" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1720-48"><a href="bias.html#cb1720-48" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1720-49"><a href="bias.html#cb1720-49" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1720-50"><a href="bias.html#cb1720-50" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span>
<span id="cb1720-51"><a href="bias.html#cb1720-51" tabindex="-1"></a></span>
<span id="cb1720-52"><a href="bias.html#cb1720-52" tabindex="-1"></a><span class="fu">anova</span>(metricInvarianceModel_fit, cfaModel_metricInvarianceV4_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Df"],"name":[1],"type":["int"],"align":["right"]},{"label":["AIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Chisq"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Chisq diff"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Df diff"],"name":[6],"type":["int"],"align":["right"]},{"label":["Pr(>Chisq)"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"54","2":"7412.554","3":"7612.738","4":"116.3133","5":"NA","6":"NA","7":"NA","_rn_":"metricInvarianceModel_fit"},{"1":"57","2":"7756.338","3":"7945.400","4":"466.0975","5":"261.8289","6":"3","7":"1.807939e-56","_rn_":"cfaModel_metricInvarianceV4_fit"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceAddConstraintsVariable5" class="section level5 hasAnchor" number="16.10.8.1.5">
<h5><span class="header-section-number">16.10.8.1.5</span> Variable 5<a href="bias.html#measurementInvarianceAddConstraintsVariable5" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Variable 5 has non-invariant intercepts.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1721"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1721-1"><a href="bias.html#cb1721-1" tabindex="-1"></a>cfaModel_metricInvarianceV5 <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1721-2"><a href="bias.html#cb1721-2" tabindex="-1"></a><span class="st"> #Fix factor loadings to be the same across groups</span></span>
<span id="cb1721-3"><a href="bias.html#cb1721-3" tabindex="-1"></a><span class="st"> visual  =~ c(lambdax1,lambdax1)*x1 + c(lambdax2,lambdax2)*x2 + c(lambdax3,lambdax3)*x3</span></span>
<span id="cb1721-4"><a href="bias.html#cb1721-4" tabindex="-1"></a><span class="st"> textual =~ c(lambdax4,lambdax4)*x4 + c(lambdax5,lambdax5)*x5 + c(lambdax6,lambdax6)*x6</span></span>
<span id="cb1721-5"><a href="bias.html#cb1721-5" tabindex="-1"></a><span class="st"> speed   =~ c(lambdax7,lambdax7)*x7 + c(lambdax8,lambdax8)*x8 + c(lambdax9,lambdax9)*x9</span></span>
<span id="cb1721-6"><a href="bias.html#cb1721-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1721-7"><a href="bias.html#cb1721-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero</span></span>
<span id="cb1721-8"><a href="bias.html#cb1721-8" tabindex="-1"></a><span class="st"> visual ~ 0</span></span>
<span id="cb1721-9"><a href="bias.html#cb1721-9" tabindex="-1"></a><span class="st"> textual ~ 0</span></span>
<span id="cb1721-10"><a href="bias.html#cb1721-10" tabindex="-1"></a><span class="st"> speed ~ 0</span></span>
<span id="cb1721-11"><a href="bias.html#cb1721-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1721-12"><a href="bias.html#cb1721-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one in group 1; free latent variances in group 2</span></span>
<span id="cb1721-13"><a href="bias.html#cb1721-13" tabindex="-1"></a><span class="st"> visual ~~ c(1, NA)*visual</span></span>
<span id="cb1721-14"><a href="bias.html#cb1721-14" tabindex="-1"></a><span class="st"> textual ~~ c(1, NA)*textual</span></span>
<span id="cb1721-15"><a href="bias.html#cb1721-15" tabindex="-1"></a><span class="st"> speed ~~ c(1, NA)*speed</span></span>
<span id="cb1721-16"><a href="bias.html#cb1721-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1721-17"><a href="bias.html#cb1721-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1721-18"><a href="bias.html#cb1721-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1721-19"><a href="bias.html#cb1721-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1721-20"><a href="bias.html#cb1721-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1721-21"><a href="bias.html#cb1721-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1721-22"><a href="bias.html#cb1721-22" tabindex="-1"></a><span class="st"> #Estimate residual variances of manifest variables</span></span>
<span id="cb1721-23"><a href="bias.html#cb1721-23" tabindex="-1"></a><span class="st"> x1 ~~ x1</span></span>
<span id="cb1721-24"><a href="bias.html#cb1721-24" tabindex="-1"></a><span class="st"> x2 ~~ x2</span></span>
<span id="cb1721-25"><a href="bias.html#cb1721-25" tabindex="-1"></a><span class="st"> x3 ~~ x3</span></span>
<span id="cb1721-26"><a href="bias.html#cb1721-26" tabindex="-1"></a><span class="st"> x4 ~~ x4</span></span>
<span id="cb1721-27"><a href="bias.html#cb1721-27" tabindex="-1"></a><span class="st"> x5 ~~ x5</span></span>
<span id="cb1721-28"><a href="bias.html#cb1721-28" tabindex="-1"></a><span class="st"> x6 ~~ x6</span></span>
<span id="cb1721-29"><a href="bias.html#cb1721-29" tabindex="-1"></a><span class="st"> x7 ~~ x7</span></span>
<span id="cb1721-30"><a href="bias.html#cb1721-30" tabindex="-1"></a><span class="st"> x8 ~~ x8</span></span>
<span id="cb1721-31"><a href="bias.html#cb1721-31" tabindex="-1"></a><span class="st"> x9 ~~ x9</span></span>
<span id="cb1721-32"><a href="bias.html#cb1721-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1721-33"><a href="bias.html#cb1721-33" tabindex="-1"></a><span class="st"> #Iteratively fix intercepts of manifest variables across groups</span></span>
<span id="cb1721-34"><a href="bias.html#cb1721-34" tabindex="-1"></a><span class="st"> x1 ~ c(intx1, intx1)*1</span></span>
<span id="cb1721-35"><a href="bias.html#cb1721-35" tabindex="-1"></a><span class="st"> x2 ~ NA*1</span></span>
<span id="cb1721-36"><a href="bias.html#cb1721-36" tabindex="-1"></a><span class="st"> x3 ~ NA*1</span></span>
<span id="cb1721-37"><a href="bias.html#cb1721-37" tabindex="-1"></a><span class="st"> x4 ~ NA*1</span></span>
<span id="cb1721-38"><a href="bias.html#cb1721-38" tabindex="-1"></a><span class="st"> x5 ~ c(intx1, intx1)*1</span></span>
<span id="cb1721-39"><a href="bias.html#cb1721-39" tabindex="-1"></a><span class="st"> x6 ~ NA*1</span></span>
<span id="cb1721-40"><a href="bias.html#cb1721-40" tabindex="-1"></a><span class="st"> x7 ~ NA*1</span></span>
<span id="cb1721-41"><a href="bias.html#cb1721-41" tabindex="-1"></a><span class="st"> x8 ~ NA*1</span></span>
<span id="cb1721-42"><a href="bias.html#cb1721-42" tabindex="-1"></a><span class="st"> x9 ~ NA*1</span></span>
<span id="cb1721-43"><a href="bias.html#cb1721-43" tabindex="-1"></a><span class="st">&#39;</span></span>
<span id="cb1721-44"><a href="bias.html#cb1721-44" tabindex="-1"></a></span>
<span id="cb1721-45"><a href="bias.html#cb1721-45" tabindex="-1"></a>cfaModel_metricInvarianceV5_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1721-46"><a href="bias.html#cb1721-46" tabindex="-1"></a>  cfaModel_metricInvarianceV5,</span>
<span id="cb1721-47"><a href="bias.html#cb1721-47" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1721-48"><a href="bias.html#cb1721-48" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1721-49"><a href="bias.html#cb1721-49" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1721-50"><a href="bias.html#cb1721-50" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span>
<span id="cb1721-51"><a href="bias.html#cb1721-51" tabindex="-1"></a></span>
<span id="cb1721-52"><a href="bias.html#cb1721-52" tabindex="-1"></a><span class="fu">anova</span>(metricInvarianceModel_fit, cfaModel_metricInvarianceV5_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Df"],"name":[1],"type":["int"],"align":["right"]},{"label":["AIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Chisq"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Chisq diff"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Df diff"],"name":[6],"type":["int"],"align":["right"]},{"label":["Pr(>Chisq)"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"54","2":"7412.554","3":"7612.738","4":"116.3133","5":"NA","6":"NA","7":"NA","_rn_":"metricInvarianceModel_fit"},{"1":"57","2":"7473.720","3":"7662.783","4":"183.4797","5":"64.17339","6":"3","7":"7.536301e-14","_rn_":"cfaModel_metricInvarianceV5_fit"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceAddConstraintsVariable6" class="section level5 hasAnchor" number="16.10.8.1.6">
<h5><span class="header-section-number">16.10.8.1.6</span> Variable 6<a href="bias.html#measurementInvarianceAddConstraintsVariable6" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Variable 6 has non-invariant intercepts.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1722"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1722-1"><a href="bias.html#cb1722-1" tabindex="-1"></a>cfaModel_metricInvarianceV6 <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1722-2"><a href="bias.html#cb1722-2" tabindex="-1"></a><span class="st"> #Fix factor loadings to be the same across groups</span></span>
<span id="cb1722-3"><a href="bias.html#cb1722-3" tabindex="-1"></a><span class="st"> visual  =~ c(lambdax1,lambdax1)*x1 + c(lambdax2,lambdax2)*x2 + c(lambdax3,lambdax3)*x3</span></span>
<span id="cb1722-4"><a href="bias.html#cb1722-4" tabindex="-1"></a><span class="st"> textual =~ c(lambdax4,lambdax4)*x4 + c(lambdax5,lambdax5)*x5 + c(lambdax6,lambdax6)*x6</span></span>
<span id="cb1722-5"><a href="bias.html#cb1722-5" tabindex="-1"></a><span class="st"> speed   =~ c(lambdax7,lambdax7)*x7 + c(lambdax8,lambdax8)*x8 + c(lambdax9,lambdax9)*x9</span></span>
<span id="cb1722-6"><a href="bias.html#cb1722-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1722-7"><a href="bias.html#cb1722-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero</span></span>
<span id="cb1722-8"><a href="bias.html#cb1722-8" tabindex="-1"></a><span class="st"> visual ~ 0</span></span>
<span id="cb1722-9"><a href="bias.html#cb1722-9" tabindex="-1"></a><span class="st"> textual ~ 0</span></span>
<span id="cb1722-10"><a href="bias.html#cb1722-10" tabindex="-1"></a><span class="st"> speed ~ 0</span></span>
<span id="cb1722-11"><a href="bias.html#cb1722-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1722-12"><a href="bias.html#cb1722-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one in group 1; free latent variances in group 2</span></span>
<span id="cb1722-13"><a href="bias.html#cb1722-13" tabindex="-1"></a><span class="st"> visual ~~ c(1, NA)*visual</span></span>
<span id="cb1722-14"><a href="bias.html#cb1722-14" tabindex="-1"></a><span class="st"> textual ~~ c(1, NA)*textual</span></span>
<span id="cb1722-15"><a href="bias.html#cb1722-15" tabindex="-1"></a><span class="st"> speed ~~ c(1, NA)*speed</span></span>
<span id="cb1722-16"><a href="bias.html#cb1722-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1722-17"><a href="bias.html#cb1722-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1722-18"><a href="bias.html#cb1722-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1722-19"><a href="bias.html#cb1722-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1722-20"><a href="bias.html#cb1722-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1722-21"><a href="bias.html#cb1722-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1722-22"><a href="bias.html#cb1722-22" tabindex="-1"></a><span class="st"> #Estimate residual variances of manifest variables</span></span>
<span id="cb1722-23"><a href="bias.html#cb1722-23" tabindex="-1"></a><span class="st"> x1 ~~ x1</span></span>
<span id="cb1722-24"><a href="bias.html#cb1722-24" tabindex="-1"></a><span class="st"> x2 ~~ x2</span></span>
<span id="cb1722-25"><a href="bias.html#cb1722-25" tabindex="-1"></a><span class="st"> x3 ~~ x3</span></span>
<span id="cb1722-26"><a href="bias.html#cb1722-26" tabindex="-1"></a><span class="st"> x4 ~~ x4</span></span>
<span id="cb1722-27"><a href="bias.html#cb1722-27" tabindex="-1"></a><span class="st"> x5 ~~ x5</span></span>
<span id="cb1722-28"><a href="bias.html#cb1722-28" tabindex="-1"></a><span class="st"> x6 ~~ x6</span></span>
<span id="cb1722-29"><a href="bias.html#cb1722-29" tabindex="-1"></a><span class="st"> x7 ~~ x7</span></span>
<span id="cb1722-30"><a href="bias.html#cb1722-30" tabindex="-1"></a><span class="st"> x8 ~~ x8</span></span>
<span id="cb1722-31"><a href="bias.html#cb1722-31" tabindex="-1"></a><span class="st"> x9 ~~ x9</span></span>
<span id="cb1722-32"><a href="bias.html#cb1722-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1722-33"><a href="bias.html#cb1722-33" tabindex="-1"></a><span class="st"> #Iteratively fix intercepts of manifest variables across groups</span></span>
<span id="cb1722-34"><a href="bias.html#cb1722-34" tabindex="-1"></a><span class="st"> x1 ~ c(intx1, intx1)*1</span></span>
<span id="cb1722-35"><a href="bias.html#cb1722-35" tabindex="-1"></a><span class="st"> x2 ~ NA*1</span></span>
<span id="cb1722-36"><a href="bias.html#cb1722-36" tabindex="-1"></a><span class="st"> x3 ~ NA*1</span></span>
<span id="cb1722-37"><a href="bias.html#cb1722-37" tabindex="-1"></a><span class="st"> x4 ~ NA*1</span></span>
<span id="cb1722-38"><a href="bias.html#cb1722-38" tabindex="-1"></a><span class="st"> x5 ~ NA*1</span></span>
<span id="cb1722-39"><a href="bias.html#cb1722-39" tabindex="-1"></a><span class="st"> x6 ~ c(intx1, intx1)*1</span></span>
<span id="cb1722-40"><a href="bias.html#cb1722-40" tabindex="-1"></a><span class="st"> x7 ~ NA*1</span></span>
<span id="cb1722-41"><a href="bias.html#cb1722-41" tabindex="-1"></a><span class="st"> x8 ~ NA*1</span></span>
<span id="cb1722-42"><a href="bias.html#cb1722-42" tabindex="-1"></a><span class="st"> x9 ~ NA*1</span></span>
<span id="cb1722-43"><a href="bias.html#cb1722-43" tabindex="-1"></a><span class="st">&#39;</span></span>
<span id="cb1722-44"><a href="bias.html#cb1722-44" tabindex="-1"></a></span>
<span id="cb1722-45"><a href="bias.html#cb1722-45" tabindex="-1"></a>cfaModel_metricInvarianceV6_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1722-46"><a href="bias.html#cb1722-46" tabindex="-1"></a>  cfaModel_metricInvarianceV6,</span>
<span id="cb1722-47"><a href="bias.html#cb1722-47" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1722-48"><a href="bias.html#cb1722-48" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1722-49"><a href="bias.html#cb1722-49" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1722-50"><a href="bias.html#cb1722-50" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span>
<span id="cb1722-51"><a href="bias.html#cb1722-51" tabindex="-1"></a></span>
<span id="cb1722-52"><a href="bias.html#cb1722-52" tabindex="-1"></a><span class="fu">anova</span>(metricInvarianceModel_fit, cfaModel_metricInvarianceV6_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Df"],"name":[1],"type":["int"],"align":["right"]},{"label":["AIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Chisq"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Chisq diff"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Df diff"],"name":[6],"type":["int"],"align":["right"]},{"label":["Pr(>Chisq)"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"54","2":"7412.554","3":"7612.738","4":"116.3133","5":"NA","6":"NA","7":"NA","_rn_":"metricInvarianceModel_fit"},{"1":"57","2":"7959.810","3":"8148.873","4":"669.5697","5":"337.3869","6":"3","7":"8.028699e-73","_rn_":"cfaModel_metricInvarianceV6_fit"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceAddConstraintsVariable7" class="section level5 hasAnchor" number="16.10.8.1.7">
<h5><span class="header-section-number">16.10.8.1.7</span> Variable 7<a href="bias.html#measurementInvarianceAddConstraintsVariable7" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Variable 7 has non-invariant intercepts.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1723"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1723-1"><a href="bias.html#cb1723-1" tabindex="-1"></a>cfaModel_metricInvarianceV7 <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1723-2"><a href="bias.html#cb1723-2" tabindex="-1"></a><span class="st"> #Fix factor loadings to be the same across groups</span></span>
<span id="cb1723-3"><a href="bias.html#cb1723-3" tabindex="-1"></a><span class="st"> visual  =~ c(lambdax1,lambdax1)*x1 + c(lambdax2,lambdax2)*x2 + c(lambdax3,lambdax3)*x3</span></span>
<span id="cb1723-4"><a href="bias.html#cb1723-4" tabindex="-1"></a><span class="st"> textual =~ c(lambdax4,lambdax4)*x4 + c(lambdax5,lambdax5)*x5 + c(lambdax6,lambdax6)*x6</span></span>
<span id="cb1723-5"><a href="bias.html#cb1723-5" tabindex="-1"></a><span class="st"> speed   =~ c(lambdax7,lambdax7)*x7 + c(lambdax8,lambdax8)*x8 + c(lambdax9,lambdax9)*x9</span></span>
<span id="cb1723-6"><a href="bias.html#cb1723-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1723-7"><a href="bias.html#cb1723-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero</span></span>
<span id="cb1723-8"><a href="bias.html#cb1723-8" tabindex="-1"></a><span class="st"> visual ~ 0</span></span>
<span id="cb1723-9"><a href="bias.html#cb1723-9" tabindex="-1"></a><span class="st"> textual ~ 0</span></span>
<span id="cb1723-10"><a href="bias.html#cb1723-10" tabindex="-1"></a><span class="st"> speed ~ 0</span></span>
<span id="cb1723-11"><a href="bias.html#cb1723-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1723-12"><a href="bias.html#cb1723-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one in group 1; free latent variances in group 2</span></span>
<span id="cb1723-13"><a href="bias.html#cb1723-13" tabindex="-1"></a><span class="st"> visual ~~ c(1, NA)*visual</span></span>
<span id="cb1723-14"><a href="bias.html#cb1723-14" tabindex="-1"></a><span class="st"> textual ~~ c(1, NA)*textual</span></span>
<span id="cb1723-15"><a href="bias.html#cb1723-15" tabindex="-1"></a><span class="st"> speed ~~ c(1, NA)*speed</span></span>
<span id="cb1723-16"><a href="bias.html#cb1723-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1723-17"><a href="bias.html#cb1723-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1723-18"><a href="bias.html#cb1723-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1723-19"><a href="bias.html#cb1723-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1723-20"><a href="bias.html#cb1723-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1723-21"><a href="bias.html#cb1723-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1723-22"><a href="bias.html#cb1723-22" tabindex="-1"></a><span class="st"> #Estimate residual variances of manifest variables</span></span>
<span id="cb1723-23"><a href="bias.html#cb1723-23" tabindex="-1"></a><span class="st"> x1 ~~ x1</span></span>
<span id="cb1723-24"><a href="bias.html#cb1723-24" tabindex="-1"></a><span class="st"> x2 ~~ x2</span></span>
<span id="cb1723-25"><a href="bias.html#cb1723-25" tabindex="-1"></a><span class="st"> x3 ~~ x3</span></span>
<span id="cb1723-26"><a href="bias.html#cb1723-26" tabindex="-1"></a><span class="st"> x4 ~~ x4</span></span>
<span id="cb1723-27"><a href="bias.html#cb1723-27" tabindex="-1"></a><span class="st"> x5 ~~ x5</span></span>
<span id="cb1723-28"><a href="bias.html#cb1723-28" tabindex="-1"></a><span class="st"> x6 ~~ x6</span></span>
<span id="cb1723-29"><a href="bias.html#cb1723-29" tabindex="-1"></a><span class="st"> x7 ~~ x7</span></span>
<span id="cb1723-30"><a href="bias.html#cb1723-30" tabindex="-1"></a><span class="st"> x8 ~~ x8</span></span>
<span id="cb1723-31"><a href="bias.html#cb1723-31" tabindex="-1"></a><span class="st"> x9 ~~ x9</span></span>
<span id="cb1723-32"><a href="bias.html#cb1723-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1723-33"><a href="bias.html#cb1723-33" tabindex="-1"></a><span class="st"> #Iteratively fix intercepts of manifest variables across groups</span></span>
<span id="cb1723-34"><a href="bias.html#cb1723-34" tabindex="-1"></a><span class="st"> x1 ~ c(intx1, intx1)*1</span></span>
<span id="cb1723-35"><a href="bias.html#cb1723-35" tabindex="-1"></a><span class="st"> x2 ~ NA*1</span></span>
<span id="cb1723-36"><a href="bias.html#cb1723-36" tabindex="-1"></a><span class="st"> x3 ~ NA*1</span></span>
<span id="cb1723-37"><a href="bias.html#cb1723-37" tabindex="-1"></a><span class="st"> x4 ~ NA*1</span></span>
<span id="cb1723-38"><a href="bias.html#cb1723-38" tabindex="-1"></a><span class="st"> x5 ~ NA*1</span></span>
<span id="cb1723-39"><a href="bias.html#cb1723-39" tabindex="-1"></a><span class="st"> x6 ~ NA*1</span></span>
<span id="cb1723-40"><a href="bias.html#cb1723-40" tabindex="-1"></a><span class="st"> x7 ~ c(intx1, intx1)*1</span></span>
<span id="cb1723-41"><a href="bias.html#cb1723-41" tabindex="-1"></a><span class="st"> x8 ~ NA*1</span></span>
<span id="cb1723-42"><a href="bias.html#cb1723-42" tabindex="-1"></a><span class="st"> x9 ~ NA*1</span></span>
<span id="cb1723-43"><a href="bias.html#cb1723-43" tabindex="-1"></a><span class="st">&#39;</span></span>
<span id="cb1723-44"><a href="bias.html#cb1723-44" tabindex="-1"></a></span>
<span id="cb1723-45"><a href="bias.html#cb1723-45" tabindex="-1"></a>cfaModel_metricInvarianceV7_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1723-46"><a href="bias.html#cb1723-46" tabindex="-1"></a>  cfaModel_metricInvarianceV7,</span>
<span id="cb1723-47"><a href="bias.html#cb1723-47" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1723-48"><a href="bias.html#cb1723-48" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1723-49"><a href="bias.html#cb1723-49" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1723-50"><a href="bias.html#cb1723-50" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span>
<span id="cb1723-51"><a href="bias.html#cb1723-51" tabindex="-1"></a></span>
<span id="cb1723-52"><a href="bias.html#cb1723-52" tabindex="-1"></a><span class="fu">anova</span>(metricInvarianceModel_fit, cfaModel_metricInvarianceV7_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Df"],"name":[1],"type":["int"],"align":["right"]},{"label":["AIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Chisq"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Chisq diff"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Df diff"],"name":[6],"type":["int"],"align":["right"]},{"label":["Pr(>Chisq)"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"54","2":"7412.554","3":"7612.738","4":"116.3133","5":"NA","6":"NA","7":"NA","_rn_":"metricInvarianceModel_fit"},{"1":"57","2":"7500.706","3":"7689.769","4":"210.4657","5":"82.5573","6":"3","7":"8.67753e-18","_rn_":"cfaModel_metricInvarianceV7_fit"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceAddConstraintsVariable8" class="section level5 hasAnchor" number="16.10.8.1.8">
<h5><span class="header-section-number">16.10.8.1.8</span> Variable 8<a href="bias.html#measurementInvarianceAddConstraintsVariable8" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Variable 8 has non-invariant intercepts.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1724"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1724-1"><a href="bias.html#cb1724-1" tabindex="-1"></a>cfaModel_metricInvarianceV8 <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1724-2"><a href="bias.html#cb1724-2" tabindex="-1"></a><span class="st"> #Fix factor loadings to be the same across groups</span></span>
<span id="cb1724-3"><a href="bias.html#cb1724-3" tabindex="-1"></a><span class="st"> visual  =~ c(lambdax1,lambdax1)*x1 + c(lambdax2,lambdax2)*x2 + c(lambdax3,lambdax3)*x3</span></span>
<span id="cb1724-4"><a href="bias.html#cb1724-4" tabindex="-1"></a><span class="st"> textual =~ c(lambdax4,lambdax4)*x4 + c(lambdax5,lambdax5)*x5 + c(lambdax6,lambdax6)*x6</span></span>
<span id="cb1724-5"><a href="bias.html#cb1724-5" tabindex="-1"></a><span class="st"> speed   =~ c(lambdax7,lambdax7)*x7 + c(lambdax8,lambdax8)*x8 + c(lambdax9,lambdax9)*x9</span></span>
<span id="cb1724-6"><a href="bias.html#cb1724-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1724-7"><a href="bias.html#cb1724-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero</span></span>
<span id="cb1724-8"><a href="bias.html#cb1724-8" tabindex="-1"></a><span class="st"> visual ~ 0</span></span>
<span id="cb1724-9"><a href="bias.html#cb1724-9" tabindex="-1"></a><span class="st"> textual ~ 0</span></span>
<span id="cb1724-10"><a href="bias.html#cb1724-10" tabindex="-1"></a><span class="st"> speed ~ 0</span></span>
<span id="cb1724-11"><a href="bias.html#cb1724-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1724-12"><a href="bias.html#cb1724-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one in group 1; free latent variances in group 2</span></span>
<span id="cb1724-13"><a href="bias.html#cb1724-13" tabindex="-1"></a><span class="st"> visual ~~ c(1, NA)*visual</span></span>
<span id="cb1724-14"><a href="bias.html#cb1724-14" tabindex="-1"></a><span class="st"> textual ~~ c(1, NA)*textual</span></span>
<span id="cb1724-15"><a href="bias.html#cb1724-15" tabindex="-1"></a><span class="st"> speed ~~ c(1, NA)*speed</span></span>
<span id="cb1724-16"><a href="bias.html#cb1724-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1724-17"><a href="bias.html#cb1724-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1724-18"><a href="bias.html#cb1724-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1724-19"><a href="bias.html#cb1724-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1724-20"><a href="bias.html#cb1724-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1724-21"><a href="bias.html#cb1724-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1724-22"><a href="bias.html#cb1724-22" tabindex="-1"></a><span class="st"> #Estimate residual variances of manifest variables</span></span>
<span id="cb1724-23"><a href="bias.html#cb1724-23" tabindex="-1"></a><span class="st"> x1 ~~ x1</span></span>
<span id="cb1724-24"><a href="bias.html#cb1724-24" tabindex="-1"></a><span class="st"> x2 ~~ x2</span></span>
<span id="cb1724-25"><a href="bias.html#cb1724-25" tabindex="-1"></a><span class="st"> x3 ~~ x3</span></span>
<span id="cb1724-26"><a href="bias.html#cb1724-26" tabindex="-1"></a><span class="st"> x4 ~~ x4</span></span>
<span id="cb1724-27"><a href="bias.html#cb1724-27" tabindex="-1"></a><span class="st"> x5 ~~ x5</span></span>
<span id="cb1724-28"><a href="bias.html#cb1724-28" tabindex="-1"></a><span class="st"> x6 ~~ x6</span></span>
<span id="cb1724-29"><a href="bias.html#cb1724-29" tabindex="-1"></a><span class="st"> x7 ~~ x7</span></span>
<span id="cb1724-30"><a href="bias.html#cb1724-30" tabindex="-1"></a><span class="st"> x8 ~~ x8</span></span>
<span id="cb1724-31"><a href="bias.html#cb1724-31" tabindex="-1"></a><span class="st"> x9 ~~ x9</span></span>
<span id="cb1724-32"><a href="bias.html#cb1724-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1724-33"><a href="bias.html#cb1724-33" tabindex="-1"></a><span class="st"> #Iteratively fix intercepts of manifest variables across groups</span></span>
<span id="cb1724-34"><a href="bias.html#cb1724-34" tabindex="-1"></a><span class="st"> x1 ~ c(intx1, intx1)*1</span></span>
<span id="cb1724-35"><a href="bias.html#cb1724-35" tabindex="-1"></a><span class="st"> x2 ~ NA*1</span></span>
<span id="cb1724-36"><a href="bias.html#cb1724-36" tabindex="-1"></a><span class="st"> x3 ~ NA*1</span></span>
<span id="cb1724-37"><a href="bias.html#cb1724-37" tabindex="-1"></a><span class="st"> x4 ~ NA*1</span></span>
<span id="cb1724-38"><a href="bias.html#cb1724-38" tabindex="-1"></a><span class="st"> x5 ~ NA*1</span></span>
<span id="cb1724-39"><a href="bias.html#cb1724-39" tabindex="-1"></a><span class="st"> x6 ~ NA*1</span></span>
<span id="cb1724-40"><a href="bias.html#cb1724-40" tabindex="-1"></a><span class="st"> x7 ~ NA*1</span></span>
<span id="cb1724-41"><a href="bias.html#cb1724-41" tabindex="-1"></a><span class="st"> x8 ~ c(intx1, intx1)*1</span></span>
<span id="cb1724-42"><a href="bias.html#cb1724-42" tabindex="-1"></a><span class="st"> x9 ~ NA*1</span></span>
<span id="cb1724-43"><a href="bias.html#cb1724-43" tabindex="-1"></a><span class="st">&#39;</span></span>
<span id="cb1724-44"><a href="bias.html#cb1724-44" tabindex="-1"></a></span>
<span id="cb1724-45"><a href="bias.html#cb1724-45" tabindex="-1"></a>cfaModel_metricInvarianceV8_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1724-46"><a href="bias.html#cb1724-46" tabindex="-1"></a>  cfaModel_metricInvarianceV8,</span>
<span id="cb1724-47"><a href="bias.html#cb1724-47" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1724-48"><a href="bias.html#cb1724-48" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1724-49"><a href="bias.html#cb1724-49" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1724-50"><a href="bias.html#cb1724-50" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span>
<span id="cb1724-51"><a href="bias.html#cb1724-51" tabindex="-1"></a></span>
<span id="cb1724-52"><a href="bias.html#cb1724-52" tabindex="-1"></a><span class="fu">anova</span>(metricInvarianceModel_fit, cfaModel_metricInvarianceV8_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Df"],"name":[1],"type":["int"],"align":["right"]},{"label":["AIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Chisq"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Chisq diff"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Df diff"],"name":[6],"type":["int"],"align":["right"]},{"label":["Pr(>Chisq)"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"54","2":"7412.554","3":"7612.738","4":"116.3133","5":"NA","6":"NA","7":"NA","_rn_":"metricInvarianceModel_fit"},{"1":"57","2":"7460.012","3":"7649.075","4":"169.7719","5":"64.74584","6":"3","7":"5.684922e-14","_rn_":"cfaModel_metricInvarianceV8_fit"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceAddConstraintsVariable9" class="section level5 hasAnchor" number="16.10.8.1.9">
<h5><span class="header-section-number">16.10.8.1.9</span> Variable 9<a href="bias.html#measurementInvarianceAddConstraintsVariable9" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Variable 9 has non-invariant intercepts.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1725"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1725-1"><a href="bias.html#cb1725-1" tabindex="-1"></a>cfaModel_metricInvarianceV9 <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1725-2"><a href="bias.html#cb1725-2" tabindex="-1"></a><span class="st"> #Fix factor loadings to be the same across groups</span></span>
<span id="cb1725-3"><a href="bias.html#cb1725-3" tabindex="-1"></a><span class="st"> visual  =~ c(lambdax1,lambdax1)*x1 + c(lambdax2,lambdax2)*x2 + c(lambdax3,lambdax3)*x3</span></span>
<span id="cb1725-4"><a href="bias.html#cb1725-4" tabindex="-1"></a><span class="st"> textual =~ c(lambdax4,lambdax4)*x4 + c(lambdax5,lambdax5)*x5 + c(lambdax6,lambdax6)*x6</span></span>
<span id="cb1725-5"><a href="bias.html#cb1725-5" tabindex="-1"></a><span class="st"> speed   =~ c(lambdax7,lambdax7)*x7 + c(lambdax8,lambdax8)*x8 + c(lambdax9,lambdax9)*x9</span></span>
<span id="cb1725-6"><a href="bias.html#cb1725-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1725-7"><a href="bias.html#cb1725-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero</span></span>
<span id="cb1725-8"><a href="bias.html#cb1725-8" tabindex="-1"></a><span class="st"> visual ~ 0</span></span>
<span id="cb1725-9"><a href="bias.html#cb1725-9" tabindex="-1"></a><span class="st"> textual ~ 0</span></span>
<span id="cb1725-10"><a href="bias.html#cb1725-10" tabindex="-1"></a><span class="st"> speed ~ 0</span></span>
<span id="cb1725-11"><a href="bias.html#cb1725-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1725-12"><a href="bias.html#cb1725-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one in group 1; free latent variances in group 2</span></span>
<span id="cb1725-13"><a href="bias.html#cb1725-13" tabindex="-1"></a><span class="st"> visual ~~ c(1, NA)*visual</span></span>
<span id="cb1725-14"><a href="bias.html#cb1725-14" tabindex="-1"></a><span class="st"> textual ~~ c(1, NA)*textual</span></span>
<span id="cb1725-15"><a href="bias.html#cb1725-15" tabindex="-1"></a><span class="st"> speed ~~ c(1, NA)*speed</span></span>
<span id="cb1725-16"><a href="bias.html#cb1725-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1725-17"><a href="bias.html#cb1725-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1725-18"><a href="bias.html#cb1725-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1725-19"><a href="bias.html#cb1725-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1725-20"><a href="bias.html#cb1725-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1725-21"><a href="bias.html#cb1725-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1725-22"><a href="bias.html#cb1725-22" tabindex="-1"></a><span class="st"> #Estimate residual variances of manifest variables</span></span>
<span id="cb1725-23"><a href="bias.html#cb1725-23" tabindex="-1"></a><span class="st"> x1 ~~ x1</span></span>
<span id="cb1725-24"><a href="bias.html#cb1725-24" tabindex="-1"></a><span class="st"> x2 ~~ x2</span></span>
<span id="cb1725-25"><a href="bias.html#cb1725-25" tabindex="-1"></a><span class="st"> x3 ~~ x3</span></span>
<span id="cb1725-26"><a href="bias.html#cb1725-26" tabindex="-1"></a><span class="st"> x4 ~~ x4</span></span>
<span id="cb1725-27"><a href="bias.html#cb1725-27" tabindex="-1"></a><span class="st"> x5 ~~ x5</span></span>
<span id="cb1725-28"><a href="bias.html#cb1725-28" tabindex="-1"></a><span class="st"> x6 ~~ x6</span></span>
<span id="cb1725-29"><a href="bias.html#cb1725-29" tabindex="-1"></a><span class="st"> x7 ~~ x7</span></span>
<span id="cb1725-30"><a href="bias.html#cb1725-30" tabindex="-1"></a><span class="st"> x8 ~~ x8</span></span>
<span id="cb1725-31"><a href="bias.html#cb1725-31" tabindex="-1"></a><span class="st"> x9 ~~ x9</span></span>
<span id="cb1725-32"><a href="bias.html#cb1725-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1725-33"><a href="bias.html#cb1725-33" tabindex="-1"></a><span class="st"> #Iteratively fix intercepts of manifest variables across groups</span></span>
<span id="cb1725-34"><a href="bias.html#cb1725-34" tabindex="-1"></a><span class="st"> x1 ~ c(intx1, intx1)*1</span></span>
<span id="cb1725-35"><a href="bias.html#cb1725-35" tabindex="-1"></a><span class="st"> x2 ~ NA*1</span></span>
<span id="cb1725-36"><a href="bias.html#cb1725-36" tabindex="-1"></a><span class="st"> x3 ~ NA*1</span></span>
<span id="cb1725-37"><a href="bias.html#cb1725-37" tabindex="-1"></a><span class="st"> x4 ~ NA*1</span></span>
<span id="cb1725-38"><a href="bias.html#cb1725-38" tabindex="-1"></a><span class="st"> x5 ~ NA*1</span></span>
<span id="cb1725-39"><a href="bias.html#cb1725-39" tabindex="-1"></a><span class="st"> x6 ~ NA*1</span></span>
<span id="cb1725-40"><a href="bias.html#cb1725-40" tabindex="-1"></a><span class="st"> x7 ~ NA*1</span></span>
<span id="cb1725-41"><a href="bias.html#cb1725-41" tabindex="-1"></a><span class="st"> x8 ~ NA*1</span></span>
<span id="cb1725-42"><a href="bias.html#cb1725-42" tabindex="-1"></a><span class="st"> x9 ~ c(intx1, intx1)*1</span></span>
<span id="cb1725-43"><a href="bias.html#cb1725-43" tabindex="-1"></a><span class="st">&#39;</span></span>
<span id="cb1725-44"><a href="bias.html#cb1725-44" tabindex="-1"></a></span>
<span id="cb1725-45"><a href="bias.html#cb1725-45" tabindex="-1"></a>cfaModel_metricInvarianceV9_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1725-46"><a href="bias.html#cb1725-46" tabindex="-1"></a>  cfaModel_metricInvarianceV9,</span>
<span id="cb1725-47"><a href="bias.html#cb1725-47" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1725-48"><a href="bias.html#cb1725-48" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1725-49"><a href="bias.html#cb1725-49" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1725-50"><a href="bias.html#cb1725-50" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span>
<span id="cb1725-51"><a href="bias.html#cb1725-51" tabindex="-1"></a></span>
<span id="cb1725-52"><a href="bias.html#cb1725-52" tabindex="-1"></a><span class="fu">anova</span>(metricInvarianceModel_fit, cfaModel_metricInvarianceV9_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Df"],"name":[1],"type":["int"],"align":["right"]},{"label":["AIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Chisq"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Chisq diff"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Df diff"],"name":[6],"type":["int"],"align":["right"]},{"label":["Pr(>Chisq)"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"54","2":"7412.554","3":"7612.738","4":"116.3133","5":"NA","6":"NA","7":"NA","_rn_":"metricInvarianceModel_fit"},{"1":"57","2":"7437.043","3":"7626.106","4":"146.8029","5":"33.95633","6":"3","7":"2.023695e-07","_rn_":"cfaModel_metricInvarianceV9_fit"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceAddConstraintsSummary" class="section level5 hasAnchor" number="16.10.8.1.10">
<h5><span class="header-section-number">16.10.8.1.10</span> Summary<a href="bias.html#measurementInvarianceAddConstraintsSummary" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>When iteratively adding constraints, item 1 showed measurement invariance of intercepts, but all other items showed measurement non-invariance of intercepts.
This could be due to the selection of starting with item 1 as the first constraint.
We could try applying constraints in a different order to see the extent to which the order influences which items are identified as showing measurement non-invariance.</p>
</div>
</div>
<div id="measurementInvarianceDropConstraints" class="section level4 hasAnchor" number="16.10.8.2">
<h4><span class="header-section-number">16.10.8.2</span> Iteratively drop constraints to identify measurement non-invariance<a href="bias.html#measurementInvarianceDropConstraints" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Another approach to identify measurement non-invariance is to iteratively drop constraints to a model that constrains the given parameter (intercepts) to be the same across groups.
So, we will use the scalar invariance model as the baseline model (in which item intercepts are constrained to be the same across groups), and we will iteratively drop constraints to identify the items for which measurement invariance becomes established (relative to the metric invariance model which allows item intercepts to differ across groups), and therefore, identifies which item(s) show non-invariant intercepts across groups.</p>
<div id="measurementInvarianceDropConstraintsVariable1" class="section level5 hasAnchor" number="16.10.8.2.1">
<h5><span class="header-section-number">16.10.8.2.1</span> Variable 1<a href="bias.html#measurementInvarianceDropConstraintsVariable1" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Measurement invariance does not yet become established when freeing intercepts of variable 1 across groups.
This suggests that, whether or not variable 1 shows non-invariant intercepts, there are other items that show non-invariant intercepts.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1726"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1726-1"><a href="bias.html#cb1726-1" tabindex="-1"></a>cfaModel_scalarInvarianceV1 <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1726-2"><a href="bias.html#cb1726-2" tabindex="-1"></a><span class="st"> #Fix factor loadings to be the same across groups</span></span>
<span id="cb1726-3"><a href="bias.html#cb1726-3" tabindex="-1"></a><span class="st"> visual  =~ c(lambdax1,lambdax1)*x1 + c(lambdax2,lambdax2)*x2 + c(lambdax3,lambdax3)*x3</span></span>
<span id="cb1726-4"><a href="bias.html#cb1726-4" tabindex="-1"></a><span class="st"> textual =~ c(lambdax4,lambdax4)*x4 + c(lambdax5,lambdax5)*x5 + c(lambdax6,lambdax6)*x6</span></span>
<span id="cb1726-5"><a href="bias.html#cb1726-5" tabindex="-1"></a><span class="st"> speed   =~ c(lambdax7,lambdax7)*x7 + c(lambdax8,lambdax8)*x8 + c(lambdax9,lambdax9)*x9</span></span>
<span id="cb1726-6"><a href="bias.html#cb1726-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1726-7"><a href="bias.html#cb1726-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero</span></span>
<span id="cb1726-8"><a href="bias.html#cb1726-8" tabindex="-1"></a><span class="st"> visual ~ 0</span></span>
<span id="cb1726-9"><a href="bias.html#cb1726-9" tabindex="-1"></a><span class="st"> textual ~ 0</span></span>
<span id="cb1726-10"><a href="bias.html#cb1726-10" tabindex="-1"></a><span class="st"> speed ~ 0</span></span>
<span id="cb1726-11"><a href="bias.html#cb1726-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1726-12"><a href="bias.html#cb1726-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one in group 1; free latent variances in group 2</span></span>
<span id="cb1726-13"><a href="bias.html#cb1726-13" tabindex="-1"></a><span class="st"> visual ~~ c(1, NA)*visual</span></span>
<span id="cb1726-14"><a href="bias.html#cb1726-14" tabindex="-1"></a><span class="st"> textual ~~ c(1, NA)*textual</span></span>
<span id="cb1726-15"><a href="bias.html#cb1726-15" tabindex="-1"></a><span class="st"> speed ~~ c(1, NA)*speed</span></span>
<span id="cb1726-16"><a href="bias.html#cb1726-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1726-17"><a href="bias.html#cb1726-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1726-18"><a href="bias.html#cb1726-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1726-19"><a href="bias.html#cb1726-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1726-20"><a href="bias.html#cb1726-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1726-21"><a href="bias.html#cb1726-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1726-22"><a href="bias.html#cb1726-22" tabindex="-1"></a><span class="st"> #Estimate residual variances of manifest variables</span></span>
<span id="cb1726-23"><a href="bias.html#cb1726-23" tabindex="-1"></a><span class="st"> x1 ~~ x1</span></span>
<span id="cb1726-24"><a href="bias.html#cb1726-24" tabindex="-1"></a><span class="st"> x2 ~~ x2</span></span>
<span id="cb1726-25"><a href="bias.html#cb1726-25" tabindex="-1"></a><span class="st"> x3 ~~ x3</span></span>
<span id="cb1726-26"><a href="bias.html#cb1726-26" tabindex="-1"></a><span class="st"> x4 ~~ x4</span></span>
<span id="cb1726-27"><a href="bias.html#cb1726-27" tabindex="-1"></a><span class="st"> x5 ~~ x5</span></span>
<span id="cb1726-28"><a href="bias.html#cb1726-28" tabindex="-1"></a><span class="st"> x6 ~~ x6</span></span>
<span id="cb1726-29"><a href="bias.html#cb1726-29" tabindex="-1"></a><span class="st"> x7 ~~ x7</span></span>
<span id="cb1726-30"><a href="bias.html#cb1726-30" tabindex="-1"></a><span class="st"> x8 ~~ x8</span></span>
<span id="cb1726-31"><a href="bias.html#cb1726-31" tabindex="-1"></a><span class="st"> x9 ~~ x9</span></span>
<span id="cb1726-32"><a href="bias.html#cb1726-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1726-33"><a href="bias.html#cb1726-33" tabindex="-1"></a><span class="st"> #Iteratively free intercepts across groups</span></span>
<span id="cb1726-34"><a href="bias.html#cb1726-34" tabindex="-1"></a><span class="st"> x1 ~ NA*1</span></span>
<span id="cb1726-35"><a href="bias.html#cb1726-35" tabindex="-1"></a><span class="st"> x2 ~ c(intx2, intx2)*1</span></span>
<span id="cb1726-36"><a href="bias.html#cb1726-36" tabindex="-1"></a><span class="st"> x3 ~ c(intx3, intx3)*1</span></span>
<span id="cb1726-37"><a href="bias.html#cb1726-37" tabindex="-1"></a><span class="st"> x4 ~ c(intx4, intx4)*1</span></span>
<span id="cb1726-38"><a href="bias.html#cb1726-38" tabindex="-1"></a><span class="st"> x5 ~ c(intx5, intx5)*1</span></span>
<span id="cb1726-39"><a href="bias.html#cb1726-39" tabindex="-1"></a><span class="st"> x6 ~ c(intx6, intx6)*1</span></span>
<span id="cb1726-40"><a href="bias.html#cb1726-40" tabindex="-1"></a><span class="st"> x7 ~ c(intx7, intx7)*1</span></span>
<span id="cb1726-41"><a href="bias.html#cb1726-41" tabindex="-1"></a><span class="st"> x8 ~ c(intx8, intx8)*1</span></span>
<span id="cb1726-42"><a href="bias.html#cb1726-42" tabindex="-1"></a><span class="st"> x9 ~ c(intx9, intx9)*1</span></span>
<span id="cb1726-43"><a href="bias.html#cb1726-43" tabindex="-1"></a><span class="st">&#39;</span></span>
<span id="cb1726-44"><a href="bias.html#cb1726-44" tabindex="-1"></a></span>
<span id="cb1726-45"><a href="bias.html#cb1726-45" tabindex="-1"></a>cfaModel_scalarInvarianceV1_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1726-46"><a href="bias.html#cb1726-46" tabindex="-1"></a>  cfaModel_scalarInvarianceV1,</span>
<span id="cb1726-47"><a href="bias.html#cb1726-47" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1726-48"><a href="bias.html#cb1726-48" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1726-49"><a href="bias.html#cb1726-49" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1726-50"><a href="bias.html#cb1726-50" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span>
<span id="cb1726-51"><a href="bias.html#cb1726-51" tabindex="-1"></a></span>
<span id="cb1726-52"><a href="bias.html#cb1726-52" tabindex="-1"></a><span class="fu">anova</span>(metricInvarianceModel_fit, cfaModel_scalarInvarianceV1_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Df"],"name":[1],"type":["int"],"align":["right"]},{"label":["AIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Chisq"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Chisq diff"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Df diff"],"name":[6],"type":["int"],"align":["right"]},{"label":["Pr(>Chisq)"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"54","2":"7412.554","3":"7612.738","4":"116.3133","5":"NA","6":"NA","7":"NA","_rn_":"metricInvarianceModel_fit"},{"1":"62","2":"7477.804","3":"7648.331","4":"197.5632","5":"82.97954","6":"8","7":"1.226467e-14","_rn_":"cfaModel_scalarInvarianceV1_fit"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceDropConstraintsVariable2" class="section level5 hasAnchor" number="16.10.8.2.2">
<h5><span class="header-section-number">16.10.8.2.2</span> Variable 2<a href="bias.html#measurementInvarianceDropConstraintsVariable2" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Measurement invariance does not yet become established when freeing intercepts of variable 2 across groups.
This suggests that, whether or not variables 1 and 2 show non-invariant intercepts, there are other items that show non-invariant intercepts.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1727"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1727-1"><a href="bias.html#cb1727-1" tabindex="-1"></a>cfaModel_scalarInvarianceV2 <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1727-2"><a href="bias.html#cb1727-2" tabindex="-1"></a><span class="st"> #Fix factor loadings to be the same across groups</span></span>
<span id="cb1727-3"><a href="bias.html#cb1727-3" tabindex="-1"></a><span class="st"> visual  =~ c(lambdax1,lambdax1)*x1 + c(lambdax2,lambdax2)*x2 + c(lambdax3,lambdax3)*x3</span></span>
<span id="cb1727-4"><a href="bias.html#cb1727-4" tabindex="-1"></a><span class="st"> textual =~ c(lambdax4,lambdax4)*x4 + c(lambdax5,lambdax5)*x5 + c(lambdax6,lambdax6)*x6</span></span>
<span id="cb1727-5"><a href="bias.html#cb1727-5" tabindex="-1"></a><span class="st"> speed   =~ c(lambdax7,lambdax7)*x7 + c(lambdax8,lambdax8)*x8 + c(lambdax9,lambdax9)*x9</span></span>
<span id="cb1727-6"><a href="bias.html#cb1727-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1727-7"><a href="bias.html#cb1727-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero</span></span>
<span id="cb1727-8"><a href="bias.html#cb1727-8" tabindex="-1"></a><span class="st"> visual ~ 0</span></span>
<span id="cb1727-9"><a href="bias.html#cb1727-9" tabindex="-1"></a><span class="st"> textual ~ 0</span></span>
<span id="cb1727-10"><a href="bias.html#cb1727-10" tabindex="-1"></a><span class="st"> speed ~ 0</span></span>
<span id="cb1727-11"><a href="bias.html#cb1727-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1727-12"><a href="bias.html#cb1727-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one in group 1; free latent variances in group 2</span></span>
<span id="cb1727-13"><a href="bias.html#cb1727-13" tabindex="-1"></a><span class="st"> visual ~~ c(1, NA)*visual</span></span>
<span id="cb1727-14"><a href="bias.html#cb1727-14" tabindex="-1"></a><span class="st"> textual ~~ c(1, NA)*textual</span></span>
<span id="cb1727-15"><a href="bias.html#cb1727-15" tabindex="-1"></a><span class="st"> speed ~~ c(1, NA)*speed</span></span>
<span id="cb1727-16"><a href="bias.html#cb1727-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1727-17"><a href="bias.html#cb1727-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1727-18"><a href="bias.html#cb1727-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1727-19"><a href="bias.html#cb1727-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1727-20"><a href="bias.html#cb1727-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1727-21"><a href="bias.html#cb1727-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1727-22"><a href="bias.html#cb1727-22" tabindex="-1"></a><span class="st"> #Estimate residual variances of manifest variables</span></span>
<span id="cb1727-23"><a href="bias.html#cb1727-23" tabindex="-1"></a><span class="st"> x1 ~~ x1</span></span>
<span id="cb1727-24"><a href="bias.html#cb1727-24" tabindex="-1"></a><span class="st"> x2 ~~ x2</span></span>
<span id="cb1727-25"><a href="bias.html#cb1727-25" tabindex="-1"></a><span class="st"> x3 ~~ x3</span></span>
<span id="cb1727-26"><a href="bias.html#cb1727-26" tabindex="-1"></a><span class="st"> x4 ~~ x4</span></span>
<span id="cb1727-27"><a href="bias.html#cb1727-27" tabindex="-1"></a><span class="st"> x5 ~~ x5</span></span>
<span id="cb1727-28"><a href="bias.html#cb1727-28" tabindex="-1"></a><span class="st"> x6 ~~ x6</span></span>
<span id="cb1727-29"><a href="bias.html#cb1727-29" tabindex="-1"></a><span class="st"> x7 ~~ x7</span></span>
<span id="cb1727-30"><a href="bias.html#cb1727-30" tabindex="-1"></a><span class="st"> x8 ~~ x8</span></span>
<span id="cb1727-31"><a href="bias.html#cb1727-31" tabindex="-1"></a><span class="st"> x9 ~~ x9</span></span>
<span id="cb1727-32"><a href="bias.html#cb1727-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1727-33"><a href="bias.html#cb1727-33" tabindex="-1"></a><span class="st"> #Iteratively free intercepts across groups</span></span>
<span id="cb1727-34"><a href="bias.html#cb1727-34" tabindex="-1"></a><span class="st"> x1 ~ NA*1</span></span>
<span id="cb1727-35"><a href="bias.html#cb1727-35" tabindex="-1"></a><span class="st"> x2 ~ NA*1</span></span>
<span id="cb1727-36"><a href="bias.html#cb1727-36" tabindex="-1"></a><span class="st"> x3 ~ c(intx3, intx3)*1</span></span>
<span id="cb1727-37"><a href="bias.html#cb1727-37" tabindex="-1"></a><span class="st"> x4 ~ c(intx4, intx4)*1</span></span>
<span id="cb1727-38"><a href="bias.html#cb1727-38" tabindex="-1"></a><span class="st"> x5 ~ c(intx5, intx5)*1</span></span>
<span id="cb1727-39"><a href="bias.html#cb1727-39" tabindex="-1"></a><span class="st"> x6 ~ c(intx6, intx6)*1</span></span>
<span id="cb1727-40"><a href="bias.html#cb1727-40" tabindex="-1"></a><span class="st"> x7 ~ c(intx7, intx7)*1</span></span>
<span id="cb1727-41"><a href="bias.html#cb1727-41" tabindex="-1"></a><span class="st"> x8 ~ c(intx8, intx8)*1</span></span>
<span id="cb1727-42"><a href="bias.html#cb1727-42" tabindex="-1"></a><span class="st"> x9 ~ c(intx9, intx9)*1</span></span>
<span id="cb1727-43"><a href="bias.html#cb1727-43" tabindex="-1"></a><span class="st">&#39;</span></span>
<span id="cb1727-44"><a href="bias.html#cb1727-44" tabindex="-1"></a></span>
<span id="cb1727-45"><a href="bias.html#cb1727-45" tabindex="-1"></a>cfaModel_scalarInvarianceV2_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1727-46"><a href="bias.html#cb1727-46" tabindex="-1"></a>  cfaModel_scalarInvarianceV2,</span>
<span id="cb1727-47"><a href="bias.html#cb1727-47" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1727-48"><a href="bias.html#cb1727-48" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1727-49"><a href="bias.html#cb1727-49" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1727-50"><a href="bias.html#cb1727-50" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span>
<span id="cb1727-51"><a href="bias.html#cb1727-51" tabindex="-1"></a></span>
<span id="cb1727-52"><a href="bias.html#cb1727-52" tabindex="-1"></a><span class="fu">anova</span>(metricInvarianceModel_fit, cfaModel_scalarInvarianceV2_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Df"],"name":[1],"type":["int"],"align":["right"]},{"label":["AIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Chisq"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Chisq diff"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Df diff"],"name":[6],"type":["int"],"align":["right"]},{"label":["Pr(>Chisq)"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"54","2":"7412.554","3":"7612.738","4":"116.3133","5":"NA","6":"NA","7":"NA","_rn_":"metricInvarianceModel_fit"},{"1":"61","2":"7475.037","3":"7649.272","4":"192.7970","5":"80.68887","6":"7","7":"9.967365e-15","_rn_":"cfaModel_scalarInvarianceV2_fit"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceDropConstraintsVariable3" class="section level5 hasAnchor" number="16.10.8.2.3">
<h5><span class="header-section-number">16.10.8.2.3</span> Variable 3<a href="bias.html#measurementInvarianceDropConstraintsVariable3" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Measurement invariance does not yet become established when freeing intercepts of variable 3 across groups.
This suggests that, whether or not variables 1, 2, and 3 show non-invariant intercepts, there are other items that show non-invariant intercepts.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1728"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1728-1"><a href="bias.html#cb1728-1" tabindex="-1"></a>cfaModel_scalarInvarianceV3 <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1728-2"><a href="bias.html#cb1728-2" tabindex="-1"></a><span class="st"> #Fix factor loadings to be the same across groups</span></span>
<span id="cb1728-3"><a href="bias.html#cb1728-3" tabindex="-1"></a><span class="st"> visual  =~ c(lambdax1,lambdax1)*x1 + c(lambdax2,lambdax2)*x2 + c(lambdax3,lambdax3)*x3</span></span>
<span id="cb1728-4"><a href="bias.html#cb1728-4" tabindex="-1"></a><span class="st"> textual =~ c(lambdax4,lambdax4)*x4 + c(lambdax5,lambdax5)*x5 + c(lambdax6,lambdax6)*x6</span></span>
<span id="cb1728-5"><a href="bias.html#cb1728-5" tabindex="-1"></a><span class="st"> speed   =~ c(lambdax7,lambdax7)*x7 + c(lambdax8,lambdax8)*x8 + c(lambdax9,lambdax9)*x9</span></span>
<span id="cb1728-6"><a href="bias.html#cb1728-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1728-7"><a href="bias.html#cb1728-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero</span></span>
<span id="cb1728-8"><a href="bias.html#cb1728-8" tabindex="-1"></a><span class="st"> visual ~ 0</span></span>
<span id="cb1728-9"><a href="bias.html#cb1728-9" tabindex="-1"></a><span class="st"> textual ~ 0</span></span>
<span id="cb1728-10"><a href="bias.html#cb1728-10" tabindex="-1"></a><span class="st"> speed ~ 0</span></span>
<span id="cb1728-11"><a href="bias.html#cb1728-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1728-12"><a href="bias.html#cb1728-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one in group 1; free latent variances in group 2</span></span>
<span id="cb1728-13"><a href="bias.html#cb1728-13" tabindex="-1"></a><span class="st"> visual ~~ c(1, NA)*visual</span></span>
<span id="cb1728-14"><a href="bias.html#cb1728-14" tabindex="-1"></a><span class="st"> textual ~~ c(1, NA)*textual</span></span>
<span id="cb1728-15"><a href="bias.html#cb1728-15" tabindex="-1"></a><span class="st"> speed ~~ c(1, NA)*speed</span></span>
<span id="cb1728-16"><a href="bias.html#cb1728-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1728-17"><a href="bias.html#cb1728-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1728-18"><a href="bias.html#cb1728-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1728-19"><a href="bias.html#cb1728-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1728-20"><a href="bias.html#cb1728-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1728-21"><a href="bias.html#cb1728-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1728-22"><a href="bias.html#cb1728-22" tabindex="-1"></a><span class="st"> #Estimate residual variances of manifest variables</span></span>
<span id="cb1728-23"><a href="bias.html#cb1728-23" tabindex="-1"></a><span class="st"> x1 ~~ x1</span></span>
<span id="cb1728-24"><a href="bias.html#cb1728-24" tabindex="-1"></a><span class="st"> x2 ~~ x2</span></span>
<span id="cb1728-25"><a href="bias.html#cb1728-25" tabindex="-1"></a><span class="st"> x3 ~~ x3</span></span>
<span id="cb1728-26"><a href="bias.html#cb1728-26" tabindex="-1"></a><span class="st"> x4 ~~ x4</span></span>
<span id="cb1728-27"><a href="bias.html#cb1728-27" tabindex="-1"></a><span class="st"> x5 ~~ x5</span></span>
<span id="cb1728-28"><a href="bias.html#cb1728-28" tabindex="-1"></a><span class="st"> x6 ~~ x6</span></span>
<span id="cb1728-29"><a href="bias.html#cb1728-29" tabindex="-1"></a><span class="st"> x7 ~~ x7</span></span>
<span id="cb1728-30"><a href="bias.html#cb1728-30" tabindex="-1"></a><span class="st"> x8 ~~ x8</span></span>
<span id="cb1728-31"><a href="bias.html#cb1728-31" tabindex="-1"></a><span class="st"> x9 ~~ x9</span></span>
<span id="cb1728-32"><a href="bias.html#cb1728-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1728-33"><a href="bias.html#cb1728-33" tabindex="-1"></a><span class="st"> #Iteratively free intercepts across groups</span></span>
<span id="cb1728-34"><a href="bias.html#cb1728-34" tabindex="-1"></a><span class="st"> x1 ~ NA*1</span></span>
<span id="cb1728-35"><a href="bias.html#cb1728-35" tabindex="-1"></a><span class="st"> x2 ~ NA*1</span></span>
<span id="cb1728-36"><a href="bias.html#cb1728-36" tabindex="-1"></a><span class="st"> x3 ~ NA*1</span></span>
<span id="cb1728-37"><a href="bias.html#cb1728-37" tabindex="-1"></a><span class="st"> x4 ~ c(intx4, intx4)*1</span></span>
<span id="cb1728-38"><a href="bias.html#cb1728-38" tabindex="-1"></a><span class="st"> x5 ~ c(intx5, intx5)*1</span></span>
<span id="cb1728-39"><a href="bias.html#cb1728-39" tabindex="-1"></a><span class="st"> x6 ~ c(intx6, intx6)*1</span></span>
<span id="cb1728-40"><a href="bias.html#cb1728-40" tabindex="-1"></a><span class="st"> x7 ~ c(intx7, intx7)*1</span></span>
<span id="cb1728-41"><a href="bias.html#cb1728-41" tabindex="-1"></a><span class="st"> x8 ~ c(intx8, intx8)*1</span></span>
<span id="cb1728-42"><a href="bias.html#cb1728-42" tabindex="-1"></a><span class="st"> x9 ~ c(intx9, intx9)*1</span></span>
<span id="cb1728-43"><a href="bias.html#cb1728-43" tabindex="-1"></a><span class="st">&#39;</span></span>
<span id="cb1728-44"><a href="bias.html#cb1728-44" tabindex="-1"></a></span>
<span id="cb1728-45"><a href="bias.html#cb1728-45" tabindex="-1"></a>cfaModel_scalarInvarianceV3_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1728-46"><a href="bias.html#cb1728-46" tabindex="-1"></a>  cfaModel_scalarInvarianceV3,</span>
<span id="cb1728-47"><a href="bias.html#cb1728-47" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1728-48"><a href="bias.html#cb1728-48" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1728-49"><a href="bias.html#cb1728-49" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1728-50"><a href="bias.html#cb1728-50" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span>
<span id="cb1728-51"><a href="bias.html#cb1728-51" tabindex="-1"></a></span>
<span id="cb1728-52"><a href="bias.html#cb1728-52" tabindex="-1"></a><span class="fu">anova</span>(metricInvarianceModel_fit, cfaModel_scalarInvarianceV3_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Df"],"name":[1],"type":["int"],"align":["right"]},{"label":["AIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Chisq"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Chisq diff"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Df diff"],"name":[6],"type":["int"],"align":["right"]},{"label":["Pr(>Chisq)"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"54","2":"7412.554","3":"7612.738","4":"116.3133","5":"NA","6":"NA","7":"NA","_rn_":"metricInvarianceModel_fit"},{"1":"60","2":"7451.422","3":"7629.364","4":"167.1822","5":"51.92834","6":"6","7":"1.927729e-09","_rn_":"cfaModel_scalarInvarianceV3_fit"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceDropConstraintsVariable4" class="section level5 hasAnchor" number="16.10.8.2.4">
<h5><span class="header-section-number">16.10.8.2.4</span> Variable 4<a href="bias.html#measurementInvarianceDropConstraintsVariable4" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Measurement invariance does not yet become established when freeing intercepts of variable 4 across groups.
This suggests that, whether or not variables 1, 2, 3, and 4 show non-invariant intercepts, there are other items that show non-invariant intercepts.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1729"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1729-1"><a href="bias.html#cb1729-1" tabindex="-1"></a>cfaModel_scalarInvarianceV4 <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1729-2"><a href="bias.html#cb1729-2" tabindex="-1"></a><span class="st"> #Fix factor loadings to be the same across groups</span></span>
<span id="cb1729-3"><a href="bias.html#cb1729-3" tabindex="-1"></a><span class="st"> visual  =~ c(lambdax1,lambdax1)*x1 + c(lambdax2,lambdax2)*x2 + c(lambdax3,lambdax3)*x3</span></span>
<span id="cb1729-4"><a href="bias.html#cb1729-4" tabindex="-1"></a><span class="st"> textual =~ c(lambdax4,lambdax4)*x4 + c(lambdax5,lambdax5)*x5 + c(lambdax6,lambdax6)*x6</span></span>
<span id="cb1729-5"><a href="bias.html#cb1729-5" tabindex="-1"></a><span class="st"> speed   =~ c(lambdax7,lambdax7)*x7 + c(lambdax8,lambdax8)*x8 + c(lambdax9,lambdax9)*x9</span></span>
<span id="cb1729-6"><a href="bias.html#cb1729-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1729-7"><a href="bias.html#cb1729-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero</span></span>
<span id="cb1729-8"><a href="bias.html#cb1729-8" tabindex="-1"></a><span class="st"> visual ~ 0</span></span>
<span id="cb1729-9"><a href="bias.html#cb1729-9" tabindex="-1"></a><span class="st"> textual ~ 0</span></span>
<span id="cb1729-10"><a href="bias.html#cb1729-10" tabindex="-1"></a><span class="st"> speed ~ 0</span></span>
<span id="cb1729-11"><a href="bias.html#cb1729-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1729-12"><a href="bias.html#cb1729-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one in group 1; free latent variances in group 2</span></span>
<span id="cb1729-13"><a href="bias.html#cb1729-13" tabindex="-1"></a><span class="st"> visual ~~ c(1, NA)*visual</span></span>
<span id="cb1729-14"><a href="bias.html#cb1729-14" tabindex="-1"></a><span class="st"> textual ~~ c(1, NA)*textual</span></span>
<span id="cb1729-15"><a href="bias.html#cb1729-15" tabindex="-1"></a><span class="st"> speed ~~ c(1, NA)*speed</span></span>
<span id="cb1729-16"><a href="bias.html#cb1729-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1729-17"><a href="bias.html#cb1729-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1729-18"><a href="bias.html#cb1729-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1729-19"><a href="bias.html#cb1729-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1729-20"><a href="bias.html#cb1729-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1729-21"><a href="bias.html#cb1729-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1729-22"><a href="bias.html#cb1729-22" tabindex="-1"></a><span class="st"> #Estimate residual variances of manifest variables</span></span>
<span id="cb1729-23"><a href="bias.html#cb1729-23" tabindex="-1"></a><span class="st"> x1 ~~ x1</span></span>
<span id="cb1729-24"><a href="bias.html#cb1729-24" tabindex="-1"></a><span class="st"> x2 ~~ x2</span></span>
<span id="cb1729-25"><a href="bias.html#cb1729-25" tabindex="-1"></a><span class="st"> x3 ~~ x3</span></span>
<span id="cb1729-26"><a href="bias.html#cb1729-26" tabindex="-1"></a><span class="st"> x4 ~~ x4</span></span>
<span id="cb1729-27"><a href="bias.html#cb1729-27" tabindex="-1"></a><span class="st"> x5 ~~ x5</span></span>
<span id="cb1729-28"><a href="bias.html#cb1729-28" tabindex="-1"></a><span class="st"> x6 ~~ x6</span></span>
<span id="cb1729-29"><a href="bias.html#cb1729-29" tabindex="-1"></a><span class="st"> x7 ~~ x7</span></span>
<span id="cb1729-30"><a href="bias.html#cb1729-30" tabindex="-1"></a><span class="st"> x8 ~~ x8</span></span>
<span id="cb1729-31"><a href="bias.html#cb1729-31" tabindex="-1"></a><span class="st"> x9 ~~ x9</span></span>
<span id="cb1729-32"><a href="bias.html#cb1729-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1729-33"><a href="bias.html#cb1729-33" tabindex="-1"></a><span class="st"> #Iteratively free intercepts across groups</span></span>
<span id="cb1729-34"><a href="bias.html#cb1729-34" tabindex="-1"></a><span class="st"> x1 ~ NA*1</span></span>
<span id="cb1729-35"><a href="bias.html#cb1729-35" tabindex="-1"></a><span class="st"> x2 ~ NA*1</span></span>
<span id="cb1729-36"><a href="bias.html#cb1729-36" tabindex="-1"></a><span class="st"> x3 ~ NA*1</span></span>
<span id="cb1729-37"><a href="bias.html#cb1729-37" tabindex="-1"></a><span class="st"> x4 ~ NA*1</span></span>
<span id="cb1729-38"><a href="bias.html#cb1729-38" tabindex="-1"></a><span class="st"> x5 ~ c(intx5, intx5)*1</span></span>
<span id="cb1729-39"><a href="bias.html#cb1729-39" tabindex="-1"></a><span class="st"> x6 ~ c(intx6, intx6)*1</span></span>
<span id="cb1729-40"><a href="bias.html#cb1729-40" tabindex="-1"></a><span class="st"> x7 ~ c(intx7, intx7)*1</span></span>
<span id="cb1729-41"><a href="bias.html#cb1729-41" tabindex="-1"></a><span class="st"> x8 ~ c(intx8, intx8)*1</span></span>
<span id="cb1729-42"><a href="bias.html#cb1729-42" tabindex="-1"></a><span class="st"> x9 ~ c(intx9, intx9)*1</span></span>
<span id="cb1729-43"><a href="bias.html#cb1729-43" tabindex="-1"></a><span class="st">&#39;</span></span>
<span id="cb1729-44"><a href="bias.html#cb1729-44" tabindex="-1"></a></span>
<span id="cb1729-45"><a href="bias.html#cb1729-45" tabindex="-1"></a>cfaModel_scalarInvarianceV4_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1729-46"><a href="bias.html#cb1729-46" tabindex="-1"></a>  cfaModel_scalarInvarianceV4,</span>
<span id="cb1729-47"><a href="bias.html#cb1729-47" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1729-48"><a href="bias.html#cb1729-48" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1729-49"><a href="bias.html#cb1729-49" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1729-50"><a href="bias.html#cb1729-50" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span>
<span id="cb1729-51"><a href="bias.html#cb1729-51" tabindex="-1"></a></span>
<span id="cb1729-52"><a href="bias.html#cb1729-52" tabindex="-1"></a><span class="fu">anova</span>(metricInvarianceModel_fit, cfaModel_scalarInvarianceV4_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Df"],"name":[1],"type":["int"],"align":["right"]},{"label":["AIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Chisq"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Chisq diff"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Df diff"],"name":[6],"type":["int"],"align":["right"]},{"label":["Pr(>Chisq)"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"54","2":"7412.554","3":"7612.738","4":"116.3133","5":"NA","6":"NA","7":"NA","_rn_":"metricInvarianceModel_fit"},{"1":"59","2":"7453.394","3":"7635.042","4":"167.1532","5":"52.06933","6":"5","7":"5.221006e-10","_rn_":"cfaModel_scalarInvarianceV4_fit"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceDropConstraintsVariable5" class="section level5 hasAnchor" number="16.10.8.2.5">
<h5><span class="header-section-number">16.10.8.2.5</span> Variable 5<a href="bias.html#measurementInvarianceDropConstraintsVariable5" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Measurement invariance does not yet become established when freeing intercepts of variable 5 across groups.
This suggests that, whether or not variables 1, 2, 3, 4, and 5 show non-invariant intercepts, there are other items that show non-invariant intercepts.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1730"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1730-1"><a href="bias.html#cb1730-1" tabindex="-1"></a>cfaModel_scalarInvarianceV5 <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1730-2"><a href="bias.html#cb1730-2" tabindex="-1"></a><span class="st"> #Fix factor loadings to be the same across groups</span></span>
<span id="cb1730-3"><a href="bias.html#cb1730-3" tabindex="-1"></a><span class="st"> visual  =~ c(lambdax1,lambdax1)*x1 + c(lambdax2,lambdax2)*x2 + c(lambdax3,lambdax3)*x3</span></span>
<span id="cb1730-4"><a href="bias.html#cb1730-4" tabindex="-1"></a><span class="st"> textual =~ c(lambdax4,lambdax4)*x4 + c(lambdax5,lambdax5)*x5 + c(lambdax6,lambdax6)*x6</span></span>
<span id="cb1730-5"><a href="bias.html#cb1730-5" tabindex="-1"></a><span class="st"> speed   =~ c(lambdax7,lambdax7)*x7 + c(lambdax8,lambdax8)*x8 + c(lambdax9,lambdax9)*x9</span></span>
<span id="cb1730-6"><a href="bias.html#cb1730-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1730-7"><a href="bias.html#cb1730-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero</span></span>
<span id="cb1730-8"><a href="bias.html#cb1730-8" tabindex="-1"></a><span class="st"> visual ~ 0</span></span>
<span id="cb1730-9"><a href="bias.html#cb1730-9" tabindex="-1"></a><span class="st"> textual ~ 0</span></span>
<span id="cb1730-10"><a href="bias.html#cb1730-10" tabindex="-1"></a><span class="st"> speed ~ 0</span></span>
<span id="cb1730-11"><a href="bias.html#cb1730-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1730-12"><a href="bias.html#cb1730-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one in group 1; free latent variances in group 2</span></span>
<span id="cb1730-13"><a href="bias.html#cb1730-13" tabindex="-1"></a><span class="st"> visual ~~ c(1, NA)*visual</span></span>
<span id="cb1730-14"><a href="bias.html#cb1730-14" tabindex="-1"></a><span class="st"> textual ~~ c(1, NA)*textual</span></span>
<span id="cb1730-15"><a href="bias.html#cb1730-15" tabindex="-1"></a><span class="st"> speed ~~ c(1, NA)*speed</span></span>
<span id="cb1730-16"><a href="bias.html#cb1730-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1730-17"><a href="bias.html#cb1730-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1730-18"><a href="bias.html#cb1730-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1730-19"><a href="bias.html#cb1730-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1730-20"><a href="bias.html#cb1730-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1730-21"><a href="bias.html#cb1730-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1730-22"><a href="bias.html#cb1730-22" tabindex="-1"></a><span class="st"> #Estimate residual variances of manifest variables</span></span>
<span id="cb1730-23"><a href="bias.html#cb1730-23" tabindex="-1"></a><span class="st"> x1 ~~ x1</span></span>
<span id="cb1730-24"><a href="bias.html#cb1730-24" tabindex="-1"></a><span class="st"> x2 ~~ x2</span></span>
<span id="cb1730-25"><a href="bias.html#cb1730-25" tabindex="-1"></a><span class="st"> x3 ~~ x3</span></span>
<span id="cb1730-26"><a href="bias.html#cb1730-26" tabindex="-1"></a><span class="st"> x4 ~~ x4</span></span>
<span id="cb1730-27"><a href="bias.html#cb1730-27" tabindex="-1"></a><span class="st"> x5 ~~ x5</span></span>
<span id="cb1730-28"><a href="bias.html#cb1730-28" tabindex="-1"></a><span class="st"> x6 ~~ x6</span></span>
<span id="cb1730-29"><a href="bias.html#cb1730-29" tabindex="-1"></a><span class="st"> x7 ~~ x7</span></span>
<span id="cb1730-30"><a href="bias.html#cb1730-30" tabindex="-1"></a><span class="st"> x8 ~~ x8</span></span>
<span id="cb1730-31"><a href="bias.html#cb1730-31" tabindex="-1"></a><span class="st"> x9 ~~ x9</span></span>
<span id="cb1730-32"><a href="bias.html#cb1730-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1730-33"><a href="bias.html#cb1730-33" tabindex="-1"></a><span class="st"> #Iteratively free intercepts across groups</span></span>
<span id="cb1730-34"><a href="bias.html#cb1730-34" tabindex="-1"></a><span class="st"> x1 ~ NA*1</span></span>
<span id="cb1730-35"><a href="bias.html#cb1730-35" tabindex="-1"></a><span class="st"> x2 ~ NA*1</span></span>
<span id="cb1730-36"><a href="bias.html#cb1730-36" tabindex="-1"></a><span class="st"> x3 ~ NA*1</span></span>
<span id="cb1730-37"><a href="bias.html#cb1730-37" tabindex="-1"></a><span class="st"> x4 ~ NA*1</span></span>
<span id="cb1730-38"><a href="bias.html#cb1730-38" tabindex="-1"></a><span class="st"> x5 ~ NA*1</span></span>
<span id="cb1730-39"><a href="bias.html#cb1730-39" tabindex="-1"></a><span class="st"> x6 ~ c(intx6, intx6)*1</span></span>
<span id="cb1730-40"><a href="bias.html#cb1730-40" tabindex="-1"></a><span class="st"> x7 ~ c(intx7, intx7)*1</span></span>
<span id="cb1730-41"><a href="bias.html#cb1730-41" tabindex="-1"></a><span class="st"> x8 ~ c(intx8, intx8)*1</span></span>
<span id="cb1730-42"><a href="bias.html#cb1730-42" tabindex="-1"></a><span class="st"> x9 ~ c(intx9, intx9)*1</span></span>
<span id="cb1730-43"><a href="bias.html#cb1730-43" tabindex="-1"></a><span class="st">&#39;</span></span>
<span id="cb1730-44"><a href="bias.html#cb1730-44" tabindex="-1"></a></span>
<span id="cb1730-45"><a href="bias.html#cb1730-45" tabindex="-1"></a>cfaModel_scalarInvarianceV5_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1730-46"><a href="bias.html#cb1730-46" tabindex="-1"></a>  cfaModel_scalarInvarianceV5,</span>
<span id="cb1730-47"><a href="bias.html#cb1730-47" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1730-48"><a href="bias.html#cb1730-48" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1730-49"><a href="bias.html#cb1730-49" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1730-50"><a href="bias.html#cb1730-50" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span>
<span id="cb1730-51"><a href="bias.html#cb1730-51" tabindex="-1"></a></span>
<span id="cb1730-52"><a href="bias.html#cb1730-52" tabindex="-1"></a><span class="fu">anova</span>(metricInvarianceModel_fit, cfaModel_scalarInvarianceV5_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Df"],"name":[1],"type":["int"],"align":["right"]},{"label":["AIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Chisq"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Chisq diff"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Df diff"],"name":[6],"type":["int"],"align":["right"]},{"label":["Pr(>Chisq)"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"54","2":"7412.554","3":"7612.738","4":"116.3133","5":"NA","6":"NA","7":"NA","_rn_":"metricInvarianceModel_fit"},{"1":"58","2":"7447.221","3":"7632.576","4":"158.9803","5":"43.71191","6":"4","7":"7.363407e-09","_rn_":"cfaModel_scalarInvarianceV5_fit"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceDropConstraintsVariable6" class="section level5 hasAnchor" number="16.10.8.2.6">
<h5><span class="header-section-number">16.10.8.2.6</span> Variable 6<a href="bias.html#measurementInvarianceDropConstraintsVariable6" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Measurement invariance does not yet become established when freeing intercepts of variable 6 across groups.
This suggests that, whether or not variables 1, 2, 3, 4, 5, and 6 show non-invariant intercepts, there are other items that show non-invariant intercepts.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1731"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1731-1"><a href="bias.html#cb1731-1" tabindex="-1"></a>cfaModel_scalarInvarianceV6 <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1731-2"><a href="bias.html#cb1731-2" tabindex="-1"></a><span class="st"> #Fix factor loadings to be the same across groups</span></span>
<span id="cb1731-3"><a href="bias.html#cb1731-3" tabindex="-1"></a><span class="st"> visual  =~ c(lambdax1,lambdax1)*x1 + c(lambdax2,lambdax2)*x2 + c(lambdax3,lambdax3)*x3</span></span>
<span id="cb1731-4"><a href="bias.html#cb1731-4" tabindex="-1"></a><span class="st"> textual =~ c(lambdax4,lambdax4)*x4 + c(lambdax5,lambdax5)*x5 + c(lambdax6,lambdax6)*x6</span></span>
<span id="cb1731-5"><a href="bias.html#cb1731-5" tabindex="-1"></a><span class="st"> speed   =~ c(lambdax7,lambdax7)*x7 + c(lambdax8,lambdax8)*x8 + c(lambdax9,lambdax9)*x9</span></span>
<span id="cb1731-6"><a href="bias.html#cb1731-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1731-7"><a href="bias.html#cb1731-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero</span></span>
<span id="cb1731-8"><a href="bias.html#cb1731-8" tabindex="-1"></a><span class="st"> visual ~ 0</span></span>
<span id="cb1731-9"><a href="bias.html#cb1731-9" tabindex="-1"></a><span class="st"> textual ~ 0</span></span>
<span id="cb1731-10"><a href="bias.html#cb1731-10" tabindex="-1"></a><span class="st"> speed ~ 0</span></span>
<span id="cb1731-11"><a href="bias.html#cb1731-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1731-12"><a href="bias.html#cb1731-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one in group 1; free latent variances in group 2</span></span>
<span id="cb1731-13"><a href="bias.html#cb1731-13" tabindex="-1"></a><span class="st"> visual ~~ c(1, NA)*visual</span></span>
<span id="cb1731-14"><a href="bias.html#cb1731-14" tabindex="-1"></a><span class="st"> textual ~~ c(1, NA)*textual</span></span>
<span id="cb1731-15"><a href="bias.html#cb1731-15" tabindex="-1"></a><span class="st"> speed ~~ c(1, NA)*speed</span></span>
<span id="cb1731-16"><a href="bias.html#cb1731-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1731-17"><a href="bias.html#cb1731-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1731-18"><a href="bias.html#cb1731-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1731-19"><a href="bias.html#cb1731-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1731-20"><a href="bias.html#cb1731-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1731-21"><a href="bias.html#cb1731-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1731-22"><a href="bias.html#cb1731-22" tabindex="-1"></a><span class="st"> #Estimate residual variances of manifest variables</span></span>
<span id="cb1731-23"><a href="bias.html#cb1731-23" tabindex="-1"></a><span class="st"> x1 ~~ x1</span></span>
<span id="cb1731-24"><a href="bias.html#cb1731-24" tabindex="-1"></a><span class="st"> x2 ~~ x2</span></span>
<span id="cb1731-25"><a href="bias.html#cb1731-25" tabindex="-1"></a><span class="st"> x3 ~~ x3</span></span>
<span id="cb1731-26"><a href="bias.html#cb1731-26" tabindex="-1"></a><span class="st"> x4 ~~ x4</span></span>
<span id="cb1731-27"><a href="bias.html#cb1731-27" tabindex="-1"></a><span class="st"> x5 ~~ x5</span></span>
<span id="cb1731-28"><a href="bias.html#cb1731-28" tabindex="-1"></a><span class="st"> x6 ~~ x6</span></span>
<span id="cb1731-29"><a href="bias.html#cb1731-29" tabindex="-1"></a><span class="st"> x7 ~~ x7</span></span>
<span id="cb1731-30"><a href="bias.html#cb1731-30" tabindex="-1"></a><span class="st"> x8 ~~ x8</span></span>
<span id="cb1731-31"><a href="bias.html#cb1731-31" tabindex="-1"></a><span class="st"> x9 ~~ x9</span></span>
<span id="cb1731-32"><a href="bias.html#cb1731-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1731-33"><a href="bias.html#cb1731-33" tabindex="-1"></a><span class="st"> #Iteratively free intercepts across groups</span></span>
<span id="cb1731-34"><a href="bias.html#cb1731-34" tabindex="-1"></a><span class="st"> x1 ~ NA*1</span></span>
<span id="cb1731-35"><a href="bias.html#cb1731-35" tabindex="-1"></a><span class="st"> x2 ~ NA*1</span></span>
<span id="cb1731-36"><a href="bias.html#cb1731-36" tabindex="-1"></a><span class="st"> x3 ~ NA*1</span></span>
<span id="cb1731-37"><a href="bias.html#cb1731-37" tabindex="-1"></a><span class="st"> x4 ~ NA*1</span></span>
<span id="cb1731-38"><a href="bias.html#cb1731-38" tabindex="-1"></a><span class="st"> x5 ~ NA*1</span></span>
<span id="cb1731-39"><a href="bias.html#cb1731-39" tabindex="-1"></a><span class="st"> x6 ~ NA*1</span></span>
<span id="cb1731-40"><a href="bias.html#cb1731-40" tabindex="-1"></a><span class="st"> x7 ~ c(intx7, intx7)*1</span></span>
<span id="cb1731-41"><a href="bias.html#cb1731-41" tabindex="-1"></a><span class="st"> x8 ~ c(intx8, intx8)*1</span></span>
<span id="cb1731-42"><a href="bias.html#cb1731-42" tabindex="-1"></a><span class="st"> x9 ~ c(intx9, intx9)*1</span></span>
<span id="cb1731-43"><a href="bias.html#cb1731-43" tabindex="-1"></a><span class="st">&#39;</span></span>
<span id="cb1731-44"><a href="bias.html#cb1731-44" tabindex="-1"></a></span>
<span id="cb1731-45"><a href="bias.html#cb1731-45" tabindex="-1"></a>cfaModel_scalarInvarianceV6_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1731-46"><a href="bias.html#cb1731-46" tabindex="-1"></a>  cfaModel_scalarInvarianceV6,</span>
<span id="cb1731-47"><a href="bias.html#cb1731-47" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1731-48"><a href="bias.html#cb1731-48" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1731-49"><a href="bias.html#cb1731-49" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1731-50"><a href="bias.html#cb1731-50" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span>
<span id="cb1731-51"><a href="bias.html#cb1731-51" tabindex="-1"></a></span>
<span id="cb1731-52"><a href="bias.html#cb1731-52" tabindex="-1"></a><span class="fu">anova</span>(metricInvarianceModel_fit, cfaModel_scalarInvarianceV6_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Df"],"name":[1],"type":["int"],"align":["right"]},{"label":["AIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Chisq"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Chisq diff"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Df diff"],"name":[6],"type":["int"],"align":["right"]},{"label":["Pr(>Chisq)"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"54","2":"7412.554","3":"7612.738","4":"116.3133","5":"NA","6":"NA","7":"NA","_rn_":"metricInvarianceModel_fit"},{"1":"57","2":"7425.539","3":"7614.601","4":"135.2983","5":"20.5684","6":"3","7":"0.0001293964","_rn_":"cfaModel_scalarInvarianceV6_fit"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceDropConstraintsVariable7" class="section level5 hasAnchor" number="16.10.8.2.7">
<h5><span class="header-section-number">16.10.8.2.7</span> Variable 7<a href="bias.html#measurementInvarianceDropConstraintsVariable7" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Measurement invariance becomes established when freeing intercepts of variable 7 across groups.
This suggests that item 7 shows non-invariant intercepts across groups, and that measurement invariance holds when constraining items 8 and 9 to be invariant across groups.
There may also be other items, especially among the preceding items (variables 1–6), that show non-invariant intercepts across groups.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1732"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1732-1"><a href="bias.html#cb1732-1" tabindex="-1"></a>cfaModel_scalarInvarianceV7 <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1732-2"><a href="bias.html#cb1732-2" tabindex="-1"></a><span class="st"> #Fix factor loadings to be the same across groups</span></span>
<span id="cb1732-3"><a href="bias.html#cb1732-3" tabindex="-1"></a><span class="st"> visual  =~ c(lambdax1,lambdax1)*x1 + c(lambdax2,lambdax2)*x2 + c(lambdax3,lambdax3)*x3</span></span>
<span id="cb1732-4"><a href="bias.html#cb1732-4" tabindex="-1"></a><span class="st"> textual =~ c(lambdax4,lambdax4)*x4 + c(lambdax5,lambdax5)*x5 + c(lambdax6,lambdax6)*x6</span></span>
<span id="cb1732-5"><a href="bias.html#cb1732-5" tabindex="-1"></a><span class="st"> speed   =~ c(lambdax7,lambdax7)*x7 + c(lambdax8,lambdax8)*x8 + c(lambdax9,lambdax9)*x9</span></span>
<span id="cb1732-6"><a href="bias.html#cb1732-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1732-7"><a href="bias.html#cb1732-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero</span></span>
<span id="cb1732-8"><a href="bias.html#cb1732-8" tabindex="-1"></a><span class="st"> visual ~ 0</span></span>
<span id="cb1732-9"><a href="bias.html#cb1732-9" tabindex="-1"></a><span class="st"> textual ~ 0</span></span>
<span id="cb1732-10"><a href="bias.html#cb1732-10" tabindex="-1"></a><span class="st"> speed ~ 0</span></span>
<span id="cb1732-11"><a href="bias.html#cb1732-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1732-12"><a href="bias.html#cb1732-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one in group 1; free latent variances in group 2</span></span>
<span id="cb1732-13"><a href="bias.html#cb1732-13" tabindex="-1"></a><span class="st"> visual ~~ c(1, NA)*visual</span></span>
<span id="cb1732-14"><a href="bias.html#cb1732-14" tabindex="-1"></a><span class="st"> textual ~~ c(1, NA)*textual</span></span>
<span id="cb1732-15"><a href="bias.html#cb1732-15" tabindex="-1"></a><span class="st"> speed ~~ c(1, NA)*speed</span></span>
<span id="cb1732-16"><a href="bias.html#cb1732-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1732-17"><a href="bias.html#cb1732-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1732-18"><a href="bias.html#cb1732-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1732-19"><a href="bias.html#cb1732-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1732-20"><a href="bias.html#cb1732-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1732-21"><a href="bias.html#cb1732-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1732-22"><a href="bias.html#cb1732-22" tabindex="-1"></a><span class="st"> #Estimate residual variances of manifest variables</span></span>
<span id="cb1732-23"><a href="bias.html#cb1732-23" tabindex="-1"></a><span class="st"> x1 ~~ x1</span></span>
<span id="cb1732-24"><a href="bias.html#cb1732-24" tabindex="-1"></a><span class="st"> x2 ~~ x2</span></span>
<span id="cb1732-25"><a href="bias.html#cb1732-25" tabindex="-1"></a><span class="st"> x3 ~~ x3</span></span>
<span id="cb1732-26"><a href="bias.html#cb1732-26" tabindex="-1"></a><span class="st"> x4 ~~ x4</span></span>
<span id="cb1732-27"><a href="bias.html#cb1732-27" tabindex="-1"></a><span class="st"> x5 ~~ x5</span></span>
<span id="cb1732-28"><a href="bias.html#cb1732-28" tabindex="-1"></a><span class="st"> x6 ~~ x6</span></span>
<span id="cb1732-29"><a href="bias.html#cb1732-29" tabindex="-1"></a><span class="st"> x7 ~~ x7</span></span>
<span id="cb1732-30"><a href="bias.html#cb1732-30" tabindex="-1"></a><span class="st"> x8 ~~ x8</span></span>
<span id="cb1732-31"><a href="bias.html#cb1732-31" tabindex="-1"></a><span class="st"> x9 ~~ x9</span></span>
<span id="cb1732-32"><a href="bias.html#cb1732-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1732-33"><a href="bias.html#cb1732-33" tabindex="-1"></a><span class="st"> #Iteratively free intercepts across groups</span></span>
<span id="cb1732-34"><a href="bias.html#cb1732-34" tabindex="-1"></a><span class="st"> x1 ~ NA*1</span></span>
<span id="cb1732-35"><a href="bias.html#cb1732-35" tabindex="-1"></a><span class="st"> x2 ~ NA*1</span></span>
<span id="cb1732-36"><a href="bias.html#cb1732-36" tabindex="-1"></a><span class="st"> x3 ~ NA*1</span></span>
<span id="cb1732-37"><a href="bias.html#cb1732-37" tabindex="-1"></a><span class="st"> x4 ~ NA*1</span></span>
<span id="cb1732-38"><a href="bias.html#cb1732-38" tabindex="-1"></a><span class="st"> x5 ~ NA*1</span></span>
<span id="cb1732-39"><a href="bias.html#cb1732-39" tabindex="-1"></a><span class="st"> x6 ~ NA*1</span></span>
<span id="cb1732-40"><a href="bias.html#cb1732-40" tabindex="-1"></a><span class="st"> x7 ~ NA*1</span></span>
<span id="cb1732-41"><a href="bias.html#cb1732-41" tabindex="-1"></a><span class="st"> x8 ~ c(intx8, intx8)*1</span></span>
<span id="cb1732-42"><a href="bias.html#cb1732-42" tabindex="-1"></a><span class="st"> x9 ~ c(intx9, intx9)*1</span></span>
<span id="cb1732-43"><a href="bias.html#cb1732-43" tabindex="-1"></a><span class="st">&#39;</span></span>
<span id="cb1732-44"><a href="bias.html#cb1732-44" tabindex="-1"></a></span>
<span id="cb1732-45"><a href="bias.html#cb1732-45" tabindex="-1"></a>cfaModel_scalarInvarianceV7_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1732-46"><a href="bias.html#cb1732-46" tabindex="-1"></a>  cfaModel_scalarInvarianceV7,</span>
<span id="cb1732-47"><a href="bias.html#cb1732-47" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1732-48"><a href="bias.html#cb1732-48" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1732-49"><a href="bias.html#cb1732-49" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1732-50"><a href="bias.html#cb1732-50" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span>
<span id="cb1732-51"><a href="bias.html#cb1732-51" tabindex="-1"></a></span>
<span id="cb1732-52"><a href="bias.html#cb1732-52" tabindex="-1"></a><span class="fu">anova</span>(metricInvarianceModel_fit, cfaModel_scalarInvarianceV7_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Df"],"name":[1],"type":["int"],"align":["right"]},{"label":["AIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Chisq"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Chisq diff"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Df diff"],"name":[6],"type":["int"],"align":["right"]},{"label":["Pr(>Chisq)"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"54","2":"7412.554","3":"7612.738","4":"116.3133","5":"NA","6":"NA","7":"NA","_rn_":"metricInvarianceModel_fit"},{"1":"56","2":"7409.214","3":"7601.984","4":"116.9740","5":"0.6448224","6":"2","7":"0.7244003","_rn_":"cfaModel_scalarInvarianceV7_fit"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceDropConstraintsVariable8" class="section level5 hasAnchor" number="16.10.8.2.8">
<h5><span class="header-section-number">16.10.8.2.8</span> Variable 8<a href="bias.html#measurementInvarianceDropConstraintsVariable8" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Measurement invariance continues to hold when freeing intercepts of variable 8 across groups.
This suggests that variable 8 does not show non-invariant intercepts across groups, at least when the other intercepts have been freed.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1733"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1733-1"><a href="bias.html#cb1733-1" tabindex="-1"></a>cfaModel_scalarInvarianceV8 <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1733-2"><a href="bias.html#cb1733-2" tabindex="-1"></a><span class="st"> #Fix factor loadings to be the same across groups</span></span>
<span id="cb1733-3"><a href="bias.html#cb1733-3" tabindex="-1"></a><span class="st"> visual  =~ c(lambdax1,lambdax1)*x1 + c(lambdax2,lambdax2)*x2 + c(lambdax3,lambdax3)*x3</span></span>
<span id="cb1733-4"><a href="bias.html#cb1733-4" tabindex="-1"></a><span class="st"> textual =~ c(lambdax4,lambdax4)*x4 + c(lambdax5,lambdax5)*x5 + c(lambdax6,lambdax6)*x6</span></span>
<span id="cb1733-5"><a href="bias.html#cb1733-5" tabindex="-1"></a><span class="st"> speed   =~ c(lambdax7,lambdax7)*x7 + c(lambdax8,lambdax8)*x8 + c(lambdax9,lambdax9)*x9</span></span>
<span id="cb1733-6"><a href="bias.html#cb1733-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1733-7"><a href="bias.html#cb1733-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero</span></span>
<span id="cb1733-8"><a href="bias.html#cb1733-8" tabindex="-1"></a><span class="st"> visual ~ 0</span></span>
<span id="cb1733-9"><a href="bias.html#cb1733-9" tabindex="-1"></a><span class="st"> textual ~ 0</span></span>
<span id="cb1733-10"><a href="bias.html#cb1733-10" tabindex="-1"></a><span class="st"> speed ~ 0</span></span>
<span id="cb1733-11"><a href="bias.html#cb1733-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1733-12"><a href="bias.html#cb1733-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one in group 1; free latent variances in group 2</span></span>
<span id="cb1733-13"><a href="bias.html#cb1733-13" tabindex="-1"></a><span class="st"> visual ~~ c(1, NA)*visual</span></span>
<span id="cb1733-14"><a href="bias.html#cb1733-14" tabindex="-1"></a><span class="st"> textual ~~ c(1, NA)*textual</span></span>
<span id="cb1733-15"><a href="bias.html#cb1733-15" tabindex="-1"></a><span class="st"> speed ~~ c(1, NA)*speed</span></span>
<span id="cb1733-16"><a href="bias.html#cb1733-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1733-17"><a href="bias.html#cb1733-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1733-18"><a href="bias.html#cb1733-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1733-19"><a href="bias.html#cb1733-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1733-20"><a href="bias.html#cb1733-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1733-21"><a href="bias.html#cb1733-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1733-22"><a href="bias.html#cb1733-22" tabindex="-1"></a><span class="st"> #Estimate residual variances of manifest variables</span></span>
<span id="cb1733-23"><a href="bias.html#cb1733-23" tabindex="-1"></a><span class="st"> x1 ~~ x1</span></span>
<span id="cb1733-24"><a href="bias.html#cb1733-24" tabindex="-1"></a><span class="st"> x2 ~~ x2</span></span>
<span id="cb1733-25"><a href="bias.html#cb1733-25" tabindex="-1"></a><span class="st"> x3 ~~ x3</span></span>
<span id="cb1733-26"><a href="bias.html#cb1733-26" tabindex="-1"></a><span class="st"> x4 ~~ x4</span></span>
<span id="cb1733-27"><a href="bias.html#cb1733-27" tabindex="-1"></a><span class="st"> x5 ~~ x5</span></span>
<span id="cb1733-28"><a href="bias.html#cb1733-28" tabindex="-1"></a><span class="st"> x6 ~~ x6</span></span>
<span id="cb1733-29"><a href="bias.html#cb1733-29" tabindex="-1"></a><span class="st"> x7 ~~ x7</span></span>
<span id="cb1733-30"><a href="bias.html#cb1733-30" tabindex="-1"></a><span class="st"> x8 ~~ x8</span></span>
<span id="cb1733-31"><a href="bias.html#cb1733-31" tabindex="-1"></a><span class="st"> x9 ~~ x9</span></span>
<span id="cb1733-32"><a href="bias.html#cb1733-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1733-33"><a href="bias.html#cb1733-33" tabindex="-1"></a><span class="st"> #Iteratively free intercepts across groups</span></span>
<span id="cb1733-34"><a href="bias.html#cb1733-34" tabindex="-1"></a><span class="st"> x1 ~ NA*1</span></span>
<span id="cb1733-35"><a href="bias.html#cb1733-35" tabindex="-1"></a><span class="st"> x2 ~ NA*1</span></span>
<span id="cb1733-36"><a href="bias.html#cb1733-36" tabindex="-1"></a><span class="st"> x3 ~ NA*1</span></span>
<span id="cb1733-37"><a href="bias.html#cb1733-37" tabindex="-1"></a><span class="st"> x4 ~ NA*1</span></span>
<span id="cb1733-38"><a href="bias.html#cb1733-38" tabindex="-1"></a><span class="st"> x5 ~ NA*1</span></span>
<span id="cb1733-39"><a href="bias.html#cb1733-39" tabindex="-1"></a><span class="st"> x6 ~ NA*1</span></span>
<span id="cb1733-40"><a href="bias.html#cb1733-40" tabindex="-1"></a><span class="st"> x7 ~ NA*1</span></span>
<span id="cb1733-41"><a href="bias.html#cb1733-41" tabindex="-1"></a><span class="st"> x8 ~ NA*1</span></span>
<span id="cb1733-42"><a href="bias.html#cb1733-42" tabindex="-1"></a><span class="st"> x9 ~ c(intx9, intx9)*1</span></span>
<span id="cb1733-43"><a href="bias.html#cb1733-43" tabindex="-1"></a><span class="st">&#39;</span></span>
<span id="cb1733-44"><a href="bias.html#cb1733-44" tabindex="-1"></a></span>
<span id="cb1733-45"><a href="bias.html#cb1733-45" tabindex="-1"></a>cfaModel_scalarInvarianceV8_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1733-46"><a href="bias.html#cb1733-46" tabindex="-1"></a>  cfaModel_scalarInvarianceV8,</span>
<span id="cb1733-47"><a href="bias.html#cb1733-47" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1733-48"><a href="bias.html#cb1733-48" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1733-49"><a href="bias.html#cb1733-49" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1733-50"><a href="bias.html#cb1733-50" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span>
<span id="cb1733-51"><a href="bias.html#cb1733-51" tabindex="-1"></a></span>
<span id="cb1733-52"><a href="bias.html#cb1733-52" tabindex="-1"></a><span class="fu">anova</span>(metricInvarianceModel_fit, cfaModel_scalarInvarianceV8_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Df"],"name":[1],"type":["int"],"align":["right"]},{"label":["AIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Chisq"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Chisq diff"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Df diff"],"name":[6],"type":["int"],"align":["right"]},{"label":["Pr(>Chisq)"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"54","2":"7412.554","3":"7612.738","4":"116.3133","5":"NA","6":"NA","7":"NA","_rn_":"metricInvarianceModel_fit"},{"1":"55","2":"7411.119","3":"7607.596","4":"116.8785","5":"0.5640824","6":"1","7":"0.45262","_rn_":"cfaModel_scalarInvarianceV8_fit"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceDropConstraintsVariable9" class="section level5 hasAnchor" number="16.10.8.2.9">
<h5><span class="header-section-number">16.10.8.2.9</span> Variable 9<a href="bias.html#measurementInvarianceDropConstraintsVariable9" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Measurement invariance continues to hold when freeing intercepts of variable 9 across groups (even after constraining intercepts of variable 8 across groups).
This suggests that variable 9 does not show non-invariant intercepts across groups, at least when the other intercepts have been freed.</p>
<details class=chunk-details open><summary class=chunk-summary><span class=chunk-summary-text>Code</span></summary>
<div class="sourceCode" id="cb1734"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1734-1"><a href="bias.html#cb1734-1" tabindex="-1"></a>cfaModel_scalarInvarianceV9 <span class="ot">&lt;-</span> <span class="st">&#39;</span></span>
<span id="cb1734-2"><a href="bias.html#cb1734-2" tabindex="-1"></a><span class="st"> #Fix factor loadings to be the same across groups</span></span>
<span id="cb1734-3"><a href="bias.html#cb1734-3" tabindex="-1"></a><span class="st"> visual  =~ c(lambdax1,lambdax1)*x1 + c(lambdax2,lambdax2)*x2 + c(lambdax3,lambdax3)*x3</span></span>
<span id="cb1734-4"><a href="bias.html#cb1734-4" tabindex="-1"></a><span class="st"> textual =~ c(lambdax4,lambdax4)*x4 + c(lambdax5,lambdax5)*x5 + c(lambdax6,lambdax6)*x6</span></span>
<span id="cb1734-5"><a href="bias.html#cb1734-5" tabindex="-1"></a><span class="st"> speed   =~ c(lambdax7,lambdax7)*x7 + c(lambdax8,lambdax8)*x8 + c(lambdax9,lambdax9)*x9</span></span>
<span id="cb1734-6"><a href="bias.html#cb1734-6" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1734-7"><a href="bias.html#cb1734-7" tabindex="-1"></a><span class="st"> #Fix latent means to zero</span></span>
<span id="cb1734-8"><a href="bias.html#cb1734-8" tabindex="-1"></a><span class="st"> visual ~ 0</span></span>
<span id="cb1734-9"><a href="bias.html#cb1734-9" tabindex="-1"></a><span class="st"> textual ~ 0</span></span>
<span id="cb1734-10"><a href="bias.html#cb1734-10" tabindex="-1"></a><span class="st"> speed ~ 0</span></span>
<span id="cb1734-11"><a href="bias.html#cb1734-11" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1734-12"><a href="bias.html#cb1734-12" tabindex="-1"></a><span class="st"> #Fix latent variances to one in group 1; free latent variances in group 2</span></span>
<span id="cb1734-13"><a href="bias.html#cb1734-13" tabindex="-1"></a><span class="st"> visual ~~ c(1, NA)*visual</span></span>
<span id="cb1734-14"><a href="bias.html#cb1734-14" tabindex="-1"></a><span class="st"> textual ~~ c(1, NA)*textual</span></span>
<span id="cb1734-15"><a href="bias.html#cb1734-15" tabindex="-1"></a><span class="st"> speed ~~ c(1, NA)*speed</span></span>
<span id="cb1734-16"><a href="bias.html#cb1734-16" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1734-17"><a href="bias.html#cb1734-17" tabindex="-1"></a><span class="st"> #Estimate covariances among latent variables</span></span>
<span id="cb1734-18"><a href="bias.html#cb1734-18" tabindex="-1"></a><span class="st"> visual ~~ textual</span></span>
<span id="cb1734-19"><a href="bias.html#cb1734-19" tabindex="-1"></a><span class="st"> visual ~~ speed</span></span>
<span id="cb1734-20"><a href="bias.html#cb1734-20" tabindex="-1"></a><span class="st"> textual ~~ speed</span></span>
<span id="cb1734-21"><a href="bias.html#cb1734-21" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1734-22"><a href="bias.html#cb1734-22" tabindex="-1"></a><span class="st"> #Estimate residual variances of manifest variables</span></span>
<span id="cb1734-23"><a href="bias.html#cb1734-23" tabindex="-1"></a><span class="st"> x1 ~~ x1</span></span>
<span id="cb1734-24"><a href="bias.html#cb1734-24" tabindex="-1"></a><span class="st"> x2 ~~ x2</span></span>
<span id="cb1734-25"><a href="bias.html#cb1734-25" tabindex="-1"></a><span class="st"> x3 ~~ x3</span></span>
<span id="cb1734-26"><a href="bias.html#cb1734-26" tabindex="-1"></a><span class="st"> x4 ~~ x4</span></span>
<span id="cb1734-27"><a href="bias.html#cb1734-27" tabindex="-1"></a><span class="st"> x5 ~~ x5</span></span>
<span id="cb1734-28"><a href="bias.html#cb1734-28" tabindex="-1"></a><span class="st"> x6 ~~ x6</span></span>
<span id="cb1734-29"><a href="bias.html#cb1734-29" tabindex="-1"></a><span class="st"> x7 ~~ x7</span></span>
<span id="cb1734-30"><a href="bias.html#cb1734-30" tabindex="-1"></a><span class="st"> x8 ~~ x8</span></span>
<span id="cb1734-31"><a href="bias.html#cb1734-31" tabindex="-1"></a><span class="st"> x9 ~~ x9</span></span>
<span id="cb1734-32"><a href="bias.html#cb1734-32" tabindex="-1"></a><span class="st"> </span></span>
<span id="cb1734-33"><a href="bias.html#cb1734-33" tabindex="-1"></a><span class="st"> #Iteratively free intercepts across groups</span></span>
<span id="cb1734-34"><a href="bias.html#cb1734-34" tabindex="-1"></a><span class="st"> x1 ~ NA*1</span></span>
<span id="cb1734-35"><a href="bias.html#cb1734-35" tabindex="-1"></a><span class="st"> x2 ~ NA*1</span></span>
<span id="cb1734-36"><a href="bias.html#cb1734-36" tabindex="-1"></a><span class="st"> x3 ~ NA*1</span></span>
<span id="cb1734-37"><a href="bias.html#cb1734-37" tabindex="-1"></a><span class="st"> x4 ~ NA*1</span></span>
<span id="cb1734-38"><a href="bias.html#cb1734-38" tabindex="-1"></a><span class="st"> x5 ~ NA*1</span></span>
<span id="cb1734-39"><a href="bias.html#cb1734-39" tabindex="-1"></a><span class="st"> x6 ~ NA*1</span></span>
<span id="cb1734-40"><a href="bias.html#cb1734-40" tabindex="-1"></a><span class="st"> x7 ~ NA*1</span></span>
<span id="cb1734-41"><a href="bias.html#cb1734-41" tabindex="-1"></a><span class="st"> x8 ~ c(intx8, intx8)*1</span></span>
<span id="cb1734-42"><a href="bias.html#cb1734-42" tabindex="-1"></a><span class="st"> x9 ~ NA*1</span></span>
<span id="cb1734-43"><a href="bias.html#cb1734-43" tabindex="-1"></a><span class="st">&#39;</span></span>
<span id="cb1734-44"><a href="bias.html#cb1734-44" tabindex="-1"></a></span>
<span id="cb1734-45"><a href="bias.html#cb1734-45" tabindex="-1"></a>cfaModel_scalarInvarianceV9_fit <span class="ot">&lt;-</span> <span class="fu">lavaan</span>(</span>
<span id="cb1734-46"><a href="bias.html#cb1734-46" tabindex="-1"></a>  cfaModel_scalarInvarianceV9,</span>
<span id="cb1734-47"><a href="bias.html#cb1734-47" tabindex="-1"></a>  <span class="at">data =</span> HolzingerSwineford1939,</span>
<span id="cb1734-48"><a href="bias.html#cb1734-48" tabindex="-1"></a>  <span class="at">group =</span> <span class="st">&quot;school&quot;</span>,</span>
<span id="cb1734-49"><a href="bias.html#cb1734-49" tabindex="-1"></a>  <span class="at">missing =</span> <span class="st">&quot;ML&quot;</span>,</span>
<span id="cb1734-50"><a href="bias.html#cb1734-50" tabindex="-1"></a>  <span class="at">estimator =</span> <span class="st">&quot;MLR&quot;</span>)</span>
<span id="cb1734-51"><a href="bias.html#cb1734-51" tabindex="-1"></a></span>
<span id="cb1734-52"><a href="bias.html#cb1734-52" tabindex="-1"></a><span class="fu">anova</span>(metricInvarianceModel_fit, cfaModel_scalarInvarianceV9_fit)</span></code></pre></div>
</details>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Df"],"name":[1],"type":["int"],"align":["right"]},{"label":["AIC"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["BIC"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Chisq"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Chisq diff"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Df diff"],"name":[6],"type":["int"],"align":["right"]},{"label":["Pr(>Chisq)"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"54","2":"7412.554","3":"7612.738","4":"116.3133","5":"NA","6":"NA","7":"NA","_rn_":"metricInvarianceModel_fit"},{"1":"55","2":"7410.964","3":"7607.441","4":"116.7236","5":"0.4023897","6":"1","7":"0.5258577","_rn_":"cfaModel_scalarInvarianceV9_fit"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="measurementInvarianceDropConstraintsSummary" class="section level5 hasAnchor" number="16.10.8.2.10">
<h5><span class="header-section-number">16.10.8.2.10</span> Summary<a href="bias.html#measurementInvarianceDropConstraintsSummary" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>When iteratively dropping constraints, measurement invariance became established when freeing the intercepts for items 1–7 across groups.
This suggests that items 7, 8, and 9 show measurement invariance of intercepts across groups.
This could be due to the selection of starting with item 1 as the first constraint to be freed.
We could try freeing constraints in a different order to see the extent to which the order influences which items are identified as showing measurement non-invariance.</p>
</div>
</div>
<div id="measurementInvarianceNextSteps" class="section level4 hasAnchor" number="16.10.8.3">
<h4><span class="header-section-number">16.10.8.3</span> Next Steps<a href="bias.html#measurementInvarianceNextSteps" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>After identifying which item(s) show measurement non-invariance, we would evaluate the measurement non-invariance according to theory and effect sizes.
We might start with the item with the largest measurement non-invariance.
If we deem the measurement non-invariance for this item is non-negligible, we have three primary options: (1) drop the item for both groups, (2) drop the item for one group but keep it for the other group, or (3) freely estimate parameters for the item across groups.
We would do this iteratively for the remaining items, by magnitude, that show non-negligible measurement non-invariance.
Thus, our model might show partial scalar invariance—some items might have intercepts that are constrained to be the same across groups, whereas other items might have intercepts that are allowed to differ across groups.
Then, we would test subsequent measurement invariance models (e.g., residual invariance) while keeping the partially freed constaints from the partial scalar invariance model.
Once we establish the best fitting model that makes as many constraints that are theoretically and empirically justified for the purposes of the study, we would use that model in subsequent tests.
As a reminder, full or partial metric invariance is helpful for comparing associations across groups, whereas full or partial scalar invariance is helpful for comparing mean levels across groups.</p>
</div>
</div>
</div>
<div id="conclusion-bias" class="section level2 hasAnchor" number="16.11">
<h2><span class="header-section-number">16.11</span> Conclusion<a href="bias.html#conclusion-bias" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Bias is a <a href="reliability.html#systematicError">systematic error</a>.
Test bias refers to a <a href="reliability.html#systematicError">systematic error</a> (in measurement, prediction, etc.) as a function of group membership.
The two broad categories of test bias include <a href="bias.html#predictiveBias">predictive bias</a> and <a href="bias.html#testStructureBias">test structure bias</a>.
<a href="bias.html#predictiveBias">Predictive bias</a> refers to differences between groups in the relation between the test and criterion—i.e., different intercepts and/or slopes of the regression line between the test and the criterion.
<a href="bias.html#testStructureBias">Test structure bias</a> refers to differences in the internal test characteristics across groups, and can be evaluated with approaches such as <a href="bias.html#measurementInvariance">measurement invariance</a> and <a href="bias.html#dif">differential item function</a>.
When detecting test bias, it is important to address it, and there are a number of ways of <a href="bias.html#correctForBias">correcting for bias</a>, including <a href="bias.html#scoreAdjustment">score adjustment</a> and <a href="bias.html#otherBiasCorrections">other approaches</a> that do not involve score adjustment.
However, even if a test does not show bias, it does not mean that the test is <a href="bias.html#fairness">fair</a>.
<a href="bias.html#fairness">Fairness</a> is not a scientific question but rather a moral, societal, and <a href="ethics.html#ethics">ethical</a> question; there are many different ways of <a href="bias.html#operationalizingFairness">operationalizing fairness</a>.
<a href="bias.html#fairness">Fairness</a> is a complex question, so do the best you can and try to minimize any negative impact of the assessment procedures.</p>
</div>
<div id="readings-bias" class="section level2 hasAnchor" number="16.12">
<h2><span class="header-section-number">16.12</span> Suggested Readings<a href="bias.html#readings-bias" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p><span class="citation">Putnick &amp; Bornstein (<a href="#ref-Putnick2016">2016</a>)</span>; <span class="citation">N. S. Cole (<a href="#ref-Cole1981">1981</a>)</span>; <span class="citation">Fernández &amp; Abe (<a href="#ref-Fernandez2018">2018</a>)</span>; <span class="citation">Reynolds &amp; Suzuki (<a href="#ref-Reynolds2012">2012</a>)</span>; <span class="citation">Sackett &amp; Wilk (<a href="#ref-Sackett1994">1994</a>)</span>; <span class="citation">Jonson &amp; Geisinger (<a href="#ref-Jonson2022">2022</a>)</span></p>
</div>
<div id="exercises-bias" class="section level2 hasAnchor" number="16.13">
<h2><span class="header-section-number">16.13</span> Exercises<a href="bias.html#exercises-bias" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="exercisesQuestions-bias" class="section level3 hasAnchor" number="16.13.1">
<h3><span class="header-section-number">16.13.1</span> Questions<a href="bias.html#exercisesQuestions-bias" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Note: Several of the following questions use data from the Children of the National Longitudinal Survey of Youth Survey (CNLSY).
The CNLSY is a publicly available longitudinal data set provided by the Bureau of Labor Statistics (<a href="https://www.bls.gov/nls/nlsy79-children.htm#topical-guide" class="uri">https://www.bls.gov/nls/nlsy79-children.htm#topical-guide</a>; archived at <a href="https://perma.cc/EH38-HDRN" class="uri">https://perma.cc/EH38-HDRN</a>).
The CNLSY data file for these exercises is located on the book’s page of the Open Science Framework (<a href="https://osf.io/3pwza" class="uri">https://osf.io/3pwza</a>).
Children’s behavior problems were rated in 1988 (time 1: T1) and then again in 1990 (time 2: T2) on the Behavior Problems Index (BPI).
Below are the items corresponding to the Antisocial subscale of the BPI:</p>
<ol style="list-style-type: decimal">
<li>cheats or tells lies</li>
<li>bullies or is cruel/mean to others</li>
<li>does not seem to feel sorry after misbehaving</li>
<li>breaks things deliberately</li>
<li>is disobedient at school</li>
<li>has trouble getting along with teachers</li>
<li>has sudden changes in mood or feeling</li>
</ol>
<ol style="list-style-type: decimal">
<li><ol style="list-style-type: lower-alpha">
<li>Determine whether there is predictive bias, as a function of sex, in predicting antisocial behavior at T2 using antisocial behavior at T2.
Describe any predictive bias observed.</li>
<li>How could you address this predictive bias?</li>
</ol></li>
<li>Assume that children are selected to receive preventative services if their score on the Antisocial scale at T1 is &gt; 6.
Assume that children at T2 have an antisocial disorder if their score at T2 is &gt; 6.
Does the Antisocial scale show fairness in terms of equal outcomes between boys and girls?</li>
<li>A lawyer argues in court that a test that is used to select students for college admissions should be able to be used because it has been demonstrated to show no evidence of bias against particular groups.
However, you know that just because a measure may be unbiased does not mean that using the measure’s scores for that purpose is fair.
How could such a measure be unbiased yet unfair?</li>
<li>Fit a multi-group <a href="irt.html#irt">item response theory</a> model to the Antisocial subscale at time 2, with the participant’s sex as a grouping factor.
<ol style="list-style-type: lower-alpha">
<li>Does a model with item parameters constrained across sexes fit significantly worse than a model with item parameters that are allowed to vary across sexes?</li>
<li>Starting with an unconstrained model that allows item parameters to vary across sexes, sequentially add constraints to item parameters across groups.
Which items differed in discrimination across groups?
Which items differed in severity across groups?
Describe any differences.</li>
<li>How could any differential item functioning be addressed?</li>
</ol></li>
<li>Examine the longitudinal measurement invariance of the Antisocial scale across T1 and T2.
Use full information maximum likelihood (FIML) to account for missing data.
Use robust standard errors to account for non-normally distributed data.
Which type(s) of longitudinal measurement invariance failed?
Which items and parameters appeared to differ across ages?</li>
</ol>
</div>
<div id="exercisesAnswers-bias" class="section level3 hasAnchor" number="16.13.2">
<h3><span class="header-section-number">16.13.2</span> Answers<a href="bias.html#exercisesAnswers-bias" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ol style="list-style-type: decimal">
<li><ol style="list-style-type: lower-alpha">
<li>There is predictive bias in terms of different slopes across sexes.
The slope is steeper for boys than girls, indicating that the measure is not as predictively accurate for girls as it is for boys.</li>
<li>You could address this predictive bias by using a different predictive measure for girls than boys, changing the criterion, or by using within-group norming.</li>
</ol></li>
<li>No, the Antisocial scale does not show equal outcomes in terms of <a href="prediction.html#selectionRatio">selection rate</a>.
The selection ratio is much lower in girls <span class="math inline">\((0.10)\)</span> compared to boys <span class="math inline">\((0.05)\)</span>.</li>
<li>The measure could be unbiased if it shows the same intercepts and slopes across groups in predicting college performance.
However, a measure can still be unfair even it is unbiased.
If there are different types of errors in different groups, the measure may not be fair.
For instance, if there are more false negative errors in Black applicants compared to White applicants and more false positive errors in White applicants than Black applicants, the measure would be unfair against Black applicants.
Even if the percent accuracy of prediction is the same for Black and White applicants, the errors have different implications for each group—White applicants who would have failed in college are more likely to be accepted whereas strong Black applicants who would have succeeded are less likely to be accepted.
To address the unfairness of the test, it will be important to use one of the various operationalizations of fairness: equal outcomes, equal opportunity, or equal odds.</li>
<li><ol style="list-style-type: lower-alpha">
<li>The model with item parameters constrained across sexes fit significantly worse than a model with item parameters that were allowed to vary across sexes, <span class="math inline">\(\chi^2[-19] = -119.12, p &lt; .001\)</span>.
This suggests that the item parameters show differential item functioning across boys and girls.</li>
<li>No items showed significant differences in discrimination across sexes, suggesting that items did not differ in their relation to the latent factor across boys and girls.
Items 1, 2, 3, 4, 5, and 6 differed in severity across sexes.
Items showed greater severity for girls than boys.
It took a higher level of antisocial behavior for a girl to be endorsed as engaging in one of the following behaviors: cheats or tells lies, bullies or is cruel/mean to others, does not seem to feel sorry after misbehaving, breaks things deliberately, is disobedient at school, and has trouble getting along with teachers.</li>
<li>The differences in item severity across boys and girls could be addressed by resolving DIF.
That is, the items that show severity DIF (items 1–6) would be allowed to differ in their severity parameter across boys and girls, but these items would be constrained to have the same discrimination across groups; non-DIF items would have severity and discrimination parameters constrained to be the same across groups.</li>
</ol></li>
<li>The configural invariance model fit well according to RMSEA <span class="math inline">\((0.09)\)</span> and SRMR <span class="math inline">\((0.06)\)</span>, but fit less well according to CFI <span class="math inline">\((0.77)\)</span>.
The metric invariance model did not fit significantly worse than the configural invariance model <span class="math inline">\((\Delta\chi^2[6] = 8.77, p = 0.187)\)</span>.
However, the scalar invariance model fit significantly worse than the metric invariance model <span class="math inline">\((\Delta\chi^2[6] = 21.28, p = 0.002)\)</span>, and the residual invariance model fit significantly worse than the scalar invariance model <span class="math inline">\((\Delta\chi^2[7] = 42.80, p &lt; .001)\)</span>.
Therefore, scalar and residual invariance failed.
The following items showed larger intercepts at T1 compared to T2: items 1, 2, 3, 4, 5, and 7.
Only item 6 showed larger intercepts at T2 than T1.
The following items showed a larger residual at T1 compared to T2: items 2–7.
Only item 1 showed a larger residual at T2 compared to T1.</li>
</ol>

</div>
</div>
</div>
<h3>References<a href="references.html#references" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="refs" class="references csl-bib-body hanging-indent" entry-spacing="0" line-spacing="2">
<div id="ref-Aguinis2010a" class="csl-entry">
Aguinis, H., Culpepper, S. A., &amp; Pierce, C. A. (2010). Revival of test bias research in preemployment testing. <em>Journal of Applied Psychology</em>, <em>95</em>(4), 648–680. <a href="https://doi.org/10.1037/a0018714">https://doi.org/10.1037/a0018714</a>
</div>
<div id="ref-AERA2014" class="csl-entry">
American Educational Research Association, American Psychological Association, &amp; National Council on Measurement in Education. (2014). <em>Standards for educational and psychological testing</em>. American Educational Research Association.
</div>
<div id="ref-Barrash2010" class="csl-entry">
Barrash, J., Stillman, A., Anderson, S. W., Uc, E. Y., Dawson, J. D., &amp; Rizzo, M. (2010). Prediction of driving ability with neuropsychological tests: Demographic adjustments diminish accuracy. <em>Journal of the International Neuropsychological Society</em>, <em>16</em>(4), 679–686. <a href="https://doi.org/10.1017/S1355617710000470">https://doi.org/10.1017/S1355617710000470</a>
</div>
<div id="ref-Bauer2020" class="csl-entry">
Bauer, D. J., Belzak, W. C. M., &amp; Cole, V. T. (2020). Simplifying the assessment of measurement invariance over multiple background variables: Using regularized moderated nonlinear factor analysis to detect differential item functioning. <em>Structural Equation Modeling: A Multidisciplinary Journal</em>, <em>27</em>(1), 43–55. <a href="https://doi.org/10.1080/10705511.2019.1642754">https://doi.org/10.1080/10705511.2019.1642754</a>
</div>
<div id="ref-Belzak2020" class="csl-entry">
Belzak, W. C. M., &amp; Bauer, D. J. (2020). Improving the assessment of measurement invariance: Using regularization to select anchor items and identify differential item functioning. <em>Psychological Methods</em>, <em>25</em>(6), 673–690. <a href="https://doi.org/10.1037/met0000253">https://doi.org/10.1037/met0000253</a>
</div>
<div id="ref-Brown1999" class="csl-entry">
Brown, R. T., Reynolds, C. R., &amp; Whitaker, J. S. (1999). Bias in mental testing since bias in mental testing. <em>School Psychology Quarterly</em>, <em>14</em>(3), 208–238. <a href="https://doi.org/10.1037/h0089007">https://doi.org/10.1037/h0089007</a>
</div>
<div id="ref-Burlew2019" class="csl-entry">
Burlew, A. K., Peteet, B. J., McCuistian, C., &amp; Miller-Roenigk, B. D. (2019). Best practices for researching diverse groups. <em>American Journal of Orthopsychiatry</em>, <em>89</em>(3), 354–368. <a href="https://doi.org/10.1037/ort0000350">https://doi.org/10.1037/ort0000350</a>
</div>
<div id="ref-Camilli2013" class="csl-entry">
Camilli, G. (2013). Ongoing issues in test fairness. <em>Educational Research and Evaluation</em>, <em>19</em>(2–3), 104–120. <a href="https://doi.org/10.1080/13803611.2013.767602">https://doi.org/10.1080/13803611.2013.767602</a>
</div>
<div id="ref-R-mirt" class="csl-entry">
Chalmers, P. (2020). <em><span class="nocase">mirt</span>: Multidimensional item response theory</em>. <a href="https://CRAN.R-project.org/package=mirt">https://CRAN.R-project.org/package=mirt</a>
</div>
<div id="ref-Chen2007a" class="csl-entry">
Chen, F. F. (2007). Sensitivity of goodness of fit indexes to lack of measurement invariance. <em>Structural Equation Modeling: A Multidisciplinary Journal</em>, <em>14</em>(3), 464–504. <a href="https://doi.org/10.1080/10705510701301834">https://doi.org/10.1080/10705510701301834</a>
</div>
<div id="ref-Cheng2016" class="csl-entry">
Cheng, Y., Shao, C., &amp; Lathrop, Q. N. (2016). The mediated MIMIC model for understanding the underlying mechanism of DIF. <em>Educational and Psychological Measurement</em>, <em>76</em>(1), 43–63. <a href="https://doi.org/10.1177/0013164415576187">https://doi.org/10.1177/0013164415576187</a>
</div>
<div id="ref-Cheung2002" class="csl-entry">
Cheung, G. W., &amp; Rensvold, R. B. (2002). Evaluating goodness-of-fit indexes for testing measurement invariance. <em>Structural Equation Modeling: A Multidisciplinary Journal</em>, <em>9</em>(2), 233–255. <a href="https://doi.org/10.1207/s15328007sem0902_5">https://doi.org/10.1207/s15328007sem0902_5</a>
</div>
<div id="ref-Clark1984" class="csl-entry">
Clark, M. J., &amp; Grandy, J. (1984). <em>Sex differences in the academic performance of <span>S</span>cholastic <span>A</span>ptitude <span>T</span>est takers: College board report no. 84-8</em>. College Board Publications.
</div>
<div id="ref-Cole1981" class="csl-entry">
Cole, N. S. (1981). Bias in testing. <em>American Psychologist</em>, <em>36</em>(10), 1067–1077. <a href="https://doi.org/10.1037/0003-066X.36.10.1067">https://doi.org/10.1037/0003-066X.36.10.1067</a>
</div>
<div id="ref-R-aMNLFA" class="csl-entry">
Cole, V., Gottfredson, N., &amp; Giordano, M. (2018). <em><span class="nocase">aMNLFA</span>: Automated fitting of moderated nonlinear factor analysis through the <span>Mplus</span> program</em>. <a href="https://CRAN.R-project.org/package=aMNLFA">https://CRAN.R-project.org/package=aMNLFA</a>
</div>
<div id="ref-GeneralAptitudeTestBattery1989" class="csl-entry">
Committee on the General Aptitude Test Battery, Commission on Behavioral and Social Sciences and Education, &amp; National Research Council. (1989). <em>Fairness in employment testing: Validity generalization, minority issues, and the general aptitude test battery</em>. National Academies Press.
</div>
<div id="ref-Counsell2020" class="csl-entry">
Counsell, A., Cribbie, R. A., &amp; Flora, D. B. (2020). Evaluating equivalence testing methods for measurement invariance. <em>Multivariate Behavioral Research</em>, <em>55</em>(2), 312–328. <a href="https://doi.org/10.1080/00273171.2019.1633617">https://doi.org/10.1080/00273171.2019.1633617</a>
</div>
<div id="ref-Curran2014" class="csl-entry">
Curran, P. J., Howard, A. L., Bainter, S. A., Lane, S. T., &amp; McGinley, J. S. (2014). The separation of between-person and within-person components of individual change over time: A latent curve model with structured residuals. <em>Journal of Consulting and Clinical Psychology</em>, <em>82</em>, 8–94. <a href="https://doi.org/10.1037/a0035297">https://doi.org/10.1037/a0035297</a>
</div>
<div id="ref-Dorans2017" class="csl-entry">
Dorans, N. J. (2017). Contributions to the quantitative assessment of item, test, and score fairness. In R. E. Bennett &amp; M. von Davier (Eds.), <em>Advancing human assessment</em> (pp. 201–230). Springer, Cham.
</div>
<div id="ref-R-dmacs" class="csl-entry">
Dueber, D. (2019). <em><span class="nocase">dmacs</span>: Measurement nonequivalence effect size calculator</em>. <a href="https://github.com/ddueber/dmacs">https://github.com/ddueber/dmacs</a>
</div>
<div id="ref-R-semPlot" class="csl-entry">
Epskamp, S. (2022). <em><span class="nocase">semPlot</span>: Path diagrams and visual analysis of various <span>SEM</span> packages’ output</em>. <a href="https://github.com/SachaEpskamp/semPlot">https://github.com/SachaEpskamp/semPlot</a>
</div>
<div id="ref-Fernandez2018" class="csl-entry">
Fernández, A. L., &amp; Abe, J. (2018). Bias in cross-cultural neuropsychological testing: Problems and possible solutions. <em>Culture and Brain</em>, <em>6</em>(1), 1–35. <a href="https://doi.org/10.1007/s40167-017-0050-2">https://doi.org/10.1007/s40167-017-0050-2</a>
</div>
<div id="ref-Fletcher2021" class="csl-entry">
Fletcher, R. R., Nakeshimana, A., &amp; Olubeko, O. (2021). Addressing fairness, bias, and appropriate use of artificial intelligence and machine learning in global health. <em>Frontiers in Artificial Intelligence</em>, <em>3</em>(116). <a href="https://doi.org/10.3389/frai.2020.561802">https://doi.org/10.3389/frai.2020.561802</a>
</div>
<div id="ref-Gipps2009" class="csl-entry">
Gipps, C., &amp; Stobart, G. (2009). Fairness in assessment. In C. Wyatt-Smith &amp; J. J. Cumming (Eds.), <em>Educational assessment in the 21st century: Connecting theory and practice</em> (pp. 105–118). Springer Netherlands. <a href="https://doi.org/10.1007/978-1-4020-9964-9_6">https://doi.org/10.1007/978-1-4020-9964-9_6</a>
</div>
<div id="ref-Gonzalez2021" class="csl-entry">
Gonzalez, O., &amp; Pelham, W. E. (2021). When does differential item functioning matter for screening? A method for empirical evaluation. <em>Assessment</em>, <em>28</em>(2), 446–456. <a href="https://doi.org/10.1177/1073191120913618">https://doi.org/10.1177/1073191120913618</a>
</div>
<div id="ref-Gottfredson1994" class="csl-entry">
Gottfredson, L. S. (1994). The science and politics of race-norming. <em>American Psychologist</em>, <em>49</em>(11), 955–963. <a href="https://doi.org/10.1037/0003-066X.49.11.955">https://doi.org/10.1037/0003-066X.49.11.955</a>
</div>
<div id="ref-Gottfredson2019" class="csl-entry">
Gottfredson, N. C., Cole, V. T., Giordano, M. L., Bauer, D. J., Hussong, A. M., &amp; Ennett, S. T. (2019). Simplifying the implementation of modern scale scoring methods with an automated <span>R</span> package: Automated moderated nonlinear factor analysis (<span class="nocase">aMNLFA</span>). <em>Addictive Behaviors</em>, <em>94</em>, 65–73. <a href="https://doi.org/10.1016/j.addbeh.2018.10.031">https://doi.org/10.1016/j.addbeh.2018.10.031</a>
</div>
<div id="ref-Gunn2020" class="csl-entry">
Gunn, H. J., Grimm, K. J., &amp; Edwards, M. C. (2020). Evaluation of six effect size measures of measurement non-invariance for continuous outcomes. <em>Structural Equation Modeling: A Multidisciplinary Journal</em>, <em>27</em>(4), 503–514. <a href="https://doi.org/10.1080/10705511.2019.1689507">https://doi.org/10.1080/10705511.2019.1689507</a>
</div>
<div id="ref-Hagquist2019" class="csl-entry">
Hagquist, C. (2019). Explaining differential item functioning focusing on the crucial role of external information – an example from the measurement of adolescent mental health. <em>BMC Medical Research Methodology</em>, <em>19</em>(1), 185. <a href="https://doi.org/10.1186/s12874-019-0828-3">https://doi.org/10.1186/s12874-019-0828-3</a>
</div>
<div id="ref-Hagquist2017" class="csl-entry">
Hagquist, C., &amp; Andrich, D. (2017). Recent advances in analysis of differential item functioning in health research using the <span>R</span>asch model. <em>Health and Quality of Life Outcomes</em>, <em>15</em>(1), 181. <a href="https://doi.org/10.1186/s12955-017-0755-0">https://doi.org/10.1186/s12955-017-0755-0</a>
</div>
<div id="ref-Hall1999" class="csl-entry">
Hall, G. C. N., Bansal, A., &amp; Lopez, I. R. (1999). Ethnicity and psychopathology: A meta-analytic review of 31 years of comparative MMPI/MMPI-2 research. <em>Psychological Assessment</em>, <em>11</em>(2), 186–197. <a href="https://doi.org/10.1037/1040-3590.11.2.186">https://doi.org/10.1037/1040-3590.11.2.186</a>
</div>
<div id="ref-Han2019" class="csl-entry">
Han, K., Colarelli, S. M., &amp; Weed, N. C. (2019). Methodological and statistical advances in the consideration of cultural diversity in assessment: A critical review of group classification and measurement invariance testing. <em>Psychological Assessment</em>, <em>31</em>(12), 1481–1496. <a href="https://doi.org/10.1037/pas0000731">https://doi.org/10.1037/pas0000731</a>
</div>
<div id="ref-Helms2006" class="csl-entry">
Helms, J. E. (2006). Fairness is not validity or cultural bias in racial-group assessment: A quantitative perspective. <em>American Psychologist</em>, <em>61</em>(8), 845–859. <a href="https://doi.org/10.1037/0003-066X.61.8.845">https://doi.org/10.1037/0003-066X.61.8.845</a>
</div>
<div id="ref-Jensen1980" class="csl-entry">
Jensen, A. R. (1980). Précis of bias in mental testing. <em>Behavioral and Brain Sciences</em>, <em>3</em>(3), 325–333. <a href="https://doi.org/10.1017/S0140525X00005161">https://doi.org/10.1017/S0140525X00005161</a>
</div>
<div id="ref-Jonson2022" class="csl-entry">
Jonson, J. L., &amp; Geisinger, K. F. (2022). <em>Fairness in educational and psychological testing: Examining theoretical, research, practice, and policy implications of the 2014 standards</em>. American Educational Research Association,.
</div>
<div id="ref-Jorgensen2018" class="csl-entry">
Jorgensen, T. D., Kite, B. A., Chen, P.-Y., &amp; Short, S. D. (2018). Permutation randomization methods for testing measurement equivalence and detecting differential item functioning in multiple-group confirmatory factor analysis. <em>Psychological Methods</em>, <em>23</em>(4), 708–728. <a href="https://doi.org/10.1037/met0000152">https://doi.org/10.1037/met0000152</a>
</div>
<div id="ref-R-semTools" class="csl-entry">
Jorgensen, T. D., Pornprasertmanit, S., Schoemann, A. M., &amp; Rosseel, Y. (2021). <em><span class="nocase">semTools</span>: Useful tools for structural equation modeling</em>. <a href="https://github.com/simsem/semTools/wiki">https://github.com/simsem/semTools/wiki</a>
</div>
<div id="ref-Kuncel2010a" class="csl-entry">
Kuncel, N. R., &amp; Hezlett, S. A. (2010). Fact and fiction in cognitive ability testing for admissions and hiring decisions. <em>Current Directions in Psychological Science</em>, <em>19</em>(6), 339–345. <a href="https://doi.org/10.1177/0963721410389459">https://doi.org/10.1177/0963721410389459</a>
</div>
<div id="ref-Lai2021" class="csl-entry">
Lai, M. H. C. (2021). Adjusting for measurement noninvariance with alignment in growth modeling. <em>Multivariate Behavioral Research</em>, 1–18. <a href="https://doi.org/10.1080/00273171.2021.1941730">https://doi.org/10.1080/00273171.2021.1941730</a>
</div>
<div id="ref-Lee2013" class="csl-entry">
Lee, K., Bull, R., &amp; Ho, R. M. H. (2013). Developmental changes in executive functioning. <em>Child Development</em>, <em>84</em>(6), 1933–1953. <a href="https://doi.org/10.1111/cdev.12096">https://doi.org/10.1111/cdev.12096</a>
</div>
<div id="ref-Little2013" class="csl-entry">
Little, T. D. (2013). <em>Longitudinal structural equation modeling</em>. The Guilford Press.
</div>
<div id="ref-Little2007" class="csl-entry">
Little, T. D., Preacher, K. J., Selig, J. P., &amp; Card, N. A. (2007). New developments in latent variable panel analyses of longitudinal data. <em>International Journal of Behavioral Development</em>, <em>31</em>(4), 357–365. <a href="https://doi.org/10.1177/0165025407077757">https://doi.org/10.1177/0165025407077757</a>
</div>
<div id="ref-Little2006a" class="csl-entry">
Little, T. D., Slegers, D. W., &amp; Card, N. A. (2006). A non-arbitrary method of identifying and scaling latent variables in SEM and MACS models. <em>Structural Equation Modeling</em>, <em>13</em>(1), 59–72. <a href="https://doi.org/10.1207/s15328007sem1301_3">https://doi.org/10.1207/s15328007sem1301_3</a>
</div>
<div id="ref-Liu2017a" class="csl-entry">
Liu, Y., Millsap, R. E., West, S. G., Tein, J.-Y., Tanaka, R., &amp; Grimm, K. J. (2017). Testing measurement invariance in longitudinal data with ordered-categorical measures. <em>Psychological Methods</em>, <em>22</em>(3), 486–506. <a href="https://doi.org/10.1037/met0000075">https://doi.org/10.1037/met0000075</a>
</div>
<div id="ref-Manly2005" class="csl-entry">
Manly, J. J. (2005). Advantages and disadvantages of separate norms for <span>A</span>frican <span>A</span>mericans. <em>The Clinical Neuropsychologist</em>, <em>19</em>(2), 270–275. <a href="https://doi.org/10.1080/13854040590945346">https://doi.org/10.1080/13854040590945346</a>
</div>
<div id="ref-Manly2007" class="csl-entry">
Manly, J. J., &amp; Echemendia, R. J. (2007). Race-specific norms: Using the model of hypertension to understand issues of race, culture, and education in neuropsychology. <em>Archives of Clinical Neuropsychology</em>, <em>22</em>(3), 319–325. <a href="https://doi.org/10.1016/j.acn.2007.01.006">https://doi.org/10.1016/j.acn.2007.01.006</a>
</div>
<div id="ref-McClelland1973" class="csl-entry">
McClelland, D. C. (1973). Testing for competence rather than for <span>“intelligence.”</span> <em>American Psychologist</em>, <em>28</em>, 1–14. <a href="https://doi.org/10.1037/h0034092">https://doi.org/10.1037/h0034092</a>
</div>
<div id="ref-Meade2010" class="csl-entry">
Meade, A. W. (2010). A taxonomy of effect size measures for the differential functioning of items and scales. <em>Journal of Applied Psychology</em>, <em>95</em>(4), 728–743. <a href="https://doi.org/10.1037/a0018966">https://doi.org/10.1037/a0018966</a>
</div>
<div id="ref-Melikyan2019" class="csl-entry">
Melikyan, Z. A., Agranovich, A. V., &amp; Puente, A. E. (2019). Fairness in psychological testing. In G. Goldstein, D. N. Allen, &amp; J. DeLuca (Eds.), <em>Handbook of psychological assessment (fourth edition)</em> (pp. 551–572). Academic Press. <a href="https://doi.org/10.1016/B978-0-12-802203-0.00018-3">https://doi.org/10.1016/B978-0-12-802203-0.00018-3</a>
</div>
<div id="ref-Millsap2011" class="csl-entry">
Millsap, R. E. (2011). <em>Statistical approaches to measurement invariance</em>. Taylor &amp; Francis.
</div>
<div id="ref-Muthen2019" class="csl-entry">
Muthén, L. K., &amp; Muthén, B. O. (2019). <em>Mplus version 8.4</em>. Muthén &amp; Muthén.
</div>
<div id="ref-Nye2019" class="csl-entry">
Nye, C. D., Bradburn, J., Olenick, J., Bialko, C., &amp; Drasgow, F. (2019). How big are my effects? Examining the magnitude of effect sizes in studies of measurement equivalence. <em>Organizational Research Methods</em>, <em>22</em>(3), 678–709. <a href="https://doi.org/10.1177/1094428118761122">https://doi.org/10.1177/1094428118761122</a>
</div>
<div id="ref-Oberski2014" class="csl-entry">
Oberski, D. L. (2014). Evaluating sensitivity of parameters of interest to measurement invariance in latent variable models. <em>Political Analysis</em>, <em>22</em>(1), 45–60. <a href="https://doi.org/10.1093/pan/mpt014">https://doi.org/10.1093/pan/mpt014</a>
</div>
<div id="ref-Oberski2015" class="csl-entry">
Oberski, D. L., Vermunt, J. K., &amp; Moors, G. B. D. (2015). Evaluating measurement invariance in categorical data latent variable models with the EPC-interest. <em>Political Analysis</em>, <em>23</em>(4), 550–563. <a href="https://doi.org/10.1093/pan/mpv020">https://doi.org/10.1093/pan/mpv020</a>
</div>
<div id="ref-Paulus2020" class="csl-entry">
Paulus, J. K., &amp; Kent, D. M. (2020). Predictably unequal: Understanding and addressing concerns that algorithmic clinical prediction may increase health disparities. <em>Npj Digital Medicine</em>, <em>3</em>(1), 99. <a href="https://doi.org/10.1038/s41746-020-0304-9">https://doi.org/10.1038/s41746-020-0304-9</a>
</div>
<div id="ref-R-petersenlab" class="csl-entry">
Petersen, I. T. (2024b). <em><span class="nocase">petersenlab</span>: A collection of <span>R</span> functions by the <span>Petersen Lab</span></em>. <a href="https://doi.org/10.32614/CRAN.package.petersenlab">https://doi.org/10.32614/CRAN.package.petersenlab</a>
</div>
<div id="ref-Petersen2020" class="csl-entry">
Petersen, I. T., Choe, D. E., &amp; LeBeau, B. (2020). Studying a moving target in development: The challenge and opportunity of heterotypic continuity. <em>Developmental Review</em>, <em>58</em>, 100935. <a href="https://doi.org/10.1016/j.dr.2020.100935">https://doi.org/10.1016/j.dr.2020.100935</a>
</div>
<div id="ref-Putnick2016" class="csl-entry">
Putnick, D. L., &amp; Bornstein, M. H. (2016). Measurement invariance conventions and reporting: The state of the art and future directions for psychological research. <em>Developmental Review</em>, <em>41</em>, 71–90. <a href="https://doi.org/10.1016/j.dr.2016.06.004">https://doi.org/10.1016/j.dr.2016.06.004</a>
</div>
<div id="ref-Raykovinpress" class="csl-entry">
Raykov, T., Marcoulides, G. A., Harrison, M., &amp; Zhang, M. (2020). On the dependability of a popular procedure for studying measurement invariance: A cause for concern? <em>Structural Equation Modeling: A Multidisciplinary Journal</em>, <em>27</em>(4), 649–656. <a href="https://doi.org/10.1080/10705511.2019.1610409">https://doi.org/10.1080/10705511.2019.1610409</a>
</div>
<div id="ref-Reynolds2021" class="csl-entry">
Reynolds, C. R., Altmann, R. A., &amp; Allen, D. N. (2021). The problem of bias in psychological assessment. In C. R. Reynolds, R. A. Altmann, &amp; D. N. Allen (Eds.), <em>Mastering modern psychological testing: Theory and methods</em> (pp. 573–613). Springer International Publishing. <a href="https://doi.org/10.1007/978-3-030-59455-8_15">https://doi.org/10.1007/978-3-030-59455-8_15</a>
</div>
<div id="ref-Reynolds2012" class="csl-entry">
Reynolds, C. R., &amp; Suzuki, L. A. (2012). Bias in psychological assessment: An empirical review and recommendations. In I. B. Weiner, J. R. Graham, &amp; J. A. Naglieri (Eds.), <em>Handbook of psychology, <span>V</span>ol. 10: <span>A</span>ssessment psychology, <span>P</span>art 1: <span>A</span>ssessment issues</em> (2nd ed., pp. 82–113).
</div>
<div id="ref-R-mnlfa" class="csl-entry">
Robitzsch, A. (2019). <em><span class="nocase">mnlfa</span>: Moderated nonlinear factor analysis</em>. <a href="https://CRAN.R-project.org/package=mnlfa">https://CRAN.R-project.org/package=mnlfa</a>
</div>
<div id="ref-R-lavaan" class="csl-entry">
Rosseel, Y., Jorgensen, T. D., &amp; Rockwood, N. (2022). <em><span class="nocase">lavaan</span>: Latent variable analysis</em>. <a href="https://lavaan.ugent.be">https://lavaan.ugent.be</a>
</div>
<div id="ref-Sackett2008" class="csl-entry">
Sackett, P. R., Borneman, M. J., &amp; Connelly, B. S. (2008). High stakes testing in higher education and employment: Appraising the evidence for validity and fairness. <em>American Psychologist</em>, <em>63</em>, 215–227. <a href="https://doi.org/10.1037/0003-066X.63.4.215">https://doi.org/10.1037/0003-066X.63.4.215</a>
</div>
<div id="ref-Sackett2001" class="csl-entry">
Sackett, P. R., Schmitt, N., Ellingson, J. E., &amp; Kabin, M. B. (2001). High-stakes testing in employment, credentialing, and higher education. <em>American Psychologist</em>, <em>56</em>, 301–318. <a href="https://doi.org/10.1037/0003-066X.56.4.302">https://doi.org/10.1037/0003-066X.56.4.302</a>
</div>
<div id="ref-Sackett1994" class="csl-entry">
Sackett, P. R., &amp; Wilk, S. L. (1994). Within-group norming and other forms of score adjustment in preemployment testing. <em>American Psychologist</em>, <em>49</em>(11), 929–954. <a href="https://doi.org/10.1037/0003-066X.49.11.929">https://doi.org/10.1037/0003-066X.49.11.929</a>
</div>
<div id="ref-Schmidt1981" class="csl-entry">
Schmidt, F. L., &amp; Hunter, J. E. (1981). Employment testing: Old theories and new research findings. <em>American Psychologist</em>, <em>36</em>(10), 1128–1137. <a href="https://doi.org/10.1037/0003-066X.36.10.1128">https://doi.org/10.1037/0003-066X.36.10.1128</a>
</div>
<div id="ref-Silverberg2009" class="csl-entry">
Silverberg, N. D., &amp; Millis, S. R. (2009). Impairment versus deficiency in neuropsychological assessment: Implications for ecological validity. <em>Journal of the International Neuropsychological Society</em>, <em>15</em>(1), 94–102. <a href="https://doi.org/10.1017/S1355617708090139">https://doi.org/10.1017/S1355617708090139</a>
</div>
<div id="ref-Thorndike1971" class="csl-entry">
Thorndike, R. L. (1971). Concepts of culture-fairness. <em>Journal of Educational Measurement</em>, <em>8</em>(2), 63–70. <a href="https://doi.org/10.1111/j.1745-3984.1971.tb00907.x">https://doi.org/10.1111/j.1745-3984.1971.tb00907.x</a>
</div>
<div id="ref-VanDeSchoot2013" class="csl-entry">
Van De Schoot, R., Kluytmans, A., Tummers, L., Lugtig, P., Hox, J., &amp; Muthen, B. (2013). Facing off with scylla and charybdis: A comparison of scalar, partial, and the novel possibility of approximate measurement invariance. <em>Frontiers in Psychology</em>, <em>4</em>(770). <a href="https://doi.org/10.3389/fpsyg.2013.00770">https://doi.org/10.3389/fpsyg.2013.00770</a>
</div>
<div id="ref-VanDeSchoot2015" class="csl-entry">
Van De Schoot, R., Schmidt, P., De Beuckelaer, A., Lek, K., &amp; Zondervan-Zwijnenburg, M. (2015). Editorial: Measurement invariance. <em>Frontiers in Psychology</em>, <em>6</em>(1064). <a href="https://doi.org/10.3389/fpsyg.2015.01064">https://doi.org/10.3389/fpsyg.2015.01064</a>
</div>
<div id="ref-Wang2014a" class="csl-entry">
Wang, T., Merkle, E. C., &amp; Zeileis, A. (2014). Score-based tests of measurement invariance: Use in practice. <em>Frontiers in Psychology</em>, <em>5</em>. <a href="https://doi.org/10.3389/fpsyg.2014.00438">https://doi.org/10.3389/fpsyg.2014.00438</a>
</div>
<div id="ref-Wang2009b" class="csl-entry">
Wang, W.-C., Shih, C.-L., &amp; Yang, C.-C. (2009). The MIMIC method with scale purification for detecting differential item functioning. <em>Educational and Psychological Measurement</em>, <em>69</em>(5), 713–731. <a href="https://doi.org/10.1177/0013164409332228">https://doi.org/10.1177/0013164409332228</a>
</div>
<div id="ref-Youngstrom2016" class="csl-entry">
Youngstrom, E. A., &amp; Van Meter, A. (2016). Empirically supported assessment of children and adolescents. <em>Clinical Psychology: Science and Practice</em>, <em>23</em>(4), 327–347. <a href="https://doi.org/10.1111/cpsp.12172">https://doi.org/10.1111/cpsp.12172</a>
</div>
<div id="ref-Zieky2006" class="csl-entry">
Zieky, M. J. (2006). Fairness review in assessment. In S. M. Downing &amp; T. M. Haladyna (Eds.), <em>Handbook of test development</em> (pp. 359–376). Routledge. <a href="https://doi.org/10.4324/9780203874776.ch16">https://doi.org/10.4324/9780203874776.ch16</a>
</div>
<div id="ref-Zieky2013" class="csl-entry">
Zieky, M. J. (2013). Fairness review in assessment. In K. F. Geisinger, B. A. Bracken, J. F. Carlson, J.-I. C. Hansen, N. R. Kuncel, S. P. Reise, &amp; M. C. Rodriguez (Eds.), <em>APA handbook of testing and assessment in psychology, <span>V</span>ol. 1: <span>T</span>est theory and testing and assessment in industrial and organizational psychology</em> (pp. 293–302). American Psychological Association. <a href="https://doi.org/10.1037/14047-017">https://doi.org/10.1037/14047-017</a>
</div>
</div>
<div id="disqus_thread"></div>
<script>
    /**
    *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
    *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables    */
    /*
    var disqus_config = function () {
    this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
    this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
    };
    */
    (function() { // DON'T EDIT BELOW THIS LINE
    var d = document, s = d.createElement('script');
    s.src = 'https://isaactpetersen.disqus.com/embed.js';
    s.setAttribute('data-timestamp', +new Date());
    (d.head || d.body).appendChild(s);
    })();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
<h3>Feedback</h3>

Please consider providing feedback about this textbook, so that I can make it as helpful as possible. You can provide feedback at the following link:

<a href="https://forms.gle/95iW4p47cuaphTek6">https://forms.gle/95iW4p47cuaphTek6</a>
<script type="text/javascript" src="includes/external-links.js"></script>
            </section>

          </div>
        </div>
      </div>
<a href="intellectual.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="interview.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/isaactpetersen/Principles-Psychological-Assessment/edit/main/17-Test-Bias.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
